<!DOCTYPE html>
<html data-html-server-rendered="true" lang="en" data-vue-tag="%7B%22lang%22:%7B%22ssr%22:%22en%22%7D%7D">
  <head>
    <title> - Galaxy Community Hub</title><meta name="gridsome:hash" content="91d4a83f9855c194893a1f5e294057505b0cd33d"><meta data-vue-tag="ssr" charset="utf-8"><meta data-vue-tag="ssr" name="generator" content="Gridsome v0.7.23"><meta data-vue-tag="ssr" data-key="viewport" name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover"><meta data-vue-tag="ssr" data-key="format-detection" name="format-detection" content="telephone=no"><meta data-vue-tag="ssr" data-key="description" name="description" content="All about Galaxy and its community."><link data-vue-tag="ssr" rel="icon" href="data:,"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="16x16" href="/assets/static/favicon.ce0531f.7d4a227415c9d9b704d56a079d49a3e1.png"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="32x32" href="/assets/static/favicon.ac8d93a.7d4a227415c9d9b704d56a079d49a3e1.png"><link rel="preload" href="/assets/css/0.styles.e1e25242.css" as="style"><link rel="preload" href="/assets/js/app.0690a31c.js" as="script"><link rel="preload" href="/assets/js/page--src--templates--article-vue.6d736ff4.js" as="script"><link rel="prefetch" href="/assets/js/page--node-modules--gridsome--app--pages--404-vue.9b913011.js"><link rel="prefetch" href="/assets/js/page--src--pages--blog-vue.c32e3571.js"><link rel="prefetch" href="/assets/js/page--src--pages--careers-vue.85d2fb31.js"><link rel="prefetch" href="/assets/js/page--src--pages--data-explorer-vue.8aa3e295.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--archive-vue.1426fe7f.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests--papercuts-vue.ab928745.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests-vue.1e8420fc.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--webinars-vue.5d6fb146.js"><link rel="prefetch" href="/assets/js/page--src--pages--events-vue.cd4ed3c8.js"><link rel="prefetch" href="/assets/js/page--src--pages--index-vue.a143f0a2.js"><link rel="prefetch" href="/assets/js/page--src--pages--news-vue.2f002266.js"><link rel="prefetch" href="/assets/js/page--src--pages--search-vue.23e742ea.js"><link rel="prefetch" href="/assets/js/page--src--pages--tests-vue.3b995c11.js"><link rel="prefetch" href="/assets/js/page--src--pages--use-vue.7c5b8683.js"><link rel="prefetch" href="/assets/js/page--src--templates--platform-vue.91145b17.js"><link rel="prefetch" href="/assets/js/page--src--templates--vue-article-vue.d9a0c32e.js"><link rel="prefetch" href="/assets/js/vendors~page--src--pages--careers-vue~page--src--pages--use-vue~page--src--templates--article-vue~pa~efade918.92a3a9c1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--index-md.b8e1f92a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--md--imgtests--imgdir--index-md.5311083e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--partials--index-md.b6884022.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--imgdir--index-md.1ee4281c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--index-md.fe03c46f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--index-md.05d8aec6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--index-md.e7b48256.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--reldir--index-md.4093d68b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--capacity-planning--index-md.8813525b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--index-md.aafe943e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--performance--cluster--legacy--index-md.3da8719f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--datatypes--index-md.9b672175.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--disk-quotas--index-md.44f752e9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--faq--index-md.3851dd50.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--data-model--index-md.6d581759.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--galaxy-ops--index-md.a3b0d0af.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--ten-simple-steps-galaxy-as-a-service--index-md.699b2e0d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--tool-config-syntax--index-md.208f876a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--writing-tests--index-md.b2caa913.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--allocation--index-md.de3e7728.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--faq--index-md.ac3b79f2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--index-md.328857f9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--ssh--index-md.d92fc074.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--troubleshooting--index-md.7c074dc2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--capacity-planning--index-md.01a397fa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--getting-started--index-md.acdcb61b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--index-md.824830b5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--building--index-md.10fb8489.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--capacity-planning--index-md.e2d98e61.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--cluster-types--index-md.df053139.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--customizing--index-md.fb1f931d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--faq--index-md.b2fab4e5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--index-md.d71b41b2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--pre-201703--index-md.f5a4c4d0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--hadoop--index-md.d7a6d969.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--ht-condor--index-md.31bcf5d4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--index-md.b7b22e1d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--openstack--index-md.d78cb425.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--services--index-md.f2027d09.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--sharing--index-md.cd3d2d6f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--troubleshooting--index-md.fc1a0be3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--userdata--index-md.e4d95203.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--future--index-md.e75566ea.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--index-md.a767ff80.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-09--index-md.66c1654b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-27--index-md.c6d537c1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-09-19--index-md.544a522a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-11-14--index-md.4746a5d0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-01-16--index-md.d1551883.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-03-20--index-md.5094871c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-05-15--index-md.1897abfc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-11-20--index-md.c4d4ced5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-02-19--index-md.311ef2ba.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-04-16--index-md.0b281218.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-06-18--index-md.7c255863.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-08-20--index-md.43ba73c2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-10-15--index-md.bc11b58b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-02-18--index-md.1a182ba1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-04-21--index-md.5e9eea61.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-06-16--index-md.b1295236.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-08-18--index-md.d8fbe7ba.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--index-md.43055631.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--webinar-tech--index-md.a44b91e1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2012--index-md.c7a0a096.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2014--index-md.46c5bc59.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--index-md.34247339.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--data--index-md.46e24851.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--datafiles--index-md.77545970.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--architecture--index-md.55f07291.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--bitbucket--index-md.718bcb6c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--css--index-md.281229bf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--getting-started--index-md.497082a2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--index-md.392b2e76.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--jsa--index-md.ba998e94.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--mercurial--index-md.14a2dad8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--security-tool-tips--index-md.17796bc0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--galaxy-project--index-md.0de41023.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--index-md.d89e2857.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--papers--index-md.4bdf7b3e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--posters--index-md.7e825096.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--presentations--index-md.084c96b9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--videos--index-md.c984f7ce.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--advanced-session--index-md.191fe1da.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--basics-session--index-md.35c32703.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--index-md.8acf4593.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--logistics--index-md.37d0ac9b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--publicity--index-md.7f2eec38.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--registration--index-md.d22dcedd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--abstracts--index-md.bbbf6ae8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--index-md.cb62d683.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--key-dates--index-md.07f8d6b1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--lightning-talks--index-md.72ab128d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--logistics--index-md.c3eeb3dd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--organizing-committee--index-md.f7a53fcd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--photos--index-md.d5102197.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--automation-strategies--index-md.6972b31d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--bioinformatics-training--index-md.648f14ff.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--end-users--index-md.ff776d4f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--workflows-and-api--index-md.540ad6ba.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--index-md.d233185c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--promotion--index-md.212eb34b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--register--index-md.b4c4f57b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--sponsorships--index-md.d0539386.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--training-day--index-md.2ee4bcd3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--index-md.1aa46f9f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--posters--index-md.a8b0ee54.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--talks--index-md.b3ee75dc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--clinical-use-hpc-file-systems--index-md.3fc5e2ed.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-admins--index-md.2f878aaa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-blast--index-md.5305db66.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-professionals--index-md.dcbf6370.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--index-md.3abd720e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--proteomics--index-md.f4d7daaf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--public-galaxy-servers--index-md.b346729f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--tool-developers--index-md.4365e2e3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--index-md.d58334f2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--key-dates--index-md.a3f7c6b8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--lightning--index-md.180bd3c7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--logistics--index-md.c889273c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--organizers--index-md.2f551bcb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--photos--index-md.38ee0e51.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--program--index-md.f2c70270.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--promotion--index-md.1e5aadf9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--register--index-md.99382e6b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--sponsorships--index-md.1e82df1d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--advance-tool-data--index-md.02e350f8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--api--index-md.23e9c64f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--index-md.7b6606de.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--intro-tool-data--index-md.1893d923.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--vms--index-md.eb7d2783.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--index-md.19ebb120.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--posters--index-md.19e77da9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--talks--index-md.69b1844f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--branch-release-merge--index-md.e83f529b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-admins--index-md.4c5b1df3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-end-users--index-md.6b3ff393.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-training-network--index-md.85db1b6f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--heterogeneous--index-md.26797010.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--index-md.ab231746.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--using-galaxy-ui-overview--index-md.3f0318f2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--hackathon--index-md.f1b7f383.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--index-md.894a1ae1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--key-dates--index-md.d3d9965a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--lightning--index-md.268064be.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--logistics--index-md.033eda0f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--organizers--index-md.73259cdb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--photos--index-md.3ee7fbac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--program--index-md.79995b28.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--promotion--index-md.59d9587a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--register--index-md.6cf374d9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--sponsor-exhibit--index-md.2aa46fae.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--admin-walkthrough--index-md.244e6bc0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--api--index-md.3aefd41a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--data-managers--index-md.ea9c133f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--index-md.1c4f2cb1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--vms--index-md.3a12839f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--cwl--index-md.29f1b923.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-management-challenges--index-md.5d4d1ee7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-splitting-and-parallelism--index-md.31c7354c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--elixir--index-md.e49743fc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--extended-workflow--index-md.ca7910bb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--fots--index-md.eea6d2b7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-admins--index-md.8df0d322.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-community-uk--index-md.69eb15c2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-scientists--index-md.3dd577a5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-training-network--index-md.d54633c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--index-md.d03261e3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--data-hack-roundup--index-md.581fdf4b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--header--index-md.72afdfea.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--index-md.adc3dfb6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2016--bofs--index-md.e539350a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--abstracts--index-md.7322e906.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--childcare--index-md.416182d5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--cofest--index-md.ab0c9286.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--conduct--index-md.e979f793.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--faq--index-md.e8fa4530.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--index-md.876ea583.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--organizers--index-md.23f78b02.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--promotion--index-md.85444af0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--registration--index-md.5ed4fbf8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--sponsors--index-md.482f848f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--training--index-md.0709a7f7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--venue--index-md.238012f6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--abstracts--index-md.dc64d26a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--childcare--index-md.234bf37e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--code-of-conduct--index-md.3a181b70.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--collaboration--index-md.003698d8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--faq--index-md.6b9a5e1b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--icebreaker--index-md.bd0460e3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--index-md.139334ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--keynotes--index-md.add6f018.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--organizers--index-md.8f9602b8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--promotion--index-md.55ce8ac1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--register--index-md.0b3372c2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--sponsors--index-md.4546b9bb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--training--index-md.9dbe3953.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--venue--index-md.1408cfc7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2014--index-md.4533e9c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2016--index-md.da0aae7f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2014--index-md.38dcd568.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2016--index-md.80c32555.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--index-md.176fc9a2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--trainingday--index-md.774b7249.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2016--index-md.c5ea9a89.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--example--vue-remark--index-md.9e4f4358.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--faqs--index-md.a910449f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-06--index-md.3a1cff95.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-07--index-md.bb5876b9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--icons--index-md.9a000132.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--index-md.3b38f73f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--logos--index-md.aa44aae7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--news-graphics--index-md.05ad5cba.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--photos--index-md.1d8f64ab.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--screenshots--index-md.a201402f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--index-md.d08ab1a3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--workflow-editor-unannotated--index-md.e72b5147.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--extract--index-md.c7671571.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--tool-panel--index-md.ac20782a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--variables--index-md.3e2afdd3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--faq--index-md.09b46d8e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--galaxy-pages--index-md.ff01861b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--interval-operations--index-md.e1c040b2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--main--faq--index-md.0e15bbd0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--tools--requirements--index-md.d2018317.js"><link rel="stylesheet" href="/assets/css/0.styles.e1e25242.css"><noscript data-vue-tag="ssr"><style>.g-image--loading{display:none;}</style></noscript>
  </head>
  <body >
    <div data-server-rendered="true" id="app" class="layout" data-v-340f7abc><header id="masthead"><nav class="navbar container justify-content-center navbar-dark bg-transparent navbar-expand-lg"><a href="/" target="_self" class="navbar-brand router-link-active"><img id="masthead-logo" src="/images/galaxy_logo_hub_white.svg" alt="Galaxy Community Hub" height="30"></a><button type="button" aria-label="Toggle navigation" class="navbar-toggler"><span class="navbar-toggler-icon"></span></button><div id="nav-collapse" class="navbar-collapse collapse" style="display:none;"><ul class="navbar-nav"><li class="nav-item"><a href="/use/" target="_self" class="nav-link">Use</a></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Learn</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/learn/" role="menuitem" target="_self" class="dropdown-item">How to use Galaxy</a></li><li role="presentation"><a role="menuitem" href="https://training.galaxyproject.org/" target="_self" class="dropdown-item">Teach with Galaxy</a></li></ul></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Community</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/community/" role="menuitem" target="_self" class="dropdown-item">About</a></li><li role="presentation"><a href="/events/" role="menuitem" target="_self" class="dropdown-item">Events</a></li><li role="presentation"><a href="/news/" role="menuitem" target="_self" class="dropdown-item">News</a></li><li role="presentation"><a href="/blog/" role="menuitem" target="_self" class="dropdown-item">Blog</a></li><li role="presentation"><a href="/publication-library/" role="menuitem" target="_self" class="dropdown-item">Publications</a></li><li role="presentation"><a href="/galaxy-project/statistics/" role="menuitem" target="_self" class="dropdown-item">Stats</a></li><li role="presentation"><a href="/careers/" role="menuitem" target="_self" class="dropdown-item">Careers</a></li></ul></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Deploy &amp; Develop</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/admin/get-galaxy/" role="menuitem" target="_self" class="dropdown-item">Install Galaxy</a></li><li role="presentation"><a href="/admin/" role="menuitem" target="_self" class="dropdown-item">Server administration</a></li><li role="presentation"><a href="/tools/" role="menuitem" target="_self" class="dropdown-item">Tools</a></li><li role="presentation"><a href="/develop/api/" role="menuitem" target="_self" class="dropdown-item">Working with the API</a></li><li role="presentation"><a href="/develop/" role="menuitem" target="_self" class="dropdown-item">Contributing</a></li><li role="presentation"><a href="/docs/" role="menuitem" target="_self" class="dropdown-item">Code documentation</a></li></ul></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Support</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/support/" role="menuitem" target="_self" class="dropdown-item">FAQ</a></li><li role="presentation"><a role="menuitem" href="https://help.galaxyproject.org/" target="_self" class="dropdown-item">Galaxy Help Forum</a></li></ul></li><li class="nav-item"><a href="/jxtx/" target="_self" class="nav-link">@jxtx</a></li></ul><ul class="navbar-nav ml-auto"><li class="form-inline"><form action="/search/" method="get" class="form-inline"><input id="search-input" name="q" type="text" placeholder="Search Galaxy" value="" class="mr-sm-2 form-control form-control-sm"><button type="submit" class="btn my-2 my-sm-0 btn-secondary btn-sm"><span class="fa fa-search"></span></button></form></li></ul></div></nav></header><section id="maincontainer" class="container"><header data-v-1944b8d1 data-v-340f7abc><a href="/" class="link active" data-v-1944b8d1>← Back to Home</a><!----><h1 class="title" data-v-1944b8d1></h1><!----><section class="metadata freetext" data-v-1944b8d1><!----><!----><!----></section><!----><!----></header><div class="content markdown" data-v-340f7abc><div class="toc-wrapper col-md-3">
<ul>
<li><a href="#tools">Tools</a></li>
<li><a href="#dependency-definitions">Dependency Definitions</a></li>
</ul>
</div>
<div class="body-wrapper col-md-9">
<ul>
<li><a href="/toolshed/">Wiki Root</a></li>
<li><a href="/toolshed/contributions/">All monthly summaries</a></li>
</ul>
<h1 id="tools"><a href="#tools" aria-hidden="true"><span class="icon icon-link"></span></a>Tools</h1>
<ul>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/nathandunn" target="_blank" rel="noopener noreferrer">nathandunn</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/nathandunn/monarchinitiative" target="_blank" rel="noopener noreferrer">monarchinitiative</a>: Enriches Related biological data using ontology driven data. Tool Suite that Pulls related Variants, Phenotypes, Diseases, Genes, and Homologues as TSV or phenopackets from the monarch initiative GOLR server  '<a href="https://monarchinitiative.org" target="_blank" rel="noopener noreferrer">https://monarchinitiative.org</a>'</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/yguitton" target="_blank" rel="noopener noreferrer">yguitton</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/yguitton/metams_rungc" target="_blank" rel="noopener noreferrer">metams_rungc</a>: [W4M][GC-MS] metaMS R Package - GC-MS data preprocessing using metaMS package. Part of the W4M project: <a href="http://workflow4metabolomics.org" target="_blank" rel="noopener noreferrer">http://workflow4metabolomics.org</a> metaMS: <a href="https://www.bioconductor.org/packages/release/bioc/html/metaMS.html" target="_blank" rel="noopener noreferrer">https://www.bioconductor.org/packages/release/bioc/html/metaMS.html</a> MS-based metabolomics data processing and compound annotation pipeline.</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/mmonsoor" target="_blank" rel="noopener noreferrer">mmonsoor</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/mmonsoor/probmetab" target="_blank" rel="noopener noreferrer">probmetab</a>: [W4M][LC-MS] ProbMetab - automatic probabilistic LC-MS based metabolome annotation. Part of the W4M project: <a href="http://workflow4metabolomics.org" target="_blank" rel="noopener noreferrer">http://workflow4metabolomics.org</a>. ProbMetab, an R package that promotes substantial improvement in automatic probabilistic liquid chromatography-mass spectrometry-based metabolome annotation. ProbMetab is developped and maintained by Silva et al. - <a href="http://labpib.fmrp.usp.br/methods/probmetab/" target="_blank" rel="noopener noreferrer">http://labpib.fmrp.usp.br/methods/probmetab/</a></li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/stemcellcommons" target="_blank" rel="noopener noreferrer">stemcellcommons</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/stemcellcommons/rnaseq_workflows" target="_blank" rel="noopener noreferrer">rnaseq_workflows</a>: RNA-seq workflows annotated for use with Refinery Platform</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/rmarenco" target="_blank" rel="noopener noreferrer">rmarenco</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/rmarenco/regtools_junctions_extract" target="_blank" rel="noopener noreferrer">regtools_junctions_extract</a>: Wrapper for the regtools junctions extract program Report splice junctions in RNA-Seq BAM file</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/rmarenco/hubarchivecreator" target="_blank" rel="noopener noreferrer">hubarchivecreator</a>: A tool to create a UCSC track hub</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/rmarenco/psltobed" target="_blank" rel="noopener noreferrer">psltobed</a>: tranform a psl format file to a bed format file</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/rmarenco/bedclip" target="_blank" rel="noopener noreferrer">bedclip</a>: upload for repository <a href="https://github.com/remimarenco/bedclip" target="_blank" rel="noopener noreferrer">https://github.com/remimarenco/bedclip</a> Remove lines from bed file that refer to off-chromosome places</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/rmarenco/blastxmltopsl" target="_blank" rel="noopener noreferrer">blastxmltopsl</a>: upload for repository <a href="https://github.com/remimarenco/blastXmlToPsl" target="_blank" rel="noopener noreferrer">https://github.com/remimarenco/blastXmlToPsl</a>. Convert blast XML output to PSLs</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/rmarenco/trfbig" target="_blank" rel="noopener noreferrer">trfbig</a>: upload for repository <a href="https://github.com/remimarenco/trfBig" target="_blank" rel="noopener noreferrer">https://github.com/remimarenco/trfBig</a> Mask tandem repeats on a big sequence file</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/rmarenco/bamtobigwig" target="_blank" rel="noopener noreferrer">bamtobigwig</a>: Convert Bam file to BigWig</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/rnateam" target="_blank" rel="noopener noreferrer">rnateam</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/rnateam/rnacommender" target="_blank" rel="noopener noreferrer">rnacommender</a>: upload for repository <a href="https://github.com/bgruening/galaxytools/tree/rna_commander/tools/rna_tools/rna_commender" target="_blank" rel="noopener noreferrer">https://github.com/bgruening/galaxytools/tree/rna_commander/tools/rna_tools/rna_commender</a> RNAcommender is a tool for genome-wide recommendation of RNA-protein interactions. RNAcommender is a tool for genome-wide recommendation of RNA-protein interactions. It is a recommender system capable of suggesting RNA targets to unexplored RNA binding proteins, by propagating the available interaction information, taking into account the protein domain composition and the RNA predicted secondary structure.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/rnateam/ribotaper" target="_blank" rel="noopener noreferrer">ribotaper</a>: upload for repository <a href="https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/ribotaper/" target="_blank" rel="noopener noreferrer">https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/ribotaper/</a> A method for defining traslated ORFs using Ribosome Profiling data. RiboTaper is a new analysis pipeline for Ribosome Profiling (Ribo-seq) experiments, which exploits the triplet periodicity of ribosomal footprints to call translated regions.</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/pmac" target="_blank" rel="noopener noreferrer">pmac</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/pmac/robustzscorenormalization" target="_blank" rel="noopener noreferrer">robustzscorenormalization</a>: This program takes a linear plate table and a plate config file as input and then performs Robust Z Score Normalization on the data Takes Cellomics High Content - Imaging data and performs Robust Z Score Normalization.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/pmac/iterativepca" target="_blank" rel="noopener noreferrer">iterativepca</a>: Iterative PCA on genotype data Perform PCA on input variant data, to filter out outliers. Repeat this process, but with the outliers removed from the input data, until no more outliers are removed.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/pmac/reformatplatestabulartolinear" target="_blank" rel="noopener noreferrer">reformatplatestabulartolinear</a>: This program converts a plates table from a tabular format to a linear format Converts a tabular plates table used in Functional Genomics and converts it back to a linear format.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/pmac/calculatezprimefactor" target="_blank" rel="noopener noreferrer">calculatezprimefactor</a>: This program takes a linear plate table and a plate config file and then calculates the Z Prime Factors for your data Designed for use in functional genomics. Takes output from a Cellomics High Content - Imaging Screen and calculates the Z Prime Factors. Requires R to be installed</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/pmac/normalizemediannegcntrl" target="_blank" rel="noopener noreferrer">normalizemediannegcntrl</a>: This program takes a linear plate table and a plate config file as input and then normalizes each plate in the table to the negative controls in the config This tool takes Cellomics High Content - Imaging data and normalizes each plate using a config file.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/pmac/reformatplateslineartotabular" target="_blank" rel="noopener noreferrer">reformatplateslineartotabular</a>: This program converts a linear plates file to a tabular format Converts a linear plates file used in Functional Genomics to a tabular format.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/pmac/map_chromosomes" target="_blank" rel="noopener noreferrer">map_chromosomes</a>: Convert between UCSC and ensembl chromosome names Converts a column in a table from UCSC to ensembl chromosome representation and vice versa.</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/bebatut" target="_blank" rel="noopener noreferrer">bebatut</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/bebatut/metaphlan2" target="_blank" rel="noopener noreferrer">metaphlan2</a>: upload for repository <a href="https://github.com/ASaiM/galaxytools/tree/master/tools/metaphlan2/" target="_blank" rel="noopener noreferrer">https://github.com/ASaiM/galaxytools/tree/master/tools/metaphlan2/</a> MetaPhlAn is a computational tool for profiling the composition of microbial communities (Bacteria, Archaea, Eukaryotes and Viruses) from metagenomic shotgun sequencing data with species level resolution</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/bgruening" target="_blank" rel="noopener noreferrer">bgruening</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/bgruening/autodock_vina" target="_blank" rel="noopener noreferrer">autodock_vina</a>: upload for repository <a href="https://github.com/leobiscassi/galaxytools/tree/autodock_vina_tools/chemicaltoolbox/autodock_vina" target="_blank" rel="noopener noreferrer">https://github.com/leobiscassi/galaxytools/tree/autodock_vina_tools/chemicaltoolbox/autodock_vina</a> Software for docking protein-compound. AutoDock Vina is an open-source program for doing molecular docking.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/bgruening/autodock_vina_prepare_receptor" target="_blank" rel="noopener noreferrer">autodock_vina_prepare_receptor</a>: upload for repository <a href="https://github.com/bgruening/galaxytools/tree/master/chemicaltoolbox/autodock_vina" target="_blank" rel="noopener noreferrer">https://github.com/bgruening/galaxytools/tree/master/chemicaltoolbox/autodock_vina</a> Software for protein-compound docking - the prepare receptor step AutoDock Vina is an open-source program for doing molecular docking.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/bgruening/autodock_vina_prepare_ligand" target="_blank" rel="noopener noreferrer">autodock_vina_prepare_ligand</a>: upload for repository <a href="https://github.com/bgruening/galaxytools/tree/master/chemicaltoolbox/autodock_vina" target="_blank" rel="noopener noreferrer">https://github.com/bgruening/galaxytools/tree/master/chemicaltoolbox/autodock_vina</a> Software for protein-compound docking - the prepare ligand step AutoDock Vina is an open-source program for doing molecular docking.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/bgruening/lighter" target="_blank" rel="noopener noreferrer">lighter</a>:  upload for repository <a href="https://github.com/mourisl/Lighter" target="_blank" rel="noopener noreferrer">https://github.com/mourisl/Lighter</a>  Lighter is a kmer-based error correction method for whole genome sequencing data Lighter is a kmer-based error correction method for whole genome sequencing data</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/bgruening/circexplorer" target="_blank" rel="noopener noreferrer">circexplorer</a>: upload for repository <a href="https://github.com/bgruening/galaxytools/tree/master/tools/circexplorer" target="_blank" rel="noopener noreferrer">https://github.com/bgruening/galaxytools/tree/master/tools/circexplorer</a> A combined strategy to identify circular RNAs (circRNAs and ciRNAs) A combined strategy to identify circular RNAs (circRNAs and ciRNAs) (Zhang et al., Complementary Sequence-Mediated Exon Circularization, Cell (2014), 159:134-147)</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/bgruening/glimmer_hmm" target="_blank" rel="noopener noreferrer">glimmer_hmm</a>: planemo upload for repository <a href="https://github.com/bgruening/galaxytools/tree/master/tools/glimmer_hmm" target="_blank" rel="noopener noreferrer">https://github.com/bgruening/galaxytools/tree/master/tools/glimmer_hmm</a> GlimmerHMM is a new gene finder based on a Generalized Hidden Markov Model (GHMM) GlimmerHMM is a new gene finder based on a Generalized Hidden Markov Model (GHMM). Although the gene finder conforms to the overall mathematical framework of a GHMM, additionally it incorporates splice site models adapted from the GeneSplicer program and a decision tree adapted from GlimmerM. It also utilizes Interpolated Markov Models for the coding and noncoding models . Currently, GlimmerHMM's GHMM structure includes introns of each phase, intergenic regions, and four types of exons (initial, internal, final, and single).</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/mvdbeek" target="_blank" rel="noopener noreferrer">mvdbeek</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/mvdbeek/docker_scriptrunner" target="_blank" rel="noopener noreferrer">docker_scriptrunner</a>: planemo upload for repository <a href="https://github.com/mvdbeek/docker_scriptrunner/" target="_blank" rel="noopener noreferrer">https://github.com/mvdbeek/docker_scriptrunner/</a> docker scriptrunner allows script execution in galaxy docker scriptrunner allows the execution of arbitrary scripts in a container, shielding the system from malicious scripts. First read the README before installing this tools, since it may run in a insecure manner.</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/chrisd" target="_blank" rel="noopener noreferrer">chrisd</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/chrisd/metasnp" target="_blank" rel="noopener noreferrer">metasnp</a>: planemo upload for repository <a href="https://github.com/cdeanj/galaxytools/tree/master/tools/metasnp" target="_blank" rel="noopener noreferrer">https://github.com/cdeanj/galaxytools/tree/master/tools/metasnp</a> A simple naive metagenomics variant caller</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/prog" target="_blank" rel="noopener noreferrer">prog</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/prog/lcmsmatching" target="_blank" rel="noopener noreferrer">lcmsmatching</a>: planemo upload for repository <a href="https://github.com/workflow4metabolomics/lcmsmatching.git" target="_blank" rel="noopener noreferrer">https://github.com/workflow4metabolomics/lcmsmatching.git</a> [Metabolomics][W4M][LC-MS] Annotate LCMS spectrum using an in-house spectra database. Part of the W4M project (<a href="http://workflow4metabolomics.org" target="_blank" rel="noopener noreferrer">http://workflow4metabolomics.org</a>). Annotate LCMS spectrum using an in-house spectra database, provided as a TSV file. Two algorithms are proposed, a simple matching on MZ and RT values, and a more complex one that makes sure for each candidate that its precursor peak is matched.</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp" target="_blank" rel="noopener noreferrer">galaxyp</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/mz_to_sqlite" target="_blank" rel="noopener noreferrer">mz_to_sqlite</a>: planemo upload for repository <a href="https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/mz_to_sqlite" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/mz_to_sqlite</a> Creates a SQLite database for proteomics data Extract proteomics files: mzIdentML, mzML, MGF, fasta, uniprot_xml into a SQLite DB</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/gcc2016_tutorial" target="_blank" rel="noopener noreferrer">gcc2016_tutorial</a>: planemo upload for repository <a href="https://github.com/galaxyproteomics/tools-galaxyp/tree/master/workflows/gcc2016%5C_tutorial" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproteomics/tools-galaxyp/tree/master/workflows/gcc2016\_tutorial</a>  GCC2016 Tutorial Workflows Workflows for PeptideShaker and NOVEL peptides analysis</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/iuc" target="_blank" rel="noopener noreferrer">iuc</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/stacks_sstacks" target="_blank" rel="noopener noreferrer">stacks_sstacks</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks</a>  Stacks: sstacks (from the Stacks tool suite) Stacks is a software pipeline for building loci from short-read sequences, such as those generated on the Illumina platform. Stacks was developed to work with restriction enzyme-based data, such as RAD-seq, for the purpose of building genetic maps and conducting population genomics and phylogeography. <a href="http://catchenlab.life.illinois.edu/stacks/" target="_blank" rel="noopener noreferrer">http://catchenlab.life.illinois.edu/stacks/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/stacks_assembleperead" target="_blank" rel="noopener noreferrer">stacks_assembleperead</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks</a>  Stacks: assemble read pairs by locus (from the Stacks tool suite)</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/stacks_rxstacks" target="_blank" rel="noopener noreferrer">stacks_rxstacks</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks</a>  Stacks: rxstacks (from the Stacks tool suite)</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/stacks_pstacks" target="_blank" rel="noopener noreferrer">stacks_pstacks</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks</a>  Stacks: pstacks (from the Stacks tool suite)</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/stacks_refmap" target="_blank" rel="noopener noreferrer">stacks_refmap</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks</a>  Stacks: reference map (from the Stacks tool suite)</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/stacks_refmap" target="_blank" rel="noopener noreferrer">stacks_refmap</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks</a>  Stacks: reference map (from the Stacks tool suite)</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/stacks_refmap" target="_blank" rel="noopener noreferrer">stacks_refmap</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks</a>  Stacks: reference map (from the Stacks tool suite)</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/stacks_populations" target="_blank" rel="noopener noreferrer">stacks_populations</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks</a>  Stacks: populations (from the Stacks tool suite)</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/suite_stacks" target="_blank" rel="noopener noreferrer">suite_stacks</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks</a>  Stacks tools for building loci from short-read sequences, such as RAD-seq</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/stacks_ustacks" target="_blank" rel="noopener noreferrer">stacks_ustacks</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks</a>  Stacks: ustacks (from the Stacks tool suite)</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/stacks_procrad" target="_blank" rel="noopener noreferrer">stacks_procrad</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks</a>  Stacks: process radtags (from the Stacks tool suite)</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/stacks_genotypes" target="_blank" rel="noopener noreferrer">stacks_genotypes</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks</a>  Stacks: genotypes (from the Stacks tool suite)</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_concat" target="_blank" rel="noopener noreferrer">bcftools_concat</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a> Wrapper for bcftools application bcftools concat. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_view" target="_blank" rel="noopener noreferrer">bcftools_view</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a> Wrapper for bcftools application bcftools view. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_roh" target="_blank" rel="noopener noreferrer">bcftools_roh</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools roh. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_plugin_mendelian" target="_blank" rel="noopener noreferrer">bcftools_plugin_mendelian</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools mendelian. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_annotate" target="_blank" rel="noopener noreferrer">bcftools_annotate</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a> Wrapper for bcftools application bcftools annotate. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_plugin_fill_tags" target="_blank" rel="noopener noreferrer">bcftools_plugin_fill_tags</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools fill-tags. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_plugin_fill_an_ac" target="_blank" rel="noopener noreferrer">bcftools_plugin_fill_an_ac</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools fill-AN-AC. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/biom_add_metadata" target="_blank" rel="noopener noreferrer">biom_add_metadata</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/biom_format" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/biom_format</a>  Wrapper for the biom-format CLI tool suite: BIOM metadata The biom-format package provides a command line interface and Python API for working with BIOM files.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/ncbi_eutils_efetch" target="_blank" rel="noopener noreferrer">ncbi_eutils_efetch</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_entrez_eutils" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_entrez_eutils</a>  Wrapper for NCBI application NCBI EFetch Entrez E-Utils provide an interface to the NCBI databases to allow querying and downloading datasets from NCBI. This repo requires that administrators should very aware of some restrictions NCBI places on the use of the Entrez service.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_plugin_impute_info" target="_blank" rel="noopener noreferrer">bcftools_plugin_impute_info</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools impute-info. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/ncbi_eutils_esearch" target="_blank" rel="noopener noreferrer">ncbi_eutils_esearch</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_entrez_eutils" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_entrez_eutils</a>  Wrapper for NCBI application NCBI ESearch Entrez E-Utils provide an interface to the NCBI databases to allow querying and downloading datasets from NCBI.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/biom_convert" target="_blank" rel="noopener noreferrer">biom_convert</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/biom_format" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/biom_format</a> Wrapper for the biom-format CLI tool suite: Convert BIOM The biom-format package provides a command line interface and Python API for working with BIOM files.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/hmmer_hmmemit" target="_blank" rel="noopener noreferrer">hmmer_hmmemit</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3</a>  Wrapper for hmmer3 application hmmemit HMMER is used for searching sequence databases for homologs of protein sequences, and for making protein sequence alignments. It implements methods using probabilistic models called profile hidden Markov models (profile HMMs). <a href="http://hmmer.janelia.org/" target="_blank" rel="noopener noreferrer">http://hmmer.janelia.org/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/hmmer_hmmconvert" target="_blank" rel="noopener noreferrer">hmmer_hmmconvert</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3</a>  Wrapper for hmmer3 application hmmconvert</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/hmmer_nhmmscan" target="_blank" rel="noopener noreferrer">hmmer_nhmmscan</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3</a>  Wrapper for hmmer3 application nhmmscan</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/hmmer_hmmalign" target="_blank" rel="noopener noreferrer">hmmer_hmmalign</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3</a> Wrapper for hmmer3 application hmmalign</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/hmmer_hmmfetch" target="_blank" rel="noopener noreferrer">hmmer_hmmfetch</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3</a>  Wrapper for hmmer3 application hmmfetch</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/hmmer_hmmbuild" target="_blank" rel="noopener noreferrer">hmmer_hmmbuild</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3</a>  Wrapper for hmmer3 application hmmbuild</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/hmmer_nhmmer" target="_blank" rel="noopener noreferrer">hmmer_nhmmer</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3</a>  Wrapper for hmmer3 application nhmmer</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/hmmer_phmmer" target="_blank" rel="noopener noreferrer">hmmer_phmmer</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3</a>  Wrapper for hmmer3 application phmmer</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/hmmer_alimask" target="_blank" rel="noopener noreferrer">hmmer_alimask</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3</a>  Wrapper for hmmer3 application alimask</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/hmmer_hmmsearch" target="_blank" rel="noopener noreferrer">hmmer_hmmsearch</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3</a>  Wrapper for hmmer3 application hmmsearch</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/hmmer_jackhmmer" target="_blank" rel="noopener noreferrer">hmmer_jackhmmer</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3</a>  Wrapper for hmmer3 application jackhmmer</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/hmmer_hmmscan" target="_blank" rel="noopener noreferrer">hmmer_hmmscan</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3</a>  Wrapper for hmmer3 application hmmscan</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_plugin_missing2ref" target="_blank" rel="noopener noreferrer">bcftools_plugin_missing2ref</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools missing2ref. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_plugin_tag2tag" target="_blank" rel="noopener noreferrer">bcftools_plugin_tag2tag</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools tag2tag. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_plugin_setgt" target="_blank" rel="noopener noreferrer">bcftools_plugin_setgt</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools setGT. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/ncbi_eutils_ecitmatch" target="_blank" rel="noopener noreferrer">ncbi_eutils_ecitmatch</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_entrez_eutils" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_entrez_eutils</a>  Wrapper for NCBI application NCBI ECitMatch Entrez E-Utils provide an interface to the NCBI databases to allow querying and downloading datasets from NCBI</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_stats" target="_blank" rel="noopener noreferrer">bcftools_stats</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools stats. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_query" target="_blank" rel="noopener noreferrer">bcftools_query</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools query. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_reheader" target="_blank" rel="noopener noreferrer">bcftools_reheader</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools reheader. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_merge" target="_blank" rel="noopener noreferrer">bcftools_merge</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a> Wrapper for bcftools application bcftools merge. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_plugin_color_chrs" target="_blank" rel="noopener noreferrer">bcftools_plugin_color_chrs</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools color-chrs. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_consensus" target="_blank" rel="noopener noreferrer">bcftools_consensus</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools consensus. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_plugin_frameshifts" target="_blank" rel="noopener noreferrer">bcftools_plugin_frameshifts</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a> Wrapper for bcftools application bcftools frameshifts. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_plugin_counts" target="_blank" rel="noopener noreferrer">bcftools_plugin_counts</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools counts. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_gtcheck" target="_blank" rel="noopener noreferrer">bcftools_gtcheck</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools gtcheck. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/data_manager_snpsift_dbnsfp" target="_blank" rel="noopener noreferrer">data_manager_snpsift_dbnsfp</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-devteam/tree/master/data_managers/data_manager_snpsift_dbnsfp" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-devteam/tree/master/data_managers/data_manager_snpsift_dbnsfp</a>  Install SnpSift dbNSFP local data Installs dbNSFP data for use by SnpSift dbNSFP. dbNSFP is an integrated database of human functional predictions from multiple algorithms (SIFT, Polyphen2, LRT and MutationTaster, PhyloP and GERP++, etc.). SnpEff and SnpSift are developed by Pablo Cingolani. <a href="http://snpeff.sourceforge.net%5B/SnpSift%5D(/toolshed/contributions/2016-07/SnpSift/).html#dbNSFP" target="_blank" rel="noopener noreferrer">http://snpeff.sourceforge.net[/SnpSift](/toolshed/contributions/2016-07/SnpSift/).html#dbNSFP</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/xpath" target="_blank" rel="noopener noreferrer">xpath</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/xpath" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/xpath</a>  XPath XML querying tool Execute XPath queries on XML files for use in downstream analysis. Useful for extracting lists of IDs or blast hits from large XML files.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_plugin_fixploidy" target="_blank" rel="noopener noreferrer">bcftools_plugin_fixploidy</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools fixploidy. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_convert_to_vcf" target="_blank" rel="noopener noreferrer">bcftools_convert_to_vcf</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools convert to vcf. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_filter" target="_blank" rel="noopener noreferrer">bcftools_filter</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools filter. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_plugin_dosage" target="_blank" rel="noopener noreferrer">bcftools_plugin_dosage</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools dosage. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/ncbi_eutils_elink" target="_blank" rel="noopener noreferrer">ncbi_eutils_elink</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_entrez_eutils" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_entrez_eutils</a>  Wrapper for NCBI application NCBI ELink Entrez E-Utils provide an interface to the NCBI databases to allow querying and downloading datasets from NCBI</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_cnv" target="_blank" rel="noopener noreferrer">bcftools_cnv</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools cnv. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/pe_histogram" target="_blank" rel="noopener noreferrer">pe_histogram</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/pe_histogram" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/pe_histogram</a>  Contains a tool that produces an insert size histogram for a paired-end BAM file. Contains a tool that produces an insert size histogram and basic statistics for a paired-end BAM file. The tool generates 2 outputs: a png image consisting of the histogram of the insert size frequency and a tabular file containing the alignment statistics.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_query_list_samples" target="_blank" rel="noopener noreferrer">bcftools_query_list_samples</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools List Samples. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/suite_ncbi_entrez" target="_blank" rel="noopener noreferrer">suite_ncbi_entrez</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_entrez_eutils" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_entrez_eutils</a>  NCBI Entrez E-Utilties allow fetching data from NCBI Databases Entrez E-Utils provide an interface to the NCBI databases to allow querying and downloading datasets from NCBI</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/prinseq" target="_blank" rel="noopener noreferrer">prinseq</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/prinseq/" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/prinseq/</a>  PRINSEQ is a tool for easy and rapid quality control and data processing of metagenomic and metatranscriptomic datasets</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/suite_biom_format" target="_blank" rel="noopener noreferrer">suite_biom_format</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/biom_format" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/biom_format</a>  BIOM-format tools The biom-format package provides a command line interface and Python API for working with BIOM files.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/suite_bcftools_1_3" target="_blank" rel="noopener noreferrer">suite_bcftools_1_3</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  A suite of Galaxy tools designed to work with version 1.3 of the bcftools package. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/simpleweather" target="_blank" rel="noopener noreferrer">simpleweather</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/weather_app" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/weather_app</a>  provides simple weather in text format Weather tool using <a href="https://wttr.in" target="_blank" rel="noopener noreferrer">https://wttr.in</a>. The tool should be treated carefully because it uses an external webpage.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/kraken_taxonomy_report" target="_blank" rel="noopener noreferrer">kraken_taxonomy_report</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/blob/master/tools/kraken_taxonomy_report/" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/blob/master/tools/kraken_taxonomy_report/</a>  Kraken taxonomy report Turns Kraken classified reads into a tabular taxonomy abundance table. Optionally outputs a newick tree.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_convert_from_vcf" target="_blank" rel="noopener noreferrer">bcftools_convert_from_vcf</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools convert from vcf. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_plugin_vcf2sex" target="_blank" rel="noopener noreferrer">bcftools_plugin_vcf2sex</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools vcf2sex. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/ncbi_eutils_einfo" target="_blank" rel="noopener noreferrer">ncbi_eutils_einfo</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_entrez_eutils" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_entrez_eutils</a>  Wrapper for NCBI application NCBI EInfo Entrez E-Utils provide an interface to the NCBI databases to allow querying and downloading datasets from NCBI</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/ncbi_eutils_epost" target="_blank" rel="noopener noreferrer">ncbi_eutils_epost</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_entrez_eutils" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_entrez_eutils</a>  Wrapper for NCBI application NCBI EPost Entrez E-Utils provide an interface to the NCBI databases to allow querying and downloading datasets from NCBI</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_isec" target="_blank" rel="noopener noreferrer">bcftools_isec</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools isec. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/bcftools_norm" target="_blank" rel="noopener noreferrer">bcftools_norm</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</a>  Wrapper for bcftools application bcftools norm. <a href="https://samtools.github.io/bcftools/" target="_blank" rel="noopener noreferrer">https://samtools.github.io/bcftools/</a></li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/ncbi_eutils_egquery" target="_blank" rel="noopener noreferrer">ncbi_eutils_egquery</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_entrez_eutils" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_entrez_eutils</a>  Wrapper for NCBI application NCBI EGQuery Entrez E-Utils provide an interface to the NCBI databases to allow querying and downloading datasets from NCBI</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/ncbi_eutils_esummary" target="_blank" rel="noopener noreferrer">ncbi_eutils_esummary</a>: planemo upload for repository <a href="https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_entrez_eutils" target="_blank" rel="noopener noreferrer">https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_entrez_eutils</a>  Wrapper for NCBI application NCBI ESummary Entrez E-Utils provide an interface to the NCBI databases to allow querying and downloading datasets from NCBI</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/ulfschaefer" target="_blank" rel="noopener noreferrer">ulfschaefer</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/ulfschaefer/phemost" target="_blank" rel="noopener noreferrer">phemost</a>: PHE MLST tool</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/ulfschaefer/data_manager_phemost" target="_blank" rel="noopener noreferrer">data_manager_phemost</a>: data manager for PHE MOST tool downloads MLST data from pubmlst.org and makes it available for the phemsot tool via the mlst_data.loc data table</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/simon-gladman" target="_blank" rel="noopener noreferrer">simon-gladman</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/simon-gladman/snippy" target="_blank" rel="noopener noreferrer">snippy</a>: Initial upload Contains the snippy tool for characterising microbial snps Snippy finds variants between a haploid reference genome and your NGS sequence reads. It will find both substitutions and insertions/deletions (indels).</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/simon-gladman/velvetoptimiser" target="_blank" rel="noopener noreferrer">velvetoptimiser</a>: Initial upload An updated wrapper for the Velvet Optimiser The Velvet Optimiser automatically finds the optimum assembly parameters for velvet over many runs. The automation is configurable. Written by Simon Gladman. This wrapper has been updated allowing for data collections and version 1.2.10 of velvet</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/luis" target="_blank" rel="noopener noreferrer">luis</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/luis/ball" target="_blank" rel="noopener noreferrer">ball</a>: BALL - The Biochemical Algorithms Library BALL is computer software consisting of the versatile C++ class framework BALL (Biochemical Algorithms Library), a library of algorithms and data structures for molecular modelling and computational structural bioinformatics, a Python interface to this library and an open source graphical interface to BALL, the molecule viewer BALLView. BALL has evolved from a commercial product into free-of-charge open-source software licensed under the GNU Lesser General Public License (LGPL). BALLView is licensed under the GNU General Public License (GPL) license. BALL and BALLView have been ported to the operating systems Linux, OS X, Solaris, and Windows. The molecule viewer BALLView, also developed by the BALL project team, is a C++ application of BALL using Qt, and OpenGL with the real-time ray tracer RTFact as render back ends. For both, BALLView offers three-dimensional and stereoscopic visualizing in several different modes, and applying directly the algorithms of the BALL library via its graphical user interface. The BALL project is developed and maintained by groups at Saarland University, Mainz University, and University of Tubingen. Both the library and the viewer are heavily used for education and research. BALL packages have been made available in the Debian project in April 2010.</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/nml" target="_blank" rel="noopener noreferrer">nml</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/nml/plasmidspades" target="_blank" rel="noopener noreferrer">plasmidspades</a>: planemo upload  Genome assembler for assemblying plasmid</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/nml/hivtrace" target="_blank" rel="noopener noreferrer">hivtrace</a>: planemo upload for repository <a href="https://github.com/phac-nml/galaxy_tools/tree/tools/hivtrace" target="_blank" rel="noopener noreferrer">https://github.com/phac-nml/galaxy_tools/tree/tools/hivtrace</a>  An application that identifies potential transmission clusters within a supplied FASTA file with an option to find potential links against the Los Alamos HIV Sequence Database.</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/brenninc" target="_blank" rel="noopener noreferrer">brenninc</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/brenninc/directory_reader_limited_by_data_table" target="_blank" rel="noopener noreferrer">directory_reader_limited_by_data_table</a>: Reads files into a data collection from a preconfigure directory on the server. Reads files from a preconfigure directory on the server. Loads all specified files from a server directory into a Data Collection and also provides a text fie with the original names. Depends on the directory_data  Data Table. See <a href="https://toolshed.g2.bx.psu.edu/view/brenninc/data_manager_for_directory_data" target="_blank" rel="noopener noreferrer">https://toolshed.g2.bx.psu.edu/view/brenninc/data_manager_for_directory_data</a>. Only preconfigured combination of path and extension work. Files will have their extension changed to one expected by galaxy and can be decompressed as set in the Data Table. Users have the ability to limit the files by prefix (start of name) and postfix (last bit of the name before the extension).</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/portiahollyoak" target="_blank" rel="noopener noreferrer">portiahollyoak</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/portiahollyoak/fastuniq" target="_blank" rel="noopener noreferrer">fastuniq</a>: planemo upload for repository <a href="https://github.com/portiahollyoak/Tools" target="_blank" rel="noopener noreferrer">https://github.com/portiahollyoak/Tools</a>  duplicate read remover FastUniq is an ultrafast de novo tool for removal of duplicates in paired short DNA sequence reads in FASTQ format. It identifies duplicates by comparing sequences between read pairs and does not require complete genome sequences as prerequisites. It is also capable of simultaneously handling reads with different lengths and results in highly efficient running time.</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/maille" target="_blank" rel="noopener noreferrer">maille</a>:</em></p>
<ul>
<li>
<p><a href="https://toolshed.g2.bx.psu.edu/view/maille/affiliation_otu" target="_blank" rel="noopener noreferrer">affiliation_otu</a>: Find Rapidly OTUs with Galaxy Solution FROGS is a galaxy/CLI workflow designed to produce an OTU count matrix from high depth sequencing amplicon data. This workflow is focused on:</p>
<ul>
<li>User-friendliness with the integration in galaxy and lots of rich graphic outputs</li>
<li>Accuracy with a clustering without global similarity threshold, the  management of multi-affiliations and management of separated PCRs in the chimera removal step</li>
<li>Speed with fast algorithms and an easy to use parallelisation</li>
<li>Scalability with algorithms designed to support the data growth</li>
</ul>
</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/lgueguen" target="_blank" rel="noopener noreferrer">lgueguen</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/lgueguen/sartools" target="_blank" rel="noopener noreferrer">sartools</a>: Planemo upload SARTools is a R package dedicated to the differential analysis of RNA-seq data. SARTools provides R tools and an environment for the statistical analysis of RNA-Seq projects: load and clean data, produce figures, perform statistical analysis/testing with DESeq2 or edgeR, export results and create final report.</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/ethevenot" target="_blank" rel="noopener noreferrer">ethevenot</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/ethevenot/biosigner" target="_blank" rel="noopener noreferrer">biosigner</a>: planemo upload for repository <a href="https://github.com/workflow4metabolomics/biosigner.git" target="_blank" rel="noopener noreferrer">https://github.com/workflow4metabolomics/biosigner.git</a> The "biosigner" module implements a new feature selection algorithm to assess the relevance of the variables for the prediction performances of the classifier (Rinaudo et al, submitted).</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/ethevenot/univariate" target="_blank" rel="noopener noreferrer">univariate</a>: planemo upload for repository <a href="https://github.com/workflow4metabolomics/univariate.git" target="_blank" rel="noopener noreferrer">https://github.com/workflow4metabolomics/univariate.git</a> Univariate statistics. The module performs two sample tests (t-test and Wilcoxon rank test), analysis of variance and Kruskal-Wallis rank test, and correlation tests (by using either the pearson or the spearman correlation).</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/ethevenot/multivariate" target="_blank" rel="noopener noreferrer">multivariate</a>: planemo upload for repository <a href="https://github.com/workflow4metabolomics/multivariate.git" target="_blank" rel="noopener noreferrer">https://github.com/workflow4metabolomics/multivariate.git</a> Multivariate analysis by PCA, PLS(-DA), and OPLS(-DA). Part of the W4M project (<a href="http://workflow4metabolomics.org" target="_blank" rel="noopener noreferrer">http://workflow4metabolomics.org</a>). "Latent variable modeling" with Principal Component Analysis ("PCA") and Partial Least Squares ("PLS") are powerful methods for "visualization", "regression", "classification", and feature selection of "omics data" where the number of variables exceeds the number of samples and with multicollinearity among variables (Wold et al, 2001; Thenenhaus, 1998; Wehrens, 2011; Eriksson et al, 2006; Trygg et al, 2007). Orthogonal Partial Least Squares ("OPLS") enables to separately model the variation correlated (predictive) to the factor of interest and the uncorrelated (orthogonal) variation (Trygg and Wold, 2002). While performing similarly to PLS, OPLS facilitates interpretation. Successful applications of these chemometrics techniques include spectroscopic data such as Raman spectroscopy, nuclear magnetic resonance (NMR), mass spectrometry (MS) in metabolomics and proteomics, but also transcriptomics data. In addition to "scores", "loadings" and "weights" plots, the module provides metrics and graphics to determine the optimal number of components (e.g. with the "R2" and "Q2" coefficients; Wold et al, 2001; Thenenhaus, 1998; Eriksson et al, 2006), check the "validity of the model" by permutation testing (Szymanska et al, 2012), detect "outliers" (Wold et al, 2001; Thenenhaus, 1998; Hubert et al, 2005), and provide several metrics to assess the importance of the variables in the model (e.g. "Variable Importance in Projection" or regression coefficients; Wold et al, 2001; Mehmood et al, 2012; Galindo-Prieto et al, 2014). The module is an implementation of the "ropls" R package available from Bioconductor (Thevenot et al, 2015).</li>
</ul>
</li>
</ul>
<h1 id="dependency-definitions"><a href="#dependency-definitions" aria-hidden="true"><span class="icon icon-link"></span></a>Dependency Definitions</h1>
<ul>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/iuc" target="_blank" rel="noopener noreferrer">iuc</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/package_meme_4_11_1" target="_blank" rel="noopener noreferrer">package_meme_4_11_1</a>: Contains a tool dependency definition that downloads and installs version 4.11.1 of the MEME suite The MEME Suite provides a unified set of tools for discovery and analysis of sequence motifs representing features such as DNA binding sites and protein interaction domains.</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/ulfschaefer" target="_blank" rel="noopener noreferrer">ulfschaefer</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/ulfschaefer/package_python_2_7_lxml_3_2_3" target="_blank" rel="noopener noreferrer">package_python_2_7_lxml_3_2_3</a>: lxml 3.2.3 for Python 2.7 This Galaxy Tool shed package installs version 3.2.3 of lxml (<a href="http://lxml.de" target="_blank" rel="noopener noreferrer">http://lxml.de</a>) for use with python 2.7.10.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/ulfschaefer/package_phemost_1_0" target="_blank" rel="noopener noreferrer">package_phemost_1_0</a>: PHE MLST tool dependency definition</li>
</ul>
</li>
</ul>
</div>
</div><footer data-v-4b3455de data-v-340f7abc><!----></footer><footer class="static-footer markdown"><p>The Galaxy Project is supported in part by NSF, NHGRI, The Huck Institutes of the Life Sciences, The Institute for CyberScience at Penn State, and Johns Hopkins University.</p>
</footer></section></div>
    <script src="/assets/js/app.0690a31c.js" defer></script><script src="/assets/js/page--src--templates--article-vue.6d736ff4.js" defer></script>
  </body>
</html>
