<!DOCTYPE html>
<html data-html-server-rendered="true" lang="en" data-vue-tag="%7B%22lang%22:%7B%22ssr%22:%22en%22%7D%7D">
  <head>
    <title> - Galaxy Community Hub</title><meta name="gridsome:hash" content="f696d459c27618375ae9beff9077a6039192ee4e"><meta data-vue-tag="ssr" charset="utf-8"><meta data-vue-tag="ssr" name="generator" content="Gridsome v0.7.23"><meta data-vue-tag="ssr" data-key="viewport" name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover"><meta data-vue-tag="ssr" data-key="format-detection" name="format-detection" content="telephone=no"><meta data-vue-tag="ssr" data-key="description" name="description" content="All about Galaxy and its community."><link data-vue-tag="ssr" rel="icon" href="data:,"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="16x16" href="/assets/static/favicon.ce0531f.7d4a227415c9d9b704d56a079d49a3e1.png"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="32x32" href="/assets/static/favicon.ac8d93a.7d4a227415c9d9b704d56a079d49a3e1.png"><link rel="preload" href="/assets/css/0.styles.ebb3d58f.css" as="style"><link rel="preload" href="/assets/js/app.431ed39e.js" as="script"><link rel="preload" href="/assets/js/page--src--templates--article-vue.077afa7b.js" as="script"><link rel="preload" href="/assets/js/page--src--templates--article-vue~page--src--templates--vue-article-vue.0d54471b.js" as="script"><link rel="prefetch" href="/assets/js/page--src--pages--404-vue.0778f6cc.js"><link rel="prefetch" href="/assets/js/page--src--pages--blog-vue.ad74c4a2.js"><link rel="prefetch" href="/assets/js/page--src--pages--careers-vue.ffdb87dc.js"><link rel="prefetch" href="/assets/js/page--src--pages--data-explorer-vue.7d92c08b.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--archive-vue.475478fc.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests--papercuts-vue.8793e971.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests-vue.85ecd7d8.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--webinars-vue.42d4404c.js"><link rel="prefetch" href="/assets/js/page--src--pages--events-vue.945a6980.js"><link rel="prefetch" href="/assets/js/page--src--pages--index-vue.7226d2dc.js"><link rel="prefetch" href="/assets/js/page--src--pages--news-vue.9b990e6a.js"><link rel="prefetch" href="/assets/js/page--src--pages--search-vue.02cebeef.js"><link rel="prefetch" href="/assets/js/page--src--pages--tests-vue.9a92ab5f.js"><link rel="prefetch" href="/assets/js/page--src--pages--use-vue.44868762.js"><link rel="prefetch" href="/assets/js/page--src--templates--platform-vue.dd6cee54.js"><link rel="prefetch" href="/assets/js/page--src--templates--vue-article-vue.4a4eb2ce.js"><link rel="prefetch" href="/assets/js/vendors~page--src--pages--404-vue.769c153b.js"><link rel="prefetch" href="/assets/js/vendors~page--src--pages--404-vue~page--src--pages--careers-vue~page--src--pages--use-vue~page--src-~1d1e8317.b4529ad8.js"><link rel="prefetch" href="/assets/js/vendors~page--src--pages--search-vue.8befcd1e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--index-md.4f5426ae.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--md--imgtests--imgdir--index-md.b08a5160.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--partials--index-md.4d0e66d8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--imgdir--index-md.166aed18.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--index-md.414821d6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--index-md.0a4114ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--index-md.3b918e0d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--reldir--index-md.1f9bacba.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--capacity-planning--index-md.6224cac9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--index-md.147aeb95.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--performance--cluster--legacy--index-md.c4fbc69d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--datatypes--index-md.6ee2085e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--disk-quotas--index-md.fd0c1ad8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--faq--index-md.cd750303.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--data-model--index-md.f1243b42.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--galaxy-ops--index-md.03f6a777.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--ten-simple-steps-galaxy-as-a-service--index-md.b0cda7d7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--tool-config-syntax--index-md.5f402dce.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--writing-tests--index-md.a8f24903.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--allocation--index-md.55aa7c38.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--faq--index-md.266f1ed7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--index-md.f16c61ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--ssh--index-md.829f2ef6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--troubleshooting--index-md.24740bba.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--capacity-planning--index-md.ec5f7f4a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--getting-started--index-md.99a27dbe.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--index-md.8b4e03ab.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--building--index-md.fad12c5a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--capacity-planning--index-md.4a8f31c9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--cluster-types--index-md.a2825b49.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--customizing--index-md.c2abaee3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--faq--index-md.21d63a7e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--index-md.f13b54d4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--pre-201703--index-md.5e720c62.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--hadoop--index-md.71d02357.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--ht-condor--index-md.1315117c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--index-md.f2ed983f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--openstack--index-md.696c6b9a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--services--index-md.b6210d57.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--sharing--index-md.c43bf8aa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--troubleshooting--index-md.903f842f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--userdata--index-md.bef916ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--future--index-md.f34d9417.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--index-md.ea8db596.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-09--index-md.09370c94.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-27--index-md.31c36f88.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-09-19--index-md.4d238f9a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-11-14--index-md.8e304a45.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-01-16--index-md.a6988118.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-03-20--index-md.e2ea4aa5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-05-15--index-md.7291d46b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-11-20--index-md.cafbc465.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-02-19--index-md.097aeffa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-04-16--index-md.451dcf9c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-06-18--index-md.59747c7b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-08-20--index-md.5c2ec0cc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-10-15--index-md.37b0e017.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-02-18--index-md.684f2b92.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-04-21--index-md.146c966c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-06-16--index-md.cb1518a2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-08-18--index-md.dfb2b024.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--index-md.d85e5326.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--webinar-tech--index-md.395e3119.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2012--index-md.ea9d8962.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2014--index-md.9e9cbd44.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--index-md.bb20208e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--data--index-md.06c97ea0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--datafiles--index-md.f0e98198.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--architecture--index-md.33a164a2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--bitbucket--index-md.eb74cd09.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--css--index-md.4fb79752.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--getting-started--index-md.fc74c666.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--index-md.283c9236.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--jsa--index-md.c49accb7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--mercurial--index-md.bbe4630a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--security-tool-tips--index-md.29e4dd11.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--galaxy-project--index-md.4df13e59.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--index-md.c3b80e1f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--papers--index-md.bbad52a7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--posters--index-md.3f74bc19.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--presentations--index-md.d4d2dff2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--videos--index-md.9cb0c58c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--advanced-session--index-md.6bd15879.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--basics-session--index-md.05ca13af.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--index-md.e97992a0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--logistics--index-md.3e4e6316.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--publicity--index-md.c40f283b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--registration--index-md.77be1242.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--abstracts--index-md.35649b89.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--index-md.356ea33f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--key-dates--index-md.1428cbc3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--lightning-talks--index-md.75e86332.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--logistics--index-md.27bb94f5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--organizing-committee--index-md.110501ba.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--photos--index-md.bdc78b82.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--automation-strategies--index-md.2d9d8096.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--bioinformatics-training--index-md.ba644bcf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--end-users--index-md.58b1c8d8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--workflows-and-api--index-md.6ec375d4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--index-md.d9fcafdc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--promotion--index-md.38e1503c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--register--index-md.54c68640.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--sponsorships--index-md.c5447529.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--training-day--index-md.2954475b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--index-md.a9629b22.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--posters--index-md.6936e94f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--talks--index-md.55e07dd9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--clinical-use-hpc-file-systems--index-md.2d27cd06.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-admins--index-md.f3b22962.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-blast--index-md.2c64fa7c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-professionals--index-md.4f0d888d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--index-md.01075520.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--proteomics--index-md.41da0de1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--public-galaxy-servers--index-md.3bd15791.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--tool-developers--index-md.3d4b6e78.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--index-md.c3b24e68.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--key-dates--index-md.b51f20c8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--lightning--index-md.5fd58060.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--logistics--index-md.6aa4f6ad.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--organizers--index-md.5e03fd24.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--photos--index-md.f6d10991.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--program--index-md.5cf83aa3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--promotion--index-md.0d42298e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--register--index-md.08fefd86.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--sponsorships--index-md.bde799e6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--advance-tool-data--index-md.d8d4916c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--api--index-md.7daa64ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--index-md.9464cfa7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--intro-tool-data--index-md.49c96a23.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--vms--index-md.f4c0d9cf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--index-md.b9f56a47.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--posters--index-md.ff4cad61.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--talks--index-md.4cadf122.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--branch-release-merge--index-md.44dc5837.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-admins--index-md.8f083421.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-end-users--index-md.c56d3c2a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-training-network--index-md.89dcab1c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--heterogeneous--index-md.1026fb27.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--index-md.698f220d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--using-galaxy-ui-overview--index-md.f0072e9c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--hackathon--index-md.0fa5189d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--index-md.1a92620f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--key-dates--index-md.5bad7912.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--lightning--index-md.d289026d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--logistics--index-md.5efc6a52.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--organizers--index-md.756da03c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--photos--index-md.7eceb6f4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--program--index-md.4288664d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--promotion--index-md.e1d6479f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--register--index-md.2357efb0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--sponsor-exhibit--index-md.c136619c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--admin-walkthrough--index-md.26cc5549.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--api--index-md.4c72bfaf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--data-managers--index-md.cdfd5a52.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--index-md.8e217549.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--vms--index-md.69d888fb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--cwl--index-md.c1cd35f3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-management-challenges--index-md.5920047d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-splitting-and-parallelism--index-md.61eede8e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--elixir--index-md.642a778a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--extended-workflow--index-md.dc858e5d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--fots--index-md.3efae184.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-admins--index-md.91dc64b7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-community-uk--index-md.c57b7a7a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-scientists--index-md.052840fd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-training-network--index-md.9a274b16.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--index-md.1f170ec8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--data-hack-roundup--index-md.da0a5aeb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--header--index-md.c293acf4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--index-md.009d5e8f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2016--bofs--index-md.20e08bdd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--abstracts--index-md.a67b99a1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--childcare--index-md.b783e73b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--cofest--index-md.badfa848.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--conduct--index-md.a617acd5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--faq--index-md.bfea97c5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--index-md.8037f1be.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--organizers--index-md.d6fd7a93.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--promotion--index-md.5d827c1b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--registration--index-md.112c0eeb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--sponsors--index-md.3aca7bc3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--training--index-md.9c114d96.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--venue--index-md.9f693a25.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--abstracts--index-md.2e85a391.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--childcare--index-md.ac80bc22.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--code-of-conduct--index-md.6c35dba5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--collaboration--index-md.c215932d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--faq--index-md.5bcf0d8f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--icebreaker--index-md.4b44ae0d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--index-md.7379eae9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--keynotes--index-md.d62e579d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--organizers--index-md.de82c948.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--promotion--index-md.6e866954.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--register--index-md.14609d86.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--sponsors--index-md.5557a981.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--training--index-md.51a05dc1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--venue--index-md.494db3be.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2014--index-md.c32db467.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2016--index-md.41eb2d7a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2014--index-md.d9f7e6c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2016--index-md.a5ac2b2f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--index-md.5388ed69.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--trainingday--index-md.3cfcfe12.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2016--index-md.dfec1f6c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--example--vue-remark--index-md.7d676959.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--faqs--index-md.c05bbde0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-06--index-md.0252d302.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-07--index-md.7d5da777.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--icons--index-md.7175ad9b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--index-md.8ae2a27a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--logos--index-md.87a50b1c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--news-graphics--index-md.eb8f25fd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--photos--index-md.796673cd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--screenshots--index-md.bc278f0e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--index-md.139748d5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--workflow-editor-unannotated--index-md.0bba8409.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--extract--index-md.fc2a5f3c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--tool-panel--index-md.5e2fead1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--variables--index-md.d815a55a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--faq--index-md.505c85de.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--galaxy-pages--index-md.1246e5d4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--interval-operations--index-md.67df01a9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--main--faq--index-md.4a8db549.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--tools--requirements--index-md.2f088bde.js"><link rel="stylesheet" href="/assets/css/0.styles.ebb3d58f.css"><script data-vue-tag="ssr" src="https://sidecar.gitter.im/dist/sidecar.v1.js" async defer></script><noscript data-vue-tag="ssr"><style>.g-image--loading{display:none;}</style></noscript>
    <script src="//unpkg.com/jquery@3.3.1/dist/jquery.slim.min.js"></script>
  </head>
  <body >
    <div data-server-rendered="true" id="app" class="layout" data-v-77022bf2><header id="masthead"><nav class="navbar container justify-content-center navbar-dark bg-transparent navbar-expand-lg"><a href="/" target="_self" class="navbar-brand router-link-active"><img id="masthead-logo" src="/images/galaxy_logo_hub_white.svg" alt="Galaxy Community Hub" height="30"></a><button type="button" aria-label="Toggle navigation" class="navbar-toggler"><span class="navbar-toggler-icon"></span></button><div id="nav-collapse" class="navbar-collapse collapse" style="display:none;"><ul class="navbar-nav"><li class="nav-item"><a href="/use/" target="_self" class="nav-link">Use</a></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Learn</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/learn/" role="menuitem" target="_self" class="dropdown-item">How to use Galaxy</a></li><li role="presentation"><a role="menuitem" href="https://training.galaxyproject.org/" target="_self" class="dropdown-item">Teach with Galaxy</a></li></ul></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Community</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/community/" role="menuitem" target="_self" class="dropdown-item">About</a></li><li role="presentation"><a href="/events/" role="menuitem" target="_self" class="dropdown-item router-link-active">Events</a></li><li role="presentation"><a href="/news/" role="menuitem" target="_self" class="dropdown-item">News</a></li><li role="presentation"><a href="/blog/" role="menuitem" target="_self" class="dropdown-item">Blog</a></li><li role="presentation"><a href="/careers/" role="menuitem" target="_self" class="dropdown-item">Careers</a></li><li role="presentation"><a href="/publication-library/" role="menuitem" target="_self" class="dropdown-item">Publications</a></li><li role="presentation"><a href="/galaxy-project/statistics/" role="menuitem" target="_self" class="dropdown-item">Stats</a></li><li role="presentation"><a href="/community/governance/" role="menuitem" target="_self" class="dropdown-item">Governance</a></li><li role="presentation"><a href="/community/coc/" role="menuitem" target="_self" class="dropdown-item">Code of Conduct</a></li></ul></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Deploy &amp; Develop</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/admin/get-galaxy/" role="menuitem" target="_self" class="dropdown-item">Install Galaxy</a></li><li role="presentation"><a href="/admin/" role="menuitem" target="_self" class="dropdown-item">Server administration</a></li><li role="presentation"><a href="/tools/" role="menuitem" target="_self" class="dropdown-item">Tools</a></li><li role="presentation"><a href="/develop/api/" role="menuitem" target="_self" class="dropdown-item">Working with the API</a></li><li role="presentation"><a href="/develop/" role="menuitem" target="_self" class="dropdown-item">Contributing</a></li><li role="presentation"><a href="/docs/" role="menuitem" target="_self" class="dropdown-item">Code documentation</a></li></ul></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Support</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/support/" role="menuitem" target="_self" class="dropdown-item">FAQ</a></li><li role="presentation"><a role="menuitem" href="https://help.galaxyproject.org/" target="_self" class="dropdown-item">Galaxy Help Forum</a></li></ul></li><li class="nav-item"><a href="/jxtx/" target="_self" class="nav-link">@jxtx</a></li></ul><ul class="navbar-nav ml-auto"><li class="form-inline"><form action="/search/" method="get" class="form-inline"><input id="search-input" name="q" type="text" placeholder="Search Galaxy" value="" class="mr-sm-2 form-control form-control-sm"><button type="submit" class="btn my-2 my-sm-0 btn-secondary btn-sm"><span class="fa fa-search"></span></button></form></li><li class="nav-item"><a href="https://github.com/galaxyproject/galaxy-hub/tree/master/content/content-md/events/gcc2012/training-day/ws1/index.md" target="_self" class="nav-link"><i class="fab fa-lg fa-github"></i>
                Edit
            </a></li></ul></div></nav></header><section id="maincontainer" class="container"><header data-v-98965260 data-v-77022bf2><a href="/" class="link active" data-v-98965260>‚Üê Back to Home</a><!----><!----><h1 class="title" data-v-98965260></h1><!----><section class="metadata freetext" data-v-98965260><!----><!----><!----></section><!----><!----><div class="clearfix" data-v-98965260></div></header><div class="content markdown" data-v-77022bf2><p>Slides are available here: <a href="https://depot.galaxyproject.org/hub/attachments/documents/presentations/gcc2012/WS1_blankenberg_gcc_2012_integrating_tools_and_data_sources.pdf" target="_blank" rel="noopener noreferrer">Slides</a>.</p>
<p>maf_slice.py:</p>
<pre><code>#!/usr/bin/env python

"""
Modified from interval2maf.py for teaching workshops

Reads a list of intervals and a maf. Produces a new maf containing the
blocks or parts of blocks in the original that overlapped the intervals.

If a MAF file, not UID, is provided the MAF file is indexed before being processed.

NOTE: If two intervals overlap the same block it will be written twice.

usage: %prog maf_file [options]
   -d, --dbkey=d: Database key, ie hg17
   -c, --chromCol=c: Column of Chr
   -s, --startCol=s: Column of Start
   -e, --endCol=e: Column of End
   -S, --strandCol=S: Column of Strand
   -t, --mafType=t: Type of MAF source to use
   -m, --mafFile=m: Path of source MAF file, if not using cached version
   -I, --mafIndex=I: Path of precomputed source MAF file index, if not using cached version
   -i, --interval_file=i:       Input interval file
   -o, --output_file=o:      Output MAF file
   -p, --species=p: Species to include in output
   -P, --split_blocks_by_species=P: Split blocks by species
   -r, --remove_all_gap_columns=r: Remove all Gap columns
   -l, --indexLocation=l: Override default maf_index.loc file
   -z, --mafIndexFile=z: Directory of local maf index file ( maf_index.loc or maf_pairwise.loc )
"""

#Dan Blankenberg
from galaxy import eggs
import pkg_resources; pkg_resources.require( "bx-python" )
from bx.cookbook import doc_optparse
import bx.align.maf
import bx.intervals.io
from galaxy.tools.util import maf_utilities
import sys

assert sys.version_info[:2] >= ( 2, 4 )

def __main__():
    index = index_filename = None
    mincols = 0
    
    #Parse Command Line
    options, args = doc_optparse.parse( __doc__ )
    
    if options.dbkey: dbkey = options.dbkey
    else: dbkey = None
    if dbkey in [None, "?"]:
        maf_utilities.tool_fail( "You must specify a proper build in order to extract alignments. You can specify your genome build by clicking on the pencil icon associated with your interval file." )
    
    species = maf_utilities.parse_species_option( options.species )
    
    if options.chromCol: chromCol = int( options.chromCol ) - 1
    else: 
        chromCol = 0
    
    if options.startCol: startCol = int( options.startCol ) - 1
    else: 
        startCol = 1
    
    if options.endCol: endCol = int( options.endCol ) - 1
    else: 
        endCol = 2
    
    if options.strandCol: strandCol = int( options.strandCol ) - 1
    else: 
        strandCol = -1
    
    if options.interval_file: interval_file = options.interval_file
    else: 
        maf_utilities.tool_fail( "Input interval file has not been specified." )
    
    if options.output_file: output_file = options.output_file
    else: 
        maf_utilities.tool_fail( "Output file has not been specified." )
    
    split_blocks_by_species = remove_all_gap_columns = False
    if options.split_blocks_by_species and options.split_blocks_by_species == 'split_blocks_by_species':
        split_blocks_by_species = True
        if options.remove_all_gap_columns and options.remove_all_gap_columns == 'remove_all_gap_columns':
            remove_all_gap_columns = True
    else:
        remove_all_gap_columns = True
    #Finish parsing command line
    
    #Open indexed access to MAFs
    if options.mafType:
        if options.indexLocation:
            index = maf_utilities.maf_index_by_uid( options.mafType, options.indexLocation )
        else:
            index = maf_utilities.maf_index_by_uid( options.mafType, options.mafIndexFile )
        if index is None:
            maf_utilities.tool_fail( "The MAF source specified (%s) appears to be invalid." % ( options.mafType ) )
    elif options.mafFile:
        index, index_filename = maf_utilities.open_or_build_maf_index( options.mafFile, options.mafIndex, species = [dbkey] )
        if index is None:
            maf_utilities.tool_fail( "Your MAF file appears to be malformed." )
    else:
        maf_utilities.tool_fail( "Desired source MAF type has not been specified." )
    
    #Create MAF writter
    out = bx.align.maf.Writer( open(output_file, "w") )
    
    #Iterate over input regions 
    num_blocks = 0
    num_regions = None
    for num_regions, region in enumerate( bx.intervals.io.NiceReaderWrapper( open( interval_file, 'r' ), chrom_col = chromCol, start_col = startCol, end_col = endCol, strand_col = strandCol, fix_strand = True, return_header = False, return_comments = False ) ):
        src = maf_utilities.src_merge( dbkey, region.chrom )
        for block in index.get_as_iterator( src, region.start, region.end ):
            if split_blocks_by_species:
                blocks = [ new_block for new_block in maf_utilities.iter_blocks_split_by_species( block ) if maf_utilities.component_overlaps_region( new_block.get_component_by_src_start( dbkey ), region ) ]
            else:
                blocks = [ block ]
            for block in blocks:
                block = maf_utilities.chop_block_by_region( block, src, region )
                if block is not None:
                    if species is not None:
                        block = block.limit_to_species( species )
                    block = maf_utilities.orient_block_by_region( block, src, region )
                    if remove_all_gap_columns:
                        block.remove_all_gap_columns()
                    out.write( block )
                    num_blocks += 1
    
    #Close output MAF
    out.close()
    
    #remove index file if created during run
    maf_utilities.remove_temp_index_file( index_filename )
    
    if num_blocks:
        print "%i MAF blocks extracted for %i regions." % ( num_blocks, ( num_regions + 1 ) )
    elif num_regions is not None:
        print "No MAF blocks could be extracted for %i regions." % ( num_regions + 1 )
    else:
        print "No valid regions have been provided."
    
if __name__ == "__main__": __main__()
</code></pre>
<p>tool_conf.xml section:</p>
<pre><code>&#x3C;toolbox>
  &#x3C;section name="Workshop Demo" id="workshop_demo">
    &#x3C;tool file="demo/maf_slice.xml" />
  &#x3C;/section>
&#x3C;/toolbox>
</code></pre>
<p>End of Exercise, adding a basic tool:</p>
<pre><code>&#x3C;tool id="maf_slice" name="Slice MAF" version="1.0.0">
  &#x3C;description>by intervals&#x3C;/description>
  &#x3C;command interpreter="python">
maf_slice.py --dbkey=hg17 --mafFile=${maf_input} --interval_file=${interval_input}  --output_file=${maf_output}
   &#x3C;/command>
   &#x3C;inputs>
    &#x3C;param format="maf" name="maf_input" label="Choose alignments" type="data"/>
    &#x3C;param format="bed" name="interval_input" type="data" label="Choose intervals"/>
   &#x3C;/inputs>
   &#x3C;outputs>
     &#x3C;data format="maf" name="maf_output"/>
   &#x3C;/outputs>
   &#x3C;tests>
   &#x3C;/tests>
   &#x3C;help>
  &#x3C;/help>
&#x3C;/tool>
</code></pre>
<p>End of Exercise adding metadata elements to tool:</p>
<pre><code>&#x3C;tool id="maf_slice" name="Slice MAF" version="1.0.1">
  &#x3C;description>by intervals&#x3C;/description>
  &#x3C;command interpreter="python">
maf_slice.py --dbkey=${interval_input.dbkey} --mafFile=${maf_input} --interval_file=${interval_input}  --output_file=${maf_output}
     --chromCol=${interval_input.metadata.chromCol} --startCol=${interval_input.metadata.startCol} --endCol=${interval_input.metadata.endCol} --strandCol=${interval_input.metadata.strandCol}
     --mafIndex=${maf_input.metadata.maf_index}
   &#x3C;/command>
   &#x3C;inputs>
    &#x3C;param format="maf" name="maf_input" label="Choose alignments" type="data"/>
    &#x3C;param format="interval" name="interval_input" type="data" label="Choose intervals"/>
   &#x3C;/inputs>
   &#x3C;outputs>
     &#x3C;data format="maf" name="maf_output"/>
   &#x3C;/outputs>
   &#x3C;tests>
   &#x3C;/tests>
   &#x3C;help>
  &#x3C;/help>
&#x3C;/tool>
</code></pre>
<p>End of Exercise, adding conditional:</p>
<pre><code>&#x3C;tool id="maf_slice" name="Slice MAF" version="1.0.2">
  &#x3C;description>by intervals&#x3C;/description>
  &#x3C;command interpreter="python">
maf_slice.py --dbkey=${interval_input.dbkey} --mafFile=${maf_source_type.maf_input} --interval_file=${interval_input}  --output_file=${maf_output}
     --chromCol=${interval_input.metadata.chromCol} --startCol=${interval_input.metadata.startCol} --endCol=${interval_input.metadata.endCol} --strandCol=${interval_input.metadata.strandCol}
     --mafIndex=${maf_source_type.maf_input.metadata.maf_index}
   &#x3C;/command>
   &#x3C;inputs>
    &#x3C;param format="interval" name="interval_input" type="data" label="Choose intervals"/>
    &#x3C;conditional name="maf_source_type">
      &#x3C;param name="maf_source" type="select" label="MAF Source">
        &#x3C;option value="cached" selected="true">Locally Cached Alignments&#x3C;/option>
        &#x3C;option value="user">Alignments in Your History&#x3C;/option>
      &#x3C;/param>
      &#x3C;when value="user">
        &#x3C;param format="maf" name="maf_input" label="Choose alignments" type="data"/>
      &#x3C;/when>
      &#x3C;when value="cached">
        &#x3C;!-- need some way to access the external data -->
      &#x3C;/when>
    &#x3C;/conditional>
   &#x3C;/inputs>
   &#x3C;outputs>
     &#x3C;data format="maf" name="maf_output"/>
   &#x3C;/outputs>
   &#x3C;tests>
   &#x3C;/tests>
   &#x3C;help>
  &#x3C;/help>
&#x3C;/tool>
</code></pre>
<p>End of Exercise, accessing .loc file:</p>
<pre><code>&#x3C;tool id="maf_slice" name="Slice MAF" version="1.0.3">
  &#x3C;description>by intervals&#x3C;/description>
  &#x3C;command interpreter="python">
     maf_slice.py --dbkey=${interval_input.dbkey} --interval_file=${interval_input}  --output_file=${maf_output}
     --chromCol=${interval_input.metadata.chromCol} --startCol=${interval_input.metadata.startCol} --endCol=${interval_input.metadata.endCol} --strandCol=${interval_input.metadata.strandCol}
     #if $maf_source_type.maf_source == "user":
         --mafFile=${maf_source_type.maf_input}
         --mafIndex=${maf_source_type.maf_input.metadata.maf_index}
     #else:
         --mafType=$maf_source_type.mafType
         --mafIndexFile=${GALAXY_DATA_INDEX_DIR}/maf_index.loc
     #end if
   &#x3C;/command>
   &#x3C;inputs>
    &#x3C;param format="interval" name="interval_input" type="data" label="Choose intervals"/>
    &#x3C;conditional name="maf_source_type">
      &#x3C;param name="maf_source" type="select" label="MAF Source">
        &#x3C;option value="cached" selected="true">Locally Cached Alignments&#x3C;/option>
        &#x3C;option value="user">Alignments in Your History&#x3C;/option>
      &#x3C;/param>
      &#x3C;when value="user">
        &#x3C;param format="maf" name="maf_input" label="Choose alignments" type="data"/>
      &#x3C;/when>
      &#x3C;when value="cached">
        &#x3C;param name="mafType" type="select" label="Choose alignments">
          &#x3C;options from_data_table="indexed_maf_files">
            &#x3C;filter type="data_meta" ref="interval_input" key="dbkey" column="dbkey" multiple="True" separator=","/>
            &#x3C;validator type="no_options" message="No alignments are available for the build associated with the selected interval file"/>
          &#x3C;/options>
        &#x3C;/param>
      &#x3C;/when>
    &#x3C;/conditional>
   &#x3C;/inputs>
   &#x3C;outputs>
     &#x3C;data format="maf" name="maf_output"/>
   &#x3C;/outputs>
   &#x3C;tests>
   &#x3C;/tests>
   &#x3C;help>
  &#x3C;/help>
&#x3C;/tool>
</code></pre>
<p>End of Exercise, selecting species from metadata values for MAF in history:</p>
<pre><code>&#x3C;tool id="maf_slice" name="Slice MAF" version="1.0.4">
  &#x3C;description>by intervals&#x3C;/description>
  &#x3C;command interpreter="python">
     maf_slice.py --dbkey=${interval_input.dbkey} --interval_file=${interval_input}  --output_file=${maf_output}
     --chromCol=${interval_input.metadata.chromCol} --startCol=${interval_input.metadata.startCol} --endCol=${interval_input.metadata.endCol} --strandCol=${interval_input.metadata.strandCol}
     #if $maf_source_type.maf_source == "user":
         --mafFile=${maf_source_type.maf_input}
         --mafIndex=${maf_source_type.maf_input.metadata.maf_index}
         --species=${maf_source_type.species}
     #else:
         --mafType=$maf_source_type.mafType
         --mafIndexFile=${GALAXY_DATA_INDEX_DIR}/maf_index.loc
     #end if
   &#x3C;/command>
   &#x3C;inputs>
    &#x3C;param format="interval" name="interval_input" type="data" label="Choose intervals"/>
    &#x3C;conditional name="maf_source_type">
      &#x3C;param name="maf_source" type="select" label="MAF Source">
        &#x3C;option value="cached" selected="true">Locally Cached Alignments&#x3C;/option>
        &#x3C;option value="user">Alignments in Your History&#x3C;/option>
      &#x3C;/param>
      &#x3C;when value="user">
        &#x3C;param format="maf" name="maf_input" label="Choose alignments" type="data"/>
        &#x3C;param name="species" type="select" display="checkboxes" multiple="true" label="Choose species" help="Select species to be included in the final alignment">
          &#x3C;options>
            &#x3C;filter type="data_meta" ref="maf_input" key="species" />
          &#x3C;/options>
        &#x3C;/param>
      &#x3C;/when>
      &#x3C;when value="cached">
        &#x3C;param name="mafType" type="select" label="Choose alignments">
          &#x3C;options from_data_table="indexed_maf_files">
            &#x3C;filter type="data_meta" ref="interval_input" key="dbkey" column="dbkey" multiple="True" separator=","/>
            &#x3C;validator type="no_options" message="No alignments are available for the build associated with the selected interval file"/>
          &#x3C;/options>
        &#x3C;/param>
      &#x3C;/when>
    &#x3C;/conditional>
   &#x3C;/inputs>
   &#x3C;outputs>
     &#x3C;data format="maf" name="maf_output"/>
   &#x3C;/outputs>
   &#x3C;tests>
   &#x3C;/tests>
   &#x3C;help>
  &#x3C;/help>
&#x3C;/tool>
</code></pre>
<p>End of exercise, selecting species available from .loc file:</p>
<pre><code>&#x3C;tool id="maf_slice" name="Slice MAF" version="1.0.5">
  &#x3C;description>by intervals&#x3C;/description>
  &#x3C;command interpreter="python">
     maf_slice.py --dbkey=${interval_input.dbkey} --interval_file=${interval_input}  --output_file=${maf_output}
     --chromCol=${interval_input.metadata.chromCol} --startCol=${interval_input.metadata.startCol} --endCol=${interval_input.metadata.endCol} --strandCol=${interval_input.metadata.strandCol}
     --species=${maf_source_type.species}
     #if $maf_source_type.maf_source == "user":
         --mafFile=${maf_source_type.maf_input}
         --mafIndex=${maf_source_type.maf_input.metadata.maf_index}
     #else:
         --mafType=$maf_source_type.mafType
         --mafIndexFile=${GALAXY_DATA_INDEX_DIR}/maf_index.loc
     #end if
   &#x3C;/command>
   &#x3C;inputs>
    &#x3C;param format="interval" name="interval_input" type="data" label="Choose intervals"/>
    &#x3C;conditional name="maf_source_type">
      &#x3C;param name="maf_source" type="select" label="MAF Source">
        &#x3C;option value="cached" selected="true">Locally Cached Alignments&#x3C;/option>
        &#x3C;option value="user">Alignments in Your History&#x3C;/option>
      &#x3C;/param>
      &#x3C;when value="user">
        &#x3C;param format="maf" name="maf_input" label="Choose alignments" type="data"/>
        &#x3C;param name="species" type="select" display="checkboxes" multiple="true" label="Choose species" help="Select species to be included in the final alignment">
          &#x3C;options>
            &#x3C;filter type="data_meta" ref="maf_input" key="species" />
          &#x3C;/options>
        &#x3C;/param>
      &#x3C;/when>
      &#x3C;when value="cached">
        &#x3C;param name="mafType" type="select" label="Choose alignments">
          &#x3C;options from_data_table="indexed_maf_files">
            &#x3C;filter type="data_meta" ref="interval_input" key="dbkey" column="dbkey" multiple="True" separator=","/>
            &#x3C;validator type="no_options" message="No alignments are available for the build associated with the selected interval file"/>
          &#x3C;/options>
        &#x3C;/param>
        &#x3C;param name="species" type="select" display="checkboxes" multiple="true" label="Choose species" help="Select species to be included in the final alignment">
          &#x3C;options from_data_table="indexed_maf_files">
            &#x3C;column name="uid" index="1"/>
            &#x3C;column name="value" index="3"/>
            &#x3C;column name="name" index="3"/>
            &#x3C;filter type="param_value" ref="mafType" column="uid"/>
            &#x3C;filter type="multiple_splitter" column="name" separator=","/>
          &#x3C;/options>
        &#x3C;/param>
      &#x3C;/when>
    &#x3C;/conditional>
   &#x3C;/inputs>
   &#x3C;outputs>
     &#x3C;data format="maf" name="maf_output"/>
   &#x3C;/outputs>
   &#x3C;tests>
   &#x3C;/tests>
   &#x3C;help>
  &#x3C;/help>
&#x3C;/tool>
</code></pre>
</div><footer data-v-54ece1d6 data-v-77022bf2><!----></footer><footer class="static-footer markdown"><p>The Galaxy Project is supported in part by NSF, NHGRI, The Huck Institutes of the Life Sciences, The Institute for CyberScience at Penn State, and Johns Hopkins University.</p>
</footer></section></div>
    <script>window.__INITIAL_STATE__={"data":{"article":{"id":"c1b6f505b57bf4cd463ec7834c816ccf","title":"","tease":"","category":null,"date":null,"days":null,"contact":"","contact_url":"","fileInfo":{"path":"build\u002Fcontent-md\u002Fevents\u002Fgcc2012\u002Ftraining-day\u002Fws1\u002Findex.md"},"authors":"","location":"","location_url":"","source_blog":"","source_blog_url":"","skip_title_render":null,"redirect":"","autotoc":null,"links":[],"image":"","images":{},"external_url":"","content":"\u003Cp\u003ESlides are available here: \u003Ca href=\"https:\u002F\u002Fdepot.galaxyproject.org\u002Fhub\u002Fattachments\u002Fdocuments\u002Fpresentations\u002Fgcc2012\u002FWS1_blankenberg_gcc_2012_integrating_tools_and_data_sources.pdf\" target=\"_blank\" rel=\"noopener noreferrer\"\u003ESlides\u003C\u002Fa\u003E.\u003C\u002Fp\u003E\n\u003Cp\u003Emaf_slice.py:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E#!\u002Fusr\u002Fbin\u002Fenv python\n\n\"\"\"\nModified from interval2maf.py for teaching workshops\n\nReads a list of intervals and a maf. Produces a new maf containing the\nblocks or parts of blocks in the original that overlapped the intervals.\n\nIf a MAF file, not UID, is provided the MAF file is indexed before being processed.\n\nNOTE: If two intervals overlap the same block it will be written twice.\n\nusage: %prog maf_file [options]\n   -d, --dbkey=d: Database key, ie hg17\n   -c, --chromCol=c: Column of Chr\n   -s, --startCol=s: Column of Start\n   -e, --endCol=e: Column of End\n   -S, --strandCol=S: Column of Strand\n   -t, --mafType=t: Type of MAF source to use\n   -m, --mafFile=m: Path of source MAF file, if not using cached version\n   -I, --mafIndex=I: Path of precomputed source MAF file index, if not using cached version\n   -i, --interval_file=i:       Input interval file\n   -o, --output_file=o:      Output MAF file\n   -p, --species=p: Species to include in output\n   -P, --split_blocks_by_species=P: Split blocks by species\n   -r, --remove_all_gap_columns=r: Remove all Gap columns\n   -l, --indexLocation=l: Override default maf_index.loc file\n   -z, --mafIndexFile=z: Directory of local maf index file ( maf_index.loc or maf_pairwise.loc )\n\"\"\"\n\n#Dan Blankenberg\nfrom galaxy import eggs\nimport pkg_resources; pkg_resources.require( \"bx-python\" )\nfrom bx.cookbook import doc_optparse\nimport bx.align.maf\nimport bx.intervals.io\nfrom galaxy.tools.util import maf_utilities\nimport sys\n\nassert sys.version_info[:2] \u003E= ( 2, 4 )\n\ndef __main__():\n    index = index_filename = None\n    mincols = 0\n    \n    #Parse Command Line\n    options, args = doc_optparse.parse( __doc__ )\n    \n    if options.dbkey: dbkey = options.dbkey\n    else: dbkey = None\n    if dbkey in [None, \"?\"]:\n        maf_utilities.tool_fail( \"You must specify a proper build in order to extract alignments. You can specify your genome build by clicking on the pencil icon associated with your interval file.\" )\n    \n    species = maf_utilities.parse_species_option( options.species )\n    \n    if options.chromCol: chromCol = int( options.chromCol ) - 1\n    else: \n        chromCol = 0\n    \n    if options.startCol: startCol = int( options.startCol ) - 1\n    else: \n        startCol = 1\n    \n    if options.endCol: endCol = int( options.endCol ) - 1\n    else: \n        endCol = 2\n    \n    if options.strandCol: strandCol = int( options.strandCol ) - 1\n    else: \n        strandCol = -1\n    \n    if options.interval_file: interval_file = options.interval_file\n    else: \n        maf_utilities.tool_fail( \"Input interval file has not been specified.\" )\n    \n    if options.output_file: output_file = options.output_file\n    else: \n        maf_utilities.tool_fail( \"Output file has not been specified.\" )\n    \n    split_blocks_by_species = remove_all_gap_columns = False\n    if options.split_blocks_by_species and options.split_blocks_by_species == 'split_blocks_by_species':\n        split_blocks_by_species = True\n        if options.remove_all_gap_columns and options.remove_all_gap_columns == 'remove_all_gap_columns':\n            remove_all_gap_columns = True\n    else:\n        remove_all_gap_columns = True\n    #Finish parsing command line\n    \n    #Open indexed access to MAFs\n    if options.mafType:\n        if options.indexLocation:\n            index = maf_utilities.maf_index_by_uid( options.mafType, options.indexLocation )\n        else:\n            index = maf_utilities.maf_index_by_uid( options.mafType, options.mafIndexFile )\n        if index is None:\n            maf_utilities.tool_fail( \"The MAF source specified (%s) appears to be invalid.\" % ( options.mafType ) )\n    elif options.mafFile:\n        index, index_filename = maf_utilities.open_or_build_maf_index( options.mafFile, options.mafIndex, species = [dbkey] )\n        if index is None:\n            maf_utilities.tool_fail( \"Your MAF file appears to be malformed.\" )\n    else:\n        maf_utilities.tool_fail( \"Desired source MAF type has not been specified.\" )\n    \n    #Create MAF writter\n    out = bx.align.maf.Writer( open(output_file, \"w\") )\n    \n    #Iterate over input regions \n    num_blocks = 0\n    num_regions = None\n    for num_regions, region in enumerate( bx.intervals.io.NiceReaderWrapper( open( interval_file, 'r' ), chrom_col = chromCol, start_col = startCol, end_col = endCol, strand_col = strandCol, fix_strand = True, return_header = False, return_comments = False ) ):\n        src = maf_utilities.src_merge( dbkey, region.chrom )\n        for block in index.get_as_iterator( src, region.start, region.end ):\n            if split_blocks_by_species:\n                blocks = [ new_block for new_block in maf_utilities.iter_blocks_split_by_species( block ) if maf_utilities.component_overlaps_region( new_block.get_component_by_src_start( dbkey ), region ) ]\n            else:\n                blocks = [ block ]\n            for block in blocks:\n                block = maf_utilities.chop_block_by_region( block, src, region )\n                if block is not None:\n                    if species is not None:\n                        block = block.limit_to_species( species )\n                    block = maf_utilities.orient_block_by_region( block, src, region )\n                    if remove_all_gap_columns:\n                        block.remove_all_gap_columns()\n                    out.write( block )\n                    num_blocks += 1\n    \n    #Close output MAF\n    out.close()\n    \n    #remove index file if created during run\n    maf_utilities.remove_temp_index_file( index_filename )\n    \n    if num_blocks:\n        print \"%i MAF blocks extracted for %i regions.\" % ( num_blocks, ( num_regions + 1 ) )\n    elif num_regions is not None:\n        print \"No MAF blocks could be extracted for %i regions.\" % ( num_regions + 1 )\n    else:\n        print \"No valid regions have been provided.\"\n    \nif __name__ == \"__main__\": __main__()\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003Etool_conf.xml section:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E&#x3C;toolbox\u003E\n  &#x3C;section name=\"Workshop Demo\" id=\"workshop_demo\"\u003E\n    &#x3C;tool file=\"demo\u002Fmaf_slice.xml\" \u002F\u003E\n  &#x3C;\u002Fsection\u003E\n&#x3C;\u002Ftoolbox\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003EEnd of Exercise, adding a basic tool:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E&#x3C;tool id=\"maf_slice\" name=\"Slice MAF\" version=\"1.0.0\"\u003E\n  &#x3C;description\u003Eby intervals&#x3C;\u002Fdescription\u003E\n  &#x3C;command interpreter=\"python\"\u003E\nmaf_slice.py --dbkey=hg17 --mafFile=${maf_input} --interval_file=${interval_input}  --output_file=${maf_output}\n   &#x3C;\u002Fcommand\u003E\n   &#x3C;inputs\u003E\n    &#x3C;param format=\"maf\" name=\"maf_input\" label=\"Choose alignments\" type=\"data\"\u002F\u003E\n    &#x3C;param format=\"bed\" name=\"interval_input\" type=\"data\" label=\"Choose intervals\"\u002F\u003E\n   &#x3C;\u002Finputs\u003E\n   &#x3C;outputs\u003E\n     &#x3C;data format=\"maf\" name=\"maf_output\"\u002F\u003E\n   &#x3C;\u002Foutputs\u003E\n   &#x3C;tests\u003E\n   &#x3C;\u002Ftests\u003E\n   &#x3C;help\u003E\n  &#x3C;\u002Fhelp\u003E\n&#x3C;\u002Ftool\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003EEnd of Exercise adding metadata elements to tool:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E&#x3C;tool id=\"maf_slice\" name=\"Slice MAF\" version=\"1.0.1\"\u003E\n  &#x3C;description\u003Eby intervals&#x3C;\u002Fdescription\u003E\n  &#x3C;command interpreter=\"python\"\u003E\nmaf_slice.py --dbkey=${interval_input.dbkey} --mafFile=${maf_input} --interval_file=${interval_input}  --output_file=${maf_output}\n     --chromCol=${interval_input.metadata.chromCol} --startCol=${interval_input.metadata.startCol} --endCol=${interval_input.metadata.endCol} --strandCol=${interval_input.metadata.strandCol}\n     --mafIndex=${maf_input.metadata.maf_index}\n   &#x3C;\u002Fcommand\u003E\n   &#x3C;inputs\u003E\n    &#x3C;param format=\"maf\" name=\"maf_input\" label=\"Choose alignments\" type=\"data\"\u002F\u003E\n    &#x3C;param format=\"interval\" name=\"interval_input\" type=\"data\" label=\"Choose intervals\"\u002F\u003E\n   &#x3C;\u002Finputs\u003E\n   &#x3C;outputs\u003E\n     &#x3C;data format=\"maf\" name=\"maf_output\"\u002F\u003E\n   &#x3C;\u002Foutputs\u003E\n   &#x3C;tests\u003E\n   &#x3C;\u002Ftests\u003E\n   &#x3C;help\u003E\n  &#x3C;\u002Fhelp\u003E\n&#x3C;\u002Ftool\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003EEnd of Exercise, adding conditional:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E&#x3C;tool id=\"maf_slice\" name=\"Slice MAF\" version=\"1.0.2\"\u003E\n  &#x3C;description\u003Eby intervals&#x3C;\u002Fdescription\u003E\n  &#x3C;command interpreter=\"python\"\u003E\nmaf_slice.py --dbkey=${interval_input.dbkey} --mafFile=${maf_source_type.maf_input} --interval_file=${interval_input}  --output_file=${maf_output}\n     --chromCol=${interval_input.metadata.chromCol} --startCol=${interval_input.metadata.startCol} --endCol=${interval_input.metadata.endCol} --strandCol=${interval_input.metadata.strandCol}\n     --mafIndex=${maf_source_type.maf_input.metadata.maf_index}\n   &#x3C;\u002Fcommand\u003E\n   &#x3C;inputs\u003E\n    &#x3C;param format=\"interval\" name=\"interval_input\" type=\"data\" label=\"Choose intervals\"\u002F\u003E\n    &#x3C;conditional name=\"maf_source_type\"\u003E\n      &#x3C;param name=\"maf_source\" type=\"select\" label=\"MAF Source\"\u003E\n        &#x3C;option value=\"cached\" selected=\"true\"\u003ELocally Cached Alignments&#x3C;\u002Foption\u003E\n        &#x3C;option value=\"user\"\u003EAlignments in Your History&#x3C;\u002Foption\u003E\n      &#x3C;\u002Fparam\u003E\n      &#x3C;when value=\"user\"\u003E\n        &#x3C;param format=\"maf\" name=\"maf_input\" label=\"Choose alignments\" type=\"data\"\u002F\u003E\n      &#x3C;\u002Fwhen\u003E\n      &#x3C;when value=\"cached\"\u003E\n        &#x3C;!-- need some way to access the external data --\u003E\n      &#x3C;\u002Fwhen\u003E\n    &#x3C;\u002Fconditional\u003E\n   &#x3C;\u002Finputs\u003E\n   &#x3C;outputs\u003E\n     &#x3C;data format=\"maf\" name=\"maf_output\"\u002F\u003E\n   &#x3C;\u002Foutputs\u003E\n   &#x3C;tests\u003E\n   &#x3C;\u002Ftests\u003E\n   &#x3C;help\u003E\n  &#x3C;\u002Fhelp\u003E\n&#x3C;\u002Ftool\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003EEnd of Exercise, accessing .loc file:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E&#x3C;tool id=\"maf_slice\" name=\"Slice MAF\" version=\"1.0.3\"\u003E\n  &#x3C;description\u003Eby intervals&#x3C;\u002Fdescription\u003E\n  &#x3C;command interpreter=\"python\"\u003E\n     maf_slice.py --dbkey=${interval_input.dbkey} --interval_file=${interval_input}  --output_file=${maf_output}\n     --chromCol=${interval_input.metadata.chromCol} --startCol=${interval_input.metadata.startCol} --endCol=${interval_input.metadata.endCol} --strandCol=${interval_input.metadata.strandCol}\n     #if $maf_source_type.maf_source == \"user\":\n         --mafFile=${maf_source_type.maf_input}\n         --mafIndex=${maf_source_type.maf_input.metadata.maf_index}\n     #else:\n         --mafType=$maf_source_type.mafType\n         --mafIndexFile=${GALAXY_DATA_INDEX_DIR}\u002Fmaf_index.loc\n     #end if\n   &#x3C;\u002Fcommand\u003E\n   &#x3C;inputs\u003E\n    &#x3C;param format=\"interval\" name=\"interval_input\" type=\"data\" label=\"Choose intervals\"\u002F\u003E\n    &#x3C;conditional name=\"maf_source_type\"\u003E\n      &#x3C;param name=\"maf_source\" type=\"select\" label=\"MAF Source\"\u003E\n        &#x3C;option value=\"cached\" selected=\"true\"\u003ELocally Cached Alignments&#x3C;\u002Foption\u003E\n        &#x3C;option value=\"user\"\u003EAlignments in Your History&#x3C;\u002Foption\u003E\n      &#x3C;\u002Fparam\u003E\n      &#x3C;when value=\"user\"\u003E\n        &#x3C;param format=\"maf\" name=\"maf_input\" label=\"Choose alignments\" type=\"data\"\u002F\u003E\n      &#x3C;\u002Fwhen\u003E\n      &#x3C;when value=\"cached\"\u003E\n        &#x3C;param name=\"mafType\" type=\"select\" label=\"Choose alignments\"\u003E\n          &#x3C;options from_data_table=\"indexed_maf_files\"\u003E\n            &#x3C;filter type=\"data_meta\" ref=\"interval_input\" key=\"dbkey\" column=\"dbkey\" multiple=\"True\" separator=\",\"\u002F\u003E\n            &#x3C;validator type=\"no_options\" message=\"No alignments are available for the build associated with the selected interval file\"\u002F\u003E\n          &#x3C;\u002Foptions\u003E\n        &#x3C;\u002Fparam\u003E\n      &#x3C;\u002Fwhen\u003E\n    &#x3C;\u002Fconditional\u003E\n   &#x3C;\u002Finputs\u003E\n   &#x3C;outputs\u003E\n     &#x3C;data format=\"maf\" name=\"maf_output\"\u002F\u003E\n   &#x3C;\u002Foutputs\u003E\n   &#x3C;tests\u003E\n   &#x3C;\u002Ftests\u003E\n   &#x3C;help\u003E\n  &#x3C;\u002Fhelp\u003E\n&#x3C;\u002Ftool\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003EEnd of Exercise, selecting species from metadata values for MAF in history:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E&#x3C;tool id=\"maf_slice\" name=\"Slice MAF\" version=\"1.0.4\"\u003E\n  &#x3C;description\u003Eby intervals&#x3C;\u002Fdescription\u003E\n  &#x3C;command interpreter=\"python\"\u003E\n     maf_slice.py --dbkey=${interval_input.dbkey} --interval_file=${interval_input}  --output_file=${maf_output}\n     --chromCol=${interval_input.metadata.chromCol} --startCol=${interval_input.metadata.startCol} --endCol=${interval_input.metadata.endCol} --strandCol=${interval_input.metadata.strandCol}\n     #if $maf_source_type.maf_source == \"user\":\n         --mafFile=${maf_source_type.maf_input}\n         --mafIndex=${maf_source_type.maf_input.metadata.maf_index}\n         --species=${maf_source_type.species}\n     #else:\n         --mafType=$maf_source_type.mafType\n         --mafIndexFile=${GALAXY_DATA_INDEX_DIR}\u002Fmaf_index.loc\n     #end if\n   &#x3C;\u002Fcommand\u003E\n   &#x3C;inputs\u003E\n    &#x3C;param format=\"interval\" name=\"interval_input\" type=\"data\" label=\"Choose intervals\"\u002F\u003E\n    &#x3C;conditional name=\"maf_source_type\"\u003E\n      &#x3C;param name=\"maf_source\" type=\"select\" label=\"MAF Source\"\u003E\n        &#x3C;option value=\"cached\" selected=\"true\"\u003ELocally Cached Alignments&#x3C;\u002Foption\u003E\n        &#x3C;option value=\"user\"\u003EAlignments in Your History&#x3C;\u002Foption\u003E\n      &#x3C;\u002Fparam\u003E\n      &#x3C;when value=\"user\"\u003E\n        &#x3C;param format=\"maf\" name=\"maf_input\" label=\"Choose alignments\" type=\"data\"\u002F\u003E\n        &#x3C;param name=\"species\" type=\"select\" display=\"checkboxes\" multiple=\"true\" label=\"Choose species\" help=\"Select species to be included in the final alignment\"\u003E\n          &#x3C;options\u003E\n            &#x3C;filter type=\"data_meta\" ref=\"maf_input\" key=\"species\" \u002F\u003E\n          &#x3C;\u002Foptions\u003E\n        &#x3C;\u002Fparam\u003E\n      &#x3C;\u002Fwhen\u003E\n      &#x3C;when value=\"cached\"\u003E\n        &#x3C;param name=\"mafType\" type=\"select\" label=\"Choose alignments\"\u003E\n          &#x3C;options from_data_table=\"indexed_maf_files\"\u003E\n            &#x3C;filter type=\"data_meta\" ref=\"interval_input\" key=\"dbkey\" column=\"dbkey\" multiple=\"True\" separator=\",\"\u002F\u003E\n            &#x3C;validator type=\"no_options\" message=\"No alignments are available for the build associated with the selected interval file\"\u002F\u003E\n          &#x3C;\u002Foptions\u003E\n        &#x3C;\u002Fparam\u003E\n      &#x3C;\u002Fwhen\u003E\n    &#x3C;\u002Fconditional\u003E\n   &#x3C;\u002Finputs\u003E\n   &#x3C;outputs\u003E\n     &#x3C;data format=\"maf\" name=\"maf_output\"\u002F\u003E\n   &#x3C;\u002Foutputs\u003E\n   &#x3C;tests\u003E\n   &#x3C;\u002Ftests\u003E\n   &#x3C;help\u003E\n  &#x3C;\u002Fhelp\u003E\n&#x3C;\u002Ftool\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003EEnd of exercise, selecting species available from .loc file:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E&#x3C;tool id=\"maf_slice\" name=\"Slice MAF\" version=\"1.0.5\"\u003E\n  &#x3C;description\u003Eby intervals&#x3C;\u002Fdescription\u003E\n  &#x3C;command interpreter=\"python\"\u003E\n     maf_slice.py --dbkey=${interval_input.dbkey} --interval_file=${interval_input}  --output_file=${maf_output}\n     --chromCol=${interval_input.metadata.chromCol} --startCol=${interval_input.metadata.startCol} --endCol=${interval_input.metadata.endCol} --strandCol=${interval_input.metadata.strandCol}\n     --species=${maf_source_type.species}\n     #if $maf_source_type.maf_source == \"user\":\n         --mafFile=${maf_source_type.maf_input}\n         --mafIndex=${maf_source_type.maf_input.metadata.maf_index}\n     #else:\n         --mafType=$maf_source_type.mafType\n         --mafIndexFile=${GALAXY_DATA_INDEX_DIR}\u002Fmaf_index.loc\n     #end if\n   &#x3C;\u002Fcommand\u003E\n   &#x3C;inputs\u003E\n    &#x3C;param format=\"interval\" name=\"interval_input\" type=\"data\" label=\"Choose intervals\"\u002F\u003E\n    &#x3C;conditional name=\"maf_source_type\"\u003E\n      &#x3C;param name=\"maf_source\" type=\"select\" label=\"MAF Source\"\u003E\n        &#x3C;option value=\"cached\" selected=\"true\"\u003ELocally Cached Alignments&#x3C;\u002Foption\u003E\n        &#x3C;option value=\"user\"\u003EAlignments in Your History&#x3C;\u002Foption\u003E\n      &#x3C;\u002Fparam\u003E\n      &#x3C;when value=\"user\"\u003E\n        &#x3C;param format=\"maf\" name=\"maf_input\" label=\"Choose alignments\" type=\"data\"\u002F\u003E\n        &#x3C;param name=\"species\" type=\"select\" display=\"checkboxes\" multiple=\"true\" label=\"Choose species\" help=\"Select species to be included in the final alignment\"\u003E\n          &#x3C;options\u003E\n            &#x3C;filter type=\"data_meta\" ref=\"maf_input\" key=\"species\" \u002F\u003E\n          &#x3C;\u002Foptions\u003E\n        &#x3C;\u002Fparam\u003E\n      &#x3C;\u002Fwhen\u003E\n      &#x3C;when value=\"cached\"\u003E\n        &#x3C;param name=\"mafType\" type=\"select\" label=\"Choose alignments\"\u003E\n          &#x3C;options from_data_table=\"indexed_maf_files\"\u003E\n            &#x3C;filter type=\"data_meta\" ref=\"interval_input\" key=\"dbkey\" column=\"dbkey\" multiple=\"True\" separator=\",\"\u002F\u003E\n            &#x3C;validator type=\"no_options\" message=\"No alignments are available for the build associated with the selected interval file\"\u002F\u003E\n          &#x3C;\u002Foptions\u003E\n        &#x3C;\u002Fparam\u003E\n        &#x3C;param name=\"species\" type=\"select\" display=\"checkboxes\" multiple=\"true\" label=\"Choose species\" help=\"Select species to be included in the final alignment\"\u003E\n          &#x3C;options from_data_table=\"indexed_maf_files\"\u003E\n            &#x3C;column name=\"uid\" index=\"1\"\u002F\u003E\n            &#x3C;column name=\"value\" index=\"3\"\u002F\u003E\n            &#x3C;column name=\"name\" index=\"3\"\u002F\u003E\n            &#x3C;filter type=\"param_value\" ref=\"mafType\" column=\"uid\"\u002F\u003E\n            &#x3C;filter type=\"multiple_splitter\" column=\"name\" separator=\",\"\u002F\u003E\n          &#x3C;\u002Foptions\u003E\n        &#x3C;\u002Fparam\u003E\n      &#x3C;\u002Fwhen\u003E\n    &#x3C;\u002Fconditional\u003E\n   &#x3C;\u002Finputs\u003E\n   &#x3C;outputs\u003E\n     &#x3C;data format=\"maf\" name=\"maf_output\"\u002F\u003E\n   &#x3C;\u002Foutputs\u003E\n   &#x3C;tests\u003E\n   &#x3C;\u002Ftests\u003E\n   &#x3C;help\u003E\n  &#x3C;\u002Fhelp\u003E\n&#x3C;\u002Ftool\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n"}},"context":{}};(function(){var s;(s=document.currentScript||document.scripts[document.scripts.length-1]).parentNode.removeChild(s);}());</script><script src="/assets/js/app.431ed39e.js" defer></script><script src="/assets/js/page--src--templates--article-vue.077afa7b.js" defer></script><script src="/assets/js/page--src--templates--article-vue~page--src--templates--vue-article-vue.0d54471b.js" defer></script>
    <script src="//unpkg.com/bootstrap@4.5.0/dist/js/bootstrap.min.js"></script>
  </body>
</html>