<!DOCTYPE html>
<html data-html-server-rendered="true" lang="en" data-vue-tag="%7B%22lang%22:%7B%22ssr%22:%22en%22%7D%7D">
  <head>
    <title> - Galaxy Community Hub</title><meta name="gridsome:hash" content="64a9e8afa68c1466f7ef441294c96aecf9a68066"><meta data-vue-tag="ssr" charset="utf-8"><meta data-vue-tag="ssr" name="generator" content="Gridsome v0.7.23"><meta data-vue-tag="ssr" data-key="viewport" name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover"><meta data-vue-tag="ssr" data-key="format-detection" name="format-detection" content="telephone=no"><meta data-vue-tag="ssr" data-key="description" name="description" content="All about Galaxy and its community."><link data-vue-tag="ssr" rel="icon" href="data:,"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="16x16" href="/assets/static/favicon.ce0531f.7d4a227415c9d9b704d56a079d49a3e1.png"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="32x32" href="/assets/static/favicon.ac8d93a.7d4a227415c9d9b704d56a079d49a3e1.png"><link rel="preload" href="/assets/css/0.styles.5b1099da.css" as="style"><link rel="preload" href="/assets/js/app.6a2146ae.js" as="script"><link rel="preload" href="/assets/js/page--src--templates--article-vue.59a31f82.js" as="script"><link rel="prefetch" href="/assets/js/page--node-modules--gridsome--app--pages--404-vue.9b913011.js"><link rel="prefetch" href="/assets/js/page--src--pages--blog-vue.b9d631d5.js"><link rel="prefetch" href="/assets/js/page--src--pages--careers-vue.7bdfa9aa.js"><link rel="prefetch" href="/assets/js/page--src--pages--data-explorer-vue.50f81452.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--archive-vue.bdcb994f.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests--papercuts-vue.9624a935.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests-vue.3da6c251.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--webinars-vue.4cf7aa8a.js"><link rel="prefetch" href="/assets/js/page--src--pages--events-vue.3f22c4a9.js"><link rel="prefetch" href="/assets/js/page--src--pages--index-vue.8b7e39d4.js"><link rel="prefetch" href="/assets/js/page--src--pages--news-vue.357930b0.js"><link rel="prefetch" href="/assets/js/page--src--pages--search-vue.1a4d1477.js"><link rel="prefetch" href="/assets/js/page--src--pages--tests-vue.093ab612.js"><link rel="prefetch" href="/assets/js/page--src--pages--use-vue.c890644b.js"><link rel="prefetch" href="/assets/js/page--src--templates--platform-vue.0db261b0.js"><link rel="prefetch" href="/assets/js/page--src--templates--vue-article-vue.454443f9.js"><link rel="prefetch" href="/assets/js/vendors~page--src--pages--careers-vue~page--src--pages--use-vue~page--src--templates--article-vue~pa~efade918.c3041708.js"><link rel="prefetch" href="/assets/js/vendors~page--src--pages--search-vue.5670e8f9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--index-md.7ef1364e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--md--imgtests--imgdir--index-md.2e934c33.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--partials--index-md.eeba3cb6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--imgdir--index-md.92886b7f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--index-md.eab5717f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--index-md.e1baaf22.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--index-md.861c549e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--reldir--index-md.aa6f3c64.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--capacity-planning--index-md.bfc3219e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--index-md.bec55add.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--performance--cluster--legacy--index-md.e37560d2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--datatypes--index-md.534701f8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--disk-quotas--index-md.33c98ae0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--faq--index-md.8a142c3e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--data-model--index-md.dbe6d521.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--galaxy-ops--index-md.ae261eb0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--ten-simple-steps-galaxy-as-a-service--index-md.7f58c8af.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--tool-config-syntax--index-md.cd8060f8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--writing-tests--index-md.67221f9c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--allocation--index-md.de0ec1f4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--faq--index-md.4669ecb3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--index-md.cf67f110.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--ssh--index-md.0e243102.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--troubleshooting--index-md.35efbe50.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--capacity-planning--index-md.dc90802a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--getting-started--index-md.2ca2f401.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--index-md.8415a706.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--building--index-md.d98aeff8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--capacity-planning--index-md.4510948e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--cluster-types--index-md.cac4089c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--customizing--index-md.e2b286b6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--faq--index-md.2b285c3a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--index-md.ea2ebc17.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--pre-201703--index-md.11c5fe6a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--hadoop--index-md.bb206ee6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--ht-condor--index-md.9da34f44.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--index-md.233a66d8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--openstack--index-md.ab85a30a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--services--index-md.10c9f181.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--sharing--index-md.d6d1b6d6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--troubleshooting--index-md.9c79821f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--userdata--index-md.53f02454.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--future--index-md.57f97361.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--index-md.9651e1c9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-09--index-md.ad9dcbc5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-27--index-md.7471bbe4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-09-19--index-md.e1e5e8d8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-11-14--index-md.9f83078a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-01-16--index-md.4539d357.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-03-20--index-md.70ed6c95.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-05-15--index-md.4688a0b3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-11-20--index-md.f2ba774b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-02-19--index-md.79bdf65d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-04-16--index-md.4a242376.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-06-18--index-md.440620d8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-08-20--index-md.376ae2ed.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-10-15--index-md.27dc34e9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-02-18--index-md.fdb95229.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-04-21--index-md.f92b0f6d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-06-16--index-md.fe11d204.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-08-18--index-md.f52ed1f8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--index-md.9f7dc20a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--webinar-tech--index-md.d949fe23.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2012--index-md.c7266273.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2014--index-md.e4286fb0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--index-md.d66e06d1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--data--index-md.20e48791.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--datafiles--index-md.2d10c564.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--architecture--index-md.4629c460.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--bitbucket--index-md.74b0114b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--css--index-md.657cefe3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--getting-started--index-md.328bffc8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--index-md.09543e4e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--jsa--index-md.6375449f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--mercurial--index-md.86dc9b7b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--security-tool-tips--index-md.f4faf0d9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--galaxy-project--index-md.3e02395c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--index-md.4d9d00ba.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--papers--index-md.c7f4278a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--posters--index-md.75155b75.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--presentations--index-md.28312b2f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--videos--index-md.53f176ff.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--advanced-session--index-md.6bd7cd89.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--basics-session--index-md.a7aa2234.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--index-md.881dccf2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--logistics--index-md.1a0b965f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--publicity--index-md.5fc8d7c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--registration--index-md.a85019e7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--abstracts--index-md.f45d8543.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--index-md.41cde30b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--key-dates--index-md.eb4461fb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--lightning-talks--index-md.416549ba.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--logistics--index-md.a40269b5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--organizing-committee--index-md.b5fc35e6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--photos--index-md.a9d9d78d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--automation-strategies--index-md.53a84cb2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--bioinformatics-training--index-md.5576edcf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--end-users--index-md.0b5fa62e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--workflows-and-api--index-md.03c0504a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--index-md.b8fa162b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--promotion--index-md.f38b09af.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--register--index-md.f4320c18.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--sponsorships--index-md.3e9b6ca4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--training-day--index-md.30df8ca8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--index-md.53afc104.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--posters--index-md.5a1c5bbd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--talks--index-md.24bf003b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--clinical-use-hpc-file-systems--index-md.54a9c0d6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-admins--index-md.4e804331.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-blast--index-md.5d5d7573.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-professionals--index-md.ee96dc73.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--index-md.e819dfd1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--proteomics--index-md.adb5209b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--public-galaxy-servers--index-md.5b535d3b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--tool-developers--index-md.934ab089.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--index-md.fc9e5d60.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--key-dates--index-md.9feb96c8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--lightning--index-md.986b956c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--logistics--index-md.aafc45da.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--organizers--index-md.8129c4fe.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--photos--index-md.9f48888b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--program--index-md.afd0b377.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--promotion--index-md.84a5ce7a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--register--index-md.e8a8cb9e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--sponsorships--index-md.f3562bcc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--advance-tool-data--index-md.f69ec61d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--api--index-md.e7ae288d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--index-md.4419ab24.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--intro-tool-data--index-md.2505a2c4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--vms--index-md.31fa2f12.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--index-md.44077d78.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--posters--index-md.968309e7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--talks--index-md.3bd80461.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--branch-release-merge--index-md.cc4e0d0a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-admins--index-md.c1aa37dd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-end-users--index-md.3ddb7f3b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-training-network--index-md.43cdc565.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--heterogeneous--index-md.810f499a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--index-md.27205369.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--using-galaxy-ui-overview--index-md.770275d4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--hackathon--index-md.23dc0e2f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--index-md.b84a33d4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--key-dates--index-md.afa58daf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--lightning--index-md.6722fb11.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--logistics--index-md.74e0dab1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--organizers--index-md.80304844.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--photos--index-md.976fd862.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--program--index-md.d8e5ebc7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--promotion--index-md.577f91cb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--register--index-md.09b57f47.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--sponsor-exhibit--index-md.d28c30a0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--admin-walkthrough--index-md.e70f4130.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--api--index-md.3e466840.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--data-managers--index-md.c4c90feb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--index-md.09eceaad.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--vms--index-md.a9f99c81.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--cwl--index-md.572d144e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-management-challenges--index-md.6519a99a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-splitting-and-parallelism--index-md.f9225e89.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--elixir--index-md.83e340c0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--extended-workflow--index-md.84d907aa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--fots--index-md.9eb9fbae.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-admins--index-md.013a0661.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-community-uk--index-md.73da832e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-scientists--index-md.8271737e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-training-network--index-md.2c5cf472.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--index-md.50db5ff8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--data-hack-roundup--index-md.54209e2d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--header--index-md.99f7b8a4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--index-md.b607d569.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2016--bofs--index-md.1a1d782b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--abstracts--index-md.8a86f254.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--childcare--index-md.46870d68.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--cofest--index-md.558efeda.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--conduct--index-md.4cdcc7b1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--faq--index-md.54d3f72a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--index-md.03d62056.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--organizers--index-md.0ca6c313.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--promotion--index-md.a0d7a442.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--registration--index-md.be067239.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--sponsors--index-md.b823edb9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--training--index-md.8e06e99f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--venue--index-md.bd503783.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--abstracts--index-md.7206d890.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--childcare--index-md.7f23990a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--code-of-conduct--index-md.9c7749ad.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--collaboration--index-md.d2df60a9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--faq--index-md.3263be33.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--icebreaker--index-md.ab42ed08.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--index-md.2726d22d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--keynotes--index-md.dfee893e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--organizers--index-md.7cdb9d53.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--promotion--index-md.3881bf6d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--register--index-md.5ca35f56.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--sponsors--index-md.8fef601a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--training--index-md.37b79dad.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--venue--index-md.7c4dd44e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2014--index-md.286c0184.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2016--index-md.6c02f7a2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2014--index-md.e1f8864b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2016--index-md.eed2cd9b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--index-md.5356abdf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--trainingday--index-md.45d2d738.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2016--index-md.837d1830.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--example--vue-remark--index-md.972b7a38.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--faqs--index-md.ecb23aeb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-06--index-md.dfe44fb6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-07--index-md.c147ef3d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--icons--index-md.b3e66524.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--index-md.27f20470.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--logos--index-md.5cf6a844.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--news-graphics--index-md.68e69490.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--photos--index-md.96cfd5fc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--screenshots--index-md.6188b2ad.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--index-md.c0be3976.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--workflow-editor-unannotated--index-md.c7e004d5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--extract--index-md.1f732370.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--tool-panel--index-md.861a5048.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--variables--index-md.c521b297.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--faq--index-md.1591ea49.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--galaxy-pages--index-md.321b9ce8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--interval-operations--index-md.cab27468.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--main--faq--index-md.7ee220fd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--tools--requirements--index-md.bcb851a8.js"><link rel="stylesheet" href="/assets/css/0.styles.5b1099da.css"><noscript data-vue-tag="ssr"><style>.g-image--loading{display:none;}</style></noscript>
  </head>
  <body >
    <div data-server-rendered="true" id="app" class="layout" data-v-14d6d1e0><header id="masthead"><nav class="container navbar navbar-expand-lg navbar-dark justify-content-center"><a href="/" class="navbar-brand active"><img id="masthead-logo" src="/images/galaxy_logo_hub_white.svg" alt="Galaxy Community Hub" height="30"></a><div id="navbarToggleWrapper" class="collapse navbar-collapse mx-auto"><ul class="navbar-nav"><li class="nav-item"><a href="/use/" role="button" aria-expanded="false" class="nav-link dropdown">
          Use
        </a></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" aria-expanded="false" class="nav-link dropdown-toggle">
          Learn
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/learn/" class="dropdown-item">How to use Galaxy</a></li><li><a href="https://training.galaxyproject.org/" class="dropdown-item">Teach with Galaxy</a></li></ul></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" class="nav-link dropdown-toggle">
          Community
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/community/" class="dropdown-item">About</a></li><li><a href="/events/" class="dropdown-item active">Events</a></li><li><a href="/news/" class="dropdown-item">News</a></li><li><a href="/blog/" class="dropdown-item">Blog</a></li><li><a href="/publication-library/" class="dropdown-item">Publications</a></li><li><a href="/galaxy-project/statistics/" class="dropdown-item">Stats</a></li><li><a href="/careers/" class="dropdown-item">Careers</a></li></ul></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" class="nav-link dropdown-toggle">
          Deploy &amp; Develop
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/admin/get-galaxy/" class="dropdown-item">Install Galaxy</a></li><li><a href="/admin/" class="dropdown-item">Server administration</a></li><li><a href="/tools/" class="dropdown-item">Tools</a></li><li><a href="/develop/api/" class="dropdown-item">Working with the API</a></li><li><a href="/develop/" class="dropdown-item">Contributing</a></li><li><a href="/docs/" class="dropdown-item">Code documentation</a></li></ul></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" class="nav-link dropdown-toggle">
          Support
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/support/" class="dropdown-item">FAQ</a></li><li><a href="https://help.galaxyproject.org/" class="dropdown-item">Galaxy Help Forum</a></li></ul></li><li class="nav-item"><a href="/jxtx/" role="button" aria-expanded="false" class="nav-link dropdown">
          @jxtx
        </a></li></ul><ul class="navbar-nav ml-auto"><li><form method="get" action="/search/" class="my-2 navbar-form"><div class="input-group"><input id="search-input" name="q" placeholder="Search Galaxy" class="form-control"><span class="input-group-btn"><button type="submit" class="btn btn-secondary"><span class="fa fa-search"></span></button></span></div></form></li></ul></div></nav></header><section id="maincontainer" class="container"><header data-v-08ecca1f data-v-14d6d1e0><a href="/" class="link active" data-v-08ecca1f>← Back to Home</a><!----><h1 class="title" data-v-08ecca1f></h1><!----><section class="metadata freetext" data-v-08ecca1f><!----><!----><!----></section><!----><!----></header><div class="content markdown" data-v-14d6d1e0><p>Slides are available here: <a href="https://depot.galaxyproject.org/hub/attachments/documents/presentations/gcc2012/WS1_blankenberg_gcc_2012_integrating_tools_and_data_sources.pdf" target="_blank" rel="noopener noreferrer">Slides</a>.</p>
<p>maf_slice.py:</p>
<pre><code>#!/usr/bin/env python

"""
Modified from interval2maf.py for teaching workshops

Reads a list of intervals and a maf. Produces a new maf containing the
blocks or parts of blocks in the original that overlapped the intervals.

If a MAF file, not UID, is provided the MAF file is indexed before being processed.

NOTE: If two intervals overlap the same block it will be written twice.

usage: %prog maf_file [options]
   -d, --dbkey=d: Database key, ie hg17
   -c, --chromCol=c: Column of Chr
   -s, --startCol=s: Column of Start
   -e, --endCol=e: Column of End
   -S, --strandCol=S: Column of Strand
   -t, --mafType=t: Type of MAF source to use
   -m, --mafFile=m: Path of source MAF file, if not using cached version
   -I, --mafIndex=I: Path of precomputed source MAF file index, if not using cached version
   -i, --interval_file=i:       Input interval file
   -o, --output_file=o:      Output MAF file
   -p, --species=p: Species to include in output
   -P, --split_blocks_by_species=P: Split blocks by species
   -r, --remove_all_gap_columns=r: Remove all Gap columns
   -l, --indexLocation=l: Override default maf_index.loc file
   -z, --mafIndexFile=z: Directory of local maf index file ( maf_index.loc or maf_pairwise.loc )
"""

#Dan Blankenberg
from galaxy import eggs
import pkg_resources; pkg_resources.require( "bx-python" )
from bx.cookbook import doc_optparse
import bx.align.maf
import bx.intervals.io
from galaxy.tools.util import maf_utilities
import sys

assert sys.version_info[:2] >= ( 2, 4 )

def __main__():
    index = index_filename = None
    mincols = 0
    
    #Parse Command Line
    options, args = doc_optparse.parse( __doc__ )
    
    if options.dbkey: dbkey = options.dbkey
    else: dbkey = None
    if dbkey in [None, "?"]:
        maf_utilities.tool_fail( "You must specify a proper build in order to extract alignments. You can specify your genome build by clicking on the pencil icon associated with your interval file." )
    
    species = maf_utilities.parse_species_option( options.species )
    
    if options.chromCol: chromCol = int( options.chromCol ) - 1
    else: 
        chromCol = 0
    
    if options.startCol: startCol = int( options.startCol ) - 1
    else: 
        startCol = 1
    
    if options.endCol: endCol = int( options.endCol ) - 1
    else: 
        endCol = 2
    
    if options.strandCol: strandCol = int( options.strandCol ) - 1
    else: 
        strandCol = -1
    
    if options.interval_file: interval_file = options.interval_file
    else: 
        maf_utilities.tool_fail( "Input interval file has not been specified." )
    
    if options.output_file: output_file = options.output_file
    else: 
        maf_utilities.tool_fail( "Output file has not been specified." )
    
    split_blocks_by_species = remove_all_gap_columns = False
    if options.split_blocks_by_species and options.split_blocks_by_species == 'split_blocks_by_species':
        split_blocks_by_species = True
        if options.remove_all_gap_columns and options.remove_all_gap_columns == 'remove_all_gap_columns':
            remove_all_gap_columns = True
    else:
        remove_all_gap_columns = True
    #Finish parsing command line
    
    #Open indexed access to MAFs
    if options.mafType:
        if options.indexLocation:
            index = maf_utilities.maf_index_by_uid( options.mafType, options.indexLocation )
        else:
            index = maf_utilities.maf_index_by_uid( options.mafType, options.mafIndexFile )
        if index is None:
            maf_utilities.tool_fail( "The MAF source specified (%s) appears to be invalid." % ( options.mafType ) )
    elif options.mafFile:
        index, index_filename = maf_utilities.open_or_build_maf_index( options.mafFile, options.mafIndex, species = [dbkey] )
        if index is None:
            maf_utilities.tool_fail( "Your MAF file appears to be malformed." )
    else:
        maf_utilities.tool_fail( "Desired source MAF type has not been specified." )
    
    #Create MAF writter
    out = bx.align.maf.Writer( open(output_file, "w") )
    
    #Iterate over input regions 
    num_blocks = 0
    num_regions = None
    for num_regions, region in enumerate( bx.intervals.io.NiceReaderWrapper( open( interval_file, 'r' ), chrom_col = chromCol, start_col = startCol, end_col = endCol, strand_col = strandCol, fix_strand = True, return_header = False, return_comments = False ) ):
        src = maf_utilities.src_merge( dbkey, region.chrom )
        for block in index.get_as_iterator( src, region.start, region.end ):
            if split_blocks_by_species:
                blocks = [ new_block for new_block in maf_utilities.iter_blocks_split_by_species( block ) if maf_utilities.component_overlaps_region( new_block.get_component_by_src_start( dbkey ), region ) ]
            else:
                blocks = [ block ]
            for block in blocks:
                block = maf_utilities.chop_block_by_region( block, src, region )
                if block is not None:
                    if species is not None:
                        block = block.limit_to_species( species )
                    block = maf_utilities.orient_block_by_region( block, src, region )
                    if remove_all_gap_columns:
                        block.remove_all_gap_columns()
                    out.write( block )
                    num_blocks += 1
    
    #Close output MAF
    out.close()
    
    #remove index file if created during run
    maf_utilities.remove_temp_index_file( index_filename )
    
    if num_blocks:
        print "%i MAF blocks extracted for %i regions." % ( num_blocks, ( num_regions + 1 ) )
    elif num_regions is not None:
        print "No MAF blocks could be extracted for %i regions." % ( num_regions + 1 )
    else:
        print "No valid regions have been provided."
    
if __name__ == "__main__": __main__()
</code></pre>
<p>tool_conf.xml section:</p>
<pre><code>&#x3C;toolbox>
  &#x3C;section name="Workshop Demo" id="workshop_demo">
    &#x3C;tool file="demo/maf_slice.xml" />
  &#x3C;/section>
&#x3C;/toolbox>
</code></pre>
<p>End of Exercise, adding a basic tool:</p>
<pre><code>&#x3C;tool id="maf_slice" name="Slice MAF" version="1.0.0">
  &#x3C;description>by intervals&#x3C;/description>
  &#x3C;command interpreter="python">
maf_slice.py --dbkey=hg17 --mafFile=${maf_input} --interval_file=${interval_input}  --output_file=${maf_output}
   &#x3C;/command>
   &#x3C;inputs>
    &#x3C;param format="maf" name="maf_input" label="Choose alignments" type="data"/>
    &#x3C;param format="bed" name="interval_input" type="data" label="Choose intervals"/>
   &#x3C;/inputs>
   &#x3C;outputs>
     &#x3C;data format="maf" name="maf_output"/>
   &#x3C;/outputs>
   &#x3C;tests>
   &#x3C;/tests>
   &#x3C;help>
  &#x3C;/help>
&#x3C;/tool>
</code></pre>
<p>End of Exercise adding metadata elements to tool:</p>
<pre><code>&#x3C;tool id="maf_slice" name="Slice MAF" version="1.0.1">
  &#x3C;description>by intervals&#x3C;/description>
  &#x3C;command interpreter="python">
maf_slice.py --dbkey=${interval_input.dbkey} --mafFile=${maf_input} --interval_file=${interval_input}  --output_file=${maf_output}
     --chromCol=${interval_input.metadata.chromCol} --startCol=${interval_input.metadata.startCol} --endCol=${interval_input.metadata.endCol} --strandCol=${interval_input.metadata.strandCol}
     --mafIndex=${maf_input.metadata.maf_index}
   &#x3C;/command>
   &#x3C;inputs>
    &#x3C;param format="maf" name="maf_input" label="Choose alignments" type="data"/>
    &#x3C;param format="interval" name="interval_input" type="data" label="Choose intervals"/>
   &#x3C;/inputs>
   &#x3C;outputs>
     &#x3C;data format="maf" name="maf_output"/>
   &#x3C;/outputs>
   &#x3C;tests>
   &#x3C;/tests>
   &#x3C;help>
  &#x3C;/help>
&#x3C;/tool>
</code></pre>
<p>End of Exercise, adding conditional:</p>
<pre><code>&#x3C;tool id="maf_slice" name="Slice MAF" version="1.0.2">
  &#x3C;description>by intervals&#x3C;/description>
  &#x3C;command interpreter="python">
maf_slice.py --dbkey=${interval_input.dbkey} --mafFile=${maf_source_type.maf_input} --interval_file=${interval_input}  --output_file=${maf_output}
     --chromCol=${interval_input.metadata.chromCol} --startCol=${interval_input.metadata.startCol} --endCol=${interval_input.metadata.endCol} --strandCol=${interval_input.metadata.strandCol}
     --mafIndex=${maf_source_type.maf_input.metadata.maf_index}
   &#x3C;/command>
   &#x3C;inputs>
    &#x3C;param format="interval" name="interval_input" type="data" label="Choose intervals"/>
    &#x3C;conditional name="maf_source_type">
      &#x3C;param name="maf_source" type="select" label="MAF Source">
        &#x3C;option value="cached" selected="true">Locally Cached Alignments&#x3C;/option>
        &#x3C;option value="user">Alignments in Your History&#x3C;/option>
      &#x3C;/param>
      &#x3C;when value="user">
        &#x3C;param format="maf" name="maf_input" label="Choose alignments" type="data"/>
      &#x3C;/when>
      &#x3C;when value="cached">
        &#x3C;!-- need some way to access the external data -->
      &#x3C;/when>
    &#x3C;/conditional>
   &#x3C;/inputs>
   &#x3C;outputs>
     &#x3C;data format="maf" name="maf_output"/>
   &#x3C;/outputs>
   &#x3C;tests>
   &#x3C;/tests>
   &#x3C;help>
  &#x3C;/help>
&#x3C;/tool>
</code></pre>
<p>End of Exercise, accessing .loc file:</p>
<pre><code>&#x3C;tool id="maf_slice" name="Slice MAF" version="1.0.3">
  &#x3C;description>by intervals&#x3C;/description>
  &#x3C;command interpreter="python">
     maf_slice.py --dbkey=${interval_input.dbkey} --interval_file=${interval_input}  --output_file=${maf_output}
     --chromCol=${interval_input.metadata.chromCol} --startCol=${interval_input.metadata.startCol} --endCol=${interval_input.metadata.endCol} --strandCol=${interval_input.metadata.strandCol}
     #if $maf_source_type.maf_source == "user":
         --mafFile=${maf_source_type.maf_input}
         --mafIndex=${maf_source_type.maf_input.metadata.maf_index}
     #else:
         --mafType=$maf_source_type.mafType
         --mafIndexFile=${GALAXY_DATA_INDEX_DIR}/maf_index.loc
     #end if
   &#x3C;/command>
   &#x3C;inputs>
    &#x3C;param format="interval" name="interval_input" type="data" label="Choose intervals"/>
    &#x3C;conditional name="maf_source_type">
      &#x3C;param name="maf_source" type="select" label="MAF Source">
        &#x3C;option value="cached" selected="true">Locally Cached Alignments&#x3C;/option>
        &#x3C;option value="user">Alignments in Your History&#x3C;/option>
      &#x3C;/param>
      &#x3C;when value="user">
        &#x3C;param format="maf" name="maf_input" label="Choose alignments" type="data"/>
      &#x3C;/when>
      &#x3C;when value="cached">
        &#x3C;param name="mafType" type="select" label="Choose alignments">
          &#x3C;options from_data_table="indexed_maf_files">
            &#x3C;filter type="data_meta" ref="interval_input" key="dbkey" column="dbkey" multiple="True" separator=","/>
            &#x3C;validator type="no_options" message="No alignments are available for the build associated with the selected interval file"/>
          &#x3C;/options>
        &#x3C;/param>
      &#x3C;/when>
    &#x3C;/conditional>
   &#x3C;/inputs>
   &#x3C;outputs>
     &#x3C;data format="maf" name="maf_output"/>
   &#x3C;/outputs>
   &#x3C;tests>
   &#x3C;/tests>
   &#x3C;help>
  &#x3C;/help>
&#x3C;/tool>
</code></pre>
<p>End of Exercise, selecting species from metadata values for MAF in history:</p>
<pre><code>&#x3C;tool id="maf_slice" name="Slice MAF" version="1.0.4">
  &#x3C;description>by intervals&#x3C;/description>
  &#x3C;command interpreter="python">
     maf_slice.py --dbkey=${interval_input.dbkey} --interval_file=${interval_input}  --output_file=${maf_output}
     --chromCol=${interval_input.metadata.chromCol} --startCol=${interval_input.metadata.startCol} --endCol=${interval_input.metadata.endCol} --strandCol=${interval_input.metadata.strandCol}
     #if $maf_source_type.maf_source == "user":
         --mafFile=${maf_source_type.maf_input}
         --mafIndex=${maf_source_type.maf_input.metadata.maf_index}
         --species=${maf_source_type.species}
     #else:
         --mafType=$maf_source_type.mafType
         --mafIndexFile=${GALAXY_DATA_INDEX_DIR}/maf_index.loc
     #end if
   &#x3C;/command>
   &#x3C;inputs>
    &#x3C;param format="interval" name="interval_input" type="data" label="Choose intervals"/>
    &#x3C;conditional name="maf_source_type">
      &#x3C;param name="maf_source" type="select" label="MAF Source">
        &#x3C;option value="cached" selected="true">Locally Cached Alignments&#x3C;/option>
        &#x3C;option value="user">Alignments in Your History&#x3C;/option>
      &#x3C;/param>
      &#x3C;when value="user">
        &#x3C;param format="maf" name="maf_input" label="Choose alignments" type="data"/>
        &#x3C;param name="species" type="select" display="checkboxes" multiple="true" label="Choose species" help="Select species to be included in the final alignment">
          &#x3C;options>
            &#x3C;filter type="data_meta" ref="maf_input" key="species" />
          &#x3C;/options>
        &#x3C;/param>
      &#x3C;/when>
      &#x3C;when value="cached">
        &#x3C;param name="mafType" type="select" label="Choose alignments">
          &#x3C;options from_data_table="indexed_maf_files">
            &#x3C;filter type="data_meta" ref="interval_input" key="dbkey" column="dbkey" multiple="True" separator=","/>
            &#x3C;validator type="no_options" message="No alignments are available for the build associated with the selected interval file"/>
          &#x3C;/options>
        &#x3C;/param>
      &#x3C;/when>
    &#x3C;/conditional>
   &#x3C;/inputs>
   &#x3C;outputs>
     &#x3C;data format="maf" name="maf_output"/>
   &#x3C;/outputs>
   &#x3C;tests>
   &#x3C;/tests>
   &#x3C;help>
  &#x3C;/help>
&#x3C;/tool>
</code></pre>
<p>End of exercise, selecting species available from .loc file:</p>
<pre><code>&#x3C;tool id="maf_slice" name="Slice MAF" version="1.0.5">
  &#x3C;description>by intervals&#x3C;/description>
  &#x3C;command interpreter="python">
     maf_slice.py --dbkey=${interval_input.dbkey} --interval_file=${interval_input}  --output_file=${maf_output}
     --chromCol=${interval_input.metadata.chromCol} --startCol=${interval_input.metadata.startCol} --endCol=${interval_input.metadata.endCol} --strandCol=${interval_input.metadata.strandCol}
     --species=${maf_source_type.species}
     #if $maf_source_type.maf_source == "user":
         --mafFile=${maf_source_type.maf_input}
         --mafIndex=${maf_source_type.maf_input.metadata.maf_index}
     #else:
         --mafType=$maf_source_type.mafType
         --mafIndexFile=${GALAXY_DATA_INDEX_DIR}/maf_index.loc
     #end if
   &#x3C;/command>
   &#x3C;inputs>
    &#x3C;param format="interval" name="interval_input" type="data" label="Choose intervals"/>
    &#x3C;conditional name="maf_source_type">
      &#x3C;param name="maf_source" type="select" label="MAF Source">
        &#x3C;option value="cached" selected="true">Locally Cached Alignments&#x3C;/option>
        &#x3C;option value="user">Alignments in Your History&#x3C;/option>
      &#x3C;/param>
      &#x3C;when value="user">
        &#x3C;param format="maf" name="maf_input" label="Choose alignments" type="data"/>
        &#x3C;param name="species" type="select" display="checkboxes" multiple="true" label="Choose species" help="Select species to be included in the final alignment">
          &#x3C;options>
            &#x3C;filter type="data_meta" ref="maf_input" key="species" />
          &#x3C;/options>
        &#x3C;/param>
      &#x3C;/when>
      &#x3C;when value="cached">
        &#x3C;param name="mafType" type="select" label="Choose alignments">
          &#x3C;options from_data_table="indexed_maf_files">
            &#x3C;filter type="data_meta" ref="interval_input" key="dbkey" column="dbkey" multiple="True" separator=","/>
            &#x3C;validator type="no_options" message="No alignments are available for the build associated with the selected interval file"/>
          &#x3C;/options>
        &#x3C;/param>
        &#x3C;param name="species" type="select" display="checkboxes" multiple="true" label="Choose species" help="Select species to be included in the final alignment">
          &#x3C;options from_data_table="indexed_maf_files">
            &#x3C;column name="uid" index="1"/>
            &#x3C;column name="value" index="3"/>
            &#x3C;column name="name" index="3"/>
            &#x3C;filter type="param_value" ref="mafType" column="uid"/>
            &#x3C;filter type="multiple_splitter" column="name" separator=","/>
          &#x3C;/options>
        &#x3C;/param>
      &#x3C;/when>
    &#x3C;/conditional>
   &#x3C;/inputs>
   &#x3C;outputs>
     &#x3C;data format="maf" name="maf_output"/>
   &#x3C;/outputs>
   &#x3C;tests>
   &#x3C;/tests>
   &#x3C;help>
  &#x3C;/help>
&#x3C;/tool>
</code></pre>
</div><footer data-v-fcf53fc4 data-v-14d6d1e0><!----></footer><footer class="static-footer markdown"><p>The Galaxy Project is supported in part by NSF, NHGRI, The Huck Institutes of the Life Sciences, The Institute for CyberScience at Penn State, and Johns Hopkins University.</p>
</footer></section></div>
    <script>window.__INITIAL_STATE__={"data":{"article":{"id":"c1b6f505b57bf4cd463ec7834c816ccf","title":"","tease":"","category":null,"date":null,"days":null,"contact":"","contact_url":"","authors":"","location":"","location_url":"","source_blog":"","source_blog_url":"","skip_title_render":null,"links":[],"image":"","images":{},"external_url":"","content":"\u003Cp\u003ESlides are available here: \u003Ca href=\"https:\u002F\u002Fdepot.galaxyproject.org\u002Fhub\u002Fattachments\u002Fdocuments\u002Fpresentations\u002Fgcc2012\u002FWS1_blankenberg_gcc_2012_integrating_tools_and_data_sources.pdf\" target=\"_blank\" rel=\"noopener noreferrer\"\u003ESlides\u003C\u002Fa\u003E.\u003C\u002Fp\u003E\n\u003Cp\u003Emaf_slice.py:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E#!\u002Fusr\u002Fbin\u002Fenv python\n\n\"\"\"\nModified from interval2maf.py for teaching workshops\n\nReads a list of intervals and a maf. Produces a new maf containing the\nblocks or parts of blocks in the original that overlapped the intervals.\n\nIf a MAF file, not UID, is provided the MAF file is indexed before being processed.\n\nNOTE: If two intervals overlap the same block it will be written twice.\n\nusage: %prog maf_file [options]\n   -d, --dbkey=d: Database key, ie hg17\n   -c, --chromCol=c: Column of Chr\n   -s, --startCol=s: Column of Start\n   -e, --endCol=e: Column of End\n   -S, --strandCol=S: Column of Strand\n   -t, --mafType=t: Type of MAF source to use\n   -m, --mafFile=m: Path of source MAF file, if not using cached version\n   -I, --mafIndex=I: Path of precomputed source MAF file index, if not using cached version\n   -i, --interval_file=i:       Input interval file\n   -o, --output_file=o:      Output MAF file\n   -p, --species=p: Species to include in output\n   -P, --split_blocks_by_species=P: Split blocks by species\n   -r, --remove_all_gap_columns=r: Remove all Gap columns\n   -l, --indexLocation=l: Override default maf_index.loc file\n   -z, --mafIndexFile=z: Directory of local maf index file ( maf_index.loc or maf_pairwise.loc )\n\"\"\"\n\n#Dan Blankenberg\nfrom galaxy import eggs\nimport pkg_resources; pkg_resources.require( \"bx-python\" )\nfrom bx.cookbook import doc_optparse\nimport bx.align.maf\nimport bx.intervals.io\nfrom galaxy.tools.util import maf_utilities\nimport sys\n\nassert sys.version_info[:2] \u003E= ( 2, 4 )\n\ndef __main__():\n    index = index_filename = None\n    mincols = 0\n    \n    #Parse Command Line\n    options, args = doc_optparse.parse( __doc__ )\n    \n    if options.dbkey: dbkey = options.dbkey\n    else: dbkey = None\n    if dbkey in [None, \"?\"]:\n        maf_utilities.tool_fail( \"You must specify a proper build in order to extract alignments. You can specify your genome build by clicking on the pencil icon associated with your interval file.\" )\n    \n    species = maf_utilities.parse_species_option( options.species )\n    \n    if options.chromCol: chromCol = int( options.chromCol ) - 1\n    else: \n        chromCol = 0\n    \n    if options.startCol: startCol = int( options.startCol ) - 1\n    else: \n        startCol = 1\n    \n    if options.endCol: endCol = int( options.endCol ) - 1\n    else: \n        endCol = 2\n    \n    if options.strandCol: strandCol = int( options.strandCol ) - 1\n    else: \n        strandCol = -1\n    \n    if options.interval_file: interval_file = options.interval_file\n    else: \n        maf_utilities.tool_fail( \"Input interval file has not been specified.\" )\n    \n    if options.output_file: output_file = options.output_file\n    else: \n        maf_utilities.tool_fail( \"Output file has not been specified.\" )\n    \n    split_blocks_by_species = remove_all_gap_columns = False\n    if options.split_blocks_by_species and options.split_blocks_by_species == 'split_blocks_by_species':\n        split_blocks_by_species = True\n        if options.remove_all_gap_columns and options.remove_all_gap_columns == 'remove_all_gap_columns':\n            remove_all_gap_columns = True\n    else:\n        remove_all_gap_columns = True\n    #Finish parsing command line\n    \n    #Open indexed access to MAFs\n    if options.mafType:\n        if options.indexLocation:\n            index = maf_utilities.maf_index_by_uid( options.mafType, options.indexLocation )\n        else:\n            index = maf_utilities.maf_index_by_uid( options.mafType, options.mafIndexFile )\n        if index is None:\n            maf_utilities.tool_fail( \"The MAF source specified (%s) appears to be invalid.\" % ( options.mafType ) )\n    elif options.mafFile:\n        index, index_filename = maf_utilities.open_or_build_maf_index( options.mafFile, options.mafIndex, species = [dbkey] )\n        if index is None:\n            maf_utilities.tool_fail( \"Your MAF file appears to be malformed.\" )\n    else:\n        maf_utilities.tool_fail( \"Desired source MAF type has not been specified.\" )\n    \n    #Create MAF writter\n    out = bx.align.maf.Writer( open(output_file, \"w\") )\n    \n    #Iterate over input regions \n    num_blocks = 0\n    num_regions = None\n    for num_regions, region in enumerate( bx.intervals.io.NiceReaderWrapper( open( interval_file, 'r' ), chrom_col = chromCol, start_col = startCol, end_col = endCol, strand_col = strandCol, fix_strand = True, return_header = False, return_comments = False ) ):\n        src = maf_utilities.src_merge( dbkey, region.chrom )\n        for block in index.get_as_iterator( src, region.start, region.end ):\n            if split_blocks_by_species:\n                blocks = [ new_block for new_block in maf_utilities.iter_blocks_split_by_species( block ) if maf_utilities.component_overlaps_region( new_block.get_component_by_src_start( dbkey ), region ) ]\n            else:\n                blocks = [ block ]\n            for block in blocks:\n                block = maf_utilities.chop_block_by_region( block, src, region )\n                if block is not None:\n                    if species is not None:\n                        block = block.limit_to_species( species )\n                    block = maf_utilities.orient_block_by_region( block, src, region )\n                    if remove_all_gap_columns:\n                        block.remove_all_gap_columns()\n                    out.write( block )\n                    num_blocks += 1\n    \n    #Close output MAF\n    out.close()\n    \n    #remove index file if created during run\n    maf_utilities.remove_temp_index_file( index_filename )\n    \n    if num_blocks:\n        print \"%i MAF blocks extracted for %i regions.\" % ( num_blocks, ( num_regions + 1 ) )\n    elif num_regions is not None:\n        print \"No MAF blocks could be extracted for %i regions.\" % ( num_regions + 1 )\n    else:\n        print \"No valid regions have been provided.\"\n    \nif __name__ == \"__main__\": __main__()\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003Etool_conf.xml section:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E&#x3C;toolbox\u003E\n  &#x3C;section name=\"Workshop Demo\" id=\"workshop_demo\"\u003E\n    &#x3C;tool file=\"demo\u002Fmaf_slice.xml\" \u002F\u003E\n  &#x3C;\u002Fsection\u003E\n&#x3C;\u002Ftoolbox\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003EEnd of Exercise, adding a basic tool:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E&#x3C;tool id=\"maf_slice\" name=\"Slice MAF\" version=\"1.0.0\"\u003E\n  &#x3C;description\u003Eby intervals&#x3C;\u002Fdescription\u003E\n  &#x3C;command interpreter=\"python\"\u003E\nmaf_slice.py --dbkey=hg17 --mafFile=${maf_input} --interval_file=${interval_input}  --output_file=${maf_output}\n   &#x3C;\u002Fcommand\u003E\n   &#x3C;inputs\u003E\n    &#x3C;param format=\"maf\" name=\"maf_input\" label=\"Choose alignments\" type=\"data\"\u002F\u003E\n    &#x3C;param format=\"bed\" name=\"interval_input\" type=\"data\" label=\"Choose intervals\"\u002F\u003E\n   &#x3C;\u002Finputs\u003E\n   &#x3C;outputs\u003E\n     &#x3C;data format=\"maf\" name=\"maf_output\"\u002F\u003E\n   &#x3C;\u002Foutputs\u003E\n   &#x3C;tests\u003E\n   &#x3C;\u002Ftests\u003E\n   &#x3C;help\u003E\n  &#x3C;\u002Fhelp\u003E\n&#x3C;\u002Ftool\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003EEnd of Exercise adding metadata elements to tool:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E&#x3C;tool id=\"maf_slice\" name=\"Slice MAF\" version=\"1.0.1\"\u003E\n  &#x3C;description\u003Eby intervals&#x3C;\u002Fdescription\u003E\n  &#x3C;command interpreter=\"python\"\u003E\nmaf_slice.py --dbkey=${interval_input.dbkey} --mafFile=${maf_input} --interval_file=${interval_input}  --output_file=${maf_output}\n     --chromCol=${interval_input.metadata.chromCol} --startCol=${interval_input.metadata.startCol} --endCol=${interval_input.metadata.endCol} --strandCol=${interval_input.metadata.strandCol}\n     --mafIndex=${maf_input.metadata.maf_index}\n   &#x3C;\u002Fcommand\u003E\n   &#x3C;inputs\u003E\n    &#x3C;param format=\"maf\" name=\"maf_input\" label=\"Choose alignments\" type=\"data\"\u002F\u003E\n    &#x3C;param format=\"interval\" name=\"interval_input\" type=\"data\" label=\"Choose intervals\"\u002F\u003E\n   &#x3C;\u002Finputs\u003E\n   &#x3C;outputs\u003E\n     &#x3C;data format=\"maf\" name=\"maf_output\"\u002F\u003E\n   &#x3C;\u002Foutputs\u003E\n   &#x3C;tests\u003E\n   &#x3C;\u002Ftests\u003E\n   &#x3C;help\u003E\n  &#x3C;\u002Fhelp\u003E\n&#x3C;\u002Ftool\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003EEnd of Exercise, adding conditional:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E&#x3C;tool id=\"maf_slice\" name=\"Slice MAF\" version=\"1.0.2\"\u003E\n  &#x3C;description\u003Eby intervals&#x3C;\u002Fdescription\u003E\n  &#x3C;command interpreter=\"python\"\u003E\nmaf_slice.py --dbkey=${interval_input.dbkey} --mafFile=${maf_source_type.maf_input} --interval_file=${interval_input}  --output_file=${maf_output}\n     --chromCol=${interval_input.metadata.chromCol} --startCol=${interval_input.metadata.startCol} --endCol=${interval_input.metadata.endCol} --strandCol=${interval_input.metadata.strandCol}\n     --mafIndex=${maf_source_type.maf_input.metadata.maf_index}\n   &#x3C;\u002Fcommand\u003E\n   &#x3C;inputs\u003E\n    &#x3C;param format=\"interval\" name=\"interval_input\" type=\"data\" label=\"Choose intervals\"\u002F\u003E\n    &#x3C;conditional name=\"maf_source_type\"\u003E\n      &#x3C;param name=\"maf_source\" type=\"select\" label=\"MAF Source\"\u003E\n        &#x3C;option value=\"cached\" selected=\"true\"\u003ELocally Cached Alignments&#x3C;\u002Foption\u003E\n        &#x3C;option value=\"user\"\u003EAlignments in Your History&#x3C;\u002Foption\u003E\n      &#x3C;\u002Fparam\u003E\n      &#x3C;when value=\"user\"\u003E\n        &#x3C;param format=\"maf\" name=\"maf_input\" label=\"Choose alignments\" type=\"data\"\u002F\u003E\n      &#x3C;\u002Fwhen\u003E\n      &#x3C;when value=\"cached\"\u003E\n        &#x3C;!-- need some way to access the external data --\u003E\n      &#x3C;\u002Fwhen\u003E\n    &#x3C;\u002Fconditional\u003E\n   &#x3C;\u002Finputs\u003E\n   &#x3C;outputs\u003E\n     &#x3C;data format=\"maf\" name=\"maf_output\"\u002F\u003E\n   &#x3C;\u002Foutputs\u003E\n   &#x3C;tests\u003E\n   &#x3C;\u002Ftests\u003E\n   &#x3C;help\u003E\n  &#x3C;\u002Fhelp\u003E\n&#x3C;\u002Ftool\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003EEnd of Exercise, accessing .loc file:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E&#x3C;tool id=\"maf_slice\" name=\"Slice MAF\" version=\"1.0.3\"\u003E\n  &#x3C;description\u003Eby intervals&#x3C;\u002Fdescription\u003E\n  &#x3C;command interpreter=\"python\"\u003E\n     maf_slice.py --dbkey=${interval_input.dbkey} --interval_file=${interval_input}  --output_file=${maf_output}\n     --chromCol=${interval_input.metadata.chromCol} --startCol=${interval_input.metadata.startCol} --endCol=${interval_input.metadata.endCol} --strandCol=${interval_input.metadata.strandCol}\n     #if $maf_source_type.maf_source == \"user\":\n         --mafFile=${maf_source_type.maf_input}\n         --mafIndex=${maf_source_type.maf_input.metadata.maf_index}\n     #else:\n         --mafType=$maf_source_type.mafType\n         --mafIndexFile=${GALAXY_DATA_INDEX_DIR}\u002Fmaf_index.loc\n     #end if\n   &#x3C;\u002Fcommand\u003E\n   &#x3C;inputs\u003E\n    &#x3C;param format=\"interval\" name=\"interval_input\" type=\"data\" label=\"Choose intervals\"\u002F\u003E\n    &#x3C;conditional name=\"maf_source_type\"\u003E\n      &#x3C;param name=\"maf_source\" type=\"select\" label=\"MAF Source\"\u003E\n        &#x3C;option value=\"cached\" selected=\"true\"\u003ELocally Cached Alignments&#x3C;\u002Foption\u003E\n        &#x3C;option value=\"user\"\u003EAlignments in Your History&#x3C;\u002Foption\u003E\n      &#x3C;\u002Fparam\u003E\n      &#x3C;when value=\"user\"\u003E\n        &#x3C;param format=\"maf\" name=\"maf_input\" label=\"Choose alignments\" type=\"data\"\u002F\u003E\n      &#x3C;\u002Fwhen\u003E\n      &#x3C;when value=\"cached\"\u003E\n        &#x3C;param name=\"mafType\" type=\"select\" label=\"Choose alignments\"\u003E\n          &#x3C;options from_data_table=\"indexed_maf_files\"\u003E\n            &#x3C;filter type=\"data_meta\" ref=\"interval_input\" key=\"dbkey\" column=\"dbkey\" multiple=\"True\" separator=\",\"\u002F\u003E\n            &#x3C;validator type=\"no_options\" message=\"No alignments are available for the build associated with the selected interval file\"\u002F\u003E\n          &#x3C;\u002Foptions\u003E\n        &#x3C;\u002Fparam\u003E\n      &#x3C;\u002Fwhen\u003E\n    &#x3C;\u002Fconditional\u003E\n   &#x3C;\u002Finputs\u003E\n   &#x3C;outputs\u003E\n     &#x3C;data format=\"maf\" name=\"maf_output\"\u002F\u003E\n   &#x3C;\u002Foutputs\u003E\n   &#x3C;tests\u003E\n   &#x3C;\u002Ftests\u003E\n   &#x3C;help\u003E\n  &#x3C;\u002Fhelp\u003E\n&#x3C;\u002Ftool\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003EEnd of Exercise, selecting species from metadata values for MAF in history:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E&#x3C;tool id=\"maf_slice\" name=\"Slice MAF\" version=\"1.0.4\"\u003E\n  &#x3C;description\u003Eby intervals&#x3C;\u002Fdescription\u003E\n  &#x3C;command interpreter=\"python\"\u003E\n     maf_slice.py --dbkey=${interval_input.dbkey} --interval_file=${interval_input}  --output_file=${maf_output}\n     --chromCol=${interval_input.metadata.chromCol} --startCol=${interval_input.metadata.startCol} --endCol=${interval_input.metadata.endCol} --strandCol=${interval_input.metadata.strandCol}\n     #if $maf_source_type.maf_source == \"user\":\n         --mafFile=${maf_source_type.maf_input}\n         --mafIndex=${maf_source_type.maf_input.metadata.maf_index}\n         --species=${maf_source_type.species}\n     #else:\n         --mafType=$maf_source_type.mafType\n         --mafIndexFile=${GALAXY_DATA_INDEX_DIR}\u002Fmaf_index.loc\n     #end if\n   &#x3C;\u002Fcommand\u003E\n   &#x3C;inputs\u003E\n    &#x3C;param format=\"interval\" name=\"interval_input\" type=\"data\" label=\"Choose intervals\"\u002F\u003E\n    &#x3C;conditional name=\"maf_source_type\"\u003E\n      &#x3C;param name=\"maf_source\" type=\"select\" label=\"MAF Source\"\u003E\n        &#x3C;option value=\"cached\" selected=\"true\"\u003ELocally Cached Alignments&#x3C;\u002Foption\u003E\n        &#x3C;option value=\"user\"\u003EAlignments in Your History&#x3C;\u002Foption\u003E\n      &#x3C;\u002Fparam\u003E\n      &#x3C;when value=\"user\"\u003E\n        &#x3C;param format=\"maf\" name=\"maf_input\" label=\"Choose alignments\" type=\"data\"\u002F\u003E\n        &#x3C;param name=\"species\" type=\"select\" display=\"checkboxes\" multiple=\"true\" label=\"Choose species\" help=\"Select species to be included in the final alignment\"\u003E\n          &#x3C;options\u003E\n            &#x3C;filter type=\"data_meta\" ref=\"maf_input\" key=\"species\" \u002F\u003E\n          &#x3C;\u002Foptions\u003E\n        &#x3C;\u002Fparam\u003E\n      &#x3C;\u002Fwhen\u003E\n      &#x3C;when value=\"cached\"\u003E\n        &#x3C;param name=\"mafType\" type=\"select\" label=\"Choose alignments\"\u003E\n          &#x3C;options from_data_table=\"indexed_maf_files\"\u003E\n            &#x3C;filter type=\"data_meta\" ref=\"interval_input\" key=\"dbkey\" column=\"dbkey\" multiple=\"True\" separator=\",\"\u002F\u003E\n            &#x3C;validator type=\"no_options\" message=\"No alignments are available for the build associated with the selected interval file\"\u002F\u003E\n          &#x3C;\u002Foptions\u003E\n        &#x3C;\u002Fparam\u003E\n      &#x3C;\u002Fwhen\u003E\n    &#x3C;\u002Fconditional\u003E\n   &#x3C;\u002Finputs\u003E\n   &#x3C;outputs\u003E\n     &#x3C;data format=\"maf\" name=\"maf_output\"\u002F\u003E\n   &#x3C;\u002Foutputs\u003E\n   &#x3C;tests\u003E\n   &#x3C;\u002Ftests\u003E\n   &#x3C;help\u003E\n  &#x3C;\u002Fhelp\u003E\n&#x3C;\u002Ftool\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003EEnd of exercise, selecting species available from .loc file:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E&#x3C;tool id=\"maf_slice\" name=\"Slice MAF\" version=\"1.0.5\"\u003E\n  &#x3C;description\u003Eby intervals&#x3C;\u002Fdescription\u003E\n  &#x3C;command interpreter=\"python\"\u003E\n     maf_slice.py --dbkey=${interval_input.dbkey} --interval_file=${interval_input}  --output_file=${maf_output}\n     --chromCol=${interval_input.metadata.chromCol} --startCol=${interval_input.metadata.startCol} --endCol=${interval_input.metadata.endCol} --strandCol=${interval_input.metadata.strandCol}\n     --species=${maf_source_type.species}\n     #if $maf_source_type.maf_source == \"user\":\n         --mafFile=${maf_source_type.maf_input}\n         --mafIndex=${maf_source_type.maf_input.metadata.maf_index}\n     #else:\n         --mafType=$maf_source_type.mafType\n         --mafIndexFile=${GALAXY_DATA_INDEX_DIR}\u002Fmaf_index.loc\n     #end if\n   &#x3C;\u002Fcommand\u003E\n   &#x3C;inputs\u003E\n    &#x3C;param format=\"interval\" name=\"interval_input\" type=\"data\" label=\"Choose intervals\"\u002F\u003E\n    &#x3C;conditional name=\"maf_source_type\"\u003E\n      &#x3C;param name=\"maf_source\" type=\"select\" label=\"MAF Source\"\u003E\n        &#x3C;option value=\"cached\" selected=\"true\"\u003ELocally Cached Alignments&#x3C;\u002Foption\u003E\n        &#x3C;option value=\"user\"\u003EAlignments in Your History&#x3C;\u002Foption\u003E\n      &#x3C;\u002Fparam\u003E\n      &#x3C;when value=\"user\"\u003E\n        &#x3C;param format=\"maf\" name=\"maf_input\" label=\"Choose alignments\" type=\"data\"\u002F\u003E\n        &#x3C;param name=\"species\" type=\"select\" display=\"checkboxes\" multiple=\"true\" label=\"Choose species\" help=\"Select species to be included in the final alignment\"\u003E\n          &#x3C;options\u003E\n            &#x3C;filter type=\"data_meta\" ref=\"maf_input\" key=\"species\" \u002F\u003E\n          &#x3C;\u002Foptions\u003E\n        &#x3C;\u002Fparam\u003E\n      &#x3C;\u002Fwhen\u003E\n      &#x3C;when value=\"cached\"\u003E\n        &#x3C;param name=\"mafType\" type=\"select\" label=\"Choose alignments\"\u003E\n          &#x3C;options from_data_table=\"indexed_maf_files\"\u003E\n            &#x3C;filter type=\"data_meta\" ref=\"interval_input\" key=\"dbkey\" column=\"dbkey\" multiple=\"True\" separator=\",\"\u002F\u003E\n            &#x3C;validator type=\"no_options\" message=\"No alignments are available for the build associated with the selected interval file\"\u002F\u003E\n          &#x3C;\u002Foptions\u003E\n        &#x3C;\u002Fparam\u003E\n        &#x3C;param name=\"species\" type=\"select\" display=\"checkboxes\" multiple=\"true\" label=\"Choose species\" help=\"Select species to be included in the final alignment\"\u003E\n          &#x3C;options from_data_table=\"indexed_maf_files\"\u003E\n            &#x3C;column name=\"uid\" index=\"1\"\u002F\u003E\n            &#x3C;column name=\"value\" index=\"3\"\u002F\u003E\n            &#x3C;column name=\"name\" index=\"3\"\u002F\u003E\n            &#x3C;filter type=\"param_value\" ref=\"mafType\" column=\"uid\"\u002F\u003E\n            &#x3C;filter type=\"multiple_splitter\" column=\"name\" separator=\",\"\u002F\u003E\n          &#x3C;\u002Foptions\u003E\n        &#x3C;\u002Fparam\u003E\n      &#x3C;\u002Fwhen\u003E\n    &#x3C;\u002Fconditional\u003E\n   &#x3C;\u002Finputs\u003E\n   &#x3C;outputs\u003E\n     &#x3C;data format=\"maf\" name=\"maf_output\"\u002F\u003E\n   &#x3C;\u002Foutputs\u003E\n   &#x3C;tests\u003E\n   &#x3C;\u002Ftests\u003E\n   &#x3C;help\u003E\n  &#x3C;\u002Fhelp\u003E\n&#x3C;\u002Ftool\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n"}},"context":{}};(function(){var s;(s=document.currentScript||document.scripts[document.scripts.length-1]).parentNode.removeChild(s);}());</script><script src="/assets/js/app.6a2146ae.js" defer></script><script src="/assets/js/page--src--templates--article-vue.59a31f82.js" defer></script>
  </body>
</html>
