<!DOCTYPE html>
<html data-html-server-rendered="true" lang="en" data-vue-tag="%7B%22lang%22:%7B%22ssr%22:%22en%22%7D%7D">
  <head>
    <title>Displaying Datasets at External Display Applications / Websites using GEDAs - Galaxy Community Hub</title><meta name="gridsome:hash" content="c32ecfdb33b075b0057598b5b9118f3554a398cf"><meta data-vue-tag="ssr" charset="utf-8"><meta data-vue-tag="ssr" name="generator" content="Gridsome v0.7.23"><meta data-vue-tag="ssr" data-key="viewport" name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover"><meta data-vue-tag="ssr" data-key="format-detection" name="format-detection" content="telephone=no"><meta data-vue-tag="ssr" data-key="description" name="description" content="All about Galaxy and its community."><link data-vue-tag="ssr" rel="icon" href="data:,"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="16x16" href="/assets/static/favicon.ce0531f.7d4a227415c9d9b704d56a079d49a3e1.png"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="32x32" href="/assets/static/favicon.ac8d93a.7d4a227415c9d9b704d56a079d49a3e1.png"><link rel="preload" href="/assets/css/0.styles.28333459.css" as="style"><link rel="preload" href="/assets/js/app.f7f9943a.js" as="script"><link rel="preload" href="/assets/js/page--src--templates--article-vue.9c961af0.js" as="script"><link rel="prefetch" href="/assets/js/page--node-modules--gridsome--app--pages--404-vue.54a07ec7.js"><link rel="prefetch" href="/assets/js/page--src--pages--blog-vue.72cfdf23.js"><link rel="prefetch" href="/assets/js/page--src--pages--careers-vue.51f7f627.js"><link rel="prefetch" href="/assets/js/page--src--pages--data-explorer-vue.cda0b563.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--archive-vue.54bd407e.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests--papercuts-vue.d73bf9af.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests-vue.f9ca327e.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--webinars-vue.2b0c4b1f.js"><link rel="prefetch" href="/assets/js/page--src--pages--events-vue.2bc0181e.js"><link rel="prefetch" href="/assets/js/page--src--pages--index-vue.9e6f30c2.js"><link rel="prefetch" href="/assets/js/page--src--pages--news-vue.c55be461.js"><link rel="prefetch" href="/assets/js/page--src--pages--tests-vue.4730afa8.js"><link rel="prefetch" href="/assets/js/page--src--pages--use-vue.8c7d1a05.js"><link rel="prefetch" href="/assets/js/page--src--templates--platform-vue.672f4f9e.js"><link rel="prefetch" href="/assets/js/page--src--templates--vue-article-vue.d03e3b77.js"><link rel="prefetch" href="/assets/js/vendors~page--src--pages--careers-vue~page--src--pages--use-vue~page--src--templates--article-vue~pa~efade918.71d44943.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--index-md.b120794f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--md--imgtests--imgdir--index-md.8a720dc7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--partials--index-md.88a2cc0b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--imgdir--index-md.a7da7426.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--index-md.ab7ce26b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--index-md.ac0f0339.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--index-md.7388aadd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--reldir--index-md.78e6d088.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--capacity-planning--index-md.532d558e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--index-md.f596bca3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--performance--cluster--legacy--index-md.9900d899.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--datatypes--index-md.37a06167.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--disk-quotas--index-md.99623880.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--faq--index-md.dfcae1f0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--data-model--index-md.f6811d25.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--galaxy-ops--index-md.da494b7d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--ten-simple-steps-galaxy-as-a-service--index-md.f0b31fa8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--tool-config-syntax--index-md.c133b049.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--writing-tests--index-md.d3ed2aee.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--allocation--index-md.88a6f777.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--faq--index-md.6b5ae7d2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--index-md.54b831ad.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--ssh--index-md.c431c940.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--troubleshooting--index-md.2bbf34e9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--capacity-planning--index-md.8d2c9c03.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--getting-started--index-md.63fe5edc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--index-md.1387a7bc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--building--index-md.ea5bc352.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--capacity-planning--index-md.bdfd10d5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--cluster-types--index-md.83943371.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--customizing--index-md.6b8b3678.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--faq--index-md.12c74d13.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--index-md.e3fa7dae.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--pre-201703--index-md.9b084a41.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--hadoop--index-md.95f1458a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--ht-condor--index-md.f973feff.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--index-md.37f11743.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--openstack--index-md.f4d0e62a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--services--index-md.e26413ae.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--sharing--index-md.c15fa6cb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--troubleshooting--index-md.953f49c4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--userdata--index-md.1116e088.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--future--index-md.ec7dff38.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--index-md.f0bd7fb5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-09--index-md.ed1d541b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-27--index-md.5a2cbe0c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-09-19--index-md.36fcc10e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-11-14--index-md.5155dfcc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-01-16--index-md.e740402b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-03-20--index-md.fa75b1de.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-05-15--index-md.ad74e436.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-11-20--index-md.6a1074f1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-02-19--index-md.bdefdea8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-04-16--index-md.ce613da7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-06-18--index-md.9253a34b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-08-20--index-md.8ac977c4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-10-15--index-md.3a558b10.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-02-18--index-md.ae82f681.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-04-21--index-md.4cd1f8fc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-06-16--index-md.7b1572c5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-08-18--index-md.5b3383d1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--index-md.da754856.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--webinar-tech--index-md.c5c416a6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2012--index-md.4eddb972.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2014--index-md.5d7b4578.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--index-md.afac4bb5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--data--index-md.f2ba0502.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--datafiles--index-md.4dfd4db7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--architecture--index-md.8035154b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--bitbucket--index-md.7cc7d8d1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--css--index-md.710209d9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--getting-started--index-md.930b3a22.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--index-md.1cbec221.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--jsa--index-md.134e6a35.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--mercurial--index-md.9026901a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--security-tool-tips--index-md.2d704e67.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--galaxy-project--index-md.230e8088.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--index-md.e5883590.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--papers--index-md.db9c7f1b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--posters--index-md.04432e3d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--presentations--index-md.92923a05.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--videos--index-md.f680b009.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--advanced-session--index-md.4227a5ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--basics-session--index-md.08c5b284.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--index-md.9844ef78.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--logistics--index-md.6f6b160a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--publicity--index-md.dfbc1a41.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--registration--index-md.5232161d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--abstracts--index-md.697fe169.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--index-md.ca51927e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--key-dates--index-md.606cfcac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--lightning-talks--index-md.6a60add5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--logistics--index-md.06ca864d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--organizing-committee--index-md.0500ed90.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--photos--index-md.ab68ac2c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--automation-strategies--index-md.2d8a9043.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--bioinformatics-training--index-md.43a55fdc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--end-users--index-md.5f46cd2a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--workflows-and-api--index-md.07da71b4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--index-md.cb8d0657.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--promotion--index-md.4eb64cde.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--register--index-md.49c36527.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--sponsorships--index-md.db0f3791.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--training-day--index-md.c8f12515.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--index-md.3c67479f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--posters--index-md.728aa734.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--talks--index-md.7f93c9db.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--clinical-use-hpc-file-systems--index-md.030a88bd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-admins--index-md.f12f7b4d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-blast--index-md.59e8820f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-professionals--index-md.dcb27276.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--index-md.cfaaca20.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--proteomics--index-md.abba045c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--public-galaxy-servers--index-md.bd34ea15.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--tool-developers--index-md.a5a0a8b6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--index-md.df6174ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--key-dates--index-md.f7630162.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--lightning--index-md.0dfb5a95.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--logistics--index-md.4759076c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--organizers--index-md.6fc11b96.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--photos--index-md.aef50972.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--program--index-md.05c5d87a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--promotion--index-md.e34ad376.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--register--index-md.8ea1785f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--sponsorships--index-md.3cde86d8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--advance-tool-data--index-md.2210b6c4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--api--index-md.57c06e35.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--index-md.4cd32016.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--intro-tool-data--index-md.75a7abbd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--vms--index-md.1c86a58b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--index-md.9abee1c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--posters--index-md.810cd33b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--talks--index-md.c9751906.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--branch-release-merge--index-md.f6ed714e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-admins--index-md.5f9fcb01.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-end-users--index-md.00b4cd3b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-training-network--index-md.a843f5f7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--heterogeneous--index-md.8313575f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--index-md.f878a9f6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--using-galaxy-ui-overview--index-md.5b284474.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--hackathon--index-md.20172bcf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--index-md.9ea6d61e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--key-dates--index-md.bac2859e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--lightning--index-md.96c8f509.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--logistics--index-md.c1318807.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--organizers--index-md.b183376c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--photos--index-md.e0b61b7a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--program--index-md.a93fc58b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--promotion--index-md.df1d2b77.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--register--index-md.54adaeb3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--sponsor-exhibit--index-md.64ef93be.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--admin-walkthrough--index-md.c82adb0e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--api--index-md.a936c5d1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--data-managers--index-md.348024f8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--index-md.030f1ebd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--vms--index-md.507d7d15.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--cwl--index-md.a8a9aa76.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-management-challenges--index-md.409c72aa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-splitting-and-parallelism--index-md.6332a3a3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--elixir--index-md.982da27f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--extended-workflow--index-md.f2b42ad0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--fots--index-md.52645a6e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-admins--index-md.ffda743c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-community-uk--index-md.060ecf60.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-scientists--index-md.ae9e7e59.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-training-network--index-md.a412fdd5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--index-md.22f27d14.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--data-hack-roundup--index-md.60bc6484.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--header--index-md.dbba4e7d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--index-md.2f85a581.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2016--bofs--index-md.5a3032e2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--abstracts--index-md.57e795e8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--childcare--index-md.096b34d2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--cofest--index-md.8b0a9b93.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--conduct--index-md.8e51f30c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--faq--index-md.0f0f1b46.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--index-md.641b44a6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--organizers--index-md.787e0ccf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--promotion--index-md.e6e58e52.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--registration--index-md.52f1b4f7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--sponsors--index-md.ef140359.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--training--index-md.ee0d3a7c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--venue--index-md.30256085.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--abstracts--index-md.137e7bc6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--childcare--index-md.84f642d3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--code-of-conduct--index-md.2789988f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--collaboration--index-md.a48ddf3b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--faq--index-md.4fd33733.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--icebreaker--index-md.ce20a106.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--index-md.9c798d19.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--keynotes--index-md.080e361c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--organizers--index-md.a3b956fb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--promotion--index-md.de985f84.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--register--index-md.05d6bc90.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--sponsors--index-md.4c398a30.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--training--index-md.ce0ef35c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--venue--index-md.4a3ddfec.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2014--index-md.d56576b3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2016--index-md.9f18f6ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2014--index-md.6fd15769.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2016--index-md.c0b34aae.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--index-md.ba91d0c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--trainingday--index-md.e89c50c3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2016--index-md.6afda26b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--example--vue-remark--index-md.c96a8cd8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--faqs--index-md.3bba7da9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-06--index-md.a5232112.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-07--index-md.19df6105.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--icons--index-md.ff92f6d4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--index-md.246f22cd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--logos--index-md.5a8a4cdf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--news-graphics--index-md.844f04b7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--photos--index-md.7a556820.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--screenshots--index-md.290a5433.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--index-md.467e2afe.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--workflow-editor-unannotated--index-md.bc93cd9f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--extract--index-md.940bc220.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--tool-panel--index-md.7ac3e984.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--variables--index-md.a3612f7c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--faq--index-md.599e559f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--galaxy-pages--index-md.51df5ee9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--interval-operations--index-md.71022235.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--main--faq--index-md.f6a425fa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--tools--requirements--index-md.1aef8fd3.js"><link rel="stylesheet" href="/assets/css/0.styles.28333459.css"><noscript data-vue-tag="ssr"><style>.g-image--loading{display:none;}</style></noscript>
  </head>
  <body >
    <div data-server-rendered="true" id="app" class="layout" data-v-3a128d75><header id="masthead"><nav class="container navbar navbar-expand-lg navbar-dark justify-content-center"><a href="/" class="navbar-brand active"><img id="masthead-logo" src="/images/galaxy_logo_hub_white.svg" alt="Galaxy Community Hub" height="30"></a><div id="navbarToggleWrapper" class="collapse navbar-collapse mx-auto"><ul class="navbar-nav"><li class="nav-item"><a href="/use/" role="button" aria-expanded="false" class="nav-link dropdown">
          Use
        </a></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" aria-expanded="false" class="nav-link dropdown-toggle">
          Learn
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/learn/" class="dropdown-item">How to use Galaxy</a></li><li><a href="https://training.galaxyproject.org/" class="dropdown-item">Teach with Galaxy</a></li></ul></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" class="nav-link dropdown-toggle">
          Community
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/community/" class="dropdown-item">About</a></li><li><a href="/events/" class="dropdown-item">Events</a></li><li><a href="/news/" class="dropdown-item">News</a></li><li><a href="/blog/" class="dropdown-item">Blog</a></li><li><a href="/publication-library/" class="dropdown-item">Publications</a></li><li><a href="/galaxy-project/statistics/" class="dropdown-item">Stats</a></li><li><a href="/careers/" class="dropdown-item">Careers</a></li></ul></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" class="nav-link dropdown-toggle">
          Deploy &amp; Develop
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/admin/get-galaxy/" class="dropdown-item">Install Galaxy</a></li><li><a href="/admin/" class="dropdown-item active">Server administration</a></li><li><a href="/tools/" class="dropdown-item">Tools</a></li><li><a href="/develop/api/" class="dropdown-item">Working with the API</a></li><li><a href="/develop/" class="dropdown-item">Contributing</a></li><li><a href="/docs/" class="dropdown-item">Code documentation</a></li></ul></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" class="nav-link dropdown-toggle">
          Support
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/support/" class="dropdown-item">FAQ</a></li><li><a href="https://help.galaxyproject.org/" class="dropdown-item">Galaxy Help Forum</a></li></ul></li><li class="nav-item"><a href="/jxtx/" role="button" aria-expanded="false" class="nav-link dropdown">
          @jxtx
        </a></li></ul></div></nav></header><section id="maincontainer" class="container"><a href="/" class="link active" data-v-3a128d75> ← Home</a><header data-v-3a128d75><!----><h1 class="title" data-v-3a128d75>Displaying Datasets at External Display Applications / Websites using GEDAs</h1><div class="metadata" data-v-3a128d75><!----><!----><!----></div></header><div class="content markdown" data-v-3a128d75><h1 id="displaying-datasets-at-external-display-applications--websites-using-gedas"><a href="#displaying-datasets-at-external-display-applications--websites-using-gedas" aria-hidden="true"><span class="icon icon-link"></span></a>Displaying Datasets at External Display Applications / Websites using GEDAs</h1>
<h2 id="about-display-applications"><a href="#about-display-applications" aria-hidden="true"><span class="icon icon-link"></span></a>About Display Applications</h2>
<p>There are many services available that allow users to upload their own data for use as viewing as e.g. 'custom tracks' in a genome browser. Examples of these include the UCSC genome browser, GeneTrack and <a href="http://gmod.org/wiki/GBrowse" target="_blank" rel="noopener noreferrer">GBrowse</a>. Adding a new external display applications requires two steps:</p>
<ol>
<li>define the display application (create a new XML definition file) and </li>
<li>instruct Galaxy to load the display application (edit datatypes_conf.xml with the location of the new XML file). </li>
</ol>
<hr>
<h2 id="citation"><a href="#citation" aria-hidden="true"><span class="icon icon-link"></span></a>Citation</h2>
<p>If you add a new external display application to Galaxy in your published work, please cite: <a href="/src/people/dan/index.md">Daniel Blankenberg</a>, <a href="/src/people/john-chilton/index.md">John Chilton</a>, and <a href="/src/people/nate/index.md">Nate Coraor</a>. <strong><a href="https://rdcu.be/b0xnC" target="_blank" rel="noopener noreferrer">Galaxy External Display Applications: Closing a dataflow interoperability loop</a></strong>. <em>Nat Methods</em> (2020). doi:10.1038/s41592-019-0727-x</p>
<hr>
<h2 id="additional-resources"><a href="#additional-resources" aria-hidden="true"><span class="icon icon-link"></span></a>Additional Resources</h2>
<ul>
<li>This preprint provides additional information on the structure and configuration of Galaxy External Display Applications (GEDAs): <a href="https://www.biorxiv.org/content/10.1101/642280v1.full" target="_blank" rel="noopener noreferrer">https://www.biorxiv.org/content/10.1101/642280v1.full</a></li>
<li>There is also an annotated XML Schema Definition (XSD) available: <a href="https://raw.githubusercontent.com/galaxyproject/galaxy/a50dcf092a1d33c5b097783131719f3f93616e75/lib/galaxy/datatypes/display_applications/xsd/geda.xsd" target="_blank" rel="noopener noreferrer">geda.xsd</a>.</li>
</ul>
<hr>
<h2 id="topics"><a href="#topics" aria-hidden="true"><span class="icon icon-link"></span></a>Topics</h2>
<ol>
<li>Basic Topics - Simple Display Applications </li>
<li>Advanced Topics - Dynamic Display Applications </li>
</ol>
<hr>
<h2 id="basic-topics"><a href="#basic-topics" aria-hidden="true"><span class="icon icon-link"></span></a>Basic Topics</h2>
<h3 id="simple-display-applications"><a href="#simple-display-applications" aria-hidden="true"><span class="icon icon-link"></span></a>Simple Display Applications</h3>
<p>Display applications are defined using XML.</p>
<h4 id="example-1"><a href="#example-1" aria-hidden="true"><span class="icon icon-link"></span></a>Example 1</h4>
<p>Lets suppose we want to write a display application which displays a BAM file at the UCSC genome browser. After familiarizing ourselves with the UCSC genome browser, we become aware of several pieces of information which are needed to display user data:</p>
<ul>
<li>The data to be displayed is provided by giving a public URL to the UCSC genome browser. </li>
<li>3 data files are needed to be provided by URL: 1) a custom track definition, 2) the BAM file and 3) the BAM index. With the following requirement: the index must have the same name as the BAM file, but have the additional suffix of '.bai' </li>
<li>The url to send data to UCSC is of the form: <code>http://genome.ucsc.edu/cgi-bin/hgTracks?db=UCSC_GENOME_BUILD&#x26;amp;hgt.customText=URL_OF_CUSTOM_TRACK</code> </li>
</ul>
<p>This display application will require 3 parameters, corresponding to the 3 data files that are needed.</p>
<ol>
<li>The custom track file is defined by a dynamically generated template-style parameter. It is referred to as 'track', and is available at <a href="http://some/url/path/track" target="_blank" rel="noopener noreferrer">http://some/url/path/track</a>. The contents of this file are created by replacing the '$' placeholders with the values indicated. </li>
<li>The BAM file, the actual dataset being used for the display, is internally referred to as 'bam_file', but will be available as <a href="http://some/url/path/galaxy.bam" target="_blank" rel="noopener noreferrer">http://some/url/path/galaxy.bam</a> </li>
<li>The index file, which is available as a metadata attribute of the BAM file, is internally referred to as 'bai_file', but will be available as <a href="http://some/url/path/galaxy.bam.bai" target="_blank" rel="noopener noreferrer">http://some/url/path/galaxy.bam.bai</a> </li>
</ol>
<p>This display application can then be defined like this example (<code>display_applications/ucsc/bam.xml/</code>):</p>
<pre><code>&#x3C;display id="ucsc_bam" version="1.0.0" name="display at UCSC">
    &#x3C;link id="main" name="main">
        &#x3C;url>http://genome.ucsc.edu/cgi-bin/hgTracks?db=${qp($bam_file.dbkey)}&#x26;amp;hgt.customText=${qp($track.url)}&#x3C;/url>
        &#x3C;param type="data" name="bam_file" url="galaxy.bam" />
        &#x3C;param type="data" name="bai_file" url="galaxy.bam.bai" metadata="bam_index" />&#x3C;!-- UCSC expects index file to exist as bam_file_name.bai -->
        &#x3C;param type="template" name="track" viewable="True">track type=bam name="${bam_file.name}" bigDataUrl=${bam_file.url} db=${bam_file.dbkey}&#x3C;/param>
    &#x3C;/link>
&#x3C;/display>
</code></pre>
<p>To instruct Galaxy to use this display app, we modify <code>datatypes_conf.xml</code> and add the following to the BAM datatype:</p>
<pre><code>&#x3C;datatype extension="bam" type="galaxy.datatypes.binary:Bam" mimetype="application/octet-stream" display_in_upload="true">
            &#x3C;display file="ucsc/bam.xml" />
        &#x3C;/datatype>
</code></pre>
<p><em>where our xml file was saved as /display_applications/ucsc/bam.xml</em></p>
<h4 id="example-2"><a href="#example-2" aria-hidden="true"><span class="icon icon-link"></span></a>Example 2</h4>
<p>Now lets create a display application that can view interval (BED-like) files at the UCSC genome browser. The UCSC genome browser natively handles BED files, but Galaxy allows a looser definition for BED files than that accepted at UCSC, so we have defined a new datatype called 'bedstrict' that has data that meets the strict definition of BED. To enable all types of intervals to be viewed at UCSC, we have created an interval to bedstrict datatype converter and defined it in the usual manner (LINK TO THIS DOCUMENTATION).</p>
<p>Items needed for this display:</p>
<ul>
<li>The data to be displayed is provided by giving a public URL to the UCSC genome browser. </li>
<li>1 data file is needed to be provided by URL: 1) the result of converting the interval file to bedstrict </li>
<li>The url to send data to UCSC is of the form: <code>http://genome.ucsc.edu/cgi-bin/hgTracks?db=UCSC_GENOME_BUILD&#x26;amp;position=CHR:START-END&#x26;amp;hgt.customText=URL_OF_STRICTBED_FILE</code> </li>
</ul>
<p>This display application requires 2 parameters:</p>
<ol>
<li>The interval file converted to bestrict, named 'bed_file', to be availabe as <a href="http://some/url/path/galaxy.bed" target="_blank" rel="noopener noreferrer">http://some/url/path/galaxy.bed</a> </li>
<li>
<p>The 'position' (viewport) in the form of CHR:START-END. This is calculated by looking at the first 10 lines of the bed file. This is not available as a URL (viewable is not set to True); but is instead substituted in as part of the URL that users are directed to for display.
`<code></code>
<display id="ucsc_interval_as_bed" version="1.0.0" name="display at UCSC"></p>
<pre><code>&#x3C;link id="main" name="main">
    &#x3C;url>http://genome.ucsc.edu/cgi-bin/hgTracks?db=${qp($bed_file.dbkey)}&#x26;amp;position=${position.qp}&#x26;amp;hgt.customText=${bed_file.qp}&#x3C;/url>
    &#x3C;param type="data" name="bed_file" url="galaxy.bed" format="bedstrict"/> &#x3C;!-- Galaxy allows BED files to contain non-standard fields beyond the first 3 columns, UCSC does not: force use of converter which will make strict BED6+ file -->
    &#x3C;param type="template" name="position" strip="True" >
</code></pre>
<p>#set line_count = 0
#set chrom = None
#set start = float( 'inf' )
#set end = 0
#for $line in open( $bed_file.file_name ):
#if $line_count > 10: ##10 max lines to check for view port
#break
#end if
#if not $line.startswith( "#" ):
#set $fields = $line.split( "\t" )
#try:
#if len( $fields ) >= max( $bed_file.metadata.startCol, $bed_file.metadata.endCol, $bed_file.metadata.chromCol ):
#if $chrom is None or $fields[$bed_file.metadata.chromCol - 1] == $chrom:
#set chrom = $fields[$bed_file.metadata.chromCol - 1]
#set start = min( $start, int( $fields[$bed_file.metadata.startCol - 1] ) )
#set end = max( $end, int( $fields[$bed_file.metadata.endCol - 1] ) )
#end if
#end if
#except:
#pass
#end try
#end if
#set line_count += 1
#end for
#if $chrom is not None:
${chrom}:${start}-${end + 1}
#else:
:-
#end if</p>
<pre><code>    &#x3C;/param>
&#x3C;/link>
</code></pre>
</display>
```
</li>
</ol>
<p><em>where the 'strip' attribute of 'position' indicates that whitespace should be removed from around the determined position before being substituted into the URL. This can also be accomplished using ordinary string operations like: <code>position=${qp(str( position).strip())}</code>. 'qp' is a shortcut to 'quote_plus' which escapes text for use as part of a URL</em></p>
<p>To instruct Galaxy to use this display app, we modify <code>datatypes_conf.xml</code> and add the following to the interval datatype:</p>
<pre><code>&#x3C;datatype extension="interval" type="galaxy.datatypes.interval:Interval" display_in_upload="true">
            &#x3C;converter file="interval_to_bedstrict_converter.xml" target_datatype="bedstrict"/>
            &#x3C;display file="ucsc/interval_as_bed.xml" inherit="True" />
        &#x3C;/datatype>
</code></pre>
<p><em>where our xml file was saved as /display_applications/ucsc/interval_as_bed.xml and inherit is set to True, so that datatypes subclassing interval (e.g. bed, bedstrict) also have the display application available</em></p>
<h4 id="example-3"><a href="#example-3" aria-hidden="true"><span class="icon icon-link"></span></a>Example 3</h4>
<p>Create a display application that can view interval (BED-like) files at GeneTrack. GeneTrack is a browser/peakcalling program that is a server distinct from Galaxy, but which is able to directly access the file system used for storing Galaxy datasets; this direct file system access requires individual Galaxy instances to have a (local) instance of GeneTrack installed.</p>
<p>Items needed for this display:</p>
<ul>
<li>1 data file is needed to be provided: 1) A GeneTrack index binary datatype </li>
<li>The data to be displayed is provided by giving an encoded filename to the GeneTrack server, which has access to the appropriate filesystem. </li>
<li>A Hashkey, used as an identity check by GeneTrack </li>
<li>GeneTrack is able to launch peakcalling jobs at Galaxy and so also requires the dataset id and the Galaxy URL used to run jobs </li>
<li>The url to send data to GeneTrack is of the form: <code>http://genetrack.g2.bx.psu.edu/galaxy?filename=ENCODED_FILENAME&#x26;amp;hashkey=HASHKEY&#x26;amp;input=DATASET_ID&#x26;amp;GALAXY_URL=POST_BACK_URL</code> </li>
</ul>
<p>This display application requires 5 parameters:</p>
<ol>
<li>The interval file converted to bed or genetrack format, named 'bed_file' </li>
<li>The bed or genetrack file converted to GeneTrack format, named 'genetrack_file' </li>
<li>'galaxy_url' containing the url that GeneTrack posts back to for calling peaks. </li>
<li>'hash_key' used as a verification check by GeneTrack </li>
<li>
<p>'encoded_filename' containing the filesystem location to the 'genetrack_file', which has been encoded using binascii.hexlify
`<code></code>
<display id="genetrack_interval" version="1.0.0" name="view in"></p>
<pre><code>&#x3C;link id="genetrack" name="GeneTrack">
    &#x3C;url target_frame="galaxy_main">http://genetrack.g2.bx.psu.edu/galaxy?filename=${encoded_filename.qp}&#x26;amp;hashkey=${hash_key.qp}&#x26;amp;input=${qp(str($genetrack_file.id))}&#x26;amp;GALAXY_URL=${galaxy_url.qp}&#x3C;/url>
    &#x3C;param type="data" name="bed_file" viewable="False" format="bed,genetrack"/> &#x3C;!-- for now, we'll explicitly take care of the multi-step conversion; walk genetrack datatype down as a conversion of genetrack to genetrack doesn't exist and would likely be pointless -->
    &#x3C;param type="data" dataset="bed_file" name="genetrack_file" format="genetrack" viewable="False" />
    &#x3C;param type="template" name="galaxy_url" strip="True" >
        ${BASE_URL}/tool_runner?tool_id=predict2genetrack
    &#x3C;/param>
    &#x3C;param type="template" name="hash_key" strip="True" >
        #from galaxy.util.hash_util import hmac_new
        ${hmac_new( $APP.config.tool_secret, $genetrack_file.file_name )}
    &#x3C;/param>
    &#x3C;param type="template" name="encoded_filename" strip="True" >
        #import binascii
        ${binascii.hexlify( $genetrack_file.file_name )}
    &#x3C;/param>
&#x3C;/link>
</code></pre>
</display>
```
</li>
</ol>
<p><em>where 'BASE_URL', 'APP' and qp are built-in parameters available to display applications</em></p>
<p>To instruct Galaxy to use this display app, we modify <code>datatypes_conf.xml</code> and add the following to the interval datatype:</p>
<pre><code>&#x3C;datatype extension="interval" type="galaxy.datatypes.interval:Interval" display_in_upload="true">
            &#x3C;converter file="interval_to_bed_converter.xml" target_datatype="bed"/>
            &#x3C;display file="genetrack.xml" inherit="True"/>
        &#x3C;/datatype>
</code></pre>
<p><em>where our xml file was saved as /display_applications/genetrack.xml and inherit is set to True, so that datatypes subclassing interval (e.g. bed, bedstrict) also have the display application available. Note that a BED to GeneTrack Converter has been defined for the BED datatype and is used to create the GeneTrack file to be displayed</em></p>
<hr>
<h2 id="advanced-topics"><a href="#advanced-topics" aria-hidden="true"><span class="icon icon-link"></span></a>Advanced Topics</h2>
<h3 id="dynamic-display-applications"><a href="#dynamic-display-applications" aria-hidden="true"><span class="icon icon-link"></span></a>Dynamic Display Applications</h3>
<p>Sometimes it can be desired to have links and parameters for a Display Application to come from an external file. In the following examples we will modify the above examples to be populated using data found in external files.</p>
<p>The file <code>tool-data/shared/ucsc/ucsc_build_sites.txt</code> contains tab delimited data, with one line per display site, consisting of columns:</p>
<ol>
<li>Site Name </li>
<li>Site URL </li>
<li>Comma-separated Genome Builds Available at this site </li>
</ol>
<p>The display will also be filtered based upon a Galaxy Application configuration variable <code>ucsc_display_sites</code> which is used to to restrict available sites to a list specified in universe_wsgi.ini; alternatively the non-desired sites can be removed by commenting or deleting them out in <code>tool-data/shared/ucsc/ucsc_build_sites.txt</code>.</p>
<h4 id="example-1-advanced"><a href="#example-1-advanced" aria-hidden="true"><span class="icon icon-link"></span></a>Example 1 Advanced</h4>
<p>Display a BAM file at the UCSC genome browser, dynamically loading sites from a text file (<code>tool-data/shared/ucsc/ucsc_build_sites.txt</code>).</p>
<pre><code>&#x3C;display id="ucsc_bam" version="1.0.0" name="display at UCSC">
    &#x3C;!-- Load links from file: one line to one link -->
    &#x3C;dynamic_links from_file="tool-data/shared/ucsc/ucsc_build_sites.txt" skip_startswith="#" id="0" name="0">


        &#x3C;!-- Define parameters by column from file, allow splitting on builds -->
        &#x3C;dynamic_param name="site_id" value="0"/>
        &#x3C;dynamic_param name="ucsc_link" value="1"/>
        &#x3C;dynamic_param name="builds" value="2" split="True" separator="," />


        &#x3C;!-- Filter out some of the links based upon matching site_id to a Galaxy application configuration parameter and by dataset dbkey -->
        &#x3C;filter>${site_id in $APP.config.ucsc_display_sites}&#x3C;/filter>
        &#x3C;filter>${dataset.dbkey in $builds}&#x3C;/filter>


        &#x3C;!-- We define url and params as normal, but values defined in dynamic_param are available by specified name -->
        &#x3C;url>${ucsc_link}db=${qp($bam_file.dbkey)}&#x26;amp;hgt.customText=${qp($track.url)}&#x3C;/url>
        &#x3C;param type="data" name="bam_file" url="galaxy_${DATASET_HASH}.bam" />
        &#x3C;param type="data" name="bai_file" url="galaxy_${DATASET_HASH}.bam.bai" metadata="bam_index" />&#x3C;!-- UCSC expects index file to exist as bam_file_name.bai -->
        &#x3C;param type="template" name="track" viewable="True">track type=bam name="${bam_file.name}" bigDataUrl=${bam_file.url} db=${bam_file.dbkey}&#x3C;/param>


    &#x3C;/dynamic_links>
&#x3C;/display>
</code></pre>
<h4 id="example-2-advanced"><a href="#example-2-advanced" aria-hidden="true"><span class="icon icon-link"></span></a>Example 2 Advanced</h4>
<p>Display an Interval file at the UCSC genome browser, dynamically loading sites from a text file (<code>tool-data/shared/ucsc/ucsc_build_sites.txt</code>).</p>
<pre><code>&#x3C;display id="ucsc_interval_as_bed" version="1.0.0" name="display at UCSC" inherit="True">
    &#x3C;!-- Load links from file: one line to one link -->
    &#x3C;dynamic_links from_file="tool-data/shared/ucsc/ucsc_build_sites.txt" skip_startswith="#" id="0" name="0">


        &#x3C;!-- Define parameters by column from file, allow splitting on builds -->
        &#x3C;dynamic_param name="site_id" value="0"/>
        &#x3C;dynamic_param name="ucsc_link" value="1"/>
        &#x3C;dynamic_param name="builds" value="2" split="True" separator="," />


        &#x3C;!-- Filter out some of the links based upon matching site_id to a Galaxy application configuration parameter and by dataset dbkey -->
        &#x3C;filter>${site_id in $APP.config.ucsc_display_sites}&#x3C;/filter>
        &#x3C;filter>${dataset.dbkey in $builds}&#x3C;/filter>


        &#x3C;!-- We define url and params as normal, but values defined in dynamic_param are available by specified name -->
        &#x3C;url>${ucsc_link}db=${qp($bed_file.dbkey)}&#x26;amp;position=${position.qp}&#x26;amp;hgt.customText=${bed_file.qp}&#x3C;/url>
        &#x3C;param type="data" name="bed_file" url="galaxy_${DATASET_HASH}.bed" format="bedstrict"/> &#x3C;!-- Galaxy allows BED files to contain non-standard fields beyond the first 3 columns, UCSC does not: force use of converter which will make strict BED6+ file -->
        &#x3C;param type="template" name="position" strip="True" >
#set line_count = 0
#set chrom = None
#set start = float( 'inf' )
#set end = 0
#for $line in open( $bed_file.file_name ):
    #if $line_count &#x26;gt; 10: ##10 max lines to check for view port
        #break
    #end if
    #if not $line.startswith( "#" ):
        #set $fields = $line.split( "\t" )
        #try:
            #if len( $fields ) &#x26;gt;= max( $bed_file.metadata.startCol, $bed_file.metadata.endCol, $bed_file.metadata.chromCol ):
                #if $chrom is None or $fields[$bed_file.metadata.chromCol - 1] == $chrom:
                    #set chrom = $fields[$bed_file.metadata.chromCol - 1]
                    #set start = min( $start, int( $fields[$bed_file.metadata.startCol - 1] ) )
                    #set end = max( $end, int( $fields[$bed_file.metadata.endCol - 1] ) )
                #end if
            #end if
        #except:
            #pass
        #end try
    #end if
    #set line_count += 1
#end for
#if $chrom is not None:
${chrom}:${start}-${end + 1}
#else:
:-
#end if
        &#x3C;/param>
    &#x3C;/dynamic_links>
&#x3C;/display>
</code></pre>
</div><footer class="static-footer markdown"><p>The Galaxy Project is supported in part by NSF, NHGRI, The Huck Institutes of the Life Sciences, The Institute for CyberScience at Penn State, and Johns Hopkins University.</p>
</footer></section></div>
    <script>window.__INITIAL_STATE__={"data":{"article":{"id":"2da14f40cae13eed0aea113c42cac4f3","title":"Displaying Datasets at External Display Applications \u002F Websites using GEDAs","tease":"","image":"","images":{},"category":null,"contact":"","date":null,"content":"\u003Ch1 id=\"displaying-datasets-at-external-display-applications--websites-using-gedas\"\u003E\u003Ca href=\"#displaying-datasets-at-external-display-applications--websites-using-gedas\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EDisplaying Datasets at External Display Applications \u002F Websites using GEDAs\u003C\u002Fh1\u003E\n\u003Ch2 id=\"about-display-applications\"\u003E\u003Ca href=\"#about-display-applications\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EAbout Display Applications\u003C\u002Fh2\u003E\n\u003Cp\u003EThere are many services available that allow users to upload their own data for use as viewing as e.g. 'custom tracks' in a genome browser. Examples of these include the UCSC genome browser, GeneTrack and \u003Ca href=\"http:\u002F\u002Fgmod.org\u002Fwiki\u002FGBrowse\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EGBrowse\u003C\u002Fa\u003E. Adding a new external display applications requires two steps:\u003C\u002Fp\u003E\n\u003Col\u003E\n\u003Cli\u003Edefine the display application (create a new XML definition file) and \u003C\u002Fli\u003E\n\u003Cli\u003Einstruct Galaxy to load the display application (edit datatypes_conf.xml with the location of the new XML file). \u003C\u002Fli\u003E\n\u003C\u002Fol\u003E\n\u003Chr\u003E\n\u003Ch2 id=\"citation\"\u003E\u003Ca href=\"#citation\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003ECitation\u003C\u002Fh2\u003E\n\u003Cp\u003EIf you add a new external display application to Galaxy in your published work, please cite: \u003Ca href=\"\u002Fsrc\u002Fpeople\u002Fdan\u002Findex.md\"\u003EDaniel Blankenberg\u003C\u002Fa\u003E, \u003Ca href=\"\u002Fsrc\u002Fpeople\u002Fjohn-chilton\u002Findex.md\"\u003EJohn Chilton\u003C\u002Fa\u003E, and \u003Ca href=\"\u002Fsrc\u002Fpeople\u002Fnate\u002Findex.md\"\u003ENate Coraor\u003C\u002Fa\u003E. \u003Cstrong\u003E\u003Ca href=\"https:\u002F\u002Frdcu.be\u002Fb0xnC\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EGalaxy External Display Applications: Closing a dataflow interoperability loop\u003C\u002Fa\u003E\u003C\u002Fstrong\u003E. \u003Cem\u003ENat Methods\u003C\u002Fem\u003E (2020). doi:10.1038\u002Fs41592-019-0727-x\u003C\u002Fp\u003E\n\u003Chr\u003E\n\u003Ch2 id=\"additional-resources\"\u003E\u003Ca href=\"#additional-resources\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EAdditional Resources\u003C\u002Fh2\u003E\n\u003Cul\u003E\n\u003Cli\u003EThis preprint provides additional information on the structure and configuration of Galaxy External Display Applications (GEDAs): \u003Ca href=\"https:\u002F\u002Fwww.biorxiv.org\u002Fcontent\u002F10.1101\u002F642280v1.full\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttps:\u002F\u002Fwww.biorxiv.org\u002Fcontent\u002F10.1101\u002F642280v1.full\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003EThere is also an annotated XML Schema Definition (XSD) available: \u003Ca href=\"https:\u002F\u002Fraw.githubusercontent.com\u002Fgalaxyproject\u002Fgalaxy\u002Fa50dcf092a1d33c5b097783131719f3f93616e75\u002Flib\u002Fgalaxy\u002Fdatatypes\u002Fdisplay_applications\u002Fxsd\u002Fgeda.xsd\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Egeda.xsd\u003C\u002Fa\u003E.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Chr\u003E\n\u003Ch2 id=\"topics\"\u003E\u003Ca href=\"#topics\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003ETopics\u003C\u002Fh2\u003E\n\u003Col\u003E\n\u003Cli\u003EBasic Topics - Simple Display Applications \u003C\u002Fli\u003E\n\u003Cli\u003EAdvanced Topics - Dynamic Display Applications \u003C\u002Fli\u003E\n\u003C\u002Fol\u003E\n\u003Chr\u003E\n\u003Ch2 id=\"basic-topics\"\u003E\u003Ca href=\"#basic-topics\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EBasic Topics\u003C\u002Fh2\u003E\n\u003Ch3 id=\"simple-display-applications\"\u003E\u003Ca href=\"#simple-display-applications\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003ESimple Display Applications\u003C\u002Fh3\u003E\n\u003Cp\u003EDisplay applications are defined using XML.\u003C\u002Fp\u003E\n\u003Ch4 id=\"example-1\"\u003E\u003Ca href=\"#example-1\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EExample 1\u003C\u002Fh4\u003E\n\u003Cp\u003ELets suppose we want to write a display application which displays a BAM file at the UCSC genome browser. After familiarizing ourselves with the UCSC genome browser, we become aware of several pieces of information which are needed to display user data:\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003EThe data to be displayed is provided by giving a public URL to the UCSC genome browser. \u003C\u002Fli\u003E\n\u003Cli\u003E3 data files are needed to be provided by URL: 1) a custom track definition, 2) the BAM file and 3) the BAM index. With the following requirement: the index must have the same name as the BAM file, but have the additional suffix of '.bai' \u003C\u002Fli\u003E\n\u003Cli\u003EThe url to send data to UCSC is of the form: \u003Ccode\u003Ehttp:\u002F\u002Fgenome.ucsc.edu\u002Fcgi-bin\u002FhgTracks?db=UCSC_GENOME_BUILD&#x26;amp;hgt.customText=URL_OF_CUSTOM_TRACK\u003C\u002Fcode\u003E \u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Cp\u003EThis display application will require 3 parameters, corresponding to the 3 data files that are needed.\u003C\u002Fp\u003E\n\u003Col\u003E\n\u003Cli\u003EThe custom track file is defined by a dynamically generated template-style parameter. It is referred to as 'track', and is available at \u003Ca href=\"http:\u002F\u002Fsome\u002Furl\u002Fpath\u002Ftrack\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttp:\u002F\u002Fsome\u002Furl\u002Fpath\u002Ftrack\u003C\u002Fa\u003E. The contents of this file are created by replacing the '$' placeholders with the values indicated. \u003C\u002Fli\u003E\n\u003Cli\u003EThe BAM file, the actual dataset being used for the display, is internally referred to as 'bam_file', but will be available as \u003Ca href=\"http:\u002F\u002Fsome\u002Furl\u002Fpath\u002Fgalaxy.bam\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttp:\u002F\u002Fsome\u002Furl\u002Fpath\u002Fgalaxy.bam\u003C\u002Fa\u003E \u003C\u002Fli\u003E\n\u003Cli\u003EThe index file, which is available as a metadata attribute of the BAM file, is internally referred to as 'bai_file', but will be available as \u003Ca href=\"http:\u002F\u002Fsome\u002Furl\u002Fpath\u002Fgalaxy.bam.bai\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttp:\u002F\u002Fsome\u002Furl\u002Fpath\u002Fgalaxy.bam.bai\u003C\u002Fa\u003E \u003C\u002Fli\u003E\n\u003C\u002Fol\u003E\n\u003Cp\u003EThis display application can then be defined like this example (\u003Ccode\u003Edisplay_applications\u002Fucsc\u002Fbam.xml\u002F\u003C\u002Fcode\u003E):\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E&#x3C;display id=\"ucsc_bam\" version=\"1.0.0\" name=\"display at UCSC\"\u003E\n    &#x3C;link id=\"main\" name=\"main\"\u003E\n        &#x3C;url\u003Ehttp:\u002F\u002Fgenome.ucsc.edu\u002Fcgi-bin\u002FhgTracks?db=${qp($bam_file.dbkey)}&#x26;amp;hgt.customText=${qp($track.url)}&#x3C;\u002Furl\u003E\n        &#x3C;param type=\"data\" name=\"bam_file\" url=\"galaxy.bam\" \u002F\u003E\n        &#x3C;param type=\"data\" name=\"bai_file\" url=\"galaxy.bam.bai\" metadata=\"bam_index\" \u002F\u003E&#x3C;!-- UCSC expects index file to exist as bam_file_name.bai --\u003E\n        &#x3C;param type=\"template\" name=\"track\" viewable=\"True\"\u003Etrack type=bam name=\"${bam_file.name}\" bigDataUrl=${bam_file.url} db=${bam_file.dbkey}&#x3C;\u002Fparam\u003E\n    &#x3C;\u002Flink\u003E\n&#x3C;\u002Fdisplay\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003ETo instruct Galaxy to use this display app, we modify \u003Ccode\u003Edatatypes_conf.xml\u003C\u002Fcode\u003E and add the following to the BAM datatype:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E&#x3C;datatype extension=\"bam\" type=\"galaxy.datatypes.binary:Bam\" mimetype=\"application\u002Foctet-stream\" display_in_upload=\"true\"\u003E\n            &#x3C;display file=\"ucsc\u002Fbam.xml\" \u002F\u003E\n        &#x3C;\u002Fdatatype\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003E\u003Cem\u003Ewhere our xml file was saved as \u002Fdisplay_applications\u002Fucsc\u002Fbam.xml\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Ch4 id=\"example-2\"\u003E\u003Ca href=\"#example-2\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EExample 2\u003C\u002Fh4\u003E\n\u003Cp\u003ENow lets create a display application that can view interval (BED-like) files at the UCSC genome browser. The UCSC genome browser natively handles BED files, but Galaxy allows a looser definition for BED files than that accepted at UCSC, so we have defined a new datatype called 'bedstrict' that has data that meets the strict definition of BED. To enable all types of intervals to be viewed at UCSC, we have created an interval to bedstrict datatype converter and defined it in the usual manner (LINK TO THIS DOCUMENTATION).\u003C\u002Fp\u003E\n\u003Cp\u003EItems needed for this display:\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003EThe data to be displayed is provided by giving a public URL to the UCSC genome browser. \u003C\u002Fli\u003E\n\u003Cli\u003E1 data file is needed to be provided by URL: 1) the result of converting the interval file to bedstrict \u003C\u002Fli\u003E\n\u003Cli\u003EThe url to send data to UCSC is of the form: \u003Ccode\u003Ehttp:\u002F\u002Fgenome.ucsc.edu\u002Fcgi-bin\u002FhgTracks?db=UCSC_GENOME_BUILD&#x26;amp;position=CHR:START-END&#x26;amp;hgt.customText=URL_OF_STRICTBED_FILE\u003C\u002Fcode\u003E \u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Cp\u003EThis display application requires 2 parameters:\u003C\u002Fp\u003E\n\u003Col\u003E\n\u003Cli\u003EThe interval file converted to bestrict, named 'bed_file', to be availabe as \u003Ca href=\"http:\u002F\u002Fsome\u002Furl\u002Fpath\u002Fgalaxy.bed\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttp:\u002F\u002Fsome\u002Furl\u002Fpath\u002Fgalaxy.bed\u003C\u002Fa\u003E \u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003EThe 'position' (viewport) in the form of CHR:START-END. This is calculated by looking at the first 10 lines of the bed file. This is not available as a URL (viewable is not set to True); but is instead substituted in as part of the URL that users are directed to for display.\n`\u003Ccode\u003E\u003C\u002Fcode\u003E\n\u003Cdisplay id=\"ucsc_interval_as_bed\" version=\"1.0.0\" name=\"display at UCSC\"\u003E\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E&#x3C;link id=\"main\" name=\"main\"\u003E\n    &#x3C;url\u003Ehttp:\u002F\u002Fgenome.ucsc.edu\u002Fcgi-bin\u002FhgTracks?db=${qp($bed_file.dbkey)}&#x26;amp;position=${position.qp}&#x26;amp;hgt.customText=${bed_file.qp}&#x3C;\u002Furl\u003E\n    &#x3C;param type=\"data\" name=\"bed_file\" url=\"galaxy.bed\" format=\"bedstrict\"\u002F\u003E &#x3C;!-- Galaxy allows BED files to contain non-standard fields beyond the first 3 columns, UCSC does not: force use of converter which will make strict BED6+ file --\u003E\n    &#x3C;param type=\"template\" name=\"position\" strip=\"True\" \u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003E#set line_count = 0\n#set chrom = None\n#set start = float( 'inf' )\n#set end = 0\n#for $line in open( $bed_file.file_name ):\n#if $line_count \u003E 10: ##10 max lines to check for view port\n#break\n#end if\n#if not $line.startswith( \"#\" ):\n#set $fields = $line.split( \"\\t\" )\n#try:\n#if len( $fields ) \u003E= max( $bed_file.metadata.startCol, $bed_file.metadata.endCol, $bed_file.metadata.chromCol ):\n#if $chrom is None or $fields[$bed_file.metadata.chromCol - 1] == $chrom:\n#set chrom = $fields[$bed_file.metadata.chromCol - 1]\n#set start = min( $start, int( $fields[$bed_file.metadata.startCol - 1] ) )\n#set end = max( $end, int( $fields[$bed_file.metadata.endCol - 1] ) )\n#end if\n#end if\n#except:\n#pass\n#end try\n#end if\n#set line_count += 1\n#end for\n#if $chrom is not None:\n${chrom}:${start}-${end + 1}\n#else:\n:-\n#end if\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E    &#x3C;\u002Fparam\u003E\n&#x3C;\u002Flink\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003C\u002Fdisplay\u003E\n```\n\u003C\u002Fli\u003E\n\u003C\u002Fol\u003E\n\u003Cp\u003E\u003Cem\u003Ewhere the 'strip' attribute of 'position' indicates that whitespace should be removed from around the determined position before being substituted into the URL. This can also be accomplished using ordinary string operations like: \u003Ccode\u003Eposition=${qp(str( position).strip())}\u003C\u002Fcode\u003E. 'qp' is a shortcut to 'quote_plus' which escapes text for use as part of a URL\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Cp\u003ETo instruct Galaxy to use this display app, we modify \u003Ccode\u003Edatatypes_conf.xml\u003C\u002Fcode\u003E and add the following to the interval datatype:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E&#x3C;datatype extension=\"interval\" type=\"galaxy.datatypes.interval:Interval\" display_in_upload=\"true\"\u003E\n            &#x3C;converter file=\"interval_to_bedstrict_converter.xml\" target_datatype=\"bedstrict\"\u002F\u003E\n            &#x3C;display file=\"ucsc\u002Finterval_as_bed.xml\" inherit=\"True\" \u002F\u003E\n        &#x3C;\u002Fdatatype\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003E\u003Cem\u003Ewhere our xml file was saved as \u002Fdisplay_applications\u002Fucsc\u002Finterval_as_bed.xml and inherit is set to True, so that datatypes subclassing interval (e.g. bed, bedstrict) also have the display application available\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Ch4 id=\"example-3\"\u003E\u003Ca href=\"#example-3\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EExample 3\u003C\u002Fh4\u003E\n\u003Cp\u003ECreate a display application that can view interval (BED-like) files at GeneTrack. GeneTrack is a browser\u002Fpeakcalling program that is a server distinct from Galaxy, but which is able to directly access the file system used for storing Galaxy datasets; this direct file system access requires individual Galaxy instances to have a (local) instance of GeneTrack installed.\u003C\u002Fp\u003E\n\u003Cp\u003EItems needed for this display:\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E1 data file is needed to be provided: 1) A GeneTrack index binary datatype \u003C\u002Fli\u003E\n\u003Cli\u003EThe data to be displayed is provided by giving an encoded filename to the GeneTrack server, which has access to the appropriate filesystem. \u003C\u002Fli\u003E\n\u003Cli\u003EA Hashkey, used as an identity check by GeneTrack \u003C\u002Fli\u003E\n\u003Cli\u003EGeneTrack is able to launch peakcalling jobs at Galaxy and so also requires the dataset id and the Galaxy URL used to run jobs \u003C\u002Fli\u003E\n\u003Cli\u003EThe url to send data to GeneTrack is of the form: \u003Ccode\u003Ehttp:\u002F\u002Fgenetrack.g2.bx.psu.edu\u002Fgalaxy?filename=ENCODED_FILENAME&#x26;amp;hashkey=HASHKEY&#x26;amp;input=DATASET_ID&#x26;amp;GALAXY_URL=POST_BACK_URL\u003C\u002Fcode\u003E \u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Cp\u003EThis display application requires 5 parameters:\u003C\u002Fp\u003E\n\u003Col\u003E\n\u003Cli\u003EThe interval file converted to bed or genetrack format, named 'bed_file' \u003C\u002Fli\u003E\n\u003Cli\u003EThe bed or genetrack file converted to GeneTrack format, named 'genetrack_file' \u003C\u002Fli\u003E\n\u003Cli\u003E'galaxy_url' containing the url that GeneTrack posts back to for calling peaks. \u003C\u002Fli\u003E\n\u003Cli\u003E'hash_key' used as a verification check by GeneTrack \u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E'encoded_filename' containing the filesystem location to the 'genetrack_file', which has been encoded using binascii.hexlify\n`\u003Ccode\u003E\u003C\u002Fcode\u003E\n\u003Cdisplay id=\"genetrack_interval\" version=\"1.0.0\" name=\"view in\"\u003E\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E&#x3C;link id=\"genetrack\" name=\"GeneTrack\"\u003E\n    &#x3C;url target_frame=\"galaxy_main\"\u003Ehttp:\u002F\u002Fgenetrack.g2.bx.psu.edu\u002Fgalaxy?filename=${encoded_filename.qp}&#x26;amp;hashkey=${hash_key.qp}&#x26;amp;input=${qp(str($genetrack_file.id))}&#x26;amp;GALAXY_URL=${galaxy_url.qp}&#x3C;\u002Furl\u003E\n    &#x3C;param type=\"data\" name=\"bed_file\" viewable=\"False\" format=\"bed,genetrack\"\u002F\u003E &#x3C;!-- for now, we'll explicitly take care of the multi-step conversion; walk genetrack datatype down as a conversion of genetrack to genetrack doesn't exist and would likely be pointless --\u003E\n    &#x3C;param type=\"data\" dataset=\"bed_file\" name=\"genetrack_file\" format=\"genetrack\" viewable=\"False\" \u002F\u003E\n    &#x3C;param type=\"template\" name=\"galaxy_url\" strip=\"True\" \u003E\n        ${BASE_URL}\u002Ftool_runner?tool_id=predict2genetrack\n    &#x3C;\u002Fparam\u003E\n    &#x3C;param type=\"template\" name=\"hash_key\" strip=\"True\" \u003E\n        #from galaxy.util.hash_util import hmac_new\n        ${hmac_new( $APP.config.tool_secret, $genetrack_file.file_name )}\n    &#x3C;\u002Fparam\u003E\n    &#x3C;param type=\"template\" name=\"encoded_filename\" strip=\"True\" \u003E\n        #import binascii\n        ${binascii.hexlify( $genetrack_file.file_name )}\n    &#x3C;\u002Fparam\u003E\n&#x3C;\u002Flink\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003C\u002Fdisplay\u003E\n```\n\u003C\u002Fli\u003E\n\u003C\u002Fol\u003E\n\u003Cp\u003E\u003Cem\u003Ewhere 'BASE_URL', 'APP' and qp are built-in parameters available to display applications\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Cp\u003ETo instruct Galaxy to use this display app, we modify \u003Ccode\u003Edatatypes_conf.xml\u003C\u002Fcode\u003E and add the following to the interval datatype:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E&#x3C;datatype extension=\"interval\" type=\"galaxy.datatypes.interval:Interval\" display_in_upload=\"true\"\u003E\n            &#x3C;converter file=\"interval_to_bed_converter.xml\" target_datatype=\"bed\"\u002F\u003E\n            &#x3C;display file=\"genetrack.xml\" inherit=\"True\"\u002F\u003E\n        &#x3C;\u002Fdatatype\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003E\u003Cem\u003Ewhere our xml file was saved as \u002Fdisplay_applications\u002Fgenetrack.xml and inherit is set to True, so that datatypes subclassing interval (e.g. bed, bedstrict) also have the display application available. Note that a BED to GeneTrack Converter has been defined for the BED datatype and is used to create the GeneTrack file to be displayed\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Chr\u003E\n\u003Ch2 id=\"advanced-topics\"\u003E\u003Ca href=\"#advanced-topics\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EAdvanced Topics\u003C\u002Fh2\u003E\n\u003Ch3 id=\"dynamic-display-applications\"\u003E\u003Ca href=\"#dynamic-display-applications\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EDynamic Display Applications\u003C\u002Fh3\u003E\n\u003Cp\u003ESometimes it can be desired to have links and parameters for a Display Application to come from an external file. In the following examples we will modify the above examples to be populated using data found in external files.\u003C\u002Fp\u003E\n\u003Cp\u003EThe file \u003Ccode\u003Etool-data\u002Fshared\u002Fucsc\u002Fucsc_build_sites.txt\u003C\u002Fcode\u003E contains tab delimited data, with one line per display site, consisting of columns:\u003C\u002Fp\u003E\n\u003Col\u003E\n\u003Cli\u003ESite Name \u003C\u002Fli\u003E\n\u003Cli\u003ESite URL \u003C\u002Fli\u003E\n\u003Cli\u003EComma-separated Genome Builds Available at this site \u003C\u002Fli\u003E\n\u003C\u002Fol\u003E\n\u003Cp\u003EThe display will also be filtered based upon a Galaxy Application configuration variable \u003Ccode\u003Eucsc_display_sites\u003C\u002Fcode\u003E which is used to to restrict available sites to a list specified in universe_wsgi.ini; alternatively the non-desired sites can be removed by commenting or deleting them out in \u003Ccode\u003Etool-data\u002Fshared\u002Fucsc\u002Fucsc_build_sites.txt\u003C\u002Fcode\u003E.\u003C\u002Fp\u003E\n\u003Ch4 id=\"example-1-advanced\"\u003E\u003Ca href=\"#example-1-advanced\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EExample 1 Advanced\u003C\u002Fh4\u003E\n\u003Cp\u003EDisplay a BAM file at the UCSC genome browser, dynamically loading sites from a text file (\u003Ccode\u003Etool-data\u002Fshared\u002Fucsc\u002Fucsc_build_sites.txt\u003C\u002Fcode\u003E).\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E&#x3C;display id=\"ucsc_bam\" version=\"1.0.0\" name=\"display at UCSC\"\u003E\n    &#x3C;!-- Load links from file: one line to one link --\u003E\n    &#x3C;dynamic_links from_file=\"tool-data\u002Fshared\u002Fucsc\u002Fucsc_build_sites.txt\" skip_startswith=\"#\" id=\"0\" name=\"0\"\u003E\n\n\n        &#x3C;!-- Define parameters by column from file, allow splitting on builds --\u003E\n        &#x3C;dynamic_param name=\"site_id\" value=\"0\"\u002F\u003E\n        &#x3C;dynamic_param name=\"ucsc_link\" value=\"1\"\u002F\u003E\n        &#x3C;dynamic_param name=\"builds\" value=\"2\" split=\"True\" separator=\",\" \u002F\u003E\n\n\n        &#x3C;!-- Filter out some of the links based upon matching site_id to a Galaxy application configuration parameter and by dataset dbkey --\u003E\n        &#x3C;filter\u003E${site_id in $APP.config.ucsc_display_sites}&#x3C;\u002Ffilter\u003E\n        &#x3C;filter\u003E${dataset.dbkey in $builds}&#x3C;\u002Ffilter\u003E\n\n\n        &#x3C;!-- We define url and params as normal, but values defined in dynamic_param are available by specified name --\u003E\n        &#x3C;url\u003E${ucsc_link}db=${qp($bam_file.dbkey)}&#x26;amp;hgt.customText=${qp($track.url)}&#x3C;\u002Furl\u003E\n        &#x3C;param type=\"data\" name=\"bam_file\" url=\"galaxy_${DATASET_HASH}.bam\" \u002F\u003E\n        &#x3C;param type=\"data\" name=\"bai_file\" url=\"galaxy_${DATASET_HASH}.bam.bai\" metadata=\"bam_index\" \u002F\u003E&#x3C;!-- UCSC expects index file to exist as bam_file_name.bai --\u003E\n        &#x3C;param type=\"template\" name=\"track\" viewable=\"True\"\u003Etrack type=bam name=\"${bam_file.name}\" bigDataUrl=${bam_file.url} db=${bam_file.dbkey}&#x3C;\u002Fparam\u003E\n\n\n    &#x3C;\u002Fdynamic_links\u003E\n&#x3C;\u002Fdisplay\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Ch4 id=\"example-2-advanced\"\u003E\u003Ca href=\"#example-2-advanced\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EExample 2 Advanced\u003C\u002Fh4\u003E\n\u003Cp\u003EDisplay an Interval file at the UCSC genome browser, dynamically loading sites from a text file (\u003Ccode\u003Etool-data\u002Fshared\u002Fucsc\u002Fucsc_build_sites.txt\u003C\u002Fcode\u003E).\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E&#x3C;display id=\"ucsc_interval_as_bed\" version=\"1.0.0\" name=\"display at UCSC\" inherit=\"True\"\u003E\n    &#x3C;!-- Load links from file: one line to one link --\u003E\n    &#x3C;dynamic_links from_file=\"tool-data\u002Fshared\u002Fucsc\u002Fucsc_build_sites.txt\" skip_startswith=\"#\" id=\"0\" name=\"0\"\u003E\n\n\n        &#x3C;!-- Define parameters by column from file, allow splitting on builds --\u003E\n        &#x3C;dynamic_param name=\"site_id\" value=\"0\"\u002F\u003E\n        &#x3C;dynamic_param name=\"ucsc_link\" value=\"1\"\u002F\u003E\n        &#x3C;dynamic_param name=\"builds\" value=\"2\" split=\"True\" separator=\",\" \u002F\u003E\n\n\n        &#x3C;!-- Filter out some of the links based upon matching site_id to a Galaxy application configuration parameter and by dataset dbkey --\u003E\n        &#x3C;filter\u003E${site_id in $APP.config.ucsc_display_sites}&#x3C;\u002Ffilter\u003E\n        &#x3C;filter\u003E${dataset.dbkey in $builds}&#x3C;\u002Ffilter\u003E\n\n\n        &#x3C;!-- We define url and params as normal, but values defined in dynamic_param are available by specified name --\u003E\n        &#x3C;url\u003E${ucsc_link}db=${qp($bed_file.dbkey)}&#x26;amp;position=${position.qp}&#x26;amp;hgt.customText=${bed_file.qp}&#x3C;\u002Furl\u003E\n        &#x3C;param type=\"data\" name=\"bed_file\" url=\"galaxy_${DATASET_HASH}.bed\" format=\"bedstrict\"\u002F\u003E &#x3C;!-- Galaxy allows BED files to contain non-standard fields beyond the first 3 columns, UCSC does not: force use of converter which will make strict BED6+ file --\u003E\n        &#x3C;param type=\"template\" name=\"position\" strip=\"True\" \u003E\n#set line_count = 0\n#set chrom = None\n#set start = float( 'inf' )\n#set end = 0\n#for $line in open( $bed_file.file_name ):\n    #if $line_count &#x26;gt; 10: ##10 max lines to check for view port\n        #break\n    #end if\n    #if not $line.startswith( \"#\" ):\n        #set $fields = $line.split( \"\\t\" )\n        #try:\n            #if len( $fields ) &#x26;gt;= max( $bed_file.metadata.startCol, $bed_file.metadata.endCol, $bed_file.metadata.chromCol ):\n                #if $chrom is None or $fields[$bed_file.metadata.chromCol - 1] == $chrom:\n                    #set chrom = $fields[$bed_file.metadata.chromCol - 1]\n                    #set start = min( $start, int( $fields[$bed_file.metadata.startCol - 1] ) )\n                    #set end = max( $end, int( $fields[$bed_file.metadata.endCol - 1] ) )\n                #end if\n            #end if\n        #except:\n            #pass\n        #end try\n    #end if\n    #set line_count += 1\n#end for\n#if $chrom is not None:\n${chrom}:${start}-${end + 1}\n#else:\n:-\n#end if\n        &#x3C;\u002Fparam\u003E\n    &#x3C;\u002Fdynamic_links\u003E\n&#x3C;\u002Fdisplay\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n"}},"context":{}};(function(){var s;(s=document.currentScript||document.scripts[document.scripts.length-1]).parentNode.removeChild(s);}());</script><script src="/assets/js/app.f7f9943a.js" defer></script><script src="/assets/js/page--src--templates--article-vue.9c961af0.js" defer></script>
  </body>
</html>
