<!DOCTYPE html>
<html data-html-server-rendered="true" lang="en" data-vue-tag="%7B%22lang%22:%7B%22ssr%22:%22en%22%7D%7D">
  <head>
    <title>Job and Tool Error Help - Galaxy Community Hub</title><meta name="gridsome:hash" content="c32ecfdb33b075b0057598b5b9118f3554a398cf"><meta data-vue-tag="ssr" charset="utf-8"><meta data-vue-tag="ssr" name="generator" content="Gridsome v0.7.23"><meta data-vue-tag="ssr" data-key="viewport" name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover"><meta data-vue-tag="ssr" data-key="format-detection" name="format-detection" content="telephone=no"><meta data-vue-tag="ssr" data-key="description" name="description" content="All about Galaxy and its community."><link data-vue-tag="ssr" rel="icon" href="data:,"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="16x16" href="/assets/static/favicon.ce0531f.7d4a227415c9d9b704d56a079d49a3e1.png"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="32x32" href="/assets/static/favicon.ac8d93a.7d4a227415c9d9b704d56a079d49a3e1.png"><link rel="preload" href="/assets/css/0.styles.28333459.css" as="style"><link rel="preload" href="/assets/js/app.f7f9943a.js" as="script"><link rel="preload" href="/assets/js/page--src--templates--article-vue.9c961af0.js" as="script"><link rel="prefetch" href="/assets/js/page--node-modules--gridsome--app--pages--404-vue.54a07ec7.js"><link rel="prefetch" href="/assets/js/page--src--pages--blog-vue.72cfdf23.js"><link rel="prefetch" href="/assets/js/page--src--pages--careers-vue.51f7f627.js"><link rel="prefetch" href="/assets/js/page--src--pages--data-explorer-vue.cda0b563.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--archive-vue.54bd407e.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests--papercuts-vue.d73bf9af.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests-vue.f9ca327e.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--webinars-vue.2b0c4b1f.js"><link rel="prefetch" href="/assets/js/page--src--pages--events-vue.2bc0181e.js"><link rel="prefetch" href="/assets/js/page--src--pages--index-vue.9e6f30c2.js"><link rel="prefetch" href="/assets/js/page--src--pages--news-vue.c55be461.js"><link rel="prefetch" href="/assets/js/page--src--pages--tests-vue.4730afa8.js"><link rel="prefetch" href="/assets/js/page--src--pages--use-vue.8c7d1a05.js"><link rel="prefetch" href="/assets/js/page--src--templates--platform-vue.672f4f9e.js"><link rel="prefetch" href="/assets/js/page--src--templates--vue-article-vue.d03e3b77.js"><link rel="prefetch" href="/assets/js/vendors~page--src--pages--careers-vue~page--src--pages--use-vue~page--src--templates--article-vue~pa~efade918.71d44943.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--index-md.b120794f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--md--imgtests--imgdir--index-md.8a720dc7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--partials--index-md.88a2cc0b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--imgdir--index-md.a7da7426.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--index-md.ab7ce26b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--index-md.ac0f0339.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--index-md.7388aadd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--reldir--index-md.78e6d088.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--capacity-planning--index-md.532d558e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--index-md.f596bca3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--performance--cluster--legacy--index-md.9900d899.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--datatypes--index-md.37a06167.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--disk-quotas--index-md.99623880.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--faq--index-md.dfcae1f0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--data-model--index-md.f6811d25.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--galaxy-ops--index-md.da494b7d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--ten-simple-steps-galaxy-as-a-service--index-md.f0b31fa8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--tool-config-syntax--index-md.c133b049.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--writing-tests--index-md.d3ed2aee.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--allocation--index-md.88a6f777.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--faq--index-md.6b5ae7d2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--index-md.54b831ad.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--ssh--index-md.c431c940.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--troubleshooting--index-md.2bbf34e9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--capacity-planning--index-md.8d2c9c03.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--getting-started--index-md.63fe5edc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--index-md.1387a7bc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--building--index-md.ea5bc352.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--capacity-planning--index-md.bdfd10d5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--cluster-types--index-md.83943371.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--customizing--index-md.6b8b3678.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--faq--index-md.12c74d13.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--index-md.e3fa7dae.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--pre-201703--index-md.9b084a41.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--hadoop--index-md.95f1458a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--ht-condor--index-md.f973feff.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--index-md.37f11743.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--openstack--index-md.f4d0e62a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--services--index-md.e26413ae.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--sharing--index-md.c15fa6cb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--troubleshooting--index-md.953f49c4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--userdata--index-md.1116e088.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--future--index-md.ec7dff38.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--index-md.f0bd7fb5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-09--index-md.ed1d541b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-27--index-md.5a2cbe0c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-09-19--index-md.36fcc10e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-11-14--index-md.5155dfcc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-01-16--index-md.e740402b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-03-20--index-md.fa75b1de.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-05-15--index-md.ad74e436.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-11-20--index-md.6a1074f1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-02-19--index-md.bdefdea8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-04-16--index-md.ce613da7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-06-18--index-md.9253a34b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-08-20--index-md.8ac977c4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-10-15--index-md.3a558b10.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-02-18--index-md.ae82f681.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-04-21--index-md.4cd1f8fc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-06-16--index-md.7b1572c5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-08-18--index-md.5b3383d1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--index-md.da754856.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--webinar-tech--index-md.c5c416a6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2012--index-md.4eddb972.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2014--index-md.5d7b4578.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--index-md.afac4bb5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--data--index-md.f2ba0502.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--datafiles--index-md.4dfd4db7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--architecture--index-md.8035154b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--bitbucket--index-md.7cc7d8d1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--css--index-md.710209d9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--getting-started--index-md.930b3a22.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--index-md.1cbec221.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--jsa--index-md.134e6a35.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--mercurial--index-md.9026901a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--security-tool-tips--index-md.2d704e67.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--galaxy-project--index-md.230e8088.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--index-md.e5883590.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--papers--index-md.db9c7f1b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--posters--index-md.04432e3d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--presentations--index-md.92923a05.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--videos--index-md.f680b009.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--advanced-session--index-md.4227a5ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--basics-session--index-md.08c5b284.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--index-md.9844ef78.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--logistics--index-md.6f6b160a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--publicity--index-md.dfbc1a41.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--registration--index-md.5232161d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--abstracts--index-md.697fe169.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--index-md.ca51927e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--key-dates--index-md.606cfcac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--lightning-talks--index-md.6a60add5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--logistics--index-md.06ca864d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--organizing-committee--index-md.0500ed90.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--photos--index-md.ab68ac2c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--automation-strategies--index-md.2d8a9043.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--bioinformatics-training--index-md.43a55fdc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--end-users--index-md.5f46cd2a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--workflows-and-api--index-md.07da71b4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--index-md.cb8d0657.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--promotion--index-md.4eb64cde.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--register--index-md.49c36527.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--sponsorships--index-md.db0f3791.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--training-day--index-md.c8f12515.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--index-md.3c67479f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--posters--index-md.728aa734.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--talks--index-md.7f93c9db.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--clinical-use-hpc-file-systems--index-md.030a88bd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-admins--index-md.f12f7b4d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-blast--index-md.59e8820f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-professionals--index-md.dcb27276.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--index-md.cfaaca20.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--proteomics--index-md.abba045c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--public-galaxy-servers--index-md.bd34ea15.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--tool-developers--index-md.a5a0a8b6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--index-md.df6174ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--key-dates--index-md.f7630162.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--lightning--index-md.0dfb5a95.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--logistics--index-md.4759076c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--organizers--index-md.6fc11b96.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--photos--index-md.aef50972.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--program--index-md.05c5d87a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--promotion--index-md.e34ad376.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--register--index-md.8ea1785f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--sponsorships--index-md.3cde86d8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--advance-tool-data--index-md.2210b6c4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--api--index-md.57c06e35.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--index-md.4cd32016.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--intro-tool-data--index-md.75a7abbd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--vms--index-md.1c86a58b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--index-md.9abee1c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--posters--index-md.810cd33b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--talks--index-md.c9751906.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--branch-release-merge--index-md.f6ed714e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-admins--index-md.5f9fcb01.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-end-users--index-md.00b4cd3b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-training-network--index-md.a843f5f7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--heterogeneous--index-md.8313575f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--index-md.f878a9f6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--using-galaxy-ui-overview--index-md.5b284474.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--hackathon--index-md.20172bcf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--index-md.9ea6d61e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--key-dates--index-md.bac2859e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--lightning--index-md.96c8f509.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--logistics--index-md.c1318807.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--organizers--index-md.b183376c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--photos--index-md.e0b61b7a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--program--index-md.a93fc58b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--promotion--index-md.df1d2b77.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--register--index-md.54adaeb3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--sponsor-exhibit--index-md.64ef93be.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--admin-walkthrough--index-md.c82adb0e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--api--index-md.a936c5d1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--data-managers--index-md.348024f8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--index-md.030f1ebd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--vms--index-md.507d7d15.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--cwl--index-md.a8a9aa76.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-management-challenges--index-md.409c72aa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-splitting-and-parallelism--index-md.6332a3a3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--elixir--index-md.982da27f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--extended-workflow--index-md.f2b42ad0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--fots--index-md.52645a6e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-admins--index-md.ffda743c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-community-uk--index-md.060ecf60.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-scientists--index-md.ae9e7e59.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-training-network--index-md.a412fdd5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--index-md.22f27d14.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--data-hack-roundup--index-md.60bc6484.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--header--index-md.dbba4e7d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--index-md.2f85a581.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2016--bofs--index-md.5a3032e2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--abstracts--index-md.57e795e8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--childcare--index-md.096b34d2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--cofest--index-md.8b0a9b93.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--conduct--index-md.8e51f30c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--faq--index-md.0f0f1b46.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--index-md.641b44a6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--organizers--index-md.787e0ccf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--promotion--index-md.e6e58e52.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--registration--index-md.52f1b4f7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--sponsors--index-md.ef140359.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--training--index-md.ee0d3a7c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--venue--index-md.30256085.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--abstracts--index-md.137e7bc6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--childcare--index-md.84f642d3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--code-of-conduct--index-md.2789988f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--collaboration--index-md.a48ddf3b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--faq--index-md.4fd33733.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--icebreaker--index-md.ce20a106.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--index-md.9c798d19.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--keynotes--index-md.080e361c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--organizers--index-md.a3b956fb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--promotion--index-md.de985f84.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--register--index-md.05d6bc90.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--sponsors--index-md.4c398a30.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--training--index-md.ce0ef35c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--venue--index-md.4a3ddfec.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2014--index-md.d56576b3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2016--index-md.9f18f6ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2014--index-md.6fd15769.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2016--index-md.c0b34aae.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--index-md.ba91d0c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--trainingday--index-md.e89c50c3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2016--index-md.6afda26b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--example--vue-remark--index-md.c96a8cd8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--faqs--index-md.3bba7da9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-06--index-md.a5232112.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-07--index-md.19df6105.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--icons--index-md.ff92f6d4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--index-md.246f22cd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--logos--index-md.5a8a4cdf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--news-graphics--index-md.844f04b7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--photos--index-md.7a556820.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--screenshots--index-md.290a5433.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--index-md.467e2afe.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--workflow-editor-unannotated--index-md.bc93cd9f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--extract--index-md.940bc220.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--tool-panel--index-md.7ac3e984.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--variables--index-md.a3612f7c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--faq--index-md.599e559f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--galaxy-pages--index-md.51df5ee9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--interval-operations--index-md.71022235.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--main--faq--index-md.f6a425fa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--tools--requirements--index-md.1aef8fd3.js"><link rel="stylesheet" href="/assets/css/0.styles.28333459.css"><noscript data-vue-tag="ssr"><style>.g-image--loading{display:none;}</style></noscript>
  </head>
  <body >
    <div data-server-rendered="true" id="app" class="layout" data-v-3a128d75><header id="masthead"><nav class="container navbar navbar-expand-lg navbar-dark justify-content-center"><a href="/" class="navbar-brand active"><img id="masthead-logo" src="/images/galaxy_logo_hub_white.svg" alt="Galaxy Community Hub" height="30"></a><div id="navbarToggleWrapper" class="collapse navbar-collapse mx-auto"><ul class="navbar-nav"><li class="nav-item"><a href="/use/" role="button" aria-expanded="false" class="nav-link dropdown">
          Use
        </a></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" aria-expanded="false" class="nav-link dropdown-toggle">
          Learn
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/learn/" class="dropdown-item">How to use Galaxy</a></li><li><a href="https://training.galaxyproject.org/" class="dropdown-item">Teach with Galaxy</a></li></ul></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" class="nav-link dropdown-toggle">
          Community
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/community/" class="dropdown-item">About</a></li><li><a href="/events/" class="dropdown-item">Events</a></li><li><a href="/news/" class="dropdown-item">News</a></li><li><a href="/blog/" class="dropdown-item">Blog</a></li><li><a href="/publication-library/" class="dropdown-item">Publications</a></li><li><a href="/galaxy-project/statistics/" class="dropdown-item">Stats</a></li><li><a href="/careers/" class="dropdown-item">Careers</a></li></ul></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" class="nav-link dropdown-toggle">
          Deploy &amp; Develop
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/admin/get-galaxy/" class="dropdown-item">Install Galaxy</a></li><li><a href="/admin/" class="dropdown-item">Server administration</a></li><li><a href="/tools/" class="dropdown-item">Tools</a></li><li><a href="/develop/api/" class="dropdown-item">Working with the API</a></li><li><a href="/develop/" class="dropdown-item">Contributing</a></li><li><a href="/docs/" class="dropdown-item">Code documentation</a></li></ul></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" class="nav-link dropdown-toggle">
          Support
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/support/" class="dropdown-item active">FAQ</a></li><li><a href="https://help.galaxyproject.org/" class="dropdown-item">Galaxy Help Forum</a></li></ul></li><li class="nav-item"><a href="/jxtx/" role="button" aria-expanded="false" class="nav-link dropdown">
          @jxtx
        </a></li></ul></div></nav></header><section id="maincontainer" class="container"><a href="/" class="link active" data-v-3a128d75> ‚Üê Home</a><header data-v-3a128d75><!----><h1 class="title" data-v-3a128d75>Job and Tool Error Help</h1><div class="metadata" data-v-3a128d75><!----><!----><!----></div></header><div class="content markdown" data-v-3a128d75><p><a href="/src/support/index.md">Back to Support Hub</a></p>
<p>Related topics</p>
<ul>
<li><a href="/src/learn/index.md">Tutorials</a></li>
<li><a href="/src/support/#getting-inputs-right">Getting Inputs Right</a></li>
<li><a href="/src/support/diff-expression/index.md">Extended Help for Differential Expression Analysis Tools</a></li>
<li><a href="/src/issues/index.md">Reporting Usage Issues or Software bugs</a></li>
<li><a href="/src/use/index.md#which-platform-platform-type-to-choose">Choices</a></li>
<li><a href="https://galaxyproject.github.io/" target="_blank" rel="noopener noreferrer">Ecosystem</a></li>
<li><a href="https://help.galaxyproject.org/" target="_blank" rel="noopener noreferrer">Galaxy Help</a></li>
<li><a href="https://galaxyproject.org/search/" target="_blank" rel="noopener noreferrer">Search all Prior <code>Q&#x26;A</code> and Galaxy Resources</a></li>
</ul>
<p>Known Issues</p>
<ul>
<li><a href="https://github.com/galaxyproject/galaxy/issues/" target="_blank" rel="noopener noreferrer">Galaxy</a></li>
<li><a href="https://github.com/galaxyproject/usegalaxy-playbook/issues/" target="_blank" rel="noopener noreferrer">Main https://usegalaxy.org</a></li>
<li><a href="https://github.com/galaxyproject/tools-devteam/issues/" target="_blank" rel="noopener noreferrer">Tools-devteam</a></li>
<li><a href="https://github.com/galaxyproject/tools-iuc/issues" target="_blank" rel="noopener noreferrer">Tools-iuc</a></li>
</ul>
<h2 id="job-and-tool-error-help"><a href="#job-and-tool-error-help" aria-hidden="true"><span class="icon icon-link"></span></a>Job and Tool Error Help</h2>
<p>So you started a job and it ended up failing. The result datasets are red in the history. </p>
<p>What to do? It depends on the <strong>failure type</strong> and where you are using Galaxy. </p>
<p><strong><em>In a rush to solve the problem?</em></strong> The top reasons for failures are listed <a href="/src/support/#getting-inputs-right"><strong>here</strong></a>. Also consider searching <strong>Known Issues</strong> and review the advanced troubleshooting help covered in <strong>Reporting Usage Issues or Software bugs</strong> (all linked above). Your problem may be something we are already working to correct or have exising prior Q and A, Galaxy help posts, documentation, and/or Galaxy tutorials that include a solution.</p>
<h3 id="query-all-galaxy-resources-with-the-search-galaxy-box-above"><a href="#query-all-galaxy-resources-with-the-search-galaxy-box-above" aria-hidden="true"><span class="icon icon-link"></span></a>Query all Galaxy resources with the "Search Galaxy" box above</h3>
<h3 id="tips"><a href="#tips" aria-hidden="true"><span class="icon icon-link"></span></a>TIPS</h3>
<ol>
<li>The general troubleshooting help applies to most Galaxy servers/tools/functions.</li>
<li>The cluster error help is server specific for <a href="/src/main/index.md">Galaxy Main</a> at <a href="http://usegalaxy.org" target="_blank" rel="noopener noreferrer">http://usegalaxy.org</a>. </li>
<li>If working at a different public Galaxy server, <a href="/src/use/index.md">contacting the admins directly</a> to address server-related problems is often necessary. Each Galaxy is independently administered.</li>
</ol>
<h2 id="determining-the-job-failure-type"><a href="#determining-the-job-failure-type" aria-hidden="true"><span class="icon icon-link"></span></a>Determining the job failure type</h2>
<p>Did the job fail with some comments that are not descriptive of the problem? Are you sure? </p>
<p>Click on the bug icon <img src="/src/images/icons/bug.png"> within the error dataset to review details about the problem. This does not need to be submitted if you can figure out and correct the problem. There is a good chance you'll able to link your issue to existing troubleshooting help, either here in the <a href="/src/support/index.md">Galaxy hub</a> or at <a href="https://help.galaxyproject.org/" target="_blank" rel="noopener noreferrer">Galaxy Help</a>. Archived Q&#x26;A may still be helpful, find it here: <a href="https://biostar.usegalaxy.org" target="_blank" rel="noopener noreferrer">Galaxy Biostars</a>.</p>
<p>Advanced troubleshooting help is covered at: <a href="/src/issues/index.md">Reporting Usage Issues or Software bugs</a></p>
<p>If you are still stuck after reviewing, please do send in the bug report and we can help.</p>
<ul>
<li>Include a link to questions asked about the same problem that you may have posted online (example: at <a href="https://help.galaxyproject.org/" target="_blank" rel="noopener noreferrer">Galaxy Help</a>)</li>
<li>Leave the input and output datasets undeleted</li>
<li>Do not delete the history immediately after submitting a bug report. Otherwise, there is a good chance we won't be able to help you.</li>
<li>Should you solve the problem before we reply, a follow-up email to let us know is appreciated.</li>
<li>The majority of submitted reports are not true software bugs. <strong>Usage problems involving incorrect input format or content cause most errors</strong>. </li>
<li>Confirming correct tool usage is where we start when diagnosing a problem, <a href="/src/support/troubleshoot-an-error/index.md"><strong>and is where you should start, too.</strong></a>. Sharing details about what you checked is always helpful.</li>
</ul>
<h3 id="type-input-problems"><a href="#type-input-problems" aria-hidden="true"><span class="icon icon-link"></span></a>Type: input problems</h3>
<p>These are the top reasons for tool failures.</p>
<h4 id="examples---input"><a href="#examples---input" aria-hidden="true"><span class="icon icon-link"></span></a>Examples - input</h4>
<ul>
<li>You need to sort your inputs.</li>
<li>The datatype or database is either not assigned or is incorrect.</li>
<li>The original fastq input was not really in fastsanger format.</li>
<li>Key fields on the tool form were left blank (please report this so we can fix it!).</li>
<li>The tool has specific formatting requirements for inputs that were not met.</li>
<li>Annotation files are a mismatch for the selected or assigned reference genome build.</li>
<li>The BED/Interval/GFF dataset is not really in specification.</li>
<li>Some of the data was generated outside of Galaxy, but later a built-in indexed genome build was used with downsteam tools. This scenerio can work, but only if those two genomes are an <strong>exact</strong> match. </li>
<li><strong>Tip</strong> "Chr1" and "chr1" and "1" do not mean the same thing to a tool (or rather, most tools!).</li>
</ul>
<h4 id="how-to-detect---input"><a href="#how-to-detect---input" aria-hidden="true"><span class="icon icon-link"></span></a>How to detect - input</h4>
<ul>
<li>Tool errors can vary. Widely.</li>
</ul>
<h4 id="how-to-resolve---input"><a href="#how-to-resolve---input" aria-hidden="true"><span class="icon icon-link"></span></a>How to resolve - input</h4>
<ul>
<li>Fix the inputs. </li>
<li>Read the tool form help.</li>
<li><strong>Follow the guidelines and troubleshooting help at the <a href="/src/support/index.md">Support Hub</a>.</strong></li>
<li><strong>Review a <a href="/src/learn/index.md">Galaxy Tutorial</a> that covers your analysis and/or tool.</strong></li>
</ul>
<h4 id="special-cases---input"><a href="#special-cases---input" aria-hidden="true"><span class="icon icon-link"></span></a>Special cases - input</h4>
<ul>
<li>There are no special cases. This problem is ubiquitous no matter where or with what tools you happen to be using.</li>
<li><em>How you think the job was executed may not actually be what was executed.</em> Use the re-run <img src="/src/images/icons/arrow-circle.png"> and job info <img src="/src/images/icons/HistoryInfo.png"> functions to review your work. </li>
</ul>
<p>Input problems are very common across any analysis that makes use of programmed tools. Learning how to format your data correctly will not only lead to successful jobs in Galaxy, but more robust and reproducible research overall. </p>
<h3 id="type-cancelled-by-admin-or-a-cluster-failure"><a href="#type-cancelled-by-admin-or-a-cluster-failure" aria-hidden="true"><span class="icon icon-link"></span></a>Type: cancelled by admin or a cluster failure</h3>
<p>The initial error message will be reported as below, and is found in the comments of a failed dataset (red dataset):</p>
<pre><code>This job failed because it was cancelled by an administrator.
Please click the bug icon to report this problem if you need help.
</code></pre>
<p>Or sometimes:</p>
<pre><code>job info:
Remote job server indicated a problem running or monitoring this job.
</code></pre>
<p>It is possible that there was a server or cluster error. However, very often input problems can contribute to these types of failures.</p>
<h4 id="how-to-resolve---cancelled"><a href="#how-to-resolve---cancelled" aria-hidden="true"><span class="icon icon-link"></span></a>How to resolve - cancelled</h4>
<p>If after reviewing inputs (see section for that job falure reason below) and re-running, and the cause of the error remains unclear, try the following:</p>
<ul>
<li>Search prior questions/answers at <a href="https://help.galaxyproject.org/" target="_blank" rel="noopener noreferrer">Galaxy Help</a> (current Q&#x26;A) or at <a href="https://biostar.usegalaxy.org" target="_blank" rel="noopener noreferrer">Galaxy Biostar</a> (archived Q&#x26;A)</li>
<li>Submit a novel question at <a href="https://help.galaxyproject.org/" target="_blank" rel="noopener noreferrer">Galaxy Help</a></li>
<li>Report the novel issue in a bug report if working at <a href="/src/main/index.md">Galaxy Main</a> (<a href="http://usegalaxy.org" target="_blank" rel="noopener noreferrer">http://usegalaxy.org</a>) or you can reproduce the error there.</li>
</ul>
<h3 id="type-exceeds-memory-allocation"><a href="#type-exceeds-memory-allocation" aria-hidden="true"><span class="icon icon-link"></span></a>Type: exceeds memory allocation</h3>
<p>The full error message will be reported as below, and can be found by clicking on the bug icon for a failed job run (red dataset):</p>
<pre><code>job info:
This job was terminated because it used more memory than it was allocated.
Please click the bug icon to report this problem if you need help.
</code></pre>
<p>Or</p>
<pre><code>stderr:
Fatal error: Exit code 1 ()
slurmstepd: error: Detected 1 oom-kill event(s) in step XXXXXXX.batch cgroup.
</code></pre>
<p>Or this somewhere in the error message (usually at the end)</p>
<pre><code>job stderr:
slurmstepd: error: Detected 1 oom-kill event(s) in step XXXXXXX.batch cgroup.
</code></pre>
<p>On rare cases when the memory quota is exceeded very quickly, an error message such as the following can appear:</p>
<pre><code>job stderr:
Fatal error: Exit code 1 ()
Traceback (most recent call last):
(other lines)
Memory Error
</code></pre>
<p>The error indicates that the job ran out of memory while executing on the cluster node that ran the job. <strong>This memory is different than the amount of disk usage in an account</strong>.</p>
<h4 id="how-to-resolve---memory"><a href="#how-to-resolve---memory" aria-hidden="true"><span class="icon icon-link"></span></a>How to resolve - memory</h4>
<p>Often memory errors can be avoided by the user executing the job:</p>
<ul>
<li>Double check the inputs to the tool. Are the <a href="/src/support/#getting-inputs-right">data properly formatted and labeled</a>?</li>
<li>Review the parameters for the tool and determine if any changes made away from the default setting (or possibly the detail settings) are compute-intensive. Make changes if they suit your research goals. See the underlying tool's documentation to better understand specific parameters. This is often linked in the Help section on a tool's execution form.</li>
<li>If the tool used was one that compares two dataset, change the order of the inputs, and test with a re-run. Some tools consume less memory when the larger dataset is entered as the first input on the tool form.</li>
<li>Also see <strong>How to resolve</strong> and <strong>Special cases</strong> in next section. Just note that a rerun will not solve this type or error until changes are made to the inputs and/or the tool settings.</li>
</ul>
<h3 id="type-execution-exceeds-maximum-allowed-job-run-time-walltime"><a href="#type-execution-exceeds-maximum-allowed-job-run-time-walltime" aria-hidden="true"><span class="icon icon-link"></span></a>Type: execution exceeds maximum allowed job run time (walltime)</h3>
<p>The full error message will be reported as below, and can be found by clicking on the bug icon for a failed job run (red dataset):</p>
<pre><code>job info:
This job was terminated because it ran longer than the maximum allowed job run time.
Please click the bug icon to report this problem if you need help.
</code></pre>
<p>Or sometimes, you'll see both of these:</p>
<pre><code>job stderr:
slurmstepd: error: *** JOB XXXX ON XXXX CANCELLED AT 2019-XX-XXTXX:XX:XX DUE TO TIME LIMIT ***

job info:
Remote job server indicated a problem running or monitoring this job.
</code></pre>
<p>The error indicates that the job execution time exceeded the "wall-time" on the cluster node that ran the job. "Wall-time" is the maximum amount of time any job has to complete before it is terminated. When using the public <a href="/src/main/index.md">Main</a> Galaxy instance at <a href="http://usegalaxy.org" target="_blank" rel="noopener noreferrer">http://usegalaxy.org</a>, see the walltime available on the different clusters <a href="/src/main/index.md">here</a>.</p>
<p>Sometimes the execution time of a job can be decreased by adjusting the inputs, parameters used, or the cluster used (try <strong><a href="/src/main/index.md">Stampede or Jetstream</a></strong>, if available for the tool form under the section <strong>Job Resource Parameters</strong>). This solution is similar to jobs that error for exceeding memory allocation.</p>
<p>If you are excuting a tool that runs on certain remote clusters (example tools: Trinity, Unicycler), the problem could be input problems, exceeding memory, or actually exceeding the "wall-time". These clusters have fixed resources.</p>
<h4 id="how-to-detect---walltime"><a href="#how-to-detect---walltime" aria-hidden="true"><span class="icon icon-link"></span></a>How to detect - walltime</h4>
<ul>
<li>A previously run job didn't re-run well.</li>
<li>The error message is cluster-specific. See examples above.</li>
<li>There is a banner at the web site noting that there are known cluster issues or maintainance is in progress.</li>
<li>Status of the server is noted as being down. Check here: <a href="https://status.galaxyproject.org/" target="_blank" rel="noopener noreferrer">https://status.galaxyproject.org/</a></li>
</ul>
<h4 id="how-to-resolve---memory-or-walltime"><a href="#how-to-resolve---memory-or-walltime" aria-hidden="true"><span class="icon icon-link"></span></a>How to resolve - memory or walltime</h4>
<ul>
<li>Rerunning the job is often the only way forward. </li>
<li>Give a longer-running cluster a try.</li>
<li>If the re-run fails again, and there are no known server issues, the issue is probably not cluster/server related but an input problem that needs to be corrected. </li>
<li>In some cases, reporting the memory or wall-time issue to our team as a bug report is a good way to let us know about tools that run out of resource. Please have the expectation that we probably can not solve your specific issue immediately, but cumulative feedback helps us to learn which tools would benefit from additional resource allocation.</li>
<li>If the job remains too large to run at <a href="/src/main/index.md">Galaxy Main</a>(<a href="http://usegalaxy.org" target="_blank" rel="noopener noreferrer">http://usegalaxy.org</a>)(<a href="http://usegalaxy.org" target="_blank" rel="noopener noreferrer">http://usegalaxy.org</a>), then moving to an instance where more memory can be allocated to jobs is the solution. A good choice is <a href="http://usegalaxy.org/cloud" target="_blank" rel="noopener noreferrer">CloudMan</a> as processing memory is easily scaled up. <a href="http://aws.amazon.com/grants/" target="_blank" rel="noopener noreferrer">AWS in Education</a> grants can help with costs. Another option is setting up a <a href="http://getgalaxy.org" target="_blank" rel="noopener noreferrer">local Galaxy</a>, if you have a computer/server with enough processing memory (16 GB minimum, but more is likely needed if the jobs are large, possibly up to 64 GB).</li>
</ul>
<h4 id="special-cases---memory-or-walltime"><a href="#special-cases---memory-or-walltime" aria-hidden="true"><span class="icon icon-link"></span></a>Special cases - memory or walltime</h4>
<ul>
<li>Certain inputs are very large and/or fragmented.</li>
<li>Example 1: A highly fragmented transcriptome/genome used as a custom reference genome. The size is generally <strong>several thousands of "chromosomes"</strong> instead of one to a few dozen. Consider only using the primary chromosomes and skipping unmapped/unplaced/other partial data. Calculating or filtering by length can reveal the extent of the fragmentation and help you to make choices about how to proceed.</li>
<li>Example 2: An entire NGS read dataset was used as a custom reference genome. The size is generally <strong>millions of "chromosomes"</strong>. <em>Fastq read datasets converted to fasta will almost never work with tools as a custom genome or build.</em></li>
</ul>
<p>Moving to a local or cloud Galaxy will often not help for these cases due to how the 3rd party wrapped tool is written. The tool would fail outside of Galaxy, too. Finding or creating a better quality (less fragmented) target genome/transcriptome is the solution.</p>
<h3 id="type-valueerror-invalid-literal-for-int-with-base-10"><a href="#type-valueerror-invalid-literal-for-int-with-base-10" aria-hidden="true"><span class="icon icon-link"></span></a>Type: ValueError: invalid literal for int() with base 10</h3>
<p>Full error is usually a longer message seen only after clicking on the bug icon <img src="/src/images/icons/bug.png">:</p>
<pre><code>stderr

...
Many lines of text, may include parameters
...
...
ValueError: invalid literal for int() with base 10: 'some-read-identifier-name'
</code></pre>
<h4 id="example---valueerror"><a href="#example---valueerror" aria-hidden="true"><span class="icon icon-link"></span></a>Example - ValueError</h4>
<ul>
<li>MACS2 produces this error the first time it is run. The original input fastq read identifiers contained spaces on the "@" line. Mapping is completed and the results are in plain text format (SAM).</li>
</ul>
<h4 id="how-to-detect---valueerror"><a href="#how-to-detect---valueerror" aria-hidden="true"><span class="icon icon-link"></span></a>How to detect - ValueError</h4>
<ul>
<li>Job errors with a message similiar to the above. MACS is not the only tool that can produce this issue, but it is the most common.</li>
</ul>
<h4 id="how-to-resolve---valueerror"><a href="#how-to-resolve---valueerror" aria-hidden="true"><span class="icon icon-link"></span></a>How to resolve - ValueError</h4>
<p>MACS/2 is not capable of interpretting sequence read names with spaces included. Two choices:</p>
<ul>
<li>Remove unmapped reads from the SAM dataset. Unmapped reads are the only data lines in SAM datasets where the full original sequence read name is present (with the included space). (Mapped reads are trimmed at the first whitespace in SAM data lines). There are several filtering tools in the groups <strong>SAMTools</strong> and <strong>Picard</strong> that can do this.</li>
<li>Convert the SAM input to BAM format with the tool <strong>SAMtools: SAM-to-BAM</strong>. When compressed input is given to MACS, the spaces are no longer an issue.</li>
</ul>
<h3 id="type-tool-and-software-problems"><a href="#type-tool-and-software-problems" aria-hidden="true"><span class="icon icon-link"></span></a>Type: Tool and software problems</h3>
<p>Software or Tool Bug? Or a usage error? Sometimes it is hard to tell. </p>
<h4 id="examples---bugs"><a href="#examples---bugs" aria-hidden="true"><span class="icon icon-link"></span></a>Examples - bugs</h4>
<p>Usage error:</p>
<ul>
<li>A mapping job runs out memory resources. The input fastq was never verified to be in <code>fastqsanger</code> format. Rescaling the quality scores with the <strong>Fastq Groomer tool</strong> creates a job that uses far less memory, and it is successful.</li>
<li>A first time you use a new tool, there is a job error. Checking the inputs reveals that the BED dataset has columns in the wrong order. Fixing the BED dataset format allows the re-run job to be successful.</li>
</ul>
<p>Software or Tool Bug:</p>
<ul>
<li>A tool fails and produces an error message stating that a dependency was not found. The tool is not on the list of <a href="/src/support/tool-issues/index.md">Known tool issues</a> and a search against <a href="/src/issues/index.md">all issues</a> does not locate a prior reported problem. You report the issue through a bug report, it is clarified and confirmed, the developers make a fix, and the updated tool is installed on the server. Future jobs are now successful.</li>
<li>When importing an older workflow, a pop-up notice informs that an upgraded tool is available. This tool is editted into the workflow. However, now the workflow fails - at that same tool that was updated. The issue is reported and fixed. Now your workflow runs without problems.</li>
</ul>
<h4 id="how-to-detect---bugs"><a href="#how-to-detect---bugs" aria-hidden="true"><span class="icon icon-link"></span></a>How to detect - bugs</h4>
<ul>
<li>The tool ended with what appears to be a tool problem message.</li>
<li>See examples above.</li>
<li><a href="/src/search">Search Galaxy Resources</a> to see if the problem is a current known problem with a workaround.</li>
</ul>
<h4 id="how-to-resolve---bugs"><a href="#how-to-resolve---bugs" aria-hidden="true"><span class="icon icon-link"></span></a>How to resolve - bugs</h4>
<ul>
<li>If you are on the public <a href="/src/main/index.md">Galaxy Main</a> (<a href="http://usegalaxy.org" target="_blank" rel="noopener noreferrer">http://usegalaxy.org</a>) server, and ran a tool that produced a red error dataset, then you will probably want to start by reviewing the error reasons here first. </li>
<li>Fixing inputs first, as needed, and rerunning is recommended.</li>
<li>If you cannot determine the problem, report the error in a bug report. Add in comments about your reasons why this seems like a tool bug and not a usage/input problem if you can.</li>
</ul>
<h4 id="special-cases---bugs"><a href="#special-cases---bugs" aria-hidden="true"><span class="icon icon-link"></span></a>Special cases - bugs</h4>
<ul>
<li>Software can fail for many reasons. If you think there is a problem, please report it.</li>
<li>Send in a bug report or ask a question at Galaxy Biostars if you want the issue vetted first.</li>
<li>If you are reasonably certain the problem is with software or tools, and not usage, please report it directly to the <a href="/src/issues/index.md">Galaxy Issue Board</a>.</li>
</ul>
</div><footer class="static-footer markdown"><p>The Galaxy Project is supported in part by NSF, NHGRI, The Huck Institutes of the Life Sciences, The Institute for CyberScience at Penn State, and Johns Hopkins University.</p>
</footer></section></div>
    <script>window.__INITIAL_STATE__={"data":{"article":{"id":"abe42116679a5a33a9c6c7c9485e7a63","title":"Job and Tool Error Help","tease":"","image":"","images":{},"category":null,"contact":"","date":null,"content":"\u003Cp\u003E\u003Ca href=\"\u002Fsrc\u002Fsupport\u002Findex.md\"\u003EBack to Support Hub\u003C\u002Fa\u003E\u003C\u002Fp\u003E\n\u003Cp\u003ERelated topics\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"\u002Fsrc\u002Flearn\u002Findex.md\"\u003ETutorials\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"\u002Fsrc\u002Fsupport\u002F#getting-inputs-right\"\u003EGetting Inputs Right\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"\u002Fsrc\u002Fsupport\u002Fdiff-expression\u002Findex.md\"\u003EExtended Help for Differential Expression Analysis Tools\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"\u002Fsrc\u002Fissues\u002Findex.md\"\u003EReporting Usage Issues or Software bugs\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"\u002Fsrc\u002Fuse\u002Findex.md#which-platform-platform-type-to-choose\"\u003EChoices\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Fgalaxyproject.github.io\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EEcosystem\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Fhelp.galaxyproject.org\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EGalaxy Help\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Fgalaxyproject.org\u002Fsearch\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003ESearch all Prior \u003Ccode\u003EQ&#x26;A\u003C\u002Fcode\u003E and Galaxy Resources\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Cp\u003EKnown Issues\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Fgithub.com\u002Fgalaxyproject\u002Fgalaxy\u002Fissues\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EGalaxy\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Fgithub.com\u002Fgalaxyproject\u002Fusegalaxy-playbook\u002Fissues\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EMain https:\u002F\u002Fusegalaxy.org\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Fgithub.com\u002Fgalaxyproject\u002Ftools-devteam\u002Fissues\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003ETools-devteam\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Fgithub.com\u002Fgalaxyproject\u002Ftools-iuc\u002Fissues\" target=\"_blank\" rel=\"noopener noreferrer\"\u003ETools-iuc\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch2 id=\"job-and-tool-error-help\"\u003E\u003Ca href=\"#job-and-tool-error-help\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EJob and Tool Error Help\u003C\u002Fh2\u003E\n\u003Cp\u003ESo you started a job and it ended up failing. The result datasets are red in the history. \u003C\u002Fp\u003E\n\u003Cp\u003EWhat to do? It depends on the \u003Cstrong\u003Efailure type\u003C\u002Fstrong\u003E and where you are using Galaxy. \u003C\u002Fp\u003E\n\u003Cp\u003E\u003Cstrong\u003E\u003Cem\u003EIn a rush to solve the problem?\u003C\u002Fem\u003E\u003C\u002Fstrong\u003E The top reasons for failures are listed \u003Ca href=\"\u002Fsrc\u002Fsupport\u002F#getting-inputs-right\"\u003E\u003Cstrong\u003Ehere\u003C\u002Fstrong\u003E\u003C\u002Fa\u003E. Also consider searching \u003Cstrong\u003EKnown Issues\u003C\u002Fstrong\u003E and review the advanced troubleshooting help covered in \u003Cstrong\u003EReporting Usage Issues or Software bugs\u003C\u002Fstrong\u003E (all linked above). Your problem may be something we are already working to correct or have exising prior Q and A, Galaxy help posts, documentation, and\u002For Galaxy tutorials that include a solution.\u003C\u002Fp\u003E\n\u003Ch3 id=\"query-all-galaxy-resources-with-the-search-galaxy-box-above\"\u003E\u003Ca href=\"#query-all-galaxy-resources-with-the-search-galaxy-box-above\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EQuery all Galaxy resources with the \"Search Galaxy\" box above\u003C\u002Fh3\u003E\n\u003Ch3 id=\"tips\"\u003E\u003Ca href=\"#tips\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003ETIPS\u003C\u002Fh3\u003E\n\u003Col\u003E\n\u003Cli\u003EThe general troubleshooting help applies to most Galaxy servers\u002Ftools\u002Ffunctions.\u003C\u002Fli\u003E\n\u003Cli\u003EThe cluster error help is server specific for \u003Ca href=\"\u002Fsrc\u002Fmain\u002Findex.md\"\u003EGalaxy Main\u003C\u002Fa\u003E at \u003Ca href=\"http:\u002F\u002Fusegalaxy.org\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttp:\u002F\u002Fusegalaxy.org\u003C\u002Fa\u003E. \u003C\u002Fli\u003E\n\u003Cli\u003EIf working at a different public Galaxy server, \u003Ca href=\"\u002Fsrc\u002Fuse\u002Findex.md\"\u003Econtacting the admins directly\u003C\u002Fa\u003E to address server-related problems is often necessary. Each Galaxy is independently administered.\u003C\u002Fli\u003E\n\u003C\u002Fol\u003E\n\u003Ch2 id=\"determining-the-job-failure-type\"\u003E\u003Ca href=\"#determining-the-job-failure-type\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EDetermining the job failure type\u003C\u002Fh2\u003E\n\u003Cp\u003EDid the job fail with some comments that are not descriptive of the problem? Are you sure? \u003C\u002Fp\u003E\n\u003Cp\u003EClick on the bug icon \u003Cimg src=\"\u002Fsrc\u002Fimages\u002Ficons\u002Fbug.png\"\u003E within the error dataset to review details about the problem. This does not need to be submitted if you can figure out and correct the problem. There is a good chance you'll able to link your issue to existing troubleshooting help, either here in the \u003Ca href=\"\u002Fsrc\u002Fsupport\u002Findex.md\"\u003EGalaxy hub\u003C\u002Fa\u003E or at \u003Ca href=\"https:\u002F\u002Fhelp.galaxyproject.org\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EGalaxy Help\u003C\u002Fa\u003E. Archived Q&#x26;A may still be helpful, find it here: \u003Ca href=\"https:\u002F\u002Fbiostar.usegalaxy.org\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EGalaxy Biostars\u003C\u002Fa\u003E.\u003C\u002Fp\u003E\n\u003Cp\u003EAdvanced troubleshooting help is covered at: \u003Ca href=\"\u002Fsrc\u002Fissues\u002Findex.md\"\u003EReporting Usage Issues or Software bugs\u003C\u002Fa\u003E\u003C\u002Fp\u003E\n\u003Cp\u003EIf you are still stuck after reviewing, please do send in the bug report and we can help.\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003EInclude a link to questions asked about the same problem that you may have posted online (example: at \u003Ca href=\"https:\u002F\u002Fhelp.galaxyproject.org\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EGalaxy Help\u003C\u002Fa\u003E)\u003C\u002Fli\u003E\n\u003Cli\u003ELeave the input and output datasets undeleted\u003C\u002Fli\u003E\n\u003Cli\u003EDo not delete the history immediately after submitting a bug report. Otherwise, there is a good chance we won't be able to help you.\u003C\u002Fli\u003E\n\u003Cli\u003EShould you solve the problem before we reply, a follow-up email to let us know is appreciated.\u003C\u002Fli\u003E\n\u003Cli\u003EThe majority of submitted reports are not true software bugs. \u003Cstrong\u003EUsage problems involving incorrect input format or content cause most errors\u003C\u002Fstrong\u003E. \u003C\u002Fli\u003E\n\u003Cli\u003EConfirming correct tool usage is where we start when diagnosing a problem, \u003Ca href=\"\u002Fsrc\u002Fsupport\u002Ftroubleshoot-an-error\u002Findex.md\"\u003E\u003Cstrong\u003Eand is where you should start, too.\u003C\u002Fstrong\u003E\u003C\u002Fa\u003E. Sharing details about what you checked is always helpful.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"type-input-problems\"\u003E\u003Ca href=\"#type-input-problems\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EType: input problems\u003C\u002Fh3\u003E\n\u003Cp\u003EThese are the top reasons for tool failures.\u003C\u002Fp\u003E\n\u003Ch4 id=\"examples---input\"\u003E\u003Ca href=\"#examples---input\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EExamples - input\u003C\u002Fh4\u003E\n\u003Cul\u003E\n\u003Cli\u003EYou need to sort your inputs.\u003C\u002Fli\u003E\n\u003Cli\u003EThe datatype or database is either not assigned or is incorrect.\u003C\u002Fli\u003E\n\u003Cli\u003EThe original fastq input was not really in fastsanger format.\u003C\u002Fli\u003E\n\u003Cli\u003EKey fields on the tool form were left blank (please report this so we can fix it!).\u003C\u002Fli\u003E\n\u003Cli\u003EThe tool has specific formatting requirements for inputs that were not met.\u003C\u002Fli\u003E\n\u003Cli\u003EAnnotation files are a mismatch for the selected or assigned reference genome build.\u003C\u002Fli\u003E\n\u003Cli\u003EThe BED\u002FInterval\u002FGFF dataset is not really in specification.\u003C\u002Fli\u003E\n\u003Cli\u003ESome of the data was generated outside of Galaxy, but later a built-in indexed genome build was used with downsteam tools. This scenerio can work, but only if those two genomes are an \u003Cstrong\u003Eexact\u003C\u002Fstrong\u003E match. \u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003ETip\u003C\u002Fstrong\u003E \"Chr1\" and \"chr1\" and \"1\" do not mean the same thing to a tool (or rather, most tools!).\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch4 id=\"how-to-detect---input\"\u003E\u003Ca href=\"#how-to-detect---input\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EHow to detect - input\u003C\u002Fh4\u003E\n\u003Cul\u003E\n\u003Cli\u003ETool errors can vary. Widely.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch4 id=\"how-to-resolve---input\"\u003E\u003Ca href=\"#how-to-resolve---input\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EHow to resolve - input\u003C\u002Fh4\u003E\n\u003Cul\u003E\n\u003Cli\u003EFix the inputs. \u003C\u002Fli\u003E\n\u003Cli\u003ERead the tool form help.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003EFollow the guidelines and troubleshooting help at the \u003Ca href=\"\u002Fsrc\u002Fsupport\u002Findex.md\"\u003ESupport Hub\u003C\u002Fa\u003E.\u003C\u002Fstrong\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003EReview a \u003Ca href=\"\u002Fsrc\u002Flearn\u002Findex.md\"\u003EGalaxy Tutorial\u003C\u002Fa\u003E that covers your analysis and\u002For tool.\u003C\u002Fstrong\u003E\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch4 id=\"special-cases---input\"\u003E\u003Ca href=\"#special-cases---input\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003ESpecial cases - input\u003C\u002Fh4\u003E\n\u003Cul\u003E\n\u003Cli\u003EThere are no special cases. This problem is ubiquitous no matter where or with what tools you happen to be using.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cem\u003EHow you think the job was executed may not actually be what was executed.\u003C\u002Fem\u003E Use the re-run \u003Cimg src=\"\u002Fsrc\u002Fimages\u002Ficons\u002Farrow-circle.png\"\u003E and job info \u003Cimg src=\"\u002Fsrc\u002Fimages\u002Ficons\u002FHistoryInfo.png\"\u003E functions to review your work. \u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Cp\u003EInput problems are very common across any analysis that makes use of programmed tools. Learning how to format your data correctly will not only lead to successful jobs in Galaxy, but more robust and reproducible research overall. \u003C\u002Fp\u003E\n\u003Ch3 id=\"type-cancelled-by-admin-or-a-cluster-failure\"\u003E\u003Ca href=\"#type-cancelled-by-admin-or-a-cluster-failure\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EType: cancelled by admin or a cluster failure\u003C\u002Fh3\u003E\n\u003Cp\u003EThe initial error message will be reported as below, and is found in the comments of a failed dataset (red dataset):\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003EThis job failed because it was cancelled by an administrator.\nPlease click the bug icon to report this problem if you need help.\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003EOr sometimes:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003Ejob info:\nRemote job server indicated a problem running or monitoring this job.\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003EIt is possible that there was a server or cluster error. However, very often input problems can contribute to these types of failures.\u003C\u002Fp\u003E\n\u003Ch4 id=\"how-to-resolve---cancelled\"\u003E\u003Ca href=\"#how-to-resolve---cancelled\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EHow to resolve - cancelled\u003C\u002Fh4\u003E\n\u003Cp\u003EIf after reviewing inputs (see section for that job falure reason below) and re-running, and the cause of the error remains unclear, try the following:\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003ESearch prior questions\u002Fanswers at \u003Ca href=\"https:\u002F\u002Fhelp.galaxyproject.org\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EGalaxy Help\u003C\u002Fa\u003E (current Q&#x26;A) or at \u003Ca href=\"https:\u002F\u002Fbiostar.usegalaxy.org\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EGalaxy Biostar\u003C\u002Fa\u003E (archived Q&#x26;A)\u003C\u002Fli\u003E\n\u003Cli\u003ESubmit a novel question at \u003Ca href=\"https:\u002F\u002Fhelp.galaxyproject.org\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EGalaxy Help\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003EReport the novel issue in a bug report if working at \u003Ca href=\"\u002Fsrc\u002Fmain\u002Findex.md\"\u003EGalaxy Main\u003C\u002Fa\u003E (\u003Ca href=\"http:\u002F\u002Fusegalaxy.org\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttp:\u002F\u002Fusegalaxy.org\u003C\u002Fa\u003E) or you can reproduce the error there.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"type-exceeds-memory-allocation\"\u003E\u003Ca href=\"#type-exceeds-memory-allocation\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EType: exceeds memory allocation\u003C\u002Fh3\u003E\n\u003Cp\u003EThe full error message will be reported as below, and can be found by clicking on the bug icon for a failed job run (red dataset):\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003Ejob info:\nThis job was terminated because it used more memory than it was allocated.\nPlease click the bug icon to report this problem if you need help.\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003EOr\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003Estderr:\nFatal error: Exit code 1 ()\nslurmstepd: error: Detected 1 oom-kill event(s) in step XXXXXXX.batch cgroup.\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003EOr this somewhere in the error message (usually at the end)\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003Ejob stderr:\nslurmstepd: error: Detected 1 oom-kill event(s) in step XXXXXXX.batch cgroup.\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003EOn rare cases when the memory quota is exceeded very quickly, an error message such as the following can appear:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003Ejob stderr:\nFatal error: Exit code 1 ()\nTraceback (most recent call last):\n(other lines)\nMemory Error\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003EThe error indicates that the job ran out of memory while executing on the cluster node that ran the job. \u003Cstrong\u003EThis memory is different than the amount of disk usage in an account\u003C\u002Fstrong\u003E.\u003C\u002Fp\u003E\n\u003Ch4 id=\"how-to-resolve---memory\"\u003E\u003Ca href=\"#how-to-resolve---memory\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EHow to resolve - memory\u003C\u002Fh4\u003E\n\u003Cp\u003EOften memory errors can be avoided by the user executing the job:\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003EDouble check the inputs to the tool. Are the \u003Ca href=\"\u002Fsrc\u002Fsupport\u002F#getting-inputs-right\"\u003Edata properly formatted and labeled\u003C\u002Fa\u003E?\u003C\u002Fli\u003E\n\u003Cli\u003EReview the parameters for the tool and determine if any changes made away from the default setting (or possibly the detail settings) are compute-intensive. Make changes if they suit your research goals. See the underlying tool's documentation to better understand specific parameters. This is often linked in the Help section on a tool's execution form.\u003C\u002Fli\u003E\n\u003Cli\u003EIf the tool used was one that compares two dataset, change the order of the inputs, and test with a re-run. Some tools consume less memory when the larger dataset is entered as the first input on the tool form.\u003C\u002Fli\u003E\n\u003Cli\u003EAlso see \u003Cstrong\u003EHow to resolve\u003C\u002Fstrong\u003E and \u003Cstrong\u003ESpecial cases\u003C\u002Fstrong\u003E in next section. Just note that a rerun will not solve this type or error until changes are made to the inputs and\u002For the tool settings.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"type-execution-exceeds-maximum-allowed-job-run-time-walltime\"\u003E\u003Ca href=\"#type-execution-exceeds-maximum-allowed-job-run-time-walltime\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EType: execution exceeds maximum allowed job run time (walltime)\u003C\u002Fh3\u003E\n\u003Cp\u003EThe full error message will be reported as below, and can be found by clicking on the bug icon for a failed job run (red dataset):\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003Ejob info:\nThis job was terminated because it ran longer than the maximum allowed job run time.\nPlease click the bug icon to report this problem if you need help.\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003EOr sometimes, you'll see both of these:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003Ejob stderr:\nslurmstepd: error: *** JOB XXXX ON XXXX CANCELLED AT 2019-XX-XXTXX:XX:XX DUE TO TIME LIMIT ***\n\njob info:\nRemote job server indicated a problem running or monitoring this job.\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003EThe error indicates that the job execution time exceeded the \"wall-time\" on the cluster node that ran the job. \"Wall-time\" is the maximum amount of time any job has to complete before it is terminated. When using the public \u003Ca href=\"\u002Fsrc\u002Fmain\u002Findex.md\"\u003EMain\u003C\u002Fa\u003E Galaxy instance at \u003Ca href=\"http:\u002F\u002Fusegalaxy.org\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttp:\u002F\u002Fusegalaxy.org\u003C\u002Fa\u003E, see the walltime available on the different clusters \u003Ca href=\"\u002Fsrc\u002Fmain\u002Findex.md\"\u003Ehere\u003C\u002Fa\u003E.\u003C\u002Fp\u003E\n\u003Cp\u003ESometimes the execution time of a job can be decreased by adjusting the inputs, parameters used, or the cluster used (try \u003Cstrong\u003E\u003Ca href=\"\u002Fsrc\u002Fmain\u002Findex.md\"\u003EStampede or Jetstream\u003C\u002Fa\u003E\u003C\u002Fstrong\u003E, if available for the tool form under the section \u003Cstrong\u003EJob Resource Parameters\u003C\u002Fstrong\u003E). This solution is similar to jobs that error for exceeding memory allocation.\u003C\u002Fp\u003E\n\u003Cp\u003EIf you are excuting a tool that runs on certain remote clusters (example tools: Trinity, Unicycler), the problem could be input problems, exceeding memory, or actually exceeding the \"wall-time\". These clusters have fixed resources.\u003C\u002Fp\u003E\n\u003Ch4 id=\"how-to-detect---walltime\"\u003E\u003Ca href=\"#how-to-detect---walltime\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EHow to detect - walltime\u003C\u002Fh4\u003E\n\u003Cul\u003E\n\u003Cli\u003EA previously run job didn't re-run well.\u003C\u002Fli\u003E\n\u003Cli\u003EThe error message is cluster-specific. See examples above.\u003C\u002Fli\u003E\n\u003Cli\u003EThere is a banner at the web site noting that there are known cluster issues or maintainance is in progress.\u003C\u002Fli\u003E\n\u003Cli\u003EStatus of the server is noted as being down. Check here: \u003Ca href=\"https:\u002F\u002Fstatus.galaxyproject.org\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttps:\u002F\u002Fstatus.galaxyproject.org\u002F\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch4 id=\"how-to-resolve---memory-or-walltime\"\u003E\u003Ca href=\"#how-to-resolve---memory-or-walltime\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EHow to resolve - memory or walltime\u003C\u002Fh4\u003E\n\u003Cul\u003E\n\u003Cli\u003ERerunning the job is often the only way forward. \u003C\u002Fli\u003E\n\u003Cli\u003EGive a longer-running cluster a try.\u003C\u002Fli\u003E\n\u003Cli\u003EIf the re-run fails again, and there are no known server issues, the issue is probably not cluster\u002Fserver related but an input problem that needs to be corrected. \u003C\u002Fli\u003E\n\u003Cli\u003EIn some cases, reporting the memory or wall-time issue to our team as a bug report is a good way to let us know about tools that run out of resource. Please have the expectation that we probably can not solve your specific issue immediately, but cumulative feedback helps us to learn which tools would benefit from additional resource allocation.\u003C\u002Fli\u003E\n\u003Cli\u003EIf the job remains too large to run at \u003Ca href=\"\u002Fsrc\u002Fmain\u002Findex.md\"\u003EGalaxy Main\u003C\u002Fa\u003E(\u003Ca href=\"http:\u002F\u002Fusegalaxy.org\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttp:\u002F\u002Fusegalaxy.org\u003C\u002Fa\u003E)(\u003Ca href=\"http:\u002F\u002Fusegalaxy.org\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttp:\u002F\u002Fusegalaxy.org\u003C\u002Fa\u003E), then moving to an instance where more memory can be allocated to jobs is the solution. A good choice is \u003Ca href=\"http:\u002F\u002Fusegalaxy.org\u002Fcloud\" target=\"_blank\" rel=\"noopener noreferrer\"\u003ECloudMan\u003C\u002Fa\u003E as processing memory is easily scaled up. \u003Ca href=\"http:\u002F\u002Faws.amazon.com\u002Fgrants\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EAWS in Education\u003C\u002Fa\u003E grants can help with costs. Another option is setting up a \u003Ca href=\"http:\u002F\u002Fgetgalaxy.org\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Elocal Galaxy\u003C\u002Fa\u003E, if you have a computer\u002Fserver with enough processing memory (16 GB minimum, but more is likely needed if the jobs are large, possibly up to 64 GB).\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch4 id=\"special-cases---memory-or-walltime\"\u003E\u003Ca href=\"#special-cases---memory-or-walltime\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003ESpecial cases - memory or walltime\u003C\u002Fh4\u003E\n\u003Cul\u003E\n\u003Cli\u003ECertain inputs are very large and\u002For fragmented.\u003C\u002Fli\u003E\n\u003Cli\u003EExample 1: A highly fragmented transcriptome\u002Fgenome used as a custom reference genome. The size is generally \u003Cstrong\u003Eseveral thousands of \"chromosomes\"\u003C\u002Fstrong\u003E instead of one to a few dozen. Consider only using the primary chromosomes and skipping unmapped\u002Funplaced\u002Fother partial data. Calculating or filtering by length can reveal the extent of the fragmentation and help you to make choices about how to proceed.\u003C\u002Fli\u003E\n\u003Cli\u003EExample 2: An entire NGS read dataset was used as a custom reference genome. The size is generally \u003Cstrong\u003Emillions of \"chromosomes\"\u003C\u002Fstrong\u003E. \u003Cem\u003EFastq read datasets converted to fasta will almost never work with tools as a custom genome or build.\u003C\u002Fem\u003E\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Cp\u003EMoving to a local or cloud Galaxy will often not help for these cases due to how the 3rd party wrapped tool is written. The tool would fail outside of Galaxy, too. Finding or creating a better quality (less fragmented) target genome\u002Ftranscriptome is the solution.\u003C\u002Fp\u003E\n\u003Ch3 id=\"type-valueerror-invalid-literal-for-int-with-base-10\"\u003E\u003Ca href=\"#type-valueerror-invalid-literal-for-int-with-base-10\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EType: ValueError: invalid literal for int() with base 10\u003C\u002Fh3\u003E\n\u003Cp\u003EFull error is usually a longer message seen only after clicking on the bug icon \u003Cimg src=\"\u002Fsrc\u002Fimages\u002Ficons\u002Fbug.png\"\u003E:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003Estderr\n\n...\nMany lines of text, may include parameters\n...\n...\nValueError: invalid literal for int() with base 10: 'some-read-identifier-name'\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Ch4 id=\"example---valueerror\"\u003E\u003Ca href=\"#example---valueerror\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EExample - ValueError\u003C\u002Fh4\u003E\n\u003Cul\u003E\n\u003Cli\u003EMACS2 produces this error the first time it is run. The original input fastq read identifiers contained spaces on the \"@\" line. Mapping is completed and the results are in plain text format (SAM).\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch4 id=\"how-to-detect---valueerror\"\u003E\u003Ca href=\"#how-to-detect---valueerror\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EHow to detect - ValueError\u003C\u002Fh4\u003E\n\u003Cul\u003E\n\u003Cli\u003EJob errors with a message similiar to the above. MACS is not the only tool that can produce this issue, but it is the most common.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch4 id=\"how-to-resolve---valueerror\"\u003E\u003Ca href=\"#how-to-resolve---valueerror\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EHow to resolve - ValueError\u003C\u002Fh4\u003E\n\u003Cp\u003EMACS\u002F2 is not capable of interpretting sequence read names with spaces included. Two choices:\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003ERemove unmapped reads from the SAM dataset. Unmapped reads are the only data lines in SAM datasets where the full original sequence read name is present (with the included space). (Mapped reads are trimmed at the first whitespace in SAM data lines). There are several filtering tools in the groups \u003Cstrong\u003ESAMTools\u003C\u002Fstrong\u003E and \u003Cstrong\u003EPicard\u003C\u002Fstrong\u003E that can do this.\u003C\u002Fli\u003E\n\u003Cli\u003EConvert the SAM input to BAM format with the tool \u003Cstrong\u003ESAMtools: SAM-to-BAM\u003C\u002Fstrong\u003E. When compressed input is given to MACS, the spaces are no longer an issue.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"type-tool-and-software-problems\"\u003E\u003Ca href=\"#type-tool-and-software-problems\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EType: Tool and software problems\u003C\u002Fh3\u003E\n\u003Cp\u003ESoftware or Tool Bug? Or a usage error? Sometimes it is hard to tell. \u003C\u002Fp\u003E\n\u003Ch4 id=\"examples---bugs\"\u003E\u003Ca href=\"#examples---bugs\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EExamples - bugs\u003C\u002Fh4\u003E\n\u003Cp\u003EUsage error:\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003EA mapping job runs out memory resources. The input fastq was never verified to be in \u003Ccode\u003Efastqsanger\u003C\u002Fcode\u003E format. Rescaling the quality scores with the \u003Cstrong\u003EFastq Groomer tool\u003C\u002Fstrong\u003E creates a job that uses far less memory, and it is successful.\u003C\u002Fli\u003E\n\u003Cli\u003EA first time you use a new tool, there is a job error. Checking the inputs reveals that the BED dataset has columns in the wrong order. Fixing the BED dataset format allows the re-run job to be successful.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Cp\u003ESoftware or Tool Bug:\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003EA tool fails and produces an error message stating that a dependency was not found. The tool is not on the list of \u003Ca href=\"\u002Fsrc\u002Fsupport\u002Ftool-issues\u002Findex.md\"\u003EKnown tool issues\u003C\u002Fa\u003E and a search against \u003Ca href=\"\u002Fsrc\u002Fissues\u002Findex.md\"\u003Eall issues\u003C\u002Fa\u003E does not locate a prior reported problem. You report the issue through a bug report, it is clarified and confirmed, the developers make a fix, and the updated tool is installed on the server. Future jobs are now successful.\u003C\u002Fli\u003E\n\u003Cli\u003EWhen importing an older workflow, a pop-up notice informs that an upgraded tool is available. This tool is editted into the workflow. However, now the workflow fails - at that same tool that was updated. The issue is reported and fixed. Now your workflow runs without problems.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch4 id=\"how-to-detect---bugs\"\u003E\u003Ca href=\"#how-to-detect---bugs\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EHow to detect - bugs\u003C\u002Fh4\u003E\n\u003Cul\u003E\n\u003Cli\u003EThe tool ended with what appears to be a tool problem message.\u003C\u002Fli\u003E\n\u003Cli\u003ESee examples above.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"\u002Fsrc\u002Fsearch\"\u003ESearch Galaxy Resources\u003C\u002Fa\u003E to see if the problem is a current known problem with a workaround.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch4 id=\"how-to-resolve---bugs\"\u003E\u003Ca href=\"#how-to-resolve---bugs\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EHow to resolve - bugs\u003C\u002Fh4\u003E\n\u003Cul\u003E\n\u003Cli\u003EIf you are on the public \u003Ca href=\"\u002Fsrc\u002Fmain\u002Findex.md\"\u003EGalaxy Main\u003C\u002Fa\u003E (\u003Ca href=\"http:\u002F\u002Fusegalaxy.org\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttp:\u002F\u002Fusegalaxy.org\u003C\u002Fa\u003E) server, and ran a tool that produced a red error dataset, then you will probably want to start by reviewing the error reasons here first. \u003C\u002Fli\u003E\n\u003Cli\u003EFixing inputs first, as needed, and rerunning is recommended.\u003C\u002Fli\u003E\n\u003Cli\u003EIf you cannot determine the problem, report the error in a bug report. Add in comments about your reasons why this seems like a tool bug and not a usage\u002Finput problem if you can.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch4 id=\"special-cases---bugs\"\u003E\u003Ca href=\"#special-cases---bugs\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003ESpecial cases - bugs\u003C\u002Fh4\u003E\n\u003Cul\u003E\n\u003Cli\u003ESoftware can fail for many reasons. If you think there is a problem, please report it.\u003C\u002Fli\u003E\n\u003Cli\u003ESend in a bug report or ask a question at Galaxy Biostars if you want the issue vetted first.\u003C\u002Fli\u003E\n\u003Cli\u003EIf you are reasonably certain the problem is with software or tools, and not usage, please report it directly to the \u003Ca href=\"\u002Fsrc\u002Fissues\u002Findex.md\"\u003EGalaxy Issue Board\u003C\u002Fa\u003E.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n"}},"context":{}};(function(){var s;(s=document.currentScript||document.scripts[document.scripts.length-1]).parentNode.removeChild(s);}());</script><script src="/assets/js/app.f7f9943a.js" defer></script><script src="/assets/js/page--src--templates--article-vue.9c961af0.js" defer></script>
  </body>
</html>
