<!DOCTYPE html>
<html data-html-server-rendered="true" lang="en" data-vue-tag="%7B%22lang%22:%7B%22ssr%22:%22en%22%7D%7D">
  <head>
    <title>Defining Data Managers - Galaxy Community Hub</title><meta name="gridsome:hash" content="a97cc003ab7b7b0128b2927bf289a961e571a0b8"><meta data-vue-tag="ssr" charset="utf-8"><meta data-vue-tag="ssr" name="generator" content="Gridsome v0.7.23"><meta data-vue-tag="ssr" data-key="viewport" name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover"><meta data-vue-tag="ssr" data-key="format-detection" name="format-detection" content="telephone=no"><meta data-vue-tag="ssr" data-key="description" name="description" content="All about Galaxy and its community."><link data-vue-tag="ssr" rel="icon" href="data:,"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="16x16" href="/assets/static/favicon.ce0531f.7d4a227415c9d9b704d56a079d49a3e1.png"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="32x32" href="/assets/static/favicon.ac8d93a.7d4a227415c9d9b704d56a079d49a3e1.png"><link rel="preload" href="/assets/css/0.styles.7e8789a2.css" as="style"><link rel="preload" href="/assets/js/app.b6749062.js" as="script"><link rel="preload" href="/assets/js/page--src--templates--article-vue.8182ee00.js" as="script"><link rel="preload" href="/assets/js/page--src--templates--article-vue~page--src--templates--vue-article-vue.3eb4ba06.js" as="script"><link rel="prefetch" href="/assets/js/page--src--pages--404-vue.757e9e96.js"><link rel="prefetch" href="/assets/js/page--src--pages--blog-vue.2f60cc3b.js"><link rel="prefetch" href="/assets/js/page--src--pages--careers-vue.e58bd442.js"><link rel="prefetch" href="/assets/js/page--src--pages--data-explorer-vue.58967285.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--archive-vue.d20a6137.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests--papercuts-vue.cf50dcc6.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests-vue.d404c7d0.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--webinars-vue.b3800c58.js"><link rel="prefetch" href="/assets/js/page--src--pages--events-vue.f8d1bb99.js"><link rel="prefetch" href="/assets/js/page--src--pages--index-vue.711d622a.js"><link rel="prefetch" href="/assets/js/page--src--pages--news-vue.8c680677.js"><link rel="prefetch" href="/assets/js/page--src--pages--search-vue.090976ac.js"><link rel="prefetch" href="/assets/js/page--src--pages--tests-vue.9ef24b0f.js"><link rel="prefetch" href="/assets/js/page--src--pages--use-vue.0be01484.js"><link rel="prefetch" href="/assets/js/page--src--templates--platform-vue.a56e417a.js"><link rel="prefetch" href="/assets/js/page--src--templates--vue-article-vue.cabaa463.js"><link rel="prefetch" href="/assets/js/vendors~page--src--pages--404-vue.769c153b.js"><link rel="prefetch" href="/assets/js/vendors~page--src--pages--404-vue~page--src--pages--careers-vue~page--src--pages--search-vue~page--s~6aaa8394.b4529ad8.js"><link rel="prefetch" href="/assets/js/vendors~page--src--pages--search-vue.8befcd1e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--index-md.11067c6d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--md--imgtests--imgdir--index-md.bf74dfc9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--partials--index-md.8656e780.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--imgdir--index-md.eca7c5ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--index-md.a7cac9b3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--index-md.958d858f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--index-md.da6161f9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--reldir--index-md.854a3057.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--capacity-planning--index-md.69b7128e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--index-md.f2d1d7aa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--performance--cluster--legacy--index-md.f9800132.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--datatypes--index-md.84ad32f5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--disk-quotas--index-md.3d889971.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--faq--index-md.910be801.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--data-model--index-md.13008439.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--galaxy-ops--index-md.a5346a8c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--ten-simple-steps-galaxy-as-a-service--index-md.e7a68355.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--tool-config-syntax--index-md.9f6b28cb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--writing-tests--index-md.4f24a155.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--allocation--index-md.6c4bec53.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--faq--index-md.e32353bc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--index-md.c0131e18.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--ssh--index-md.76a5a35d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--troubleshooting--index-md.ff16ed17.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--capacity-planning--index-md.7ebdba3e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--getting-started--index-md.0bee47f1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--index-md.cdda9a36.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--building--index-md.d1fb9af5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--capacity-planning--index-md.a45c87f5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--cluster-types--index-md.9c14189b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--customizing--index-md.9627664f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--faq--index-md.b2f845db.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--index-md.4fc6ce5f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--pre-201703--index-md.5b8c873a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--hadoop--index-md.a01cef54.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--ht-condor--index-md.0dcbc49e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--index-md.d5e2a096.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--openstack--index-md.eed9b01d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--services--index-md.db34aabc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--sharing--index-md.479cd135.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--troubleshooting--index-md.61ed81f7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--userdata--index-md.08ce96da.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--future--index-md.fef4a8cb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--index-md.78f52817.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-09--index-md.ac03a5b6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-27--index-md.a56bfa70.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-09-19--index-md.a0976ba4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-11-14--index-md.a9153135.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-01-16--index-md.1ba29422.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-03-20--index-md.0db664a3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-05-15--index-md.d37592fd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-11-20--index-md.a409dc1a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-02-19--index-md.42a2979f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-04-16--index-md.40d17d01.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-06-18--index-md.66f73e0e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-08-20--index-md.db54c8a7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-10-15--index-md.e234a0f3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-02-18--index-md.24a180df.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-04-21--index-md.0ba24e10.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-06-16--index-md.9e797eb1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-08-18--index-md.d83e8b4a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--index-md.540866ce.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--webinar-tech--index-md.a861e596.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2012--index-md.436f25c1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2014--index-md.26f60517.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--index-md.8c39551b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--data--index-md.89dfaed6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--datafiles--index-md.b8ea37ea.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--architecture--index-md.42a21fab.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--bitbucket--index-md.ed830e89.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--css--index-md.d7eb156b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--getting-started--index-md.bf1ead58.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--index-md.8eac10cf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--jsa--index-md.037a145c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--mercurial--index-md.4b546d11.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--security-tool-tips--index-md.8cdb7df5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--galaxy-project--index-md.4686d49d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--index-md.83cee777.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--papers--index-md.4b8ecfa4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--posters--index-md.d1e3edb4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--presentations--index-md.0e2d2e1a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--videos--index-md.70752028.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--advanced-session--index-md.b9f3ed1e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--basics-session--index-md.2bcf225d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--index-md.b8d05422.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--logistics--index-md.9a26fa92.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--publicity--index-md.cf406358.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--registration--index-md.a1309166.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--abstracts--index-md.b5b4584c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--index-md.b57031f3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--key-dates--index-md.8090038a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--lightning-talks--index-md.bc6442f2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--logistics--index-md.be6fe2d9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--organizing-committee--index-md.1c7932ef.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--photos--index-md.8bfc91d3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--automation-strategies--index-md.aab816f0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--bioinformatics-training--index-md.8b587560.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--end-users--index-md.f4036fe7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--workflows-and-api--index-md.f117be80.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--index-md.6eb6a1f1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--promotion--index-md.8a3e997f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--register--index-md.d6e73d37.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--sponsorships--index-md.9a501620.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--training-day--index-md.028bfddb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--index-md.f56e0e40.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--posters--index-md.54e334df.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--talks--index-md.e03ca9cd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--clinical-use-hpc-file-systems--index-md.479d8fca.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-admins--index-md.84645809.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-blast--index-md.4165d5c8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-professionals--index-md.4784e072.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--index-md.9735e2e7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--proteomics--index-md.235cc338.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--public-galaxy-servers--index-md.ca6be78b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--tool-developers--index-md.5488ae6c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--index-md.b6fe9d05.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--key-dates--index-md.95045adf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--lightning--index-md.0f476bf0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--logistics--index-md.3f7fab58.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--organizers--index-md.6b745e0b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--photos--index-md.46f289a6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--program--index-md.2dc90221.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--promotion--index-md.f86302a0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--register--index-md.b8ebbc6b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--sponsorships--index-md.5bd62dd5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--advance-tool-data--index-md.85274240.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--api--index-md.16264cdc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--index-md.01d1a1a9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--intro-tool-data--index-md.4db9ced0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--vms--index-md.f2bd63c3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--index-md.2bb7a0c5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--posters--index-md.79573dfe.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--talks--index-md.d3c01835.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--branch-release-merge--index-md.c6b43399.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-admins--index-md.0a5eb972.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-end-users--index-md.912197e9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-training-network--index-md.ee1827b5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--heterogeneous--index-md.9baea693.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--index-md.fdd1ffad.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--using-galaxy-ui-overview--index-md.9e227d33.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--hackathon--index-md.ed2f45ef.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--index-md.f1ed9a04.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--key-dates--index-md.fae4ba9d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--lightning--index-md.80c44468.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--logistics--index-md.841fc0fd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--organizers--index-md.7f343588.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--photos--index-md.2a63fc2a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--program--index-md.5ae4685e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--promotion--index-md.6e706c8a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--register--index-md.70e5c55a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--sponsor-exhibit--index-md.cf443bd4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--admin-walkthrough--index-md.83edf099.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--api--index-md.16d8430c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--data-managers--index-md.8b01904d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--index-md.420c7759.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--vms--index-md.8440c898.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--cwl--index-md.4c687cfe.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-management-challenges--index-md.46cbd5a6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-splitting-and-parallelism--index-md.596d67fa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--elixir--index-md.b760684b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--extended-workflow--index-md.7abfaf7f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--fots--index-md.292a9f73.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-admins--index-md.86489759.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-community-uk--index-md.2f296ab3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-scientists--index-md.0650adbc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-training-network--index-md.bd7432f3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--index-md.698afae1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--data-hack-roundup--index-md.81d8b289.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--header--index-md.6a085933.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--index-md.9d5304e9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2016--bofs--index-md.e45fe6e9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--abstracts--index-md.4a1538a9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--childcare--index-md.627ff44c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--cofest--index-md.a8dc4701.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--conduct--index-md.cbb9369e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--faq--index-md.a59a28a2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--index-md.bfbac994.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--organizers--index-md.fa504f7f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--promotion--index-md.e195e4c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--registration--index-md.ff34324b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--sponsors--index-md.35239aeb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--training--index-md.35b38787.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--venue--index-md.af316896.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--abstracts--index-md.a964faf5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--childcare--index-md.d77de6de.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--code-of-conduct--index-md.eafab0ba.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--collaboration--index-md.56a546a0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--faq--index-md.7336048d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--icebreaker--index-md.00e425d8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--index-md.efb025b5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--keynotes--index-md.2d849c13.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--organizers--index-md.194881fd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--promotion--index-md.bee008c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--register--index-md.1b4ec5c7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--sponsors--index-md.561d5e54.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--training--index-md.8dd4291b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--venue--index-md.72afd1d4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2014--index-md.16cb7d43.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2016--index-md.cb7bab71.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2014--index-md.59dba3f9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2016--index-md.4280b00e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--index-md.f21026a7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--trainingday--index-md.de9750eb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2016--index-md.327bdf0d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--example--vue-remark--index-md.cbaafdd5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--faqs--index-md.be042548.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-06--index-md.20f3500b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-07--index-md.87c20dee.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--icons--index-md.cd46c11c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--index-md.14afa90b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--logos--index-md.0fbf405d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--news-graphics--index-md.0fa5ff73.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--photos--index-md.c9d4f3be.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--screenshots--index-md.bc7be700.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--index-md.8e64f404.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--workflow-editor-unannotated--index-md.b60ef6c1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--extract--index-md.92f9a05c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--tool-panel--index-md.086fd516.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--variables--index-md.c9116760.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--faq--index-md.3a40d93c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--galaxy-pages--index-md.0feedd43.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--interval-operations--index-md.4adb4e28.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--main--faq--index-md.d705d512.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--tools--requirements--index-md.6870b69b.js"><link rel="stylesheet" href="/assets/css/0.styles.7e8789a2.css"><script data-vue-tag="ssr" src="https://sidecar.gitter.im/dist/sidecar.v1.js" async defer></script><noscript data-vue-tag="ssr"><style>.g-image--loading{display:none;}</style></noscript>
  </head>
  <body >
    <div data-server-rendered="true" id="app" class="layout" data-v-29bca0ce><header id="masthead"><nav class="navbar container justify-content-center navbar-dark bg-transparent navbar-expand-lg"><a href="/" target="_self" class="navbar-brand router-link-active"><img id="masthead-logo" src="/images/galaxy_logo_hub_white.svg" alt="Galaxy Community Hub" height="30"></a><button type="button" aria-label="Toggle navigation" class="navbar-toggler"><span class="navbar-toggler-icon"></span></button><div id="nav-collapse" class="navbar-collapse collapse" style="display:none;"><ul class="navbar-nav"><li class="nav-item"><a href="/use/" target="_self" class="nav-link">Use</a></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Learn</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/learn/" role="menuitem" target="_self" class="dropdown-item">How to use Galaxy</a></li><li role="presentation"><a role="menuitem" href="https://training.galaxyproject.org/" target="_self" class="dropdown-item">Teach with Galaxy</a></li></ul></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Community</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/community/" role="menuitem" target="_self" class="dropdown-item">About</a></li><li role="presentation"><a href="/events/" role="menuitem" target="_self" class="dropdown-item">Events</a></li><li role="presentation"><a href="/news/" role="menuitem" target="_self" class="dropdown-item">News</a></li><li role="presentation"><a href="/blog/" role="menuitem" target="_self" class="dropdown-item">Blog</a></li><li role="presentation"><a href="/publication-library/" role="menuitem" target="_self" class="dropdown-item">Publications</a></li><li role="presentation"><a href="/galaxy-project/statistics/" role="menuitem" target="_self" class="dropdown-item">Stats</a></li><li role="presentation"><a href="/careers/" role="menuitem" target="_self" class="dropdown-item">Careers</a></li></ul></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Deploy &amp; Develop</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/admin/get-galaxy/" role="menuitem" target="_self" class="dropdown-item">Install Galaxy</a></li><li role="presentation"><a href="/admin/" role="menuitem" target="_self" class="dropdown-item router-link-active">Server administration</a></li><li role="presentation"><a href="/tools/" role="menuitem" target="_self" class="dropdown-item">Tools</a></li><li role="presentation"><a href="/develop/api/" role="menuitem" target="_self" class="dropdown-item">Working with the API</a></li><li role="presentation"><a href="/develop/" role="menuitem" target="_self" class="dropdown-item">Contributing</a></li><li role="presentation"><a href="/docs/" role="menuitem" target="_self" class="dropdown-item">Code documentation</a></li></ul></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Support</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/support/" role="menuitem" target="_self" class="dropdown-item">FAQ</a></li><li role="presentation"><a role="menuitem" href="https://help.galaxyproject.org/" target="_self" class="dropdown-item">Galaxy Help Forum</a></li></ul></li><li class="nav-item"><a href="/jxtx/" target="_self" class="nav-link">@jxtx</a></li></ul><ul class="navbar-nav ml-auto"><li class="form-inline"><form action="/search/" method="get" class="form-inline"><input id="search-input" name="q" type="text" placeholder="Search Galaxy" value="" class="mr-sm-2 form-control form-control-sm"><button type="submit" class="btn my-2 my-sm-0 btn-secondary btn-sm"><span class="fa fa-search"></span></button></form></li></ul></div></nav></header><section id="maincontainer" class="container"><header data-v-98965260 data-v-29bca0ce><a href="/" class="link active" data-v-98965260>‚Üê Back to Home</a><!----><!----><h1 class="title" data-v-98965260>Defining Data Managers</h1><!----><section class="metadata freetext" data-v-98965260><!----><!----><!----></section><!----><!----><div class="clearfix" data-v-98965260></div></header><div class="content markdown" data-v-29bca0ce><div class="toc-wrapper col-md-3">
<ul>
<li>
<p><a href="#data-manager-components">Data Manager Components</a></p>
<ul>
<li><a href="#data-manager-configuration">Data Manager Configuration</a></li>
<li><a href="#data-manager-tool">Data Manager Tool</a></li>
</ul>
</li>
<li><a href="#data-manager-server-configuration-options">Data Manager Server Configuration Options</a></li>
<li>
<p><a href="#an-example-single-entry-data_manager_config_file">An example single entry data_manager_config_file</a></p>
<ul>
<li><a href="#an-example-data_managerfetch_genome_all_fastaxml">An example data_manager/fetch_genome_all_fasta.xml</a></li>
<li>
<p><a href="#an-example-data_manager_fetch_genome_all_fastapy">An example data_manager_fetch_genome_all_fasta.py</a></p>
<ul>
<li><a href="#example-json-input-to-tool-dbkey-is-saccer2">Example JSON input to tool, dbkey is sacCer2</a></li>
<li><a href="#example-json-output-from-tool-to-galaxy-dbkey-is-saccer2">Example JSON Output from tool to galaxy, dbkey is sacCer2</a></li>
<li><a href="#new-entry-in-data-table-dbkey-is-saccer2">New Entry in Data Table, dbkey is sacCer2</a></li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
<div class="body-wrapper col-md-9">
<p>This page describes how to define a Data Manager.</p>
<hr>
<hr>
<h1 id="data-manager-components"><a href="#data-manager-components" aria-hidden="true"><span class="icon icon-link"></span></a>Data Manager Components</h1>
<p>Data Managers are composed of two components:</p>
<ul>
<li>Data Manager configuration (e.g. <em>data_manager_conf.xml</em>)</li>
<li>Data Manager Tool</li>
</ul>
<h2 id="data-manager-configuration"><a href="#data-manager-configuration" aria-hidden="true"><span class="icon icon-link"></span></a>Data Manager Configuration</h2>
<p>The Data Manager Configuration (e.g. <em>data_manager_conf.xml</em>) defines the set of available Data Managers using an <a href="/admin/tools/data-managers/data-manager-xml-syntax/">XML description</a>. Each Data Manager can add entries to one or more <a href="/admin/tools/data-tables/">Tool Data Tables</a>. For each Tool Data Table under consideration, the expected output entry columns, and how to handle the Data Manager Tool results, are defined.</p>
<h2 id="data-manager-tool"><a href="#data-manager-tool" aria-hidden="true"><span class="icon icon-link"></span></a>Data Manager Tool</h2>
<p>A Data Manager Tool is a special class of <a href="/admin/tools/adding-tools/">Galaxy Tool</a>. Data Manager Tools do not appear in the standard Tool Panel and can only be accessed by a Galaxy Administrator. Additionally, the initial content of a Data Manager's output file contains a JSON dictionary with a listing of the Tool parameters and Job settings (i.e. they are a type of OutputParameterJSONTool, this is also available for DataSourceTools). There is no requirement for the underlying Data Manager tool to make use of these contents, but they are provided as a handy way to transfer all of the tool and job parameters without requiring a different command-line argument for each necessary piece of information.</p>
<p>The primary difference between a standard Galaxy Tool and a Data Manager Tool is that the primary output dataset of a Data Manager Tool <strong>must</strong> be a file containing a JSON description of the new entries to add to a Tool Data Table. The on-disk content to be referenced by the Data Manager Tool, if any, is stored within the <em>extra_files_path</em> of the output dataset created by the tool.</p>
<hr>
<h1 id="data-manager-server-configuration-options"><a href="#data-manager-server-configuration-options" aria-hidden="true"><span class="icon icon-link"></span></a>Data Manager Server Configuration Options</h1>
<p>In your "<a href="https://github.com/galaxyproject/galaxy/blob/dev/config/galaxy.ini.sample" target="_blank" rel="noopener noreferrer">galaxy.ini</a>" file these settings exist in the <code>[app:main]</code> section:</p>
<pre><code class="language-python"># Data manager configuration options
enable_data_manager_user_view = True
data_manager_config_file = data_manager_conf.xml 
shed_data_manager_config_file = shed_data_manager_conf.xml 
galaxy_data_manager_data_path = tool-data
</code></pre>
<p>Where <em>enable_data_manager_user_view</em> allows non-admin users to view the available data that has been managed.</p>
<p>Where <em>data_manager_config_file</em> defines the local xml file to use for loading the configurations of locally defined data managers.</p>
<p>Where <em>shed_data_manager_config_file</em> defines the local xml file to use for saving and loading the configurations of locally defined data managers.</p>
<p>Where <em>galaxy_data_manager_data_path</em> defines the location to use for storing the files created by Data Managers. When not configured it defaults to the value of <em>tool_data_path</em>.</p>
<h1 id="an-example-single-entry-data_manager_config_file"><a href="#an-example-single-entry-data_manager_config_file" aria-hidden="true"><span class="icon icon-link"></span></a>An example single entry data_manager_config_file</h1>
<pre><code class="language-xml">&#x3C;?xml version="1.0"?>
&#x3C;data_managers> &#x3C;!-- The root element -->
    &#x3C;data_manager tool_file="data_manager/fetch_genome_all_fasta.xml" id="fetch_genome_all_fasta"> &#x3C;!-- Defines a single Data Manager Tool that can update one or more Data Tables -->
        &#x3C;data_table name="all_fasta"> &#x3C;!-- Defines a Data Table to be modified. -->
            &#x3C;output> &#x3C;!-- Handle the output of the Data Manager Tool -->
                &#x3C;column name="value" /> &#x3C;!-- columns that are going to be specified by the Data Manager Tool -->
                &#x3C;column name="dbkey" />
                &#x3C;column name="name" />
                &#x3C;column name="path" output_ref="out_file" >  &#x3C;!-- The value of this column will be modified based upon data in "out_file". example value "phiX.fa" -->
                    &#x3C;move type="file"> &#x3C;!-- Moving a file from the extra files path of "out_file" -->
                        &#x3C;source>${path}&#x3C;/source> &#x3C;!-- File name within the extra files path -->
                        &#x3C;target base="${GALAXY_DATA_MANAGER_DATA_PATH}">${dbkey}/seq/${path}&#x3C;/target> &#x3C;!-- Target Location to store the file, directories are created as needed -->
                    &#x3C;/move>
                    &#x3C;value_translation>${GALAXY_DATA_MANAGER_DATA_PATH}/${dbkey}/seq/${path}&#x3C;/value_translation> &#x3C;!-- Store this value in the final Data Table -->
                &#x3C;/column>
            &#x3C;/output>
        &#x3C;/data_table>
        &#x3C;!-- additional data_tables can be configured from a single Data Manager -->
    &#x3C;/data_manager>
&#x3C;/data_managers>
</code></pre>
<h2 id="an-example-data_managerfetch_genome_all_fastaxml"><a href="#an-example-data_managerfetch_genome_all_fastaxml" aria-hidden="true"><span class="icon icon-link"></span></a>An example data_manager/fetch_genome_all_fasta.xml</h2>
<p>This Tool Config calls a python script <em>data_manager_fetch_genome_all_fasta.py</em> and provides a single file <em>out_file</em> and the description from the dbkey dropdown menu for input.</p>
<p>The starting contents of out_file contain information from Galaxy about the tool, including input parameter values, in the JSON format. Data Manager tools are expected to be able to parse this file. The Data Manager tool will also put the return output values for its results in this file; additional files to be moved can be placed in the <em>extra_files_path</em> of <em>out_file</em>.</p>
<pre><code class="language-xml" id="data_manager_fetch_genome_all_fasta">&#x3C;tool id="data_manager_fetch_genome_all_fasta" name="Reference Genome" version="0.0.1" tool_type="manage_data">
    &#x3C;description>fetching&#x3C;/description>
    &#x3C;command interpreter="python">data_manager_fetch_genome_all_fasta.py "${out_file}" --dbkey_description ${ dbkey.get_display_text() }&#x3C;/command>
    &#x3C;inputs>
        &#x3C;param name="dbkey" type="genomebuild" label="DBKEY to assign to data" />
        &#x3C;param type="text" name="sequence_name" value="" label="Name of sequence" />
        &#x3C;param type="text" name="sequence_desc" value="" label="Description of sequence" />
        &#x3C;param type="text" name="sequence_id" value="" label="ID for sequence" />
        &#x3C;conditional name="reference_source">
          &#x3C;param name="reference_source_selector" type="select" label="Choose the source for the reference genome">
            &#x3C;option value="ucsc">UCSC&#x3C;/option>
            &#x3C;option value="ncbi">NCBI&#x3C;/option>
            &#x3C;option value="url">URL&#x3C;/option>
            &#x3C;option value="history">History&#x3C;/option>
            &#x3C;option value="directory">Directory on Server&#x3C;/option>
          &#x3C;/param>
          &#x3C;when value="ucsc">
            &#x3C;param type="text" name="requested_dbkey" value="" label="UCSC's DBKEY for source FASTA" optional="False" />
          &#x3C;/when>
          &#x3C;when value="ncbi">
            &#x3C;param type="text" name="requested_identifier" value="" label="NCBI identifier" optional="False" />
          &#x3C;/when>
          &#x3C;when value="url">
            &#x3C;param type="text" area="True" name="user_url" value="http://" label="URLs" optional="False" />
          &#x3C;/when>
          &#x3C;when value="history">
            &#x3C;param name="input_fasta" type="data" format="fasta" label="FASTA File" multiple="False" optional="False" />
          &#x3C;/when>
          &#x3C;when value="directory">
            &#x3C;param type="text" name="fasta_filename" value="" label="Full path to FASTA File on disk" optional="False" />
            &#x3C;param type="boolean" name="create_symlink" truevalue="create_symlink" falsevalue="copy_file" label="Create symlink to orignal data instead of copying" checked="False" />
          &#x3C;/when>
        &#x3C;/conditional>
    &#x3C;/inputs>
    &#x3C;outputs>
        &#x3C;data name="out_file" format="data_manager_json"/>
    &#x3C;/outputs>
    &#x3C;!-- 
    &#x3C;tests>
        &#x3C;test>
            DON'T FORGET TO DEFINE SOME TOOL TESTS
        &#x3C;/test>
    &#x3C;/tests>
    -->
    &#x3C;help>
**What it does**

Fetches a reference genome from various sources (UCSC, NCBI, URL, Galaxy History, or a server directory) and populates the "all_fasta" data table.

------



.. class:: infomark

**Notice:** If you leave name, description, or id blank, it will be generated automatically. 

    &#x3C;/help>
&#x3C;/tool>
</code></pre>
<h2 id="an-example-data_manager_fetch_genome_all_fastapy"><a href="#an-example-data_manager_fetch_genome_all_fastapy" aria-hidden="true"><span class="icon icon-link"></span></a>An example data_manager_fetch_genome_all_fasta.py</h2>
<pre><code class="language-python">#!/usr/bin/env python
#Dan Blankenberg

import sys
import os
import tempfile
import shutil
import optparse
import urllib2
from ftplib import FTP
import tarfile

from galaxy.util.json import from_json_string, to_json_string


CHUNK_SIZE = 2**20 #1mb

def cleanup_before_exit( tmp_dir ):
    if tmp_dir and os.path.exists( tmp_dir ):
        shutil.rmtree( tmp_dir )

def stop_err(msg):
    sys.stderr.write(msg)
    sys.exit(1)
    
def get_dbkey_id_name( params, dbkey_description=None):
    dbkey = params['param_dict']['dbkey']
    #TODO: ensure sequence_id is unique and does not already appear in location file
    sequence_id = params['param_dict']['sequence_id']
    if not sequence_id:
        sequence_id = dbkey #uuid.uuid4() generate and use an uuid instead?
    
    sequence_name = params['param_dict']['sequence_name']
    if not sequence_name:
        sequence_name = dbkey_description
        if not sequence_name:
            sequence_name = dbkey
    return dbkey, sequence_id, sequence_name

def download_from_ucsc( data_manager_dict, params, target_directory, dbkey, sequence_id, sequence_name ):
    UCSC_FTP_SERVER = 'hgdownload.cse.ucsc.edu'
    UCSC_CHROM_FA_FILENAME = 'chromFa.tar.gz' #FIXME: this file is actually variable...
    UCSC_DOWNLOAD_PATH = '/goldenPath/%s/bigZips/' + UCSC_CHROM_FA_FILENAME
    COMPRESSED_EXTENSIONS = [ '.tar.gz', '.tar.bz2', '.zip', '.fa.gz', '.fa.bz2' ]
    
    email = params['param_dict']['__user_email__']
    if not email:
        email = 'anonymous@example.com'

    ucsc_dbkey = params['param_dict']['reference_source']['requested_dbkey'] or dbkey
    ftp = FTP( UCSC_FTP_SERVER )
    ftp.login( 'anonymous', email )
    ucsc_file_name = UCSC_DOWNLOAD_PATH % ucsc_dbkey
    
    tmp_dir = tempfile.mkdtemp( prefix='tmp-data-manager-ucsc-' )
    ucsc_fasta_filename = os.path.join( tmp_dir, UCSC_CHROM_FA_FILENAME )
    
    fasta_base_filename = "%s.fa" % sequence_id
    fasta_filename = os.path.join( target_directory, fasta_base_filename )
    fasta_writer = open( fasta_filename, 'wb+' )
    
    tmp_extract_dir = os.path.join ( tmp_dir, 'extracted_fasta' )
    os.mkdir( tmp_extract_dir )
    
    tmp_fasta = open( ucsc_fasta_filename, 'wb+' )
    
    ftp.retrbinary( 'RETR %s' % ucsc_file_name, tmp_fasta.write )
    
    tmp_fasta.seek( 0 )
    fasta_tar = tarfile.open( fileobj=tmp_fasta, mode='r:*' )
    
    fasta_reader = [ fasta_tar.extractfile( member ) for member in fasta_tar.getmembers() ]
    
    data_table_entry = _stream_fasta_to_file( fasta_reader, target_directory, dbkey, sequence_id, sequence_name )
    _add_data_table_entry( data_manager_dict, data_table_entry )
    
    fasta_tar.close()
    tmp_fasta.close()
    cleanup_before_exit( tmp_dir )

def download_from_ncbi( data_manager_dict, params, target_directory, dbkey, sequence_id, sequence_name ):
    NCBI_DOWNLOAD_URL = 'http://togows.dbcls.jp/entry/ncbi-nucleotide/%s.fasta' #FIXME: taken from dave's genome manager...why some japan site?
    
    requested_identifier = params['param_dict']['reference_source']['requested_identifier']
    url = NCBI_DOWNLOAD_URL % requested_identifier
    fasta_reader = urllib2.urlopen( url )
    
    data_table_entry = _stream_fasta_to_file( fasta_reader, target_directory, dbkey, sequence_id, sequence_name )
    _add_data_table_entry( data_manager_dict, data_table_entry )

def download_from_url( data_manager_dict, params, target_directory, dbkey, sequence_id, sequence_name ):
    urls = filter( bool, map( lambda x: x.strip(), params['param_dict']['reference_source']['user_url'].split( '\n' ) ) )
    fasta_reader = [ urllib2.urlopen( url ) for url in urls ]
    
    data_table_entry = _stream_fasta_to_file( fasta_reader, target_directory, dbkey, sequence_id, sequence_name )
    _add_data_table_entry( data_manager_dict, data_table_entry )

def download_from_history( data_manager_dict, params, target_directory, dbkey, sequence_id, sequence_name ):
    #TODO: allow multiple FASTA input files
    input_filename = params['param_dict']['reference_source']['input_fasta']
    if isinstance( input_filename, list ):
        fasta_reader = [ open( filename, 'rb' ) for filename in input_filename ]
    else:
        fasta_reader = open( input_filename )
    
    data_table_entry = _stream_fasta_to_file( fasta_reader, target_directory, dbkey, sequence_id, sequence_name )
    _add_data_table_entry( data_manager_dict, data_table_entry )

def copy_from_directory( data_manager_dict, params, target_directory, dbkey, sequence_id, sequence_name ):
    input_filename = params['param_dict']['reference_source']['fasta_filename']
    create_symlink = params['param_dict']['reference_source']['create_symlink'] == 'create_symlink'
    if create_symlink:
        data_table_entry = _create_symlink( input_filename, target_directory, dbkey, sequence_id, sequence_name )
    else:
        if isinstance( input_filename, list ):
            fasta_reader = [ open( filename, 'rb' ) for filename in input_filename ]
        else:
            fasta_reader = open( input_filename )    
        data_table_entry = _stream_fasta_to_file( fasta_reader, target_directory, dbkey, sequence_id, sequence_name )
    _add_data_table_entry( data_manager_dict, data_table_entry )

def _add_data_table_entry( data_manager_dict, data_table_entry ):
    data_manager_dict['data_tables'] = data_manager_dict.get( 'data_tables', {} )
    data_manager_dict['data_tables']['all_fasta'] = data_manager_dict['data_tables'].get( 'all_fasta', [] )
    data_manager_dict['data_tables']['all_fasta'].append( data_table_entry )
    return data_manager_dict

def _stream_fasta_to_file( fasta_stream, target_directory, dbkey, sequence_id, sequence_name, close_stream=True ):
    fasta_base_filename = "%s.fa" % sequence_id
    fasta_filename = os.path.join( target_directory, fasta_base_filename )
    fasta_writer = open( fasta_filename, 'wb+' )
    
    if isinstance( fasta_stream, list ) and len( fasta_stream ) == 1:
        fasta_stream = fasta_stream[0]
    
    if isinstance( fasta_stream, list ):
        last_char = None
        for fh in fasta_stream:
            if last_char not in [ None, '\n', '\r' ]:
                fasta_writer.write( '\n' )
            while True:
                data = fh.read( CHUNK_SIZE )
                if data:
                    fasta_writer.write( data )
                    last_char = data[-1]
                else:
                    break
            if close_stream:
                fh.close()
    else:
        while True:
            data = fasta_stream.read( CHUNK_SIZE )
            if data:
                fasta_writer.write( data )
            else:
                break
        if close_stream:
            fasta_stream.close()
    
    fasta_writer.close()
    
    return dict( value=sequence_id, dbkey=dbkey, name=sequence_name, path=fasta_base_filename )

def _create_symlink( input_filename, target_directory, dbkey, sequence_id, sequence_name ):
    fasta_base_filename = "%s.fa" % sequence_id
    fasta_filename = os.path.join( target_directory, fasta_base_filename )
    os.symlink( input_filename, fasta_filename )
    return dict( value=sequence_id, dbkey=dbkey, name=sequence_name, path=fasta_base_filename )

REFERENCE_SOURCE_TO_DOWNLOAD = dict( ucsc=download_from_ucsc, ncbi=download_from_ncbi, url=download_from_url, history=download_from_history, directory=copy_from_directory )


def main():
    #Parse Command Line
    parser = optparse.OptionParser()
    parser.add_option( '-d', '--dbkey_description', dest='dbkey_description', action='store', type="string", default=None, help='dbkey_description' )
    (options, args) = parser.parse_args()
    
    filename = args[0]
    
    params = from_json_string( open( filename ).read() )
    target_directory = params[ 'output_data' ][0]['extra_files_path']
    os.mkdir( target_directory )
    data_manager_dict = {}
    
    dbkey, sequence_id, sequence_name = get_dbkey_id_name( params, dbkey_description=options.dbkey_description ) 
    
    if dbkey in [ None, *, '?' ]:
        raise Exception( '"%s" is not a valid dbkey. You must specify a valid dbkey.' % ( dbkey ) )
    
    #Fetch the FASTA
    REFERENCE_SOURCE_TO_DOWNLOAD[ params['param_dict']['reference_source']['reference_source_selector'] ]( data_manager_dict, params, target_directory, dbkey, sequence_id, sequence_name )
    
    #save info to json file
    open( filename, 'wb' ).write( to_json_string( data_manager_dict ) )
        
if __name__ == "__main__": main()
</code></pre>
<h3 id="example-json-input-to-tool-dbkey-is-saccer2"><a href="#example-json-input-to-tool-dbkey-is-saccer2" aria-hidden="true"><span class="icon icon-link"></span></a>Example JSON input to tool, dbkey is sacCer2</h3>
<pre><code class="language-javascript">{
   "param_dict":{
      "__datatypes_config__":"/Users/dan/galaxy-central/database/tmp/tmphyQRH3",
      "__get_data_table_entry__":"&#x3C;function get_data_table_entry at 0x10d435b90>",
      "userId":"1",
      "userEmail":"dan@bx.psu.edu",
      "dbkey":"sacCer2",
      "sequence_desc":"",
      "GALAXY_DATA_INDEX_DIR":"/Users/dan/galaxy-central/tool-data",
      "__admin_users__":"dan@bx.psu.edu,dan+2@bx.psu.edu",
      "__app__":"galaxy.app:UniverseApplication",
      "__user_email__":"dan@bx.psu.edu",
      "sequence_name":"",
      "GALAXY_DATATYPES_CONF_FILE":"/Users/dan/galaxy-central/database/tmp/tmphyQRH3",
      "__user_name__":"danb",
      "sequence_id":"",
      "reference_source":{
         "reference_source_selector":"ncbi",
         "requested_identifier":"sacCer2",
         "__current_case__":"1"
      },
      "__new_file_path__":"/Users/dan/galaxy-central/database/tmp",
      "__user_id__":"1",
      "out_file":"/Users/dan/galaxy-central/database/files/000/dataset_200.dat",
      "GALAXY_ROOT_DIR":"/Users/dan/galaxy-central",
      "__tool_data_path__":"/Users/dan/galaxy-central/tool-data",
      "__root_dir__":"/Users/dan/galaxy-central",
      "chromInfo":"/Users/dan/galaxy-central/tool-data/shared/ucsc/chrom/sacCer2.len"
   },
   "output_data":[
      {
         "extra_files_path":"/Users/dan/galaxy-central/database/job_working_directory/000/202/dataset_200_files",
         "file_name":"/Users/dan/galaxy-central/database/files/000/dataset_200.dat",
         "ext":"data_manager_json",
         "out_data_name":"out_file",
         "hda_id":201,
         "dataset_id":200
      }
   ],
   "job_config":{
      "GALAXY_ROOT_DIR":"/Users/dan/galaxy-central",
      "GALAXY_DATATYPES_CONF_FILE":"/Users/dan/galaxy-central/database/tmp/tmphyQRH3",
      "TOOL_PROVIDED_JOB_METADATA_FILE":"galaxy.json"
   }
}
</code></pre>
<h3 id="example-json-output-from-tool-to-galaxy-dbkey-is-saccer2"><a href="#example-json-output-from-tool-to-galaxy-dbkey-is-saccer2" aria-hidden="true"><span class="icon icon-link"></span></a>Example JSON Output from tool to galaxy, dbkey is sacCer2</h3>
<pre><code class="language-javascript">{
   "data_tables":{
      "all_fasta":[
         {
            "path":"sacCer2.fa",
            "dbkey":"sacCer2",
            "name":"S. cerevisiae June 2008 (SGD/sacCer2) (sacCer2)",
            "value":"sacCer2"
         }
      ]
   }
}
</code></pre>
<h3 id="new-entry-in-data-table-dbkey-is-saccer2"><a href="#new-entry-in-data-table-dbkey-is-saccer2" aria-hidden="true"><span class="icon icon-link"></span></a>New Entry in Data Table, dbkey is sacCer2</h3>
<pre><code class="language-text">#&#x3C;unique_build_id>  &#x3C;dbkey>     &#x3C;display_name>  &#x3C;file_path>
sacCer2 sacCer2 S. cerevisiae June 2008 (SGD/sacCer2) (sacCer2) /Users/dan/galaxy-central/tool-data/sacCer2/seq/sacCer2.fa
</code></pre>
<hr>
<p><a href="/admin/tools/data-managers/">Admin/Tools/DataManagers</a></p>
</div>
</div><footer data-v-54ece1d6 data-v-29bca0ce><!----></footer><footer class="static-footer markdown"><p>The Galaxy Project is supported in part by NSF, NHGRI, The Huck Institutes of the Life Sciences, The Institute for CyberScience at Penn State, and Johns Hopkins University.</p>
</footer></section></div>
    <script>window.__INITIAL_STATE__={"data":{"article":{"id":"8924efe22012fe7ee3055f452d76bcdd","title":"Defining Data Managers","tease":"","category":null,"date":null,"days":null,"contact":"","contact_url":"","authors":"","location":"","location_url":"","source_blog":"","source_blog_url":"","skip_title_render":null,"redirect":"","autotoc":null,"links":[],"image":"","images":{},"external_url":"","content":"\u003Cdiv class=\"toc-wrapper col-md-3\"\u003E\n\u003Cul\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Ca href=\"#data-manager-components\"\u003EData Manager Components\u003C\u002Fa\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"#data-manager-configuration\"\u003EData Manager Configuration\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#data-manager-tool\"\u003EData Manager Tool\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#data-manager-server-configuration-options\"\u003EData Manager Server Configuration Options\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Ca href=\"#an-example-single-entry-data_manager_config_file\"\u003EAn example single entry data_manager_config_file\u003C\u002Fa\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"#an-example-data_managerfetch_genome_all_fastaxml\"\u003EAn example data_manager\u002Ffetch_genome_all_fasta.xml\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Ca href=\"#an-example-data_manager_fetch_genome_all_fastapy\"\u003EAn example data_manager_fetch_genome_all_fasta.py\u003C\u002Fa\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"#example-json-input-to-tool-dbkey-is-saccer2\"\u003EExample JSON input to tool, dbkey is sacCer2\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#example-json-output-from-tool-to-galaxy-dbkey-is-saccer2\"\u003EExample JSON Output from tool to galaxy, dbkey is sacCer2\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#new-entry-in-data-table-dbkey-is-saccer2\"\u003ENew Entry in Data Table, dbkey is sacCer2\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fdiv\u003E\n\u003Cdiv class=\"body-wrapper col-md-9\"\u003E\n\u003Cp\u003EThis page describes how to define a Data Manager.\u003C\u002Fp\u003E\n\u003Chr\u003E\n\u003Chr\u003E\n\u003Ch1 id=\"data-manager-components\"\u003E\u003Ca href=\"#data-manager-components\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EData Manager Components\u003C\u002Fh1\u003E\n\u003Cp\u003EData Managers are composed of two components:\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003EData Manager configuration (e.g. \u003Cem\u003Edata_manager_conf.xml\u003C\u002Fem\u003E)\u003C\u002Fli\u003E\n\u003Cli\u003EData Manager Tool\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch2 id=\"data-manager-configuration\"\u003E\u003Ca href=\"#data-manager-configuration\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EData Manager Configuration\u003C\u002Fh2\u003E\n\u003Cp\u003EThe Data Manager Configuration (e.g. \u003Cem\u003Edata_manager_conf.xml\u003C\u002Fem\u003E) defines the set of available Data Managers using an \u003Ca href=\"\u002Fadmin\u002Ftools\u002Fdata-managers\u002Fdata-manager-xml-syntax\u002F\"\u003EXML description\u003C\u002Fa\u003E. Each Data Manager can add entries to one or more \u003Ca href=\"\u002Fadmin\u002Ftools\u002Fdata-tables\u002F\"\u003ETool Data Tables\u003C\u002Fa\u003E. For each Tool Data Table under consideration, the expected output entry columns, and how to handle the Data Manager Tool results, are defined.\u003C\u002Fp\u003E\n\u003Ch2 id=\"data-manager-tool\"\u003E\u003Ca href=\"#data-manager-tool\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EData Manager Tool\u003C\u002Fh2\u003E\n\u003Cp\u003EA Data Manager Tool is a special class of \u003Ca href=\"\u002Fadmin\u002Ftools\u002Fadding-tools\u002F\"\u003EGalaxy Tool\u003C\u002Fa\u003E. Data Manager Tools do not appear in the standard Tool Panel and can only be accessed by a Galaxy Administrator. Additionally, the initial content of a Data Manager's output file contains a JSON dictionary with a listing of the Tool parameters and Job settings (i.e. they are a type of OutputParameterJSONTool, this is also available for DataSourceTools). There is no requirement for the underlying Data Manager tool to make use of these contents, but they are provided as a handy way to transfer all of the tool and job parameters without requiring a different command-line argument for each necessary piece of information.\u003C\u002Fp\u003E\n\u003Cp\u003EThe primary difference between a standard Galaxy Tool and a Data Manager Tool is that the primary output dataset of a Data Manager Tool \u003Cstrong\u003Emust\u003C\u002Fstrong\u003E be a file containing a JSON description of the new entries to add to a Tool Data Table. The on-disk content to be referenced by the Data Manager Tool, if any, is stored within the \u003Cem\u003Eextra_files_path\u003C\u002Fem\u003E of the output dataset created by the tool.\u003C\u002Fp\u003E\n\u003Chr\u003E\n\u003Ch1 id=\"data-manager-server-configuration-options\"\u003E\u003Ca href=\"#data-manager-server-configuration-options\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EData Manager Server Configuration Options\u003C\u002Fh1\u003E\n\u003Cp\u003EIn your \"\u003Ca href=\"https:\u002F\u002Fgithub.com\u002Fgalaxyproject\u002Fgalaxy\u002Fblob\u002Fdev\u002Fconfig\u002Fgalaxy.ini.sample\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Egalaxy.ini\u003C\u002Fa\u003E\" file these settings exist in the \u003Ccode\u003E[app:main]\u003C\u002Fcode\u003E section:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode class=\"language-python\"\u003E# Data manager configuration options\nenable_data_manager_user_view = True\ndata_manager_config_file = data_manager_conf.xml \nshed_data_manager_config_file = shed_data_manager_conf.xml \ngalaxy_data_manager_data_path = tool-data\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003EWhere \u003Cem\u003Eenable_data_manager_user_view\u003C\u002Fem\u003E allows non-admin users to view the available data that has been managed.\u003C\u002Fp\u003E\n\u003Cp\u003EWhere \u003Cem\u003Edata_manager_config_file\u003C\u002Fem\u003E defines the local xml file to use for loading the configurations of locally defined data managers.\u003C\u002Fp\u003E\n\u003Cp\u003EWhere \u003Cem\u003Eshed_data_manager_config_file\u003C\u002Fem\u003E defines the local xml file to use for saving and loading the configurations of locally defined data managers.\u003C\u002Fp\u003E\n\u003Cp\u003EWhere \u003Cem\u003Egalaxy_data_manager_data_path\u003C\u002Fem\u003E defines the location to use for storing the files created by Data Managers. When not configured it defaults to the value of \u003Cem\u003Etool_data_path\u003C\u002Fem\u003E.\u003C\u002Fp\u003E\n\u003Ch1 id=\"an-example-single-entry-data_manager_config_file\"\u003E\u003Ca href=\"#an-example-single-entry-data_manager_config_file\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EAn example single entry data_manager_config_file\u003C\u002Fh1\u003E\n\u003Cpre\u003E\u003Ccode class=\"language-xml\"\u003E&#x3C;?xml version=\"1.0\"?\u003E\n&#x3C;data_managers\u003E &#x3C;!-- The root element --\u003E\n    &#x3C;data_manager tool_file=\"data_manager\u002Ffetch_genome_all_fasta.xml\" id=\"fetch_genome_all_fasta\"\u003E &#x3C;!-- Defines a single Data Manager Tool that can update one or more Data Tables --\u003E\n        &#x3C;data_table name=\"all_fasta\"\u003E &#x3C;!-- Defines a Data Table to be modified. --\u003E\n            &#x3C;output\u003E &#x3C;!-- Handle the output of the Data Manager Tool --\u003E\n                &#x3C;column name=\"value\" \u002F\u003E &#x3C;!-- columns that are going to be specified by the Data Manager Tool --\u003E\n                &#x3C;column name=\"dbkey\" \u002F\u003E\n                &#x3C;column name=\"name\" \u002F\u003E\n                &#x3C;column name=\"path\" output_ref=\"out_file\" \u003E  &#x3C;!-- The value of this column will be modified based upon data in \"out_file\". example value \"phiX.fa\" --\u003E\n                    &#x3C;move type=\"file\"\u003E &#x3C;!-- Moving a file from the extra files path of \"out_file\" --\u003E\n                        &#x3C;source\u003E${path}&#x3C;\u002Fsource\u003E &#x3C;!-- File name within the extra files path --\u003E\n                        &#x3C;target base=\"${GALAXY_DATA_MANAGER_DATA_PATH}\"\u003E${dbkey}\u002Fseq\u002F${path}&#x3C;\u002Ftarget\u003E &#x3C;!-- Target Location to store the file, directories are created as needed --\u003E\n                    &#x3C;\u002Fmove\u003E\n                    &#x3C;value_translation\u003E${GALAXY_DATA_MANAGER_DATA_PATH}\u002F${dbkey}\u002Fseq\u002F${path}&#x3C;\u002Fvalue_translation\u003E &#x3C;!-- Store this value in the final Data Table --\u003E\n                &#x3C;\u002Fcolumn\u003E\n            &#x3C;\u002Foutput\u003E\n        &#x3C;\u002Fdata_table\u003E\n        &#x3C;!-- additional data_tables can be configured from a single Data Manager --\u003E\n    &#x3C;\u002Fdata_manager\u003E\n&#x3C;\u002Fdata_managers\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Ch2 id=\"an-example-data_managerfetch_genome_all_fastaxml\"\u003E\u003Ca href=\"#an-example-data_managerfetch_genome_all_fastaxml\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EAn example data_manager\u002Ffetch_genome_all_fasta.xml\u003C\u002Fh2\u003E\n\u003Cp\u003EThis Tool Config calls a python script \u003Cem\u003Edata_manager_fetch_genome_all_fasta.py\u003C\u002Fem\u003E and provides a single file \u003Cem\u003Eout_file\u003C\u002Fem\u003E and the description from the dbkey dropdown menu for input.\u003C\u002Fp\u003E\n\u003Cp\u003EThe starting contents of out_file contain information from Galaxy about the tool, including input parameter values, in the JSON format. Data Manager tools are expected to be able to parse this file. The Data Manager tool will also put the return output values for its results in this file; additional files to be moved can be placed in the \u003Cem\u003Eextra_files_path\u003C\u002Fem\u003E of \u003Cem\u003Eout_file\u003C\u002Fem\u003E.\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode class=\"language-xml\" id=\"data_manager_fetch_genome_all_fasta\"\u003E&#x3C;tool id=\"data_manager_fetch_genome_all_fasta\" name=\"Reference Genome\" version=\"0.0.1\" tool_type=\"manage_data\"\u003E\n    &#x3C;description\u003Efetching&#x3C;\u002Fdescription\u003E\n    &#x3C;command interpreter=\"python\"\u003Edata_manager_fetch_genome_all_fasta.py \"${out_file}\" --dbkey_description ${ dbkey.get_display_text() }&#x3C;\u002Fcommand\u003E\n    &#x3C;inputs\u003E\n        &#x3C;param name=\"dbkey\" type=\"genomebuild\" label=\"DBKEY to assign to data\" \u002F\u003E\n        &#x3C;param type=\"text\" name=\"sequence_name\" value=\"\" label=\"Name of sequence\" \u002F\u003E\n        &#x3C;param type=\"text\" name=\"sequence_desc\" value=\"\" label=\"Description of sequence\" \u002F\u003E\n        &#x3C;param type=\"text\" name=\"sequence_id\" value=\"\" label=\"ID for sequence\" \u002F\u003E\n        &#x3C;conditional name=\"reference_source\"\u003E\n          &#x3C;param name=\"reference_source_selector\" type=\"select\" label=\"Choose the source for the reference genome\"\u003E\n            &#x3C;option value=\"ucsc\"\u003EUCSC&#x3C;\u002Foption\u003E\n            &#x3C;option value=\"ncbi\"\u003ENCBI&#x3C;\u002Foption\u003E\n            &#x3C;option value=\"url\"\u003EURL&#x3C;\u002Foption\u003E\n            &#x3C;option value=\"history\"\u003EHistory&#x3C;\u002Foption\u003E\n            &#x3C;option value=\"directory\"\u003EDirectory on Server&#x3C;\u002Foption\u003E\n          &#x3C;\u002Fparam\u003E\n          &#x3C;when value=\"ucsc\"\u003E\n            &#x3C;param type=\"text\" name=\"requested_dbkey\" value=\"\" label=\"UCSC's DBKEY for source FASTA\" optional=\"False\" \u002F\u003E\n          &#x3C;\u002Fwhen\u003E\n          &#x3C;when value=\"ncbi\"\u003E\n            &#x3C;param type=\"text\" name=\"requested_identifier\" value=\"\" label=\"NCBI identifier\" optional=\"False\" \u002F\u003E\n          &#x3C;\u002Fwhen\u003E\n          &#x3C;when value=\"url\"\u003E\n            &#x3C;param type=\"text\" area=\"True\" name=\"user_url\" value=\"http:\u002F\u002F\" label=\"URLs\" optional=\"False\" \u002F\u003E\n          &#x3C;\u002Fwhen\u003E\n          &#x3C;when value=\"history\"\u003E\n            &#x3C;param name=\"input_fasta\" type=\"data\" format=\"fasta\" label=\"FASTA File\" multiple=\"False\" optional=\"False\" \u002F\u003E\n          &#x3C;\u002Fwhen\u003E\n          &#x3C;when value=\"directory\"\u003E\n            &#x3C;param type=\"text\" name=\"fasta_filename\" value=\"\" label=\"Full path to FASTA File on disk\" optional=\"False\" \u002F\u003E\n            &#x3C;param type=\"boolean\" name=\"create_symlink\" truevalue=\"create_symlink\" falsevalue=\"copy_file\" label=\"Create symlink to orignal data instead of copying\" checked=\"False\" \u002F\u003E\n          &#x3C;\u002Fwhen\u003E\n        &#x3C;\u002Fconditional\u003E\n    &#x3C;\u002Finputs\u003E\n    &#x3C;outputs\u003E\n        &#x3C;data name=\"out_file\" format=\"data_manager_json\"\u002F\u003E\n    &#x3C;\u002Foutputs\u003E\n    &#x3C;!-- \n    &#x3C;tests\u003E\n        &#x3C;test\u003E\n            DON'T FORGET TO DEFINE SOME TOOL TESTS\n        &#x3C;\u002Ftest\u003E\n    &#x3C;\u002Ftests\u003E\n    --\u003E\n    &#x3C;help\u003E\n**What it does**\n\nFetches a reference genome from various sources (UCSC, NCBI, URL, Galaxy History, or a server directory) and populates the \"all_fasta\" data table.\n\n------\n\n\n\n.. class:: infomark\n\n**Notice:** If you leave name, description, or id blank, it will be generated automatically. \n\n    &#x3C;\u002Fhelp\u003E\n&#x3C;\u002Ftool\u003E\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Ch2 id=\"an-example-data_manager_fetch_genome_all_fastapy\"\u003E\u003Ca href=\"#an-example-data_manager_fetch_genome_all_fastapy\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EAn example data_manager_fetch_genome_all_fasta.py\u003C\u002Fh2\u003E\n\u003Cpre\u003E\u003Ccode class=\"language-python\"\u003E#!\u002Fusr\u002Fbin\u002Fenv python\n#Dan Blankenberg\n\nimport sys\nimport os\nimport tempfile\nimport shutil\nimport optparse\nimport urllib2\nfrom ftplib import FTP\nimport tarfile\n\nfrom galaxy.util.json import from_json_string, to_json_string\n\n\nCHUNK_SIZE = 2**20 #1mb\n\ndef cleanup_before_exit( tmp_dir ):\n    if tmp_dir and os.path.exists( tmp_dir ):\n        shutil.rmtree( tmp_dir )\n\ndef stop_err(msg):\n    sys.stderr.write(msg)\n    sys.exit(1)\n    \ndef get_dbkey_id_name( params, dbkey_description=None):\n    dbkey = params['param_dict']['dbkey']\n    #TODO: ensure sequence_id is unique and does not already appear in location file\n    sequence_id = params['param_dict']['sequence_id']\n    if not sequence_id:\n        sequence_id = dbkey #uuid.uuid4() generate and use an uuid instead?\n    \n    sequence_name = params['param_dict']['sequence_name']\n    if not sequence_name:\n        sequence_name = dbkey_description\n        if not sequence_name:\n            sequence_name = dbkey\n    return dbkey, sequence_id, sequence_name\n\ndef download_from_ucsc( data_manager_dict, params, target_directory, dbkey, sequence_id, sequence_name ):\n    UCSC_FTP_SERVER = 'hgdownload.cse.ucsc.edu'\n    UCSC_CHROM_FA_FILENAME = 'chromFa.tar.gz' #FIXME: this file is actually variable...\n    UCSC_DOWNLOAD_PATH = '\u002FgoldenPath\u002F%s\u002FbigZips\u002F' + UCSC_CHROM_FA_FILENAME\n    COMPRESSED_EXTENSIONS = [ '.tar.gz', '.tar.bz2', '.zip', '.fa.gz', '.fa.bz2' ]\n    \n    email = params['param_dict']['__user_email__']\n    if not email:\n        email = 'anonymous@example.com'\n\n    ucsc_dbkey = params['param_dict']['reference_source']['requested_dbkey'] or dbkey\n    ftp = FTP( UCSC_FTP_SERVER )\n    ftp.login( 'anonymous', email )\n    ucsc_file_name = UCSC_DOWNLOAD_PATH % ucsc_dbkey\n    \n    tmp_dir = tempfile.mkdtemp( prefix='tmp-data-manager-ucsc-' )\n    ucsc_fasta_filename = os.path.join( tmp_dir, UCSC_CHROM_FA_FILENAME )\n    \n    fasta_base_filename = \"%s.fa\" % sequence_id\n    fasta_filename = os.path.join( target_directory, fasta_base_filename )\n    fasta_writer = open( fasta_filename, 'wb+' )\n    \n    tmp_extract_dir = os.path.join ( tmp_dir, 'extracted_fasta' )\n    os.mkdir( tmp_extract_dir )\n    \n    tmp_fasta = open( ucsc_fasta_filename, 'wb+' )\n    \n    ftp.retrbinary( 'RETR %s' % ucsc_file_name, tmp_fasta.write )\n    \n    tmp_fasta.seek( 0 )\n    fasta_tar = tarfile.open( fileobj=tmp_fasta, mode='r:*' )\n    \n    fasta_reader = [ fasta_tar.extractfile( member ) for member in fasta_tar.getmembers() ]\n    \n    data_table_entry = _stream_fasta_to_file( fasta_reader, target_directory, dbkey, sequence_id, sequence_name )\n    _add_data_table_entry( data_manager_dict, data_table_entry )\n    \n    fasta_tar.close()\n    tmp_fasta.close()\n    cleanup_before_exit( tmp_dir )\n\ndef download_from_ncbi( data_manager_dict, params, target_directory, dbkey, sequence_id, sequence_name ):\n    NCBI_DOWNLOAD_URL = 'http:\u002F\u002Ftogows.dbcls.jp\u002Fentry\u002Fncbi-nucleotide\u002F%s.fasta' #FIXME: taken from dave's genome manager...why some japan site?\n    \n    requested_identifier = params['param_dict']['reference_source']['requested_identifier']\n    url = NCBI_DOWNLOAD_URL % requested_identifier\n    fasta_reader = urllib2.urlopen( url )\n    \n    data_table_entry = _stream_fasta_to_file( fasta_reader, target_directory, dbkey, sequence_id, sequence_name )\n    _add_data_table_entry( data_manager_dict, data_table_entry )\n\ndef download_from_url( data_manager_dict, params, target_directory, dbkey, sequence_id, sequence_name ):\n    urls = filter( bool, map( lambda x: x.strip(), params['param_dict']['reference_source']['user_url'].split( '\\n' ) ) )\n    fasta_reader = [ urllib2.urlopen( url ) for url in urls ]\n    \n    data_table_entry = _stream_fasta_to_file( fasta_reader, target_directory, dbkey, sequence_id, sequence_name )\n    _add_data_table_entry( data_manager_dict, data_table_entry )\n\ndef download_from_history( data_manager_dict, params, target_directory, dbkey, sequence_id, sequence_name ):\n    #TODO: allow multiple FASTA input files\n    input_filename = params['param_dict']['reference_source']['input_fasta']\n    if isinstance( input_filename, list ):\n        fasta_reader = [ open( filename, 'rb' ) for filename in input_filename ]\n    else:\n        fasta_reader = open( input_filename )\n    \n    data_table_entry = _stream_fasta_to_file( fasta_reader, target_directory, dbkey, sequence_id, sequence_name )\n    _add_data_table_entry( data_manager_dict, data_table_entry )\n\ndef copy_from_directory( data_manager_dict, params, target_directory, dbkey, sequence_id, sequence_name ):\n    input_filename = params['param_dict']['reference_source']['fasta_filename']\n    create_symlink = params['param_dict']['reference_source']['create_symlink'] == 'create_symlink'\n    if create_symlink:\n        data_table_entry = _create_symlink( input_filename, target_directory, dbkey, sequence_id, sequence_name )\n    else:\n        if isinstance( input_filename, list ):\n            fasta_reader = [ open( filename, 'rb' ) for filename in input_filename ]\n        else:\n            fasta_reader = open( input_filename )    \n        data_table_entry = _stream_fasta_to_file( fasta_reader, target_directory, dbkey, sequence_id, sequence_name )\n    _add_data_table_entry( data_manager_dict, data_table_entry )\n\ndef _add_data_table_entry( data_manager_dict, data_table_entry ):\n    data_manager_dict['data_tables'] = data_manager_dict.get( 'data_tables', {} )\n    data_manager_dict['data_tables']['all_fasta'] = data_manager_dict['data_tables'].get( 'all_fasta', [] )\n    data_manager_dict['data_tables']['all_fasta'].append( data_table_entry )\n    return data_manager_dict\n\ndef _stream_fasta_to_file( fasta_stream, target_directory, dbkey, sequence_id, sequence_name, close_stream=True ):\n    fasta_base_filename = \"%s.fa\" % sequence_id\n    fasta_filename = os.path.join( target_directory, fasta_base_filename )\n    fasta_writer = open( fasta_filename, 'wb+' )\n    \n    if isinstance( fasta_stream, list ) and len( fasta_stream ) == 1:\n        fasta_stream = fasta_stream[0]\n    \n    if isinstance( fasta_stream, list ):\n        last_char = None\n        for fh in fasta_stream:\n            if last_char not in [ None, '\\n', '\\r' ]:\n                fasta_writer.write( '\\n' )\n            while True:\n                data = fh.read( CHUNK_SIZE )\n                if data:\n                    fasta_writer.write( data )\n                    last_char = data[-1]\n                else:\n                    break\n            if close_stream:\n                fh.close()\n    else:\n        while True:\n            data = fasta_stream.read( CHUNK_SIZE )\n            if data:\n                fasta_writer.write( data )\n            else:\n                break\n        if close_stream:\n            fasta_stream.close()\n    \n    fasta_writer.close()\n    \n    return dict( value=sequence_id, dbkey=dbkey, name=sequence_name, path=fasta_base_filename )\n\ndef _create_symlink( input_filename, target_directory, dbkey, sequence_id, sequence_name ):\n    fasta_base_filename = \"%s.fa\" % sequence_id\n    fasta_filename = os.path.join( target_directory, fasta_base_filename )\n    os.symlink( input_filename, fasta_filename )\n    return dict( value=sequence_id, dbkey=dbkey, name=sequence_name, path=fasta_base_filename )\n\nREFERENCE_SOURCE_TO_DOWNLOAD = dict( ucsc=download_from_ucsc, ncbi=download_from_ncbi, url=download_from_url, history=download_from_history, directory=copy_from_directory )\n\n\ndef main():\n    #Parse Command Line\n    parser = optparse.OptionParser()\n    parser.add_option( '-d', '--dbkey_description', dest='dbkey_description', action='store', type=\"string\", default=None, help='dbkey_description' )\n    (options, args) = parser.parse_args()\n    \n    filename = args[0]\n    \n    params = from_json_string( open( filename ).read() )\n    target_directory = params[ 'output_data' ][0]['extra_files_path']\n    os.mkdir( target_directory )\n    data_manager_dict = {}\n    \n    dbkey, sequence_id, sequence_name = get_dbkey_id_name( params, dbkey_description=options.dbkey_description ) \n    \n    if dbkey in [ None, *, '?' ]:\n        raise Exception( '\"%s\" is not a valid dbkey. You must specify a valid dbkey.' % ( dbkey ) )\n    \n    #Fetch the FASTA\n    REFERENCE_SOURCE_TO_DOWNLOAD[ params['param_dict']['reference_source']['reference_source_selector'] ]( data_manager_dict, params, target_directory, dbkey, sequence_id, sequence_name )\n    \n    #save info to json file\n    open( filename, 'wb' ).write( to_json_string( data_manager_dict ) )\n        \nif __name__ == \"__main__\": main()\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Ch3 id=\"example-json-input-to-tool-dbkey-is-saccer2\"\u003E\u003Ca href=\"#example-json-input-to-tool-dbkey-is-saccer2\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EExample JSON input to tool, dbkey is sacCer2\u003C\u002Fh3\u003E\n\u003Cpre\u003E\u003Ccode class=\"language-javascript\"\u003E{\n   \"param_dict\":{\n      \"__datatypes_config__\":\"\u002FUsers\u002Fdan\u002Fgalaxy-central\u002Fdatabase\u002Ftmp\u002FtmphyQRH3\",\n      \"__get_data_table_entry__\":\"&#x3C;function get_data_table_entry at 0x10d435b90\u003E\",\n      \"userId\":\"1\",\n      \"userEmail\":\"dan@bx.psu.edu\",\n      \"dbkey\":\"sacCer2\",\n      \"sequence_desc\":\"\",\n      \"GALAXY_DATA_INDEX_DIR\":\"\u002FUsers\u002Fdan\u002Fgalaxy-central\u002Ftool-data\",\n      \"__admin_users__\":\"dan@bx.psu.edu,dan+2@bx.psu.edu\",\n      \"__app__\":\"galaxy.app:UniverseApplication\",\n      \"__user_email__\":\"dan@bx.psu.edu\",\n      \"sequence_name\":\"\",\n      \"GALAXY_DATATYPES_CONF_FILE\":\"\u002FUsers\u002Fdan\u002Fgalaxy-central\u002Fdatabase\u002Ftmp\u002FtmphyQRH3\",\n      \"__user_name__\":\"danb\",\n      \"sequence_id\":\"\",\n      \"reference_source\":{\n         \"reference_source_selector\":\"ncbi\",\n         \"requested_identifier\":\"sacCer2\",\n         \"__current_case__\":\"1\"\n      },\n      \"__new_file_path__\":\"\u002FUsers\u002Fdan\u002Fgalaxy-central\u002Fdatabase\u002Ftmp\",\n      \"__user_id__\":\"1\",\n      \"out_file\":\"\u002FUsers\u002Fdan\u002Fgalaxy-central\u002Fdatabase\u002Ffiles\u002F000\u002Fdataset_200.dat\",\n      \"GALAXY_ROOT_DIR\":\"\u002FUsers\u002Fdan\u002Fgalaxy-central\",\n      \"__tool_data_path__\":\"\u002FUsers\u002Fdan\u002Fgalaxy-central\u002Ftool-data\",\n      \"__root_dir__\":\"\u002FUsers\u002Fdan\u002Fgalaxy-central\",\n      \"chromInfo\":\"\u002FUsers\u002Fdan\u002Fgalaxy-central\u002Ftool-data\u002Fshared\u002Fucsc\u002Fchrom\u002FsacCer2.len\"\n   },\n   \"output_data\":[\n      {\n         \"extra_files_path\":\"\u002FUsers\u002Fdan\u002Fgalaxy-central\u002Fdatabase\u002Fjob_working_directory\u002F000\u002F202\u002Fdataset_200_files\",\n         \"file_name\":\"\u002FUsers\u002Fdan\u002Fgalaxy-central\u002Fdatabase\u002Ffiles\u002F000\u002Fdataset_200.dat\",\n         \"ext\":\"data_manager_json\",\n         \"out_data_name\":\"out_file\",\n         \"hda_id\":201,\n         \"dataset_id\":200\n      }\n   ],\n   \"job_config\":{\n      \"GALAXY_ROOT_DIR\":\"\u002FUsers\u002Fdan\u002Fgalaxy-central\",\n      \"GALAXY_DATATYPES_CONF_FILE\":\"\u002FUsers\u002Fdan\u002Fgalaxy-central\u002Fdatabase\u002Ftmp\u002FtmphyQRH3\",\n      \"TOOL_PROVIDED_JOB_METADATA_FILE\":\"galaxy.json\"\n   }\n}\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Ch3 id=\"example-json-output-from-tool-to-galaxy-dbkey-is-saccer2\"\u003E\u003Ca href=\"#example-json-output-from-tool-to-galaxy-dbkey-is-saccer2\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EExample JSON Output from tool to galaxy, dbkey is sacCer2\u003C\u002Fh3\u003E\n\u003Cpre\u003E\u003Ccode class=\"language-javascript\"\u003E{\n   \"data_tables\":{\n      \"all_fasta\":[\n         {\n            \"path\":\"sacCer2.fa\",\n            \"dbkey\":\"sacCer2\",\n            \"name\":\"S. cerevisiae June 2008 (SGD\u002FsacCer2) (sacCer2)\",\n            \"value\":\"sacCer2\"\n         }\n      ]\n   }\n}\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Ch3 id=\"new-entry-in-data-table-dbkey-is-saccer2\"\u003E\u003Ca href=\"#new-entry-in-data-table-dbkey-is-saccer2\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003ENew Entry in Data Table, dbkey is sacCer2\u003C\u002Fh3\u003E\n\u003Cpre\u003E\u003Ccode class=\"language-text\"\u003E#&#x3C;unique_build_id\u003E  &#x3C;dbkey\u003E     &#x3C;display_name\u003E  &#x3C;file_path\u003E\nsacCer2 sacCer2 S. cerevisiae June 2008 (SGD\u002FsacCer2) (sacCer2) \u002FUsers\u002Fdan\u002Fgalaxy-central\u002Ftool-data\u002FsacCer2\u002Fseq\u002FsacCer2.fa\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Chr\u003E\n\u003Cp\u003E\u003Ca href=\"\u002Fadmin\u002Ftools\u002Fdata-managers\u002F\"\u003EAdmin\u002FTools\u002FDataManagers\u003C\u002Fa\u003E\u003C\u002Fp\u003E\n\u003C\u002Fdiv\u003E\n"}},"context":{}};(function(){var s;(s=document.currentScript||document.scripts[document.scripts.length-1]).parentNode.removeChild(s);}());</script><script src="/assets/js/app.b6749062.js" defer></script><script src="/assets/js/page--src--templates--article-vue.8182ee00.js" defer></script><script src="/assets/js/page--src--templates--article-vue~page--src--templates--vue-article-vue.3eb4ba06.js" defer></script>
  </body>
</html>
