<!DOCTYPE html>
<html data-html-server-rendered="true" lang="en" data-vue-tag="%7B%22lang%22:%7B%22ssr%22:%22en%22%7D%7D">
  <head>
    <title>April 2015 Tool Shed Contributions - Galaxy Community Hub</title><meta name="gridsome:hash" content="91d4a83f9855c194893a1f5e294057505b0cd33d"><meta data-vue-tag="ssr" charset="utf-8"><meta data-vue-tag="ssr" name="generator" content="Gridsome v0.7.23"><meta data-vue-tag="ssr" data-key="viewport" name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover"><meta data-vue-tag="ssr" data-key="format-detection" name="format-detection" content="telephone=no"><meta data-vue-tag="ssr" data-key="description" name="description" content="All about Galaxy and its community."><link data-vue-tag="ssr" rel="icon" href="data:,"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="16x16" href="/assets/static/favicon.ce0531f.7d4a227415c9d9b704d56a079d49a3e1.png"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="32x32" href="/assets/static/favicon.ac8d93a.7d4a227415c9d9b704d56a079d49a3e1.png"><link rel="preload" href="/assets/css/0.styles.e1e25242.css" as="style"><link rel="preload" href="/assets/js/app.0690a31c.js" as="script"><link rel="preload" href="/assets/js/page--src--templates--article-vue.6d736ff4.js" as="script"><link rel="prefetch" href="/assets/js/page--node-modules--gridsome--app--pages--404-vue.9b913011.js"><link rel="prefetch" href="/assets/js/page--src--pages--blog-vue.c32e3571.js"><link rel="prefetch" href="/assets/js/page--src--pages--careers-vue.85d2fb31.js"><link rel="prefetch" href="/assets/js/page--src--pages--data-explorer-vue.8aa3e295.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--archive-vue.1426fe7f.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests--papercuts-vue.ab928745.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests-vue.1e8420fc.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--webinars-vue.5d6fb146.js"><link rel="prefetch" href="/assets/js/page--src--pages--events-vue.cd4ed3c8.js"><link rel="prefetch" href="/assets/js/page--src--pages--index-vue.a143f0a2.js"><link rel="prefetch" href="/assets/js/page--src--pages--news-vue.2f002266.js"><link rel="prefetch" href="/assets/js/page--src--pages--search-vue.23e742ea.js"><link rel="prefetch" href="/assets/js/page--src--pages--tests-vue.3b995c11.js"><link rel="prefetch" href="/assets/js/page--src--pages--use-vue.7c5b8683.js"><link rel="prefetch" href="/assets/js/page--src--templates--platform-vue.91145b17.js"><link rel="prefetch" href="/assets/js/page--src--templates--vue-article-vue.d9a0c32e.js"><link rel="prefetch" href="/assets/js/vendors~page--src--pages--careers-vue~page--src--pages--use-vue~page--src--templates--article-vue~pa~efade918.92a3a9c1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--index-md.b8e1f92a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--md--imgtests--imgdir--index-md.5311083e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--partials--index-md.b6884022.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--imgdir--index-md.1ee4281c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--index-md.fe03c46f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--index-md.05d8aec6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--index-md.e7b48256.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--reldir--index-md.4093d68b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--capacity-planning--index-md.8813525b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--index-md.aafe943e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--performance--cluster--legacy--index-md.3da8719f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--datatypes--index-md.9b672175.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--disk-quotas--index-md.44f752e9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--faq--index-md.3851dd50.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--data-model--index-md.6d581759.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--galaxy-ops--index-md.a3b0d0af.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--ten-simple-steps-galaxy-as-a-service--index-md.699b2e0d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--tool-config-syntax--index-md.208f876a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--writing-tests--index-md.b2caa913.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--allocation--index-md.de3e7728.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--faq--index-md.ac3b79f2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--index-md.328857f9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--ssh--index-md.d92fc074.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--troubleshooting--index-md.7c074dc2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--capacity-planning--index-md.01a397fa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--getting-started--index-md.acdcb61b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--index-md.824830b5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--building--index-md.10fb8489.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--capacity-planning--index-md.e2d98e61.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--cluster-types--index-md.df053139.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--customizing--index-md.fb1f931d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--faq--index-md.b2fab4e5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--index-md.d71b41b2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--pre-201703--index-md.f5a4c4d0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--hadoop--index-md.d7a6d969.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--ht-condor--index-md.31bcf5d4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--index-md.b7b22e1d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--openstack--index-md.d78cb425.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--services--index-md.f2027d09.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--sharing--index-md.cd3d2d6f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--troubleshooting--index-md.fc1a0be3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--userdata--index-md.e4d95203.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--future--index-md.e75566ea.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--index-md.a767ff80.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-09--index-md.66c1654b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-27--index-md.c6d537c1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-09-19--index-md.544a522a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-11-14--index-md.4746a5d0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-01-16--index-md.d1551883.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-03-20--index-md.5094871c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-05-15--index-md.1897abfc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-11-20--index-md.c4d4ced5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-02-19--index-md.311ef2ba.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-04-16--index-md.0b281218.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-06-18--index-md.7c255863.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-08-20--index-md.43ba73c2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-10-15--index-md.bc11b58b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-02-18--index-md.1a182ba1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-04-21--index-md.5e9eea61.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-06-16--index-md.b1295236.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-08-18--index-md.d8fbe7ba.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--index-md.43055631.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--webinar-tech--index-md.a44b91e1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2012--index-md.c7a0a096.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2014--index-md.46c5bc59.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--index-md.34247339.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--data--index-md.46e24851.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--datafiles--index-md.77545970.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--architecture--index-md.55f07291.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--bitbucket--index-md.718bcb6c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--css--index-md.281229bf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--getting-started--index-md.497082a2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--index-md.392b2e76.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--jsa--index-md.ba998e94.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--mercurial--index-md.14a2dad8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--security-tool-tips--index-md.17796bc0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--galaxy-project--index-md.0de41023.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--index-md.d89e2857.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--papers--index-md.4bdf7b3e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--posters--index-md.7e825096.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--presentations--index-md.084c96b9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--videos--index-md.c984f7ce.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--advanced-session--index-md.191fe1da.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--basics-session--index-md.35c32703.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--index-md.8acf4593.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--logistics--index-md.37d0ac9b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--publicity--index-md.7f2eec38.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--registration--index-md.d22dcedd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--abstracts--index-md.bbbf6ae8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--index-md.cb62d683.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--key-dates--index-md.07f8d6b1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--lightning-talks--index-md.72ab128d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--logistics--index-md.c3eeb3dd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--organizing-committee--index-md.f7a53fcd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--photos--index-md.d5102197.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--automation-strategies--index-md.6972b31d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--bioinformatics-training--index-md.648f14ff.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--end-users--index-md.ff776d4f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--workflows-and-api--index-md.540ad6ba.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--index-md.d233185c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--promotion--index-md.212eb34b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--register--index-md.b4c4f57b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--sponsorships--index-md.d0539386.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--training-day--index-md.2ee4bcd3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--index-md.1aa46f9f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--posters--index-md.a8b0ee54.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--talks--index-md.b3ee75dc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--clinical-use-hpc-file-systems--index-md.3fc5e2ed.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-admins--index-md.2f878aaa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-blast--index-md.5305db66.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-professionals--index-md.dcbf6370.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--index-md.3abd720e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--proteomics--index-md.f4d7daaf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--public-galaxy-servers--index-md.b346729f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--tool-developers--index-md.4365e2e3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--index-md.d58334f2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--key-dates--index-md.a3f7c6b8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--lightning--index-md.180bd3c7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--logistics--index-md.c889273c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--organizers--index-md.2f551bcb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--photos--index-md.38ee0e51.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--program--index-md.f2c70270.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--promotion--index-md.1e5aadf9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--register--index-md.99382e6b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--sponsorships--index-md.1e82df1d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--advance-tool-data--index-md.02e350f8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--api--index-md.23e9c64f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--index-md.7b6606de.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--intro-tool-data--index-md.1893d923.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--vms--index-md.eb7d2783.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--index-md.19ebb120.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--posters--index-md.19e77da9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--talks--index-md.69b1844f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--branch-release-merge--index-md.e83f529b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-admins--index-md.4c5b1df3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-end-users--index-md.6b3ff393.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-training-network--index-md.85db1b6f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--heterogeneous--index-md.26797010.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--index-md.ab231746.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--using-galaxy-ui-overview--index-md.3f0318f2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--hackathon--index-md.f1b7f383.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--index-md.894a1ae1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--key-dates--index-md.d3d9965a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--lightning--index-md.268064be.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--logistics--index-md.033eda0f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--organizers--index-md.73259cdb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--photos--index-md.3ee7fbac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--program--index-md.79995b28.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--promotion--index-md.59d9587a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--register--index-md.6cf374d9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--sponsor-exhibit--index-md.2aa46fae.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--admin-walkthrough--index-md.244e6bc0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--api--index-md.3aefd41a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--data-managers--index-md.ea9c133f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--index-md.1c4f2cb1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--vms--index-md.3a12839f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--cwl--index-md.29f1b923.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-management-challenges--index-md.5d4d1ee7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-splitting-and-parallelism--index-md.31c7354c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--elixir--index-md.e49743fc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--extended-workflow--index-md.ca7910bb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--fots--index-md.eea6d2b7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-admins--index-md.8df0d322.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-community-uk--index-md.69eb15c2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-scientists--index-md.3dd577a5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-training-network--index-md.d54633c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--index-md.d03261e3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--data-hack-roundup--index-md.581fdf4b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--header--index-md.72afdfea.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--index-md.adc3dfb6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2016--bofs--index-md.e539350a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--abstracts--index-md.7322e906.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--childcare--index-md.416182d5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--cofest--index-md.ab0c9286.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--conduct--index-md.e979f793.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--faq--index-md.e8fa4530.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--index-md.876ea583.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--organizers--index-md.23f78b02.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--promotion--index-md.85444af0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--registration--index-md.5ed4fbf8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--sponsors--index-md.482f848f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--training--index-md.0709a7f7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--venue--index-md.238012f6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--abstracts--index-md.dc64d26a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--childcare--index-md.234bf37e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--code-of-conduct--index-md.3a181b70.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--collaboration--index-md.003698d8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--faq--index-md.6b9a5e1b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--icebreaker--index-md.bd0460e3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--index-md.139334ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--keynotes--index-md.add6f018.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--organizers--index-md.8f9602b8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--promotion--index-md.55ce8ac1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--register--index-md.0b3372c2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--sponsors--index-md.4546b9bb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--training--index-md.9dbe3953.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--venue--index-md.1408cfc7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2014--index-md.4533e9c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2016--index-md.da0aae7f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2014--index-md.38dcd568.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2016--index-md.80c32555.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--index-md.176fc9a2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--trainingday--index-md.774b7249.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2016--index-md.c5ea9a89.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--example--vue-remark--index-md.9e4f4358.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--faqs--index-md.a910449f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-06--index-md.3a1cff95.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-07--index-md.bb5876b9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--icons--index-md.9a000132.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--index-md.3b38f73f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--logos--index-md.aa44aae7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--news-graphics--index-md.05ad5cba.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--photos--index-md.1d8f64ab.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--screenshots--index-md.a201402f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--index-md.d08ab1a3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--workflow-editor-unannotated--index-md.e72b5147.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--extract--index-md.c7671571.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--tool-panel--index-md.ac20782a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--variables--index-md.3e2afdd3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--faq--index-md.09b46d8e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--galaxy-pages--index-md.ff01861b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--interval-operations--index-md.e1c040b2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--main--faq--index-md.0e15bbd0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--tools--requirements--index-md.d2018317.js"><link rel="stylesheet" href="/assets/css/0.styles.e1e25242.css"><noscript data-vue-tag="ssr"><style>.g-image--loading{display:none;}</style></noscript>
  </head>
  <body >
    <div data-server-rendered="true" id="app" class="layout" data-v-340f7abc><header id="masthead"><nav class="navbar container justify-content-center navbar-dark bg-transparent navbar-expand-lg"><a href="/" target="_self" class="navbar-brand router-link-active"><img id="masthead-logo" src="/images/galaxy_logo_hub_white.svg" alt="Galaxy Community Hub" height="30"></a><button type="button" aria-label="Toggle navigation" class="navbar-toggler"><span class="navbar-toggler-icon"></span></button><div id="nav-collapse" class="navbar-collapse collapse" style="display:none;"><ul class="navbar-nav"><li class="nav-item"><a href="/use/" target="_self" class="nav-link">Use</a></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Learn</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/learn/" role="menuitem" target="_self" class="dropdown-item">How to use Galaxy</a></li><li role="presentation"><a role="menuitem" href="https://training.galaxyproject.org/" target="_self" class="dropdown-item">Teach with Galaxy</a></li></ul></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Community</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/community/" role="menuitem" target="_self" class="dropdown-item">About</a></li><li role="presentation"><a href="/events/" role="menuitem" target="_self" class="dropdown-item">Events</a></li><li role="presentation"><a href="/news/" role="menuitem" target="_self" class="dropdown-item">News</a></li><li role="presentation"><a href="/blog/" role="menuitem" target="_self" class="dropdown-item">Blog</a></li><li role="presentation"><a href="/publication-library/" role="menuitem" target="_self" class="dropdown-item">Publications</a></li><li role="presentation"><a href="/galaxy-project/statistics/" role="menuitem" target="_self" class="dropdown-item">Stats</a></li><li role="presentation"><a href="/careers/" role="menuitem" target="_self" class="dropdown-item">Careers</a></li></ul></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Deploy &amp; Develop</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/admin/get-galaxy/" role="menuitem" target="_self" class="dropdown-item">Install Galaxy</a></li><li role="presentation"><a href="/admin/" role="menuitem" target="_self" class="dropdown-item">Server administration</a></li><li role="presentation"><a href="/tools/" role="menuitem" target="_self" class="dropdown-item">Tools</a></li><li role="presentation"><a href="/develop/api/" role="menuitem" target="_self" class="dropdown-item">Working with the API</a></li><li role="presentation"><a href="/develop/" role="menuitem" target="_self" class="dropdown-item">Contributing</a></li><li role="presentation"><a href="/docs/" role="menuitem" target="_self" class="dropdown-item">Code documentation</a></li></ul></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Support</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/support/" role="menuitem" target="_self" class="dropdown-item">FAQ</a></li><li role="presentation"><a role="menuitem" href="https://help.galaxyproject.org/" target="_self" class="dropdown-item">Galaxy Help Forum</a></li></ul></li><li class="nav-item"><a href="/jxtx/" target="_self" class="nav-link">@jxtx</a></li></ul><ul class="navbar-nav ml-auto"><li class="form-inline"><form action="/search/" method="get" class="form-inline"><input id="search-input" name="q" type="text" placeholder="Search Galaxy" value="" class="mr-sm-2 form-control form-control-sm"><button type="submit" class="btn my-2 my-sm-0 btn-secondary btn-sm"><span class="fa fa-search"></span></button></form></li></ul></div></nav></header><section id="maincontainer" class="container"><header data-v-1944b8d1 data-v-340f7abc><a href="/" class="link active" data-v-1944b8d1>← Back to Home</a><!----><h1 class="title" data-v-1944b8d1>April 2015 Tool Shed Contributions</h1><!----><section class="metadata freetext" data-v-1944b8d1><!----><!----><!----></section><!----><!----></header><div class="content markdown" data-v-340f7abc><div class="toc-wrapper col-md-3">
<ul>
<li><a href="#featured-updates">Featured Updates</a></li>
<li><a href="#tools">Tools</a></li>
<li><a href="#suites">Suites</a></li>
<li><a href="#dependency-definitions">Dependency Definitions</a></li>
</ul>
</div>
<div class="body-wrapper col-md-9">
<ul>
<li><a href="/toolshed/">Wiki Root</a></li>
<li><a href="/toolshed/contributions/">All monthly summaries</a></li>
</ul>
<h3 id="featured-updates"><a href="#featured-updates" aria-hidden="true"><span class="icon icon-link"></span></a>Featured Updates</h3>
<ul>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/nilesh" target="_blank" rel="noopener noreferrer">lparsons</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/nilesh/rseqc" target="_blank" rel="noopener noreferrer">rseqc</a>: Fixed bam2wig</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/vipints" target="_blank" rel="noopener noreferrer">vipints</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/vipints/fml_gff3togtf" target="_blank" rel="noopener noreferrer">fml_gff3togtf</a>: version 2.1.0</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/crs4" target="_blank" rel="noopener noreferrer">crs4</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/crs4/kggseq_variant_selection" target="_blank" rel="noopener noreferrer">kggseq_variant_selection</a>: Update to KggSeq v0.7_20150118</li>
</ul>
</li>
</ul>
<h3 id="tools"><a href="#tools" aria-hidden="true"><span class="icon icon-link"></span></a>Tools</h3>
<ul>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/arkarachai-fungtammasan" target="_blank" rel="noopener noreferrer">arkarachai-fungtammasan</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/arkarachai-fungtammasan/str_fm" target="_blank" rel="noopener noreferrer">str_fm</a>: Pipeline to profile and genotype microsatellites from short read data This repository contains three sets of tools: 1 create microsatellite length profile, 2 correct for sequencing errors and report genotype, 3 estimate minimum sequencing read depth.</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/peterjc" target="_blank" rel="noopener noreferrer">peterjc</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/peterjc/blast_top_hit_species" target="_blank" rel="noopener noreferrer">blast_top_hit_species</a>: v0.0.1 Workflow to count species of top nr BLASTX hits of a transcriptome This is a non-trivial example workflow using the NCBI BLAST+ wrappers, intended only for crude crude contamination assessment of a transcriptome assembly. This would ideally include a visualisation of the finally tally table as a Pie Chart, currently not possible with the Galaxy Visualization Framework.</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/devteam" target="_blank" rel="noopener noreferrer">devteam</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/devteam/samtools_stats" target="_blank" rel="noopener noreferrer">samtools_stats</a>: Generate statistics for a BAM or SAM file This tool runs the samtools stats command in the SAMtools toolkit, collecting statistics from BAM files. The output can be visualized using plot-bamstats.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/devteam/samtools_reheader" target="_blank" rel="noopener noreferrer">samtools_reheader</a>: Replace SAM header. Add or replace the header in a SAM or BAM file with a new header.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/devteam/samtools_calmd" target="_blank" rel="noopener noreferrer">samtools_calmd</a>: recalculate MD/NM tags Generates the MD tag using <code>samtools calmd</code> command.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/devteam/data_manager_bowtie2_index_builder" target="_blank" rel="noopener noreferrer">data_manager_bowtie2_index_builder</a>: Data Manager for building bowtie2 indexes Data Manager for building bowtie2 indexes</li>
<li>
<p><a href="https://toolshed.g2.bx.psu.edu/view/devteam/samtools_sort" target="_blank" rel="noopener noreferrer">samtools_sort</a>: Sort alignments by leftmost coordinates or read name. This tool uses the SAMtools toolkit to sort alignments by leftmost coordinates or read names.</p>
<ul>
<li>Repository-Maintainer: <a href="/people/bjoern-gruening/">Björn Grüning</a></li>
<li>Repository-Development: <a href="https://github.com/bgruening/galaxytools" target="_blank" rel="noopener noreferrer">https://github.com/bgruening/galaxytools</a></li>
</ul>
</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/devteam/samtools_idxstats" target="_blank" rel="noopener noreferrer">samtools_idxstats</a>: BAM mapping statistics (using samtools idxstats) Runs "samtools idxstats" to get mapping statistics from a sorted index BAM file as a tabular output file.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/devteam/samtools_split" target="_blank" rel="noopener noreferrer">samtools_split</a>:  Split a BAM file by read group This tool runs the samtools split command in the SAMtools toolkit to split BAM files by read group.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/devteam/cd_hit_dup" target="_blank" rel="noopener noreferrer">cd_hit_dup</a>: simple tool for removing duplicates from sequencing reads  cd-hit-dup is a simple tool for removing duplicates from sequencing reads, with optional step to detect and remove chimeric reads. A number of options are provided to tune how the duplicates are removed.</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/yutaka-saito" target="_blank" rel="noopener noreferrer">yutaka-saito</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/yutaka-saito/bisulfighter" target="_blank" rel="noopener noreferrer">bisulfighter</a>: A pipeline for accurate detection of methylated cytosines and differentially methylated regions A pipeline for accurate detection of methylated cytosines and differentially methylated regions</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/mir-bioinf" target="_blank" rel="noopener noreferrer">mir-bioinf</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/mir-bioinf/multi_join_left" target="_blank" rel="noopener noreferrer">multi_join_left</a>: Joins multiple files serially Use this tool to join multiple tab-delimited files serially on one column; performs a left-join starting with the first file specified (first specified file is printed in full, then only matching lines are printed from each subsequent file).</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/mir-bioinf/heatmap_colormanipulation" target="_blank" rel="noopener noreferrer">heatmap_colormanipulation</a>: Initial upload Generates heatmap with nonlinear color mapping options Uses R's heatmap.2 function and gplots package. Good for visualization of disparate data.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/mir-bioinf/fix_excel_date_symbols" target="_blank" rel="noopener noreferrer">fix_excel_date_symbols</a>: Initial upload Can recover human or mouse gene symbols from dates (usually caused by Excel) When Excel sees a gene symbol like SEPT9, it thinks to itself 'must be a date' and automatically converts it to the date 9-Sep (permanent conversion). As an 'after-the-fact' remedy, this tool converts dates like 9-Sept, 3-March, etc, in the chosen column(s) back to the gene symbols SEPT9 and MARCH3, etc. (respectively).</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/mir-bioinf/column_range_string_generator" target="_blank" rel="noopener noreferrer">column_range_string_generator</a>: Initial upload Generates column string for input to Cut Columns tool Use this tool to generate a string that can be copied to the buffer then pasted into the Cut Column tool to save time (rather than needing to individually type out C1,C2,C3..... etc.).</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp" target="_blank" rel="noopener noreferrer">galaxyp</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/unipept" target="_blank" rel="noopener noreferrer">unipept</a>: Unipept retrieves taxonomy for tryptic peptides Unipept metaproteomics analysis. Unipept retrieves taxonomy for tryptic peptides using the unipept API.</li>
</ul>
</li>
</ul>
<h3 id="suites"><a href="#suites" aria-hidden="true"><span class="icon icon-link"></span></a>Suites</h3>
<ul>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/arkarachai-fungtammasan" target="_blank" rel="noopener noreferrer">arkarachai-fungtammasan</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/arkarachai-fungtammasan/suite_microsatellite_ngs_1_0" target="_blank" rel="noopener noreferrer">suite_microsatellite_ngs_1_0</a>: all dependency for microsattelite_ngs package and microsattelite_ngs itself</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/arkarachai-fungtammasan/suite_str_fm_1_0" target="_blank" rel="noopener noreferrer">suite_str_fm_1_0</a>: all dependency for str_fm package and str_fm itself</li>
</ul>
</li>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/devteam" target="_blank" rel="noopener noreferrer">devteam</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/devteam/suite_samtools_1_2" target="_blank" rel="noopener noreferrer">suite_samtools_1_2</a>: A suite of Galaxy utilities associated with version 1.2 of the SAMtools package. SAM (Sequence Alignment/Map) format is a generic format for storing large nucleotide sequence alignments. This repository suite associates selected repositories containing Galaxy utilities that require version 1.2 of the SAMTools package. These associated Galaxy utilities consist of a Galaxy Data Manager contained in the repository named data_manager_sam_fasta_index_builder and Galaxy tools contained in several separate repositories.</li>
</ul>
</li>
</ul>
<h3 id="dependency-definitions"><a href="#dependency-definitions" aria-hidden="true"><span class="icon icon-link"></span></a>Dependency Definitions</h3>
<ul>
<li>
<p><em>From <a href="https://toolshed.g2.bx.psu.edu/view/iuc" target="_blank" rel="noopener noreferrer">iuc</a>:</em></p>
<ul>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/package_cutadapt_1_8" target="_blank" rel="noopener noreferrer">package_cutadapt_1_8</a>: Contains a tool dependency definition that downloads and compiles version 1.8 of cutadapt Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/package_cd_hit_auxtools_0_5_2012-03-07_fix_dan" target="_blank" rel="noopener noreferrer">package_cd_hit_auxtools_0_5_2012_03_07_fix_dan</a>: Builds and installs cd-hit-auxtools including cd-hit-dup, cd-hit-lap, and read-linker.</li>
<li><a href="https://toolshed.g2.bx.psu.edu/view/iuc/package_nwalign_0_3_1" target="_blank" rel="noopener noreferrer">package_nwalign_0_3_1</a>: Contains a tool dependency definition for version 0.3.1 of the python nwalign package nwalign is an implementation of Needleman-Wunsch global sequence alignment. nwalign is a resource efficient alternative to scikit alignment.global_pairwise_align_nucleotide and BioPython pairwise2</li>
</ul>
</li>
</ul>
</div>
</div><footer data-v-4b3455de data-v-340f7abc><!----></footer><footer class="static-footer markdown"><p>The Galaxy Project is supported in part by NSF, NHGRI, The Huck Institutes of the Life Sciences, The Institute for CyberScience at Penn State, and Johns Hopkins University.</p>
</footer></section></div>
    <script>window.__INITIAL_STATE__={"data":{"article":{"id":"8bea819f82acac7aaa20fad3f4a2d044","title":"April 2015 Tool Shed Contributions","tease":"","category":null,"date":null,"days":null,"contact":"","contact_url":"","authors":"","location":"","location_url":"","source_blog":"","source_blog_url":"","skip_title_render":null,"links":[],"image":"","images":{},"external_url":"","content":"\u003Cdiv class=\"toc-wrapper col-md-3\"\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"#featured-updates\"\u003EFeatured Updates\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#tools\"\u003ETools\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#suites\"\u003ESuites\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#dependency-definitions\"\u003EDependency Definitions\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fdiv\u003E\n\u003Cdiv class=\"body-wrapper col-md-9\"\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"\u002Ftoolshed\u002F\"\u003EWiki Root\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"\u002Ftoolshed\u002Fcontributions\u002F\"\u003EAll monthly summaries\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"featured-updates\"\u003E\u003Ca href=\"#featured-updates\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EFeatured Updates\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cem\u003EFrom \u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fnilesh\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Elparsons\u003C\u002Fa\u003E:\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fnilesh\u002Frseqc\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Erseqc\u003C\u002Fa\u003E: Fixed bam2wig\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cem\u003EFrom \u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fvipints\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Evipints\u003C\u002Fa\u003E:\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fvipints\u002Ffml_gff3togtf\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Efml_gff3togtf\u003C\u002Fa\u003E: version 2.1.0\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cem\u003EFrom \u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fcrs4\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ecrs4\u003C\u002Fa\u003E:\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fcrs4\u002Fkggseq_variant_selection\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ekggseq_variant_selection\u003C\u002Fa\u003E: Update to KggSeq v0.7_20150118\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"tools\"\u003E\u003Ca href=\"#tools\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003ETools\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cem\u003EFrom \u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Farkarachai-fungtammasan\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Earkarachai-fungtammasan\u003C\u002Fa\u003E:\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Farkarachai-fungtammasan\u002Fstr_fm\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Estr_fm\u003C\u002Fa\u003E: Pipeline to profile and genotype microsatellites from short read data This repository contains three sets of tools: 1 create microsatellite length profile, 2 correct for sequencing errors and report genotype, 3 estimate minimum sequencing read depth.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cem\u003EFrom \u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fpeterjc\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Epeterjc\u003C\u002Fa\u003E:\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fpeterjc\u002Fblast_top_hit_species\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Eblast_top_hit_species\u003C\u002Fa\u003E: v0.0.1 Workflow to count species of top nr BLASTX hits of a transcriptome This is a non-trivial example workflow using the NCBI BLAST+ wrappers, intended only for crude crude contamination assessment of a transcriptome assembly. This would ideally include a visualisation of the finally tally table as a Pie Chart, currently not possible with the Galaxy Visualization Framework.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cem\u003EFrom \u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fdevteam\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Edevteam\u003C\u002Fa\u003E:\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fdevteam\u002Fsamtools_stats\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Esamtools_stats\u003C\u002Fa\u003E: Generate statistics for a BAM or SAM file This tool runs the samtools stats command in the SAMtools toolkit, collecting statistics from BAM files. The output can be visualized using plot-bamstats.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fdevteam\u002Fsamtools_reheader\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Esamtools_reheader\u003C\u002Fa\u003E: Replace SAM header. Add or replace the header in a SAM or BAM file with a new header.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fdevteam\u002Fsamtools_calmd\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Esamtools_calmd\u003C\u002Fa\u003E: recalculate MD\u002FNM tags Generates the MD tag using \u003Ccode\u003Esamtools calmd\u003C\u002Fcode\u003E command.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fdevteam\u002Fdata_manager_bowtie2_index_builder\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Edata_manager_bowtie2_index_builder\u003C\u002Fa\u003E: Data Manager for building bowtie2 indexes Data Manager for building bowtie2 indexes\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fdevteam\u002Fsamtools_sort\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Esamtools_sort\u003C\u002Fa\u003E: Sort alignments by leftmost coordinates or read name. This tool uses the SAMtools toolkit to sort alignments by leftmost coordinates or read names.\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003ERepository-Maintainer: \u003Ca href=\"\u002Fpeople\u002Fbjoern-gruening\u002F\"\u003EBjörn Grüning\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003ERepository-Development: \u003Ca href=\"https:\u002F\u002Fgithub.com\u002Fbgruening\u002Fgalaxytools\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttps:\u002F\u002Fgithub.com\u002Fbgruening\u002Fgalaxytools\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fdevteam\u002Fsamtools_idxstats\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Esamtools_idxstats\u003C\u002Fa\u003E: BAM mapping statistics (using samtools idxstats) Runs \"samtools idxstats\" to get mapping statistics from a sorted index BAM file as a tabular output file.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fdevteam\u002Fsamtools_split\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Esamtools_split\u003C\u002Fa\u003E:  Split a BAM file by read group This tool runs the samtools split command in the SAMtools toolkit to split BAM files by read group.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fdevteam\u002Fcd_hit_dup\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ecd_hit_dup\u003C\u002Fa\u003E: simple tool for removing duplicates from sequencing reads  cd-hit-dup is a simple tool for removing duplicates from sequencing reads, with optional step to detect and remove chimeric reads. A number of options are provided to tune how the duplicates are removed.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cem\u003EFrom \u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fyutaka-saito\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Eyutaka-saito\u003C\u002Fa\u003E:\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fyutaka-saito\u002Fbisulfighter\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ebisulfighter\u003C\u002Fa\u003E: A pipeline for accurate detection of methylated cytosines and differentially methylated regions A pipeline for accurate detection of methylated cytosines and differentially methylated regions\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cem\u003EFrom \u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fmir-bioinf\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Emir-bioinf\u003C\u002Fa\u003E:\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fmir-bioinf\u002Fmulti_join_left\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Emulti_join_left\u003C\u002Fa\u003E: Joins multiple files serially Use this tool to join multiple tab-delimited files serially on one column; performs a left-join starting with the first file specified (first specified file is printed in full, then only matching lines are printed from each subsequent file).\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fmir-bioinf\u002Fheatmap_colormanipulation\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Eheatmap_colormanipulation\u003C\u002Fa\u003E: Initial upload Generates heatmap with nonlinear color mapping options Uses R's heatmap.2 function and gplots package. Good for visualization of disparate data.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fmir-bioinf\u002Ffix_excel_date_symbols\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Efix_excel_date_symbols\u003C\u002Fa\u003E: Initial upload Can recover human or mouse gene symbols from dates (usually caused by Excel) When Excel sees a gene symbol like SEPT9, it thinks to itself 'must be a date' and automatically converts it to the date 9-Sep (permanent conversion). As an 'after-the-fact' remedy, this tool converts dates like 9-Sept, 3-March, etc, in the chosen column(s) back to the gene symbols SEPT9 and MARCH3, etc. (respectively).\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fmir-bioinf\u002Fcolumn_range_string_generator\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ecolumn_range_string_generator\u003C\u002Fa\u003E: Initial upload Generates column string for input to Cut Columns tool Use this tool to generate a string that can be copied to the buffer then pasted into the Cut Column tool to save time (rather than needing to individually type out C1,C2,C3..... etc.).\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cem\u003EFrom \u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fgalaxyp\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Egalaxyp\u003C\u002Fa\u003E:\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fgalaxyp\u002Funipept\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Eunipept\u003C\u002Fa\u003E: Unipept retrieves taxonomy for tryptic peptides Unipept metaproteomics analysis. Unipept retrieves taxonomy for tryptic peptides using the unipept API.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"suites\"\u003E\u003Ca href=\"#suites\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003ESuites\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cem\u003EFrom \u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Farkarachai-fungtammasan\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Earkarachai-fungtammasan\u003C\u002Fa\u003E:\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Farkarachai-fungtammasan\u002Fsuite_microsatellite_ngs_1_0\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Esuite_microsatellite_ngs_1_0\u003C\u002Fa\u003E: all dependency for microsattelite_ngs package and microsattelite_ngs itself\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Farkarachai-fungtammasan\u002Fsuite_str_fm_1_0\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Esuite_str_fm_1_0\u003C\u002Fa\u003E: all dependency for str_fm package and str_fm itself\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cem\u003EFrom \u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fdevteam\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Edevteam\u003C\u002Fa\u003E:\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fdevteam\u002Fsuite_samtools_1_2\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Esuite_samtools_1_2\u003C\u002Fa\u003E: A suite of Galaxy utilities associated with version 1.2 of the SAMtools package. SAM (Sequence Alignment\u002FMap) format is a generic format for storing large nucleotide sequence alignments. This repository suite associates selected repositories containing Galaxy utilities that require version 1.2 of the SAMTools package. These associated Galaxy utilities consist of a Galaxy Data Manager contained in the repository named data_manager_sam_fasta_index_builder and Galaxy tools contained in several separate repositories.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"dependency-definitions\"\u003E\u003Ca href=\"#dependency-definitions\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EDependency Definitions\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cem\u003EFrom \u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fiuc\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Eiuc\u003C\u002Fa\u003E:\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fiuc\u002Fpackage_cutadapt_1_8\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Epackage_cutadapt_1_8\u003C\u002Fa\u003E: Contains a tool dependency definition that downloads and compiles version 1.8 of cutadapt Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fiuc\u002Fpackage_cd_hit_auxtools_0_5_2012-03-07_fix_dan\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Epackage_cd_hit_auxtools_0_5_2012_03_07_fix_dan\u003C\u002Fa\u003E: Builds and installs cd-hit-auxtools including cd-hit-dup, cd-hit-lap, and read-linker.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002Fview\u002Fiuc\u002Fpackage_nwalign_0_3_1\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Epackage_nwalign_0_3_1\u003C\u002Fa\u003E: Contains a tool dependency definition for version 0.3.1 of the python nwalign package nwalign is an implementation of Needleman-Wunsch global sequence alignment. nwalign is a resource efficient alternative to scikit alignment.global_pairwise_align_nucleotide and BioPython pairwise2\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fdiv\u003E\n"}},"context":{}};(function(){var s;(s=document.currentScript||document.scripts[document.scripts.length-1]).parentNode.removeChild(s);}());</script><script src="/assets/js/app.0690a31c.js" defer></script><script src="/assets/js/page--src--templates--article-vue.6d736ff4.js" defer></script>
  </body>
</html>
