<!DOCTYPE html>
<html data-html-server-rendered="true" lang="en" data-vue-tag="%7B%22lang%22:%7B%22ssr%22:%22en%22%7D%7D">
  <head>
    <title>June 2013 Galaxy Update - Galaxy Community Hub</title><meta name="gridsome:hash" content="c32ecfdb33b075b0057598b5b9118f3554a398cf"><meta data-vue-tag="ssr" charset="utf-8"><meta data-vue-tag="ssr" name="generator" content="Gridsome v0.7.23"><meta data-vue-tag="ssr" data-key="viewport" name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover"><meta data-vue-tag="ssr" data-key="format-detection" name="format-detection" content="telephone=no"><meta data-vue-tag="ssr" data-key="description" name="description" content="All about Galaxy and its community."><link data-vue-tag="ssr" rel="icon" href="data:,"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="16x16" href="/assets/static/favicon.ce0531f.7d4a227415c9d9b704d56a079d49a3e1.png"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="32x32" href="/assets/static/favicon.ac8d93a.7d4a227415c9d9b704d56a079d49a3e1.png"><link rel="preload" href="/assets/css/0.styles.28333459.css" as="style"><link rel="preload" href="/assets/js/app.f7f9943a.js" as="script"><link rel="preload" href="/assets/js/page--src--templates--vue-article-vue.d03e3b77.js" as="script"><link rel="preload" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-06--index-md.a5232112.js" as="script"><link rel="prefetch" href="/assets/js/page--node-modules--gridsome--app--pages--404-vue.54a07ec7.js"><link rel="prefetch" href="/assets/js/page--src--pages--blog-vue.72cfdf23.js"><link rel="prefetch" href="/assets/js/page--src--pages--careers-vue.51f7f627.js"><link rel="prefetch" href="/assets/js/page--src--pages--data-explorer-vue.cda0b563.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--archive-vue.54bd407e.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests--papercuts-vue.d73bf9af.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests-vue.f9ca327e.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--webinars-vue.2b0c4b1f.js"><link rel="prefetch" href="/assets/js/page--src--pages--events-vue.2bc0181e.js"><link rel="prefetch" href="/assets/js/page--src--pages--index-vue.9e6f30c2.js"><link rel="prefetch" href="/assets/js/page--src--pages--news-vue.c55be461.js"><link rel="prefetch" href="/assets/js/page--src--pages--tests-vue.4730afa8.js"><link rel="prefetch" href="/assets/js/page--src--pages--use-vue.8c7d1a05.js"><link rel="prefetch" href="/assets/js/page--src--templates--article-vue.9c961af0.js"><link rel="prefetch" href="/assets/js/page--src--templates--platform-vue.672f4f9e.js"><link rel="prefetch" href="/assets/js/vendors~page--src--pages--careers-vue~page--src--pages--use-vue~page--src--templates--article-vue~pa~efade918.71d44943.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--index-md.b120794f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--md--imgtests--imgdir--index-md.8a720dc7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--partials--index-md.88a2cc0b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--imgdir--index-md.a7da7426.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--index-md.ab7ce26b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--index-md.ac0f0339.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--index-md.7388aadd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--reldir--index-md.78e6d088.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--capacity-planning--index-md.532d558e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--index-md.f596bca3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--performance--cluster--legacy--index-md.9900d899.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--datatypes--index-md.37a06167.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--disk-quotas--index-md.99623880.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--faq--index-md.dfcae1f0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--data-model--index-md.f6811d25.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--galaxy-ops--index-md.da494b7d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--ten-simple-steps-galaxy-as-a-service--index-md.f0b31fa8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--tool-config-syntax--index-md.c133b049.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--writing-tests--index-md.d3ed2aee.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--allocation--index-md.88a6f777.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--faq--index-md.6b5ae7d2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--index-md.54b831ad.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--ssh--index-md.c431c940.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--troubleshooting--index-md.2bbf34e9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--capacity-planning--index-md.8d2c9c03.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--getting-started--index-md.63fe5edc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--index-md.1387a7bc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--building--index-md.ea5bc352.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--capacity-planning--index-md.bdfd10d5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--cluster-types--index-md.83943371.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--customizing--index-md.6b8b3678.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--faq--index-md.12c74d13.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--index-md.e3fa7dae.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--pre-201703--index-md.9b084a41.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--hadoop--index-md.95f1458a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--ht-condor--index-md.f973feff.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--index-md.37f11743.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--openstack--index-md.f4d0e62a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--services--index-md.e26413ae.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--sharing--index-md.c15fa6cb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--troubleshooting--index-md.953f49c4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--userdata--index-md.1116e088.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--future--index-md.ec7dff38.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--index-md.f0bd7fb5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-09--index-md.ed1d541b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-27--index-md.5a2cbe0c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-09-19--index-md.36fcc10e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-11-14--index-md.5155dfcc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-01-16--index-md.e740402b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-03-20--index-md.fa75b1de.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-05-15--index-md.ad74e436.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-11-20--index-md.6a1074f1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-02-19--index-md.bdefdea8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-04-16--index-md.ce613da7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-06-18--index-md.9253a34b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-08-20--index-md.8ac977c4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-10-15--index-md.3a558b10.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-02-18--index-md.ae82f681.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-04-21--index-md.4cd1f8fc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-06-16--index-md.7b1572c5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-08-18--index-md.5b3383d1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--index-md.da754856.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--webinar-tech--index-md.c5c416a6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2012--index-md.4eddb972.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2014--index-md.5d7b4578.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--index-md.afac4bb5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--data--index-md.f2ba0502.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--datafiles--index-md.4dfd4db7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--architecture--index-md.8035154b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--bitbucket--index-md.7cc7d8d1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--css--index-md.710209d9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--getting-started--index-md.930b3a22.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--index-md.1cbec221.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--jsa--index-md.134e6a35.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--mercurial--index-md.9026901a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--security-tool-tips--index-md.2d704e67.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--galaxy-project--index-md.230e8088.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--index-md.e5883590.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--papers--index-md.db9c7f1b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--posters--index-md.04432e3d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--presentations--index-md.92923a05.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--videos--index-md.f680b009.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--advanced-session--index-md.4227a5ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--basics-session--index-md.08c5b284.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--index-md.9844ef78.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--logistics--index-md.6f6b160a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--publicity--index-md.dfbc1a41.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--registration--index-md.5232161d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--abstracts--index-md.697fe169.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--index-md.ca51927e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--key-dates--index-md.606cfcac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--lightning-talks--index-md.6a60add5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--logistics--index-md.06ca864d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--organizing-committee--index-md.0500ed90.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--photos--index-md.ab68ac2c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--automation-strategies--index-md.2d8a9043.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--bioinformatics-training--index-md.43a55fdc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--end-users--index-md.5f46cd2a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--workflows-and-api--index-md.07da71b4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--index-md.cb8d0657.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--promotion--index-md.4eb64cde.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--register--index-md.49c36527.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--sponsorships--index-md.db0f3791.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--training-day--index-md.c8f12515.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--index-md.3c67479f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--posters--index-md.728aa734.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--talks--index-md.7f93c9db.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--clinical-use-hpc-file-systems--index-md.030a88bd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-admins--index-md.f12f7b4d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-blast--index-md.59e8820f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-professionals--index-md.dcb27276.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--index-md.cfaaca20.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--proteomics--index-md.abba045c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--public-galaxy-servers--index-md.bd34ea15.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--tool-developers--index-md.a5a0a8b6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--index-md.df6174ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--key-dates--index-md.f7630162.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--lightning--index-md.0dfb5a95.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--logistics--index-md.4759076c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--organizers--index-md.6fc11b96.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--photos--index-md.aef50972.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--program--index-md.05c5d87a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--promotion--index-md.e34ad376.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--register--index-md.8ea1785f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--sponsorships--index-md.3cde86d8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--advance-tool-data--index-md.2210b6c4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--api--index-md.57c06e35.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--index-md.4cd32016.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--intro-tool-data--index-md.75a7abbd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--vms--index-md.1c86a58b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--index-md.9abee1c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--posters--index-md.810cd33b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--talks--index-md.c9751906.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--branch-release-merge--index-md.f6ed714e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-admins--index-md.5f9fcb01.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-end-users--index-md.00b4cd3b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-training-network--index-md.a843f5f7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--heterogeneous--index-md.8313575f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--index-md.f878a9f6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--using-galaxy-ui-overview--index-md.5b284474.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--hackathon--index-md.20172bcf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--index-md.9ea6d61e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--key-dates--index-md.bac2859e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--lightning--index-md.96c8f509.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--logistics--index-md.c1318807.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--organizers--index-md.b183376c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--photos--index-md.e0b61b7a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--program--index-md.a93fc58b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--promotion--index-md.df1d2b77.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--register--index-md.54adaeb3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--sponsor-exhibit--index-md.64ef93be.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--admin-walkthrough--index-md.c82adb0e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--api--index-md.a936c5d1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--data-managers--index-md.348024f8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--index-md.030f1ebd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--vms--index-md.507d7d15.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--cwl--index-md.a8a9aa76.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-management-challenges--index-md.409c72aa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-splitting-and-parallelism--index-md.6332a3a3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--elixir--index-md.982da27f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--extended-workflow--index-md.f2b42ad0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--fots--index-md.52645a6e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-admins--index-md.ffda743c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-community-uk--index-md.060ecf60.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-scientists--index-md.ae9e7e59.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-training-network--index-md.a412fdd5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--index-md.22f27d14.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--data-hack-roundup--index-md.60bc6484.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--header--index-md.dbba4e7d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--index-md.2f85a581.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2016--bofs--index-md.5a3032e2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--abstracts--index-md.57e795e8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--childcare--index-md.096b34d2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--cofest--index-md.8b0a9b93.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--conduct--index-md.8e51f30c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--faq--index-md.0f0f1b46.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--index-md.641b44a6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--organizers--index-md.787e0ccf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--promotion--index-md.e6e58e52.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--registration--index-md.52f1b4f7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--sponsors--index-md.ef140359.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--training--index-md.ee0d3a7c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--venue--index-md.30256085.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--abstracts--index-md.137e7bc6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--childcare--index-md.84f642d3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--code-of-conduct--index-md.2789988f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--collaboration--index-md.a48ddf3b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--faq--index-md.4fd33733.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--icebreaker--index-md.ce20a106.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--index-md.9c798d19.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--keynotes--index-md.080e361c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--organizers--index-md.a3b956fb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--promotion--index-md.de985f84.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--register--index-md.05d6bc90.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--sponsors--index-md.4c398a30.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--training--index-md.ce0ef35c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--venue--index-md.4a3ddfec.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2014--index-md.d56576b3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2016--index-md.9f18f6ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2014--index-md.6fd15769.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2016--index-md.c0b34aae.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--index-md.ba91d0c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--trainingday--index-md.e89c50c3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2016--index-md.6afda26b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--example--vue-remark--index-md.c96a8cd8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--faqs--index-md.3bba7da9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-07--index-md.19df6105.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--icons--index-md.ff92f6d4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--index-md.246f22cd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--logos--index-md.5a8a4cdf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--news-graphics--index-md.844f04b7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--photos--index-md.7a556820.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--screenshots--index-md.290a5433.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--index-md.467e2afe.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--workflow-editor-unannotated--index-md.bc93cd9f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--extract--index-md.940bc220.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--tool-panel--index-md.7ac3e984.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--variables--index-md.a3612f7c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--faq--index-md.599e559f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--galaxy-pages--index-md.51df5ee9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--interval-operations--index-md.71022235.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--main--faq--index-md.f6a425fa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--tools--requirements--index-md.1aef8fd3.js"><link rel="stylesheet" href="/assets/css/0.styles.28333459.css"><noscript data-vue-tag="ssr"><style>.g-image--loading{display:none;}</style></noscript>
  </head>
  <body >
    <div data-server-rendered="true" id="app" class="layout" data-v-4735ab8b><header id="masthead"><nav class="container navbar navbar-expand-lg navbar-dark justify-content-center"><a href="/" class="navbar-brand active"><img id="masthead-logo" src="/images/galaxy_logo_hub_white.svg" alt="Galaxy Community Hub" height="30"></a><div id="navbarToggleWrapper" class="collapse navbar-collapse mx-auto"><ul class="navbar-nav"><li class="nav-item"><a href="/use/" role="button" aria-expanded="false" class="nav-link dropdown">
          Use
        </a></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" aria-expanded="false" class="nav-link dropdown-toggle">
          Learn
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/learn/" class="dropdown-item">How to use Galaxy</a></li><li><a href="https://training.galaxyproject.org/" class="dropdown-item">Teach with Galaxy</a></li></ul></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" class="nav-link dropdown-toggle">
          Community
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/community/" class="dropdown-item">About</a></li><li><a href="/events/" class="dropdown-item">Events</a></li><li><a href="/news/" class="dropdown-item">News</a></li><li><a href="/blog/" class="dropdown-item">Blog</a></li><li><a href="/publication-library/" class="dropdown-item">Publications</a></li><li><a href="/galaxy-project/statistics/" class="dropdown-item">Stats</a></li><li><a href="/careers/" class="dropdown-item">Careers</a></li></ul></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" class="nav-link dropdown-toggle">
          Deploy &amp; Develop
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/admin/get-galaxy/" class="dropdown-item">Install Galaxy</a></li><li><a href="/admin/" class="dropdown-item">Server administration</a></li><li><a href="/tools/" class="dropdown-item">Tools</a></li><li><a href="/develop/api/" class="dropdown-item">Working with the API</a></li><li><a href="/develop/" class="dropdown-item">Contributing</a></li><li><a href="/docs/" class="dropdown-item">Code documentation</a></li></ul></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" class="nav-link dropdown-toggle">
          Support
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/support/" class="dropdown-item">FAQ</a></li><li><a href="https://help.galaxyproject.org/" class="dropdown-item">Galaxy Help Forum</a></li></ul></li><li class="nav-item"><a href="/jxtx/" role="button" aria-expanded="false" class="nav-link dropdown">
          @jxtx
        </a></li></ul></div></nav></header><section id="maincontainer" class="container"><a href="/" class="link active" data-v-4735ab8b> ← Home</a><header data-v-4735ab8b><!----><h1 class="title" data-v-4735ab8b>June 2013 Galaxy Update</h1><section class="metadata" data-v-4735ab8b><!----><!----><!----></section></header><article class="content markdown" data-v-4735ab8b><div data-v-4735ab8b><div class="left"><a href="/src/galaxy-updates/index.md"><img src="/src/images/logos/GalaxyUpdate200.png" alt="Galaxy Updates" width="150"></a></div><p>Welcome to the June 2013 <em><a href="/src/galaxy-updates/index.md">Galaxy Update</a></em>, a monthly summary of what is going on in the Galaxy community. <em><a href="/src/galaxy-updates/index.md">Galaxy Updates</a></em> complement the <em><a href="/src/docs/index.md">Galaxy Development News Briefs</a></em> which accompany new Galaxy releases and focus on Galaxy code updates.</p><br><h2 id="new-public-servers"><a href="#new-public-servers" aria-hidden="true"><span class="icon icon-link"></span></a>New Public Servers</h2><p><a href="http://www.odose.nl/" target="_blank" rel="noopener noreferrer">ODoSE</a> and <a href="http://services.cbib.u-bordeaux2.fr/galaxy/" target="_blank" rel="noopener noreferrer">CBiB Galaxy</a> joined <a href="/src/use/index.md">the list of over 30 publicly accessible Galaxy servers</a> in May.</p><h3 id="odose"><a href="#odose" aria-hidden="true"><span class="icon icon-link"></span></a>ODoSE</h3><div class="left"><a href="http://www.odose.nl/"><img src="/src/images/logos/odose-scatter-plot.png" alt="ODoSE: Ortholog Direction of Selection Engine" width="200"></a></div><p>The <a href="http://www.odose.nl/" target="_blank" rel="noopener noreferrer">Ortholog Direction of Selection Engine (ODoSE)</a> supports genome-wide calculation of adaptive divergence in prokaryotes:</p><p>&quot;The web-based graphical user-interface ODoSE (Ortholog Direction of Selection Engine) identifies all orthologs between a set of strains and allows the calculation of a novel extension of the [McDonald Kreitman] MK test, the Direction of Selection (DoS) statistic as well as the calculation of the Neutrality Index (NI) statistic for all genes shared by two taxonomic groups combined. The engine also generates the site frequency spectrum for each gene allowing one to apply more advanced methods for estimating the distribution of fitness effects and rates of adaptive evolution.&quot;</p><p>See <a href="http://bit.ly/10ydSiO" target="_blank" rel="noopener noreferrer">ODoSE: A Webserver for Genome-Wide Calculation of Adaptive Divergence in Prokaryotes</a> by Michiel Vos, Tim A. H. te Beek, Marc A. van Driel, Martijn A. Huynen, Adam Eyre-Walker, Mark W. J. van Passel; PLoS ONE, Vol. 8, No. 5. (6 May 2013), e62447, doi:10.1371/journal.pone.0062447</p><div class="right"><br>[[<a href="http://services.cbib.u-bordeaux2.fr/galaxy/"><img src="/src/images/logos/cbib-logo-300.png" alt="CBiB Galaxy" width="200"></a></div><h3 id="cbib-galaxy"><a href="#cbib-galaxy" aria-hidden="true"><span class="icon icon-link"></span></a>CBiB Galaxy</h3><p><a href="http://services.cbib.u-bordeaux2.fr/galaxy/" target="_blank" rel="noopener noreferrer">CBiB Galaxy</a> is a general purpose Galaxy instance  that includes EMBOSS (a software analysis package for molecular biology) and fibronectin (diversity analysis of synthetic libraries of a Fibronectin domain).  For now, the computational resources of the Galaxy instance of the CBiB are relatively limited. However, a migration to a dedicated server will be made ​​soon.  </p><p><a href="http://services.cbib.u-bordeaux2.fr/galaxy/" target="_blank" rel="noopener noreferrer">CBiB Galaxy</a> is sponsored by <a href="http://www.cbib.u-bordeaux2.fr/" target="_blank" rel="noopener noreferrer">Centre de Bioinformatique de Bordeaux</a>.</p><div class="left"><table><tr><th> # </th><th> Tag </th></tr><tr><td style="text-align:right;"> 24 </td><td><a href="http://www.citeulike.org/group/16008/tag/methods">methods</a></td></tr><tr><td style="text-align:right;"> 10 </td><td><a href="http://www.citeulike.org/group/16008/tag/workbench">workbench</a></td></tr><tr><td style="text-align:right;"> 9 </td><td><a href="http://www.citeulike.org/group/16008/tag/usemain">usemain</a></td></tr><tr><td style="text-align:right;"> 2 </td><td><a href="http://www.citeulike.org/group/16008/tag/isgalaxy">isgalaxy</a></td></tr><tr><td style="text-align:right;"> 2 </td><td><a href="http://www.citeulike.org/group/16008/tag/uselocal">uselocal</a></td></tr><tr><td style="text-align:right;"> 2 </td><td><a href="http://www.citeulike.org/group/16008/tag/unknown">unknown</a></td></tr><tr><td style="text-align:right;"> 1 </td><td><a href="http://www.citeulike.org/group/16008/tag/cloud">cloud</a></td></tr><tr><td style="text-align:right;"> 1 </td><td><a href="http://www.citeulike.org/group/16008/tag/shared">shared</a></td></tr></table></div><h2 id="new-papers"><a href="#new-papers" aria-hidden="true"><span class="icon icon-link"></span></a>New Papers</h2><p>39 new papers were added to the Galaxy CiteULike Group in May, including:</p><ul><li><p>&quot;<a href="https://doi.org/10.1093/nar/gkt342" target="_blank" rel="noopener noreferrer">The Genomic HyperBrowser: an analysis web server for genome-scale data</a>&quot; by Geir K. Sandve, Sveinung Gundersen, Morten Johansen, <em>et al.</em>; <em><a href="http://nar.oxfordjournals.org/" target="_blank" rel="noopener noreferrer">Nucleic Acids Research</a></em> (30 April 2013)</p></li><li><p>&quot;<a href="https://doi.org/10.1371/journal.pone.0062447" target="_blank" rel="noopener noreferrer">ODoSE: A Webserver for Genome-Wide Calculation of Adaptive Divergence in Prokaryotes</a>&quot; by Michiel Vos, Tim A. H. te Beek, Marc A. van Driel, Martijn A. Huynen, Adam Eyre-Walker, Mark W. J. van Passel; <em><a href="http://www.plosone.org/" target="_blank" rel="noopener noreferrer">PLoS ONE</a></em>, Vol. 8, No. 5. (6 May 2013), e62447</p></li></ul><p>There are now almost <a href="http://www.citeulike.org/group/16008/" target="_blank" rel="noopener noreferrer">1000 papers in the Galaxy CiteULike Group</a>.</p><br><h2 id="whos-hiring"><a href="#whos-hiring" aria-hidden="true"><span class="icon icon-link"></span></a>Who's Hiring</h2><div class="right"><a href="/src/galaxy-is-hiring/index.md"><img src="/src/images/news-graphics/GalaxyIsHiringWordCloud2.png" alt="Please Help! Yes you!" width="200"></a></div><p>The Galaxy is expanding! Please help it grow.</p><ul><li><strong>The <a href="/src/galaxy-is-hiring/index.md">Galaxy Project is hiring software engineers and post-docs</a>!</strong> at both Emory and Penn State.</li><li><a href="http://www.nature.com/naturejobs/science/jobs/321283-Senior-Developer-Stem-Cell-Bioinformatics-Core" target="_blank" rel="noopener noreferrer">Senior Developer</a>, Stem Cell Bioinformatics Core, Sage Bionetworks, Seattle, WA, United States</li><li><a href="http://bit.ly/VkRtrb" target="_blank" rel="noopener noreferrer">Bioinformatics Support Group Leader</a> @ LSU</li></ul><p>Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the <a href="/src/news/index.md">Galaxy News feed</a> and include it in next month's <a href="/src/galaxy-updates/index.md">update</a>.</p><h2 id="gcc2013"><a href="#gcc2013" aria-hidden="true"><span class="icon icon-link"></span></a>GCC2013</h2><div class="center"><a href="/src/events/gcc2013/index.md"><img src="/src/images/logos/GCC2013Logo400.png" alt="2013 Galaxy Community Conference (GCC2013)" width="300"></a></div><table><tr><th>    <strong><a href="/src/events/gcc2013/register/index.md">Register by 14 June</a> &amp; avoid late fees</strong>    </th></tr></table><br><p>The <a href="/src/events/gcc2013/index.md">2013 Galaxy Community Conference (GCC2013)</a> will be held 30 June through July 2 in Oslo Norway, at the <a href="http://uio.no" target="_blank" rel="noopener noreferrer">University of Oslo</a>.</p><div class="right"><a href="/src/events/gcc2013/training-day/index.md"><img src="/src/images/logos/GCC2013TrainingDayLogo200.png" alt="2013 Galaxy Community Conference (GCC2013) Training Day" width="150"></a></div><p><a href="/src/events/gcc2013/index.md">GCC2013</a> is an opportunity to participate in two full days of presentations, discussions, poster sessions, keynotes, lightning talks and breakouts, all about high-throughput biology and the tools that support it. The conference also includes a <a href="/src/events/gcc2013/training-day/index.md">Training Day</a> for the second year in a row, this year with more in-depth topic coverage, more concurrent sessions, and more topics.</p><p><strong><a href="/src/events/gcc2013/register/index.md">Register by 14 June</a></strong> and <em>and avoid the 100% fee increase for late registration.</em>  Registration is very affordable, with combined registration (<a href="/src/events/gcc2013/training-day/index.md">Training Day</a> + main meeting) starting at ~ €165 for post-docs and students.  Registering now also assures you a spot in the Training Day workshops you want to attend.  Once a session becomes full, it will be closed to new registrations.  Note that you can still register after 14 June, it will just cost you much more.</p><br><div class="right"><br><p class="markdown" data-v-4735ab8b><div class="alert alert-info float-right text-center">
[**BoFs**](/src/events/gcc2013/bof/index.md)<br />
[GalaxyAdmins](/src/events/gcc2013/bof/galaxy-admins/index.md)<br />
[Proteomics](/src/events/gcc2013/bof/proteomics/index.md)<br />
[Tool Developers](/src/events/gcc2013/bof/tool-developers/index.md)<br />
[Clinical Use / HPC Storage](/src/events/gcc2013/bof/clinical-use-hpc-file-systems/index.md)<br />
[BLAST+ wrappers](/src/events/gcc2013/bof/galaxy-blast/index.md)<br />
[Galaxy Professionals](/src/events/gcc2013/bof/galaxy-professionals/index.md)<br />
[(Birds of Prey)](/src/events/gcc2013/bof/galaxy-professionals/index.md)<br />
[Public Servers](/src/events/gcc2013/bof/public-galaxy-servers/index.md)<br />
</div></p></div><h3 id="birds-of-a-feather-sessions"><a href="#birds-of-a-feather-sessions" aria-hidden="true"><span class="icon icon-link"></span></a>Birds of a Feather Sessions</h3><div class="left"><a href="/src/events/gcc2013/bof/index.md"><img src="/src/images/logos/GCC2013BoFLogo.png" alt="" width="180"></a></div><p>Past Galaxy Community Conferences have been <em>the</em> event for networking in the Galaxy: There is no better place to meet and learn from others doing high-throughput biology.  GCC2013 extends this tradition by including <em><a href="/src/events/gcc2013/bof/index.md">Birds of a Feather (BoF)</a></em> meetups at the event.  Birds of a Feather meetups are informal gatherings where participants group together based on common interests.  If you have something in the list at right you want to meet about, or you have a whole new topic, then please join or start a <a href="/src/events/gcc2013/bof/index.md">GCC2013 BoF</a>.</p><br><div class="right"><br><a href="http://bioteam.net/"><img src="/src/images/logos/BioTeamLogo154.gif" alt="BioTeam" width="140"></a></div><h3 id="sponsorships"><a href="#sponsorships" aria-hidden="true"><span class="icon icon-link"></span></a>Sponsorships</h3><p>We are pleased to announce that <a href="http://bioteam.net" target="_blank" rel="noopener noreferrer">BioTeam</a> is a <a href="/src/events/gcc2013/sponsorships/index.md">Gold Sponsor of GCC2013</a>. </p><p>There are still <a href="/src/events/gcc2013/sponsorships/index.md">Silver and Bronze sponsorships</a> available. Please contact the <a href="/src/events/gcc2013/organizers/index.md">Organizers</a> if your organization would like to help sponsor this event. </p><br><h3 id="software-carpentry-boot-camp"><a href="#software-carpentry-boot-camp" aria-hidden="true"><span class="icon icon-link"></span></a>Software Carpentry Boot Camp</h3><p>Directly following GCC2013, there is a unique possibility to attend a <a href="http://bit.ly/14EmcSA" target="_blank" rel="noopener noreferrer">two-day Software Carpentry Boot Camp at the University of Oslo</a> (in a building close to where the GCC is held). <a href="http://software-carpentry.org/bootcamps/" target="_blank" rel="noopener noreferrer">Software Carpentry Boot Camps</a> aim to to help scientists and engineers become more productive by teaching them basic computing skills like program design, version control, testing, and task automation. In this two-day boot camp, short tutorials will alternate with hands-on practical exercises.</p><p>The course is aimed at postgraduate students and other scientists who are familiar with basic programming concepts (like loops, conditionals, arrays, and functions) but need help to translate this knowledge into practical tools to help them work more productively.</p><p>Content: The syllabus for this boot camp will include:</p><div class="right"><a href="http://bit.ly/14EmcSA"><img src="/src/images/logos/SoftwareCarpentryLogo.png" alt="Software Carpentry Boot Camp, 3-4 July, University of Oslo" width="200"></a></div><ul><li>using the shell to do more in less time</li><li>using version control to manage and share information</li><li>basic Python programming</li><li>how (and how much) to test programs</li></ul><p>Visit the <a href="http://software-carpentry.org/bootcamps/2013-07-oslo.html" target="_blank" rel="noopener noreferrer">Boot Camp Page</a> for more information, and registration.</p><br><h2 id="other-upcoming-events"><a href="#other-upcoming-events" aria-hidden="true"><span class="icon icon-link"></span></a>Other Upcoming Events</h2><div class="center"><a href="https://www.eshg.org/sunday2013.0.html#W1"><img src="/src/images/logos/ESHG2013Narrow.png" alt="Galaxy Workshop @ ESHG" height="80px"></a><a href="/src/events/asms2013/index.md"><img src="/src/images/logos/ASMS2013LogoClipped.png" alt="61st ASMS Conference" height="80px"></a><a href="/src/events/gcc2013/index.md"><img src="/src/images/logos/GCC2013Logo200.png" alt="2013 Galaxy Community Conference (GCC2013)" width="135px"></a><a href="/src/events/rt-workshop-tour2013/index.md"><img src="/src/images/logos/RTGalaxyWorkshopTour2013-300.png" alt="" width="140px"></a></div><div class="right"><a href="http://bit.ly/gxycal"><img src="/src/images/icons/CalendarIcon.gif"></a></div> 
See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on these and other events.
<h3 id="events"><a href="#events" aria-hidden="true"><span class="icon icon-link"></span></a>Events</h3><table><tr class="th"><th> Date </th><th> Topic/Event </th><th> Venue/Location </th><th> Contact </th></tr><tr><th> June 6-7 </th><td><em><a href="http://bit.ly/VyBEcS">Informatics on High Throughput Sequencing Data Workshop</a></em></td><td> Toronto, Ontario, Cananda </td><td><a href="http://bioinformatics.ca/person/cbw-experts/francis-ouellette">Francis Ouellette </a></td></tr><tr><th> June 8-11 </th><td><em><a href="https://www.eshg.org/sunday2013.0.html#W1">WS08: Biomedical Data Analysis with Galaxy Workshop</a></em></td><td><strong><a href="https://www.eshg.org/eshg2013.0.html">European Human Genetics Conference (ESHG2013)</a></strong>, Paris, France </td><td><a href="/src/people/anton/index.md">Anton Nekrutenko</a>, <a href="/src/people/enis-afgan/index.md">Enis Afgan</a></td></tr><tr><th> June 9-13 </th><td><strong><a href="/src/events/asms2013/index.md">61st ASMS Conference on Mass Spectrometry and Allied Topics</a></strong><div class="indent">Includes at least 1 workshop and 9 posters either about or utilizing Galaxy</div></td><td> Minneapolis, Minnesota, United States </td><td><a href="/src/people/john-chilton/index.md">John Chilton</a></td></tr><tr><th> June 13-14 </th><td><a href="http://www.cdb.riken.jp/deepseq2013/index.html">Genomics and Epigenomics with Deep Sequencing, the 24th CDB Meeting</a></td><td> RIKEN, CDB, Kobe, Japan </td><td><a href="/src/people/anton/index.md">Anton Nekrutenko</a></td></tr><tr><th> June 20-22 </th><td><em><a href="http://www.fged.org/conferences/upcoming-conference/workshops-and-tutorials-1/">W1: Next-generation sequencing data analysis with GenomeSpace and Galaxy</a></em></td><td><a href="http://www.fged.org/conferences/upcoming-conference2/">FGED Conference</a>, Seattle, Washington </td><td><a href="/src/people/dave-clements/index.md">Dave Clements</a></td></tr><tr><th> June 21 </th><td><em><a href="http://bit.ly/17RKWHJ">RNA-Seq analysis using Galaxy</a></em><div class="indent">Also <a href="http://bit.ly/ZZe8Kt">offered on 4 July</a> at <a href="http://www.qut.edu.au/about/campuses-and-locations/kelvin-grove">QUT Kelvin Grove Campus</a></div></td><td><a href="http://www.griffith.edu.au/about-griffith/campuses/nathan-campus">Griffith University Nathan Campus</a>, Nathan, Australia </td><td><a href="mailto:m DOT crowe AT qfab DOT org">Mark Crowe</a></td></tr><tr><th> June 30 - July 2 </th><td><strong><em><a href="/src/events/gcc2013/index.md">2013 Galaxy Community Conference (GCC2013)</a></em></strong></td><td> University of Oslo, Oslo, Norway </td><td><a href="/src/events/gcc2013/organizers/index.md">Organizers</a></td></tr><tr><th> July 4 </th><td><em><a href="http://bit.ly/ZZe8Kt">RNA-Seq analysis using Galaxy</a></em><div class="indent">Also <a href="http://bit.ly/17RKWHJ">offered on 21 June</a> at <a href="http://www.griffith.edu.au/about-griffith/campuses/nathan-campus">Griffith University Nathan Campus</a></div></td><td><a href="http://www.qut.edu.au/about/campuses-and-locations/kelvin-grove">QUT Kelvin Grove Campus</a>, Kelvin Grove, Australia </td><td><a href="mailto:m DOT crowe AT qfab DOT org">Mark Crowe</a></td></tr><tr><th rowspan="2"> July 19-23 </th><td><em><a href="http://www.iscb.org/cms_addon/conferences/ismbeccb2013/workshops.php#W05">The “How To Guide” for Establishing a Successful Bioinformatics Network</a></em></td><td rowspan="2"><strong><a href="http://www.iscb.org/ismbeccb2013">ISMB/ECCB 2013</a></strong> &amp; <strong><a href="http://www.open-bio.org/wiki/BOSC_2013">BOSC 2013</a></strong>, Berlin, Germany </td><td rowspan="2"><a href="/src/people/jeremy-goecks/index.md">JeremyGoecks</a></td></tr><tr><td><em><a href="http://www.iscb.org/cms_addon/conferences/ismbeccb2013/workshops.php#W06">Integrated Visualization and Computing for Biomedical Research</a></em></td></tr><tr><th rowspan="4"> July 18-23 </th><td><em><a href="/src/events/niehs2013/index.md">Introduction to Galaxy Workshop</a></em><div class="indent">National Institute of Environmental Health Sciences (NIEHS)</div></td><td rowspan="4"><a href="/src/events/rt-workshop-tour2013/index.md">2013 Research Triangle Workshop Tour</a>, North Carolina, United States </td><td rowspan="4"><a href="/src/people/dave-clements/index.md">Dave Clements</a></td></tr><tr><td><em><a href="/src/events/unc2013/index.md">Introduction to Galaxy Workshop</a></em><div class="indent">University of North Carolina, Chapel Hill</div></td></tr><tr><td><em><a href="http://bit.ly/ZFVLG1">Galaxy Installation Tutorial</a></em><div class="indent"><strong><a href="http://bit.ly/ZFVLG1">2013 GMOD Summer School</a></strong></div></td></tr><tr><td><em><a href="/src/events/ncsu2013/index.md">Introduction to Galaxy Workshop</a></em><div class="indent"> North Carolina State University</div></td></tr><tr><th rowspan="3"> July 21-25 </th><td><em><a href="http://bit.ly/116LWZB">Experiences in building a Next-Generation Sequencing Analysis Service using Galaxy, Globus Online, and Amazon Web Services</a></em></td><td rowspan="3"><strong><a href="https://www.xsede.org/web/xsede13">XSEDE13</a></strong>, San Diego, California, United States </td><td><a href="http://www.mcs.anl.gov/about/people_detail.php?id=347">Ravi Madduri</a></td></tr><tr><td><em><a href="http://bit.ly/Zx4rje">A Sustainable National Gateway for Biological Computation</a></em></td><td> Josephine Palencia </td></tr><tr><td><em><a href="http://bit.ly/124Jm33">Supporting Genomics and other Biological Research</a></em></td><td> Richard LeDuc </td></tr></table><div class="left"><br><a href="http://rnaseq.pathogenportal.org/"><img src="/src/images/screenshots/PathogenPortalGalaxyLandingPage.png" alt="Pathogen Portal Galaxy" height="120"></a></div><h2 id="galaxyadmins"><a href="#galaxyadmins" aria-hidden="true"><span class="icon icon-link"></span></a>GalaxyAdmins</h2><div class="right"><a href="/src/community/galaxy-admins/meetups/2013-05-15/index.md"><img src="/src/images/logos/GalaxyAdmins.png" alt="GalaxyAdmins" height="50"></a></div><p>The <a href="/src/community/galaxy-admins/meetups/2013-05-15/index.md">May GalaxyAdmins Meetup</a> featured Andrew Warren of the <a href="https://www.vbi.vt.edu/" target="_blank" rel="noopener noreferrer">Virginia Bioinformatics Institute</a> at Virginia Tech speaking on their Galaxy deployment <em><a href="http://rnaseq.pathogenportal.org/" target="_blank" rel="noopener noreferrer">RNA Rocket</a></em> for <a href="http://pathogenportal.org" target="_blank" rel="noopener noreferrer">Pathogen Portal</a>. <a href="/src/people/dannon-baker/index.md">Dannon Baker</a>also gave a Galaxy Project update.  Slides and a screencast are now available.</p><br><h2 id="duplicate-accounts-on-main"><a href="#duplicate-accounts-on-main" aria-hidden="true"><span class="icon icon-link"></span></a>Duplicate Accounts on Main</h2><div class="right"><a href="https://main.g2.bx.psu.edu/static/terms.html"><img src="/src/images/logos/UseGalaxyOrgLogoShadow200.png" alt="Main's Terms of Use"></a><br><a href="https://main.g2.bx.psu.edu/static/terms.html"><img src="/src/images/logos/UseGalaxyOrgLogoShadow200.png" alt="Main's Terms of Use"></a></div><p>The <a href="https://main.g2.bx.psu.edu/static/terms.html" target="_blank" rel="noopener noreferrer">terms of use</a> for the Galaxy Project's free Galaxy server, <a href="http://usegalaxy.org" target="_blank" rel="noopener noreferrer">http://usegalaxy.org</a> (a.k.a. <a href="/src/main/index.md">Main</a>) specify </p><div class="indent">
Your access to the Service is provided under the condition that you abide by any published quotas on data storage, job submissions, or any other limitations placed on the public Service. *Attempts to subvert these limits by creating multiple accounts or through any other method may result in termination of all associated accounts.*
</div><br>
In the past we have dealt with multiple accounts on an ad-hoc basis.  In May 2013, an automated system was put into place to detect, disable, and eventually delete duplicate accounts.  If the automated system determines that an account is a duplicate, the account will be disabled and the account owner will receive an email asking if this is the account they wish to keep, and if so, to follow the directions in the email.
<p>If your account is suspended and you believe that you are not in violation of Main's terms of use, then please contact [terms_and_conditions AT galaxyproject DOT org](mailto:terms_and_conditions AT galaxyproject DOT org).</p><h2 id="galaxy-distributions"><a href="#galaxy-distributions" aria-hidden="true"><span class="icon icon-link"></span></a>Galaxy Distributions</h2><p>The most recent official distribution was on <a href="/src/archive/dev-news-briefs/2013-04-01/index.md">April 1, 2013</a>.  However, <a href="/src/news/2013-04-08-galaxy-security-release/index.md">a security update was released on April 8</a>.  See <a href="/src/news/2013-04-08-galaxy-security-release/index.md">this news item</a> for details.  We are planning the next release for early June.</p><h2 id="tool-shed-contributions"><a href="#tool-shed-contributions" aria-hidden="true"><span class="icon icon-link"></span></a>Tool Shed Contributions</h2><div class="right"><a href="http://toolshed.g2.bx.psu.edu/"><img src="/src/images/logos/ToolShed.jpg" alt="Galaxy Tool Shed" width="150"></a></div><ul><li><a href="http://bit.ly/1071hG2" target="_blank" rel="noopener noreferrer">rxlr_venn_workflow</a>: a workflow for comparing 3 RXLR prediction methods with a Venn Diagram </li><li><a href="http://bit.ly/14bpN9c" target="_blank" rel="noopener noreferrer">vcf_to_variantdb</a>: Send UnifiedGenotyper VCF to a !VariantDB instance  </li><li><a href="http://bit.ly/15VS5cs" target="_blank" rel="noopener noreferrer">iquant</a>: isobaric tag quantification program </li><li><a href="http://bit.ly/15VSvzq" target="_blank" rel="noopener noreferrer">appendfdr</a>: compute false discovery rate of tabular data using decoy entries </li><li><a href="http://bit.ly/ZOh5e1" target="_blank" rel="noopener noreferrer">bumbershoot</a>: Tabb lab tools idqonvert, myrimatch, tagrecon, all for protein identification.</li><li><a href="http://bit.ly/135vHq9" target="_blank" rel="noopener noreferrer">dbbuilder</a>: download protein databases from common sources  </li><li><a href="http://bit.ly/17SbTei" target="_blank" rel="noopener noreferrer">nbic_fasta</a>: Galaxy-P unofficial fork of NBIC Galaxy tools for FASTA manipulation</li><li><a href="http://bit.ly/17ScCfB" target="_blank" rel="noopener noreferrer">maxquant</a>: large mass-spec data quantitative proteomics analysis, especially high-resolution mass spectometry </li><li><a href="http://bit.ly/14bsTKv" target="_blank" rel="noopener noreferrer">msconvert</a>: Tool wrappers for msconvert, distributed as part of Proteowizard.</li><li><a href="http://bit.ly/YLoLSY" target="_blank" rel="noopener noreferrer">peptideshaker</a>: Identify proteins combining X! Tandem, OMSSA (using SearchGUI) and PeptideShaker </li><li><a href="http://bit.ly/ZULT0e" target="_blank" rel="noopener noreferrer">psm_eval</a>: Peptide-Spectrum-Match reevaluation tool  </li><li><a href="http://bit.ly/13QaSCN" target="_blank" rel="noopener noreferrer">allele_counts</a>: Parse Naive Variant Detector output, count alleles and calculate minor allele frequencies.</li><li><a href="http://bit.ly/1avPD9D" target="_blank" rel="noopener noreferrer">snp_analysis_conversion</a>: Format conversion for SNP Analysis tools from Miller Lab @ PSU. </li></ul><h2 id="other-news"><a href="#other-news" aria-hidden="true"><span class="icon icon-link"></span></a>Other News</h2><ul><li>Just discovered <a href="http://bit.ly/167H7AP" target="_blank" rel="noopener noreferrer">these tutorials</a> by Lance Parsons on RNA-Seq, ChIP-Seq, SNP &amp; Indels, and IGV all using Galaxy   </li><li><a href="http://j.mp/10Tk2Kq" target="_blank" rel="noopener noreferrer">Scaling variant detection pipelines for whole genome sequencing analysis</a></li><li>There is now a <a href="http://gplus.to/galaxyportal" target="_blank" rel="noopener noreferrer">Google+ Galaxy Group</a>.  Thanks to Samuel Lampa for setting this up.</li></ul></div></article><footer class="static-footer markdown"><p>The Galaxy Project is supported in part by NSF, NHGRI, The Huck Institutes of the Life Sciences, The Institute for CyberScience at Penn State, and Johns Hopkins University.</p>
</footer></section></div>
    <script>window.__INITIAL_STATE__={"data":{"article":{"id":"47c76a5c7705c63c34bb49770fb97007","title":"June 2013 Galaxy Update","tease":"","skip_title_render":null,"image":"","images":{},"contact":"","inserts":[{"name":"\u002Fevents\u002Fgcc2013\u002Fbof\u002Flinkbox","content":"\u003Cdiv class=\"alert alert-info float-right text-center\"\u003E\n[**BoFs**](\u002Fsrc\u002Fevents\u002Fgcc2013\u002Fbof\u002Findex.md)\u003Cbr \u002F\u003E\n[GalaxyAdmins](\u002Fsrc\u002Fevents\u002Fgcc2013\u002Fbof\u002Fgalaxy-admins\u002Findex.md)\u003Cbr \u002F\u003E\n[Proteomics](\u002Fsrc\u002Fevents\u002Fgcc2013\u002Fbof\u002Fproteomics\u002Findex.md)\u003Cbr \u002F\u003E\n[Tool Developers](\u002Fsrc\u002Fevents\u002Fgcc2013\u002Fbof\u002Ftool-developers\u002Findex.md)\u003Cbr \u002F\u003E\n[Clinical Use \u002F HPC Storage](\u002Fsrc\u002Fevents\u002Fgcc2013\u002Fbof\u002Fclinical-use-hpc-file-systems\u002Findex.md)\u003Cbr \u002F\u003E\n[BLAST+ wrappers](\u002Fsrc\u002Fevents\u002Fgcc2013\u002Fbof\u002Fgalaxy-blast\u002Findex.md)\u003Cbr \u002F\u003E\n[Galaxy Professionals](\u002Fsrc\u002Fevents\u002Fgcc2013\u002Fbof\u002Fgalaxy-professionals\u002Findex.md)\u003Cbr \u002F\u003E\n[(Birds of Prey)](\u002Fsrc\u002Fevents\u002Fgcc2013\u002Fbof\u002Fgalaxy-professionals\u002Findex.md)\u003Cbr \u002F\u003E\n[Public Servers](\u002Fsrc\u002Fevents\u002Fgcc2013\u002Fbof\u002Fpublic-galaxy-servers\u002Findex.md)\u003Cbr \u002F\u003E\n\u003C\u002Fdiv\u003E\n"}],"date":null,"content":"\u003Cdiv class='left'\u003E\u003Ca href='\u002Fsrc\u002Fgalaxy-updates\u002Findex.md'\u003E\u003Cimg src=\"\u002Fsrc\u002Fimages\u002Flogos\u002FGalaxyUpdate200.png\" alt=\"Galaxy Updates\" width=150 \u002F\u003E\u003C\u002Fa\u003E\u003C\u002Fdiv\u003E\n\nWelcome to the June 2013 *[Galaxy Update](\u002Fsrc\u002Fgalaxy-updates\u002Findex.md)*, a monthly summary of what is going on in the Galaxy community. *[Galaxy Updates](\u002Fsrc\u002Fgalaxy-updates\u002Findex.md)* complement the *[Galaxy Development News Briefs](\u002Fsrc\u002Fdocs\u002Findex.md)* which accompany new Galaxy releases and focus on Galaxy code updates.\n\n\u003Cbr \u002F\u003E\n\n## New Public Servers\n\n[ODoSE](http:\u002F\u002Fwww.odose.nl\u002F) and [CBiB Galaxy](http:\u002F\u002Fservices.cbib.u-bordeaux2.fr\u002Fgalaxy\u002F) joined [the list of over 30 publicly accessible Galaxy servers](\u002Fsrc\u002Fuse\u002Findex.md) in May.\n\n### ODoSE\n\n\u003Cdiv class='left'\u003E\u003Ca href='http:\u002F\u002Fwww.odose.nl\u002F'\u003E\u003Cimg src=\"\u002Fsrc\u002Fimages\u002Flogos\u002Fodose-scatter-plot.png\" alt=\"ODoSE: Ortholog Direction of Selection Engine\" width=\"200\" \u002F\u003E\u003C\u002Fa\u003E\u003C\u002Fdiv\u003E\n\nThe [Ortholog Direction of Selection Engine (ODoSE)](http:\u002F\u002Fwww.odose.nl\u002F) supports genome-wide calculation of adaptive divergence in prokaryotes:\n\n\"The web-based graphical user-interface ODoSE (Ortholog Direction of Selection Engine) identifies all orthologs between a set of strains and allows the calculation of a novel extension of the [McDonald Kreitman] MK test, the Direction of Selection (DoS) statistic as well as the calculation of the Neutrality Index (NI) statistic for all genes shared by two taxonomic groups combined. The engine also generates the site frequency spectrum for each gene allowing one to apply more advanced methods for estimating the distribution of fitness effects and rates of adaptive evolution.\"\n\nSee [ODoSE: A Webserver for Genome-Wide Calculation of Adaptive Divergence in Prokaryotes](http:\u002F\u002Fbit.ly\u002F10ydSiO) by Michiel Vos, Tim A. H. te Beek, Marc A. van Driel, Martijn A. Huynen, Adam Eyre-Walker, Mark W. J. van Passel; PLoS ONE, Vol. 8, No. 5. (6 May 2013), e62447, doi:10.1371\u002Fjournal.pone.0062447\n\n\u003Cdiv class='right'\u003E\u003Cbr \u002F\u003E[[\u003Ca href='http:\u002F\u002Fservices.cbib.u-bordeaux2.fr\u002Fgalaxy\u002F'\u003E\u003Cimg src=\"\u002Fsrc\u002Fimages\u002Flogos\u002Fcbib-logo-300.png\" alt=\"CBiB Galaxy\" width=\"200\" \u002F\u003E\u003C\u002Fa\u003E\u003C\u002Fdiv\u003E\n\n### CBiB Galaxy\n\n[CBiB Galaxy](http:\u002F\u002Fservices.cbib.u-bordeaux2.fr\u002Fgalaxy\u002F) is a general purpose Galaxy instance  that includes EMBOSS (a software analysis package for molecular biology) and fibronectin (diversity analysis of synthetic libraries of a Fibronectin domain).  For now, the computational resources of the Galaxy instance of the CBiB are relatively limited. However, a migration to a dedicated server will be made ​​soon.  \n\n[CBiB Galaxy](http:\u002F\u002Fservices.cbib.u-bordeaux2.fr\u002Fgalaxy\u002F) is sponsored by [Centre de Bioinformatique de Bordeaux](http:\u002F\u002Fwww.cbib.u-bordeaux2.fr\u002F).\n\n\u003Cdiv class='left'\u003E\n\u003Ctable\u003E\n  \u003Ctr\u003E\n    \u003Cth\u003E # \u003C\u002Fth\u003E\n    \u003Cth\u003E Tag \u003C\u002Fth\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Ctd style=\" text-align: right;\"\u003E 24 \u003C\u002Ftd\u003E\n    \u003Ctd\u003E \u003Ca href='http:\u002F\u002Fwww.citeulike.org\u002Fgroup\u002F16008\u002Ftag\u002Fmethods'\u003Emethods\u003C\u002Fa\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Ctd style=\" text-align: right;\"\u003E 10 \u003C\u002Ftd\u003E\n    \u003Ctd\u003E \u003Ca href='http:\u002F\u002Fwww.citeulike.org\u002Fgroup\u002F16008\u002Ftag\u002Fworkbench'\u003Eworkbench\u003C\u002Fa\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Ctd style=\" text-align: right;\"\u003E 9 \u003C\u002Ftd\u003E\n    \u003Ctd\u003E \u003Ca href='http:\u002F\u002Fwww.citeulike.org\u002Fgroup\u002F16008\u002Ftag\u002Fusemain'\u003Eusemain\u003C\u002Fa\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Ctd style=\" text-align: right;\"\u003E 2 \u003C\u002Ftd\u003E\n    \u003Ctd\u003E \u003Ca href='http:\u002F\u002Fwww.citeulike.org\u002Fgroup\u002F16008\u002Ftag\u002Fisgalaxy'\u003Eisgalaxy\u003C\u002Fa\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Ctd style=\" text-align: right;\"\u003E 2 \u003C\u002Ftd\u003E\n    \u003Ctd\u003E \u003Ca href='http:\u002F\u002Fwww.citeulike.org\u002Fgroup\u002F16008\u002Ftag\u002Fuselocal'\u003Euselocal\u003C\u002Fa\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Ctd style=\" text-align: right;\"\u003E 2 \u003C\u002Ftd\u003E\n    \u003Ctd\u003E \u003Ca href='http:\u002F\u002Fwww.citeulike.org\u002Fgroup\u002F16008\u002Ftag\u002Funknown'\u003Eunknown\u003C\u002Fa\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Ctd style=\" text-align: right;\"\u003E 1 \u003C\u002Ftd\u003E\n    \u003Ctd\u003E \u003Ca href='http:\u002F\u002Fwww.citeulike.org\u002Fgroup\u002F16008\u002Ftag\u002Fcloud'\u003Ecloud\u003C\u002Fa\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Ctd style=\" text-align: right;\"\u003E 1 \u003C\u002Ftd\u003E\n    \u003Ctd\u003E \u003Ca href='http:\u002F\u002Fwww.citeulike.org\u002Fgroup\u002F16008\u002Ftag\u002Fshared'\u003Eshared\u003C\u002Fa\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n\u003C\u002Ftable\u003E\n\n\u003C\u002Fdiv\u003E\n\n## New Papers\n\n39 new papers were added to the Galaxy CiteULike Group in May, including:\n\n* \"[The Genomic HyperBrowser: an analysis web server for genome-scale data](https:\u002F\u002Fdoi.org\u002F10.1093\u002Fnar\u002Fgkt342)\" by Geir K. Sandve, Sveinung Gundersen, Morten Johansen, *et al.*; *[Nucleic Acids Research](http:\u002F\u002Fnar.oxfordjournals.org\u002F)* (30 April 2013)\n\n* \"[ODoSE: A Webserver for Genome-Wide Calculation of Adaptive Divergence in Prokaryotes](https:\u002F\u002Fdoi.org\u002F10.1371\u002Fjournal.pone.0062447)\" by Michiel Vos, Tim A. H. te Beek, Marc A. van Driel, Martijn A. Huynen, Adam Eyre-Walker, Mark W. J. van Passel; *[PLoS ONE](http:\u002F\u002Fwww.plosone.org\u002F)*, Vol. 8, No. 5. (6 May 2013), e62447\n\nThere are now almost [1000 papers in the Galaxy CiteULike Group](http:\u002F\u002Fwww.citeulike.org\u002Fgroup\u002F16008\u002F).\n\n\u003Cbr \u002F\u003E\n\n## Who's Hiring\n\n\u003Cdiv class='right'\u003E\u003Ca href='\u002Fsrc\u002Fgalaxy-is-hiring\u002Findex.md'\u003E\u003Cimg src=\"\u002Fsrc\u002Fimages\u002Fnews-graphics\u002FGalaxyIsHiringWordCloud2.png\" alt=\"Please Help! Yes you!\" width=\"200\" \u002F\u003E\u003C\u002Fa\u003E\u003C\u002Fdiv\u003E\n\nThe Galaxy is expanding! Please help it grow.\n\n* **The [Galaxy Project is hiring software engineers and post-docs](\u002Fsrc\u002Fgalaxy-is-hiring\u002Findex.md)!** at both Emory and Penn State.\n* [Senior Developer](http:\u002F\u002Fwww.nature.com\u002Fnaturejobs\u002Fscience\u002Fjobs\u002F321283-Senior-Developer-Stem-Cell-Bioinformatics-Core), Stem Cell Bioinformatics Core, Sage Bionetworks, Seattle, WA, United States\n* [Bioinformatics Support Group Leader](http:\u002F\u002Fbit.ly\u002FVkRtrb) @ LSU\n\nGot a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](\u002Fsrc\u002Fnews\u002Findex.md) and include it in next month's [update](\u002Fsrc\u002Fgalaxy-updates\u002Findex.md).\n\n\n## GCC2013\n\n\u003Cdiv class='center'\u003E\u003Ca href='\u002Fsrc\u002Fevents\u002Fgcc2013\u002Findex.md'\u003E\u003Cimg src=\"\u002Fsrc\u002Fimages\u002Flogos\u002FGCC2013Logo400.png\" alt=\"2013 Galaxy Community Conference (GCC2013)\" width=\"300\" \u002F\u003E\u003C\u002Fa\u003E \u003C\u002Fdiv\u003E\n\n\u003Ctable\u003E\n  \u003Ctr\u003E\n    \u003Cth\u003E &nbsp;&nbsp; \u003Cstrong\u003E\u003Ca href='\u002Fsrc\u002Fevents\u002Fgcc2013\u002Fregister\u002Findex.md'\u003ERegister by 14 June\u003C\u002Fa\u003E & avoid late fees\u003C\u002Fstrong\u003E &nbsp;&nbsp; \u003C\u002Fth\u003E\n  \u003C\u002Ftr\u003E\n\u003C\u002Ftable\u003E\n\n\n\u003Cbr \u002F\u003E\n\nThe [2013 Galaxy Community Conference (GCC2013)](\u002Fsrc\u002Fevents\u002Fgcc2013\u002Findex.md) will be held 30 June through July 2 in Oslo Norway, at the [University of Oslo](http:\u002F\u002Fuio.no).\n\n\u003Cdiv class='right'\u003E\u003Ca href='\u002Fsrc\u002Fevents\u002Fgcc2013\u002Ftraining-day\u002Findex.md'\u003E\u003Cimg src=\"\u002Fsrc\u002Fimages\u002Flogos\u002FGCC2013TrainingDayLogo200.png\" alt=\"2013 Galaxy Community Conference (GCC2013) Training Day\" width=\"150\" \u002F\u003E\u003C\u002Fa\u003E\u003C\u002Fdiv\u003E\n\n[GCC2013](\u002Fsrc\u002Fevents\u002Fgcc2013\u002Findex.md) is an opportunity to participate in two full days of presentations, discussions, poster sessions, keynotes, lightning talks and breakouts, all about high-throughput biology and the tools that support it. The conference also includes a [Training Day](\u002Fsrc\u002Fevents\u002Fgcc2013\u002Ftraining-day\u002Findex.md) for the second year in a row, this year with more in-depth topic coverage, more concurrent sessions, and more topics.\n\n**[Register by 14 June](\u002Fsrc\u002Fevents\u002Fgcc2013\u002Fregister\u002Findex.md)** and *and avoid the 100% fee increase for late registration.*  Registration is very affordable, with combined registration ([Training Day](\u002Fsrc\u002Fevents\u002Fgcc2013\u002Ftraining-day\u002Findex.md) + main meeting) starting at ~ €165 for post-docs and students.  Registering now also assures you a spot in the Training Day workshops you want to attend.  Once a session becomes full, it will be closed to new registrations.  Note that you can still register after 14 June, it will just cost you much more.\n\n\u003Cbr \u002F\u003E\n\n\u003Cdiv class='right'\u003E\u003Cbr \u002F\u003E\n\n\u003Cslot name=\"\u002Fevents\u002Fgcc2013\u002Fbof\u002Flinkbox\" \u002F\u003E\n\n\u003C\u002Fdiv\u003E\n\n### Birds of a Feather Sessions\n\n\u003Cdiv class='left'\u003E\u003Ca href='\u002Fsrc\u002Fevents\u002Fgcc2013\u002Fbof\u002Findex.md'\u003E\u003Cimg src=\"\u002Fsrc\u002Fimages\u002Flogos\u002FGCC2013BoFLogo.png\" alt=\"\" width=\"180\" \u002F\u003E\u003C\u002Fa\u003E\u003C\u002Fdiv\u003E\n\nPast Galaxy Community Conferences have been *the* event for networking in the Galaxy: There is no better place to meet and learn from others doing high-throughput biology.  GCC2013 extends this tradition by including *[Birds of a Feather (BoF)](\u002Fsrc\u002Fevents\u002Fgcc2013\u002Fbof\u002Findex.md)* meetups at the event.  Birds of a Feather meetups are informal gatherings where participants group together based on common interests.  If you have something in the list at right you want to meet about, or you have a whole new topic, then please join or start a [GCC2013 BoF](\u002Fsrc\u002Fevents\u002Fgcc2013\u002Fbof\u002Findex.md).\n\n\u003Cbr \u002F\u003E\n\n\u003Cdiv class='right'\u003E\u003Cbr \u002F\u003E\u003Ca href='http:\u002F\u002Fbioteam.net\u002F'\u003E\u003Cimg src=\"\u002Fsrc\u002Fimages\u002Flogos\u002FBioTeamLogo154.gif\" alt=\"BioTeam\" width=\"140\" \u002F\u003E\u003C\u002Fa\u003E\u003C\u002Fdiv\u003E\n\n### Sponsorships\n\nWe are pleased to announce that [BioTeam](http:\u002F\u002Fbioteam.net) is a [Gold Sponsor of GCC2013](\u002Fsrc\u002Fevents\u002Fgcc2013\u002Fsponsorships\u002Findex.md). \n\nThere are still [Silver and Bronze sponsorships](\u002Fsrc\u002Fevents\u002Fgcc2013\u002Fsponsorships\u002Findex.md) available. Please contact the [Organizers](\u002Fsrc\u002Fevents\u002Fgcc2013\u002Forganizers\u002Findex.md) if your organization would like to help sponsor this event. \n\n\u003Cbr \u002F\u003E\n\n### Software Carpentry Boot Camp\n\nDirectly following GCC2013, there is a unique possibility to attend a [two-day Software Carpentry Boot Camp at the University of Oslo](http:\u002F\u002Fbit.ly\u002F14EmcSA) (in a building close to where the GCC is held). [Software Carpentry Boot Camps](http:\u002F\u002Fsoftware-carpentry.org\u002Fbootcamps\u002F) aim to to help scientists and engineers become more productive by teaching them basic computing skills like program design, version control, testing, and task automation. In this two-day boot camp, short tutorials will alternate with hands-on practical exercises.\n\nThe course is aimed at postgraduate students and other scientists who are familiar with basic programming concepts (like loops, conditionals, arrays, and functions) but need help to translate this knowledge into practical tools to help them work more productively.\n\nContent: The syllabus for this boot camp will include:\n\n\u003Cdiv class='right'\u003E\u003Ca href='http:\u002F\u002Fbit.ly\u002F14EmcSA'\u003E\u003Cimg src=\"\u002Fsrc\u002Fimages\u002Flogos\u002FSoftwareCarpentryLogo.png\" alt=\"Software Carpentry Boot Camp, 3-4 July, University of Oslo\" width=\"200\" \u002F\u003E\u003C\u002Fa\u003E\u003C\u002Fdiv\u003E\n\n* using the shell to do more in less time\n* using version control to manage and share information\n* basic Python programming\n* how (and how much) to test programs\n\nVisit the [Boot Camp Page](http:\u002F\u002Fsoftware-carpentry.org\u002Fbootcamps\u002F2013-07-oslo.html) for more information, and registration.\n\n\u003Cbr \u002F\u003E\n\n## Other Upcoming Events\n\n\u003Cdiv class='center'\u003E \n\u003Ca href='https:\u002F\u002Fwww.eshg.org\u002Fsunday2013.0.html#W1'\u003E\u003Cimg src=\"\u002Fsrc\u002Fimages\u002Flogos\u002FESHG2013Narrow.png\" alt=\"Galaxy Workshop @ ESHG\" height=\"80px\" \u002F\u003E\u003C\u002Fa\u003E\n\u003Ca href='\u002Fsrc\u002Fevents\u002Fasms2013\u002Findex.md'\u003E\u003Cimg src=\"\u002Fsrc\u002Fimages\u002Flogos\u002FASMS2013LogoClipped.png\" alt=\"61st ASMS Conference\" height=\"80px\" \u002F\u003E\u003C\u002Fa\u003E\n\u003Ca href='\u002Fsrc\u002Fevents\u002Fgcc2013\u002Findex.md'\u003E\u003Cimg src=\"\u002Fsrc\u002Fimages\u002Flogos\u002FGCC2013Logo200.png\" alt=\"2013 Galaxy Community Conference (GCC2013)\" width=\"135px\" \u002F\u003E\u003C\u002Fa\u003E\n\u003Ca href='\u002Fsrc\u002Fevents\u002Frt-workshop-tour2013\u002Findex.md'\u003E\u003Cimg src=\"\u002Fsrc\u002Fimages\u002Flogos\u002FRTGalaxyWorkshopTour2013-300.png\" alt=\"\" width=\"140px\" \u002F\u003E\u003C\u002Fa\u003E\n\u003C\u002Fdiv\u003E\n\n\u003Cdiv class='right'\u003E\u003Ca href='http:\u002F\u002Fbit.ly\u002Fgxycal'\u003E\u003Cimg src=\"\u002Fsrc\u002Fimages\u002Ficons\u002FCalendarIcon.gif\" \u002F\u003E\u003C\u002Fa\u003E\u003C\u002Fdiv\u003E \nSee the [Galaxy Events Google Calendar](http:\u002F\u002Fbit.ly\u002Fgxycal) for details on these and other events.\n\n### Events\n\n\u003Ctable\u003E\n  \u003Ctr class=\"th\" \u003E\n    \u003Cth\u003E Date \u003C\u002Fth\u003E\n    \u003Cth\u003E Topic\u002FEvent \u003C\u002Fth\u003E\n    \u003Cth\u003E Venue\u002FLocation \u003C\u002Fth\u003E\n    \u003Cth\u003E Contact \u003C\u002Fth\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Cth\u003E June 6-7 \u003C\u002Fth\u003E\n    \u003Ctd\u003E \u003Cem\u003E\u003Ca href='http:\u002F\u002Fbit.ly\u002FVyBEcS'\u003EInformatics on High Throughput Sequencing Data Workshop\u003C\u002Fa\u003E\u003C\u002Fem\u003E \u003C\u002Ftd\u003E\n    \u003Ctd\u003E Toronto, Ontario, Cananda \u003C\u002Ftd\u003E\n    \u003Ctd\u003E \u003Ca href='http:\u002F\u002Fbioinformatics.ca\u002Fperson\u002Fcbw-experts\u002Ffrancis-ouellette'\u003EFrancis Ouellette \u003C\u002Fa\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Cth\u003E June 8-11 \u003C\u002Fth\u003E\n    \u003Ctd\u003E \u003Cem\u003E\u003Ca href='https:\u002F\u002Fwww.eshg.org\u002Fsunday2013.0.html#W1'\u003EWS08: Biomedical Data Analysis with Galaxy Workshop\u003C\u002Fa\u003E\u003C\u002Fem\u003E \u003C\u002Ftd\u003E\n    \u003Ctd\u003E \u003Cstrong\u003E\u003Ca href='https:\u002F\u002Fwww.eshg.org\u002Feshg2013.0.html'\u003EEuropean Human Genetics Conference (ESHG2013)\u003C\u002Fa\u003E\u003C\u002Fstrong\u003E, Paris, France \u003C\u002Ftd\u003E\n    \u003Ctd\u003E \u003Ca href='\u002Fsrc\u002Fpeople\u002Fanton\u002Findex.md'\u003EAnton Nekrutenko\u003C\u002Fa\u003E, \u003Ca href='\u002Fsrc\u002Fpeople\u002Fenis-afgan\u002Findex.md'\u003EEnis Afgan\u003C\u002Fa\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Cth\u003E June 9-13 \u003C\u002Fth\u003E\n    \u003Ctd\u003E \u003Cstrong\u003E\u003Ca href='\u002Fsrc\u002Fevents\u002Fasms2013\u002Findex.md'\u003E61st ASMS Conference on Mass Spectrometry and Allied Topics\u003C\u002Fa\u003E\u003C\u002Fstrong\u003E\u003Cdiv class='indent'\u003EIncludes at least 1 workshop and 9 posters either about or utilizing Galaxy\u003C\u002Fdiv\u003E \u003C\u002Ftd\u003E\n    \u003Ctd\u003E Minneapolis, Minnesota, United States \u003C\u002Ftd\u003E\n    \u003Ctd\u003E \u003Ca href='\u002Fsrc\u002Fpeople\u002Fjohn-chilton\u002Findex.md'\u003EJohn Chilton\u003C\u002Fa\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Cth\u003E June 13-14 \u003C\u002Fth\u003E\n    \u003Ctd\u003E \u003Ca href='http:\u002F\u002Fwww.cdb.riken.jp\u002Fdeepseq2013\u002Findex.html'\u003EGenomics and Epigenomics with Deep Sequencing, the 24th CDB Meeting\u003C\u002Fa\u003E \u003C\u002Ftd\u003E\n    \u003Ctd\u003E RIKEN, CDB, Kobe, Japan \u003C\u002Ftd\u003E\n    \u003Ctd\u003E \u003Ca href='\u002Fsrc\u002Fpeople\u002Fanton\u002Findex.md'\u003EAnton Nekrutenko\u003C\u002Fa\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Cth\u003E June 20-22 \u003C\u002Fth\u003E\n    \u003Ctd\u003E \u003Cem\u003E\u003Ca href='http:\u002F\u002Fwww.fged.org\u002Fconferences\u002Fupcoming-conference\u002Fworkshops-and-tutorials-1\u002F'\u003EW1: Next-generation sequencing data analysis with GenomeSpace and Galaxy\u003C\u002Fa\u003E\u003C\u002Fem\u003E \u003C\u002Ftd\u003E\n    \u003Ctd\u003E \u003Ca href='http:\u002F\u002Fwww.fged.org\u002Fconferences\u002Fupcoming-conference2\u002F'\u003EFGED Conference\u003C\u002Fa\u003E, Seattle, Washington \u003C\u002Ftd\u003E\n    \u003Ctd\u003E \u003Ca href='\u002Fsrc\u002Fpeople\u002Fdave-clements\u002Findex.md'\u003EDave Clements\u003C\u002Fa\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Cth\u003E June 21 \u003C\u002Fth\u003E\n    \u003Ctd\u003E \u003Cem\u003E\u003Ca href='http:\u002F\u002Fbit.ly\u002F17RKWHJ'\u003ERNA-Seq analysis using Galaxy\u003C\u002Fa\u003E\u003C\u002Fem\u003E\u003Cdiv class='indent'\u003EAlso \u003Ca href='http:\u002F\u002Fbit.ly\u002FZZe8Kt'\u003Eoffered on 4 July\u003C\u002Fa\u003E at \u003Ca href='http:\u002F\u002Fwww.qut.edu.au\u002Fabout\u002Fcampuses-and-locations\u002Fkelvin-grove'\u003EQUT Kelvin Grove Campus\u003C\u002Fa\u003E \u003C\u002Fdiv\u003E \u003C\u002Ftd\u003E\n    \u003Ctd\u003E \u003Ca href='http:\u002F\u002Fwww.griffith.edu.au\u002Fabout-griffith\u002Fcampuses\u002Fnathan-campus'\u003EGriffith University Nathan Campus\u003C\u002Fa\u003E, Nathan, Australia \u003C\u002Ftd\u003E\n    \u003Ctd\u003E \u003Ca href=\"mailto:m DOT crowe AT qfab DOT org\"\u003EMark Crowe\u003C\u002Fa\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Cth\u003E June 30 - July&nbsp;2 \u003C\u002Fth\u003E\n    \u003Ctd\u003E \u003Cstrong\u003E\u003Cem\u003E\u003Ca href='\u002Fsrc\u002Fevents\u002Fgcc2013\u002Findex.md'\u003E2013 Galaxy Community Conference (GCC2013)\u003C\u002Fa\u003E\u003C\u002Fem\u003E\u003C\u002Fstrong\u003E \u003C\u002Ftd\u003E\n    \u003Ctd\u003E University of Oslo, Oslo, Norway \u003C\u002Ftd\u003E\n    \u003Ctd\u003E \u003Ca href='\u002Fsrc\u002Fevents\u002Fgcc2013\u002Forganizers\u002Findex.md'\u003EOrganizers\u003C\u002Fa\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Cth\u003E July 4 \u003C\u002Fth\u003E\n    \u003Ctd\u003E \u003Cem\u003E\u003Ca href='http:\u002F\u002Fbit.ly\u002FZZe8Kt'\u003ERNA-Seq analysis using Galaxy\u003C\u002Fa\u003E\u003C\u002Fem\u003E\u003Cdiv class='indent'\u003EAlso \u003Ca href='http:\u002F\u002Fbit.ly\u002F17RKWHJ'\u003Eoffered on 21 June\u003C\u002Fa\u003E at \u003Ca href='http:\u002F\u002Fwww.griffith.edu.au\u002Fabout-griffith\u002Fcampuses\u002Fnathan-campus'\u003EGriffith University Nathan Campus\u003C\u002Fa\u003E\u003C\u002Fdiv\u003E \u003C\u002Ftd\u003E\n    \u003Ctd\u003E \u003Ca href='http:\u002F\u002Fwww.qut.edu.au\u002Fabout\u002Fcampuses-and-locations\u002Fkelvin-grove'\u003EQUT Kelvin Grove Campus\u003C\u002Fa\u003E, Kelvin Grove, Australia \u003C\u002Ftd\u003E\n    \u003Ctd\u003E \u003Ca href=\"mailto:m DOT crowe AT qfab DOT org\"\u003EMark Crowe\u003C\u002Fa\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Cth rowspan=2\u003E July 19-23 \u003C\u002Fth\u003E\n    \u003Ctd\u003E \u003Cem\u003E\u003Ca href='http:\u002F\u002Fwww.iscb.org\u002Fcms_addon\u002Fconferences\u002Fismbeccb2013\u002Fworkshops.php#W05'\u003EThe “How To Guide” for Establishing a Successful Bioinformatics Network\u003C\u002Fa\u003E\u003C\u002Fem\u003E \u003C\u002Ftd\u003E\n    \u003Ctd rowspan=2\u003E \u003Cstrong\u003E\u003Ca href='http:\u002F\u002Fwww.iscb.org\u002Fismbeccb2013'\u003EISMB\u002FECCB 2013\u003C\u002Fa\u003E\u003C\u002Fstrong\u003E & \u003Cstrong\u003E\u003Ca href='http:\u002F\u002Fwww.open-bio.org\u002Fwiki\u002FBOSC_2013'\u003EBOSC 2013\u003C\u002Fa\u003E\u003C\u002Fstrong\u003E, Berlin, Germany \u003C\u002Ftd\u003E\n    \u003Ctd rowspan=2\u003E \u003Ca href='\u002Fsrc\u002Fpeople\u002Fjeremy-goecks\u002Findex.md'\u003EJeremyGoecks\u003C\u002Fa\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Ctd\u003E \u003Cem\u003E\u003Ca href='http:\u002F\u002Fwww.iscb.org\u002Fcms_addon\u002Fconferences\u002Fismbeccb2013\u002Fworkshops.php#W06'\u003EIntegrated Visualization and Computing for Biomedical Research\u003C\u002Fa\u003E\u003C\u002Fem\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Cth rowspan=4\u003E July 18-23 \u003C\u002Fth\u003E\n    \u003Ctd\u003E \u003Cem\u003E\u003Ca href='\u002Fsrc\u002Fevents\u002Fniehs2013\u002Findex.md'\u003EIntroduction to Galaxy Workshop\u003C\u002Fa\u003E\u003C\u002Fem\u003E\u003Cdiv class='indent'\u003ENational Institute of Environmental Health Sciences (NIEHS)\u003C\u002Fdiv\u003E \u003C\u002Ftd\u003E\n    \u003Ctd rowspan=4\u003E \u003Ca href='\u002Fsrc\u002Fevents\u002Frt-workshop-tour2013\u002Findex.md'\u003E2013 Research Triangle Workshop Tour\u003C\u002Fa\u003E, North Carolina, United States \u003C\u002Ftd\u003E\n    \u003Ctd rowspan=4\u003E \u003Ca href='\u002Fsrc\u002Fpeople\u002Fdave-clements\u002Findex.md'\u003EDave Clements\u003C\u002Fa\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Ctd\u003E \u003Cem\u003E\u003Ca href='\u002Fsrc\u002Fevents\u002Func2013\u002Findex.md'\u003EIntroduction to Galaxy Workshop\u003C\u002Fa\u003E\u003C\u002Fem\u003E \u003Cdiv class='indent'\u003EUniversity of North Carolina, Chapel Hill\u003C\u002Fdiv\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Ctd\u003E \u003Cem\u003E\u003Ca href='http:\u002F\u002Fbit.ly\u002FZFVLG1'\u003EGalaxy Installation Tutorial\u003C\u002Fa\u003E\u003C\u002Fem\u003E \u003Cdiv class='indent'\u003E\u003Cstrong\u003E\u003Ca href='http:\u002F\u002Fbit.ly\u002FZFVLG1'\u003E2013 GMOD Summer School\u003C\u002Fa\u003E\u003C\u002Fstrong\u003E\u003C\u002Fdiv\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Ctd\u003E \u003Cem\u003E\u003Ca href='\u002Fsrc\u002Fevents\u002Fncsu2013\u002Findex.md'\u003EIntroduction to Galaxy Workshop\u003C\u002Fa\u003E\u003C\u002Fem\u003E \u003Cdiv class='indent'\u003E North Carolina State University\u003C\u002Fdiv\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Cth rowspan=3\u003E July 21-25 \u003C\u002Fth\u003E\n    \u003Ctd\u003E \u003Cem\u003E\u003Ca href='http:\u002F\u002Fbit.ly\u002F116LWZB'\u003EExperiences in building a Next-Generation Sequencing Analysis Service using Galaxy, Globus Online, and Amazon Web Services\u003C\u002Fa\u003E\u003C\u002Fem\u003E \u003C\u002Ftd\u003E\n    \u003Ctd rowspan=3\u003E \u003Cstrong\u003E\u003Ca href='https:\u002F\u002Fwww.xsede.org\u002Fweb\u002Fxsede13'\u003EXSEDE13\u003C\u002Fa\u003E\u003C\u002Fstrong\u003E, San Diego, California, United States \u003C\u002Ftd\u003E\n    \u003Ctd\u003E \u003Ca href='http:\u002F\u002Fwww.mcs.anl.gov\u002Fabout\u002Fpeople_detail.php?id=347'\u003ERavi Madduri\u003C\u002Fa\u003E \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Ctd\u003E \u003Cem\u003E\u003Ca href='http:\u002F\u002Fbit.ly\u002FZx4rje'\u003EA Sustainable National Gateway for Biological Computation\u003C\u002Fa\u003E\u003C\u002Fem\u003E \u003C\u002Ftd\u003E\n    \u003Ctd\u003E Josephine Palencia \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n  \u003Ctr\u003E\n    \u003Ctd\u003E \u003Cem\u003E\u003Ca href='http:\u002F\u002Fbit.ly\u002F124Jm33'\u003ESupporting Genomics and other Biological Research\u003C\u002Fa\u003E\u003C\u002Fem\u003E \u003C\u002Ftd\u003E\n    \u003Ctd\u003E Richard LeDuc \u003C\u002Ftd\u003E\n  \u003C\u002Ftr\u003E\n\u003C\u002Ftable\u003E\n\n\n\u003Cdiv class='left'\u003E\u003Cbr \u002F\u003E\u003Ca href='http:\u002F\u002Frnaseq.pathogenportal.org\u002F'\u003E\u003Cimg src=\"\u002Fsrc\u002Fimages\u002Fscreenshots\u002FPathogenPortalGalaxyLandingPage.png\" alt=\"Pathogen Portal Galaxy\" height=\"120\" \u002F\u003E\u003C\u002Fa\u003E\u003C\u002Fdiv\u003E\n\n## GalaxyAdmins\n\n\u003Cdiv class='right'\u003E\u003Ca href='\u002Fsrc\u002Fcommunity\u002Fgalaxy-admins\u002Fmeetups\u002F2013-05-15\u002Findex.md'\u003E\u003Cimg src=\"\u002Fsrc\u002Fimages\u002Flogos\u002FGalaxyAdmins.png\" alt=\"GalaxyAdmins\" height=\"50\" \u002F\u003E\u003C\u002Fa\u003E\u003C\u002Fdiv\u003E\n\nThe [May GalaxyAdmins Meetup](\u002Fsrc\u002Fcommunity\u002Fgalaxy-admins\u002Fmeetups\u002F2013-05-15\u002Findex.md) featured Andrew Warren of the [Virginia Bioinformatics Institute](https:\u002F\u002Fwww.vbi.vt.edu\u002F) at Virginia Tech speaking on their Galaxy deployment *[RNA Rocket](http:\u002F\u002Frnaseq.pathogenportal.org\u002F)* for [Pathogen Portal](http:\u002F\u002Fpathogenportal.org). [Dannon Baker](\u002Fsrc\u002Fpeople\u002Fdannon-baker\u002Findex.md)also gave a Galaxy Project update.  Slides and a screencast are now available.\n\n\u003Cbr \u002F\u003E\n\n## Duplicate Accounts on Main\n\n\u003Cdiv class='right'\u003E\u003Ca href='https:\u002F\u002Fmain.g2.bx.psu.edu\u002Fstatic\u002Fterms.html'\u003E\u003Cimg src=\"\u002Fsrc\u002Fimages\u002Flogos\u002FUseGalaxyOrgLogoShadow200.png\" alt=\"Main's Terms of Use\"  \u002F\u003E\u003C\u002Fa\u003E\u003Cbr \u002F\u003E\u003Ca href='https:\u002F\u002Fmain.g2.bx.psu.edu\u002Fstatic\u002Fterms.html'\u003E\u003Cimg src=\"\u002Fsrc\u002Fimages\u002Flogos\u002FUseGalaxyOrgLogoShadow200.png\" alt=\"Main's Terms of Use\"  \u002F\u003E\u003C\u002Fa\u003E\u003C\u002Fdiv\u003E\n\nThe [terms of use](https:\u002F\u002Fmain.g2.bx.psu.edu\u002Fstatic\u002Fterms.html) for the Galaxy Project's free Galaxy server, [http:\u002F\u002Fusegalaxy.org](http:\u002F\u002Fusegalaxy.org) (a.k.a. [Main](\u002Fsrc\u002Fmain\u002Findex.md)) specify \n\n\u003Cdiv class='indent'\u003E\nYour access to the Service is provided under the condition that you abide by any published quotas on data storage, job submissions, or any other limitations placed on the public Service. *Attempts to subvert these limits by creating multiple accounts or through any other method may result in termination of all associated accounts.*\n\u003C\u002Fdiv\u003E\n\n\u003Cbr \u002F\u003E\nIn the past we have dealt with multiple accounts on an ad-hoc basis.  In May 2013, an automated system was put into place to detect, disable, and eventually delete duplicate accounts.  If the automated system determines that an account is a duplicate, the account will be disabled and the account owner will receive an email asking if this is the account they wish to keep, and if so, to follow the directions in the email.\n\nIf your account is suspended and you believe that you are not in violation of Main's terms of use, then please contact [terms_and_conditions AT galaxyproject DOT org](mailto:terms_and_conditions AT galaxyproject DOT org).\n\n## Galaxy Distributions\n\nThe most recent official distribution was on [April 1, 2013](\u002Fsrc\u002Farchive\u002Fdev-news-briefs\u002F2013-04-01\u002Findex.md).  However, [a security update was released on April 8](\u002Fsrc\u002Fnews\u002F2013-04-08-galaxy-security-release\u002Findex.md).  See [this news item](\u002Fsrc\u002Fnews\u002F2013-04-08-galaxy-security-release\u002Findex.md) for details.  We are planning the next release for early June.\n\n\n## Tool Shed Contributions\n\n\u003Cdiv class='right'\u003E\u003Ca href='http:\u002F\u002Ftoolshed.g2.bx.psu.edu\u002F'\u003E\u003Cimg src=\"\u002Fsrc\u002Fimages\u002Flogos\u002FToolShed.jpg\" alt=\"Galaxy Tool Shed\" width=150 \u002F\u003E\u003C\u002Fa\u003E\u003C\u002Fdiv\u003E\n\n* [rxlr_venn_workflow](http:\u002F\u002Fbit.ly\u002F1071hG2): a workflow for comparing 3 RXLR prediction methods with a Venn Diagram \n* [vcf_to_variantdb](http:\u002F\u002Fbit.ly\u002F14bpN9c): Send UnifiedGenotyper VCF to a !VariantDB instance  \n* [iquant](http:\u002F\u002Fbit.ly\u002F15VS5cs): isobaric tag quantification program \n* [appendfdr](http:\u002F\u002Fbit.ly\u002F15VSvzq): compute false discovery rate of tabular data using decoy entries \n* [bumbershoot](http:\u002F\u002Fbit.ly\u002FZOh5e1): Tabb lab tools idqonvert, myrimatch, tagrecon, all for protein identification.\n* [dbbuilder](http:\u002F\u002Fbit.ly\u002F135vHq9): download protein databases from common sources  \n* [nbic_fasta](http:\u002F\u002Fbit.ly\u002F17SbTei): Galaxy-P unofficial fork of NBIC Galaxy tools for FASTA manipulation\n* [maxquant](http:\u002F\u002Fbit.ly\u002F17ScCfB): large mass-spec data quantitative proteomics analysis, especially high-resolution mass spectometry \n* [msconvert](http:\u002F\u002Fbit.ly\u002F14bsTKv): Tool wrappers for msconvert, distributed as part of Proteowizard.\n* [peptideshaker](http:\u002F\u002Fbit.ly\u002FYLoLSY): Identify proteins combining X! Tandem, OMSSA (using SearchGUI) and PeptideShaker \n* [psm_eval](http:\u002F\u002Fbit.ly\u002FZULT0e): Peptide-Spectrum-Match reevaluation tool  \n* [allele_counts](http:\u002F\u002Fbit.ly\u002F13QaSCN): Parse Naive Variant Detector output, count alleles and calculate minor allele frequencies.\n* [snp_analysis_conversion](http:\u002F\u002Fbit.ly\u002F1avPD9D): Format conversion for SNP Analysis tools from Miller Lab @ PSU. \n\n## Other News\n\n* Just discovered [these tutorials](http:\u002F\u002Fbit.ly\u002F167H7AP) by Lance Parsons on RNA-Seq, ChIP-Seq, SNP & Indels, and IGV all using Galaxy   \n* [Scaling variant detection pipelines for whole genome sequencing analysis](http:\u002F\u002Fj.mp\u002F10Tk2Kq)\n* There is now a [Google+ Galaxy Group](http:\u002F\u002Fgplus.to\u002Fgalaxyportal).  Thanks to Samuel Lampa for setting this up.\n"}},"context":{}};(function(){var s;(s=document.currentScript||document.scripts[document.scripts.length-1]).parentNode.removeChild(s);}());</script><script src="/assets/js/app.f7f9943a.js" defer></script><script src="/assets/js/page--src--templates--vue-article-vue.d03e3b77.js" defer></script><script src="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-06--index-md.a5232112.js" defer></script>
  </body>
</html>
