<!DOCTYPE html>
<html data-html-server-rendered="true" lang="en" data-vue-tag="%7B%22lang%22:%7B%22ssr%22:%22en%22%7D%7D">
  <head>
    <title>Custom Genomes - Galaxy Community Hub</title><meta name="gridsome:hash" content="eaf962330836a104724e8b210d95933603316eda"><meta data-vue-tag="ssr" charset="utf-8"><meta data-vue-tag="ssr" name="generator" content="Gridsome v0.7.23"><meta data-vue-tag="ssr" data-key="viewport" name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover"><meta data-vue-tag="ssr" data-key="format-detection" name="format-detection" content="telephone=no"><meta data-vue-tag="ssr" data-key="description" name="description" content="All about Galaxy and its community."><link data-vue-tag="ssr" rel="icon" href="data:,"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="16x16" href="/assets/static/favicon.ce0531f.7d4a227415c9d9b704d56a079d49a3e1.png"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="32x32" href="/assets/static/favicon.ac8d93a.7d4a227415c9d9b704d56a079d49a3e1.png"><link rel="preload" href="/assets/css/0.styles.8bf9ec2e.css" as="style"><link rel="preload" href="/assets/js/app.4169b14c.js" as="script"><link rel="preload" href="/assets/js/page--src--templates--article-vue.7f90f7d3.js" as="script"><link rel="preload" href="/assets/js/page--src--templates--article-vue~page--src--templates--vue-article-vue.c289395b.js" as="script"><link rel="prefetch" href="/assets/js/page--src--pages--404-vue.757e9e96.js"><link rel="prefetch" href="/assets/js/page--src--pages--blog-vue.a020a667.js"><link rel="prefetch" href="/assets/js/page--src--pages--careers-vue.f8bcd2e3.js"><link rel="prefetch" href="/assets/js/page--src--pages--data-explorer-vue.58967285.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--archive-vue.e9003ff3.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests--papercuts-vue.f6facc0a.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests-vue.c3e8e2b2.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--webinars-vue.a50a8e37.js"><link rel="prefetch" href="/assets/js/page--src--pages--events-vue.6b44f711.js"><link rel="prefetch" href="/assets/js/page--src--pages--index-vue.e21f2675.js"><link rel="prefetch" href="/assets/js/page--src--pages--news-vue.d5cd41ec.js"><link rel="prefetch" href="/assets/js/page--src--pages--search-vue.090976ac.js"><link rel="prefetch" href="/assets/js/page--src--pages--tests-vue.9ef24b0f.js"><link rel="prefetch" href="/assets/js/page--src--pages--use-vue.0be01484.js"><link rel="prefetch" href="/assets/js/page--src--templates--platform-vue.a56e417a.js"><link rel="prefetch" href="/assets/js/page--src--templates--vue-article-vue.bfa2ba22.js"><link rel="prefetch" href="/assets/js/vendors~page--src--pages--404-vue.769c153b.js"><link rel="prefetch" href="/assets/js/vendors~page--src--pages--404-vue~page--src--pages--careers-vue~page--src--pages--search-vue~page--s~6aaa8394.b4529ad8.js"><link rel="prefetch" href="/assets/js/vendors~page--src--pages--search-vue.8befcd1e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--index-md.11067c6d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--md--imgtests--imgdir--index-md.bf74dfc9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--partials--index-md.8656e780.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--imgdir--index-md.eca7c5ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--index-md.a7cac9b3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--index-md.958d858f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--index-md.da6161f9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--reldir--index-md.854a3057.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--capacity-planning--index-md.69b7128e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--index-md.f2d1d7aa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--performance--cluster--legacy--index-md.f9800132.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--datatypes--index-md.84ad32f5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--disk-quotas--index-md.3d889971.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--faq--index-md.910be801.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--data-model--index-md.13008439.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--galaxy-ops--index-md.a5346a8c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--ten-simple-steps-galaxy-as-a-service--index-md.e7a68355.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--tool-config-syntax--index-md.9f6b28cb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--writing-tests--index-md.4f24a155.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--allocation--index-md.6c4bec53.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--faq--index-md.e32353bc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--index-md.c0131e18.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--ssh--index-md.76a5a35d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--troubleshooting--index-md.ff16ed17.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--capacity-planning--index-md.7ebdba3e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--getting-started--index-md.0bee47f1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--index-md.cdda9a36.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--building--index-md.d1fb9af5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--capacity-planning--index-md.a45c87f5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--cluster-types--index-md.9c14189b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--customizing--index-md.9627664f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--faq--index-md.b2f845db.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--index-md.4fc6ce5f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--pre-201703--index-md.5b8c873a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--hadoop--index-md.a01cef54.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--ht-condor--index-md.0dcbc49e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--index-md.d5e2a096.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--openstack--index-md.eed9b01d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--services--index-md.db34aabc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--sharing--index-md.479cd135.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--troubleshooting--index-md.61ed81f7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--userdata--index-md.08ce96da.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--future--index-md.fef4a8cb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--index-md.78f52817.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-09--index-md.ac03a5b6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-27--index-md.a56bfa70.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-09-19--index-md.a0976ba4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-11-14--index-md.a9153135.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-01-16--index-md.1ba29422.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-03-20--index-md.0db664a3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-05-15--index-md.d37592fd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-11-20--index-md.a409dc1a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-02-19--index-md.42a2979f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-04-16--index-md.40d17d01.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-06-18--index-md.66f73e0e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-08-20--index-md.db54c8a7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-10-15--index-md.e234a0f3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-02-18--index-md.24a180df.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-04-21--index-md.0ba24e10.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-06-16--index-md.9e797eb1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-08-18--index-md.d83e8b4a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--index-md.540866ce.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--webinar-tech--index-md.a861e596.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2012--index-md.436f25c1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2014--index-md.26f60517.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--index-md.8c39551b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--data--index-md.89dfaed6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--datafiles--index-md.b8ea37ea.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--architecture--index-md.42a21fab.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--bitbucket--index-md.ed830e89.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--css--index-md.d7eb156b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--getting-started--index-md.bf1ead58.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--index-md.8eac10cf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--jsa--index-md.037a145c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--mercurial--index-md.4b546d11.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--security-tool-tips--index-md.8cdb7df5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--galaxy-project--index-md.4686d49d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--index-md.83cee777.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--papers--index-md.4b8ecfa4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--posters--index-md.d1e3edb4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--presentations--index-md.0e2d2e1a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--videos--index-md.70752028.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--advanced-session--index-md.b9f3ed1e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--basics-session--index-md.2bcf225d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--index-md.b8d05422.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--logistics--index-md.9a26fa92.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--publicity--index-md.cf406358.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--registration--index-md.a1309166.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--abstracts--index-md.b5b4584c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--index-md.b57031f3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--key-dates--index-md.8090038a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--lightning-talks--index-md.bc6442f2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--logistics--index-md.be6fe2d9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--organizing-committee--index-md.1c7932ef.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--photos--index-md.8bfc91d3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--automation-strategies--index-md.aab816f0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--bioinformatics-training--index-md.8b587560.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--end-users--index-md.f4036fe7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--workflows-and-api--index-md.f117be80.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--index-md.6eb6a1f1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--promotion--index-md.8a3e997f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--register--index-md.d6e73d37.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--sponsorships--index-md.9a501620.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--training-day--index-md.028bfddb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--index-md.f56e0e40.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--posters--index-md.54e334df.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--talks--index-md.e03ca9cd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--clinical-use-hpc-file-systems--index-md.479d8fca.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-admins--index-md.84645809.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-blast--index-md.4165d5c8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-professionals--index-md.4784e072.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--index-md.9735e2e7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--proteomics--index-md.235cc338.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--public-galaxy-servers--index-md.ca6be78b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--tool-developers--index-md.5488ae6c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--index-md.b6fe9d05.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--key-dates--index-md.95045adf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--lightning--index-md.0f476bf0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--logistics--index-md.3f7fab58.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--organizers--index-md.6b745e0b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--photos--index-md.46f289a6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--program--index-md.2dc90221.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--promotion--index-md.f86302a0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--register--index-md.b8ebbc6b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--sponsorships--index-md.5bd62dd5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--advance-tool-data--index-md.85274240.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--api--index-md.16264cdc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--index-md.01d1a1a9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--intro-tool-data--index-md.4db9ced0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--vms--index-md.f2bd63c3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--index-md.2bb7a0c5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--posters--index-md.79573dfe.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--talks--index-md.d3c01835.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--branch-release-merge--index-md.c6b43399.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-admins--index-md.0a5eb972.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-end-users--index-md.912197e9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-training-network--index-md.ee1827b5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--heterogeneous--index-md.9baea693.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--index-md.fdd1ffad.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--using-galaxy-ui-overview--index-md.9e227d33.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--hackathon--index-md.ed2f45ef.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--index-md.f1ed9a04.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--key-dates--index-md.fae4ba9d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--lightning--index-md.80c44468.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--logistics--index-md.841fc0fd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--organizers--index-md.7f343588.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--photos--index-md.2a63fc2a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--program--index-md.5ae4685e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--promotion--index-md.6e706c8a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--register--index-md.70e5c55a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--sponsor-exhibit--index-md.cf443bd4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--admin-walkthrough--index-md.83edf099.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--api--index-md.16d8430c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--data-managers--index-md.8b01904d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--index-md.420c7759.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--vms--index-md.8440c898.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--cwl--index-md.4c687cfe.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-management-challenges--index-md.46cbd5a6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-splitting-and-parallelism--index-md.596d67fa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--elixir--index-md.b760684b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--extended-workflow--index-md.7abfaf7f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--fots--index-md.292a9f73.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-admins--index-md.86489759.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-community-uk--index-md.2f296ab3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-scientists--index-md.0650adbc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-training-network--index-md.bd7432f3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--index-md.698afae1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--data-hack-roundup--index-md.81d8b289.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--header--index-md.6a085933.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--index-md.9d5304e9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2016--bofs--index-md.e45fe6e9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--abstracts--index-md.1357c177.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--childcare--index-md.65f808f4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--cofest--index-md.e336528c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--conduct--index-md.cbb9369e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--faq--index-md.81a14824.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--index-md.eb9ca6ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--organizers--index-md.fa504f7f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--promotion--index-md.e195e4c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--registration--index-md.ff34324b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--sponsors--index-md.35239aeb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--training--index-md.35b38787.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--venue--index-md.b23ba2f9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--abstracts--index-md.a964faf5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--childcare--index-md.d77de6de.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--code-of-conduct--index-md.eafab0ba.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--collaboration--index-md.56a546a0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--faq--index-md.7336048d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--icebreaker--index-md.00e425d8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--index-md.efb025b5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--keynotes--index-md.2d849c13.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--organizers--index-md.194881fd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--promotion--index-md.bee008c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--register--index-md.1b4ec5c7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--sponsors--index-md.561d5e54.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--training--index-md.8dd4291b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--venue--index-md.72afd1d4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2014--index-md.16cb7d43.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2016--index-md.cb7bab71.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2014--index-md.59dba3f9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2016--index-md.4280b00e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--index-md.f21026a7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--trainingday--index-md.de9750eb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2016--index-md.327bdf0d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--example--vue-remark--index-md.b7f18147.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--faqs--index-md.be042548.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-06--index-md.20f3500b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-07--index-md.87c20dee.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--icons--index-md.cd46c11c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--index-md.14afa90b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--logos--index-md.0fbf405d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--news-graphics--index-md.0fa5ff73.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--photos--index-md.c9d4f3be.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--screenshots--index-md.bc7be700.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--index-md.8e64f404.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--workflow-editor-unannotated--index-md.b60ef6c1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--extract--index-md.92f9a05c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--tool-panel--index-md.086fd516.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--variables--index-md.c9116760.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--faq--index-md.3a40d93c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--galaxy-pages--index-md.0feedd43.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--interval-operations--index-md.4adb4e28.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--main--faq--index-md.d705d512.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--tools--requirements--index-md.6870b69b.js"><link rel="stylesheet" href="/assets/css/0.styles.8bf9ec2e.css"><script data-vue-tag="ssr" src="https://sidecar.gitter.im/dist/sidecar.v1.js" async defer></script><noscript data-vue-tag="ssr"><style>.g-image--loading{display:none;}</style></noscript>
  </head>
  <body >
    <div data-server-rendered="true" id="app" class="layout" data-v-818dbe78><header id="masthead"><nav class="navbar container justify-content-center navbar-dark bg-transparent navbar-expand-lg"><a href="/" target="_self" class="navbar-brand router-link-active"><img id="masthead-logo" src="/images/galaxy_logo_hub_white.svg" alt="Galaxy Community Hub" height="30"></a><button type="button" aria-label="Toggle navigation" class="navbar-toggler"><span class="navbar-toggler-icon"></span></button><div id="nav-collapse" class="navbar-collapse collapse" style="display:none;"><ul class="navbar-nav"><li class="nav-item"><a href="/use/" target="_self" class="nav-link">Use</a></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Learn</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/learn/" role="menuitem" target="_self" class="dropdown-item router-link-active">How to use Galaxy</a></li><li role="presentation"><a role="menuitem" href="https://training.galaxyproject.org/" target="_self" class="dropdown-item">Teach with Galaxy</a></li></ul></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Community</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/community/" role="menuitem" target="_self" class="dropdown-item">About</a></li><li role="presentation"><a href="/events/" role="menuitem" target="_self" class="dropdown-item">Events</a></li><li role="presentation"><a href="/news/" role="menuitem" target="_self" class="dropdown-item">News</a></li><li role="presentation"><a href="/blog/" role="menuitem" target="_self" class="dropdown-item">Blog</a></li><li role="presentation"><a href="/publication-library/" role="menuitem" target="_self" class="dropdown-item">Publications</a></li><li role="presentation"><a href="/galaxy-project/statistics/" role="menuitem" target="_self" class="dropdown-item">Stats</a></li><li role="presentation"><a href="/careers/" role="menuitem" target="_self" class="dropdown-item">Careers</a></li></ul></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Deploy &amp; Develop</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/admin/get-galaxy/" role="menuitem" target="_self" class="dropdown-item">Install Galaxy</a></li><li role="presentation"><a href="/admin/" role="menuitem" target="_self" class="dropdown-item">Server administration</a></li><li role="presentation"><a href="/tools/" role="menuitem" target="_self" class="dropdown-item">Tools</a></li><li role="presentation"><a href="/develop/api/" role="menuitem" target="_self" class="dropdown-item">Working with the API</a></li><li role="presentation"><a href="/develop/" role="menuitem" target="_self" class="dropdown-item">Contributing</a></li><li role="presentation"><a href="/docs/" role="menuitem" target="_self" class="dropdown-item">Code documentation</a></li></ul></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Support</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/support/" role="menuitem" target="_self" class="dropdown-item">FAQ</a></li><li role="presentation"><a role="menuitem" href="https://help.galaxyproject.org/" target="_self" class="dropdown-item">Galaxy Help Forum</a></li></ul></li><li class="nav-item"><a href="/jxtx/" target="_self" class="nav-link">@jxtx</a></li></ul><ul class="navbar-nav ml-auto"><li class="form-inline"><form action="/search/" method="get" class="form-inline"><input id="search-input" name="q" type="text" placeholder="Search Galaxy" value="" class="mr-sm-2 form-control form-control-sm"><button type="submit" class="btn my-2 my-sm-0 btn-secondary btn-sm"><span class="fa fa-search"></span></button></form></li></ul></div></nav></header><section id="maincontainer" class="container"><header data-v-173c38b0 data-v-818dbe78><a href="/" class="link active" data-v-173c38b0>← Back to Home</a><!----><!----><h1 class="title" data-v-173c38b0>Custom Genomes</h1><!----><section class="metadata freetext" data-v-173c38b0><!----><!----><!----></section><!----><!----></header><div class="content markdown" data-v-818dbe78><div class="toc-wrapper col-md-3">
<ul>
<li><a href="#what-is-a-custom-reference-genome-">What is a "Custom Reference Genome" ?</a></li>
<li><a href="#overview">Overview</a></li>
<li><a href="#screencasts--tutorials">Screencasts &#x26; Tutorials</a></li>
<li><a href="#sources">Sources</a></li>
<li><a href="#format">Format</a></li>
<li><a href="#custom-builds">Custom Builds</a></li>
<li><a href="#sorting">Sorting</a></li>
<li>
<p><a href="#troubleshooting">Troubleshooting</a></p>
<ul>
<li>
<ul>
<li><a href="#1-custom-genome-not-assigned-as-a-hreflearndatatypesfastafastaa-format">1. Custom genome not assigned as <a href="/learn/datatypes/#fasta">FASTA</a> format</a></li>
<li><a href="#2-incomplete-custom-genome-file-load">2. Incomplete Custom genome file load</a></li>
<li><a href="#3-extra-spaces-extra-lines-inconsistent-line-wrapping-or-any-deviation-from-strict-a-hreflearndatatypesfastafastaa-format">3. Extra spaces, extra lines, inconsistent line wrapping, or any deviation from strict <a href="/learn/datatypes/#fasta">FASTA</a> format</a></li>
<li><a href="#4-inconsistent-line-wrapping-common-if-merging-chromosomes-from-various-genbank-records-eg-primary-chroms-with-mito">4. Inconsistent line wrapping, common if merging chromosomes from various Genbank records (e.g. primary chroms with mito)</a></li>
<li><a href="#5-unsorted-fasta-genome-file">5. Unsorted fasta genome file</a></li>
<li><a href="#6-identifier-and-description-in--title-lines-used-inconsistently-by-tools-in-the-same-analysis">6. Identifier and Description in ">" title lines used inconsistently by tools in the same analysis</a></li>
<li><a href="#7-unassigned-database">7. Unassigned database</a></li>
</ul>
</li>
</ul>
</li>
<li>
<p><a href="#a-problem-or-not-a-problem">A problem or not a problem?</a></p>
<ul>
<li>
<ul>
<li><a href="#a-did-you-use-the-same-exact-reference-genome-for-all-steps-in-the-same-analysis-pipeline">A. Did you use the same exact reference genome for all steps in the same analysis pipeline?</a></li>
<li><a href="#b-any-tool-including-cufflinksmergediff-reports-a-missingproblem-transcriptsgtf-file">B. Any tool, including Cufflinks/merge/diff, reports a missing/problem transcripts.gtf file.</a></li>
<li><a href="#c-what-if-the-inputs-are-truly-not-a-match">C. What if the inputs are truly not a match?</a></li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
<div class="body-wrapper col-md-9">
<p><a href="/support/">Support Hub</a></p>
<p><a href="/learn/">Learning Hub</a></p>
<hr>
<h1 id="what-is-a-custom-reference-genome-"><a href="#what-is-a-custom-reference-genome-" aria-hidden="true"><span class="icon icon-link"></span></a>What is a "Custom Reference Genome" ?</h1>
<p>A reference genome contains the nucleotide sequence of the chromosomes, scaffolds, transcripts, or contigs for a single species. It is representative of a specific genome build or release.</p>
<p>In <a href="http://usegalaxy.org" target="_blank" rel="noopener noreferrer">Galaxy</a>, a <strong>custom reference genome</strong> is a <a href="/learn/datatypes/#fasta">FASTA</a> formatted <a href="/learn/#datasets">dataset</a> that can be used in place of a <strong>native reference genome</strong> with most tools.</p>
<ul>
<li><strong>custom</strong>: a dataset <em>from the history</em> loaded by users</li>
<li><strong>native</strong>: <em>local</em> or <em>cached</em> by administrators (see <a href="/admin/data-preparation/">Admin/DataPreparation</a>)</li>
</ul>
<h1 id="overview"><a href="#overview" aria-hidden="true"><span class="icon icon-link"></span></a>Overview</h1>
<p>There are <strong>five basic steps</strong> to using a <em>Custom Reference Genome</em>:</p>
<ul>
<li>Obtain a <strong><a href="/learn/datatypes/#fasta">FASTA</a> copy of the target genome</strong></li>
<li><strong><a href="/ftp-upload/">FTP</a> the genome to Galaxy</strong> and load into a history as a <a href="/learn/#datasets">dataset</a></li>
<li><strong>Clean up the format</strong> with the tool <strong>NormalizeFasta</strong> using the options to wrap sequence lines at 80 bases and to trim the title line at the first whitespace.</li>
<li>Make sure the <a href="https://galaxyproject.org/support/chrom-identifiers" target="_blank" rel="noopener noreferrer">chromosome identifiers</a> are a match for other inputs</li>
<li><strong>Set a tool form's options to use a custom reference genome from the history</strong> and select the loaded genome</li>
</ul>
<h1 id="screencasts--tutorials"><a href="#screencasts--tutorials" aria-hidden="true"><span class="icon icon-link"></span></a>Screencasts &#x26; Tutorials</h1>
<table>
<thead>
<tr>
<th>Screencast</th>
<th>Topic</th>
</tr>
</thead>
<tbody>
<tr>
<td><strong><a href="https://vimeo.com/123108417" target="_blank" rel="noopener noreferrer">NGS101-8 Mapping to YOUR Reference</a></strong></td>
<td>demonstrates <strong>BWA-MEM</strong> tool form options using a custom genome from the history</td>
</tr>
<tr>
<td><strong><a href="https://vimeo.com/75918922" target="_blank" rel="noopener noreferrer">Custom Genomes</a></strong></td>
<td>explains format, usage, and how to load a genome that works with tools</td>
</tr>
</tbody>
</table>
<h1 id="sources"><a href="#sources" aria-hidden="true"><span class="icon icon-link"></span></a>Sources</h1>
<ul>
<li>UCSC, Ensembl, NCBI/GenBank</li>
<li>Other Research project associated with specific genome projects</li>
<li>Internal research projects</li>
<li>Selected genomes can be found in "Data Libraries" on <a href="/main/">Main</a> for use at <a href="http://usegalaxy.org" target="_blank" rel="noopener noreferrer">http://usegalaxy.org</a>. Example: <strong>hg19</strong> is available for GATK under that sub-directory.</li>
</ul>
<h1 id="format"><a href="#format" aria-hidden="true"><span class="icon icon-link"></span></a>Format</h1>
<ul>
<li>Custom Genomes are required to be in <a href="/learn/datatypes/#fasta">FASTA</a> format</li>
<li>The data should be formatted as <a href="/learn/datatypes/#fasta">FASTA</a> prior to upload into Galaxy</li>
<li>The dataset will need to be labeled as <a href="/learn/datatypes/#fasta">FASTA</a> after loaded (if not automatically assigned)</li>
<li>Did you use <strong>NormalizeFasta</strong> with the options to wrap sequence lines at 80 bases and to trim the title line at the first whitespace?</li>
</ul>
<h1 id="custom-builds"><a href="#custom-builds" aria-hidden="true"><span class="icon icon-link"></span></a>Custom Builds</h1>
<p>Some tools and functions require that the <a href="/support/datatypes-and-tools/">'database' attribute is assigned</a> or that a Custom Reference Genome is set up as a <em>Custom Build</em> prior to use. Examples are the tools <strong>Featurecounts</strong>, <strong>Extract Genomic DNA</strong>, certain <strong>Picard</strong> tools, and the functions under  <strong>Visualization</strong>.</p>
<p>Once created, a <em>Custom Build</em> is added to the list <strong>Database/Build:</strong> on the dataset 'Edit Attributes' and 'Upload File' tool forms and is available for 'Visualizations'. These can be assigned or used just like any other reference genome.</p>
<ul>
<li>Start with an existing fasta Custom Reference Genome in your history. It is <em>very important</em> make sure the <a href="/learn/custom-genomes/#format">format</a> is correct.</li>
<li>Go to the top "User" menu and select "Custom Builds"</li>
<li>Enter in the labels (no spaces and no special characters other than "_")</li>
<li>Select the fasta Custom Reference Genome</li>
<li>Submit and wait for the build to finish loading before assigning to a dataset or using to start a new Visualization</li>
<li>Note: It is fine to navigate away from this form and come back to it later to check for status. The larger the fasta file and busier the Galaxy instance is, the longer the processing will take.</li>
</ul>
<h1 id="sorting"><a href="#sorting" aria-hidden="true"><span class="icon icon-link"></span></a>Sorting</h1>
<p>Certain tools expect that reference genomes are sorted in <a href="http://en.wikipedia.org/wiki/Lexicographical_order" target="_blank" rel="noopener noreferrer">lexicographical order</a>. These tools are often downstream of the initial mapping tools, which means that a large investment in a project has already been made (i.e. a long mapping process), before a problem with sorting pops up in conclusion layer tools. No one likes to start over!</p>
<p>How to avoid? Always sort your <a href="/learn/datatypes/#fasta">FASTA</a> reference genome dataset at the beginning of a project. Many sources only provide sorted genomes, but double checking is your own responsibility, and super easy in Galaxy. So easy that there isn't even a shared workflow, just a recipe (but feel free to make your own):</p>
<p>Quick lexicographical sort recipe:</p>
<pre><code>1. Convert Formats -> FASTA-to-Tabular
2. Filter and Sort -> Sort
       on column: c1 
       with flavor: Alphabetical
       everything in: Ascending order
3. Convert Formats -> Tabular-to-FASTA
</code></pre>
<p>The above sorting method is for most tools, but not all. In particular, GATK tools have a tool-specific sort order requirement. The <a href="http://www.broadinstitute.org" target="_blank" rel="noopener noreferrer">Broad Institute</a> FAQ with input format instructions is <strong><a href="http://www.broadinstitute.org/gark/guide/article?id=1204" target="_blank" rel="noopener noreferrer">here</a></strong>.</p>
<h1 id="troubleshooting"><a href="#troubleshooting" aria-hidden="true"><span class="icon icon-link"></span></a>Troubleshooting</h1>
<p>If a custom genome dataset is producing errors, double check the format and that the chromosome identifiers between <strong>ALL</strong> inputs. Clicking on the green bug icon <img src="/images/icons/bug.png"> will often provide a description of the problem. This does not automatically submit a bug report, and it is not always necessary to do so, but it is a good way to get more information about why a job is failing.</p>
<h3 id="1-custom-genome-not-assigned-as-a-hreflearndatatypesfastafastaa-format"><a href="#1-custom-genome-not-assigned-as-a-hreflearndatatypesfastafastaa-format" aria-hidden="true"><span class="icon icon-link"></span></a>1. Custom genome not assigned as <a href="/learn/datatypes/#fasta">FASTA</a> format</h3>
<ul>
<li><strong>Symptoms include</strong>: Dataset not included in custom genome pull down menu on tool forms.</li>
<li><strong>Solution</strong>: Check datatype assigned to dataset and assign <strong>fasta</strong> format.</li>
<li><strong>How</strong>: Click on the dataset's pencil icon <img src="/images/icons/pencil.png"> to reach the "Edit Attributes" page, and in the <a href="/learn/datatypes/">datatypes</a> section, type in "fasta", and save.</li>
</ul>
<h3 id="2-incomplete-custom-genome-file-load"><a href="#2-incomplete-custom-genome-file-load" aria-hidden="true"><span class="icon icon-link"></span></a>2. Incomplete Custom genome file load</h3>
<ul>
<li><strong>Symptoms include</strong>: Tool errors result the first time you use the Custom genome.</li>
<li><strong>Solution</strong>: Use <strong>Text Manipulation → Select last lines from a dataset</strong> to check last 10 lines to see if file is truncated.</li>
<li><strong>How</strong>: Reload the dataset (switch to <strong><a href="/ftp-upload/">FTP</a></strong> if not using already). Check your FTP client logs to make sure the load is complete.</li>
</ul>
<h3 id="3-extra-spaces-extra-lines-inconsistent-line-wrapping-or-any-deviation-from-strict-a-hreflearndatatypesfastafastaa-format"><a href="#3-extra-spaces-extra-lines-inconsistent-line-wrapping-or-any-deviation-from-strict-a-hreflearndatatypesfastafastaa-format" aria-hidden="true"><span class="icon icon-link"></span></a>3. Extra spaces, extra lines, inconsistent line wrapping, or any deviation from strict <a href="/learn/datatypes/#fasta">FASTA</a> format</h3>
<ul>
<li><strong>Symptoms include</strong>: RNA-seq tools (<strong>Cufflinks, Cuffcompare, Cuffmerge, Cuffdiff</strong>) fails with error <code>Error: sequence lines in a FASTA record must have the same length!</code>.</li>
<li><strong>Solution</strong>: File tested and corrected locally then re-upload or test/fix within Galaxy, then re-run.</li>
<li><strong>How</strong>:</li>
<li><strong>Quick re-formatting</strong> Run the tool through the tool <strong>NormalizeFasta</strong> using the options to wrap sequence lines at 80 bases and to trim the title line at the first whitespace.</li>
<li><strong>Optional Detailed re-formatting</strong> Start with <strong>FASTA manipulation → FASTA Width formatter </strong> with a value between 40-80 (60 is common) to reformat wrapping. Next, use <strong>Filter and Sort → Select</strong> with ">" to examine identifiers. Use a combination of <strong>Convert Formats → FASTA-to-Tabular</strong>, <strong>Text Manipulation</strong> tools, then <strong>Tabular-to-FASTA</strong> to correct.</li>
<li><strong>With either of the above</strong>, finish by using <strong>Filter and Sort → Select</strong> with <code>^\w*$</code> to search for empty lines (use "NOT matching" to remove these lines and output a properly format fasta dataset).</li>
</ul>
<h3 id="4-inconsistent-line-wrapping-common-if-merging-chromosomes-from-various-genbank-records-eg-primary-chroms-with-mito"><a href="#4-inconsistent-line-wrapping-common-if-merging-chromosomes-from-various-genbank-records-eg-primary-chroms-with-mito" aria-hidden="true"><span class="icon icon-link"></span></a>4. Inconsistent line wrapping, common if merging chromosomes from various Genbank records (e.g. primary chroms with mito)</h3>
<ul>
<li><strong>Symptoms include</strong>: Tools (<strong>SAMTools</strong>, <strong>Extract Genomic DNA</strong>, but rarely alignment tools) may complain about unexpected line lengths/missing identifiers. Or they may just fail for what appears to be a cluster error.</li>
<li><strong>Solution</strong>: File tested and corrected locally then re-upload or <strong>test/fix within Galaxy</strong>.</li>
<li><strong>How</strong>: Use <strong><strong>NormalizeFasta</strong></strong> using the options to wrap sequence lines at 80 bases and to trim the title line at the first whitespace. Finish by using <strong>Filter and Sort → Select</strong> with <code>^\w*$</code> to search for empty lines (use "NOT matching" to remove these lines and output a properly format fasta dataset).</li>
</ul>
<h3 id="5-unsorted-fasta-genome-file"><a href="#5-unsorted-fasta-genome-file" aria-hidden="true"><span class="icon icon-link"></span></a>5. Unsorted fasta genome file</h3>
<ul>
<li><strong>Symptoms include</strong>: Tools such as <strong>Extract Genomic DNA</strong> report problems with sequence lengths.</li>
<li><strong>Solution</strong>: First try sorting and re-formatting in Galaxy then re-run.</li>
<li><strong>How</strong>: To sort, follow instructions for <a href="/learn/custom-genomes/#sorting">Sorting</a> a Custom Genome.</li>
</ul>
<h3 id="6-identifier-and-description-in--title-lines-used-inconsistently-by-tools-in-the-same-analysis"><a href="#6-identifier-and-description-in--title-lines-used-inconsistently-by-tools-in-the-same-analysis" aria-hidden="true"><span class="icon icon-link"></span></a>6. Identifier and Description in ">" title lines used inconsistently by tools in the same analysis</h3>
<ul>
<li><strong>Symptoms include</strong>: Will generally manifest as a false genome-mismatch problem.</li>
<li><strong>Solution</strong>: Remove the description content and re-run all tools/workflows that used this input. Mapping tools will usually not fail, but downstream tools will. When this comes up, it usually means that an analysis needs to be started over from the mapping step to correct the problems. No one enjoys redoing this work. Avoid the problems by formatting the genome, by double checking that the same reference genome was used for all steps, and by making certain the 'identifiers' are a match between all planned inputs (including reference annotation such as GTF data) <strong>before using your custom genome</strong>.</li>
<li><strong>How</strong>: To drop the title line description content, use <strong>NormalizeFasta</strong> using the options to wrap sequence lines at 80 bases and to trim the title line at the first whitespace. Next, double check that the <a href="https://galaxyproject.org/support/chrom-identifiers" target="_blank" rel="noopener noreferrer">chromosome identifiers are an exact match between all inputs</a>.</li>
</ul>
<h3 id="7-unassigned-database"><a href="#7-unassigned-database" aria-hidden="true"><span class="icon icon-link"></span></a>7. Unassigned database</h3>
<ul>
<li><strong>Symptoms include</strong>: Tools report that no build is available for the assigned reference genome.</li>
<li><strong>Solution</strong> This occurs with tools that require an assigned <em>database</em> metadata attribute. <strong>SAMTools</strong> and <strong>Picard</strong> often require this assignment.</li>
<li><strong>How</strong>: Create a <a href="/learn/custom-genomes/#custom-builds">Custom Build</a> and assign it to the dataset</li>
</ul>
<h1 id="a-problem-or-not-a-problem"><a href="#a-problem-or-not-a-problem" aria-hidden="true"><span class="icon icon-link"></span></a>A problem or not a problem?</h1>
<p>Certain job errors with RNA-seq tools can at first appear to look like a format problem with a custom reference genome, but are actually a bit more complicated...</p>
<h3 id="a-did-you-use-the-same-exact-reference-genome-for-all-steps-in-the-same-analysis-pipeline"><a href="#a-did-you-use-the-same-exact-reference-genome-for-all-steps-in-the-same-analysis-pipeline" aria-hidden="true"><span class="icon icon-link"></span></a>A. Did you use the same exact reference genome for all steps in the same analysis pipeline?</h3>
<p>Alignment tools (BWA, Bowtie, Tophat) are generally tolerant of minor formatting problems with reference genomes. However, downstream tools tend to have more stringent format requirements. To avoid having to reprocess, <strong>a best practice is to verify that the formatting is correct before any steps are started</strong>.</p>
<h3 id="b-any-tool-including-cufflinksmergediff-reports-a-missingproblem-transcriptsgtf-file"><a href="#b-any-tool-including-cufflinksmergediff-reports-a-missingproblem-transcriptsgtf-file" aria-hidden="true"><span class="icon icon-link"></span></a>B. Any tool, including Cufflinks/merge/diff, reports a missing/problem transcripts.gtf file.</h3>
<p>This generally indicates a mismatch in the chromosome identifiers between the reference genome used for the mapping/alignment step, and the reference annotation GTF data and/or the reference genome used in downstream steps.</p>
<h3 id="c-what-if-the-inputs-are-truly-not-a-match"><a href="#c-what-if-the-inputs-are-truly-not-a-match" aria-hidden="true"><span class="icon icon-link"></span></a>C. What if the inputs are truly not a match?</h3>
<p>The problem can sometimes be corrected by altering the chromosome identifiers in the GTF file or the reference genome. However, making changes can introduce scientific genome-mismatch problems. It is generally much better to find reference data (genomes, reference files) that are confirmed to be a match.</p>
</div>
</div><footer data-v-54ece1d6 data-v-818dbe78><!----></footer><footer class="static-footer markdown"><p>The Galaxy Project is supported in part by NSF, NHGRI, The Huck Institutes of the Life Sciences, The Institute for CyberScience at Penn State, and Johns Hopkins University.</p>
</footer></section></div>
    <script>window.__INITIAL_STATE__={"data":{"article":{"id":"ec13212c27addb885c9c3b9e7299b0be","title":"Custom Genomes","tease":"","category":null,"date":null,"days":null,"contact":"","contact_url":"","authors":"","location":"","location_url":"","source_blog":"","source_blog_url":"","skip_title_render":null,"redirect":"","links":[],"image":"","images":{},"external_url":"","content":"\u003Cdiv class=\"toc-wrapper col-md-3\"\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"#what-is-a-custom-reference-genome-\"\u003EWhat is a \"Custom Reference Genome\" ?\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#overview\"\u003EOverview\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#screencasts--tutorials\"\u003EScreencasts &#x26; Tutorials\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#sources\"\u003ESources\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#format\"\u003EFormat\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#custom-builds\"\u003ECustom Builds\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#sorting\"\u003ESorting\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Ca href=\"#troubleshooting\"\u003ETroubleshooting\u003C\u002Fa\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"#1-custom-genome-not-assigned-as-a-hreflearndatatypesfastafastaa-format\"\u003E1. Custom genome not assigned as \u003Ca href=\"\u002Flearn\u002Fdatatypes\u002F#fasta\"\u003EFASTA\u003C\u002Fa\u003E format\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#2-incomplete-custom-genome-file-load\"\u003E2. Incomplete Custom genome file load\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#3-extra-spaces-extra-lines-inconsistent-line-wrapping-or-any-deviation-from-strict-a-hreflearndatatypesfastafastaa-format\"\u003E3. Extra spaces, extra lines, inconsistent line wrapping, or any deviation from strict \u003Ca href=\"\u002Flearn\u002Fdatatypes\u002F#fasta\"\u003EFASTA\u003C\u002Fa\u003E format\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#4-inconsistent-line-wrapping-common-if-merging-chromosomes-from-various-genbank-records-eg-primary-chroms-with-mito\"\u003E4. Inconsistent line wrapping, common if merging chromosomes from various Genbank records (e.g. primary chroms with mito)\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#5-unsorted-fasta-genome-file\"\u003E5. Unsorted fasta genome file\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#6-identifier-and-description-in--title-lines-used-inconsistently-by-tools-in-the-same-analysis\"\u003E6. Identifier and Description in \"\u003E\" title lines used inconsistently by tools in the same analysis\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#7-unassigned-database\"\u003E7. Unassigned database\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Ca href=\"#a-problem-or-not-a-problem\"\u003EA problem or not a problem?\u003C\u002Fa\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"#a-did-you-use-the-same-exact-reference-genome-for-all-steps-in-the-same-analysis-pipeline\"\u003EA. Did you use the same exact reference genome for all steps in the same analysis pipeline?\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#b-any-tool-including-cufflinksmergediff-reports-a-missingproblem-transcriptsgtf-file\"\u003EB. Any tool, including Cufflinks\u002Fmerge\u002Fdiff, reports a missing\u002Fproblem transcripts.gtf file.\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#c-what-if-the-inputs-are-truly-not-a-match\"\u003EC. What if the inputs are truly not a match?\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fdiv\u003E\n\u003Cdiv class=\"body-wrapper col-md-9\"\u003E\n\u003Cp\u003E\u003Ca href=\"\u002Fsupport\u002F\"\u003ESupport Hub\u003C\u002Fa\u003E\u003C\u002Fp\u003E\n\u003Cp\u003E\u003Ca href=\"\u002Flearn\u002F\"\u003ELearning Hub\u003C\u002Fa\u003E\u003C\u002Fp\u003E\n\u003Chr\u003E\n\u003Ch1 id=\"what-is-a-custom-reference-genome-\"\u003E\u003Ca href=\"#what-is-a-custom-reference-genome-\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EWhat is a \"Custom Reference Genome\" ?\u003C\u002Fh1\u003E\n\u003Cp\u003EA reference genome contains the nucleotide sequence of the chromosomes, scaffolds, transcripts, or contigs for a single species. It is representative of a specific genome build or release.\u003C\u002Fp\u003E\n\u003Cp\u003EIn \u003Ca href=\"http:\u002F\u002Fusegalaxy.org\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EGalaxy\u003C\u002Fa\u003E, a \u003Cstrong\u003Ecustom reference genome\u003C\u002Fstrong\u003E is a \u003Ca href=\"\u002Flearn\u002Fdatatypes\u002F#fasta\"\u003EFASTA\u003C\u002Fa\u003E formatted \u003Ca href=\"\u002Flearn\u002F#datasets\"\u003Edataset\u003C\u002Fa\u003E that can be used in place of a \u003Cstrong\u003Enative reference genome\u003C\u002Fstrong\u003E with most tools.\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Cstrong\u003Ecustom\u003C\u002Fstrong\u003E: a dataset \u003Cem\u003Efrom the history\u003C\u002Fem\u003E loaded by users\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003Enative\u003C\u002Fstrong\u003E: \u003Cem\u003Elocal\u003C\u002Fem\u003E or \u003Cem\u003Ecached\u003C\u002Fem\u003E by administrators (see \u003Ca href=\"\u002Fadmin\u002Fdata-preparation\u002F\"\u003EAdmin\u002FDataPreparation\u003C\u002Fa\u003E)\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch1 id=\"overview\"\u003E\u003Ca href=\"#overview\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EOverview\u003C\u002Fh1\u003E\n\u003Cp\u003EThere are \u003Cstrong\u003Efive basic steps\u003C\u002Fstrong\u003E to using a \u003Cem\u003ECustom Reference Genome\u003C\u002Fem\u003E:\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003EObtain a \u003Cstrong\u003E\u003Ca href=\"\u002Flearn\u002Fdatatypes\u002F#fasta\"\u003EFASTA\u003C\u002Fa\u003E copy of the target genome\u003C\u002Fstrong\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003E\u003Ca href=\"\u002Fftp-upload\u002F\"\u003EFTP\u003C\u002Fa\u003E the genome to Galaxy\u003C\u002Fstrong\u003E and load into a history as a \u003Ca href=\"\u002Flearn\u002F#datasets\"\u003Edataset\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003EClean up the format\u003C\u002Fstrong\u003E with the tool \u003Cstrong\u003ENormalizeFasta\u003C\u002Fstrong\u003E using the options to wrap sequence lines at 80 bases and to trim the title line at the first whitespace.\u003C\u002Fli\u003E\n\u003Cli\u003EMake sure the \u003Ca href=\"https:\u002F\u002Fgalaxyproject.org\u002Fsupport\u002Fchrom-identifiers\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Echromosome identifiers\u003C\u002Fa\u003E are a match for other inputs\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003ESet a tool form's options to use a custom reference genome from the history\u003C\u002Fstrong\u003E and select the loaded genome\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch1 id=\"screencasts--tutorials\"\u003E\u003Ca href=\"#screencasts--tutorials\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EScreencasts &#x26; Tutorials\u003C\u002Fh1\u003E\n\u003Ctable\u003E\n\u003Cthead\u003E\n\u003Ctr\u003E\n\u003Cth\u003EScreencast\u003C\u002Fth\u003E\n\u003Cth\u003ETopic\u003C\u002Fth\u003E\n\u003C\u002Ftr\u003E\n\u003C\u002Fthead\u003E\n\u003Ctbody\u003E\n\u003Ctr\u003E\n\u003Ctd\u003E\u003Cstrong\u003E\u003Ca href=\"https:\u002F\u002Fvimeo.com\u002F123108417\" target=\"_blank\" rel=\"noopener noreferrer\"\u003ENGS101-8 Mapping to YOUR Reference\u003C\u002Fa\u003E\u003C\u002Fstrong\u003E\u003C\u002Ftd\u003E\n\u003Ctd\u003Edemonstrates \u003Cstrong\u003EBWA-MEM\u003C\u002Fstrong\u003E tool form options using a custom genome from the history\u003C\u002Ftd\u003E\n\u003C\u002Ftr\u003E\n\u003Ctr\u003E\n\u003Ctd\u003E\u003Cstrong\u003E\u003Ca href=\"https:\u002F\u002Fvimeo.com\u002F75918922\" target=\"_blank\" rel=\"noopener noreferrer\"\u003ECustom Genomes\u003C\u002Fa\u003E\u003C\u002Fstrong\u003E\u003C\u002Ftd\u003E\n\u003Ctd\u003Eexplains format, usage, and how to load a genome that works with tools\u003C\u002Ftd\u003E\n\u003C\u002Ftr\u003E\n\u003C\u002Ftbody\u003E\n\u003C\u002Ftable\u003E\n\u003Ch1 id=\"sources\"\u003E\u003Ca href=\"#sources\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003ESources\u003C\u002Fh1\u003E\n\u003Cul\u003E\n\u003Cli\u003EUCSC, Ensembl, NCBI\u002FGenBank\u003C\u002Fli\u003E\n\u003Cli\u003EOther Research project associated with specific genome projects\u003C\u002Fli\u003E\n\u003Cli\u003EInternal research projects\u003C\u002Fli\u003E\n\u003Cli\u003ESelected genomes can be found in \"Data Libraries\" on \u003Ca href=\"\u002Fmain\u002F\"\u003EMain\u003C\u002Fa\u003E for use at \u003Ca href=\"http:\u002F\u002Fusegalaxy.org\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttp:\u002F\u002Fusegalaxy.org\u003C\u002Fa\u003E. Example: \u003Cstrong\u003Ehg19\u003C\u002Fstrong\u003E is available for GATK under that sub-directory.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch1 id=\"format\"\u003E\u003Ca href=\"#format\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EFormat\u003C\u002Fh1\u003E\n\u003Cul\u003E\n\u003Cli\u003ECustom Genomes are required to be in \u003Ca href=\"\u002Flearn\u002Fdatatypes\u002F#fasta\"\u003EFASTA\u003C\u002Fa\u003E format\u003C\u002Fli\u003E\n\u003Cli\u003EThe data should be formatted as \u003Ca href=\"\u002Flearn\u002Fdatatypes\u002F#fasta\"\u003EFASTA\u003C\u002Fa\u003E prior to upload into Galaxy\u003C\u002Fli\u003E\n\u003Cli\u003EThe dataset will need to be labeled as \u003Ca href=\"\u002Flearn\u002Fdatatypes\u002F#fasta\"\u003EFASTA\u003C\u002Fa\u003E after loaded (if not automatically assigned)\u003C\u002Fli\u003E\n\u003Cli\u003EDid you use \u003Cstrong\u003ENormalizeFasta\u003C\u002Fstrong\u003E with the options to wrap sequence lines at 80 bases and to trim the title line at the first whitespace?\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch1 id=\"custom-builds\"\u003E\u003Ca href=\"#custom-builds\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003ECustom Builds\u003C\u002Fh1\u003E\n\u003Cp\u003ESome tools and functions require that the \u003Ca href=\"\u002Fsupport\u002Fdatatypes-and-tools\u002F\"\u003E'database' attribute is assigned\u003C\u002Fa\u003E or that a Custom Reference Genome is set up as a \u003Cem\u003ECustom Build\u003C\u002Fem\u003E prior to use. Examples are the tools \u003Cstrong\u003EFeaturecounts\u003C\u002Fstrong\u003E, \u003Cstrong\u003EExtract Genomic DNA\u003C\u002Fstrong\u003E, certain \u003Cstrong\u003EPicard\u003C\u002Fstrong\u003E tools, and the functions under  \u003Cstrong\u003EVisualization\u003C\u002Fstrong\u003E.\u003C\u002Fp\u003E\n\u003Cp\u003EOnce created, a \u003Cem\u003ECustom Build\u003C\u002Fem\u003E is added to the list \u003Cstrong\u003EDatabase\u002FBuild:\u003C\u002Fstrong\u003E on the dataset 'Edit Attributes' and 'Upload File' tool forms and is available for 'Visualizations'. These can be assigned or used just like any other reference genome.\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003EStart with an existing fasta Custom Reference Genome in your history. It is \u003Cem\u003Every important\u003C\u002Fem\u003E make sure the \u003Ca href=\"\u002Flearn\u002Fcustom-genomes\u002F#format\"\u003Eformat\u003C\u002Fa\u003E is correct.\u003C\u002Fli\u003E\n\u003Cli\u003EGo to the top \"User\" menu and select \"Custom Builds\"\u003C\u002Fli\u003E\n\u003Cli\u003EEnter in the labels (no spaces and no special characters other than \"_\")\u003C\u002Fli\u003E\n\u003Cli\u003ESelect the fasta Custom Reference Genome\u003C\u002Fli\u003E\n\u003Cli\u003ESubmit and wait for the build to finish loading before assigning to a dataset or using to start a new Visualization\u003C\u002Fli\u003E\n\u003Cli\u003ENote: It is fine to navigate away from this form and come back to it later to check for status. The larger the fasta file and busier the Galaxy instance is, the longer the processing will take.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch1 id=\"sorting\"\u003E\u003Ca href=\"#sorting\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003ESorting\u003C\u002Fh1\u003E\n\u003Cp\u003ECertain tools expect that reference genomes are sorted in \u003Ca href=\"http:\u002F\u002Fen.wikipedia.org\u002Fwiki\u002FLexicographical_order\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Elexicographical order\u003C\u002Fa\u003E. These tools are often downstream of the initial mapping tools, which means that a large investment in a project has already been made (i.e. a long mapping process), before a problem with sorting pops up in conclusion layer tools. No one likes to start over!\u003C\u002Fp\u003E\n\u003Cp\u003EHow to avoid? Always sort your \u003Ca href=\"\u002Flearn\u002Fdatatypes\u002F#fasta\"\u003EFASTA\u003C\u002Fa\u003E reference genome dataset at the beginning of a project. Many sources only provide sorted genomes, but double checking is your own responsibility, and super easy in Galaxy. So easy that there isn't even a shared workflow, just a recipe (but feel free to make your own):\u003C\u002Fp\u003E\n\u003Cp\u003EQuick lexicographical sort recipe:\u003C\u002Fp\u003E\n\u003Cpre\u003E\u003Ccode\u003E1. Convert Formats -\u003E FASTA-to-Tabular\n2. Filter and Sort -\u003E Sort\n       on column: c1 \n       with flavor: Alphabetical\n       everything in: Ascending order\n3. Convert Formats -\u003E Tabular-to-FASTA\n\u003C\u002Fcode\u003E\u003C\u002Fpre\u003E\n\u003Cp\u003EThe above sorting method is for most tools, but not all. In particular, GATK tools have a tool-specific sort order requirement. The \u003Ca href=\"http:\u002F\u002Fwww.broadinstitute.org\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EBroad Institute\u003C\u002Fa\u003E FAQ with input format instructions is \u003Cstrong\u003E\u003Ca href=\"http:\u002F\u002Fwww.broadinstitute.org\u002Fgark\u002Fguide\u002Farticle?id=1204\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehere\u003C\u002Fa\u003E\u003C\u002Fstrong\u003E.\u003C\u002Fp\u003E\n\u003Ch1 id=\"troubleshooting\"\u003E\u003Ca href=\"#troubleshooting\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003ETroubleshooting\u003C\u002Fh1\u003E\n\u003Cp\u003EIf a custom genome dataset is producing errors, double check the format and that the chromosome identifiers between \u003Cstrong\u003EALL\u003C\u002Fstrong\u003E inputs. Clicking on the green bug icon \u003Cimg src=\"\u002Fimages\u002Ficons\u002Fbug.png\"\u003E will often provide a description of the problem. This does not automatically submit a bug report, and it is not always necessary to do so, but it is a good way to get more information about why a job is failing.\u003C\u002Fp\u003E\n\u003Ch3 id=\"1-custom-genome-not-assigned-as-a-hreflearndatatypesfastafastaa-format\"\u003E\u003Ca href=\"#1-custom-genome-not-assigned-as-a-hreflearndatatypesfastafastaa-format\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003E1. Custom genome not assigned as \u003Ca href=\"\u002Flearn\u002Fdatatypes\u002F#fasta\"\u003EFASTA\u003C\u002Fa\u003E format\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Cstrong\u003ESymptoms include\u003C\u002Fstrong\u003E: Dataset not included in custom genome pull down menu on tool forms.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003ESolution\u003C\u002Fstrong\u003E: Check datatype assigned to dataset and assign \u003Cstrong\u003Efasta\u003C\u002Fstrong\u003E format.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003EHow\u003C\u002Fstrong\u003E: Click on the dataset's pencil icon \u003Cimg src=\"\u002Fimages\u002Ficons\u002Fpencil.png\"\u003E to reach the \"Edit Attributes\" page, and in the \u003Ca href=\"\u002Flearn\u002Fdatatypes\u002F\"\u003Edatatypes\u003C\u002Fa\u003E section, type in \"fasta\", and save.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"2-incomplete-custom-genome-file-load\"\u003E\u003Ca href=\"#2-incomplete-custom-genome-file-load\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003E2. Incomplete Custom genome file load\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Cstrong\u003ESymptoms include\u003C\u002Fstrong\u003E: Tool errors result the first time you use the Custom genome.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003ESolution\u003C\u002Fstrong\u003E: Use \u003Cstrong\u003EText Manipulation → Select last lines from a dataset\u003C\u002Fstrong\u003E to check last 10 lines to see if file is truncated.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003EHow\u003C\u002Fstrong\u003E: Reload the dataset (switch to \u003Cstrong\u003E\u003Ca href=\"\u002Fftp-upload\u002F\"\u003EFTP\u003C\u002Fa\u003E\u003C\u002Fstrong\u003E if not using already). Check your FTP client logs to make sure the load is complete.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"3-extra-spaces-extra-lines-inconsistent-line-wrapping-or-any-deviation-from-strict-a-hreflearndatatypesfastafastaa-format\"\u003E\u003Ca href=\"#3-extra-spaces-extra-lines-inconsistent-line-wrapping-or-any-deviation-from-strict-a-hreflearndatatypesfastafastaa-format\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003E3. Extra spaces, extra lines, inconsistent line wrapping, or any deviation from strict \u003Ca href=\"\u002Flearn\u002Fdatatypes\u002F#fasta\"\u003EFASTA\u003C\u002Fa\u003E format\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Cstrong\u003ESymptoms include\u003C\u002Fstrong\u003E: RNA-seq tools (\u003Cstrong\u003ECufflinks, Cuffcompare, Cuffmerge, Cuffdiff\u003C\u002Fstrong\u003E) fails with error \u003Ccode\u003EError: sequence lines in a FASTA record must have the same length!\u003C\u002Fcode\u003E.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003ESolution\u003C\u002Fstrong\u003E: File tested and corrected locally then re-upload or test\u002Ffix within Galaxy, then re-run.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003EHow\u003C\u002Fstrong\u003E:\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003EQuick re-formatting\u003C\u002Fstrong\u003E Run the tool through the tool \u003Cstrong\u003ENormalizeFasta\u003C\u002Fstrong\u003E using the options to wrap sequence lines at 80 bases and to trim the title line at the first whitespace.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003EOptional Detailed re-formatting\u003C\u002Fstrong\u003E Start with \u003Cstrong\u003EFASTA manipulation → FASTA Width formatter \u003C\u002Fstrong\u003E with a value between 40-80 (60 is common) to reformat wrapping. Next, use \u003Cstrong\u003EFilter and Sort → Select\u003C\u002Fstrong\u003E with \"\u003E\" to examine identifiers. Use a combination of \u003Cstrong\u003EConvert Formats → FASTA-to-Tabular\u003C\u002Fstrong\u003E, \u003Cstrong\u003EText Manipulation\u003C\u002Fstrong\u003E tools, then \u003Cstrong\u003ETabular-to-FASTA\u003C\u002Fstrong\u003E to correct.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003EWith either of the above\u003C\u002Fstrong\u003E, finish by using \u003Cstrong\u003EFilter and Sort → Select\u003C\u002Fstrong\u003E with \u003Ccode\u003E^\\w*$\u003C\u002Fcode\u003E to search for empty lines (use \"NOT matching\" to remove these lines and output a properly format fasta dataset).\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"4-inconsistent-line-wrapping-common-if-merging-chromosomes-from-various-genbank-records-eg-primary-chroms-with-mito\"\u003E\u003Ca href=\"#4-inconsistent-line-wrapping-common-if-merging-chromosomes-from-various-genbank-records-eg-primary-chroms-with-mito\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003E4. Inconsistent line wrapping, common if merging chromosomes from various Genbank records (e.g. primary chroms with mito)\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Cstrong\u003ESymptoms include\u003C\u002Fstrong\u003E: Tools (\u003Cstrong\u003ESAMTools\u003C\u002Fstrong\u003E, \u003Cstrong\u003EExtract Genomic DNA\u003C\u002Fstrong\u003E, but rarely alignment tools) may complain about unexpected line lengths\u002Fmissing identifiers. Or they may just fail for what appears to be a cluster error.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003ESolution\u003C\u002Fstrong\u003E: File tested and corrected locally then re-upload or \u003Cstrong\u003Etest\u002Ffix within Galaxy\u003C\u002Fstrong\u003E.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003EHow\u003C\u002Fstrong\u003E: Use \u003Cstrong\u003E\u003Cstrong\u003ENormalizeFasta\u003C\u002Fstrong\u003E\u003C\u002Fstrong\u003E using the options to wrap sequence lines at 80 bases and to trim the title line at the first whitespace. Finish by using \u003Cstrong\u003EFilter and Sort → Select\u003C\u002Fstrong\u003E with \u003Ccode\u003E^\\w*$\u003C\u002Fcode\u003E to search for empty lines (use \"NOT matching\" to remove these lines and output a properly format fasta dataset).\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"5-unsorted-fasta-genome-file\"\u003E\u003Ca href=\"#5-unsorted-fasta-genome-file\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003E5. Unsorted fasta genome file\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Cstrong\u003ESymptoms include\u003C\u002Fstrong\u003E: Tools such as \u003Cstrong\u003EExtract Genomic DNA\u003C\u002Fstrong\u003E report problems with sequence lengths.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003ESolution\u003C\u002Fstrong\u003E: First try sorting and re-formatting in Galaxy then re-run.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003EHow\u003C\u002Fstrong\u003E: To sort, follow instructions for \u003Ca href=\"\u002Flearn\u002Fcustom-genomes\u002F#sorting\"\u003ESorting\u003C\u002Fa\u003E a Custom Genome.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"6-identifier-and-description-in--title-lines-used-inconsistently-by-tools-in-the-same-analysis\"\u003E\u003Ca href=\"#6-identifier-and-description-in--title-lines-used-inconsistently-by-tools-in-the-same-analysis\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003E6. Identifier and Description in \"\u003E\" title lines used inconsistently by tools in the same analysis\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Cstrong\u003ESymptoms include\u003C\u002Fstrong\u003E: Will generally manifest as a false genome-mismatch problem.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003ESolution\u003C\u002Fstrong\u003E: Remove the description content and re-run all tools\u002Fworkflows that used this input. Mapping tools will usually not fail, but downstream tools will. When this comes up, it usually means that an analysis needs to be started over from the mapping step to correct the problems. No one enjoys redoing this work. Avoid the problems by formatting the genome, by double checking that the same reference genome was used for all steps, and by making certain the 'identifiers' are a match between all planned inputs (including reference annotation such as GTF data) \u003Cstrong\u003Ebefore using your custom genome\u003C\u002Fstrong\u003E.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003EHow\u003C\u002Fstrong\u003E: To drop the title line description content, use \u003Cstrong\u003ENormalizeFasta\u003C\u002Fstrong\u003E using the options to wrap sequence lines at 80 bases and to trim the title line at the first whitespace. Next, double check that the \u003Ca href=\"https:\u002F\u002Fgalaxyproject.org\u002Fsupport\u002Fchrom-identifiers\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Echromosome identifiers are an exact match between all inputs\u003C\u002Fa\u003E.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"7-unassigned-database\"\u003E\u003Ca href=\"#7-unassigned-database\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003E7. Unassigned database\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Cstrong\u003ESymptoms include\u003C\u002Fstrong\u003E: Tools report that no build is available for the assigned reference genome.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003ESolution\u003C\u002Fstrong\u003E This occurs with tools that require an assigned \u003Cem\u003Edatabase\u003C\u002Fem\u003E metadata attribute. \u003Cstrong\u003ESAMTools\u003C\u002Fstrong\u003E and \u003Cstrong\u003EPicard\u003C\u002Fstrong\u003E often require this assignment.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003EHow\u003C\u002Fstrong\u003E: Create a \u003Ca href=\"\u002Flearn\u002Fcustom-genomes\u002F#custom-builds\"\u003ECustom Build\u003C\u002Fa\u003E and assign it to the dataset\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch1 id=\"a-problem-or-not-a-problem\"\u003E\u003Ca href=\"#a-problem-or-not-a-problem\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EA problem or not a problem?\u003C\u002Fh1\u003E\n\u003Cp\u003ECertain job errors with RNA-seq tools can at first appear to look like a format problem with a custom reference genome, but are actually a bit more complicated...\u003C\u002Fp\u003E\n\u003Ch3 id=\"a-did-you-use-the-same-exact-reference-genome-for-all-steps-in-the-same-analysis-pipeline\"\u003E\u003Ca href=\"#a-did-you-use-the-same-exact-reference-genome-for-all-steps-in-the-same-analysis-pipeline\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EA. Did you use the same exact reference genome for all steps in the same analysis pipeline?\u003C\u002Fh3\u003E\n\u003Cp\u003EAlignment tools (BWA, Bowtie, Tophat) are generally tolerant of minor formatting problems with reference genomes. However, downstream tools tend to have more stringent format requirements. To avoid having to reprocess, \u003Cstrong\u003Ea best practice is to verify that the formatting is correct before any steps are started\u003C\u002Fstrong\u003E.\u003C\u002Fp\u003E\n\u003Ch3 id=\"b-any-tool-including-cufflinksmergediff-reports-a-missingproblem-transcriptsgtf-file\"\u003E\u003Ca href=\"#b-any-tool-including-cufflinksmergediff-reports-a-missingproblem-transcriptsgtf-file\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EB. Any tool, including Cufflinks\u002Fmerge\u002Fdiff, reports a missing\u002Fproblem transcripts.gtf file.\u003C\u002Fh3\u003E\n\u003Cp\u003EThis generally indicates a mismatch in the chromosome identifiers between the reference genome used for the mapping\u002Falignment step, and the reference annotation GTF data and\u002For the reference genome used in downstream steps.\u003C\u002Fp\u003E\n\u003Ch3 id=\"c-what-if-the-inputs-are-truly-not-a-match\"\u003E\u003Ca href=\"#c-what-if-the-inputs-are-truly-not-a-match\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EC. What if the inputs are truly not a match?\u003C\u002Fh3\u003E\n\u003Cp\u003EThe problem can sometimes be corrected by altering the chromosome identifiers in the GTF file or the reference genome. However, making changes can introduce scientific genome-mismatch problems. It is generally much better to find reference data (genomes, reference files) that are confirmed to be a match.\u003C\u002Fp\u003E\n\u003C\u002Fdiv\u003E\n"}},"context":{}};(function(){var s;(s=document.currentScript||document.scripts[document.scripts.length-1]).parentNode.removeChild(s);}());</script><script src="/assets/js/app.4169b14c.js" defer></script><script src="/assets/js/page--src--templates--article-vue.7f90f7d3.js" defer></script><script src="/assets/js/page--src--templates--article-vue~page--src--templates--vue-article-vue.c289395b.js" defer></script>
  </body>
</html>
