<!DOCTYPE html>
<html data-html-server-rendered="true" lang="en" data-vue-tag="%7B%22lang%22:%7B%22ssr%22:%22en%22%7D%7D">
  <head>
    <title>Galaxy Platform Directory: Servers, Clouds, and Deployable Resources - Galaxy Community Hub</title><meta name="gridsome:hash" content="c32ecfdb33b075b0057598b5b9118f3554a398cf"><meta data-vue-tag="ssr" charset="utf-8"><meta data-vue-tag="ssr" name="generator" content="Gridsome v0.7.23"><meta data-vue-tag="ssr" data-key="viewport" name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover"><meta data-vue-tag="ssr" data-key="format-detection" name="format-detection" content="telephone=no"><meta data-vue-tag="ssr" data-key="description" name="description" content="All about Galaxy and its community."><link data-vue-tag="ssr" rel="icon" href="data:,"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="16x16" href="/assets/static/favicon.ce0531f.7d4a227415c9d9b704d56a079d49a3e1.png"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="32x32" href="/assets/static/favicon.ac8d93a.7d4a227415c9d9b704d56a079d49a3e1.png"><link rel="preload" href="/assets/css/0.styles.28333459.css" as="style"><link rel="preload" href="/assets/js/app.f7f9943a.js" as="script"><link rel="preload" href="/assets/js/page--src--pages--use-vue.8c7d1a05.js" as="script"><link rel="prefetch" href="/assets/js/page--node-modules--gridsome--app--pages--404-vue.54a07ec7.js"><link rel="prefetch" href="/assets/js/page--src--pages--blog-vue.72cfdf23.js"><link rel="prefetch" href="/assets/js/page--src--pages--careers-vue.51f7f627.js"><link rel="prefetch" href="/assets/js/page--src--pages--data-explorer-vue.cda0b563.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--archive-vue.54bd407e.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests--papercuts-vue.d73bf9af.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests-vue.f9ca327e.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--webinars-vue.2b0c4b1f.js"><link rel="prefetch" href="/assets/js/page--src--pages--events-vue.2bc0181e.js"><link rel="prefetch" href="/assets/js/page--src--pages--index-vue.9e6f30c2.js"><link rel="prefetch" href="/assets/js/page--src--pages--news-vue.c55be461.js"><link rel="prefetch" href="/assets/js/page--src--pages--tests-vue.4730afa8.js"><link rel="prefetch" href="/assets/js/page--src--templates--article-vue.9c961af0.js"><link rel="prefetch" href="/assets/js/page--src--templates--platform-vue.672f4f9e.js"><link rel="prefetch" href="/assets/js/page--src--templates--vue-article-vue.d03e3b77.js"><link rel="prefetch" href="/assets/js/vendors~page--src--pages--careers-vue~page--src--pages--use-vue~page--src--templates--article-vue~pa~efade918.71d44943.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--index-md.b120794f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--md--imgtests--imgdir--index-md.8a720dc7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--partials--index-md.88a2cc0b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--imgdir--index-md.a7da7426.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--index-md.ab7ce26b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--index-md.ac0f0339.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--index-md.7388aadd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--reldir--index-md.78e6d088.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--capacity-planning--index-md.532d558e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--index-md.f596bca3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--performance--cluster--legacy--index-md.9900d899.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--datatypes--index-md.37a06167.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--disk-quotas--index-md.99623880.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--faq--index-md.dfcae1f0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--data-model--index-md.f6811d25.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--galaxy-ops--index-md.da494b7d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--ten-simple-steps-galaxy-as-a-service--index-md.f0b31fa8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--tool-config-syntax--index-md.c133b049.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--writing-tests--index-md.d3ed2aee.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--allocation--index-md.88a6f777.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--faq--index-md.6b5ae7d2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--index-md.54b831ad.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--ssh--index-md.c431c940.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--troubleshooting--index-md.2bbf34e9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--capacity-planning--index-md.8d2c9c03.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--getting-started--index-md.63fe5edc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--index-md.1387a7bc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--building--index-md.ea5bc352.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--capacity-planning--index-md.bdfd10d5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--cluster-types--index-md.83943371.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--customizing--index-md.6b8b3678.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--faq--index-md.12c74d13.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--index-md.e3fa7dae.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--pre-201703--index-md.9b084a41.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--hadoop--index-md.95f1458a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--ht-condor--index-md.f973feff.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--index-md.37f11743.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--openstack--index-md.f4d0e62a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--services--index-md.e26413ae.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--sharing--index-md.c15fa6cb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--troubleshooting--index-md.953f49c4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--userdata--index-md.1116e088.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--future--index-md.ec7dff38.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--index-md.f0bd7fb5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-09--index-md.ed1d541b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-27--index-md.5a2cbe0c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-09-19--index-md.36fcc10e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-11-14--index-md.5155dfcc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-01-16--index-md.e740402b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-03-20--index-md.fa75b1de.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-05-15--index-md.ad74e436.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-11-20--index-md.6a1074f1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-02-19--index-md.bdefdea8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-04-16--index-md.ce613da7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-06-18--index-md.9253a34b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-08-20--index-md.8ac977c4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-10-15--index-md.3a558b10.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-02-18--index-md.ae82f681.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-04-21--index-md.4cd1f8fc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-06-16--index-md.7b1572c5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-08-18--index-md.5b3383d1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--index-md.da754856.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--webinar-tech--index-md.c5c416a6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2012--index-md.4eddb972.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2014--index-md.5d7b4578.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--index-md.afac4bb5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--data--index-md.f2ba0502.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--datafiles--index-md.4dfd4db7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--architecture--index-md.8035154b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--bitbucket--index-md.7cc7d8d1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--css--index-md.710209d9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--getting-started--index-md.930b3a22.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--index-md.1cbec221.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--jsa--index-md.134e6a35.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--mercurial--index-md.9026901a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--security-tool-tips--index-md.2d704e67.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--galaxy-project--index-md.230e8088.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--index-md.e5883590.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--papers--index-md.db9c7f1b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--posters--index-md.04432e3d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--presentations--index-md.92923a05.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--videos--index-md.f680b009.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--advanced-session--index-md.4227a5ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--basics-session--index-md.08c5b284.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--index-md.9844ef78.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--logistics--index-md.6f6b160a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--publicity--index-md.dfbc1a41.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--registration--index-md.5232161d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--abstracts--index-md.697fe169.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--index-md.ca51927e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--key-dates--index-md.606cfcac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--lightning-talks--index-md.6a60add5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--logistics--index-md.06ca864d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--organizing-committee--index-md.0500ed90.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--photos--index-md.ab68ac2c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--automation-strategies--index-md.2d8a9043.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--bioinformatics-training--index-md.43a55fdc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--end-users--index-md.5f46cd2a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--workflows-and-api--index-md.07da71b4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--index-md.cb8d0657.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--promotion--index-md.4eb64cde.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--register--index-md.49c36527.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--sponsorships--index-md.db0f3791.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--training-day--index-md.c8f12515.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--index-md.3c67479f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--posters--index-md.728aa734.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--talks--index-md.7f93c9db.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--clinical-use-hpc-file-systems--index-md.030a88bd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-admins--index-md.f12f7b4d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-blast--index-md.59e8820f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-professionals--index-md.dcb27276.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--index-md.cfaaca20.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--proteomics--index-md.abba045c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--public-galaxy-servers--index-md.bd34ea15.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--tool-developers--index-md.a5a0a8b6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--index-md.df6174ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--key-dates--index-md.f7630162.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--lightning--index-md.0dfb5a95.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--logistics--index-md.4759076c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--organizers--index-md.6fc11b96.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--photos--index-md.aef50972.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--program--index-md.05c5d87a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--promotion--index-md.e34ad376.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--register--index-md.8ea1785f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--sponsorships--index-md.3cde86d8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--advance-tool-data--index-md.2210b6c4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--api--index-md.57c06e35.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--index-md.4cd32016.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--intro-tool-data--index-md.75a7abbd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--vms--index-md.1c86a58b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--index-md.9abee1c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--posters--index-md.810cd33b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--talks--index-md.c9751906.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--branch-release-merge--index-md.f6ed714e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-admins--index-md.5f9fcb01.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-end-users--index-md.00b4cd3b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-training-network--index-md.a843f5f7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--heterogeneous--index-md.8313575f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--index-md.f878a9f6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--using-galaxy-ui-overview--index-md.5b284474.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--hackathon--index-md.20172bcf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--index-md.9ea6d61e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--key-dates--index-md.bac2859e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--lightning--index-md.96c8f509.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--logistics--index-md.c1318807.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--organizers--index-md.b183376c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--photos--index-md.e0b61b7a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--program--index-md.a93fc58b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--promotion--index-md.df1d2b77.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--register--index-md.54adaeb3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--sponsor-exhibit--index-md.64ef93be.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--admin-walkthrough--index-md.c82adb0e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--api--index-md.a936c5d1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--data-managers--index-md.348024f8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--index-md.030f1ebd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--vms--index-md.507d7d15.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--cwl--index-md.a8a9aa76.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-management-challenges--index-md.409c72aa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-splitting-and-parallelism--index-md.6332a3a3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--elixir--index-md.982da27f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--extended-workflow--index-md.f2b42ad0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--fots--index-md.52645a6e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-admins--index-md.ffda743c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-community-uk--index-md.060ecf60.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-scientists--index-md.ae9e7e59.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-training-network--index-md.a412fdd5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--index-md.22f27d14.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--data-hack-roundup--index-md.60bc6484.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--header--index-md.dbba4e7d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--index-md.2f85a581.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2016--bofs--index-md.5a3032e2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--abstracts--index-md.57e795e8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--childcare--index-md.096b34d2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--cofest--index-md.8b0a9b93.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--conduct--index-md.8e51f30c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--faq--index-md.0f0f1b46.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--index-md.641b44a6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--organizers--index-md.787e0ccf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--promotion--index-md.e6e58e52.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--registration--index-md.52f1b4f7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--sponsors--index-md.ef140359.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--training--index-md.ee0d3a7c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--venue--index-md.30256085.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--abstracts--index-md.137e7bc6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--childcare--index-md.84f642d3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--code-of-conduct--index-md.2789988f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--collaboration--index-md.a48ddf3b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--faq--index-md.4fd33733.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--icebreaker--index-md.ce20a106.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--index-md.9c798d19.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--keynotes--index-md.080e361c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--organizers--index-md.a3b956fb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--promotion--index-md.de985f84.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--register--index-md.05d6bc90.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--sponsors--index-md.4c398a30.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--training--index-md.ce0ef35c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--venue--index-md.4a3ddfec.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2014--index-md.d56576b3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2016--index-md.9f18f6ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2014--index-md.6fd15769.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2016--index-md.c0b34aae.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--index-md.ba91d0c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--trainingday--index-md.e89c50c3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2016--index-md.6afda26b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--example--vue-remark--index-md.c96a8cd8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--faqs--index-md.3bba7da9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-06--index-md.a5232112.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-07--index-md.19df6105.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--icons--index-md.ff92f6d4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--index-md.246f22cd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--logos--index-md.5a8a4cdf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--news-graphics--index-md.844f04b7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--photos--index-md.7a556820.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--screenshots--index-md.290a5433.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--index-md.467e2afe.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--workflow-editor-unannotated--index-md.bc93cd9f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--extract--index-md.940bc220.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--tool-panel--index-md.7ac3e984.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--variables--index-md.a3612f7c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--faq--index-md.599e559f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--galaxy-pages--index-md.51df5ee9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--interval-operations--index-md.71022235.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--main--faq--index-md.f6a425fa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--tools--requirements--index-md.1aef8fd3.js"><link rel="stylesheet" href="/assets/css/0.styles.28333459.css"><noscript data-vue-tag="ssr"><style>.g-image--loading{display:none;}</style></noscript>
  </head>
  <body >
    <div data-server-rendered="true" id="app" class="layout"><header id="masthead"><nav class="container navbar navbar-expand-lg navbar-dark justify-content-center"><a href="/" class="navbar-brand active"><img id="masthead-logo" src="/images/galaxy_logo_hub_white.svg" alt="Galaxy Community Hub" height="30"></a><div id="navbarToggleWrapper" class="collapse navbar-collapse mx-auto"><ul class="navbar-nav"><li class="nav-item"><a href="/use/" aria-current="page" role="button" aria-expanded="false" class="nav-link dropdown active--exact active">
          Use
        </a></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" aria-expanded="false" class="nav-link dropdown-toggle">
          Learn
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/learn/" class="dropdown-item">How to use Galaxy</a></li><li><a href="https://training.galaxyproject.org/" class="dropdown-item">Teach with Galaxy</a></li></ul></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" class="nav-link dropdown-toggle">
          Community
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/community/" class="dropdown-item">About</a></li><li><a href="/events/" class="dropdown-item">Events</a></li><li><a href="/news/" class="dropdown-item">News</a></li><li><a href="/blog/" class="dropdown-item">Blog</a></li><li><a href="/publication-library/" class="dropdown-item">Publications</a></li><li><a href="/galaxy-project/statistics/" class="dropdown-item">Stats</a></li><li><a href="/careers/" class="dropdown-item">Careers</a></li></ul></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" class="nav-link dropdown-toggle">
          Deploy &amp; Develop
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/admin/get-galaxy/" class="dropdown-item">Install Galaxy</a></li><li><a href="/admin/" class="dropdown-item">Server administration</a></li><li><a href="/tools/" class="dropdown-item">Tools</a></li><li><a href="/develop/api/" class="dropdown-item">Working with the API</a></li><li><a href="/develop/" class="dropdown-item">Contributing</a></li><li><a href="/docs/" class="dropdown-item">Code documentation</a></li></ul></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" class="nav-link dropdown-toggle">
          Support
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/support/" class="dropdown-item">FAQ</a></li><li><a href="https://help.galaxyproject.org/" class="dropdown-item">Galaxy Help Forum</a></li></ul></li><li class="nav-item"><a href="/jxtx/" role="button" aria-expanded="false" class="nav-link dropdown">
          @jxtx
        </a></li></ul></div></nav></header><section id="maincontainer" class="container"><h1 class="page-title">Galaxy Platform Directory: Servers, Clouds, and Deployable Resources</h1><div class="markdown"><img class="float-right" style="max-width: 35%;" src="/src/use/use-resource-banner.png" class="img-fluid" alt="125+ platforms to use Galaxy on">
<p>This directory lists platforms where you can use or deploy your own Galaxy Server with minimal effort.</p>
<ul>
<li><em>Almost all of them are free to use (Commercial clouds are the exception)</em></li>
<li><em>Almost all of them are open to everyone (Academic clouds are the exception).</em></li>
<li><em>All of the resources can be used, either immediately, or shortly after getting an account.</em></li>
</ul>
<p>These resources cover a wide spectrum of domains all across life sciences. There are resources for genomics (lots of them), metagenomics, transcriptomics, proteomics, drug discovery and even some outside biology like natural language processing (a couple of them), and social science.</p>
<!---
The rest of this page (the actual directories) are generated by
layouts/use_index.pug
--->
</div><table class="table table-striped"><thead><tr><th>Resource</th><th>Server</th><th>Summary</th></tr></thead><tbody><tr><td class="title"><a href="/use/ab-openlab/">AB-OpenLab</a></td><td class="server"><a href="http://ab-openlab.csir.res.in/frog/">Server</a></td><td class="summary markdown">FROG stands for <strong>F</strong>inge<strong>R</strong>printing <strong>O</strong>ntology of <strong>G</strong>enomic variations. FROG fingerprints have been devised to capture genomic variations at various levels.</td></tr><tr><td class="title"><a href="/use/abims/">ABiMS</a></td><td class="server"><a href="http://abims.sb-roscoff.fr">Server</a></td><td class="summary markdown">Analysis and bioinformatics for Marine Science.</td></tr><tr><td class="title"><a href="/use/ageseq-aspendb/">AGEseq @ AspenDB</a></td><td class="server"><a href="http://aspendb.uga.edu:8085/">Server</a></td><td class="summary markdown">This site provides a user-friendly interface for AGEseq (Analysis of Genome Editing by Sequencing) in a Galaxy instance.</td></tr><tr><td class="title"><a href="/use/anastasia/">ANASTASIA</a></td><td class="server"><a href="http://motherbox.chemeng.ntua.gr/anastasia_dev/">Server</a></td><td class="summary markdown"><strong>A</strong>utomated <strong>N</strong>ucleotide <strong>A</strong>minoacid <strong>S</strong>equences <strong>T</strong>ranslational pl<strong>A</strong>tform for <strong>S</strong>ystemic <strong>I</strong>nterpretation and <strong>A</strong>nalysis</td></tr><tr><td class="title"><a href="/use/apostl/">APOSTL</a></td><td class="server"><a href="http://apostl.moffitt.org/">Server</a></td><td class="summary markdown">Automated Processing of SAINT Templated Layouts (APOSTL) is a freely available software suite and analysis pipeline for reproducible, interactive analysis of AP-MS data.</td></tr><tr><td class="title"><a href="/use/arga/">ARGA</a></td><td class="server"><a href="http://mem.rcees.ac.cn:8083/">Server</a></td><td class="summary markdown">Antibiotic Resistance Gene Analyzer</td></tr><tr><td class="title"><a href="/use/args-oap/">ARGs-OAP</a></td><td class="server"><a href="http://smile.hku.hk/SARGs">Server</a></td><td class="summary markdown">Fast annotation and classification of antibiotic resistance gene-like sequences from metagenomic data.</td></tr><tr><td class="title"><a href="/use/ariaweb/">ARIAweb</a></td><td class="server"><a href="https://ariaweb.pasteur.fr/">Server</a></td><td class="summary markdown">Automated NOE assignment and NMR structure calculation</td></tr><tr><td class="title"><a href="/use/aries/">ARIES</a></td><td class="server"><a href="https://w3.iss.it/site/aries/">Server</a></td><td class="summary markdown">Analysis of genomic data in the field of public health and food safety, with the aim of deploying a comprehensive bioinformatics approach to the study of food-borne zoonoses and infectious diseases at the human and animal interface.</td></tr><tr><td class="title"><a href="/use/asaim/">ASaiM</a></td><td class="server"><a href="https://github.com/ASaiM/framework">Server</a></td><td class="summary markdown">Integrates tools specifically chosen for metagenomic and metatranscriptomic studies and hierarchically organized to orient user choice toward the best tool for a given task.</td></tr><tr><td class="title"><a href="/use/atacgraph/">ATACgraph</a></td><td class="server"><a href="https://github.com/RitataLU/ATACgraph">Server</a></td><td class="summary markdown">ATAC-seq for biologists with minimal bioinformatics knowledge to analyze chromatin accessibility</td></tr><tr><td class="title"><a href="/use/aws/">Amazon Web Services (AWS)</a></td><td class="server"><a href="https://aws.amazon.com/">Server</a></td><td class="summary markdown">Set up you own general purpose genomics server runing AWS using GVL and CloudMan.</td></tr><tr><td class="title"><a href="/use/anvil/">AnVIL</a></td><td class="server"><a href="https://anvilproject.org/">Server</a></td><td class="summary markdown">Run Galaxy inside the secure AnVIL environment on Google Cloud Platform.</td></tr><tr><td class="title"><a href="/use/bf2i-map/">BF2I-MAP</a></td><td class="server"><a href="http://bf2i-galaxy.insa-lyon.fr:8080/">Server</a></td><td class="summary markdown">Locally developed as well as standard tools are available on a 16-core server with 32GB memory, accessible to all users, and located in the BF2I lab.</td></tr><tr><td class="title"><a href="/use/bipaa/">BIPAA (BioInformatics Platform for Agroecosystem Arthropods)</a></td><td class="server"><a href="https://bipaa.genouest.org/">Server</a></td><td class="summary markdown">Insect genomics (aphids, parasito√Ød wasps, lepidopterans)</td></tr><tr><td class="title"><a href="/use/bridge/">BRIDGE</a></td><td class="server"><a href="https://github.com/scientificomputing/BRIDGE">Server</a></td><td class="summary markdown">Prepare, simulate and analyse and visualise molecular simulations, especially molecular dynamics and free energies.</td></tr><tr><td class="title"><a href="/use/biobix/">BioBix</a></td><td class="server"><a href="http://galaxy.ugent.be/">Server</a></td><td class="summary markdown">A general genomics server. Includes the PROTEOFORMER pipeline tools.</td></tr><tr><td class="title"><a href="/use/biociphers-lab-galaxy/">BioCiphers Lab Galaxy</a></td><td class="server"><a href="http://biociphers.org/">Server</a></td><td class="summary markdown">BCL Galaxy gives a user friendly interface for analysis tools developed by the <a href="http://biociphers.org/">BioCiphers Lab</a> at the University of Pennsylvania.</td></tr><tr><td class="title"><a href="/use/biocompute-object/">BioCompute Object Galaxy</a></td><td class="server"><a href="https://galaxy.aws.biochemistry.gwu.edu/">Server</a></td><td class="summary markdown"><a href="https://biocomputeobject.org/about.html">BioCompute Objects</a>  is a standard for communication of High-throughput Sequencing (HTS) analysis results, data set creation and curation, and bioinformatics verification protocols.</td></tr><tr><td class="title"><a href="/use/biodivine/">BioDivine Toolset</a></td><td class="server"><a href="https://sybila.fi.muni.cz/tools.html">Server</a></td><td class="summary markdown">Systems biology tools implemented by the Sybila team.</td></tr><tr><td class="title"><a href="/use/biomas/">BioMaS</a></td><td class="server"><a href="http://galaxy.cloud.ba.infn.it:8080/">Server</a></td><td class="summary markdown">Taxonomic studies of environmental microbial communities</td></tr><tr><td class="title"><a href="/use/biomina/">Biomina</a></td><td class="server"><a href="http://143.169.238.104/galaxy/">Server</a></td><td class="summary markdown">A general purpose Galaxy instance that includes most standard tools for DNA/RNA sequencing, plus extra tools for panel resequencing, variant annotation and some tools for Illumina SNParray analysis.</td></tr><tr><td class="title"><a href="/use/bitlab/">BitLAB</a></td><td class="server"><a href="https://pistacho.ac.uma.es/galaxy-bitlab">Server</a></td><td class="summary markdown">Detailed analysis of metagenomes.</td></tr><tr><td class="title"><a href="/use/cafu/">CAFU</a></td><td class="server"><a href="https://github.com/cma2015/CAFU">Server</a></td><td class="summary markdown">A Galaxy framework for exploring unmapped RNA-Seq data</td></tr><tr><td class="title"><a href="/use/cbib-galaxy/">CBiB Galaxy</a></td><td class="server"><a href="http://services.cbib.u-bordeaux2.fr/galaxy/">Server</a></td><td class="summary markdown">A general purpose Galaxy instance that includes EMBOSS (a software analysis package for molecular biology) and fibronectin (diversity analysis of synthetic libraries of a Fibronectin domain).</td></tr><tr><td class="title"><a href="/use/cirm-cfbp/">CIRM-CFBP</a></td><td class="server"><a href="https://iris.angers.inra.fr/galaxypub-cfbp">Server</a></td><td class="summary markdown">Web based access to tools (<a href="https://sourcesup.renater.fr/wiki/skif/">SkIf</a>, <a href="https://sourcesup.renater.fr/wiki/ki-s/">KI-S</a>) of IRHS-EmerSys lab.</td></tr><tr><td class="title"><a href="/use/climb/">CLIMB</a></td><td class="server"><a href="http://www.climb.ac.uk/">Server</a></td><td class="summary markdown">The CLIMB project (Cloud Infrastructure for Microbial Bioinformatics) develops and deploys a world leading cyber-infrastructure for microbial bioinformatics; providing free cloud-based compute, storage, and analysis tools for academic microbiologists in the UK.</td></tr><tr><td class="title"><a href="/use/clip-explorer/">CLIP-Explorer</a></td><td class="server"><a href="https://clipseq.usegalaxy.eu/">Server</a></td><td class="summary markdown">The CLIP-Explorer is a webserver to process, analyse and visualise CLIP-Seq data.</td></tr><tr><td class="title"><a href="/use/covid-19/">COVID-19</a></td><td class="server"><a href="https://covid19.galaxyproject.org/">Server</a></td><td class="summary markdown">A companion to our study describing the analysis of early COVID-19 data</td></tr><tr><td class="title"><a href="/use/cravat-p/">CRAVAT-P</a></td><td class="server"><a href="https://jraysajulga.github.io/cravatp-galaxy-docker/">Server</a></td><td class="summary markdown">A Docker image containing a fully-operational Galaxy instance with pre-installed demonstration material for CRAVAT-P.</td></tr><tr><td class="title"><a href="/use/cell-organizer/">CellOrganizer for Galaxy</a></td><td class="server"><a href="http://www.cellorganizer.org/public-galaxy-server/">Server</a></td><td class="summary markdown">Create generative models of the spatial organization of cells from microscope images and automatically provide geometries for spatial simulations of cell processes and behaviors</td></tr><tr><td class="title"><a href="/use/center-for-phage-technology-cpt/">Center for Phage Technology (CPT)</a></td><td class="server"><a href="https://cpt.tamu.edu/galaxy-public/">Server</a></td><td class="summary markdown">Phage biology and automated annotation.</td></tr><tr><td class="title"><a href="/use/chemflow/">ChemFlow</a></td><td class="server"><a href="https://vm-chemflow-francegrille.eu/">Server</a></td><td class="summary markdown">Chemometrics without programming: infrared spectrometry on a Galaxy platform.</td></tr><tr><td class="title"><a href="/use/cart/">Chemical Annotation Retrieval Toolkit (CART)</a></td><td class="server"><a href="http://cart.embl.de/">Server</a></td><td class="summary markdown">Retrieves annotations on biological effects of chemicals and determines which ones are enriched.</td></tr><tr><td class="title"><a href="/use/chemicaltoolbox/">ChemicalToolbox</a></td><td class="server"><a href="https://cheminformatics.usegalaxy.eu/">Server</a></td><td class="summary markdown">For processing, analysing and visualising chemical data, and performing molecular simulations.</td></tr><tr><td class="title"><a href="/use/cistrome-analysis-pipeline/">Cistrome Analysis Pipeline</a></td><td class="server"><a href="http://cistrome.org/ap/root">Server</a></td><td class="summary markdown">ChIP-chip/seq and gene expression data</td></tr><tr><td class="title"><a href="/use/climate-science-workbench/">Climate Science Workbench</a></td><td class="server"><a href="https://climate.usegalaxy.eu/">Server</a></td><td class="summary markdown">A comprehensive set of analysis tools and consolidated workflows for climate analysis and visualisation</td></tr><tr><td class="title"><a href="/use/cossci/">CoSSci</a></td><td class="server"><a href="http://socscicompute.ss.uci.edu/">Server</a></td><td class="summary markdown">Tools for solving <a href="http://en.wikipedia.org/wiki/Galton&#x27;s_problem">Galton's problem</a> in Comparative Research and complex network problems in Social Science.</td></tr><tr><td class="title"><a href="/use/codon-harmonizer/">Codon Harmonizer</a></td><td class="server"><a href="http://codonharmonizer.systemsbiology.nl/">Server</a></td><td class="summary markdown">Easily generate codon harmonized variants of gene sequences for heterologous expression</td></tr><tr><td class="title"><a href="/use/coloc-stats/">Coloc-stats</a></td><td class="server"><a href="https://hyperbrowser.uio.no/coloc-stats">Server</a></td><td class="summary markdown">a unified web interface to perform co-localization analysis of genomic features</td></tr><tr><td class="title"><a href="/use/corgat/">CorGAT</a></td><td class="server"><a href="http://corgat.cloud.ba.infn.it/galaxy">Server</a></td><td class="summary markdown">Align complete assemblies of SARS-CoV-2 genomes wih the reference genomic sequence, to obtain a list of polymorphic positions and to annotate genetic variants.</td></tr><tr><td class="title"><a href="/use/coral-snp/">CoralSNP</a></td><td class="server"><a href="https://coralsnp.science.psu.edu/galaxy/">Server</a></td><td class="summary markdown">Standard Tools for Acroporid Genotyping (STAG)</td></tr><tr><td class="title"><a href="/use/cropgalaxy/">CropGalaxy</a></td><td class="server"><a href="http://cropgalaxy.excellenceinbreeding.org/">Server</a></td><td class="summary markdown">Hosts tools for the <a href="/src/use/ricegalaxy/index.md">RiceGalaxy</a>, <a href="http://gobiiproject.org/">GOBii project</a>, <a href="http://cegsb.icrisat.org/high-throughput-genotyping-project-htpg/">HTPG project</a> and the <a href="http://excellenceinbreeding.org/">Excellence in Breeding (EiB) platform</a>, including genomic selection, marker selection, GWAS, imputation, file conversion, and cluster analysis.</td></tr><tr><td class="title"><a href="/use/dintor/">Dintor</a></td><td class="server"><a href="http://dintor.eurac.edu/">Server</a></td><td class="summary markdown">GWA and NGS tools and modules for functional annotation of genes and gene products</td></tr><tr><td class="title"><a href="/use/ewas-galaxy/">EWAS-Galaxy</a></td><td class="server"><a href="https://github.com/kpbioteam/docker_ewas">Server</a></td><td class="summary markdown">Galaxy Docker image for population epigenetics data analysis</td></tr><tr><td class="title"><a href="/use/epigenomics/">Epigenomics Galaxy</a></td><td class="server"><a href="https://hub.docker.com/r/mpaya/epigenomics_galaxy">Server</a></td><td class="summary markdown">Galaxy Docker image for epigenomics analysis</td></tr><tr><td class="title"><a href="/use/erasmus-mc/">Erasmus MC</a></td><td class="server"><a href="https://bioinf-galaxian.erasmusmc.nl/galaxy/">Server</a></td><td class="summary markdown">General purpose genomics analysis, featuring many standard tools plus many additional tools.</td></tr><tr><td class="title"><a href="/use/freebioinfo/">FreeBioinfo</a></td><td class="server"><a href="http://www.freebioinfo.org/">Server</a></td><td class="summary markdown">Galaxy for metagenomic analysis.</td></tr><tr><td class="title"><a href="/use/g-onramp/">G-OnRamp</a></td><td class="server"><a href="http://gonramp.wustl.edu/?page_id=1031">Server</a></td><td class="summary markdown">G-OnRamp provides a user-friendly, web-based platform for collaborative, end-to-end annotation of eukaryotic genomes using UCSC Assembly Hubs and JBrowse/Apollo genome browsers with evidence tracks derived from sequence alignments, ab initio gene predictors, RNA-Seq data, and repeat finders.</td></tr><tr><td class="title"><a href="/use/gaslini/">GASLINI</a></td><td class="server"><a href="http://igg.cloud.ba.infn.it/galaxy">Server</a></td><td class="summary markdown">Supports PIPE-T, a tool for Reverse transcription quantitative real-time polymerase chain reaction (RT-qPCR) analysis.</td></tr><tr><td class="title"><a href="/use/gcac/">GCAC : Galaxy for Compound Activity Classification</a></td><td class="server"><a href="https://github.com/LynnLab-JNU/gcac-galaxy-tools">Server</a></td><td class="summary markdown">GCAC is a web enabled platform for predictive model building using R and caret package for virtual screening.</td></tr><tr><td class="title"><a href="/use/galaxeast/">GalaxEast</a></td><td class="server"><a href="http://www.galaxeast.fr">Server</a></td><td class="summary markdown">Integrative 'omics data analysis</td></tr><tr><td class="title"><a href="/use/galaxy-africa/">Galaxy Africa</a></td><td class="server"><a href="https://africa.usegalaxy.eu/">Server</a></td><td class="summary markdown">Devoted to assisting researchers on the African continent accelerate their genomics research and analyses.</td></tr><tr><td class="title"><a href="/use/blast/">Galaxy BLAST+</a></td><td class="server"><a href="https://github.com/bgruening/docker-galaxy-blast">Server</a></td><td class="summary markdown">A complete and production ready Galaxy instance with installed BLAST+ tools.</td></tr><tr><td class="title"><a href="/use/galaxy-docker/">Galaxy Docker</a></td><td class="server"><a href="https://github.com/bgruening/docker-galaxy-stable">Server</a></td><td class="summary markdown">The Galaxy Docker Image is an easy distributable full-fledged Galaxy installation, that can be used for testing, teaching and presenting new tools and features.</td></tr><tr><td class="title"><a href="/use/exome-seq/">Galaxy Exome-Seq</a></td><td class="server"><a href="https://github.com/bgruening/galaxy-exome-seq">Server</a></td><td class="summary markdown">Galaxy Docker repository for Exome sequencing data analysis</td></tr><tr><td class="title"><a href="/use/gga/">Galaxy Genome Annotation (GGA)</a></td><td class="server"><a href="https://github.com/galaxy-genome-annotation">Server</a></td><td class="summary markdown">Galaxy for genome annotation</td></tr><tr><td class="title"><a href="/use/imaging/">Galaxy Imaging</a></td><td class="server"><a href="https://github.com/bgruening/galaxy-imaging">Server</a></td><td class="summary markdown">Galaxy Docker repository for image analysis</td></tr><tr><td class="title"><a href="/use/galaxy-india/">Galaxy India</a></td><td class="server"><a href="https://india.usegalaxy.eu/">Server</a></td><td class="summary markdown">This is the Indian galaxy server, Welcome Home!</td></tr><tr><td class="title"><a href="/use/galaxy-integrated-omics-gio/">Galaxy Integrated Omics (GIO)</a></td><td class="server"><a href="http://gio.sbcs.qmul.ac.uk/">Server</a></td><td class="summary markdown">Proteomics Informed by Transcriptomics (PIT) methodology, and selection of surrogate peptides for targeted proteomics.</td></tr><tr><td class="title"><a href="/use/galaxy-test/">Galaxy Test</a></td><td class="server"><a href="https://test.galaxyproject.org/">Server</a></td><td class="summary markdown">Beta version of Galaxy Main</td></tr><tr><td class="title"><a href="/use/msi/">Galaxy Workbench for Mass Spectrometry Imaging</a></td><td class="server"><a href="https://github.com/foellmelanie/docker-galaxy-msi">Server</a></td><td class="summary markdown">Galaxy Docker image for mass spec imaging</td></tr><tr><td class="title"><a href="/use/basic-sequence/">Galaxy basic sequence manipulation</a></td><td class="server"><a href="https://hub.docker.com/r/bgruening/galaxy-sequence-tools/">Server</a></td><td class="summary markdown">Galaxy Docker repository for basic sequence data analysis.</td></tr><tr><td class="title"><a href="/use/ecology/">Galaxy for Ecology</a></td><td class="server"><a href="https://ecology.usegalaxy.eu/">Server</a></td><td class="summary markdown">A web platform to get, process, analyze and visualize ecological data</td></tr><tr><td class="title"><a href="/use/galaxy-p/">Galaxy-P</a></td><td class="server"><a href="http://galaxyp.org/">Server</a></td><td class="summary markdown">Galaxy-P is a multiple 'omics' data analysis platform with particular emphasis on mass spectrometry based proteomics. Galaxy-P is developed at the <a href="http://umn.edu/">University of Minnesota</a>, deployed at the <a href="http://msi.umn.edu/">Minnesota Supercomputing Institute</a>.</td></tr><tr><td class="title"><a href="/use/synbiocad/">Galaxy-SynBioCAD</a></td><td class="server"><a href="https://www.jfaulon.com/galaxy-synbiocad-portal/">Server</a></td><td class="summary markdown">Synthetic biology and metabolic engineering</td></tr><tr><td class="title"><a href="/use/galaxy-genouest/">Galaxy@GenOuest</a></td><td class="server"><a href="https://galaxy.genouest.org">Server</a></td><td class="summary markdown">A general purpose Galaxy server Includes tools developped by <a href="http://www.irisa.fr/dyliss/">Dyliss</a> and <a href="https://team.inria.fr/genscale/">GenScale</a> bioinformatics research teams in Rennes, France.</td></tr><tr><td class="title"><a href="/use/galaxy-prabi/">Galaxy@PRABI</a></td><td class="server"><a href="http://galaxy.prabi.fr/">Server</a></td><td class="summary markdown">Includes bioinformatics tools developed by the research teams working in the perimeter of the PRABI core facility, including <em>kissplice/kissDE, TETools, SEX-DETector,</em> and <em>priam</em>.</td></tr><tr><td class="title"><a href="/use/galaxy-pasteur/">Galaxy@Pasteur</a></td><td class="server"><a href="https://galaxy.pasteur.fr/">Server</a></td><td class="summary markdown">General purpose genomics analysis server.</td></tr><tr><td class="title"><a href="/use/galaxytrakr/">GalaxyTrakr</a></td><td class="server"><a href="https://galaxytrakr.org/">Server</a></td><td class="summary markdown">Supporting food-borne pathogen research</td></tr><tr><td class="title"><a href="/use/genap/">GenAP</a></td><td class="server"><a href="https://www.genap.ca/">Server</a></td><td class="summary markdown">All Canadian researchers are <a href="https://ccdb.computecanada.ca/account_application">eligible for a GenAP account</a>. International researchers that have a Canadian collaborator can also be sponsored and become eligible to a GenAP account as well.</td></tr><tr><td class="title"><a href="/use/genelab/">GeneLab</a></td><td class="server"><a href="https://genelab.nasa.gov/">Server</a></td><td class="summary markdown">NASA GeneLab is a comprehensive space-related omics database in which users can upload, download, share, store, and analyze spaceflight and corresponding model organism data.</td></tr><tr><td class="title"><a href="/use/genomic-hyperbrowser/">Genomic Hyperbrowser</a></td><td class="server"><a href="http://hyperbrowser.uio.no/hb/">Server</a></td><td class="summary markdown">Statistical methodology and computing power to handle a variety of biological inquires on genomic datasets</td></tr><tr><td class="title"><a href="/use/gigagalaxy/">GigaGalaxy</a></td><td class="server"><a href="http://gigagalaxy.net/">Server</a></td><td class="summary markdown">GigaGalaxy hosts tools and data analyses from papers published in <em><a href="http://www.gigasciencejournal.com/">GigaScience</a></em>.</td></tr><tr><td class="title"><a href="/use/globus-genomics/">Globus Genomics</a></td><td class="server"><a href="http://globusgenomics.org/">Server</a></td><td class="summary markdown">Globus Genomics provides hardened high-performance Galaxy servers running with Globus on AWS.</td></tr><tr><td class="title"><a href="/use/grappa/">GrAPPA</a></td><td class="server"><a href="https://grappa.eecs.utk.edu/">Server</a></td><td class="summary markdown">GrAPPA is a web-based interface constructed on the Galaxy framework for graph theoretical tools. It contains novel combinatorial methods integrated into a complete microarray analysis toolchain, from uploading raw high-throughput data to visualization of results.</td></tr><tr><td class="title"><a href="/use/graphclust/">GraphClust</a></td><td class="server"><a href="https://github.com/BackofenLab/GraphClust-2">Server</a></td><td class="summary markdown"><a href="https://github.com/BackofenLab/GraphClust-2">GraphClust2</a> is a workflow for scalable clustering of RNAs based on sequence and secondary structures feature. GraphClust2 is implemented within the Galaxy framework and consists a set of integrated Galaxy tools and flavors of the linear-time clustering workflow.</td></tr><tr><td class="title"><a href="/use/halogen-bonding/">Halogen Bonding</a></td><td class="server"><a href="http://halogenbonding.com/">Server</a></td><td class="summary markdown">Visualize halogen bond contacts in the protein binding site. Analyze a protein binding site for halogen bonding hotspots. Evaluate halogen contacts with the protein backbone using the scoring function XBScore.</td></tr><tr><td class="title"><a href="/use/hardwood-genomics/">Hardwood Genomics</a></td><td class="server"><a href="https://www.hardwoodgenomics.org/content/tripal-galaxy-workflows">Server</a></td><td class="summary markdown">A simplified, workflow-centic interface to Galaxy, focused on genomics data from hardwood trees.</td></tr><tr><td class="title"><a href="/use/hic-explorer/">HiCExplorer</a></td><td class="server"><a href="https://hicexplorer.usegalaxy.eu">Server</a></td><td class="summary markdown">Tools to process and visualize chromosome conformation.</td></tr><tr><td class="title"><a href="/use/human-cell-atlas/">Human Cell Atlas</a></td><td class="server"><a href="https://humancellatlas.usegalaxy.eu/">Server</a></td><td class="summary markdown">Tools and workflows for the analysis of Single Cell RNA-Seq data. It includes a module that connects to the Matrix Service API of the <a href="https://www.humancellatlas.org/">Human Cell Atlas</a>‚Äôs Data Coordination Platform that enables retrieval of gene expression matrices from any data sets in the Human Cell Atlas.</td></tr><tr><td class="title"><a href="/use/huttenhower-lab/">Huttenhower Lab</a></td><td class="server"><a href="http://huttenhower.sph.harvard.edu/galaxy/">Server</a></td><td class="summary markdown">metagenomic and functional genomic analyses, intended for research and academic use</td></tr><tr><td class="title"><a href="/use/hyphy/">HyPhy HIV NGS Tools</a></td><td class="server"><a href="https://galaxy.hyphy.org/">Server</a></td><td class="summary markdown">Tools for HIV sequencing analysis.</td></tr><tr><td class="title"><a href="/use/ibers-aberystwyth/">IBERS @ Aberystwyth</a></td><td class="server"><a href="http://share-galaxy.ibers.aber.ac.uk/">Server</a></td><td class="summary markdown">Software, tools and pages written by IBERS researchers for release to the wider community.</td></tr><tr><td class="title"><a href="/use/idenap/">IDENAP</a></td><td class="server"><a href="http://mem.rcees.ac.cn:8081/">Server</a></td><td class="summary markdown">Inter-Domain Ecological Network Analysis Pipeline</td></tr><tr><td class="title"><a href="/use/ipk-blast/">IPK Galaxy Blast Suite</a></td><td class="server"><a href="https://galaxy-web.ipk-gatersleben.de">Server</a></td><td class="summary markdown">Part of the IPK Crop Analysis Tools Suite (CATS). Offers access to genomic references published by IPK and collaborative projects</td></tr><tr><td class="title"><a href="/use/immport-galaxy/">ImmPort Galaxy</a></td><td class="server"><a href="https://www.immportgalaxy.org/">Server</a></td><td class="summary markdown">Flow Cytometry Analysis.</td></tr><tr><td class="title"><a href="/use/interactomix/">InteractoMIX</a></td><td class="server"><a href="http://galaxy.interactomix.com/">Server</a></td><td class="summary markdown">A one-stop resource offering a wide range of computational analyses from proteome-wide interactomes to structure of protein complexes</td></tr><tr><td class="title"><a href="/use/jetstream/">Jetstream</a></td><td class="server"><a href="http://jetstream-cloud.org//">Server</a></td><td class="summary markdown"><a href="http://jetstream-cloud.org/">Jetstream</a> is part of <a href="https://www.xsede.org/">XSEDE</a>, a "collection of integrated advanced digital resources and services" and is funded by NSF.</td></tr><tr><td class="title"><a href="/use/lapps-grid/">LAPPS Grid</a></td><td class="server"><a href="http://galaxy.lappsgrid.org/">Server</a></td><td class="summary markdown">An open, interoperable web service platform for natural language processing (NLP) research and development.</td></tr><tr><td class="title"><a href="/use/langille-lab/">Langille Lab</a></td><td class="server"><a href="http://galaxy.morganlangille.com/">Server</a></td><td class="summary markdown">Prediction of the unobserved character states in a community of organisms from phylogenetic information.</td></tr><tr><td class="title"><a href="/use/laniakea/">Laniakea</a></td><td class="server"><a href="https://laniakea-elixir-it.github.io/">Server</a></td><td class="summary markdown">Laniakea provides the possibility to automate the creation of Galaxy-based virtualized environments through an easy setup procedure, providing an on-demand workspace ready to be used by life scientists and bioinformaticians, with built-in storage encryption for user data.</td></tr><tr><td class="title"><a href="/use/lebanese-university/">Lebanese University Galaxy</a></td><td class="server"><a href="http://galaxy.ul.edu.lb/">Server</a></td><td class="summary markdown">Includes the SpCLUST tool set for fast and reliable clustering of potentially divergent biological sequences</td></tr><tr><td class="title"><a href="/use/lisis/">LiSIs</a></td><td class="server"><a href="http://lisis.cs.ucy.ac.cy/">Server</a></td><td class="summary markdown">A platform for Virtual Screening.</td></tr><tr><td class="title"><a href="/use/lifeportal-oslo/">Lifeportal</a></td><td class="server"><a href="https://lifeportal.uio.no">Server</a></td><td class="summary markdown">a Galaxy instance to satisfy the computational needs of¬†the Norwegian research community in life¬†sciences but also of any other user willing to use the service.</td></tr><tr><td class="title"><a href="/use/mississippi/">MISSISSIPPI</a></td><td class="server"><a href="http://mississippi.fr/">Server</a></td><td class="summary markdown">Support for analyses of RNA and small RNA sequencing datasets as well as for epigenetics or metagenomics studies.</td></tr><tr><td class="title"><a href="/use/marea4galaxy/">MaREA4Galaxy</a></td><td class="server"><a href="http://bimib.disco.unimib.it:5555/">Server</a></td><td class="summary markdown">Metabolic Reaction Enrichment Analysis and visualization of RNA-seq data</td></tr><tr><td class="title"><a href="/use/ml-workbench/">Machine Learning Workbench</a></td><td class="server"><a href="https://ml.usegalaxy.eu/">Server</a></td><td class="summary markdown">A comprehensive set of data preprocessing, machine learning, deep learning and visualisation tools, consolidated workflows for end-to-end machine learning analysis and training materials to showcase the usage of these tools.</td></tr><tr><td class="title"><a href="/use/majewski-lab/">Majewski Lab</a></td><td class="server"><a href="http://genomequebec.mcgill.ca/exomeai">Server</a></td><td class="summary markdown">Detection of recurrent Allelic Imbalance in tumors using whole Exome sequencing data.</td></tr><tr><td class="title"><a href="/use/mandoiu/">Mandoiu Lab</a></td><td class="server"><a href="https://neo.engr.uconn.edu/">Server</a></td><td class="summary markdown">A customized installation of the Galaxy framework deploying bioinformatic tools and pipelines developed in the <a href="http://dna.engr.uconn.edu/">Mandoiu Lab</a> at the <a href="http://www.uconn.edu/">University of Connecticut</a> for high-throughput sequencing and immunogenomics data analysis</td></tr><tr><td class="title"><a href="/use/martin-luther-u-halle-wittenberg/">Martin Luther U. Halle-Wittenberg</a></td><td class="server"><a href="http://galaxy.informatik.uni-halle.de">Server</a></td><td class="summary markdown">Provides implementations of <a href="http://www.jstacs.de/index.php/GeMoMa">GeMoMa</a>, InMoDe, CRISPRer, TALgetter, TALgetterLong, TALENoffer, and Dimont</td></tr><tr><td class="title"><a href="/use/metacentrum/">MetaCentrum</a></td><td class="server"><a href="https://galaxy.metacentrum.cz/">Server</a></td><td class="summary markdown">MetaCentrum regional virtual organization of the Czech National Grid Infrastructure (NGI) and is accessible to employees and students of research and academic organizations of the Czech Republic and for scientific purposes only.</td></tr><tr><td class="title"><a href="/use/metabolomics/">Metabolomics</a></td><td class="server"><a href="https://metabolomics.usegalaxy.eu/">Server</a></td><td class="summary markdown">A community driven metabolomics Galaxy service</td></tr><tr><td class="title"><a href="/use/metagenomics/">Metagenomics</a></td><td class="server"><a href="https://metagenomics.usegalaxy.eu/">Server</a></td><td class="summary markdown">a webserver to process, analyse and visualize Metagenomic and Microbiota data in general.</td></tr><tr><td class="title"><a href="/use/metaproteomics-gateway/">Metaproteomics Gateway</a></td><td class="server"><a href="http://z.umn.edu/metaproteomicsgateway">Server</a></td><td class="summary markdown">Metaproteomics gateway providing access to documentation and other instructional materials, and an opportunity for hands-on training using example datasets and optimized metaproteomics workflows</td></tr><tr><td class="title"><a href="/use/mimodd-nacreousmap/">MiModD NacreousMap</a></td><td class="server"><a href="http://mapping-by-sequencing.vm.uni-freiburg.de:8080/">Server</a></td><td class="summary markdown">NacreousMap mapping/plotting tool of MiModD for users of MiModD who do not want to install the required dependencies (R and rpy2) for graphical output from this tool on their local system.</td></tr><tr><td class="title"><a href="/use/migale/">Migale</a></td><td class="server"><a href="http://galaxy.migale.inra.fr/">Server</a></td><td class="summary markdown">A general purpose genomics server with an emphasis on metagenomics, including FROGS and several Migale developed tools.</td></tr><tr><td class="title"><a href="/use/mpdsdm/">Molecular Property Diagnostic Suite - Diabetes Mellitus Portal (MPDSDM)</a></td><td class="server"><a href="http://www.mpds-diabetes.in/">Server</a></td><td class="summary markdown">A disease-specific open source web portal for Diabetes Mellitus.</td></tr><tr><td class="title"><a href="/use/mpdstb/">Molecular Property Diagnostic Suite - Tuberculosis (MPDSTB)</a></td><td class="server"><a href="http://mpds.osdd.net/">Server</a></td><td class="summary markdown">A disease-specific web portal for Tuberculosis for drug discovery and development.</td></tr><tr><td class="title"><a href="/use/ng-tax/">NG-Tax</a></td><td class="server"><a href="http://wurssb.gitlab.io/ngtax/index.html">Server</a></td><td class="summary markdown">A Semantic Framework for High-throughput Marker Gene (Amplicon) Analysis</td></tr><tr><td class="title"><a href="/use/ngphylogeny/">NGPhylogeny.fr</a></td><td class="server"><a href="https://ngphylogeny.fr/">Server</a></td><td class="summary markdown">Robust phylogenetic analysis for everyone.</td></tr><tr><td class="title"><a href="/use/nanogalaxy/">NanoGalaxy</a></td><td class="server"><a href="https://nanopore.usegalaxy.eu/">Server</a></td><td class="summary markdown">a webserver to process, analyse and visualize Oxford Nanopore Technologies (ONT) data and similar long-reads technologies.</td></tr><tr><td class="title"><a href="/use/nectar/">Nectar Cloud</a></td><td class="server"><a href="https://nectar.org.au/research-cloud/">Server</a></td><td class="summary markdown">Set up a Galaxy server on Australia's Nectar research cloud.</td></tr><tr><td class="title"><a href="/use/network-analyst/">NetworkAnalyst</a></td><td class="server"><a href="https://www.networkanalyst.ca/">Server</a></td><td class="summary markdown">For mapping and quantification of raw RNA-seq data</td></tr><tr><td class="title"><a href="/use/osddlinux-livegalaxy/">OSDDlinux LiveGalaxy</a></td><td class="server"><a href="http://osddlinux.osdd.net:8001/">Server</a></td><td class="summary markdown">Drug discovery</td></tr><tr><td class="title"><a href="/use/oqtans/">Oqtans</a></td><td class="server"><a href="http://galaxy.inf.ethz.ch">Server</a></td><td class="summary markdown">Customized Galaxy, extended with machine learning based tools for sequence and tiling array data analysis.</td></tr><tr><td class="title"><a href="/use/orione/">Orione</a></td><td class="server"><a href="http://orione.crs4.it/">Server</a></td><td class="summary markdown">A Galaxy based web server for microbiology. <a href="http://orione.crs4.it/">Orione</a> includes all post mapping or assembling steps from scaffolding to complete annotation pipelines.</td></tr><tr><td class="title"><a href="/use/osiris/">Osiris</a></td><td class="server"><a href="http://galaxy-dev.cnsi.ucsb.edu/osiris/">Server</a></td><td class="summary markdown">Phylogenetics</td></tr><tr><td class="title"><a href="/use/pia/">PIA</a></td><td class="server"><a href="http://galaxy-dev.cnsi.ucsb.edu/pia/">Server</a></td><td class="summary markdown">Phylogenetics and gene interaction</td></tr><tr><td class="title"><a href="/use/palfinder/">Palfinder</a></td><td class="server"><a href="https://palfinder.ls.manchester.ac.uk/">Server</a></td><td class="summary markdown">Microsatellite development.</td></tr><tr><td class="title"><a href="/use/pepsimili/">PepSimili</a></td><td class="server"><a href="http://pepsimili.e-nios.com:8080/">Server</a></td><td class="summary markdown">Integrated workflow tool for mapping of massive peptide repertoires on whole proteomes and delivering streamlined, systems-level biological interpretation.</td></tr><tr><td class="title"><a href="/use/phagepromoter/">PhagePromotor</a></td><td class="server"><a href="https://galaxy.bio.di.uminho.pt/">Server</a></td><td class="summary markdown">Predict promoter sequences in phage genomes, using machine learning models.</td></tr><tr><td class="title"><a href="/use/phenomenal/">PhenoMeNal</a></td><td class="server"><a href="https://portal.phenomenal-h2020.eu/home">Server</a></td><td class="summary markdown">Large scale computing for medical informatics</td></tr><tr><td class="title"><a href="/use/pirate/">PiRATE: a Pipeline to Retrieve and Annotate Transposable Elements</a></td><td class="server"><a href="https://www.seanoe.org/data/00406/51795/">Server</a></td><td class="summary markdown">Detect, classify and annotate TEs of non-model organisms.</td></tr><tr><td class="title"><a href="/use/plants/">Plant Anaylsis Workbench</a></td><td class="server"><a href="https://plants.usegalaxy.eu/">Server</a></td><td class="summary markdown">A community driven plant data analsys Galaxy service</td></tr><tr><td class="title"><a href="/use/plantgenie/">PlantGenIE</a></td><td class="server"><a href="http://galaxy.plantgenie.org:8080/">Server</a></td><td class="summary markdown">Plant Genome Integrative Explorer Galaxy instance</td></tr><tr><td class="title"><a href="/use/prestige/">PreSTIGE</a></td><td class="server"><a href="http://prestige.case.edu/">Server</a></td><td class="summary markdown">Predicting Specific Tissue Interactions of Genes and Enhancers</td></tr><tr><td class="title"><a href="/use/proteore/">ProteoRE</a></td><td class="server"><a href="http://www.proteore.org/">Server</a></td><td class="summary markdown">ProteoRE (Proteomics Research Environment)</td></tr><tr><td class="title"><a href="/use/proteogenomics-gateway/">Proteogenomics Gateway</a></td><td class="server"><a href="http://z.umn.edu/proteogenomicsgateway">Server</a></td><td class="summary markdown">Provides access to documentation and other instructional materials, and an opportunity for hands-on training using example datasets and optimized proteogenomics workflows. The main goal of this server is to provide documentation to facilitate training and mastery of these software and workflows.</td></tr><tr><td class="title"><a href="/use/protologger/">Protologger</a></td><td class="server"><a href="http://protologger.de/">Server</a></td><td class="summary markdown">All-in-one genome description tool, aimed at simplifying gathering the data required for writing protologues. This includes providing taxonomic, functional and ecological insights.</td></tr><tr><td class="title"><a href="/use/quantp/">QuanTP</a></td><td class="server"><a href="https://z.umn.edu/quantpdocker">Server</a></td><td class="summary markdown">A Docker image containing a fully-operational Galaxy instance with pre-installed QuanTP tools.</td></tr><tr><td class="title"><a href="/use/rna-modification-calling/">RNA Modification Calling</a></td><td class="server"><a href="https://github.com/HelmGroup/Galaxy_modification_calling">Server</a></td><td class="summary markdown">Galaxy Docker image for RNA analysis and modification calling</td></tr><tr><td class="title"><a href="/use/rna-workbench/">RNA Workbench</a></td><td class="server"><a href="https://github.com/bgruening/galaxy-rna-workbench">Server</a></td><td class="summary markdown">more than 50 bioinformatics tools dedicated to different research areas of RNA biology, including RNA structure analysis, RNA alignment, RNA annotation, RNA-protein interaction, ribosome profiling, RNA-Seq analysis, and RNA target prediction.</td></tr><tr><td class="title"><a href="/use/repeatexplorer/">RepeatExplorer</a></td><td class="server"><a href="http://repeatexplorer.org/">Server</a></td><td class="summary markdown">Graph-based clustering and characterization of repetitive sequences, and detection of transposable element protein coding domains.</td></tr><tr><td class="title"><a href="/use/ribogalaxy/">RiboGalaxy</a></td><td class="server"><a href="http://ribogalaxy.ucc.ie/">Server</a></td><td class="summary markdown">Tools for the analysis and visualization of ribo-seq data obtained with the ribosome profiling technique.</td></tr><tr><td class="title"><a href="/use/rice-galaxy/">Rice Galaxy</a></td><td class="server"><a href="https://github.com/InternationalRiceResearchInstitute/RiceGalaxy">Server</a></td><td class="summary markdown">Tools for designing single-nucleotide polymorphism assays, analyzing genome-wide association studies, population diversity, rice‚àíbacterial pathogen diagnostics, and a suite of published genomic prediction methods.</td></tr><tr><td class="title"><a href="/use/ena-submission/">SARS-CoV-2 ENA Submission</a></td><td class="server"><a href="https://rdm.elixir-belgium.org/covid-19/sarscov2_submission.html">Server</a></td><td class="summary markdown">Galaxy Docker image for submitting SARS-CoV-2 data to ENA</td></tr><tr><td class="title"><a href="/use/shaman/">SHAMAN</a></td><td class="server"><a href="https://shaman.pasteur.fr/">Server</a></td><td class="summary markdown">A user-friendly website for metataxonomic analysis from raw reads to statistical analysis</td></tr><tr><td class="title"><a href="/use/snic-science-cloud/">SNIC Science Cloud (SSC)</a></td><td class="server"><a href="https://cloud.snic.se/">Server</a></td><td class="summary markdown">A national-scale cloud infrastructure for Swedish Academia</td></tr><tr><td class="title"><a href="/use/sciensano/">Sciensano</a></td><td class="server"><a href="https://www.sciensano.be/">Server</a></td><td class="summary markdown">General purpose Galaxy server optimized for microbiological genomics applications of interest for public health.</td></tr><tr><td class="title"><a href="/use/shapiro-lab/">Shapiro Lab</a></td><td class="server"><a href="https://binkley2.ncifcrf.gov/users/bshapiro/software.html">Server</a></td><td class="summary markdown">RNA structure and design tools including RNA2D3D and StructureLab</td></tr><tr><td class="title"><a href="/use/singlecell/">Single Cell Omics workbench</a></td><td class="server"><a href="https://singlecell.usegalaxy.eu/">Server</a></td><td class="summary markdown">The Single Cell Omics workbench is a comprehensive set of analysis tools and consolidated workflows</td></tr><tr><td class="title"><a href="/use/south-green/">South Green</a></td><td class="server"><a href="http://galaxy.southgreen.fr/galaxy/">Server</a></td><td class="summary markdown">Genomic resource analysis of southern and Mediterranean plants.</td></tr><tr><td class="title"><a href="/use/streetscience/">Street Science</a></td><td class="server"><a href="https://streetscience.usegalaxy.eu/">Server</a></td><td class="summary markdown">a Galaxy dedicated for the Street Science Community</td></tr><tr><td class="title"><a href="/use/tgm/">TGM</a></td><td class="server"><a href="http://r3lab.uni.lu/web/tgm-pipeline/">Server</a></td><td class="summary markdown">The <em>TGM pipeline</em> is a workflow combining tranSMART server, Galaxy Server and MINERVA platform to enable visually-aided exploration, analysis and interpretation of high-throughput translational medicine data.</td></tr><tr><td class="title"><a href="/use/treegenes/">TreeGenes</a></td><td class="server"><a href="http://treegenesdb.org/">Server</a></td><td class="summary markdown">TreeGenes integrates genotype, phenotype, and environmental data that can be processed for association mapping and landscape genomics, with a focus on forest trees. This functionality is provided by <a href="https://github.com/tripal/tripal_galaxy">Tripal Galaxy</a>, a module for <a href="http://tripal.info">Tripal</a> that provides seamless support for running Galaxy workflows directly in the site. Users can analyze data provided by TreeGenes or their own.</td></tr><tr><td class="title"><a href="/use/usmi-galaxy-demonstrator/">USMI Galaxy Demonstrator</a></td><td class="server"><a href="http://bioinformatics.hsanmartino.it:8080/">Server</a></td><td class="summary markdown">This Galaxy implementation is meant to demonstrate integration of microbial domain Biological Resource Center (mBRC) catalogues into the bioinformatics data integration environment.</td></tr><tr><td class="title"><a href="/use/usegalaxy-be/">UseGalaxy.be</a></td><td class="server"><a href="https://usegalaxy.be/">Server</a></td><td class="summary markdown">General purpose genomics Galaxy server</td></tr><tr><td class="title"><a href="/use/usegalaxy-eu/">UseGalaxy.eu</a></td><td class="server"><a href="https://usegalaxy.eu">Server</a></td><td class="summary markdown">The European UseGalaxy server.</td></tr><tr><td class="title"><a href="/use/usegalaxy-fr/">UseGalaxy.fr</a></td><td class="server"><a href="https://usegalaxy.fr/">Server</a></td><td class="summary markdown">General purpose genomics Galaxy server</td></tr><tr><td class="title"><a href="/use/usegalaxy-no/">UseGalaxy.no</a></td><td class="server"><a href="https://usegalaxy.no/">Server</a></td><td class="summary markdown">A central, joint &#x26; free Galaxy instance for Norway-based researchers and their collaborators, with ~2000 bioinformatics tools, 200GB short-term storage and project storage on nels.bioinfo.no.</td></tr><tr><td class="title"><a href="/use/usegalaxy-org/">UseGalaxy.org (Main)</a></td><td class="server"><a href="https://usegalaxy.org">Server</a></td><td class="summary markdown">The <a href="/galaxy-project/">Galaxy Project</a> free public server; biomedical research</td></tr><tr><td class="title"><a href="/use/usegalaxy-au/">UseGalaxy.org.au</a></td><td class="server"><a href="https://usegalaxy.org.au">Server</a></td><td class="summary markdown">The Australian UseGalaxy server.</td></tr><tr><td class="title"><a href="/use/veupathdb/">VEuPathDB Galaxy</a></td><td class="server"><a href="https://veupathdb.org/veupathdb/app/galaxy-orientation">Server</a></td><td class="summary markdown">A free, interactive, web-based platform for large-scale data analysis focused on eukaryotic pathogens. Merges the former VectorBase and EuPathDB servers.</td></tr><tr><td class="title"><a href="/use/vinyl/">VINYL (Variant prIoritizatioN bY survivaL analysis)</a></td><td class="server"><a href="http://www.beaconlab.it/VINYL/">Server</a></td><td class="summary markdown">A flexible and fully automated system for the functional annotation and prioritization of genetic variants.</td></tr><tr><td class="title"><a href="/use/vinther-lab/">Vinther Lab</a></td><td class="server"><a href="http://galaxy.bio.ku.dk/">Server</a></td><td class="summary markdown">RNA structure-probing data analysis to improve the prediction of RNA secondary and tertiary structure and allow structural changes to be identified and investigated.</td></tr><tr><td class="title"><a href="/use/viramp/">VirAmp</a></td><td class="server"><a href="http://viramp.com/">Server</a></td><td class="summary markdown">VirAmp is a web-based semi-<em>de novo</em> fast virus genome assembly pipeline designed for extremely high coverage NGS data.</td></tr><tr><td class="title"><a href="/use/vvv/">Viral Variant Visualizer (VVV)</a></td><td class="server"><a href="https://viralvariant.anses.fr/">Server</a></td><td class="summary markdown">Determine the composition of a viral population</td></tr><tr><td class="title"><a href="/use/virology/">Virology</a></td><td class="server"><a href="https://virology.usegalaxy.eu/">Server</a></td><td class="summary markdown">A comprehensive set of analysis tools and consolidated workflows for virology work, including SARS-CoV-2.</td></tr><tr><td class="title"><a href="/use/workflow4metabolomics/">Workflow4Metabolomics</a></td><td class="server"><a href="http://workflow4metabolomics.org/">Server</a></td><td class="summary markdown">A collaborative portal dedicated to metabolomics data processing, analysis and annotation.</td></tr><tr><td class="title"><a href="/use/deepts/">deepTS</a></td><td class="server"><a href="https://github.com/cma2015/deepTS">Server</a></td><td class="summary markdown">Explore transcriptional switches from pairwise, temporal, and population RNA-Seq using deepTS.</td></tr><tr><td class="title"><a href="/use/epigeec/">epiGeEC</a></td><td class="server"><a href="https://epigeec.genap.ca/galaxy/">Server</a></td><td class="summary markdown">Compare user-provided epigenomic datasets with thousands of public datasets.</td></tr><tr><td class="title"><a href="/use/immuneml/">immuneML</a></td><td class="server"><a href="https://immuneml.uio.no/">Server</a></td><td class="summary markdown">Machine learning-based analysis and classification of adaptive immune receptors and repertoires (AIRR).</td></tr><tr><td class="title"><a href="/use/qiime2/">q2Galaxy</a></td><td class="server"><a href="https://github.com/qiime2/q2galaxy">Server</a></td><td class="summary markdown">QIIME 2 Galaxy Docker image.</td></tr><tr><td class="title"><a href="/use/wqap/">wQAP</a></td><td class="server"><a href="http://life2cloud.com:6005/qap">Server</a></td><td class="summary markdown">QAP is an open source software that can be used to analyze virus quasispecies using CBS, NGS and TGS data. wQAP (web QAP), is the online version of QAP.</td></tr></tbody></table><!----><footer class="static-footer markdown"><p>The Galaxy Project is supported in part by NSF, NHGRI, The Huck Institutes of the Life Sciences, The Institute for CyberScience at Penn State, and Johns Hopkins University.</p>
</footer></section></div>
    <script src="/assets/js/app.f7f9943a.js" defer></script><script src="/assets/js/page--src--pages--use-vue.8c7d1a05.js" defer></script>
  </body>
</html>
