<!DOCTYPE html>
<html data-html-server-rendered="true" lang="en" data-vue-tag="%7B%22lang%22:%7B%22ssr%22:%22en%22%7D%7D">
  <head>
    <title>Galaxy Platform Directory: Servers, Clouds, and Deployable Resources - Galaxy Community Hub</title><meta name="gridsome:hash" content="91d4a83f9855c194893a1f5e294057505b0cd33d"><meta data-vue-tag="ssr" charset="utf-8"><meta data-vue-tag="ssr" name="generator" content="Gridsome v0.7.23"><meta data-vue-tag="ssr" data-key="viewport" name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover"><meta data-vue-tag="ssr" data-key="format-detection" name="format-detection" content="telephone=no"><meta data-vue-tag="ssr" data-key="description" name="description" content="All about Galaxy and its community."><link data-vue-tag="ssr" rel="icon" href="data:,"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="16x16" href="/assets/static/favicon.ce0531f.7d4a227415c9d9b704d56a079d49a3e1.png"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="32x32" href="/assets/static/favicon.ac8d93a.7d4a227415c9d9b704d56a079d49a3e1.png"><link rel="preload" href="/assets/css/0.styles.e1e25242.css" as="style"><link rel="preload" href="/assets/js/app.0690a31c.js" as="script"><link rel="preload" href="/assets/js/page--src--pages--use-vue.7c5b8683.js" as="script"><link rel="prefetch" href="/assets/js/page--node-modules--gridsome--app--pages--404-vue.9b913011.js"><link rel="prefetch" href="/assets/js/page--src--pages--blog-vue.c32e3571.js"><link rel="prefetch" href="/assets/js/page--src--pages--careers-vue.85d2fb31.js"><link rel="prefetch" href="/assets/js/page--src--pages--data-explorer-vue.8aa3e295.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--archive-vue.1426fe7f.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests--papercuts-vue.ab928745.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests-vue.1e8420fc.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--webinars-vue.5d6fb146.js"><link rel="prefetch" href="/assets/js/page--src--pages--events-vue.cd4ed3c8.js"><link rel="prefetch" href="/assets/js/page--src--pages--index-vue.a143f0a2.js"><link rel="prefetch" href="/assets/js/page--src--pages--news-vue.2f002266.js"><link rel="prefetch" href="/assets/js/page--src--pages--search-vue.23e742ea.js"><link rel="prefetch" href="/assets/js/page--src--pages--tests-vue.3b995c11.js"><link rel="prefetch" href="/assets/js/page--src--templates--article-vue.6d736ff4.js"><link rel="prefetch" href="/assets/js/page--src--templates--platform-vue.91145b17.js"><link rel="prefetch" href="/assets/js/page--src--templates--vue-article-vue.d9a0c32e.js"><link rel="prefetch" href="/assets/js/vendors~page--src--pages--careers-vue~page--src--pages--use-vue~page--src--templates--article-vue~pa~efade918.92a3a9c1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--index-md.b8e1f92a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--md--imgtests--imgdir--index-md.5311083e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--partials--index-md.b6884022.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--imgdir--index-md.1ee4281c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--index-md.fe03c46f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--index-md.05d8aec6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--index-md.e7b48256.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--reldir--index-md.4093d68b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--capacity-planning--index-md.8813525b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--index-md.aafe943e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--performance--cluster--legacy--index-md.3da8719f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--datatypes--index-md.9b672175.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--disk-quotas--index-md.44f752e9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--faq--index-md.3851dd50.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--data-model--index-md.6d581759.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--galaxy-ops--index-md.a3b0d0af.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--ten-simple-steps-galaxy-as-a-service--index-md.699b2e0d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--tool-config-syntax--index-md.208f876a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--writing-tests--index-md.b2caa913.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--allocation--index-md.de3e7728.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--faq--index-md.ac3b79f2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--index-md.328857f9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--ssh--index-md.d92fc074.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--troubleshooting--index-md.7c074dc2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--capacity-planning--index-md.01a397fa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--getting-started--index-md.acdcb61b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--index-md.824830b5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--building--index-md.10fb8489.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--capacity-planning--index-md.e2d98e61.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--cluster-types--index-md.df053139.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--customizing--index-md.fb1f931d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--faq--index-md.b2fab4e5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--index-md.d71b41b2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--pre-201703--index-md.f5a4c4d0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--hadoop--index-md.d7a6d969.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--ht-condor--index-md.31bcf5d4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--index-md.b7b22e1d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--openstack--index-md.d78cb425.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--services--index-md.f2027d09.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--sharing--index-md.cd3d2d6f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--troubleshooting--index-md.fc1a0be3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--userdata--index-md.e4d95203.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--future--index-md.e75566ea.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--index-md.a767ff80.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-09--index-md.66c1654b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-27--index-md.c6d537c1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-09-19--index-md.544a522a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-11-14--index-md.4746a5d0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-01-16--index-md.d1551883.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-03-20--index-md.5094871c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-05-15--index-md.1897abfc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-11-20--index-md.c4d4ced5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-02-19--index-md.311ef2ba.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-04-16--index-md.0b281218.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-06-18--index-md.7c255863.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-08-20--index-md.43ba73c2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-10-15--index-md.bc11b58b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-02-18--index-md.1a182ba1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-04-21--index-md.5e9eea61.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-06-16--index-md.b1295236.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-08-18--index-md.d8fbe7ba.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--index-md.43055631.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--webinar-tech--index-md.a44b91e1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2012--index-md.c7a0a096.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2014--index-md.46c5bc59.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--index-md.34247339.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--data--index-md.46e24851.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--datafiles--index-md.77545970.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--architecture--index-md.55f07291.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--bitbucket--index-md.718bcb6c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--css--index-md.281229bf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--getting-started--index-md.497082a2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--index-md.392b2e76.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--jsa--index-md.ba998e94.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--mercurial--index-md.14a2dad8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--security-tool-tips--index-md.17796bc0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--galaxy-project--index-md.0de41023.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--index-md.d89e2857.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--papers--index-md.4bdf7b3e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--posters--index-md.7e825096.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--presentations--index-md.084c96b9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--videos--index-md.c984f7ce.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--advanced-session--index-md.191fe1da.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--basics-session--index-md.35c32703.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--index-md.8acf4593.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--logistics--index-md.37d0ac9b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--publicity--index-md.7f2eec38.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--registration--index-md.d22dcedd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--abstracts--index-md.bbbf6ae8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--index-md.cb62d683.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--key-dates--index-md.07f8d6b1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--lightning-talks--index-md.72ab128d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--logistics--index-md.c3eeb3dd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--organizing-committee--index-md.f7a53fcd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--photos--index-md.d5102197.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--automation-strategies--index-md.6972b31d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--bioinformatics-training--index-md.648f14ff.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--end-users--index-md.ff776d4f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--workflows-and-api--index-md.540ad6ba.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--index-md.d233185c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--promotion--index-md.212eb34b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--register--index-md.b4c4f57b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--sponsorships--index-md.d0539386.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--training-day--index-md.2ee4bcd3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--index-md.1aa46f9f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--posters--index-md.a8b0ee54.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--talks--index-md.b3ee75dc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--clinical-use-hpc-file-systems--index-md.3fc5e2ed.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-admins--index-md.2f878aaa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-blast--index-md.5305db66.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-professionals--index-md.dcbf6370.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--index-md.3abd720e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--proteomics--index-md.f4d7daaf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--public-galaxy-servers--index-md.b346729f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--tool-developers--index-md.4365e2e3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--index-md.d58334f2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--key-dates--index-md.a3f7c6b8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--lightning--index-md.180bd3c7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--logistics--index-md.c889273c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--organizers--index-md.2f551bcb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--photos--index-md.38ee0e51.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--program--index-md.f2c70270.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--promotion--index-md.1e5aadf9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--register--index-md.99382e6b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--sponsorships--index-md.1e82df1d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--advance-tool-data--index-md.02e350f8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--api--index-md.23e9c64f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--index-md.7b6606de.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--intro-tool-data--index-md.1893d923.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--vms--index-md.eb7d2783.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--index-md.19ebb120.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--posters--index-md.19e77da9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--talks--index-md.69b1844f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--branch-release-merge--index-md.e83f529b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-admins--index-md.4c5b1df3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-end-users--index-md.6b3ff393.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-training-network--index-md.85db1b6f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--heterogeneous--index-md.26797010.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--index-md.ab231746.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--using-galaxy-ui-overview--index-md.3f0318f2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--hackathon--index-md.f1b7f383.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--index-md.894a1ae1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--key-dates--index-md.d3d9965a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--lightning--index-md.268064be.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--logistics--index-md.033eda0f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--organizers--index-md.73259cdb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--photos--index-md.3ee7fbac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--program--index-md.79995b28.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--promotion--index-md.59d9587a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--register--index-md.6cf374d9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--sponsor-exhibit--index-md.2aa46fae.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--admin-walkthrough--index-md.244e6bc0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--api--index-md.3aefd41a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--data-managers--index-md.ea9c133f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--index-md.1c4f2cb1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--vms--index-md.3a12839f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--cwl--index-md.29f1b923.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-management-challenges--index-md.5d4d1ee7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-splitting-and-parallelism--index-md.31c7354c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--elixir--index-md.e49743fc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--extended-workflow--index-md.ca7910bb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--fots--index-md.eea6d2b7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-admins--index-md.8df0d322.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-community-uk--index-md.69eb15c2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-scientists--index-md.3dd577a5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-training-network--index-md.d54633c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--index-md.d03261e3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--data-hack-roundup--index-md.581fdf4b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--header--index-md.72afdfea.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--index-md.adc3dfb6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2016--bofs--index-md.e539350a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--abstracts--index-md.7322e906.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--childcare--index-md.416182d5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--cofest--index-md.ab0c9286.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--conduct--index-md.e979f793.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--faq--index-md.e8fa4530.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--index-md.876ea583.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--organizers--index-md.23f78b02.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--promotion--index-md.85444af0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--registration--index-md.5ed4fbf8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--sponsors--index-md.482f848f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--training--index-md.0709a7f7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--venue--index-md.238012f6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--abstracts--index-md.dc64d26a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--childcare--index-md.234bf37e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--code-of-conduct--index-md.3a181b70.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--collaboration--index-md.003698d8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--faq--index-md.6b9a5e1b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--icebreaker--index-md.bd0460e3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--index-md.139334ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--keynotes--index-md.add6f018.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--organizers--index-md.8f9602b8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--promotion--index-md.55ce8ac1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--register--index-md.0b3372c2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--sponsors--index-md.4546b9bb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--training--index-md.9dbe3953.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--venue--index-md.1408cfc7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2014--index-md.4533e9c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2016--index-md.da0aae7f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2014--index-md.38dcd568.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2016--index-md.80c32555.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--index-md.176fc9a2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--trainingday--index-md.774b7249.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2016--index-md.c5ea9a89.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--example--vue-remark--index-md.9e4f4358.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--faqs--index-md.a910449f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-06--index-md.3a1cff95.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-07--index-md.bb5876b9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--icons--index-md.9a000132.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--index-md.3b38f73f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--logos--index-md.aa44aae7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--news-graphics--index-md.05ad5cba.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--photos--index-md.1d8f64ab.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--screenshots--index-md.a201402f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--index-md.d08ab1a3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--workflow-editor-unannotated--index-md.e72b5147.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--extract--index-md.c7671571.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--tool-panel--index-md.ac20782a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--variables--index-md.3e2afdd3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--faq--index-md.09b46d8e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--galaxy-pages--index-md.ff01861b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--interval-operations--index-md.e1c040b2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--main--faq--index-md.0e15bbd0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--tools--requirements--index-md.d2018317.js"><link rel="stylesheet" href="/assets/css/0.styles.e1e25242.css"><noscript data-vue-tag="ssr"><style>.g-image--loading{display:none;}</style></noscript>
  </head>
  <body >
    <div data-server-rendered="true" id="app" class="layout" data-v-b620dc4c><header id="masthead"><nav class="navbar container justify-content-center navbar-dark bg-transparent navbar-expand-lg"><a href="/" target="_self" class="navbar-brand router-link-active"><img id="masthead-logo" src="/images/galaxy_logo_hub_white.svg" alt="Galaxy Community Hub" height="30"></a><button type="button" aria-label="Toggle navigation" class="navbar-toggler"><span class="navbar-toggler-icon"></span></button><div id="nav-collapse" class="navbar-collapse collapse" style="display:none;"><ul class="navbar-nav"><li class="nav-item"><a href="/use/" aria-current="page" target="_self" class="nav-link router-link-exact-active router-link-active">Use</a></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Learn</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/learn/" role="menuitem" target="_self" class="dropdown-item">How to use Galaxy</a></li><li role="presentation"><a role="menuitem" href="https://training.galaxyproject.org/" target="_self" class="dropdown-item">Teach with Galaxy</a></li></ul></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Community</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/community/" role="menuitem" target="_self" class="dropdown-item">About</a></li><li role="presentation"><a href="/events/" role="menuitem" target="_self" class="dropdown-item">Events</a></li><li role="presentation"><a href="/news/" role="menuitem" target="_self" class="dropdown-item">News</a></li><li role="presentation"><a href="/blog/" role="menuitem" target="_self" class="dropdown-item">Blog</a></li><li role="presentation"><a href="/publication-library/" role="menuitem" target="_self" class="dropdown-item">Publications</a></li><li role="presentation"><a href="/galaxy-project/statistics/" role="menuitem" target="_self" class="dropdown-item">Stats</a></li><li role="presentation"><a href="/careers/" role="menuitem" target="_self" class="dropdown-item">Careers</a></li></ul></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Deploy &amp; Develop</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/admin/get-galaxy/" role="menuitem" target="_self" class="dropdown-item">Install Galaxy</a></li><li role="presentation"><a href="/admin/" role="menuitem" target="_self" class="dropdown-item">Server administration</a></li><li role="presentation"><a href="/tools/" role="menuitem" target="_self" class="dropdown-item">Tools</a></li><li role="presentation"><a href="/develop/api/" role="menuitem" target="_self" class="dropdown-item">Working with the API</a></li><li role="presentation"><a href="/develop/" role="menuitem" target="_self" class="dropdown-item">Contributing</a></li><li role="presentation"><a href="/docs/" role="menuitem" target="_self" class="dropdown-item">Code documentation</a></li></ul></li><li class="nav-item b-nav-dropdown dropdown"><a role="button" aria-haspopup="true" aria-expanded="false" href="#" target="_self" class="nav-link dropdown-toggle"><span>Support</span></a><ul tabindex="-1" class="dropdown-menu"><li role="presentation"><a href="/support/" role="menuitem" target="_self" class="dropdown-item">FAQ</a></li><li role="presentation"><a role="menuitem" href="https://help.galaxyproject.org/" target="_self" class="dropdown-item">Galaxy Help Forum</a></li></ul></li><li class="nav-item"><a href="/jxtx/" target="_self" class="nav-link">@jxtx</a></li></ul><ul class="navbar-nav ml-auto"><li class="form-inline"><form action="/search/" method="get" class="form-inline"><input id="search-input" name="q" type="text" placeholder="Search Galaxy" value="" class="mr-sm-2 form-control form-control-sm"><button type="submit" class="btn my-2 my-sm-0 btn-secondary btn-sm"><span class="fa fa-search"></span></button></form></li></ul></div></nav></header><section id="maincontainer" class="container"><h1 class="page-title" data-v-b620dc4c>Galaxy Platform Directory: Servers, Clouds, and Deployable Resources</h1><div class="markdown" data-v-b620dc4c><div class="img-sizer float-right" style="max-width: 35%;">
<p><img class="g-image g-image--lazy g-image--loading" src="data:image/svg+xml,%3csvg fill='none' viewBox='0 0 1100 402' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink'%3e%3cdefs%3e%3cfilter id='__svg-blur-3612ab8e8a4de515b02eccd7b705cc05'%3e%3cfeGaussianBlur in='SourceGraphic' stdDeviation='40'/%3e%3c/filter%3e%3c/defs%3e%3cimage x='0' y='0' filter='url(%23__svg-blur-3612ab8e8a4de515b02eccd7b705cc05)' width='1100' height='402' xlink:href='data:image/png%3bbase64%2ciVBORw0KGgoAAAANSUhEUgAAAEAAAAAXCAIAAAA0gN7GAAAACXBIWXMAAAsSAAALEgHS3X78AAAP1UlEQVRYw504Z1BbaZLa27u6q7uZ3durq72tGu94bJODMWAyxh4bY7ANBmOTo0GYnJFAZASIJCGyyckgkkCASCIjIURGZIQEIkoCRMZkdJ/w3t6vuR/3quurfv1193v9dffr7geRkX/%2bWyD96FdRKU3Jh09FpTREJDWk5J7JyP8K6AABRAnZp5IPhcgDCXVRaU1ABPzgFgDApeR%2bBbdi0ppiMk%2b%2b47cMQBbwPLuVfQpuwQp0istoiUk/uRX/%2bxbg/y7y6//xhgAgv7UBhJXUX1vbe%2brqW9o5eEOdA3TfWko81AIaFVX1VDXfmlm5KGu8UX2ib2nr7uIRpKCi%2b1BRG1AAPFbTk1d%2bBTjNbdws7dxlFJ7LK%2bsoqurKKryQe/wSyOoZWKtovgVPUdcyAGx6BlZCDcq6aloGQC0QB6rklYQico%2b1/58GAL2PlHRQCWk6r80jkGgA3n7h4VFobV0zD%2b%2bQtIyC1Iz8uMT0zC9F%2bYU4AJHRGGxaXmxcalpmAQabHRufBgxDhMbl5JehEtLR2Gz993Z%2b8ChkTHI0KgXsfskpQcYmJ6fmWtl7hEYk%2bMORAYHIvIJyVGIaYI5PFIqERSa6e4eISGkAe8ArAftv4TcNAM4C29qyii8AyCi8kJZ/bmzm5OQKhwXFODoHgBd6bwoFR4tEYcORaECJisW6eiJcPBDAUT7%2b4cAwU0uXT05%2bdo4%2bvrBIK1uPV28snd0DA27FgYsCQ1CBwbF%2bMKS3fwRQ6%2bweVEtoAczAt9GxWBgiJgKJAQqBhcAeqAssPjHD1MJFTFpLTkkH2PA9dKUePpOW%2b9%2b4ggiD7JEwRqUVnkvIPrkvrvxAXEVUTEVKWlMWCIDglnkiIaMu9VBTVFIVIOLS6i9eGj16/EJCRkNCRlNMWl1CSkW4CmVV74ur3BNXBquYtIaELABNcRkNUXGln%2b7K3n2geOcXhZ8fPP5FVBE85ecH8gD5aO6o/lRfVFoNqL0nqiQioQK27okpgWdJymi81DMREVf6%2bf6juw/k74sqK6nqqj55q6z5Rl7l1W16CDME8lhVV%2beNubmFk5ujm4s7LBARF5qQ4I4Oj22IrZwtrpuuKaUVUkeozEUWg8HqI48ODY22dvVSaLRe6mB7N5k5N8WidSwM9WyssB38o39UNP7rU%2bsfFYxdQ9A8LofJWmavrNa29ZaV13j7u1hYvzWz0K9pwNc11BR8ze/s6cTXEnLTM1YXGXh826s3ofrGMbr64Tn5NUzmYk5FbVhmbmz6l9LSirT0HHh4UHkVvrK0Oiky0c3Z3%2bijAwjmRyq6EJCpBka25vYeFs5%2boZHowpImfM9MSVPNNJN8Lji7Epzyv/H2jnYFN4LV9WPWMp%2b/s7d9ej61sc3cWGUw5vfYs%2bf8zZutBcE3jjci%2bl/V7P/6yuVflCyD0IUCgeDy7Ozk4nxkhjXP4rPYI1QankDI7%2bzqps2y9i4uT84uZ2ZnQ2EBgsO9weF5Q5NUI9PMoLCafurUjUBQ0d5X3dhQ00HZ4J5v8L719/f1kKmESkJiENLL2d/SzsfA2FFZ8y0wQPfNO2srexcnPz9YMAqNKSxqnWnP8RyrjPuKa%2bjvJTfWE8fHxy8urw4PT4%2bODvg727MbPOr80vDSKqmrY7yzgT01tMWaPuKtfg5O%2bkHF6qcX0D%2bpWyFTi4ABq99OuddX1BE6ZWiBzV6YnhlpbycSW9t61wXF9DMKY/P4cL%2bqBj89SMWX1xuZYY1MU4LDK%2bsb%2b64uz2s7elpysppbu2ljK9PzHNrAIKm9u6GpPbu4wj841sTS/a2RvYoGMEBNDxhgb%2bMGh0WCJEtN%2b1pMmiFlO49Vx1QRWsfHRgcHB5lM5vX1zc3N9eHh4d4uf2P/eJS9MclaIbUSRloqqcQK%2bkDvDo9jD4v7QcXyzgvoj8oW4VihB1Z2T9bW93spdPrc5twcs719qK6O1NPVub7LJ43SOBur19fX4GjWV5dLskr1DBP13iW9N8MS6noml9n22dWm0enYyub5RT5z5YBOp5M6usvT8%2bJ9Q9xs3U2s/m6Aup6%2bka2JjUsAIio4LCEnF4%2bnLJFwMbzxRoHwujm5vW5xwenp6cHB/va385nlFRBF5Dnm2tTg9fnJXGc1d6jVCur1B02HX157/VHrU0h60f4u//ji4lggoDPYrLXDjc3N8fF5av9IV2c3gzmxw6fu7vGBTmADWEnttABEBTq147NHcXPTwBCDqV3YIpnf5oXr4G6ecvjXDCaTShtprKqPd4O727hZ2fu%2b%2b%2bAISgpEQUVHW9fko4mNk4svqCaBwUmoVFxgVjWO0ESjUslkamsrqaend3RkfHJiltJHayAQC/ANuSW4xHJCZFE1qamxMh1VnYPtayYY2bj/k/Sb/1A0%2bp3oy09%2byH5KP31ymsFkVdU04Kray3GVcXHo2BhwRrk5Ofl5OSXVNfiJickJ%2btTs3FxRCf6DWaSnb7qXX3pySvHw8FBtHcEtLd89LgtX0drQOlBLaCwoKs/IyAvxC7G2cALHr61nDkoeRF75pcYzfVBuTSygJhaO5lYuUGf/zx4BiNDIRDQ2GZsejUqCwyNM7ezkXynIv3zs5OX2JbsAnZqVmVOMxmZmZOWmJKdkfsnLzS/2DYp19YuGh6Gj4jI8/cLCIqL0dPX0dHQT0amoJCQmJTk17UsSOiU0FBkRjgoJiYIFhKWmZfj6BWAwmJSUjKjoxLj4VBg8PACGSExKTsKkJsUlICOjU9KyMJhUREhkYBAoNQiLT16mllA9fYunL94rKL%2bCKKtp6%2boZf3Ly94b5%2bAT6lpa3DQ6MLMzNbm9vC4OYvcIc7Fhm81kb653jzV3jpBnmtDCwbsB3QrC3u3NxcQGQqythGJQTBoLQpPSyoQI8va5lkMvlEOsJdVW4LS5//5K9dUG/EZxtrm0P9E4dH562NQwS63oWFiZ7umao1F7mMuPi4hIoWV5iTU1NXgK95%2bdkCqWPQrm6uTk5PadPzrR09H/52hyDKfCGRTu5%2busb26oIQ%2bjx81d6JqBSBofFRESjcwrrOzspw8OjjEXW7uHJ8uzEWFNZ38Dst9PL/d0D7jqXxWIvrW8trXOXl9lj4xOzM9PtrU3NxMaWZmJueVsGbjgsuTk%2br5fYMba0xOJv7%2bzweIsLjL0jztR64zKfNjDZ1NM%2bPNA3RWqi5WTgFhbHZ1gXlNLumc7qvb3D6anptrY2CqV/dw9c%2bwMDNFJ7xyZ3a22DSx0YLCgnphfUJmLzzR3gb40tNbXfPVJ5CQG91ztjezdPBOhSMCm5lTUdQ0NTLNYSl7d1c5thkwssxvIuOJvjb7tbvM31jQ1w4jc3Z8ANvO31i4tTwdWR4OIIMLBmV8entobn9oaHN6fmVs/OT8/BMZ6d7u/uH5yv7Z4vHF%2bvrfHmeTz%2byMD8CmuDs7UjuPhGKZ%2bvhg2PldPOL4Ue4PG4gzRwDYDPzjQwaHr66vr6%2bNvp0PAosbU7MafOPzLTFx7j6hFobAoFrSTkkdJL9acGOq/NjE0cDD/Y67w1f2fyydTOw8rus/NndxcXTxsbRxs7109Qt8/OHnb2rjZQD%2b/IRK/weD8k5jMcafvJyRHqbm0H/WBm6WjtGuAT7RsYAvMP8/YNDgmNBNEcERmLCImIjELFRKNjY7GhoSgELDg4JMLVC%2b7rEwB19NV95isjH6r1DB4VFRMdHRcVFevq5mVhaWNmbmNgaOro5I6Kw6JiUb7%2bCBc3H2cXH0sbFwMjK63nhiqab0CvCgGJrK1nYmLu%2bNnVz8HZ86W5v3dEam5mrnVA2M9G5rJQf4XPQV%2bKcERiu6F7vLiJr1IwVq%2bH86ad/byH98f0zvv/LHcfIiH3UF772YvYnOhIQpiOrksXeWR8fIzH2xofn9jc5DAYi2vr6xzgU%2b4qjUYo7KB8SC41Q2WVNpG6e3q7%2bkDMtHR1tzGW2OTp8e6xAVxdXVt7f2V1KzwoFVdWvbKAJw7M43BVZeXV2fllfogYG3t3w4/Ql68tQBsBUdR4pWdo4eDkFRQSAw%2bJ8YzIxTdTpoi1bqlZPwWh/9PS/b%2bMrAidowcHN67ZpHuwhGdZJdZjpzZDByJdRxBs/5//7c6H1/%2buqCRhZmgbWRP0a7bfkz80Dretra5xrq%2bvQEgsLi6COgiCUZj1V9XEbk/FmFJJZKF1XC6rrRPQQHG/Oj/a39vjHuyNbmyytncmJpcX2MdbO8ds9vLqMqsZZBqrOz0VW1yKy8wq9PDy9/AJsXWEG5lA1bT0QQ68fmtk6%2bETGoCINbDwhAam5Nd2Ls7N%2bpaU/eDgLmXwUeGDRU0LdXXtGEMgW%2bYUmheU%2b1M2fXqX5fCMPyMb/nLnH5UUICoq/yB9F2KPMfDN8GUGondWS5iMbmGBur7e4HKX2OybG6ExZydJLX0fwnCkgLqe5OrWberg4cn%2b0f7W7vYamznL2%2bHNc9YXVrcG6ayl9fOlFd7c/Dzw3gZ3h7853dra1tzSgcfXx6Pi/eHRNg4wA2MHYQ4AL7x5Z%2bPhFewfiDSwcIMiMtBfa%2bsplXZfUiCOAX%2b08LgLhVU0kafntzZ39y4Fx/hBukct3bacppNLVgkvgvzu99LiEGWl36vJQ0LTbFrnGmdn6WRaYl9/5SV//3hk5LCyaqWp%2bfLm%2bvbDu9A/ioRmVRAH6bWUYdLg%2bNkh72ifs8Hhgm53hbc2trY4sbxFHl5aWjtb5x2sczaXWCzGwvzQyFQjsbmlrbOluS0nKQoegPhuAEgDCGivDd7bgVkkBpUcEpVgF5haVE8apTcX9HQGNPdFkQYQ%2bI6GzuHJOe7KJv/gaL%2b2bwRaQDbEtkojav9infDTnXvPn8lraIjLSP8Jg0ngHwrGphhD01sT8%2bzrg/1zOn2vrW2VTAav/%2b1i%2b/CUt8ZZnOPtttEXMHWd9eRBYNP56f75yfY2j3Nyery0z5ndZNImFzi7gk329vLU/NLy6uXl5cHBHqjZPX1UUkdvfl6Rb0CEHTQQpIEa8ABoJV68%2bmhi8Rl8mOydA9QNnd3hMXl5RcVFZY2Exvq6%2bvKyipSM/NzCqjJcbX19Y2Ri5nsPlLZDpLJF2J2n9qJiko8egQFI7oGIrLsXrBU0jI1NVbXE8oqqrvbO4lQwc%2bZm5xW2t3c1EpvwhJoyXDm5uxfUJyck1jEiEfRFtbUEYiOxvp7Y1EKqaK7Hf8XlZ5eU1fWVpuRgfGHZ%2bcVNxBZ8bUNuXjEGmxmPTocjYsDcrG8snAeErYRwFvvbXwYtcWlNEUkwVamKSKrdE1O%2bK/IYrGC8EpVQFZVUAVsPJMCWqpiUmoS0uiQY02Q1ACdg%2bJ8tNVFJNbCKSamLSqoL6UJE7TuPqJCuAei/iAqHPlk5LcAG9D%2bQAAzqQpBQFZNQByIAvw/o0hoisk8AJ8BFhC%2bgLtQvBXz95Pbfh9b3fxb/DTCfZQTZ5dXBAAAAAElFTkSuQmCC' /%3e%3c/svg%3e" width="1100" alt="125+ platforms to use Galaxy on" data-srcset="/assets/static/use-resource-banner.82a2fbd.9a59c7336b4eb29ce5c910a631d552f2.png 480w, /assets/static/use-resource-banner.a6806d3.9a59c7336b4eb29ce5c910a631d552f2.png 1100w" data-sizes="(max-width: 1100px) 100vw, 1100px" data-src="/assets/static/use-resource-banner.a6806d3.9a59c7336b4eb29ce5c910a631d552f2.png"><noscript><img class="g-image g-image--lazy g-image--loaded" src="/assets/static/use-resource-banner.a6806d3.9a59c7336b4eb29ce5c910a631d552f2.png" width="1100" alt="125+ platforms to use Galaxy on"></noscript></p>
</div>
<p>This directory lists platforms where you can use or deploy your own Galaxy Server with minimal effort.</p>
<ul>
<li><em>Almost all of them are free to use (Commercial clouds are the exception)</em></li>
<li><em>Almost all of them are open to everyone (Academic clouds are the exception).</em></li>
<li><em>All of the resources can be used, either immediately, or shortly after getting an account.</em></li>
</ul>
<p>These resources cover a wide spectrum of domains all across life sciences. There are resources for genomics (lots of them), metagenomics, transcriptomics, proteomics, drug discovery and even some outside biology like natural language processing (a couple of them), and social science.</p>
<!---
The rest of this page (the actual directories) are generated by
layouts/use_index.pug
--->
</div><ul id="resource-tabs" role="tablist" class="nav nav-tabs nav-fill" data-v-b620dc4c><li class="nav-item" data-v-b620dc4c><a id="usegalaxy-tab" data-toggle-group="use-panes" data-toggle="tab" role="tab" aria-controls="usegalaxy-pane" aria-selected="true" class="nav-link  active" data-v-b620dc4c><strong data-v-b620dc4c>UseGalaxy</strong></a></li><li class="nav-item" data-v-b620dc4c><a id="all-resources-tab" data-toggle-group="use-panes" data-toggle="tab" role="tab" aria-controls="all-resources-pane" aria-selected="true" class="nav-link" data-v-b620dc4c><strong data-v-b620dc4c>All</strong></a></li><li class="nav-item" data-v-b620dc4c><a id="public-server-tab" data-toggle-group="use-panes" data-toggle="tab" role="tab" aria-controls="public-server-pane" aria-selected="true" class="nav-link" data-v-b620dc4c><strong data-v-b620dc4c>Public Servers</strong></a></li><li class="nav-item" data-v-b620dc4c><a id="academic-cloud-tab" data-toggle-group="use-panes" data-toggle="tab" role="tab" aria-controls="academic-cloud-pane" aria-selected="true" class="nav-link" data-v-b620dc4c><strong data-v-b620dc4c>Academic Clouds</strong></a></li><li class="nav-item" data-v-b620dc4c><a id="commercial-cloud-tab" data-toggle-group="use-panes" data-toggle="tab" role="tab" aria-controls="commercial-cloud-pane" aria-selected="true" class="nav-link" data-v-b620dc4c><strong data-v-b620dc4c>Commercial Clouds</strong></a></li><li class="nav-item" data-v-b620dc4c><a id="containers-tab" data-toggle-group="use-panes" data-toggle="tab" role="tab" aria-controls="containers-pane" aria-selected="true" class="nav-link" data-v-b620dc4c><strong data-v-b620dc4c>Containers</strong></a></li><li class="nav-item" data-v-b620dc4c><a id="vms-tab" data-toggle-group="use-panes" data-toggle="tab" role="tab" aria-controls="vms-pane" aria-selected="true" class="nav-link" data-v-b620dc4c><strong data-v-b620dc4c>VMs</strong></a></li></ul><div id="resource-tabs-content" class="tab-content" data-v-b620dc4c><div id="usegalaxy-pane" data-toggle-group="use-panes" aria-labelledby="usegalaxy-tab" role="tabpanel" class="tab-pane fade show active" data-v-b620dc4c><h2 class="nav-item" data-v-b620dc4c>
                    UseGalaxy Resources
                </h2><p class="markdown" data-v-b620dc4c><p><a href="/usegalaxy/">UseGalaxy servers</a> implement a common core set of tools and reference genomes, and are open to anyone to use.  They also contain tools and genomes that are local to each server. Each is backed by significant computational resources and they are excellent places to get started with Galaxy, and to share and publish your results.</p>
</p><p class="d-none" data-v-b620dc4c>dummy text</p><table class="table table-striped" data-v-b620dc4c><thead data-v-b620dc4c><tr data-v-b620dc4c><th data-v-b620dc4c>
                            Resource
                        </th><th data-v-b620dc4c>
                            Server
                        </th><th data-v-b620dc4c>
                            Summary
                        </th><th data-v-b620dc4c>
                            Keywords
                        </th></tr></thead><tbody data-v-b620dc4c><tr data-v-b620dc4c><td class="title"><a href="/use/usegalaxy-eu/">UseGalaxy.eu</a></td><td class="link"><a href="https://usegalaxy.eu">
            Server
        </a></td><td class="summary markdown">The European UseGalaxy server.</td><!----><td class="keywords"><a href="/use/#usegalaxy-dir">
            UseGalaxy
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/usegalaxy-org/">UseGalaxy.org (Main)</a></td><td class="link"><a href="https://usegalaxy.org">
            Server
        </a></td><td class="summary markdown">The <a href="/galaxy-project/">Galaxy Project</a> free public server; biomedical research</td><!----><td class="keywords"><a href="/use/#usegalaxy-dir">
            UseGalaxy
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/usegalaxy-au/">UseGalaxy.org.au</a></td><td class="link"><a href="https://usegalaxy.org.au/">
            Server
        </a></td><td class="summary markdown">The Australian UseGalaxy server.</td><!----><td class="keywords"><a href="/use/#usegalaxy-dir">
            UseGalaxy
        </a></td></tr></tbody></table><p class="markdown" data-v-b620dc4c><p>You can <a href="https://status.galaxyproject.org/" target="_blank" rel="noopener noreferrer">check the status of all UseGalaxy resources here</a></p>
</p></div><div id="all-resources-pane" data-toggle-group="use-panes" aria-labelledby="all-resources-tab" role="tabpanel" class="tab-pane fade show" data-v-b620dc4c><h2 class="nav-item" data-v-b620dc4c>
                    All Resources
                </h2><p class="markdown" data-v-b620dc4c><p>Complete listing of resources for using Galaxy.</p>
</p><p class="d-none" data-v-b620dc4c>dummy text</p><table class="table table-striped" data-v-b620dc4c><thead data-v-b620dc4c><tr data-v-b620dc4c><th data-v-b620dc4c>
                            Resource
                        </th><th data-v-b620dc4c>
                            Server
                        </th><th data-v-b620dc4c>
                            Cloud
                        </th><th data-v-b620dc4c>
                            Deployable
                        </th><th data-v-b620dc4c>
                            Summary
                        </th><th data-v-b620dc4c>
                            Keywords
                        </th></tr></thead><tbody data-v-b620dc4c><tr data-v-b620dc4c><td class="title"><a href="/use/ab-openlab/">AB-OpenLab</a></td><td class="link"><a href="http://ab-openlab.csir.res.in/frog/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">FROG stands for <strong>F</strong>inge<strong>R</strong>printing <strong>O</strong>ntology of <strong>G</strong>enomic variations. FROG fingerprints have been devised to capture genomic variations at various levels.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/abims/">ABiMS</a></td><td class="link"><a href="http://galaxy.sb-roscoff.fr/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Analysis and bioinformatics for Marine Science.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/abims-tools/">ABiMS-Tools</a></td><td class="link"><a href="http://galaxy3.sb-roscoff.fr/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Hosts tools/resources developed at the <a href="http://www.sb-roscoff.fr/">Station Biologique de Roscoff</a> in collaboration with <a href="http://abims.sb-roscoff.fr/">ABiMS</a>.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ageseq-aspendb/">AGEseq @ AspenDB</a></td><td class="link"><a href="http://aspendb.uga.edu:8085/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">This site provides a user-friendly interface for AGEseq (Analysis of Genome Editing by Sequencing) in a Galaxy instance.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/aws/">Amazon Web Services (AWS)</a></td><td class="link"></td><td class="link"><a href="https://launch.usegalaxy.org/">
            Commercial
        </a></td><td class="link"></td><td class="summary markdown">Set up you own general purpose genomics server runing AWS using GVL and CloudMan.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/anastasia/">ANASTASIA</a></td><td class="link"><a href="http://motherbox.chemeng.ntua.gr/anastasia_dev/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown"><strong>A</strong>utomated <strong>N</strong>ucleotide <strong>A</strong>minoacid <strong>S</strong>equences <strong>T</strong>ranslational pl<strong>A</strong>tform for <strong>S</strong>ystemic <strong>I</strong>nterpretation and <strong>A</strong>nalysis</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/anvil/">AnVIL</a></td><td class="link"></td><td class="link"><a href="https://anvilproject.org/">
            Academic
        </a><a href="https://anvilproject.org/">
            Commercial
        </a></td><td class="link"></td><td class="summary markdown">Run Galaxy inside the secure AnVIL environment on Google Cloud Platform.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/apostl/">APOSTL</a></td><td class="link"><a href="http://apostl.moffitt.org/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Automated Processing of SAINT Templated Layouts (APOSTL) is a freely available software suite and analysis pipeline for reproducible, interactive analysis of AP-MS data.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/arga/">ARGA</a></td><td class="link"><a href="http://mem.rcees.ac.cn:8083/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Antibiotic Resistance Gene Analyzer</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/args-oap/">ARGs-OAP</a></td><td class="link"><a href="http://smile.hku.hk/SARGs">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Fast annotation and classification of antibiotic resistance gene-like sequences from metagenomic data.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ariaweb/">ARIAweb</a></td><td class="link"><a href="https://ariaweb.pasteur.fr/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Automated NOE assignment and NMR structure calculation</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/aries/">ARIES</a></td><td class="link"><a href="https://w3.iss.it/site/aries/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Analysis of genomic data in the field of public health and food safety, with the aim of deploying a comprehensive bioinformatics approach to the study of food-borne zoonoses and infectious diseases at the human and animal interface.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/asaim/">ASaiM</a></td><td class="link"></td><td class="link"></td><td class="link"><a href="https://quay.io/repository/bebatut/asaim-framework?tag=latest&amp;tab=tags">
            Container
        </a></td><td class="summary markdown">Integrates tools specifically chosen for metagenomic and metatranscriptomic studies and hierarchically organized to orient user choice toward the best tool for a given task.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/atacgraph/">ATACgraph</a></td><td class="link"></td><td class="link"></td><td class="link"><a href="https://hub.docker.com/r/lsbnb/galaxy_atacgraph">
            Container
        </a></td><td class="summary markdown">ATAC-seq for biologists with minimal bioinformatics knowledge to analyze chromatin accessibility</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/bf2i-map/">BF2I-MAP</a></td><td class="link"><a href="http://bf2i-galaxy.insa-lyon.fr:8080/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Locally developed as well as standard tools are available on a 16-core server with 32GB memory, accessible to all users, and located in the BF2I lab.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/biobix/">BioBix</a></td><td class="link"><a href="http://galaxy.ugent.be/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">A general genomics server. Includes the PROTEOFORMER pipeline tools.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/biociphers-lab-galaxy/">BioCiphers Lab Galaxy</a></td><td class="link"><a href="http://avispa.biociphers.org/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">BCL Galaxy gives a user friendly interface for analysis tools developed by the <a href="http://biociphers.org/">BioCiphers Lab</a> at the University of Pennsylvania.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/biocompute-object/">BioCompute Object Galaxy</a></td><td class="link"><a href="https://galaxy.aws.biochemistry.gwu.edu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown"><a href="https://biocomputeobject.org/about.html">BioCompute Objects</a>  is a standard for communication of High-throughput Sequencing (HTS) analysis results, data set creation and curation, and bioinformatics verification protocols.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/biodivine/">BioDivine Toolset</a></td><td class="link"><a href="https://biodivine-vm.fi.muni.cz/galaxy/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Systems biology tools implemented by the Sybila team.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/biomas/">BioMaS</a></td><td class="link"><a href="http://galaxy.cloud.ba.infn.it:8080/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Taxonomic studies of environmental microbial communities</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/biomina/">Biomina</a></td><td class="link"><a href="http://143.169.238.104/galaxy/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">A general purpose Galaxy instance that includes most standard tools for DNA/RNA sequencing, plus extra tools for panel resequencing, variant annotation and some tools for Illumina SNParray analysis.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/bipaa/">BIPAA (BioInformatics Platform for Agroecosystem Arthropods)</a></td><td class="link"><a href="https://bipaa-galaxy.genouest.org">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Insect genomics (aphids, parasitoïd wasps, lepidopterans)</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/bitlab/">BitLAB</a></td><td class="link"><a href="https://pistacho.ac.uma.es/galaxy-bitlab">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Detailed analysis of metagenomes.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/bridge/">BRIDGE</a></td><td class="link"><a href="https://galaxy-compchem.ilifu.ac.za/">
            Server
        </a></td><td class="link"></td><td class="link"><a href="https://github.com/scientificomputing/bridge-docker">
            Container
        </a></td><td class="summary markdown">Prepare, simulate and analyse and visualise molecular simulations, especially molecular dynamics and free energies.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/cafu/">CAFU</a></td><td class="link"><a href="http://omicstudio.cloud:4001/">
            Server
        </a></td><td class="link"></td><td class="link"><a href="https://hub.docker.com/r/malab/cafu">
            Container
        </a></td><td class="summary markdown">A Galaxy framework for exploring unmapped RNA-Seq data</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/cbib-galaxy/">CBiB Galaxy</a></td><td class="link"><a href="http://services.cbib.u-bordeaux2.fr/galaxy/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">A general purpose Galaxy instance that includes EMBOSS (a software analysis package for molecular biology) and fibronectin (diversity analysis of synthetic libraries of a Fibronectin domain).</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/cell-organizer/">CellOrganizer for Galaxy</a></td><td class="link"><a href="http://galaxy3.compbio.cs.cmu.edu:9001">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Create generative models of the spatial organization of cells from microscope images and automatically provide geometries for spatial simulations of cell processes and behaviors</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/center-for-phage-technology-cpt/">Center for Phage Technology (CPT)</a></td><td class="link"><a href="https://cpt.tamu.edu/galaxy-pub/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Phage biology and automated annotation.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/chemflow/">ChemFlow</a></td><td class="link"><a href="https://vm-chemflow-francegrille.eu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Chemometrics without programming: infrared spectrometry on a Galaxy platform.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/cart/">Chemical Annotation Retrieval Toolkit (CART)</a></td><td class="link"><a href="http://cart.embl.de/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Retrieves annotations on biological effects of chemicals and determines which ones are enriched.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/chemicaltoolbox/">ChemicalToolbox</a></td><td class="link"><a href="https://cheminformatics.usegalaxy.eu/">
            Server
        </a></td><td class="link"></td><td class="link"><a href="https://quay.io/repository/bgruening/galaxy-chemicaltoolbox">
            Container
        </a></td><td class="summary markdown">For processing, analysing and visualising chemical data, and performing molecular simulations.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/cirm-cfbp/">CIRM-CFBP</a></td><td class="link"><a href="https://iris.angers.inra.fr/galaxypub-cfbp">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Web based access to tools (<a href="https://sourcesup.renater.fr/wiki/skif/">SkIf</a>, <a href="https://sourcesup.renater.fr/wiki/ki-s/">KI-S</a>) of IRHS-EmerSys lab.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/cistrome-analysis-pipeline/">Cistrome Analysis Pipeline</a></td><td class="link"><a href="http://cistrome.org/ap/root">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">ChIP-chip/seq and gene expression data</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/climate-science-workbench/">Climate Science Workbench</a></td><td class="link"><a href="https://climate.usegalaxy.eu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">A comprehensive set of analysis tools and consolidated workflows for climate analysis and visualisation</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/climb/">CLIMB</a></td><td class="link"></td><td class="link"><a href="http://www.climb.ac.uk/">
            Academic
        </a></td><td class="link"></td><td class="summary markdown">The CLIMB project (Cloud Infrastructure for Microbial Bioinformatics) develops and deploys a world leading cyber-infrastructure for microbial bioinformatics; providing free cloud-based compute, storage, and analysis tools for academic microbiologists in the UK.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/clip-explorer/">CLIP-Explorer</a></td><td class="link"><a href="https://clipseq.usegalaxy.eu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">The CLIP-Explorer is a webserver to process, analyse and visualise CLIP-Seq data.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/codon-harmonizer/">Codon Harmonizer</a></td><td class="link"><a href="http://codonharmonizer.systemsbiology.nl/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Easily generate codon harmonized variants of gene sequences for heterologous expression</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/coloc-stats/">Coloc-stats</a></td><td class="link"><a href="https://hyperbrowser.uio.no/coloc-stats">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">a unified web interface to perform co-localization analysis of genomic features</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/coral-snp/">CoralSNP</a></td><td class="link"><a href="https://coralsnp.science.psu.edu/galaxy/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Standard Tools for Acroporid Genotyping (STAG)</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/corgat/">CorGAT</a></td><td class="link"><a href="http://corgat.cloud.ba.infn.it/galaxy">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Align complete assemblies of SARS-CoV-2 genomes wih the reference genomic sequence, to obtain a list of polymorphic positions and to annotate genetic variants.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/cossci/">CoSSci</a></td><td class="link"><a href="http://socscicompute.ss.uci.edu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Tools for solving <a href="http://en.wikipedia.org/wiki/Galton&#x27;s_problem">Galton's problem</a> in Comparative Research and complex network problems in Social Science.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/covid-19/">COVID-19</a></td><td class="link"><a href="https://covid19.usegalaxy.eu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">A companion to our study describing the analysis of early COVID-19 data</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/cravat-p/">CRAVAT-P</a></td><td class="link"></td><td class="link"></td><td class="link"><a href="https://hub.docker.com/r/jraysajulga/galaxy-cravatp-docker/">
            Container
        </a></td><td class="summary markdown">A Docker image containing a fully-operational Galaxy instance with pre-installed demonstration material for CRAVAT-P.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/cropgalaxy/">CropGalaxy</a></td><td class="link"><a href="http://cropgalaxy.excellenceinbreeding.org/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Hosts tools for the <a href="/src/use/ricegalaxy/index.md">RiceGalaxy</a>, <a href="http://gobiiproject.org/">GOBii project</a>, <a href="http://cegsb.icrisat.org/high-throughput-genotyping-project-htpg/">HTPG project</a> and the <a href="http://excellenceinbreeding.org/">Excellence in Breeding (EiB) platform</a>, including genomic selection, marker selection, GWAS, imputation, file conversion, and cluster analysis.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/deepts/">deepTS</a></td><td class="link"><a href="https://deepts.nwafu.edu.cn/">
            Server
        </a></td><td class="link"></td><td class="link"><a href="https://hub.docker.com/r/malab/deepts">
            Container
        </a></td><td class="summary markdown">Explore transcriptional switches from pairwise, temporal, and population RNA-Seq using deepTS.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/dintor/">Dintor</a></td><td class="link"><a href="http://dintor.eurac.edu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">GWA and NGS tools and modules for functional annotation of genes and gene products</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/epigeec/">epiGeEC</a></td><td class="link"><a href="https://epigeec.genap.ca/galaxy/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Compare user-provided epigenomic datasets with thousands of public datasets.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/epigenomics/">Epigenomics Galaxy</a></td><td class="link"></td><td class="link"></td><td class="link"><a href="https://hub.docker.com/r/mpaya/epigenomics_galaxy">
            Container
        </a></td><td class="summary markdown">Galaxy Docker image for epigenomics analysis</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/erasmus-mc/">Erasmus MC</a></td><td class="link"><a href="https://erasmusmc.usegalaxy.eu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">General purpose genomics analysis, featuring many standard tools plus many additional tools.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ewas-galaxy/">EWAS-Galaxy</a></td><td class="link"></td><td class="link"></td><td class="link"><a href="https://github.com/kpbioteam/docker_ewas">
            Container
        </a></td><td class="summary markdown">Galaxy Docker image for population epigenetics data analysis</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/freebioinfo/">FreeBioinfo</a></td><td class="link"><a href="http://www.freebioinfo.org/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Galaxy for metagenomic analysis.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/g-onramp/">G-OnRamp</a></td><td class="link"></td><td class="link"><a href="https://launch.usegalaxy.org/catalog">
            Commercial
        </a></td><td class="link"><a href="https://wustl.app.box.com/v/g-onramp-vm-v1">
            VM
        </a></td><td class="summary markdown">G-OnRamp provides a user-friendly, web-based platform for collaborative, end-to-end annotation of eukaryotic genomes using UCSC Assembly Hubs and JBrowse/Apollo genome browsers with evidence tracks derived from sequence alignments, ab initio gene predictors, RNA-Seq data, and repeat finders.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/galaxeast/">GalaxEast</a></td><td class="link"><a href="http://www.galaxeast.fr">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Integrative 'omics data analysis</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/galaxy-africa/">Galaxy Africa</a></td><td class="link"><a href="https://africa.usegalaxy.eu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Devoted to assisting researchers on the African continent accelerate their genomics research and analyses.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/basic-sequence/">Galaxy basic sequence manipulation</a></td><td class="link"></td><td class="link"></td><td class="link"><a href="https://hub.docker.com/r/bgruening/galaxy-sequence-tools/">
            Container
        </a></td><td class="summary markdown">Galaxy Docker repository for basic sequence data analysis.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/blast/">Galaxy BLAST+</a></td><td class="link"></td><td class="link"></td><td class="link"><a href="https://hub.docker.com/r/bgruening/galaxy-blast/">
            Container
        </a></td><td class="summary markdown">A complete and production ready Galaxy instance with installed BLAST+ tools.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/galaxy-docker/">Galaxy Docker</a></td><td class="link"></td><td class="link"></td><td class="link"><a href="https://github.com/bgruening/docker-galaxy-stable">
            Container
        </a></td><td class="summary markdown">The Galaxy Docker Image is an easy distributable full-fledged Galaxy installation, that can be used for testing, teaching and presenting new tools and features.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/exome-seq/">Galaxy Exome-Seq</a></td><td class="link"></td><td class="link"></td><td class="link"><a href="https://hub.docker.com/r/bgruening/galaxy-exome-seq/">
            Container
        </a></td><td class="summary markdown">Galaxy Docker repository for Exome sequencing data analysis</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ecology/">Galaxy for Ecology</a></td><td class="link"><a href="https://ecology.usegalaxy.eu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">A web platform to get, process, analyze and visualize ecological data</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/gga/">Galaxy Genome Annotation (GGA)</a></td><td class="link"><a href="https://annotation.usegalaxy.eu/">
            Server
        </a></td><td class="link"></td><td class="link"><a href="https://hub.docker.com/r/bgruening/galaxy-genome-annotation/">
            Container
        </a></td><td class="summary markdown">Galaxy for genome annotation</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/imaging/">Galaxy Imaging</a></td><td class="link"><a href="https://imaging.usegalaxy.eu/">
            Server
        </a></td><td class="link"></td><td class="link"><a href="https://hub.docker.com/r/bgruening/galaxy-imaging/">
            Container
        </a></td><td class="summary markdown">Galaxy Docker repository for image analysis</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/galaxy-india/">Galaxy India</a></td><td class="link"><a href="https://india.usegalaxy.eu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">This is the Indian galaxy server, Welcome Home!</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/galaxy-integrated-omics-gio/">Galaxy Integrated Omics (GIO)</a></td><td class="link"><a href="http://gio.sbcs.qmul.ac.uk/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Proteomics Informed by Transcriptomics (PIT) methodology, and selection of surrogate peptides for targeted proteomics.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/galaxy-test/">Galaxy Test</a></td><td class="link"><a href="https://test.galaxyproject.org/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Beta version of Galaxy Main</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/msi/">Galaxy Workbench for Mass Spectrometry Imaging</a></td><td class="link"></td><td class="link"></td><td class="link"><a href="https://hub.docker.com/r/mcfoell/galaxy-msi">
            Container
        </a></td><td class="summary markdown">Galaxy Docker image for mass spec imaging</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/galaxy-p/">Galaxy-P</a></td><td class="link"><a href="https://usegalaxyp.org/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Galaxy-P is a multiple 'omics' data analysis platform with particular emphasis on mass spectrometry based proteomics. Galaxy-P is developed at the <a href="http://umn.edu/">University of Minnesota</a>, deployed at the <a href="http://msi.umn.edu/">Minnesota Supercomputing Institute</a>.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/synbiocad/">Galaxy-SynBioCAD</a></td><td class="link"><a href="https://galaxy-synbiocad.org/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Synthetic biology and metabolic engineering</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/galaxy-genouest/">Galaxy@GenOuest</a></td><td class="link"><a href="https://galaxy.genouest.org">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">A general purpose Galaxy server Includes tools developped by <a href="http://www.irisa.fr/dyliss/">Dyliss</a> and <a href="https://team.inria.fr/genscale/">GenScale</a> bioinformatics research teams in Rennes, France.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/galaxy-pasteur/">Galaxy@Pasteur</a></td><td class="link"><a href="https://galaxy.pasteur.fr/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">General purpose genomics analysis server.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/galaxy-prabi/">Galaxy@PRABI</a></td><td class="link"><a href="http://galaxy.prabi.fr/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Includes bioinformatics tools developed by the research teams working in the perimeter of the PRABI core facility, including <em>kissplice/kissDE, TETools, SEX-DETector,</em> and <em>priam</em>.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/galaxytrakr/">GalaxyTrakr</a></td><td class="link"><a href="https://galaxytrakr.org/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Supporting food-borne pathogen research</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/gaslini/">GASLINI</a></td><td class="link"><a href="http://igg.cloud.ba.infn.it/galaxy">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Supports PIPE-T, a tool for Reverse transcription quantitative real-time polymerase chain reaction (RT-qPCR) analysis.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/gcac/">GCAC : Galaxy for Compound Activity Classification</a></td><td class="link"><a href="http://ccbb.jnu.ac.in/gcac">
            Server
        </a></td><td class="link"></td><td class="link"><a href="https://drive.google.com/file/d/1ZQM5x-9J49rvPy9SoWOrM3Xm1MHZdr1I/view?usp=sharing">
            VM
        </a></td><td class="summary markdown">GCAC is a web enabled platform for predictive model building using R and caret package for virtual screening.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/genap/">GenAP</a></td><td class="link"></td><td class="link"><a href="https://www.genap.ca/">
            Academic
        </a></td><td class="link"></td><td class="summary markdown">All Canadian researchers are <a href="https://ccdb.computecanada.ca/account_application">eligible for a GenAP account</a>. International researchers that have a Canadian collaborator can also be sponsored and become eligible to a GenAP account as well.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/genelab/">GeneLab</a></td><td class="link"><a href="https://galaxy.genelab.nasa.gov/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">NASA GeneLab is a comprehensive space-related omics database in which users can upload, download, share, store, and analyze spaceflight and corresponding model organism data.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/genomic-hyperbrowser/">Genomic Hyperbrowser</a></td><td class="link"><a href="http://hyperbrowser.uio.no/hb/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Statistical methodology and computing power to handle a variety of biological inquires on genomic datasets</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/gigagalaxy/">GigaGalaxy</a></td><td class="link"><a href="http://gigagalaxy.net/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">GigaGalaxy hosts tools and data analyses from papers published in <em><a href="http://www.gigasciencejournal.com/">GigaScience</a></em>.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/globus-genomics/">Globus Genomics</a></td><td class="link"></td><td class="link"><a href="http://globusgenomics.org/">
            Academic
        </a><a href="http://globusgenomics.org/">
            Commercial
        </a></td><td class="link"></td><td class="summary markdown">Globus Genomics provides hardened high-performance Galaxy servers running with Globus on AWS.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/graphclust/">GraphClust</a></td><td class="link"><a href="https://graphclust.usegalaxy.eu/">
            Server
        </a></td><td class="link"></td><td class="link"><a href="https://hub.docker.com/r/backofenlab/docker-galaxy-graphclust">
            Container
        </a></td><td class="summary markdown"><a href="https://github.com/BackofenLab/GraphClust-2">GraphClust2</a> is a workflow for scalable clustering of RNAs based on sequence and secondary structures feature. GraphClust2 is implemented within the Galaxy framework and consists a set of integrated Galaxy tools and flavors of the linear-time clustering workflow.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/grappa/">GrAPPA</a></td><td class="link"><a href="https://grappa.eecs.utk.edu/grappa/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">GrAPPA is a web-based interface constructed on the Galaxy framework for graph theoretical tools. It contains novel combinatorial methods integrated into a complete microarray analysis toolchain, from uploading raw high-throughput data to visualization of results.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/halogen-bonding/">Halogen Bonding</a></td><td class="link"><a href="http://134.2.17.68:8081/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Visualize halogen bond contacts in the protein binding site. Analyze a protein binding site for halogen bonding hotspots. Evaluate halogen contacts with the protein backbone using the scoring function XBScore.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/hardwood-genomics/">Hardwood Genomics</a></td><td class="link"><a href="https://www.hardwoodgenomics.org/content/tripal-galaxy-workflows">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">A simplified, workflow-centic interface to Galaxy, focused on genomics data from hardwood trees.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/hic-explorer/">HiCExplorer</a></td><td class="link"><a href="https://hicexplorer.usegalaxy.eu">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Tools to process and visualize chromosome conformation.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/human-cell-atlas/">Human Cell Atlas</a></td><td class="link"><a href="https://humancellatlas.usegalaxy.eu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Tools and workflows for the analysis of Single Cell RNA-Seq data. It includes a module that connects to the Matrix Service API of the <a href="https://www.humancellatlas.org/">Human Cell Atlas</a>’s Data Coordination Platform that enables retrieval of gene expression matrices from any data sets in the Human Cell Atlas.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/huttenhower-lab/">Huttenhower Lab</a></td><td class="link"><a href="http://huttenhower.sph.harvard.edu/galaxy/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">metagenomic and functional genomic analyses, intended for research and academic use</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/hyphy/">HyPhy HIV NGS Tools</a></td><td class="link"><a href="https://galaxy.hyphy.org/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Tools for HIV sequencing analysis.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ibers-aberystwyth/">IBERS @ Aberystwyth</a></td><td class="link"><a href="http://share-galaxy.ibers.aber.ac.uk/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Software, tools and pages written by IBERS researchers for release to the wider community.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/idenap/">IDENAP</a></td><td class="link"><a href="http://mem.rcees.ac.cn:8081/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Inter-Domain Ecological Network Analysis Pipeline</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/immport-galaxy/">ImmPort Galaxy</a></td><td class="link"><a href="https://www.immportgalaxy.org/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Flow Cytometry Analysis.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/immuneml/">immuneML</a></td><td class="link"><a href="https://galaxy.immuneml.uio.no/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Machine learning-based analysis and classification of adaptive immune receptors and repertoires (AIRR).</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/interactomix/">InteractoMIX</a></td><td class="link"><a href="http://galaxy.interactomix.com/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">A one-stop resource offering a wide range of computational analyses from proteome-wide interactomes to structure of protein complexes</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ipk-blast/">IPK Galaxy Blast Suite</a></td><td class="link"><a href="https://galaxy-web.ipk-gatersleben.de">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Part of the IPK Crop Analysis Tools Suite (CATS). Offers access to genomic references published by IPK and collaborative projects</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/jetstream/">Jetstream</a></td><td class="link"></td><td class="link"><a href="http://jetstream-cloud.org/">
            Academic
        </a></td><td class="link"></td><td class="summary markdown"><a href="http://jetstream-cloud.org/">Jetstream</a> is part of <a href="https://www.xsede.org/">XSEDE</a>, a "collection of integrated advanced digital resources and services" and is funded by NSF.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/langille-lab/">Langille Lab</a></td><td class="link"><a href="http://galaxy.morganlangille.com/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Prediction of the unobserved character states in a community of organisms from phylogenetic information.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/laniakea/">Laniakea</a></td><td class="link"></td><td class="link"><a href="https://laniakea-dashboard.cloud.ba.infn.it/login">
            Academic
        </a></td><td class="link"></td><td class="summary markdown">Laniakea provides the possibility to automate the creation of Galaxy-based virtualized environments through an easy setup procedure, providing an on-demand workspace ready to be used by life scientists and bioinformaticians, with built-in storage encryption for user data.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/lapps-grid/">LAPPS Grid</a></td><td class="link"><a href="http://galaxy.lappsgrid.org/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">An open, interoperable web service platform for natural language processing (NLP) research and development.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/lebanese-university/">Lebanese University Galaxy</a></td><td class="link"><a href="http://galaxy.ul.edu.lb/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Includes the SpCLUST tool set for fast and reliable clustering of potentially divergent biological sequences</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/lifeportal-oslo/">Lifeportal</a></td><td class="link"><a href="https://lifeportal.uio.no">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">a Galaxy instance to satisfy the computational needs of the Norwegian research community in life sciences but also of any other user willing to use the service.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/lisis/">LiSIs</a></td><td class="link"><a href="http://lisis.cs.ucy.ac.cy/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">A platform for Virtual Screening.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ml-workbench/">Machine Learning Workbench</a></td><td class="link"><a href="https://ml.usegalaxy.eu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">A comprehensive set of data preprocessing, machine learning, deep learning and visualisation tools, consolidated workflows for end-to-end machine learning analysis and training materials to showcase the usage of these tools.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/majewski-lab/">Majewski Lab</a></td><td class="link"><a href="http://genomequebec.mcgill.ca/exomeai">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Detection of recurrent Allelic Imbalance in tumors using whole Exome sequencing data.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/mandoiu/">Mandoiu Lab</a></td><td class="link"><a href="https://neo.engr.uconn.edu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">A customized installation of the Galaxy framework deploying bioinformatic tools and pipelines developed in the <a href="http://dna.engr.uconn.edu/">Mandoiu Lab</a> at the <a href="http://www.uconn.edu/">University of Connecticut</a> for high-throughput sequencing and immunogenomics data analysis</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/marea4galaxy/">MaREA4Galaxy</a></td><td class="link"><a href="http://bimib.disco.unimib.it:5555/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Metabolic Reaction Enrichment Analysis and visualization of RNA-seq data</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/martin-luther-u-halle-wittenberg/">Martin Luther U. Halle-Wittenberg</a></td><td class="link"><a href="http://galaxy.informatik.uni-halle.de">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Provides implementations of <a href="http://www.jstacs.de/index.php/GeMoMa">GeMoMa</a>, InMoDe, CRISPRer, TALgetter, TALgetterLong, TALENoffer, and Dimont</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/metabolomics/">Metabolomics</a></td><td class="link"><a href="https://metabolomics.usegalaxy.eu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">A community driven metabolomics Galaxy service</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/metacentrum/">MetaCentrum</a></td><td class="link"></td><td class="link"><a href="https://galaxy.metacentrum.cz/">
            Academic
        </a></td><td class="link"></td><td class="summary markdown">MetaCentrum regional virtual organization of the Czech National Grid Infrastructure (NGI) and is accessible to employees and students of research and academic organizations of the Czech Republic and for scientific purposes only.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/metagenomics/">Metagenomics</a></td><td class="link"><a href="https://metagenomics.usegalaxy.eu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">a webserver to process, analyse and visualize Metagenomic and Microbiota data in general.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/metaproteomics-gateway/">Metaproteomics Gateway</a></td><td class="link"><a href="http://z.umn.edu/metaproteomicsgateway">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Metaproteomics gateway providing access to documentation and other instructional materials, and an opportunity for hands-on training using example datasets and optimized metaproteomics workflows</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/migale/">Migale</a></td><td class="link"><a href="http://galaxy.migale.inra.fr/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">A general purpose genomics server with an emphasis on metagenomics, including FROGS and several Migale developed tools.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/mimodd-nacreousmap/">MiModD NacreousMap</a></td><td class="link"><a href="http://mapping-by-sequencing.vm.uni-freiburg.de:8080/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">NacreousMap mapping/plotting tool of MiModD for users of MiModD who do not want to install the required dependencies (R and rpy2) for graphical output from this tool on their local system.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/mississippi/">MISSISSIPPI</a></td><td class="link"><a href="http://mississippi.fr/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Support for analyses of RNA and small RNA sequencing datasets as well as for epigenetics or metagenomics studies.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/mpdsdm/">Molecular Property Diagnostic Suite - Diabetes Mellitus Portal (MPDSDM)</a></td><td class="link"><a href="http://www.mpds-diabetes.in/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">A disease-specific open source web portal for Diabetes Mellitus.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/mpdstb/">Molecular Property Diagnostic Suite - Tuberculosis (MPDSTB)</a></td><td class="link"><a href="http://mpds.osdd.net/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">A disease-specific web portal for Tuberculosis for drug discovery and development.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/nanogalaxy/">NanoGalaxy</a></td><td class="link"><a href="https://nanopore.usegalaxy.eu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">a webserver to process, analyse and visualize Oxford Nanopore Technologies (ONT) data and similar long-reads technologies.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/nectar/">Nectar Cloud</a></td><td class="link"></td><td class="link"><a href="https://nectar.org.au/research-cloud/">
            Academic
        </a></td><td class="link"></td><td class="summary markdown">Set up a Galaxy server on Australia's Nectar research cloud.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/network-analyst/">NetworkAnalyst</a></td><td class="link"><a href="https://galaxy.networkanalyst.ca/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">For mapping and quantification of raw RNA-seq data</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ng-tax/">NG-Tax</a></td><td class="link"><a href="https://ngtax.systemsbiology.nl/">
            Server
        </a></td><td class="link"></td><td class="link"><a href="http://wurssb.gitlab.io/ngtax/galaxy.html">
            Container
        </a></td><td class="summary markdown">A Semantic Framework for High-throughput Marker Gene (Amplicon) Analysis</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ngphylogeny/">NGPhylogeny.fr</a></td><td class="link"><a href="https://ngphylogeny.fr/">
            Server
        </a></td><td class="link"></td><td class="link"><a href="https://hub.docker.com/r/evolbioinfo/ngphylogeny/">
            Container
        </a></td><td class="summary markdown">Robust phylogenetic analysis for everyone.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/oqtans/">Oqtans</a></td><td class="link"><a href="http://galaxy.inf.ethz.ch">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Customized Galaxy, extended with machine learning based tools for sequence and tiling array data analysis.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/orione/">Orione</a></td><td class="link"><a href="http://orione.crs4.it/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">A Galaxy based web server for microbiology. <a href="http://orione.crs4.it/">Orione</a> includes all post mapping or assembling steps from scaffolding to complete annotation pipelines.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/osddlinux-livegalaxy/">OSDDlinux LiveGalaxy</a></td><td class="link"><a href="http://osddlinux.osdd.net:8001/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Drug discovery</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/osiris/">Osiris</a></td><td class="link"><a href="http://galaxy-dev.cnsi.ucsb.edu/osiris/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Phylogenetics</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/palfinder/">Palfinder</a></td><td class="link"><a href="https://palfinder.ls.manchester.ac.uk/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Microsatellite development.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/pepsimili/">PepSimili</a></td><td class="link"><a href="http://pepsimili.e-nios.com:8080/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Integrated workflow tool for mapping of massive peptide repertoires on whole proteomes and delivering streamlined, systems-level biological interpretation.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/phagepromoter/">PhagePromotor</a></td><td class="link"><a href="https://galaxy.bio.di.uminho.pt/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Predict promoter sequences in phage genomes, using machine learning models.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/phenomenal/">PhenoMeNal</a></td><td class="link"></td><td class="link"><a href="https://portal.phenomenal-h2020.eu/cloud-research-environment-setup">
            Academic
        </a><a href="https://portal.phenomenal-h2020.eu/cloud-research-environment-setup">
            Commercial
        </a></td><td class="link"></td><td class="summary markdown">Large scale computing for medical informatics</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/pia/">PIA</a></td><td class="link"><a href="http://galaxy-dev.cnsi.ucsb.edu/pia/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Phylogenetics and gene interaction</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/pirate/">PiRATE: a Pipeline to Retrieve and Annotate Transposable Elements</a></td><td class="link"></td><td class="link"></td><td class="link"><a href="https://www.seanoe.org/data/00406/51795/">
            VM
        </a></td><td class="summary markdown">Detect, classify and annotate TEs of non-model organisms.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/plants/">Plant Anaylsis Workbench</a></td><td class="link"><a href="https://plants.usegalaxy.eu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">A community driven plant data analsys Galaxy service</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/plantgenie/">PlantGenIE</a></td><td class="link"><a href="http://galaxy.plantgenie.org:8080/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Plant Genome Integrative Explorer Galaxy instance</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/prestige/">PreSTIGE</a></td><td class="link"><a href="http://prestige.case.edu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Predicting Specific Tissue Interactions of Genes and Enhancers</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/proteogenomics-gateway/">Proteogenomics Gateway</a></td><td class="link"><a href="http://z.umn.edu/proteogenomicsgateway">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Provides access to documentation and other instructional materials, and an opportunity for hands-on training using example datasets and optimized proteogenomics workflows. The main goal of this server is to provide documentation to facilitate training and mastery of these software and workflows.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/proteore/">ProteoRE</a></td><td class="link"><a href="http://www.proteore.org/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">ProteoRE (Proteomics Research Environment)</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/protologger/">Protologger</a></td><td class="link"><a href="http://protologger.de/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">All-in-one genome description tool, aimed at simplifying gathering the data required for writing protologues. This includes providing taxonomic, functional and ecological insights.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/qiime2/">q2Galaxy</a></td><td class="link"></td><td class="link"></td><td class="link"><a href="https://quay.io/repository/qiime2/q2galaxy">
            Container
        </a></td><td class="summary markdown">QIIME 2 Galaxy Docker image.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/quantp/">QuanTP</a></td><td class="link"></td><td class="link"></td><td class="link"><a href="https://z.umn.edu/quantpdocker">
            Container
        </a></td><td class="summary markdown">A Docker image containing a fully-operational Galaxy instance with pre-installed QuanTP tools.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/repeatexplorer/">RepeatExplorer</a></td><td class="link"><a href="https://repeatexplorer-elixir.cerit-sc.cz/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Graph-based clustering and characterization of repetitive sequences, and detection of transposable element protein coding domains.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ribogalaxy/">RiboGalaxy</a></td><td class="link"><a href="http://ribogalaxy.ucc.ie/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Tools for the analysis and visualization of ribo-seq data obtained with the ribosome profiling technique.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/rice-galaxy/">Rice Galaxy</a></td><td class="link"><a href="http://galaxy.irri.org/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Tools for designing single-nucleotide polymorphism assays, analyzing genome-wide association studies, population diversity, rice−bacterial pathogen diagnostics, and a suite of published genomic prediction methods.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/rna-modification-calling/">RNA Modification Calling</a></td><td class="link"></td><td class="link"></td><td class="link"><a href="https://github.com/HelmGroup/Galaxy_modification_calling">
            Container
        </a></td><td class="summary markdown">Galaxy Docker image for RNA analysis and modification calling</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/rna-workbench/">RNA Workbench</a></td><td class="link"><a href="https://rna.usegalaxy.eu/">
            Server
        </a></td><td class="link"></td><td class="link"><a href="https://github.com/bgruening/galaxy-rna-workbench">
            Container
        </a></td><td class="summary markdown">more than 50 bioinformatics tools dedicated to different research areas of RNA biology, including RNA structure analysis, RNA alignment, RNA annotation, RNA-protein interaction, ribosome profiling, RNA-Seq analysis, and RNA target prediction.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ena-submission/">SARS-CoV-2 ENA Submission</a></td><td class="link"></td><td class="link"></td><td class="link"><a href="https://quay.io/repository/galaxy/ena-upload?tag=latest&amp;tab=tags">
            Container
        </a></td><td class="summary markdown">Galaxy Docker image for submitting SARS-CoV-2 data to ENA</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/sciensano/">Sciensano</a></td><td class="link"><a href="https://galaxy.sciensano.be/policy/disclaimer.html">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">General purpose Galaxy server optimized for microbiological genomics applications of interest for public health.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/shaman/">SHAMAN</a></td><td class="link"><a href="https://shaman.pasteur.fr/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">A user-friendly website for metataxonomic analysis from raw reads to statistical analysis</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/shapiro-lab/">Shapiro Lab</a></td><td class="link"></td><td class="link"></td><td class="link"><a href="https://binkley2.ncifcrf.gov/users/bshapiro//software.html">
            VM
        </a></td><td class="summary markdown">RNA structure and design tools including RNA2D3D and StructureLab</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/singlecell/">Single Cell Omics workbench</a></td><td class="link"><a href="https://singlecell.usegalaxy.eu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">The Single Cell Omics workbench is a comprehensive set of analysis tools and consolidated workflows</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/snic-science-cloud/">SNIC Science Cloud (SSC)</a></td><td class="link"></td><td class="link"><a href="https://cloud.snic.se/">
            Academic
        </a></td><td class="link"></td><td class="summary markdown">A national-scale cloud infrastructure for Swedish Academia</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/streetscience/">Street Science</a></td><td class="link"><a href="https://streetscience.usegalaxy.eu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">a Galaxy dedicated for the Street Science Community</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/tgm/">TGM</a></td><td class="link"><a href="http://tgm-pipeline.uni.lu/galaxy/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">The <em>TGM pipeline</em> is a workflow combining tranSMART server, Galaxy Server and MINERVA platform to enable visually-aided exploration, analysis and interpretation of high-throughput translational medicine data.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/treegenes/">TreeGenes</a></td><td class="link"><a href="https://treegenes.cam.uchc.edu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">TreeGenes integrates genotype, phenotype, and environmental data that can be processed for association mapping and landscape genomics, with a focus on forest trees. This functionality is provided by <a href="https://github.com/tripal/tripal_galaxy">Tripal Galaxy</a>, a module for <a href="http://tripal.info">Tripal</a> that provides seamless support for running Galaxy workflows directly in the site. Users can analyze data provided by TreeGenes or their own.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/usegalaxy-be/">UseGalaxy.be</a></td><td class="link"><a href="https://usegalaxy.be/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">General purpose genomics Galaxy server</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/usegalaxy-eu/">UseGalaxy.eu</a></td><td class="link"><a href="https://usegalaxy.eu">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">The European UseGalaxy server.</td><!----><td class="keywords"><a href="/use/#usegalaxy-dir">
            UseGalaxy
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/usegalaxy-fr/">UseGalaxy.fr</a></td><td class="link"><a href="https://usegalaxy.fr/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">General purpose genomics Galaxy server</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/usegalaxy-no/">UseGalaxy.no</a></td><td class="link"><a href="https://usegalaxy.no">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">A central, joint &#x26; free Galaxy instance for Norway-based researchers and their collaborators, with ~2000 bioinformatics tools, 200GB short-term storage and project storage on nels.bioinfo.no.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/usegalaxy-org/">UseGalaxy.org (Main)</a></td><td class="link"><a href="https://usegalaxy.org">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">The <a href="/galaxy-project/">Galaxy Project</a> free public server; biomedical research</td><!----><td class="keywords"><a href="/use/#usegalaxy-dir">
            UseGalaxy
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/usegalaxy-au/">UseGalaxy.org.au</a></td><td class="link"><a href="https://usegalaxy.org.au/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">The Australian UseGalaxy server.</td><!----><td class="keywords"><a href="/use/#usegalaxy-dir">
            UseGalaxy
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/usmi-galaxy-demonstrator/">USMI Galaxy Demonstrator</a></td><td class="link"><a href="http://bioinformatics.hsanmartino.it:8080/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">This Galaxy implementation is meant to demonstrate integration of microbial domain Biological Resource Center (mBRC) catalogues into the bioinformatics data integration environment.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/veupathdb/">VEuPathDB Galaxy</a></td><td class="link"></td><td class="link"><a href="https://veupathdb.org/veupathdb/app/galaxy-orientation">
            Academic
        </a></td><td class="link"></td><td class="summary markdown">A free, interactive, web-based platform for large-scale data analysis focused on eukaryotic pathogens. Merges the former VectorBase and EuPathDB servers.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/vinther-lab/">Vinther Lab</a></td><td class="link"><a href="http://galaxy.bio.ku.dk/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">RNA structure-probing data analysis to improve the prediction of RNA secondary and tertiary structure and allow structural changes to be identified and investigated.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/vinyl/">VINYL (Variant prIoritizatioN bY survivaL analysis)</a></td><td class="link"><a href="http://www.beaconlab.it/VINYL">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">A flexible and fully automated system for the functional annotation and prioritization of genetic variants.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/vvv/">Viral Variant Visualizer (VVV)</a></td><td class="link"><a href="https://viralvariant.anses.fr/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">Determine the composition of a viral population</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/viramp/">VirAmp</a></td><td class="link"><a href="http://viramp.com/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">VirAmp is a web-based semi-<em>de novo</em> fast virus genome assembly pipeline designed for extremely high coverage NGS data.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/virology/">Virology</a></td><td class="link"><a href="https://virology.usegalaxy.eu/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">A comprehensive set of analysis tools and consolidated workflows for virology work, including SARS-CoV-2.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/workflow4metabolomics/">Workflow4Metabolomics</a></td><td class="link"><a href="https://workflow4metabolomics.usegalaxy.fr">
            Server
        </a></td><td class="link"></td><td class="link"><a href="https://hub.docker.com/r/workflow4metabolomics/galaxy-workflow4metabolomics/">
            Container
        </a></td><td class="summary markdown">A collaborative portal dedicated to metabolomics data processing, analysis and annotation.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/wqap/">wQAP</a></td><td class="link"><a href="http://life2cloud.com:6005/">
            Server
        </a></td><td class="link"></td><td class="link"></td><td class="summary markdown">QAP is an open source software that can be used to analyze virus quasispecies using CBS, NGS and TGS data. wQAP (web QAP), is the online version of QAP.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr></tbody></table><!----></div><div id="public-server-pane" data-toggle-group="use-panes" aria-labelledby="public-server-tab" role="tabpanel" class="tab-pane fade show" data-v-b620dc4c><h2 class="nav-item" data-v-b620dc4c>
                    Public Servers
                </h2><p class="markdown" data-v-b620dc4c><p>Public Servers are acccessible to (at least) any academic researcher in the world.  Some require you to create/request an account, and they may have restrictive quotas, but often, to use these, just go the website and start running analyses. These are free to use.</p>
<p>The folks are Freiburg have put together a <a href="https://stats.galaxyproject.eu/d/000000020/public-galaxy-servers?orgId=1" target="_blank" rel="noopener noreferrer">public Galaxy server dashboard</a> to track the status and some of the supported features of the servers listed here. See the <a href="/blog/2017-10-public-galaxy-dashboard/">blog post</a> for more info.</p>
</p><p class="d-none" data-v-b620dc4c>dummy text</p><table class="table table-striped" data-v-b620dc4c><thead data-v-b620dc4c><tr data-v-b620dc4c><th data-v-b620dc4c>Resource</th><th data-v-b620dc4c>Link</th><th data-v-b620dc4c>Summary</th><th data-v-b620dc4c>Keywords</th></tr></thead><tbody data-v-b620dc4c><tr data-v-b620dc4c><td class="title"><a href="/use/ab-openlab/">AB-OpenLab</a></td><td class="link"><a href="http://ab-openlab.csir.res.in/frog/">
            Server
        </a></td><td class="summary markdown">FROG stands for <strong>F</strong>inge<strong>R</strong>printing <strong>O</strong>ntology of <strong>G</strong>enomic variations. FROG fingerprints have been devised to capture genomic variations at various levels.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/abims/">ABiMS</a></td><td class="link"><a href="http://galaxy.sb-roscoff.fr/">
            Server
        </a></td><td class="summary markdown">Analysis and bioinformatics for Marine Science.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/abims-tools/">ABiMS-Tools</a></td><td class="link"><a href="http://galaxy3.sb-roscoff.fr/">
            Server
        </a></td><td class="summary markdown">Hosts tools/resources developed at the <a href="http://www.sb-roscoff.fr/">Station Biologique de Roscoff</a> in collaboration with <a href="http://abims.sb-roscoff.fr/">ABiMS</a>.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ageseq-aspendb/">AGEseq @ AspenDB</a></td><td class="link"><a href="http://aspendb.uga.edu:8085/">
            Server
        </a></td><td class="summary markdown">This site provides a user-friendly interface for AGEseq (Analysis of Genome Editing by Sequencing) in a Galaxy instance.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/anastasia/">ANASTASIA</a></td><td class="link"><a href="http://motherbox.chemeng.ntua.gr/anastasia_dev/">
            Server
        </a></td><td class="summary markdown"><strong>A</strong>utomated <strong>N</strong>ucleotide <strong>A</strong>minoacid <strong>S</strong>equences <strong>T</strong>ranslational pl<strong>A</strong>tform for <strong>S</strong>ystemic <strong>I</strong>nterpretation and <strong>A</strong>nalysis</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/apostl/">APOSTL</a></td><td class="link"><a href="http://apostl.moffitt.org/">
            Server
        </a></td><td class="summary markdown">Automated Processing of SAINT Templated Layouts (APOSTL) is a freely available software suite and analysis pipeline for reproducible, interactive analysis of AP-MS data.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/arga/">ARGA</a></td><td class="link"><a href="http://mem.rcees.ac.cn:8083/">
            Server
        </a></td><td class="summary markdown">Antibiotic Resistance Gene Analyzer</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/args-oap/">ARGs-OAP</a></td><td class="link"><a href="http://smile.hku.hk/SARGs">
            Server
        </a></td><td class="summary markdown">Fast annotation and classification of antibiotic resistance gene-like sequences from metagenomic data.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ariaweb/">ARIAweb</a></td><td class="link"><a href="https://ariaweb.pasteur.fr/">
            Server
        </a></td><td class="summary markdown">Automated NOE assignment and NMR structure calculation</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/aries/">ARIES</a></td><td class="link"><a href="https://w3.iss.it/site/aries/">
            Server
        </a></td><td class="summary markdown">Analysis of genomic data in the field of public health and food safety, with the aim of deploying a comprehensive bioinformatics approach to the study of food-borne zoonoses and infectious diseases at the human and animal interface.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/bf2i-map/">BF2I-MAP</a></td><td class="link"><a href="http://bf2i-galaxy.insa-lyon.fr:8080/">
            Server
        </a></td><td class="summary markdown">Locally developed as well as standard tools are available on a 16-core server with 32GB memory, accessible to all users, and located in the BF2I lab.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/biobix/">BioBix</a></td><td class="link"><a href="http://galaxy.ugent.be/">
            Server
        </a></td><td class="summary markdown">A general genomics server. Includes the PROTEOFORMER pipeline tools.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/biociphers-lab-galaxy/">BioCiphers Lab Galaxy</a></td><td class="link"><a href="http://avispa.biociphers.org/">
            Server
        </a></td><td class="summary markdown">BCL Galaxy gives a user friendly interface for analysis tools developed by the <a href="http://biociphers.org/">BioCiphers Lab</a> at the University of Pennsylvania.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/biocompute-object/">BioCompute Object Galaxy</a></td><td class="link"><a href="https://galaxy.aws.biochemistry.gwu.edu/">
            Server
        </a></td><td class="summary markdown"><a href="https://biocomputeobject.org/about.html">BioCompute Objects</a>  is a standard for communication of High-throughput Sequencing (HTS) analysis results, data set creation and curation, and bioinformatics verification protocols.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/biodivine/">BioDivine Toolset</a></td><td class="link"><a href="https://biodivine-vm.fi.muni.cz/galaxy/">
            Server
        </a></td><td class="summary markdown">Systems biology tools implemented by the Sybila team.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/biomas/">BioMaS</a></td><td class="link"><a href="http://galaxy.cloud.ba.infn.it:8080/">
            Server
        </a></td><td class="summary markdown">Taxonomic studies of environmental microbial communities</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/biomina/">Biomina</a></td><td class="link"><a href="http://143.169.238.104/galaxy/">
            Server
        </a></td><td class="summary markdown">A general purpose Galaxy instance that includes most standard tools for DNA/RNA sequencing, plus extra tools for panel resequencing, variant annotation and some tools for Illumina SNParray analysis.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/bipaa/">BIPAA (BioInformatics Platform for Agroecosystem Arthropods)</a></td><td class="link"><a href="https://bipaa-galaxy.genouest.org">
            Server
        </a></td><td class="summary markdown">Insect genomics (aphids, parasitoïd wasps, lepidopterans)</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/bitlab/">BitLAB</a></td><td class="link"><a href="https://pistacho.ac.uma.es/galaxy-bitlab">
            Server
        </a></td><td class="summary markdown">Detailed analysis of metagenomes.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/bridge/">BRIDGE</a></td><td class="link"><a href="https://galaxy-compchem.ilifu.ac.za/">
            Server
        </a></td><td class="summary markdown">Prepare, simulate and analyse and visualise molecular simulations, especially molecular dynamics and free energies.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/cafu/">CAFU</a></td><td class="link"><a href="http://omicstudio.cloud:4001/">
            Server
        </a></td><td class="summary markdown">A Galaxy framework for exploring unmapped RNA-Seq data</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/cbib-galaxy/">CBiB Galaxy</a></td><td class="link"><a href="http://services.cbib.u-bordeaux2.fr/galaxy/">
            Server
        </a></td><td class="summary markdown">A general purpose Galaxy instance that includes EMBOSS (a software analysis package for molecular biology) and fibronectin (diversity analysis of synthetic libraries of a Fibronectin domain).</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/cell-organizer/">CellOrganizer for Galaxy</a></td><td class="link"><a href="http://galaxy3.compbio.cs.cmu.edu:9001">
            Server
        </a></td><td class="summary markdown">Create generative models of the spatial organization of cells from microscope images and automatically provide geometries for spatial simulations of cell processes and behaviors</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/center-for-phage-technology-cpt/">Center for Phage Technology (CPT)</a></td><td class="link"><a href="https://cpt.tamu.edu/galaxy-pub/">
            Server
        </a></td><td class="summary markdown">Phage biology and automated annotation.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/chemflow/">ChemFlow</a></td><td class="link"><a href="https://vm-chemflow-francegrille.eu/">
            Server
        </a></td><td class="summary markdown">Chemometrics without programming: infrared spectrometry on a Galaxy platform.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/cart/">Chemical Annotation Retrieval Toolkit (CART)</a></td><td class="link"><a href="http://cart.embl.de/">
            Server
        </a></td><td class="summary markdown">Retrieves annotations on biological effects of chemicals and determines which ones are enriched.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/chemicaltoolbox/">ChemicalToolbox</a></td><td class="link"><a href="https://cheminformatics.usegalaxy.eu/">
            Server
        </a></td><td class="summary markdown">For processing, analysing and visualising chemical data, and performing molecular simulations.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/cirm-cfbp/">CIRM-CFBP</a></td><td class="link"><a href="https://iris.angers.inra.fr/galaxypub-cfbp">
            Server
        </a></td><td class="summary markdown">Web based access to tools (<a href="https://sourcesup.renater.fr/wiki/skif/">SkIf</a>, <a href="https://sourcesup.renater.fr/wiki/ki-s/">KI-S</a>) of IRHS-EmerSys lab.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/cistrome-analysis-pipeline/">Cistrome Analysis Pipeline</a></td><td class="link"><a href="http://cistrome.org/ap/root">
            Server
        </a></td><td class="summary markdown">ChIP-chip/seq and gene expression data</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/climate-science-workbench/">Climate Science Workbench</a></td><td class="link"><a href="https://climate.usegalaxy.eu/">
            Server
        </a></td><td class="summary markdown">A comprehensive set of analysis tools and consolidated workflows for climate analysis and visualisation</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/clip-explorer/">CLIP-Explorer</a></td><td class="link"><a href="https://clipseq.usegalaxy.eu/">
            Server
        </a></td><td class="summary markdown">The CLIP-Explorer is a webserver to process, analyse and visualise CLIP-Seq data.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/codon-harmonizer/">Codon Harmonizer</a></td><td class="link"><a href="http://codonharmonizer.systemsbiology.nl/">
            Server
        </a></td><td class="summary markdown">Easily generate codon harmonized variants of gene sequences for heterologous expression</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/coloc-stats/">Coloc-stats</a></td><td class="link"><a href="https://hyperbrowser.uio.no/coloc-stats">
            Server
        </a></td><td class="summary markdown">a unified web interface to perform co-localization analysis of genomic features</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/coral-snp/">CoralSNP</a></td><td class="link"><a href="https://coralsnp.science.psu.edu/galaxy/">
            Server
        </a></td><td class="summary markdown">Standard Tools for Acroporid Genotyping (STAG)</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/corgat/">CorGAT</a></td><td class="link"><a href="http://corgat.cloud.ba.infn.it/galaxy">
            Server
        </a></td><td class="summary markdown">Align complete assemblies of SARS-CoV-2 genomes wih the reference genomic sequence, to obtain a list of polymorphic positions and to annotate genetic variants.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/cossci/">CoSSci</a></td><td class="link"><a href="http://socscicompute.ss.uci.edu/">
            Server
        </a></td><td class="summary markdown">Tools for solving <a href="http://en.wikipedia.org/wiki/Galton&#x27;s_problem">Galton's problem</a> in Comparative Research and complex network problems in Social Science.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/covid-19/">COVID-19</a></td><td class="link"><a href="https://covid19.usegalaxy.eu/">
            Server
        </a></td><td class="summary markdown">A companion to our study describing the analysis of early COVID-19 data</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/cropgalaxy/">CropGalaxy</a></td><td class="link"><a href="http://cropgalaxy.excellenceinbreeding.org/">
            Server
        </a></td><td class="summary markdown">Hosts tools for the <a href="/src/use/ricegalaxy/index.md">RiceGalaxy</a>, <a href="http://gobiiproject.org/">GOBii project</a>, <a href="http://cegsb.icrisat.org/high-throughput-genotyping-project-htpg/">HTPG project</a> and the <a href="http://excellenceinbreeding.org/">Excellence in Breeding (EiB) platform</a>, including genomic selection, marker selection, GWAS, imputation, file conversion, and cluster analysis.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/deepts/">deepTS</a></td><td class="link"><a href="https://deepts.nwafu.edu.cn/">
            Server
        </a></td><td class="summary markdown">Explore transcriptional switches from pairwise, temporal, and population RNA-Seq using deepTS.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/dintor/">Dintor</a></td><td class="link"><a href="http://dintor.eurac.edu/">
            Server
        </a></td><td class="summary markdown">GWA and NGS tools and modules for functional annotation of genes and gene products</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/epigeec/">epiGeEC</a></td><td class="link"><a href="https://epigeec.genap.ca/galaxy/">
            Server
        </a></td><td class="summary markdown">Compare user-provided epigenomic datasets with thousands of public datasets.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/erasmus-mc/">Erasmus MC</a></td><td class="link"><a href="https://erasmusmc.usegalaxy.eu/">
            Server
        </a></td><td class="summary markdown">General purpose genomics analysis, featuring many standard tools plus many additional tools.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/freebioinfo/">FreeBioinfo</a></td><td class="link"><a href="http://www.freebioinfo.org/">
            Server
        </a></td><td class="summary markdown">Galaxy for metagenomic analysis.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/galaxeast/">GalaxEast</a></td><td class="link"><a href="http://www.galaxeast.fr">
            Server
        </a></td><td class="summary markdown">Integrative 'omics data analysis</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/galaxy-africa/">Galaxy Africa</a></td><td class="link"><a href="https://africa.usegalaxy.eu/">
            Server
        </a></td><td class="summary markdown">Devoted to assisting researchers on the African continent accelerate their genomics research and analyses.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ecology/">Galaxy for Ecology</a></td><td class="link"><a href="https://ecology.usegalaxy.eu/">
            Server
        </a></td><td class="summary markdown">A web platform to get, process, analyze and visualize ecological data</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/gga/">Galaxy Genome Annotation (GGA)</a></td><td class="link"><a href="https://annotation.usegalaxy.eu/">
            Server
        </a></td><td class="summary markdown">Galaxy for genome annotation</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/imaging/">Galaxy Imaging</a></td><td class="link"><a href="https://imaging.usegalaxy.eu/">
            Server
        </a></td><td class="summary markdown">Galaxy Docker repository for image analysis</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/galaxy-india/">Galaxy India</a></td><td class="link"><a href="https://india.usegalaxy.eu/">
            Server
        </a></td><td class="summary markdown">This is the Indian galaxy server, Welcome Home!</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/galaxy-integrated-omics-gio/">Galaxy Integrated Omics (GIO)</a></td><td class="link"><a href="http://gio.sbcs.qmul.ac.uk/">
            Server
        </a></td><td class="summary markdown">Proteomics Informed by Transcriptomics (PIT) methodology, and selection of surrogate peptides for targeted proteomics.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/galaxy-test/">Galaxy Test</a></td><td class="link"><a href="https://test.galaxyproject.org/">
            Server
        </a></td><td class="summary markdown">Beta version of Galaxy Main</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/galaxy-p/">Galaxy-P</a></td><td class="link"><a href="https://usegalaxyp.org/">
            Server
        </a></td><td class="summary markdown">Galaxy-P is a multiple 'omics' data analysis platform with particular emphasis on mass spectrometry based proteomics. Galaxy-P is developed at the <a href="http://umn.edu/">University of Minnesota</a>, deployed at the <a href="http://msi.umn.edu/">Minnesota Supercomputing Institute</a>.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/synbiocad/">Galaxy-SynBioCAD</a></td><td class="link"><a href="https://galaxy-synbiocad.org/">
            Server
        </a></td><td class="summary markdown">Synthetic biology and metabolic engineering</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/galaxy-genouest/">Galaxy@GenOuest</a></td><td class="link"><a href="https://galaxy.genouest.org">
            Server
        </a></td><td class="summary markdown">A general purpose Galaxy server Includes tools developped by <a href="http://www.irisa.fr/dyliss/">Dyliss</a> and <a href="https://team.inria.fr/genscale/">GenScale</a> bioinformatics research teams in Rennes, France.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/galaxy-pasteur/">Galaxy@Pasteur</a></td><td class="link"><a href="https://galaxy.pasteur.fr/">
            Server
        </a></td><td class="summary markdown">General purpose genomics analysis server.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/galaxy-prabi/">Galaxy@PRABI</a></td><td class="link"><a href="http://galaxy.prabi.fr/">
            Server
        </a></td><td class="summary markdown">Includes bioinformatics tools developed by the research teams working in the perimeter of the PRABI core facility, including <em>kissplice/kissDE, TETools, SEX-DETector,</em> and <em>priam</em>.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/galaxytrakr/">GalaxyTrakr</a></td><td class="link"><a href="https://galaxytrakr.org/">
            Server
        </a></td><td class="summary markdown">Supporting food-borne pathogen research</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/gaslini/">GASLINI</a></td><td class="link"><a href="http://igg.cloud.ba.infn.it/galaxy">
            Server
        </a></td><td class="summary markdown">Supports PIPE-T, a tool for Reverse transcription quantitative real-time polymerase chain reaction (RT-qPCR) analysis.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/gcac/">GCAC : Galaxy for Compound Activity Classification</a></td><td class="link"><a href="http://ccbb.jnu.ac.in/gcac">
            Server
        </a></td><td class="summary markdown">GCAC is a web enabled platform for predictive model building using R and caret package for virtual screening.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/genelab/">GeneLab</a></td><td class="link"><a href="https://galaxy.genelab.nasa.gov/">
            Server
        </a></td><td class="summary markdown">NASA GeneLab is a comprehensive space-related omics database in which users can upload, download, share, store, and analyze spaceflight and corresponding model organism data.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/genomic-hyperbrowser/">Genomic Hyperbrowser</a></td><td class="link"><a href="http://hyperbrowser.uio.no/hb/">
            Server
        </a></td><td class="summary markdown">Statistical methodology and computing power to handle a variety of biological inquires on genomic datasets</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/gigagalaxy/">GigaGalaxy</a></td><td class="link"><a href="http://gigagalaxy.net/">
            Server
        </a></td><td class="summary markdown">GigaGalaxy hosts tools and data analyses from papers published in <em><a href="http://www.gigasciencejournal.com/">GigaScience</a></em>.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/graphclust/">GraphClust</a></td><td class="link"><a href="https://graphclust.usegalaxy.eu/">
            Server
        </a></td><td class="summary markdown"><a href="https://github.com/BackofenLab/GraphClust-2">GraphClust2</a> is a workflow for scalable clustering of RNAs based on sequence and secondary structures feature. GraphClust2 is implemented within the Galaxy framework and consists a set of integrated Galaxy tools and flavors of the linear-time clustering workflow.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/grappa/">GrAPPA</a></td><td class="link"><a href="https://grappa.eecs.utk.edu/grappa/">
            Server
        </a></td><td class="summary markdown">GrAPPA is a web-based interface constructed on the Galaxy framework for graph theoretical tools. It contains novel combinatorial methods integrated into a complete microarray analysis toolchain, from uploading raw high-throughput data to visualization of results.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/halogen-bonding/">Halogen Bonding</a></td><td class="link"><a href="http://134.2.17.68:8081/">
            Server
        </a></td><td class="summary markdown">Visualize halogen bond contacts in the protein binding site. Analyze a protein binding site for halogen bonding hotspots. Evaluate halogen contacts with the protein backbone using the scoring function XBScore.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/hardwood-genomics/">Hardwood Genomics</a></td><td class="link"><a href="https://www.hardwoodgenomics.org/content/tripal-galaxy-workflows">
            Server
        </a></td><td class="summary markdown">A simplified, workflow-centic interface to Galaxy, focused on genomics data from hardwood trees.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/hic-explorer/">HiCExplorer</a></td><td class="link"><a href="https://hicexplorer.usegalaxy.eu">
            Server
        </a></td><td class="summary markdown">Tools to process and visualize chromosome conformation.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/human-cell-atlas/">Human Cell Atlas</a></td><td class="link"><a href="https://humancellatlas.usegalaxy.eu/">
            Server
        </a></td><td class="summary markdown">Tools and workflows for the analysis of Single Cell RNA-Seq data. It includes a module that connects to the Matrix Service API of the <a href="https://www.humancellatlas.org/">Human Cell Atlas</a>’s Data Coordination Platform that enables retrieval of gene expression matrices from any data sets in the Human Cell Atlas.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/huttenhower-lab/">Huttenhower Lab</a></td><td class="link"><a href="http://huttenhower.sph.harvard.edu/galaxy/">
            Server
        </a></td><td class="summary markdown">metagenomic and functional genomic analyses, intended for research and academic use</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/hyphy/">HyPhy HIV NGS Tools</a></td><td class="link"><a href="https://galaxy.hyphy.org/">
            Server
        </a></td><td class="summary markdown">Tools for HIV sequencing analysis.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ibers-aberystwyth/">IBERS @ Aberystwyth</a></td><td class="link"><a href="http://share-galaxy.ibers.aber.ac.uk/">
            Server
        </a></td><td class="summary markdown">Software, tools and pages written by IBERS researchers for release to the wider community.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/idenap/">IDENAP</a></td><td class="link"><a href="http://mem.rcees.ac.cn:8081/">
            Server
        </a></td><td class="summary markdown">Inter-Domain Ecological Network Analysis Pipeline</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/immport-galaxy/">ImmPort Galaxy</a></td><td class="link"><a href="https://www.immportgalaxy.org/">
            Server
        </a></td><td class="summary markdown">Flow Cytometry Analysis.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/immuneml/">immuneML</a></td><td class="link"><a href="https://galaxy.immuneml.uio.no/">
            Server
        </a></td><td class="summary markdown">Machine learning-based analysis and classification of adaptive immune receptors and repertoires (AIRR).</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/interactomix/">InteractoMIX</a></td><td class="link"><a href="http://galaxy.interactomix.com/">
            Server
        </a></td><td class="summary markdown">A one-stop resource offering a wide range of computational analyses from proteome-wide interactomes to structure of protein complexes</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ipk-blast/">IPK Galaxy Blast Suite</a></td><td class="link"><a href="https://galaxy-web.ipk-gatersleben.de">
            Server
        </a></td><td class="summary markdown">Part of the IPK Crop Analysis Tools Suite (CATS). Offers access to genomic references published by IPK and collaborative projects</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/langille-lab/">Langille Lab</a></td><td class="link"><a href="http://galaxy.morganlangille.com/">
            Server
        </a></td><td class="summary markdown">Prediction of the unobserved character states in a community of organisms from phylogenetic information.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/lapps-grid/">LAPPS Grid</a></td><td class="link"><a href="http://galaxy.lappsgrid.org/">
            Server
        </a></td><td class="summary markdown">An open, interoperable web service platform for natural language processing (NLP) research and development.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/lebanese-university/">Lebanese University Galaxy</a></td><td class="link"><a href="http://galaxy.ul.edu.lb/">
            Server
        </a></td><td class="summary markdown">Includes the SpCLUST tool set for fast and reliable clustering of potentially divergent biological sequences</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/lifeportal-oslo/">Lifeportal</a></td><td class="link"><a href="https://lifeportal.uio.no">
            Server
        </a></td><td class="summary markdown">a Galaxy instance to satisfy the computational needs of the Norwegian research community in life sciences but also of any other user willing to use the service.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/lisis/">LiSIs</a></td><td class="link"><a href="http://lisis.cs.ucy.ac.cy/">
            Server
        </a></td><td class="summary markdown">A platform for Virtual Screening.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ml-workbench/">Machine Learning Workbench</a></td><td class="link"><a href="https://ml.usegalaxy.eu/">
            Server
        </a></td><td class="summary markdown">A comprehensive set of data preprocessing, machine learning, deep learning and visualisation tools, consolidated workflows for end-to-end machine learning analysis and training materials to showcase the usage of these tools.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/majewski-lab/">Majewski Lab</a></td><td class="link"><a href="http://genomequebec.mcgill.ca/exomeai">
            Server
        </a></td><td class="summary markdown">Detection of recurrent Allelic Imbalance in tumors using whole Exome sequencing data.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/mandoiu/">Mandoiu Lab</a></td><td class="link"><a href="https://neo.engr.uconn.edu/">
            Server
        </a></td><td class="summary markdown">A customized installation of the Galaxy framework deploying bioinformatic tools and pipelines developed in the <a href="http://dna.engr.uconn.edu/">Mandoiu Lab</a> at the <a href="http://www.uconn.edu/">University of Connecticut</a> for high-throughput sequencing and immunogenomics data analysis</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/marea4galaxy/">MaREA4Galaxy</a></td><td class="link"><a href="http://bimib.disco.unimib.it:5555/">
            Server
        </a></td><td class="summary markdown">Metabolic Reaction Enrichment Analysis and visualization of RNA-seq data</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/martin-luther-u-halle-wittenberg/">Martin Luther U. Halle-Wittenberg</a></td><td class="link"><a href="http://galaxy.informatik.uni-halle.de">
            Server
        </a></td><td class="summary markdown">Provides implementations of <a href="http://www.jstacs.de/index.php/GeMoMa">GeMoMa</a>, InMoDe, CRISPRer, TALgetter, TALgetterLong, TALENoffer, and Dimont</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/metabolomics/">Metabolomics</a></td><td class="link"><a href="https://metabolomics.usegalaxy.eu/">
            Server
        </a></td><td class="summary markdown">A community driven metabolomics Galaxy service</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/metagenomics/">Metagenomics</a></td><td class="link"><a href="https://metagenomics.usegalaxy.eu/">
            Server
        </a></td><td class="summary markdown">a webserver to process, analyse and visualize Metagenomic and Microbiota data in general.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/metaproteomics-gateway/">Metaproteomics Gateway</a></td><td class="link"><a href="http://z.umn.edu/metaproteomicsgateway">
            Server
        </a></td><td class="summary markdown">Metaproteomics gateway providing access to documentation and other instructional materials, and an opportunity for hands-on training using example datasets and optimized metaproteomics workflows</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/migale/">Migale</a></td><td class="link"><a href="http://galaxy.migale.inra.fr/">
            Server
        </a></td><td class="summary markdown">A general purpose genomics server with an emphasis on metagenomics, including FROGS and several Migale developed tools.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/mimodd-nacreousmap/">MiModD NacreousMap</a></td><td class="link"><a href="http://mapping-by-sequencing.vm.uni-freiburg.de:8080/">
            Server
        </a></td><td class="summary markdown">NacreousMap mapping/plotting tool of MiModD for users of MiModD who do not want to install the required dependencies (R and rpy2) for graphical output from this tool on their local system.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/mississippi/">MISSISSIPPI</a></td><td class="link"><a href="http://mississippi.fr/">
            Server
        </a></td><td class="summary markdown">Support for analyses of RNA and small RNA sequencing datasets as well as for epigenetics or metagenomics studies.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/mpdsdm/">Molecular Property Diagnostic Suite - Diabetes Mellitus Portal (MPDSDM)</a></td><td class="link"><a href="http://www.mpds-diabetes.in/">
            Server
        </a></td><td class="summary markdown">A disease-specific open source web portal for Diabetes Mellitus.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/mpdstb/">Molecular Property Diagnostic Suite - Tuberculosis (MPDSTB)</a></td><td class="link"><a href="http://mpds.osdd.net/">
            Server
        </a></td><td class="summary markdown">A disease-specific web portal for Tuberculosis for drug discovery and development.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/nanogalaxy/">NanoGalaxy</a></td><td class="link"><a href="https://nanopore.usegalaxy.eu/">
            Server
        </a></td><td class="summary markdown">a webserver to process, analyse and visualize Oxford Nanopore Technologies (ONT) data and similar long-reads technologies.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/network-analyst/">NetworkAnalyst</a></td><td class="link"><a href="https://galaxy.networkanalyst.ca/">
            Server
        </a></td><td class="summary markdown">For mapping and quantification of raw RNA-seq data</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ng-tax/">NG-Tax</a></td><td class="link"><a href="https://ngtax.systemsbiology.nl/">
            Server
        </a></td><td class="summary markdown">A Semantic Framework for High-throughput Marker Gene (Amplicon) Analysis</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ngphylogeny/">NGPhylogeny.fr</a></td><td class="link"><a href="https://ngphylogeny.fr/">
            Server
        </a></td><td class="summary markdown">Robust phylogenetic analysis for everyone.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/oqtans/">Oqtans</a></td><td class="link"><a href="http://galaxy.inf.ethz.ch">
            Server
        </a></td><td class="summary markdown">Customized Galaxy, extended with machine learning based tools for sequence and tiling array data analysis.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/orione/">Orione</a></td><td class="link"><a href="http://orione.crs4.it/">
            Server
        </a></td><td class="summary markdown">A Galaxy based web server for microbiology. <a href="http://orione.crs4.it/">Orione</a> includes all post mapping or assembling steps from scaffolding to complete annotation pipelines.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/osddlinux-livegalaxy/">OSDDlinux LiveGalaxy</a></td><td class="link"><a href="http://osddlinux.osdd.net:8001/">
            Server
        </a></td><td class="summary markdown">Drug discovery</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/osiris/">Osiris</a></td><td class="link"><a href="http://galaxy-dev.cnsi.ucsb.edu/osiris/">
            Server
        </a></td><td class="summary markdown">Phylogenetics</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/palfinder/">Palfinder</a></td><td class="link"><a href="https://palfinder.ls.manchester.ac.uk/">
            Server
        </a></td><td class="summary markdown">Microsatellite development.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/pepsimili/">PepSimili</a></td><td class="link"><a href="http://pepsimili.e-nios.com:8080/">
            Server
        </a></td><td class="summary markdown">Integrated workflow tool for mapping of massive peptide repertoires on whole proteomes and delivering streamlined, systems-level biological interpretation.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/phagepromoter/">PhagePromotor</a></td><td class="link"><a href="https://galaxy.bio.di.uminho.pt/">
            Server
        </a></td><td class="summary markdown">Predict promoter sequences in phage genomes, using machine learning models.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/pia/">PIA</a></td><td class="link"><a href="http://galaxy-dev.cnsi.ucsb.edu/pia/">
            Server
        </a></td><td class="summary markdown">Phylogenetics and gene interaction</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/plants/">Plant Anaylsis Workbench</a></td><td class="link"><a href="https://plants.usegalaxy.eu/">
            Server
        </a></td><td class="summary markdown">A community driven plant data analsys Galaxy service</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/plantgenie/">PlantGenIE</a></td><td class="link"><a href="http://galaxy.plantgenie.org:8080/">
            Server
        </a></td><td class="summary markdown">Plant Genome Integrative Explorer Galaxy instance</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/prestige/">PreSTIGE</a></td><td class="link"><a href="http://prestige.case.edu/">
            Server
        </a></td><td class="summary markdown">Predicting Specific Tissue Interactions of Genes and Enhancers</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/proteogenomics-gateway/">Proteogenomics Gateway</a></td><td class="link"><a href="http://z.umn.edu/proteogenomicsgateway">
            Server
        </a></td><td class="summary markdown">Provides access to documentation and other instructional materials, and an opportunity for hands-on training using example datasets and optimized proteogenomics workflows. The main goal of this server is to provide documentation to facilitate training and mastery of these software and workflows.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/proteore/">ProteoRE</a></td><td class="link"><a href="http://www.proteore.org/">
            Server
        </a></td><td class="summary markdown">ProteoRE (Proteomics Research Environment)</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/protologger/">Protologger</a></td><td class="link"><a href="http://protologger.de/">
            Server
        </a></td><td class="summary markdown">All-in-one genome description tool, aimed at simplifying gathering the data required for writing protologues. This includes providing taxonomic, functional and ecological insights.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/repeatexplorer/">RepeatExplorer</a></td><td class="link"><a href="https://repeatexplorer-elixir.cerit-sc.cz/">
            Server
        </a></td><td class="summary markdown">Graph-based clustering and characterization of repetitive sequences, and detection of transposable element protein coding domains.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ribogalaxy/">RiboGalaxy</a></td><td class="link"><a href="http://ribogalaxy.ucc.ie/">
            Server
        </a></td><td class="summary markdown">Tools for the analysis and visualization of ribo-seq data obtained with the ribosome profiling technique.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/rice-galaxy/">Rice Galaxy</a></td><td class="link"><a href="http://galaxy.irri.org/">
            Server
        </a></td><td class="summary markdown">Tools for designing single-nucleotide polymorphism assays, analyzing genome-wide association studies, population diversity, rice−bacterial pathogen diagnostics, and a suite of published genomic prediction methods.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/rna-workbench/">RNA Workbench</a></td><td class="link"><a href="https://rna.usegalaxy.eu/">
            Server
        </a></td><td class="summary markdown">more than 50 bioinformatics tools dedicated to different research areas of RNA biology, including RNA structure analysis, RNA alignment, RNA annotation, RNA-protein interaction, ribosome profiling, RNA-Seq analysis, and RNA target prediction.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/sciensano/">Sciensano</a></td><td class="link"><a href="https://galaxy.sciensano.be/policy/disclaimer.html">
            Server
        </a></td><td class="summary markdown">General purpose Galaxy server optimized for microbiological genomics applications of interest for public health.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/shaman/">SHAMAN</a></td><td class="link"><a href="https://shaman.pasteur.fr/">
            Server
        </a></td><td class="summary markdown">A user-friendly website for metataxonomic analysis from raw reads to statistical analysis</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/singlecell/">Single Cell Omics workbench</a></td><td class="link"><a href="https://singlecell.usegalaxy.eu/">
            Server
        </a></td><td class="summary markdown">The Single Cell Omics workbench is a comprehensive set of analysis tools and consolidated workflows</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/streetscience/">Street Science</a></td><td class="link"><a href="https://streetscience.usegalaxy.eu/">
            Server
        </a></td><td class="summary markdown">a Galaxy dedicated for the Street Science Community</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/tgm/">TGM</a></td><td class="link"><a href="http://tgm-pipeline.uni.lu/galaxy/">
            Server
        </a></td><td class="summary markdown">The <em>TGM pipeline</em> is a workflow combining tranSMART server, Galaxy Server and MINERVA platform to enable visually-aided exploration, analysis and interpretation of high-throughput translational medicine data.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/treegenes/">TreeGenes</a></td><td class="link"><a href="https://treegenes.cam.uchc.edu/">
            Server
        </a></td><td class="summary markdown">TreeGenes integrates genotype, phenotype, and environmental data that can be processed for association mapping and landscape genomics, with a focus on forest trees. This functionality is provided by <a href="https://github.com/tripal/tripal_galaxy">Tripal Galaxy</a>, a module for <a href="http://tripal.info">Tripal</a> that provides seamless support for running Galaxy workflows directly in the site. Users can analyze data provided by TreeGenes or their own.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/usegalaxy-be/">UseGalaxy.be</a></td><td class="link"><a href="https://usegalaxy.be/">
            Server
        </a></td><td class="summary markdown">General purpose genomics Galaxy server</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/usegalaxy-eu/">UseGalaxy.eu</a></td><td class="link"><a href="https://usegalaxy.eu">
            Server
        </a></td><td class="summary markdown">The European UseGalaxy server.</td><!----><td class="keywords"><a href="/use/#usegalaxy-dir">
            UseGalaxy
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/usegalaxy-fr/">UseGalaxy.fr</a></td><td class="link"><a href="https://usegalaxy.fr/">
            Server
        </a></td><td class="summary markdown">General purpose genomics Galaxy server</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/usegalaxy-no/">UseGalaxy.no</a></td><td class="link"><a href="https://usegalaxy.no">
            Server
        </a></td><td class="summary markdown">A central, joint &#x26; free Galaxy instance for Norway-based researchers and their collaborators, with ~2000 bioinformatics tools, 200GB short-term storage and project storage on nels.bioinfo.no.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/usegalaxy-org/">UseGalaxy.org (Main)</a></td><td class="link"><a href="https://usegalaxy.org">
            Server
        </a></td><td class="summary markdown">The <a href="/galaxy-project/">Galaxy Project</a> free public server; biomedical research</td><!----><td class="keywords"><a href="/use/#usegalaxy-dir">
            UseGalaxy
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/usegalaxy-au/">UseGalaxy.org.au</a></td><td class="link"><a href="https://usegalaxy.org.au/">
            Server
        </a></td><td class="summary markdown">The Australian UseGalaxy server.</td><!----><td class="keywords"><a href="/use/#usegalaxy-dir">
            UseGalaxy
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/usmi-galaxy-demonstrator/">USMI Galaxy Demonstrator</a></td><td class="link"><a href="http://bioinformatics.hsanmartino.it:8080/">
            Server
        </a></td><td class="summary markdown">This Galaxy implementation is meant to demonstrate integration of microbial domain Biological Resource Center (mBRC) catalogues into the bioinformatics data integration environment.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/vinther-lab/">Vinther Lab</a></td><td class="link"><a href="http://galaxy.bio.ku.dk/">
            Server
        </a></td><td class="summary markdown">RNA structure-probing data analysis to improve the prediction of RNA secondary and tertiary structure and allow structural changes to be identified and investigated.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/vinyl/">VINYL (Variant prIoritizatioN bY survivaL analysis)</a></td><td class="link"><a href="http://www.beaconlab.it/VINYL">
            Server
        </a></td><td class="summary markdown">A flexible and fully automated system for the functional annotation and prioritization of genetic variants.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/vvv/">Viral Variant Visualizer (VVV)</a></td><td class="link"><a href="https://viralvariant.anses.fr/">
            Server
        </a></td><td class="summary markdown">Determine the composition of a viral population</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/viramp/">VirAmp</a></td><td class="link"><a href="http://viramp.com/">
            Server
        </a></td><td class="summary markdown">VirAmp is a web-based semi-<em>de novo</em> fast virus genome assembly pipeline designed for extremely high coverage NGS data.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/virology/">Virology</a></td><td class="link"><a href="https://virology.usegalaxy.eu/">
            Server
        </a></td><td class="summary markdown">A comprehensive set of analysis tools and consolidated workflows for virology work, including SARS-CoV-2.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/workflow4metabolomics/">Workflow4Metabolomics</a></td><td class="link"><a href="https://workflow4metabolomics.usegalaxy.fr">
            Server
        </a></td><td class="summary markdown">A collaborative portal dedicated to metabolomics data processing, analysis and annotation.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/wqap/">wQAP</a></td><td class="link"><a href="http://life2cloud.com:6005/">
            Server
        </a></td><td class="summary markdown">QAP is an open source software that can be used to analyze virus quasispecies using CBS, NGS and TGS data. wQAP (web QAP), is the online version of QAP.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr></tbody></table><!----></div><div id="academic-cloud-pane" data-toggle-group="use-panes" aria-labelledby="academic-cloud-tab" role="tabpanel" class="tab-pane fade show" data-v-b620dc4c><h2 class="nav-item" data-v-b620dc4c>
                    Academic Cloud Services
                </h2><p class="markdown" data-v-b620dc4c><p>These are academic cloud providers where ready to run Galaxy instances are available. Many of these are restricted to users within a particular geographic domain, for example, a country, province, or consortium of countries. These are often free to eligible researchers.</p>
</p><p class="d-none" data-v-b620dc4c>dummy text</p><table class="table table-striped" data-v-b620dc4c><thead data-v-b620dc4c><tr data-v-b620dc4c><th data-v-b620dc4c>Resource</th><th data-v-b620dc4c>Link</th><th data-v-b620dc4c>Summary</th><th data-v-b620dc4c>Keywords</th></tr></thead><tbody data-v-b620dc4c><tr data-v-b620dc4c><td class="title"><a href="/use/anvil/">AnVIL</a></td><td class="link"><a href="https://anvilproject.org/">
            Academic
        </a></td><td class="summary markdown">Run Galaxy inside the secure AnVIL environment on Google Cloud Platform.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/climb/">CLIMB</a></td><td class="link"><a href="http://www.climb.ac.uk/">
            Academic
        </a></td><td class="summary markdown">The CLIMB project (Cloud Infrastructure for Microbial Bioinformatics) develops and deploys a world leading cyber-infrastructure for microbial bioinformatics; providing free cloud-based compute, storage, and analysis tools for academic microbiologists in the UK.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/genap/">GenAP</a></td><td class="link"><a href="https://www.genap.ca/">
            Academic
        </a></td><td class="summary markdown">All Canadian researchers are <a href="https://ccdb.computecanada.ca/account_application">eligible for a GenAP account</a>. International researchers that have a Canadian collaborator can also be sponsored and become eligible to a GenAP account as well.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/globus-genomics/">Globus Genomics</a></td><td class="link"><a href="http://globusgenomics.org/">
            Academic
        </a></td><td class="summary markdown">Globus Genomics provides hardened high-performance Galaxy servers running with Globus on AWS.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/jetstream/">Jetstream</a></td><td class="link"><a href="http://jetstream-cloud.org/">
            Academic
        </a></td><td class="summary markdown"><a href="http://jetstream-cloud.org/">Jetstream</a> is part of <a href="https://www.xsede.org/">XSEDE</a>, a "collection of integrated advanced digital resources and services" and is funded by NSF.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/laniakea/">Laniakea</a></td><td class="link"><a href="https://laniakea-dashboard.cloud.ba.infn.it/login">
            Academic
        </a></td><td class="summary markdown">Laniakea provides the possibility to automate the creation of Galaxy-based virtualized environments through an easy setup procedure, providing an on-demand workspace ready to be used by life scientists and bioinformaticians, with built-in storage encryption for user data.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/metacentrum/">MetaCentrum</a></td><td class="link"><a href="https://galaxy.metacentrum.cz/">
            Academic
        </a></td><td class="summary markdown">MetaCentrum regional virtual organization of the Czech National Grid Infrastructure (NGI) and is accessible to employees and students of research and academic organizations of the Czech Republic and for scientific purposes only.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/nectar/">Nectar Cloud</a></td><td class="link"><a href="https://nectar.org.au/research-cloud/">
            Academic
        </a></td><td class="summary markdown">Set up a Galaxy server on Australia's Nectar research cloud.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/phenomenal/">PhenoMeNal</a></td><td class="link"><a href="https://portal.phenomenal-h2020.eu/cloud-research-environment-setup">
            Academic
        </a></td><td class="summary markdown">Large scale computing for medical informatics</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/snic-science-cloud/">SNIC Science Cloud (SSC)</a></td><td class="link"><a href="https://cloud.snic.se/">
            Academic
        </a></td><td class="summary markdown">A national-scale cloud infrastructure for Swedish Academia</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/veupathdb/">VEuPathDB Galaxy</a></td><td class="link"><a href="https://veupathdb.org/veupathdb/app/galaxy-orientation">
            Academic
        </a></td><td class="summary markdown">A free, interactive, web-based platform for large-scale data analysis focused on eukaryotic pathogens. Merges the former VectorBase and EuPathDB servers.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr></tbody></table><!----></div><div id="commercial-cloud-pane" data-toggle-group="use-panes" aria-labelledby="commercial-cloud-tab" role="tabpanel" class="tab-pane fade show" data-v-b620dc4c><h2 class="nav-item" data-v-b620dc4c>
                    Commercial Cloud Services
                </h2><p class="markdown" data-v-b620dc4c><p>These are commercial cloud providers where ready to run Galaxy instances are available. You need to pay for these, but you only need to pay for as long as you need the instance to be up.</p>
</p><p class="d-none" data-v-b620dc4c>dummy text</p><table class="table table-striped" data-v-b620dc4c><thead data-v-b620dc4c><tr data-v-b620dc4c><th data-v-b620dc4c>Resource</th><th data-v-b620dc4c>Link</th><th data-v-b620dc4c>Summary</th><th data-v-b620dc4c>Keywords</th></tr></thead><tbody data-v-b620dc4c><tr data-v-b620dc4c><td class="title"><a href="/use/aws/">Amazon Web Services (AWS)</a></td><td class="link"><a href="https://launch.usegalaxy.org/">
            Commercial
        </a></td><td class="summary markdown">Set up you own general purpose genomics server runing AWS using GVL and CloudMan.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/anvil/">AnVIL</a></td><td class="link"><a href="https://anvilproject.org/">
            Commercial
        </a></td><td class="summary markdown">Run Galaxy inside the secure AnVIL environment on Google Cloud Platform.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/g-onramp/">G-OnRamp</a></td><td class="link"><a href="https://launch.usegalaxy.org/catalog">
            Commercial
        </a></td><td class="summary markdown">G-OnRamp provides a user-friendly, web-based platform for collaborative, end-to-end annotation of eukaryotic genomes using UCSC Assembly Hubs and JBrowse/Apollo genome browsers with evidence tracks derived from sequence alignments, ab initio gene predictors, RNA-Seq data, and repeat finders.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/globus-genomics/">Globus Genomics</a></td><td class="link"><a href="http://globusgenomics.org/">
            Commercial
        </a></td><td class="summary markdown">Globus Genomics provides hardened high-performance Galaxy servers running with Globus on AWS.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/phenomenal/">PhenoMeNal</a></td><td class="link"><a href="https://portal.phenomenal-h2020.eu/cloud-research-environment-setup">
            Commercial
        </a></td><td class="summary markdown">Large scale computing for medical informatics</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr></tbody></table><!----></div><div id="containers-pane" data-toggle-group="use-panes" aria-labelledby="containers-tab" role="tabpanel" class="tab-pane fade show" data-v-b620dc4c><h2 class="nav-item" data-v-b620dc4c>
                    Containers
                </h2><p class="markdown" data-v-b620dc4c><p>These Galaxy instances are prepackaged using container technology, usually Docker. You run these locally by first installing the container technology (Docker is easy), and then launching the containerized Galaxy within that technology. These use local resources.</p>
</p><p class="d-none" data-v-b620dc4c>dummy text</p><table class="table table-striped" data-v-b620dc4c><thead data-v-b620dc4c><tr data-v-b620dc4c><th data-v-b620dc4c>Resource</th><th data-v-b620dc4c>Link</th><th data-v-b620dc4c>Summary</th><th data-v-b620dc4c>Keywords</th></tr></thead><tbody data-v-b620dc4c><tr data-v-b620dc4c><td class="title"><a href="/use/asaim/">ASaiM</a></td><td class="link"><a href="https://quay.io/repository/bebatut/asaim-framework?tag=latest&amp;tab=tags">
            Container
        </a></td><td class="summary markdown">Integrates tools specifically chosen for metagenomic and metatranscriptomic studies and hierarchically organized to orient user choice toward the best tool for a given task.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/atacgraph/">ATACgraph</a></td><td class="link"><a href="https://hub.docker.com/r/lsbnb/galaxy_atacgraph">
            Container
        </a></td><td class="summary markdown">ATAC-seq for biologists with minimal bioinformatics knowledge to analyze chromatin accessibility</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/bridge/">BRIDGE</a></td><td class="link"><a href="https://github.com/scientificomputing/bridge-docker">
            Container
        </a></td><td class="summary markdown">Prepare, simulate and analyse and visualise molecular simulations, especially molecular dynamics and free energies.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/cafu/">CAFU</a></td><td class="link"><a href="https://hub.docker.com/r/malab/cafu">
            Container
        </a></td><td class="summary markdown">A Galaxy framework for exploring unmapped RNA-Seq data</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/chemicaltoolbox/">ChemicalToolbox</a></td><td class="link"><a href="https://quay.io/repository/bgruening/galaxy-chemicaltoolbox">
            Container
        </a></td><td class="summary markdown">For processing, analysing and visualising chemical data, and performing molecular simulations.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/cravat-p/">CRAVAT-P</a></td><td class="link"><a href="https://hub.docker.com/r/jraysajulga/galaxy-cravatp-docker/">
            Container
        </a></td><td class="summary markdown">A Docker image containing a fully-operational Galaxy instance with pre-installed demonstration material for CRAVAT-P.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/deepts/">deepTS</a></td><td class="link"><a href="https://hub.docker.com/r/malab/deepts">
            Container
        </a></td><td class="summary markdown">Explore transcriptional switches from pairwise, temporal, and population RNA-Seq using deepTS.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/epigenomics/">Epigenomics Galaxy</a></td><td class="link"><a href="https://hub.docker.com/r/mpaya/epigenomics_galaxy">
            Container
        </a></td><td class="summary markdown">Galaxy Docker image for epigenomics analysis</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ewas-galaxy/">EWAS-Galaxy</a></td><td class="link"><a href="https://github.com/kpbioteam/docker_ewas">
            Container
        </a></td><td class="summary markdown">Galaxy Docker image for population epigenetics data analysis</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/basic-sequence/">Galaxy basic sequence manipulation</a></td><td class="link"><a href="https://hub.docker.com/r/bgruening/galaxy-sequence-tools/">
            Container
        </a></td><td class="summary markdown">Galaxy Docker repository for basic sequence data analysis.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/blast/">Galaxy BLAST+</a></td><td class="link"><a href="https://hub.docker.com/r/bgruening/galaxy-blast/">
            Container
        </a></td><td class="summary markdown">A complete and production ready Galaxy instance with installed BLAST+ tools.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/galaxy-docker/">Galaxy Docker</a></td><td class="link"><a href="https://github.com/bgruening/docker-galaxy-stable">
            Container
        </a></td><td class="summary markdown">The Galaxy Docker Image is an easy distributable full-fledged Galaxy installation, that can be used for testing, teaching and presenting new tools and features.</td><!----><td class="keywords"><a href="/use/#genomics">
            Genomics
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/exome-seq/">Galaxy Exome-Seq</a></td><td class="link"><a href="https://hub.docker.com/r/bgruening/galaxy-exome-seq/">
            Container
        </a></td><td class="summary markdown">Galaxy Docker repository for Exome sequencing data analysis</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/gga/">Galaxy Genome Annotation (GGA)</a></td><td class="link"><a href="https://hub.docker.com/r/bgruening/galaxy-genome-annotation/">
            Container
        </a></td><td class="summary markdown">Galaxy for genome annotation</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/imaging/">Galaxy Imaging</a></td><td class="link"><a href="https://hub.docker.com/r/bgruening/galaxy-imaging/">
            Container
        </a></td><td class="summary markdown">Galaxy Docker repository for image analysis</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/msi/">Galaxy Workbench for Mass Spectrometry Imaging</a></td><td class="link"><a href="https://hub.docker.com/r/mcfoell/galaxy-msi">
            Container
        </a></td><td class="summary markdown">Galaxy Docker image for mass spec imaging</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/graphclust/">GraphClust</a></td><td class="link"><a href="https://hub.docker.com/r/backofenlab/docker-galaxy-graphclust">
            Container
        </a></td><td class="summary markdown"><a href="https://github.com/BackofenLab/GraphClust-2">GraphClust2</a> is a workflow for scalable clustering of RNAs based on sequence and secondary structures feature. GraphClust2 is implemented within the Galaxy framework and consists a set of integrated Galaxy tools and flavors of the linear-time clustering workflow.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ng-tax/">NG-Tax</a></td><td class="link"><a href="http://wurssb.gitlab.io/ngtax/galaxy.html">
            Container
        </a></td><td class="summary markdown">A Semantic Framework for High-throughput Marker Gene (Amplicon) Analysis</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ngphylogeny/">NGPhylogeny.fr</a></td><td class="link"><a href="https://hub.docker.com/r/evolbioinfo/ngphylogeny/">
            Container
        </a></td><td class="summary markdown">Robust phylogenetic analysis for everyone.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/qiime2/">q2Galaxy</a></td><td class="link"><a href="https://quay.io/repository/qiime2/q2galaxy">
            Container
        </a></td><td class="summary markdown">QIIME 2 Galaxy Docker image.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/quantp/">QuanTP</a></td><td class="link"><a href="https://z.umn.edu/quantpdocker">
            Container
        </a></td><td class="summary markdown">A Docker image containing a fully-operational Galaxy instance with pre-installed QuanTP tools.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/rna-modification-calling/">RNA Modification Calling</a></td><td class="link"><a href="https://github.com/HelmGroup/Galaxy_modification_calling">
            Container
        </a></td><td class="summary markdown">Galaxy Docker image for RNA analysis and modification calling</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/rna-workbench/">RNA Workbench</a></td><td class="link"><a href="https://github.com/bgruening/galaxy-rna-workbench">
            Container
        </a></td><td class="summary markdown">more than 50 bioinformatics tools dedicated to different research areas of RNA biology, including RNA structure analysis, RNA alignment, RNA annotation, RNA-protein interaction, ribosome profiling, RNA-Seq analysis, and RNA target prediction.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/ena-submission/">SARS-CoV-2 ENA Submission</a></td><td class="link"><a href="https://quay.io/repository/galaxy/ena-upload?tag=latest&amp;tab=tags">
            Container
        </a></td><td class="summary markdown">Galaxy Docker image for submitting SARS-CoV-2 data to ENA</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/workflow4metabolomics/">Workflow4Metabolomics</a></td><td class="link"><a href="https://hub.docker.com/r/workflow4metabolomics/galaxy-workflow4metabolomics/">
            Container
        </a></td><td class="summary markdown">A collaborative portal dedicated to metabolomics data processing, analysis and annotation.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr></tbody></table><!----></div><div id="vms-pane" data-toggle-group="use-panes" aria-labelledby="vms-tab" role="tabpanel" class="tab-pane fade show" data-v-b620dc4c><h2 class="nav-item" data-v-b620dc4c>
                    Virtual Machines (VMs)
                </h2><p class="markdown" data-v-b620dc4c><p>VMs are similar to containers but use a different technology. Containers take significant advantage of your computer's underlying operating system. Virtual Machines include an entire supporting operating system, and are significantly larger than containers. You run these locally by first installing a VM player like VirtualBox, and then downloading and launching the VM within that player. These use local resources.</p>
</p><p class="d-none" data-v-b620dc4c>dummy text</p><table class="table table-striped" data-v-b620dc4c><thead data-v-b620dc4c><tr data-v-b620dc4c><th data-v-b620dc4c>Resource</th><th data-v-b620dc4c>Link</th><th data-v-b620dc4c>Summary</th><th data-v-b620dc4c>Keywords</th></tr></thead><tbody data-v-b620dc4c><tr data-v-b620dc4c><td class="title"><a href="/use/g-onramp/">G-OnRamp</a></td><td class="link"><a href="https://wustl.app.box.com/v/g-onramp-vm-v1">
            VM
        </a></td><td class="summary markdown">G-OnRamp provides a user-friendly, web-based platform for collaborative, end-to-end annotation of eukaryotic genomes using UCSC Assembly Hubs and JBrowse/Apollo genome browsers with evidence tracks derived from sequence alignments, ab initio gene predictors, RNA-Seq data, and repeat finders.</td><!----><td class="keywords"><a href="/use/#domain">
            Domain
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/gcac/">GCAC : Galaxy for Compound Activity Classification</a></td><td class="link"><a href="https://drive.google.com/file/d/1ZQM5x-9J49rvPy9SoWOrM3Xm1MHZdr1I/view?usp=sharing">
            VM
        </a></td><td class="summary markdown">GCAC is a web enabled platform for predictive model building using R and caret package for virtual screening.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/pirate/">PiRATE: a Pipeline to Retrieve and Annotate Transposable Elements</a></td><td class="link"><a href="https://www.seanoe.org/data/00406/51795/">
            VM
        </a></td><td class="summary markdown">Detect, classify and annotate TEs of non-model organisms.</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr><tr data-v-b620dc4c><td class="title"><a href="/use/shapiro-lab/">Shapiro Lab</a></td><td class="link"><a href="https://binkley2.ncifcrf.gov/users/bshapiro//software.html">
            VM
        </a></td><td class="summary markdown">RNA structure and design tools including RNA2D3D and StructureLab</td><!----><td class="keywords"><a href="/use/#tool-publishing">
            Tools
        </a></td></tr></tbody></table><!----></div></div><hr data-v-b620dc4c><footer class="page-footer markdown" data-v-b620dc4c><h2 id="about-this-directory"><a href="#about-this-directory" aria-hidden="true"><span class="icon icon-link"></span></a>About This Directory</h2>
<p>The Galaxy <a href="/community/">Community</a> has created Galaxy instances in many different forms and for many different applications. This directory lists the options we know about, including:</p>
<ul>
<li>Galaxy servers,</li>
<li>cloud services that support Galaxy instances, and</li>
<li>virtual machines and containers that can be easily deployed for your own server.</li>
</ul>
<h2 id="which-platform--platform-type-to-choose"><a href="#which-platform--platform-type-to-choose" aria-hidden="true"><span class="icon icon-link"></span></a>Which Platform / Platform Type to Choose?</h2>
<p>Your choices depends upon your needs. If you want to use Galaxy with a minimum of setup work, then try starting with a UseGalaxy orPublic Server. There are over 125 of these platforms to pcik from all of which can be use with little or no setup on your part. However if none of those meet your needs then here are some guidelines for other cases.</p>
<table>
<thead>
<tr>
<th></th>
<th>UseGalaxy Servers</th>
<th>Public Servers</th>
<th><a href="https://galaxyproject.eu/tiaas.html" target="_blank" rel="noopener noreferrer">TIaaS</a></th>
<th>Academic Clouds</th>
<th>Commercial Clouds</th>
<th>Containers</th>
<th>VMs</th>
<th>Local</th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>Free to use</strong></td>
<td>Yes</td>
<td>Yes</td>
<td>Yes</td>
<td>Yes<sup>1</sup></td>
<td>No</td>
<td>Yes</td>
<td>Yes</td>
<td>Yes</td>
</tr>
<tr>
<td><strong>Uses your local compute infrastructure</strong></td>
<td>No</td>
<td>No</td>
<td>No</td>
<td>No</td>
<td>No</td>
<td>Yes<sup>2</sup></td>
<td>Yes<sup>2</sup></td>
<td>Yes</td>
</tr>
<tr>
<td><strong>Datasets total > 250GB</strong> (including intermediate)</td>
<td>No</td>
<td>?<sup>5</sup></td>
<td>Yes</td>
<td>Yes</td>
<td>Yes</td>
<td>Yes<sup>3</sup></td>
<td>Yes<sup>3</sup></td>
<td>Yes</td>
</tr>
<tr>
<td><strong>Computational requirements are similarly large</strong></td>
<td>No</td>
<td>?<sup>5</sup></td>
<td>Yes</td>
<td>Yes</td>
<td>Yes</td>
<td>Yes<sup>3</sup></td>
<td>Yes<sup>3</sup></td>
<td>Yes</td>
</tr>
<tr>
<td><strong>Share Galaxy objects outside your organization</strong></td>
<td>Yes</td>
<td>Yes</td>
<td>Yes</td>
<td>Yes</td>
<td>Yes</td>
<td>Yes<sup>4</sup></td>
<td>Yes<sup>4</sup></td>
<td>Yes<sup>5</sup></td>
</tr>
<tr>
<td><strong>Install custom tools and reference genomces</strong></td>
<td>No</td>
<td>No</td>
<td>No</td>
<td>Yes<sup>5</sup></td>
<td>Yes</td>
<td>Yes</td>
<td>Yes</td>
<td>Yes</td>
</tr>
<tr>
<td><strong>Have absolute data security requirements</strong></td>
<td>No</td>
<td>No</td>
<td>No</td>
<td>?<sup>5</sup></td>
<td>?<sup>5</sup></td>
<td>?<sup>5</sup></td>
<td>?<sup>5</sup></td>
<td>Yes</td>
</tr>
</tbody>
</table>
<p><sup>1</sup> Depends on provider, and if you are eligible for the service<br>
<sup>2</sup> These technologies can be deployed on clouds or locally.<br>
<sup>3</sup> Depends on the size of the system you are running it on.<br>
<sup>4</sup> With these technologies you can save the server and share the entire platform with them.<br>
<sup>5</sup> Depends on configuration.</p>
<p>Most of these platform types are defined above (click on each tab), but two aren't:</p>
<h3 id="tiaas"><a href="#tiaas" aria-hidden="true"><span class="icon icon-link"></span></a>TIaaS</h3>
<p>Training Infrastructure as a Service (TIaaS) is a service offered by some UseGalaxy servers to specifically support training use cases. See <a href="https://galaxyproject.eu/tiaas.html" target="_blank" rel="noopener noreferrer">TIaaS</a> for more information.</p>
<h3 id="local"><a href="#local" aria-hidden="true"><span class="icon icon-link"></span></a>Local</h3>
<p>Finally, you can install your own local instance of Galaxy from scratch. See <a href="/admin/get-galaxy/">Get Galaxy</a> for details on how to do this. This has become much easier to do in recent years.</p>
<h2 id="resource-keywords"><a href="#resource-keywords" aria-hidden="true"><span class="icon icon-link"></span></a>Resource Keywords</h2>
<p>Each resource has a set of keywords associated with them.</p>
<h3 id="real-keywords"><a href="#real-keywords" aria-hidden="true"><span class="icon icon-link"></span></a>Real keywords</h3>
<p><em>(What about listing these by the resource's domain, for example, the research methodology (e.g., RNA-Seq), the clade (e.g., crops), or field e.g., metagenomics) that a resource is targeted at? We are working on it.)</em></p>
<h3 id="scope"><a href="#scope" aria-hidden="true"><span class="icon icon-link"></span></a>Scope</h3>
<p>Initially, there is only one keyword per resource and that defines the scope of the resource:</p>
<h4 id="usegalaxy"><a href="#usegalaxy" aria-hidden="true"><span class="icon icon-link"></span></a>UseGalaxy</h4>
<p>UseGalaxy servers implement a common core set of tools and reference genomes, and are open to anyone to use.</p>
<h4 id="genomics"><a href="#genomics" aria-hidden="true"><span class="icon icon-link"></span></a>Genomics</h4>
<p>These servers implement a broad range of tools and aren't specific to any part of the tree of life, or to any type of analysis.</p>
<h4 id="domain"><a href="#domain" aria-hidden="true"><span class="icon icon-link"></span></a>Domain</h4>
<p>Domain resources specialize in either a particular branch of the tree of life or in particular types of analysis. Within their specializations, domain resources offer a wide variety of tools.</p>
<h4 id="tool-publishing"><a href="#tool-publishing" aria-hidden="true"><span class="icon icon-link"></span></a>Tool Publishing</h4>
<p>Tool publishing resources make tools easily available so that researchers can use them without having to install and use command line versions. The distinction between <em>domain</em> resources and <em>tool publishing</em> resources is fuzzy. In general, tool publishing resources expose tools from a particular organization, while domain resources include tools / datasets from other organizations and have a larger goal in mind.</p>
<h2 id="add-a-resource-to-this-directory"><a href="#add-a-resource-to-this-directory" aria-hidden="true"><span class="icon icon-link"></span></a>Add a resource to this directory</h2>
<p>If you maintain a Galaxy resource that can be listed on this directory, then please <strong><a href="https://docs.google.com/forms/d/e/1FAIpQLSd4xzUP3pSYjTHiLoRE10zSJbplDVW7EbGy8EJ9JS0yy2SySg/viewform" target="_blank" rel="noopener noreferrer">describe your Galaxy resource here</a></strong>. We'll contact you with any question and then post it this directory.</p>
</footer><footer class="static-footer markdown"><p>The Galaxy Project is supported in part by NSF, NHGRI, The Huck Institutes of the Life Sciences, The Institute for CyberScience at Penn State, and Johns Hopkins University.</p>
</footer></section></div>
    <script src="/assets/js/app.0690a31c.js" defer></script><script src="/assets/js/page--src--pages--use-vue.7c5b8683.js" defer></script>
  </body>
</html>
