<!DOCTYPE html>
<html data-html-server-rendered="true" lang="en" data-vue-tag="%7B%22lang%22:%7B%22ssr%22:%22en%22%7D%7D">
  <head>
    <title>Development News Brief - Galaxy Community Hub</title><meta name="gridsome:hash" content="c32ecfdb33b075b0057598b5b9118f3554a398cf"><meta data-vue-tag="ssr" charset="utf-8"><meta data-vue-tag="ssr" name="generator" content="Gridsome v0.7.23"><meta data-vue-tag="ssr" data-key="viewport" name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover"><meta data-vue-tag="ssr" data-key="format-detection" name="format-detection" content="telephone=no"><meta data-vue-tag="ssr" data-key="description" name="description" content="All about Galaxy and its community."><link data-vue-tag="ssr" rel="icon" href="data:,"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="16x16" href="/assets/static/favicon.ce0531f.7d4a227415c9d9b704d56a079d49a3e1.png"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="32x32" href="/assets/static/favicon.ac8d93a.7d4a227415c9d9b704d56a079d49a3e1.png"><link rel="preload" href="/assets/css/0.styles.28333459.css" as="style"><link rel="preload" href="/assets/js/app.f7f9943a.js" as="script"><link rel="preload" href="/assets/js/page--src--templates--article-vue.9c961af0.js" as="script"><link rel="prefetch" href="/assets/js/page--node-modules--gridsome--app--pages--404-vue.54a07ec7.js"><link rel="prefetch" href="/assets/js/page--src--pages--blog-vue.72cfdf23.js"><link rel="prefetch" href="/assets/js/page--src--pages--careers-vue.51f7f627.js"><link rel="prefetch" href="/assets/js/page--src--pages--data-explorer-vue.cda0b563.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--archive-vue.54bd407e.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests--papercuts-vue.d73bf9af.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests-vue.f9ca327e.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--webinars-vue.2b0c4b1f.js"><link rel="prefetch" href="/assets/js/page--src--pages--events-vue.2bc0181e.js"><link rel="prefetch" href="/assets/js/page--src--pages--index-vue.9e6f30c2.js"><link rel="prefetch" href="/assets/js/page--src--pages--news-vue.c55be461.js"><link rel="prefetch" href="/assets/js/page--src--pages--tests-vue.4730afa8.js"><link rel="prefetch" href="/assets/js/page--src--pages--use-vue.8c7d1a05.js"><link rel="prefetch" href="/assets/js/page--src--templates--platform-vue.672f4f9e.js"><link rel="prefetch" href="/assets/js/page--src--templates--vue-article-vue.d03e3b77.js"><link rel="prefetch" href="/assets/js/vendors~page--src--pages--careers-vue~page--src--pages--use-vue~page--src--templates--article-vue~pa~efade918.71d44943.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--index-md.b120794f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--md--imgtests--imgdir--index-md.8a720dc7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--partials--index-md.88a2cc0b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--imgdir--index-md.a7da7426.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--index-md.ab7ce26b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--index-md.ac0f0339.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--index-md.7388aadd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--reldir--index-md.78e6d088.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--capacity-planning--index-md.532d558e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--index-md.f596bca3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--performance--cluster--legacy--index-md.9900d899.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--datatypes--index-md.37a06167.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--disk-quotas--index-md.99623880.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--faq--index-md.dfcae1f0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--data-model--index-md.f6811d25.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--galaxy-ops--index-md.da494b7d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--ten-simple-steps-galaxy-as-a-service--index-md.f0b31fa8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--tool-config-syntax--index-md.c133b049.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--writing-tests--index-md.d3ed2aee.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--allocation--index-md.88a6f777.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--faq--index-md.6b5ae7d2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--index-md.54b831ad.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--ssh--index-md.c431c940.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--troubleshooting--index-md.2bbf34e9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--capacity-planning--index-md.8d2c9c03.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--getting-started--index-md.63fe5edc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--index-md.1387a7bc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--building--index-md.ea5bc352.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--capacity-planning--index-md.bdfd10d5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--cluster-types--index-md.83943371.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--customizing--index-md.6b8b3678.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--faq--index-md.12c74d13.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--index-md.e3fa7dae.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--pre-201703--index-md.9b084a41.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--hadoop--index-md.95f1458a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--ht-condor--index-md.f973feff.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--index-md.37f11743.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--openstack--index-md.f4d0e62a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--services--index-md.e26413ae.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--sharing--index-md.c15fa6cb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--troubleshooting--index-md.953f49c4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--userdata--index-md.1116e088.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--future--index-md.ec7dff38.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--index-md.f0bd7fb5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-09--index-md.ed1d541b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-27--index-md.5a2cbe0c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-09-19--index-md.36fcc10e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-11-14--index-md.5155dfcc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-01-16--index-md.e740402b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-03-20--index-md.fa75b1de.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-05-15--index-md.ad74e436.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-11-20--index-md.6a1074f1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-02-19--index-md.bdefdea8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-04-16--index-md.ce613da7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-06-18--index-md.9253a34b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-08-20--index-md.8ac977c4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-10-15--index-md.3a558b10.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-02-18--index-md.ae82f681.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-04-21--index-md.4cd1f8fc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-06-16--index-md.7b1572c5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-08-18--index-md.5b3383d1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--index-md.da754856.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--webinar-tech--index-md.c5c416a6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2012--index-md.4eddb972.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2014--index-md.5d7b4578.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--index-md.afac4bb5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--data--index-md.f2ba0502.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--datafiles--index-md.4dfd4db7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--architecture--index-md.8035154b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--bitbucket--index-md.7cc7d8d1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--css--index-md.710209d9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--getting-started--index-md.930b3a22.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--index-md.1cbec221.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--jsa--index-md.134e6a35.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--mercurial--index-md.9026901a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--security-tool-tips--index-md.2d704e67.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--galaxy-project--index-md.230e8088.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--index-md.e5883590.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--papers--index-md.db9c7f1b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--posters--index-md.04432e3d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--presentations--index-md.92923a05.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--videos--index-md.f680b009.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--advanced-session--index-md.4227a5ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--basics-session--index-md.08c5b284.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--index-md.9844ef78.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--logistics--index-md.6f6b160a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--publicity--index-md.dfbc1a41.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--registration--index-md.5232161d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--abstracts--index-md.697fe169.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--index-md.ca51927e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--key-dates--index-md.606cfcac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--lightning-talks--index-md.6a60add5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--logistics--index-md.06ca864d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--organizing-committee--index-md.0500ed90.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--photos--index-md.ab68ac2c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--automation-strategies--index-md.2d8a9043.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--bioinformatics-training--index-md.43a55fdc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--end-users--index-md.5f46cd2a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--workflows-and-api--index-md.07da71b4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--index-md.cb8d0657.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--promotion--index-md.4eb64cde.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--register--index-md.49c36527.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--sponsorships--index-md.db0f3791.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--training-day--index-md.c8f12515.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--index-md.3c67479f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--posters--index-md.728aa734.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--talks--index-md.7f93c9db.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--clinical-use-hpc-file-systems--index-md.030a88bd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-admins--index-md.f12f7b4d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-blast--index-md.59e8820f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-professionals--index-md.dcb27276.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--index-md.cfaaca20.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--proteomics--index-md.abba045c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--public-galaxy-servers--index-md.bd34ea15.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--tool-developers--index-md.a5a0a8b6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--index-md.df6174ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--key-dates--index-md.f7630162.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--lightning--index-md.0dfb5a95.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--logistics--index-md.4759076c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--organizers--index-md.6fc11b96.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--photos--index-md.aef50972.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--program--index-md.05c5d87a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--promotion--index-md.e34ad376.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--register--index-md.8ea1785f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--sponsorships--index-md.3cde86d8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--advance-tool-data--index-md.2210b6c4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--api--index-md.57c06e35.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--index-md.4cd32016.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--intro-tool-data--index-md.75a7abbd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--vms--index-md.1c86a58b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--index-md.9abee1c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--posters--index-md.810cd33b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--talks--index-md.c9751906.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--branch-release-merge--index-md.f6ed714e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-admins--index-md.5f9fcb01.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-end-users--index-md.00b4cd3b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-training-network--index-md.a843f5f7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--heterogeneous--index-md.8313575f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--index-md.f878a9f6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--using-galaxy-ui-overview--index-md.5b284474.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--hackathon--index-md.20172bcf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--index-md.9ea6d61e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--key-dates--index-md.bac2859e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--lightning--index-md.96c8f509.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--logistics--index-md.c1318807.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--organizers--index-md.b183376c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--photos--index-md.e0b61b7a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--program--index-md.a93fc58b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--promotion--index-md.df1d2b77.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--register--index-md.54adaeb3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--sponsor-exhibit--index-md.64ef93be.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--admin-walkthrough--index-md.c82adb0e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--api--index-md.a936c5d1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--data-managers--index-md.348024f8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--index-md.030f1ebd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--vms--index-md.507d7d15.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--cwl--index-md.a8a9aa76.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-management-challenges--index-md.409c72aa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-splitting-and-parallelism--index-md.6332a3a3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--elixir--index-md.982da27f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--extended-workflow--index-md.f2b42ad0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--fots--index-md.52645a6e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-admins--index-md.ffda743c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-community-uk--index-md.060ecf60.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-scientists--index-md.ae9e7e59.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-training-network--index-md.a412fdd5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--index-md.22f27d14.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--data-hack-roundup--index-md.60bc6484.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--header--index-md.dbba4e7d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--index-md.2f85a581.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2016--bofs--index-md.5a3032e2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--abstracts--index-md.57e795e8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--childcare--index-md.096b34d2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--cofest--index-md.8b0a9b93.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--conduct--index-md.8e51f30c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--faq--index-md.0f0f1b46.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--index-md.641b44a6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--organizers--index-md.787e0ccf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--promotion--index-md.e6e58e52.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--registration--index-md.52f1b4f7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--sponsors--index-md.ef140359.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--training--index-md.ee0d3a7c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--venue--index-md.30256085.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--abstracts--index-md.137e7bc6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--childcare--index-md.84f642d3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--code-of-conduct--index-md.2789988f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--collaboration--index-md.a48ddf3b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--faq--index-md.4fd33733.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--icebreaker--index-md.ce20a106.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--index-md.9c798d19.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--keynotes--index-md.080e361c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--organizers--index-md.a3b956fb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--promotion--index-md.de985f84.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--register--index-md.05d6bc90.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--sponsors--index-md.4c398a30.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--training--index-md.ce0ef35c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--venue--index-md.4a3ddfec.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2014--index-md.d56576b3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2016--index-md.9f18f6ac.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2014--index-md.6fd15769.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2016--index-md.c0b34aae.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--index-md.ba91d0c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--trainingday--index-md.e89c50c3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2016--index-md.6afda26b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--example--vue-remark--index-md.c96a8cd8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--faqs--index-md.3bba7da9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-06--index-md.a5232112.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-07--index-md.19df6105.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--icons--index-md.ff92f6d4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--index-md.246f22cd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--logos--index-md.5a8a4cdf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--news-graphics--index-md.844f04b7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--photos--index-md.7a556820.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--screenshots--index-md.290a5433.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--index-md.467e2afe.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--workflow-editor-unannotated--index-md.bc93cd9f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--extract--index-md.940bc220.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--tool-panel--index-md.7ac3e984.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--variables--index-md.a3612f7c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--faq--index-md.599e559f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--galaxy-pages--index-md.51df5ee9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--interval-operations--index-md.71022235.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--main--faq--index-md.f6a425fa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--tools--requirements--index-md.1aef8fd3.js"><link rel="stylesheet" href="/assets/css/0.styles.28333459.css"><noscript data-vue-tag="ssr"><style>.g-image--loading{display:none;}</style></noscript>
  </head>
  <body >
    <div data-server-rendered="true" id="app" class="layout" data-v-3a128d75><header id="masthead"><nav class="container navbar navbar-expand-lg navbar-dark justify-content-center"><a href="/" class="navbar-brand active"><img id="masthead-logo" src="/images/galaxy_logo_hub_white.svg" alt="Galaxy Community Hub" height="30"></a><div id="navbarToggleWrapper" class="collapse navbar-collapse mx-auto"><ul class="navbar-nav"><li class="nav-item"><a href="/use/" role="button" aria-expanded="false" class="nav-link dropdown">
          Use
        </a></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" aria-expanded="false" class="nav-link dropdown-toggle">
          Learn
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/learn/" class="dropdown-item">How to use Galaxy</a></li><li><a href="https://training.galaxyproject.org/" class="dropdown-item">Teach with Galaxy</a></li></ul></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" class="nav-link dropdown-toggle">
          Community
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/community/" class="dropdown-item">About</a></li><li><a href="/events/" class="dropdown-item">Events</a></li><li><a href="/news/" class="dropdown-item">News</a></li><li><a href="/blog/" class="dropdown-item">Blog</a></li><li><a href="/publication-library/" class="dropdown-item">Publications</a></li><li><a href="/galaxy-project/statistics/" class="dropdown-item">Stats</a></li><li><a href="/careers/" class="dropdown-item">Careers</a></li></ul></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" class="nav-link dropdown-toggle">
          Deploy &amp; Develop
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/admin/get-galaxy/" class="dropdown-item">Install Galaxy</a></li><li><a href="/admin/" class="dropdown-item">Server administration</a></li><li><a href="/tools/" class="dropdown-item">Tools</a></li><li><a href="/develop/api/" class="dropdown-item">Working with the API</a></li><li><a href="/develop/" class="dropdown-item">Contributing</a></li><li><a href="/docs/" class="dropdown-item">Code documentation</a></li></ul></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" class="nav-link dropdown-toggle">
          Support
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/support/" class="dropdown-item">FAQ</a></li><li><a href="https://help.galaxyproject.org/" class="dropdown-item">Galaxy Help Forum</a></li></ul></li><li class="nav-item"><a href="/jxtx/" role="button" aria-expanded="false" class="nav-link dropdown">
          @jxtx
        </a></li></ul></div></nav></header><section id="maincontainer" class="container"><a href="/" class="link active" data-v-3a128d75> ‚Üê Home</a><header data-v-3a128d75><!----><h1 class="title" data-v-3a128d75>Development News Brief</h1><div class="metadata" data-v-3a128d75><!----><!----><p class="date" data-v-3a128d75>8 April 2011</p></div></header><div class="content markdown" data-v-3a128d75><hr>
<p><a href="http://bitbucket.org/galaxy/galaxy-central/wiki/GetGalaxy" target="_blank" rel="noopener noreferrer">Get Galaxy!</a>
new: <code>% hg clone http://www.bx.psu.edu/hg/galaxy galaxy-dist</code><br />
upgrade: <code>% hg pull -u -r 50e249442c5a</code></p>
<hr>
<h2 id="key-upcoming-galaxy-events"><a href="#key-upcoming-galaxy-events" aria-hidden="true"><span class="icon icon-link"></span></a>Key Upcoming Galaxy Events</h2>
<p><strong>May 24-26, 2011 Galaxy Community Conference</strong>, Lunteren, The Netherlands.</p>
<ul>
<li><a href="http://galaxy.psu.edu/gcc2011/" target="_blank" rel="noopener noreferrer">http://galaxy.psu.edu/gcc2011/</a></li>
<li><strong>April 24</strong> is the Early registration deadline (save 20%!)</li>
<li><a href="http://galaxy.psu.edu/gcc2011/Register.html" target="_blank" rel="noopener noreferrer">http://galaxy.psu.edu/gcc2011/Register.html</a></li>
</ul>
<p><strong>April 13-14, NBIC Galaxy Hackathon</strong>, Belgium.  Please submit your suggestions!  </p>
<ul>
<li><a href="http://wiki.nbic.nl/index.php/NBIC_Galaxy_Hackathon_project" target="_blank" rel="noopener noreferrer">http://wiki.nbic.nl/index.php/NBIC_Galaxy_Hackathon_project</a></li>
</ul>
<hr>
<h2 id="whats-new"><a href="#whats-new" aria-hidden="true"><span class="icon icon-link"></span></a>What's New</h2>
<h3 id="workflow-api"><a href="#workflow-api" aria-hidden="true"><span class="icon icon-link"></span></a>Workflow API</h3>
<p>An <strong>API for executing workflows</strong> has been added.
#### Usage examples</p>
<p>/scripts/api/workflow_execute.py<br />
/scripts/api/example_watch_folder.py.</p>
<h3 id="move-data-library-items"><a href="#move-data-library-items" aria-hidden="true"><span class="icon icon-link"></span></a>Move Data Library Items</h3>
<p>We have introduced a feature that enables you to move library datasets or folders (including all folder contents) to other locations within the same data library or to a different data library altogether. </p>
<p>Galaxy administrators can perform this feature on any data library item and can move items to any data library.  Users that are not Galaxy administrators must be given the modify library item permission on an item in order to move it, and they'll need the add library item permission on the desired target data library folder in order for it to be displayed in the select list of targets. </p>
<h4 id="moving-a-single-dataset"><a href="#moving-a-single-dataset" aria-hidden="true"><span class="icon icon-link"></span></a>Moving a single dataset</h4>
<p>To move a single dataset, select the <strong>Move this dataset</strong> option from the dataset's pop-up menu.</p>
<p><img src="/src/archive/dev-news-briefs/2011-04-08/2011_04_08_g1_move_dataset.png"></p>
<p>The default behavior is to move items to other locations within the same data library, so you'll initially be presented with a list of valid folders from which to choose for the new target location.</p>
<p><img src="/src/archive/dev-news-briefs/2011-04-08/2011_04_08_g2_select_folder.png"></p>
<p>After selecting the desired folder, clicking the <strong>Move</strong> button will move the dataset to the selected folder.</p>
<h4 id="moving-a-folder"><a href="#moving-a-folder" aria-hidden="true"><span class="icon icon-link"></span></a>Moving a Folder</h4>
<p>Moving a folder is very similar to moving a single dataset - just select the <strong>Move this folder</strong> option from the folder's pop-up menu.</p>
<p><img src="/src/archive/dev-news-briefs/2011-04-08/2011_04_08_g3_move_folder.png"></p>
<p>To move the folder to a different data library, after selecting the <strong>Move this folder</strong> option, click the <strong>Choose another data library</strong> link on the <strong>Move data library items</strong> page. The  folders select list will change to a select list of data libraries to which you are authorized to move the item.  When you select a target data library, the select list will change again to display the list of folders in the selected data library to which you are authorized to move the item. Clicking the <strong>Move</strong> button will move the folder and all of it's contents to the selected target folder within the selected target data library.  </p>
<p>The target folders select list is filtered to include only valid folders to which you can move the item.  For example, you cannot move a folder to one of it's own sub-folders in one step.  To do this, the sub-folder must be moved outside of it's parent, and then the parent can be moved to the folder that it previously contained.</p>
<p><img src="/src/archive/dev-news-briefs/2011-04-08/2011_04_08_g4_move_folder1.png"></p>
<hr>
<h2 id="updated--improved"><a href="#updated--improved" aria-hidden="true"><span class="icon icon-link"></span></a>Updated &#x26; Improved</h2>
<h3 id="data-content"><a href="#data-content" aria-hidden="true"><span class="icon icon-link"></span></a>Data Content</h3>
<ul>
<li>
<p><strong>Genomes</strong></p>
<ul>
<li>
<p><strong>New &#x26; included in NGS Tools</strong></p>
<ul>
<li><em>Saccharomcyes cerevisiae</em>: Saccharomcyes_cerevisiae_S288C_SGD2010</li>
<li><em>Arabidopsis lyrata</em>: Araly1</li>
<li>Purple Sea Urchin: strPur3 and Spur_v2.6</li>
<li>Hydra: Hydra_JCVI</li>
<li>Zebrafish: danRer7</li>
<li>Poplar: Ptrichocarpa_156</li>
<li>Chimpanzee: panTro3</li>
<li>Northern White-Cheeked Gibbon: nomLeu1</li>
<li>Korean Man AK1: Homo_sapiens_AK1</li>
</ul>
</li>
<li>
<p><strong>New &#x26; not included in NGS Tools</strong></p>
<ul>
<li><em>Caenorhabditis remanei</em>: caeRem2</li>
</ul>
</li>
<li>
<p><strong>Existing genomes added to NGS Tools</strong></p>
<ul>
<li>hg19 Canonical female (no Y chromosome)</li>
<li><em>Streptococcus pneumoniae R6</em>: 278</li>
<li><em>Drosophila virilis</em>: droVir3 and droVir2 </li>
</ul>
</li>
</ul>
</li>
<li>
<p><strong>New </strong>LiftOver<strong> Files</strong></p>
<ul>
<li>caeRem2 --> caePb1, caeRem2 --> caeRem3, caeRem2 --> cb3, caeRem2 --> ce4, caeRem2 --> priPac1, calJac3 --> hg18, canFam2 --> monDom5, danRer6 --> danRer7, danRer7 --> fr2, danRer7 --> gasAcu1, danRer7 --> hg19, danRer7 --> mm9, danRer7 --> oryLat2, danRer7 --> panTro3, danRer7 --> tetNig2, danRer7 --> xenTro2, droVir3 --> droVir2, fr2 --> danRer7, gasAcu1 --> danRer7, hg18 --> calJac3, hg19 --> danRer7, mm9 --> danRer7, panTro3 --> danRer7, panTro3 --> hg19, ponAbe2 --> calJac3, ponAbe2 --> monDom5, strPur2 --> ci2, tetNig2 --> danRer7, xenTro2 --> danRer7</li>
</ul>
</li>
<li>
<p><strong>Add Genomes to Your Instance</strong></p>
<ul>
<li><a href="http://bitbucket.org/galaxy/galaxy-central/wiki/NGSLocalSetup" target="_blank" rel="noopener noreferrer">http://bitbucket.org/galaxy/galaxy-central/wiki/NGSLocalSetup</a></li>
</ul>
</li>
<li>
<p><strong>Current Galaxy Main Genomes</strong></p>
<ul>
<li><a href="http://bitbucket.org/galaxy/galaxy-central/wiki/GenomeData" target="_blank" rel="noopener noreferrer">http://bitbucket.org/galaxy/galaxy-central/wiki/GenomeData</a> </li>
</ul>
</li>
</ul>
<h3 id="current-tools"><a href="#current-tools" aria-hidden="true"><span class="icon icon-link"></span></a>Current Tools</h3>
<ul>
<li>Add more verbose error reporting to <strong>FASTQ Groomer</strong> tool. Provides more information to allow users to determine what is wrong with <em>FASTQ</em> files with invalid format.</li>
<li>Enhance <em>Bowtie</em> wrapper to accept non-Sanger variant <em>FASTQ</em> files.</li>
<li>Allow <strong>Upload Tool</strong> to function on <code>https</code> URLs.</li>
<li>
<p>Add count <em>GFF</em> features tool and tests: </p>
<ul>
<li>Filter and Sort -->  GFF -->  <strong>Filter GFF file by feature count using simple expressions</strong>.</li>
<li>Tool counts the number of features in a <em>GFF</em> file. Note: this is different than the number of lines because a single <em>GFF</em> feature can often span multiple lines.</li>
</ul>
</li>
<li>
<p><em>Tophat</em> v1.2.0 support: </p>
<ul>
<li>(a) allow indel search.</li>
<li>(b) max insertion and max deletion lengths.</li>
<li>(c) library type.</li>
</ul>
</li>
<li>Updated <strong><code>gff_filter_by_feature_count</code></strong> tool now accepts and correctly handles all <em>GTF</em>, <em>GFF</em>, and <em>GFF3</em> files.</li>
<li>Changes for <strong>detecting and loading <em>BAM</em> data type</strong>
<em>Samtools</em> version 0.1.13 or newer produces an error condition when attempting to index an unsorted <em>BAM</em> file.  To determine if a <em>BAM</em> file is sorted, we first use <em>Samtools</em> to check the headers.  If this does not provide a definitive answer and <em>Samtools</em> version 0.1.13 or newer is being used, we index the <em>BAM</em> file to see if it produces the error.  This process provides a more robust approach to determining if the <em>BAM</em> file is sorted.</li>
</ul>
<h3 id="new-tools"><a href="#new-tools" aria-hidden="true"><span class="icon icon-link"></span></a>New Tools</h3>
<ul>
<li><strong>Multiple Alignments: <em>ClustalW</em></strong> multiple sequence alignment program for DNA or proteins.</li>
<li>
<p><strong>Motif Tools: Sequence Logo</strong> generator for <em>FASTA</em> data (example: <em>ClustalW</em> alignment).</p>
<ul>
<li>Both tools originated from the Community Tool Shed (see below).</li>
<li>The <strong>Sequence Logo</strong> tool uses <em>Weblogo3</em> wrapped into <strong>Galaxy</strong> to generate a sequence logo. The input file must be a <em>FASTA</em> file in your current history. It is recommended for viewing multiple-sequence alignment output from the <em>ClustalW</em> tool. Set the <em>ClustalW</em> output to <em>FASTA</em> to create the input for this tool. </li>
</ul>
</li>
</ul>
<h4 id="a-typical-output-looks-like-this"><a href="#a-typical-output-looks-like-this" aria-hidden="true"><span class="icon icon-link"></span></a>A typical output looks like this:</h4>
<p><img src="/src/archive/dev-news-briefs/2011-04-08/2011_04_08_rgWebLogo3_test.jpg"></p>
<h3 id="community-tools-tool-shed"><a href="#community-tools-tool-shed" aria-hidden="true"><span class="icon icon-link"></span></a>Community Tools (Tool Shed)</h3>
<ul>
<li>Tuning: Clarified what is being searched in the <strong>Tool Shed</strong>.</li>
<li>New <strong><em>ClustalW</em></strong> wrapper (see <a href="http://www.clustal.org" target="_blank" rel="noopener noreferrer">http://www.clustal.org</a>) for protein/dna multiple alignments based on the Galaxy <em>ClustalW</em> wrapper posted by <a href="/src/people/hansrudolf-hotz/index.md">Hans-Rudolf Hotz</a> <a href="http://lists.bx.psu.edu/pipermail/galaxy-dev/2010-November/003732.html" target="_blank" rel="noopener noreferrer">in an email on the developer list</a></li>
<li>New <em>Weblogo3</em> wrapper (<a href="http://weblogo.berkeley.edu" target="_blank" rel="noopener noreferrer">http://weblogo.berkeley.edu</a>) that creates sequence logos from <em>FASTA</em> data such as the output from a <em>ClustalW</em> alignment.</li>
</ul>
<h3 id="data-libraries"><a href="#data-libraries" aria-hidden="true"><span class="icon icon-link"></span></a>Data Libraries</h3>
<ul>
<li>Disabled problematic eager loading on data libraries.  Very large data libraries will load two to three times quicker.</li>
<li>Upload Improvement with example walk-through: </li>
</ul>
<h4 id="uploading-data-library-datasets"><a href="#uploading-data-library-datasets" aria-hidden="true"><span class="icon icon-link"></span></a>Uploading data library datasets</h4>
<ol>
<li>Apply a corrected version of the patch from Peter Cock that flips the "Preserve directory structure?" setting when uploading library datasets from filesystem paths.  The checkbox is now "yes" instead of the original "no", and is checked by default.</li>
</ol>
<p><img src="/src/archive/dev-news-briefs/2011-04-08/2011_04_08_g5_preserve.png"></p>
<ol start="2">
<li>Flip the behavior of the "Copy data into Galaxy" feature when uploading library datasets (similar to 1 above) to be more clear and logical.  Instead of a single checkbox, this is now a select list with clearly defined options.  The default setting is to copy the files into the configured Galaxy file store.</li>
</ol>
<p><img src="/src/archive/dev-news-briefs/2011-04-08/2011_04_08_g6_copy_files.png"></p>
<ol start="3">
<li>Allow importing items from a history to replace a library dataset with a new version.  Previously, you could only replace a library dataset with a new version by uploading a single file.</li>
</ol>
<p><img src="/src/archive/dev-news-briefs/2011-04-08/2011_04_08_g7_replace_dataset.png"></p>
<h3 id="workflows"><a href="#workflows" aria-hidden="true"><span class="icon icon-link"></span></a>Workflows</h3>
<ul>
<li>Users can now import copies of their own <strong>Workflows</strong> and <strong>Histories</strong>.</li>
</ul>
<h3 id="trackster"><a href="#trackster" aria-hidden="true"><span class="icon icon-link"></span></a>Trackster</h3>
<ul>
<li>
<p><strong>Enhancements</strong>:</p>
<ul>
<li>Implemented a data manager for <strong>Trackster</strong> and drawing is now completely tracked and controlled.</li>
<li>Put show_insertions and show_differences in read track <code>config</code>.</li>
<li>
<p>Filtering now supports:</p>
<ul>
<li>(a) score columns in <em>BED</em> and <em>GFF</em>.</li>
<li>(b) <em>GTF</em> attributes.</li>
</ul>
</li>
<li>Added <em>BED</em> and <em>GFF</em> support to visual analytics framework, and enable GOPS intersect and subtract tools to work with visual analytics.</li>
</ul>
</li>
<li>
<p><strong>Improvements</strong>:</p>
<ul>
<li>Insertions and deletions now shown in reads.</li>
<li>Save and restore mode for feature and read tracks.</li>
<li>Bug fixes for drawing feature tracks in <code>Squish</code> and <code>Dense</code> mode.</li>
<li>Menus and menu items now work correctly.</li>
<li>Remove form from navigation controls so that enter key works properly and sets the chrom/low/high.</li>
<li>Enable full keyboard navigation via arrow keys.</li>
</ul>
</li>
<li>
<p><strong>Bug fixes</strong>:</p>
<ul>
<li>
<p>Fix issues with navigation input: </p>
<ul>
<li>(a) arrow keys no longer perform navigation</li>
<li>(b) invalid chromosome names are handled well.</li>
</ul>
</li>
<li>Fix feature track bugs so that intervals are correctly drawn as half-open.</li>
<li>Use gap to correctly position read connector in <code>Pack</code> mode.</li>
<li>Fix read id bug in Trackster <em>BAM</em> data provider.</li>
</ul>
</li>
</ul>
<h4 id="example-of-trackster-visualization"><a href="#example-of-trackster-visualization" aria-hidden="true"><span class="icon icon-link"></span></a>Example of Trackster visualization:</h4>
<p>Track of reads mapped using <em>Tophat</em> (spliced) including the transcript assembled from those reads.</p>
<p><img src="/src/archive/dev-news-briefs/2011-04-08/2011_04_08_j1_trackster.png"></p>
<h3 id="user-interface-ui"><a href="#user-interface-ui" aria-hidden="true"><span class="icon icon-link"></span></a>User Interface (UI)</h3>
<ul>
<li>SGD Yeast genome added as a <em>Gbrowse</em> display site.</li>
<li><em>IGV</em> added as an external display application. This is not enabled on Galaxy main, but is available for local instances.</li>
</ul>
<h3 id="cloudman"><a href="#cloudman" aria-hidden="true"><span class="icon icon-link"></span></a>CloudMan</h3>
<ul>
<li><code>FTP</code> data upload enabled.
<a href="http://bitbucket.org/galaxy/galaxy-central/wiki/cloud" target="_blank" rel="noopener noreferrer">http://bitbucket.org/galaxy/galaxy-central/wiki/cloud</a></li>
</ul>
<h3 id="source"><a href="#source" aria-hidden="true"><span class="icon icon-link"></span></a>Source</h3>
<ul>
<li>Improvements in reliability and speed regarding manipulation of instance data volumes.</li>
<li>The value set in '<code>new_file_path</code>' in <code>universe_wsgi.ini</code> will now be used for creation of all temporary files.  This obviates the need for setting <code>$TEMP</code> when running data source tools on the cluster.</li>
<li>
<p>Datatypes:</p>
<ul>
<li>Add a '<code>subclass</code>' flag to datatype definitions in <code>datatypes_conf.xml</code> that allows dynamic creation of a subclass.</li>
<li>Allow composite datatype datasets to be populated in the <strong>Upload</strong> tool from files that were uploaded to the Galaxy FTP server.</li>
</ul>
</li>
<li>
<p>Tool configuration enhancements:</p>
<ul>
<li>Allow <code>DataToolParameter</code> to be filtered on attributes other than <code>dbkey</code> (dynamic options).</li>
<li>Allow <code>FromParamToolOutputActionOption</code> and <code>ParamValueToolOutputActionOptionFilter</code> to access attributes of parameter values.</li>
</ul>
</li>
</ul>
<h3 id="tool-test-framework"><a href="#tool-test-framework" aria-hidden="true"><span class="icon icon-link"></span></a>Tool Test Framework</h3>
<ul>
<li>Specifying '<code>ftype="bam"</code>' as a parameter on a test's <code>&#x3C;output></code> tag will now cause the test framework to use '<code>samtools view</code>' to convert the file to <em>SAM</em>.  This should make it easier to debug why the test tool output differs from your baseline test output.</li>
</ul>
<p>Test framework enhancements:</p>
<ul>
<li>Allow <code>toolbox tests</code> to upload a file found located in <code>subdirectories</code>. </li>
<li>Fix a bug occurring on the determination of uploaded dataset name during the handling of the removal of <code>.gzip</code> or <code>.zip</code> extension from the uploaded filenames.</li>
<li>When using <code>re_match comparison method</code> in functional tests and line counts do not match, print out first 40 lines of the history file.</li>
</ul>
<h3 id="admin-menu"><a href="#admin-menu" aria-hidden="true"><span class="icon icon-link"></span></a>Admin Menu</h3>
<ul>
<li>Add a checkbox to the <strong>Create Group</strong> page that if checked will create a new <em><code>Role</code></em> with the same name. This provides a similar feature the the existing checkbox on the <strong>Create Role</strong> page. </li>
<li>However, the behavior is now changed such that new associations are created when the checkbox is checked, whereas before, only the <em><code>Group</code></em> or <em><code>Role</code></em> objects with the same name were created, but not associated with anything.</li>
</ul>
<h3 id="bug-fixes"><a href="#bug-fixes" aria-hidden="true"><span class="icon icon-link"></span></a>Bug Fixes</h3>
<ul>
<li>Following the <strong>login</strong> link from the <strong>Logout</strong> page no longer redirects you to the <strong>Logout</strong> page once you have logged in.</li>
<li>The peek setting method, <code>set_peek()</code>, should now function consistently across datatypes descending from <code>Text</code>, particularly with respect to line count estimation.</li>
<li>Apply patch from Ry4an Brase to correct "NoneType preference on the jobs view" issue.</li>
<li>Don't alter the contents of a file while uploading to a data library if using one of the <strong>Upload files from file system paths</strong> or <strong>Upload a directory of files</strong> options in conjunction with the <strong>Link to files without copying into Galaxy</strong> option. This partially resolves the issue where a supposedly sorted <em>BAM</em> file was being resorted upon upload to a data library when using this option. A better implementation of determining whether a <em>BAM</em> file has been sorted (so that it does not get resorted) remains to be done.</li>
<li>
<p><strong>Extract Genomic DNA</strong> tool: </p>
<ul>
<li>Do not fix strand for <em>GFF</em> input.</li>
<li>Handle <em>non-GFF</em> files when interpret features is <code>true</code>.</li>
</ul>
</li>
<li>Set default value for <em>Cuffdiff's</em> minimum alignment count parameter to reflect v0.9.3.</li>
<li>Enhance <em>GFF</em> reader to handle headers and comments and fix bug so that new feature starts when <code>transcript_id</code> or <code>gene_id</code> changes.</li>
<li>Suppress most logging for <em>Cufflinks</em> and <em>Cuffdiff</em> as the logging can generate output that is incompatible with <code>UTF-8 format</code> used by database.</li>
<li>Set message type to error when history deletion fails due to sharing.</li>
<li>Show masthead links to <strong>Published Visualizations</strong> and <strong>Published Pages</strong> only if <strong>Tracks</strong> or <strong>Pages</strong> are enabled.</li>
<li>Shared visualizations work again.</li>
<li>Fix bug in <code>recently used tool submenu</code>.</li>
<li>Change default value for <em>Tophat</em> coverage search to <code>true</code>.</li>
</ul>
<hr>
<h2 id="about-galaxy"><a href="#about-galaxy" aria-hidden="true"><span class="icon icon-link"></span></a>About Galaxy</h2>
<p>The <strong>Galaxy team</strong> is a part of <a href="http://www.bx.psu.edu/" target="_blank" rel="noopener noreferrer">BX</a> at <a href="http://www.psu.edu/" target="_blank" rel="noopener noreferrer">Penn State</a>, and the <a href="http://www.biology.emory.edu/" target="_blank" rel="noopener noreferrer">Biology</a> and <a href="http://www.mathcs.emory.edu/" target="_blank" rel="noopener noreferrer">Mathematics and Computer Science</a> departments at <a href="http://www.emory.edu/home/index.html" target="_blank" rel="noopener noreferrer">Emory University</a>. </p>
<p><strong>Galaxy</strong> is supported in part by <a href="http://www.nsf.gov/" target="_blank" rel="noopener noreferrer">NSF</a>, <a href="http://www.genome.gov/" target="_blank" rel="noopener noreferrer">NHGRI</a>, the <a href="http://www.huck.psu.edu/" target="_blank" rel="noopener noreferrer">Huck Institutes of the Life Sciences</a>, and <a href="http://www.ics.psu.edu/" target="_blank" rel="noopener noreferrer">The Institute for CyberScience at Penn State</a>, and <a href="http://www.emory.edu/home/index.html" target="_blank" rel="noopener noreferrer">Emory University</a>.</p>
<p><a href="http://bitbucket.org/galaxy/galaxy-central/wiki/GalaxyTeam" target="_blank" rel="noopener noreferrer">GalaxyTeam</a></p>
<p><a href="http://usegalaxy.org" target="_blank" rel="noopener noreferrer">Use Galaxy!</a></p>
<p><a href="http://galaxyproject.org" target="_blank" rel="noopener noreferrer">GalaxyProject.org</a></p>
<p>Join us at <strong>Twitter</strong></p>
<p><code>#usegalaxy</code></p>
<p><a href="http://twitter.com/#!/search/galaxyproject" target="_blank" rel="noopener noreferrer">http://twitter.com/#!/search/galaxyproject</a> </p>
<hr>
</div><footer class="static-footer markdown"><p>The Galaxy Project is supported in part by NSF, NHGRI, The Huck Institutes of the Life Sciences, The Institute for CyberScience at Penn State, and Johns Hopkins University.</p>
</footer></section></div>
    <script>window.__INITIAL_STATE__={"data":{"article":{"id":"0482a649457bd6251bba0751ce9b23ab","title":"Development News Brief","tease":"","image":"","images":{},"category":null,"contact":"","date":"8 April 2011","content":"\u003Chr\u003E\n\u003Cp\u003E\u003Ca href=\"http:\u002F\u002Fbitbucket.org\u002Fgalaxy\u002Fgalaxy-central\u002Fwiki\u002FGetGalaxy\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EGet Galaxy!\u003C\u002Fa\u003E\nnew: \u003Ccode\u003E% hg clone http:\u002F\u002Fwww.bx.psu.edu\u002Fhg\u002Fgalaxy galaxy-dist\u003C\u002Fcode\u003E\u003Cbr \u002F\u003E\nupgrade: \u003Ccode\u003E% hg pull -u -r 50e249442c5a\u003C\u002Fcode\u003E\u003C\u002Fp\u003E\n\u003Chr\u003E\n\u003Ch2 id=\"key-upcoming-galaxy-events\"\u003E\u003Ca href=\"#key-upcoming-galaxy-events\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EKey Upcoming Galaxy Events\u003C\u002Fh2\u003E\n\u003Cp\u003E\u003Cstrong\u003EMay 24-26, 2011 Galaxy Community Conference\u003C\u002Fstrong\u003E, Lunteren, The Netherlands.\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"http:\u002F\u002Fgalaxy.psu.edu\u002Fgcc2011\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttp:\u002F\u002Fgalaxy.psu.edu\u002Fgcc2011\u002F\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cstrong\u003EApril 24\u003C\u002Fstrong\u003E is the Early registration deadline (save 20%!)\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"http:\u002F\u002Fgalaxy.psu.edu\u002Fgcc2011\u002FRegister.html\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttp:\u002F\u002Fgalaxy.psu.edu\u002Fgcc2011\u002FRegister.html\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Cp\u003E\u003Cstrong\u003EApril 13-14, NBIC Galaxy Hackathon\u003C\u002Fstrong\u003E, Belgium.  Please submit your suggestions!  \u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"http:\u002F\u002Fwiki.nbic.nl\u002Findex.php\u002FNBIC_Galaxy_Hackathon_project\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttp:\u002F\u002Fwiki.nbic.nl\u002Findex.php\u002FNBIC_Galaxy_Hackathon_project\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Chr\u003E\n\u003Ch2 id=\"whats-new\"\u003E\u003Ca href=\"#whats-new\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EWhat's New\u003C\u002Fh2\u003E\n\u003Ch3 id=\"workflow-api\"\u003E\u003Ca href=\"#workflow-api\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EWorkflow API\u003C\u002Fh3\u003E\n\u003Cp\u003EAn \u003Cstrong\u003EAPI for executing workflows\u003C\u002Fstrong\u003E has been added.\n#### Usage examples\u003C\u002Fp\u003E\n\u003Cp\u003E\u002Fscripts\u002Fapi\u002Fworkflow_execute.py\u003Cbr \u002F\u003E\n\u002Fscripts\u002Fapi\u002Fexample_watch_folder.py.\u003C\u002Fp\u003E\n\u003Ch3 id=\"move-data-library-items\"\u003E\u003Ca href=\"#move-data-library-items\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EMove Data Library Items\u003C\u002Fh3\u003E\n\u003Cp\u003EWe have introduced a feature that enables you to move library datasets or folders (including all folder contents) to other locations within the same data library or to a different data library altogether. \u003C\u002Fp\u003E\n\u003Cp\u003EGalaxy administrators can perform this feature on any data library item and can move items to any data library.  Users that are not Galaxy administrators must be given the modify library item permission on an item in order to move it, and they'll need the add library item permission on the desired target data library folder in order for it to be displayed in the select list of targets. \u003C\u002Fp\u003E\n\u003Ch4 id=\"moving-a-single-dataset\"\u003E\u003Ca href=\"#moving-a-single-dataset\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EMoving a single dataset\u003C\u002Fh4\u003E\n\u003Cp\u003ETo move a single dataset, select the \u003Cstrong\u003EMove this dataset\u003C\u002Fstrong\u003E option from the dataset's pop-up menu.\u003C\u002Fp\u003E\n\u003Cp\u003E\u003Cimg src=\"\u002Fsrc\u002Farchive\u002Fdev-news-briefs\u002F2011-04-08\u002F2011_04_08_g1_move_dataset.png\"\u003E\u003C\u002Fp\u003E\n\u003Cp\u003EThe default behavior is to move items to other locations within the same data library, so you'll initially be presented with a list of valid folders from which to choose for the new target location.\u003C\u002Fp\u003E\n\u003Cp\u003E\u003Cimg src=\"\u002Fsrc\u002Farchive\u002Fdev-news-briefs\u002F2011-04-08\u002F2011_04_08_g2_select_folder.png\"\u003E\u003C\u002Fp\u003E\n\u003Cp\u003EAfter selecting the desired folder, clicking the \u003Cstrong\u003EMove\u003C\u002Fstrong\u003E button will move the dataset to the selected folder.\u003C\u002Fp\u003E\n\u003Ch4 id=\"moving-a-folder\"\u003E\u003Ca href=\"#moving-a-folder\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EMoving a Folder\u003C\u002Fh4\u003E\n\u003Cp\u003EMoving a folder is very similar to moving a single dataset - just select the \u003Cstrong\u003EMove this folder\u003C\u002Fstrong\u003E option from the folder's pop-up menu.\u003C\u002Fp\u003E\n\u003Cp\u003E\u003Cimg src=\"\u002Fsrc\u002Farchive\u002Fdev-news-briefs\u002F2011-04-08\u002F2011_04_08_g3_move_folder.png\"\u003E\u003C\u002Fp\u003E\n\u003Cp\u003ETo move the folder to a different data library, after selecting the \u003Cstrong\u003EMove this folder\u003C\u002Fstrong\u003E option, click the \u003Cstrong\u003EChoose another data library\u003C\u002Fstrong\u003E link on the \u003Cstrong\u003EMove data library items\u003C\u002Fstrong\u003E page. The  folders select list will change to a select list of data libraries to which you are authorized to move the item.  When you select a target data library, the select list will change again to display the list of folders in the selected data library to which you are authorized to move the item. Clicking the \u003Cstrong\u003EMove\u003C\u002Fstrong\u003E button will move the folder and all of it's contents to the selected target folder within the selected target data library.  \u003C\u002Fp\u003E\n\u003Cp\u003EThe target folders select list is filtered to include only valid folders to which you can move the item.  For example, you cannot move a folder to one of it's own sub-folders in one step.  To do this, the sub-folder must be moved outside of it's parent, and then the parent can be moved to the folder that it previously contained.\u003C\u002Fp\u003E\n\u003Cp\u003E\u003Cimg src=\"\u002Fsrc\u002Farchive\u002Fdev-news-briefs\u002F2011-04-08\u002F2011_04_08_g4_move_folder1.png\"\u003E\u003C\u002Fp\u003E\n\u003Chr\u003E\n\u003Ch2 id=\"updated--improved\"\u003E\u003Ca href=\"#updated--improved\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EUpdated &#x26; Improved\u003C\u002Fh2\u003E\n\u003Ch3 id=\"data-content\"\u003E\u003Ca href=\"#data-content\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EData Content\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cstrong\u003EGenomes\u003C\u002Fstrong\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cstrong\u003ENew &#x26; included in NGS Tools\u003C\u002Fstrong\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Cem\u003ESaccharomcyes cerevisiae\u003C\u002Fem\u003E: Saccharomcyes_cerevisiae_S288C_SGD2010\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cem\u003EArabidopsis lyrata\u003C\u002Fem\u003E: Araly1\u003C\u002Fli\u003E\n\u003Cli\u003EPurple Sea Urchin: strPur3 and Spur_v2.6\u003C\u002Fli\u003E\n\u003Cli\u003EHydra: Hydra_JCVI\u003C\u002Fli\u003E\n\u003Cli\u003EZebrafish: danRer7\u003C\u002Fli\u003E\n\u003Cli\u003EPoplar: Ptrichocarpa_156\u003C\u002Fli\u003E\n\u003Cli\u003EChimpanzee: panTro3\u003C\u002Fli\u003E\n\u003Cli\u003ENorthern White-Cheeked Gibbon: nomLeu1\u003C\u002Fli\u003E\n\u003Cli\u003EKorean Man AK1: Homo_sapiens_AK1\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cstrong\u003ENew &#x26; not included in NGS Tools\u003C\u002Fstrong\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Cem\u003ECaenorhabditis remanei\u003C\u002Fem\u003E: caeRem2\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cstrong\u003EExisting genomes added to NGS Tools\u003C\u002Fstrong\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003Ehg19 Canonical female (no Y chromosome)\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cem\u003EStreptococcus pneumoniae R6\u003C\u002Fem\u003E: 278\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cem\u003EDrosophila virilis\u003C\u002Fem\u003E: droVir3 and droVir2 \u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cstrong\u003ENew \u003C\u002Fstrong\u003ELiftOver\u003Cstrong\u003E Files\u003C\u002Fstrong\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003EcaeRem2 --\u003E caePb1, caeRem2 --\u003E caeRem3, caeRem2 --\u003E cb3, caeRem2 --\u003E ce4, caeRem2 --\u003E priPac1, calJac3 --\u003E hg18, canFam2 --\u003E monDom5, danRer6 --\u003E danRer7, danRer7 --\u003E fr2, danRer7 --\u003E gasAcu1, danRer7 --\u003E hg19, danRer7 --\u003E mm9, danRer7 --\u003E oryLat2, danRer7 --\u003E panTro3, danRer7 --\u003E tetNig2, danRer7 --\u003E xenTro2, droVir3 --\u003E droVir2, fr2 --\u003E danRer7, gasAcu1 --\u003E danRer7, hg18 --\u003E calJac3, hg19 --\u003E danRer7, mm9 --\u003E danRer7, panTro3 --\u003E danRer7, panTro3 --\u003E hg19, ponAbe2 --\u003E calJac3, ponAbe2 --\u003E monDom5, strPur2 --\u003E ci2, tetNig2 --\u003E danRer7, xenTro2 --\u003E danRer7\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cstrong\u003EAdd Genomes to Your Instance\u003C\u002Fstrong\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"http:\u002F\u002Fbitbucket.org\u002Fgalaxy\u002Fgalaxy-central\u002Fwiki\u002FNGSLocalSetup\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttp:\u002F\u002Fbitbucket.org\u002Fgalaxy\u002Fgalaxy-central\u002Fwiki\u002FNGSLocalSetup\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cstrong\u003ECurrent Galaxy Main Genomes\u003C\u002Fstrong\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"http:\u002F\u002Fbitbucket.org\u002Fgalaxy\u002Fgalaxy-central\u002Fwiki\u002FGenomeData\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttp:\u002F\u002Fbitbucket.org\u002Fgalaxy\u002Fgalaxy-central\u002Fwiki\u002FGenomeData\u003C\u002Fa\u003E \u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"current-tools\"\u003E\u003Ca href=\"#current-tools\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003ECurrent Tools\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003EAdd more verbose error reporting to \u003Cstrong\u003EFASTQ Groomer\u003C\u002Fstrong\u003E tool. Provides more information to allow users to determine what is wrong with \u003Cem\u003EFASTQ\u003C\u002Fem\u003E files with invalid format.\u003C\u002Fli\u003E\n\u003Cli\u003EEnhance \u003Cem\u003EBowtie\u003C\u002Fem\u003E wrapper to accept non-Sanger variant \u003Cem\u003EFASTQ\u003C\u002Fem\u003E files.\u003C\u002Fli\u003E\n\u003Cli\u003EAllow \u003Cstrong\u003EUpload Tool\u003C\u002Fstrong\u003E to function on \u003Ccode\u003Ehttps\u003C\u002Fcode\u003E URLs.\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003EAdd count \u003Cem\u003EGFF\u003C\u002Fem\u003E features tool and tests: \u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003EFilter and Sort --\u003E  GFF --\u003E  \u003Cstrong\u003EFilter GFF file by feature count using simple expressions\u003C\u002Fstrong\u003E.\u003C\u002Fli\u003E\n\u003Cli\u003ETool counts the number of features in a \u003Cem\u003EGFF\u003C\u002Fem\u003E file. Note: this is different than the number of lines because a single \u003Cem\u003EGFF\u003C\u002Fem\u003E feature can often span multiple lines.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cem\u003ETophat\u003C\u002Fem\u003E v1.2.0 support: \u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E(a) allow indel search.\u003C\u002Fli\u003E\n\u003Cli\u003E(b) max insertion and max deletion lengths.\u003C\u002Fli\u003E\n\u003Cli\u003E(c) library type.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003EUpdated \u003Cstrong\u003E\u003Ccode\u003Egff_filter_by_feature_count\u003C\u002Fcode\u003E\u003C\u002Fstrong\u003E tool now accepts and correctly handles all \u003Cem\u003EGTF\u003C\u002Fem\u003E, \u003Cem\u003EGFF\u003C\u002Fem\u003E, and \u003Cem\u003EGFF3\u003C\u002Fem\u003E files.\u003C\u002Fli\u003E\n\u003Cli\u003EChanges for \u003Cstrong\u003Edetecting and loading \u003Cem\u003EBAM\u003C\u002Fem\u003E data type\u003C\u002Fstrong\u003E\n\u003Cem\u003ESamtools\u003C\u002Fem\u003E version 0.1.13 or newer produces an error condition when attempting to index an unsorted \u003Cem\u003EBAM\u003C\u002Fem\u003E file.  To determine if a \u003Cem\u003EBAM\u003C\u002Fem\u003E file is sorted, we first use \u003Cem\u003ESamtools\u003C\u002Fem\u003E to check the headers.  If this does not provide a definitive answer and \u003Cem\u003ESamtools\u003C\u002Fem\u003E version 0.1.13 or newer is being used, we index the \u003Cem\u003EBAM\u003C\u002Fem\u003E file to see if it produces the error.  This process provides a more robust approach to determining if the \u003Cem\u003EBAM\u003C\u002Fem\u003E file is sorted.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"new-tools\"\u003E\u003Ca href=\"#new-tools\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003ENew Tools\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Cstrong\u003EMultiple Alignments: \u003Cem\u003EClustalW\u003C\u002Fem\u003E\u003C\u002Fstrong\u003E multiple sequence alignment program for DNA or proteins.\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cstrong\u003EMotif Tools: Sequence Logo\u003C\u002Fstrong\u003E generator for \u003Cem\u003EFASTA\u003C\u002Fem\u003E data (example: \u003Cem\u003EClustalW\u003C\u002Fem\u003E alignment).\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003EBoth tools originated from the Community Tool Shed (see below).\u003C\u002Fli\u003E\n\u003Cli\u003EThe \u003Cstrong\u003ESequence Logo\u003C\u002Fstrong\u003E tool uses \u003Cem\u003EWeblogo3\u003C\u002Fem\u003E wrapped into \u003Cstrong\u003EGalaxy\u003C\u002Fstrong\u003E to generate a sequence logo. The input file must be a \u003Cem\u003EFASTA\u003C\u002Fem\u003E file in your current history. It is recommended for viewing multiple-sequence alignment output from the \u003Cem\u003EClustalW\u003C\u002Fem\u003E tool. Set the \u003Cem\u003EClustalW\u003C\u002Fem\u003E output to \u003Cem\u003EFASTA\u003C\u002Fem\u003E to create the input for this tool. \u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch4 id=\"a-typical-output-looks-like-this\"\u003E\u003Ca href=\"#a-typical-output-looks-like-this\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EA typical output looks like this:\u003C\u002Fh4\u003E\n\u003Cp\u003E\u003Cimg src=\"\u002Fsrc\u002Farchive\u002Fdev-news-briefs\u002F2011-04-08\u002F2011_04_08_rgWebLogo3_test.jpg\"\u003E\u003C\u002Fp\u003E\n\u003Ch3 id=\"community-tools-tool-shed\"\u003E\u003Ca href=\"#community-tools-tool-shed\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003ECommunity Tools (Tool Shed)\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003ETuning: Clarified what is being searched in the \u003Cstrong\u003ETool Shed\u003C\u002Fstrong\u003E.\u003C\u002Fli\u003E\n\u003Cli\u003ENew \u003Cstrong\u003E\u003Cem\u003EClustalW\u003C\u002Fem\u003E\u003C\u002Fstrong\u003E wrapper (see \u003Ca href=\"http:\u002F\u002Fwww.clustal.org\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttp:\u002F\u002Fwww.clustal.org\u003C\u002Fa\u003E) for protein\u002Fdna multiple alignments based on the Galaxy \u003Cem\u003EClustalW\u003C\u002Fem\u003E wrapper posted by \u003Ca href=\"\u002Fsrc\u002Fpeople\u002Fhansrudolf-hotz\u002Findex.md\"\u003EHans-Rudolf Hotz\u003C\u002Fa\u003E \u003Ca href=\"http:\u002F\u002Flists.bx.psu.edu\u002Fpipermail\u002Fgalaxy-dev\u002F2010-November\u002F003732.html\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ein an email on the developer list\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003ENew \u003Cem\u003EWeblogo3\u003C\u002Fem\u003E wrapper (\u003Ca href=\"http:\u002F\u002Fweblogo.berkeley.edu\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttp:\u002F\u002Fweblogo.berkeley.edu\u003C\u002Fa\u003E) that creates sequence logos from \u003Cem\u003EFASTA\u003C\u002Fem\u003E data such as the output from a \u003Cem\u003EClustalW\u003C\u002Fem\u003E alignment.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"data-libraries\"\u003E\u003Ca href=\"#data-libraries\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EData Libraries\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003EDisabled problematic eager loading on data libraries.  Very large data libraries will load two to three times quicker.\u003C\u002Fli\u003E\n\u003Cli\u003EUpload Improvement with example walk-through: \u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch4 id=\"uploading-data-library-datasets\"\u003E\u003Ca href=\"#uploading-data-library-datasets\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EUploading data library datasets\u003C\u002Fh4\u003E\n\u003Col\u003E\n\u003Cli\u003EApply a corrected version of the patch from Peter Cock that flips the \"Preserve directory structure?\" setting when uploading library datasets from filesystem paths.  The checkbox is now \"yes\" instead of the original \"no\", and is checked by default.\u003C\u002Fli\u003E\n\u003C\u002Fol\u003E\n\u003Cp\u003E\u003Cimg src=\"\u002Fsrc\u002Farchive\u002Fdev-news-briefs\u002F2011-04-08\u002F2011_04_08_g5_preserve.png\"\u003E\u003C\u002Fp\u003E\n\u003Col start=\"2\"\u003E\n\u003Cli\u003EFlip the behavior of the \"Copy data into Galaxy\" feature when uploading library datasets (similar to 1 above) to be more clear and logical.  Instead of a single checkbox, this is now a select list with clearly defined options.  The default setting is to copy the files into the configured Galaxy file store.\u003C\u002Fli\u003E\n\u003C\u002Fol\u003E\n\u003Cp\u003E\u003Cimg src=\"\u002Fsrc\u002Farchive\u002Fdev-news-briefs\u002F2011-04-08\u002F2011_04_08_g6_copy_files.png\"\u003E\u003C\u002Fp\u003E\n\u003Col start=\"3\"\u003E\n\u003Cli\u003EAllow importing items from a history to replace a library dataset with a new version.  Previously, you could only replace a library dataset with a new version by uploading a single file.\u003C\u002Fli\u003E\n\u003C\u002Fol\u003E\n\u003Cp\u003E\u003Cimg src=\"\u002Fsrc\u002Farchive\u002Fdev-news-briefs\u002F2011-04-08\u002F2011_04_08_g7_replace_dataset.png\"\u003E\u003C\u002Fp\u003E\n\u003Ch3 id=\"workflows\"\u003E\u003Ca href=\"#workflows\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EWorkflows\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003EUsers can now import copies of their own \u003Cstrong\u003EWorkflows\u003C\u002Fstrong\u003E and \u003Cstrong\u003EHistories\u003C\u002Fstrong\u003E.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"trackster\"\u003E\u003Ca href=\"#trackster\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003ETrackster\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cstrong\u003EEnhancements\u003C\u002Fstrong\u003E:\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003EImplemented a data manager for \u003Cstrong\u003ETrackster\u003C\u002Fstrong\u003E and drawing is now completely tracked and controlled.\u003C\u002Fli\u003E\n\u003Cli\u003EPut show_insertions and show_differences in read track \u003Ccode\u003Econfig\u003C\u002Fcode\u003E.\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003EFiltering now supports:\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E(a) score columns in \u003Cem\u003EBED\u003C\u002Fem\u003E and \u003Cem\u003EGFF\u003C\u002Fem\u003E.\u003C\u002Fli\u003E\n\u003Cli\u003E(b) \u003Cem\u003EGTF\u003C\u002Fem\u003E attributes.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003EAdded \u003Cem\u003EBED\u003C\u002Fem\u003E and \u003Cem\u003EGFF\u003C\u002Fem\u003E support to visual analytics framework, and enable GOPS intersect and subtract tools to work with visual analytics.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cstrong\u003EImprovements\u003C\u002Fstrong\u003E:\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003EInsertions and deletions now shown in reads.\u003C\u002Fli\u003E\n\u003Cli\u003ESave and restore mode for feature and read tracks.\u003C\u002Fli\u003E\n\u003Cli\u003EBug fixes for drawing feature tracks in \u003Ccode\u003ESquish\u003C\u002Fcode\u003E and \u003Ccode\u003EDense\u003C\u002Fcode\u003E mode.\u003C\u002Fli\u003E\n\u003Cli\u003EMenus and menu items now work correctly.\u003C\u002Fli\u003E\n\u003Cli\u003ERemove form from navigation controls so that enter key works properly and sets the chrom\u002Flow\u002Fhigh.\u003C\u002Fli\u003E\n\u003Cli\u003EEnable full keyboard navigation via arrow keys.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cstrong\u003EBug fixes\u003C\u002Fstrong\u003E:\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\n\u003Cp\u003EFix issues with navigation input: \u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E(a) arrow keys no longer perform navigation\u003C\u002Fli\u003E\n\u003Cli\u003E(b) invalid chromosome names are handled well.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003EFix feature track bugs so that intervals are correctly drawn as half-open.\u003C\u002Fli\u003E\n\u003Cli\u003EUse gap to correctly position read connector in \u003Ccode\u003EPack\u003C\u002Fcode\u003E mode.\u003C\u002Fli\u003E\n\u003Cli\u003EFix read id bug in Trackster \u003Cem\u003EBAM\u003C\u002Fem\u003E data provider.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch4 id=\"example-of-trackster-visualization\"\u003E\u003Ca href=\"#example-of-trackster-visualization\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EExample of Trackster visualization:\u003C\u002Fh4\u003E\n\u003Cp\u003ETrack of reads mapped using \u003Cem\u003ETophat\u003C\u002Fem\u003E (spliced) including the transcript assembled from those reads.\u003C\u002Fp\u003E\n\u003Cp\u003E\u003Cimg src=\"\u002Fsrc\u002Farchive\u002Fdev-news-briefs\u002F2011-04-08\u002F2011_04_08_j1_trackster.png\"\u003E\u003C\u002Fp\u003E\n\u003Ch3 id=\"user-interface-ui\"\u003E\u003Ca href=\"#user-interface-ui\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EUser Interface (UI)\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003ESGD Yeast genome added as a \u003Cem\u003EGbrowse\u003C\u002Fem\u003E display site.\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Cem\u003EIGV\u003C\u002Fem\u003E added as an external display application. This is not enabled on Galaxy main, but is available for local instances.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"cloudman\"\u003E\u003Ca href=\"#cloudman\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003ECloudMan\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ccode\u003EFTP\u003C\u002Fcode\u003E data upload enabled.\n\u003Ca href=\"http:\u002F\u002Fbitbucket.org\u002Fgalaxy\u002Fgalaxy-central\u002Fwiki\u002Fcloud\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttp:\u002F\u002Fbitbucket.org\u002Fgalaxy\u002Fgalaxy-central\u002Fwiki\u002Fcloud\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"source\"\u003E\u003Ca href=\"#source\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003ESource\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003EImprovements in reliability and speed regarding manipulation of instance data volumes.\u003C\u002Fli\u003E\n\u003Cli\u003EThe value set in '\u003Ccode\u003Enew_file_path\u003C\u002Fcode\u003E' in \u003Ccode\u003Euniverse_wsgi.ini\u003C\u002Fcode\u003E will now be used for creation of all temporary files.  This obviates the need for setting \u003Ccode\u003E$TEMP\u003C\u002Fcode\u003E when running data source tools on the cluster.\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003EDatatypes:\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003EAdd a '\u003Ccode\u003Esubclass\u003C\u002Fcode\u003E' flag to datatype definitions in \u003Ccode\u003Edatatypes_conf.xml\u003C\u002Fcode\u003E that allows dynamic creation of a subclass.\u003C\u002Fli\u003E\n\u003Cli\u003EAllow composite datatype datasets to be populated in the \u003Cstrong\u003EUpload\u003C\u002Fstrong\u003E tool from files that were uploaded to the Galaxy FTP server.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003ETool configuration enhancements:\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003EAllow \u003Ccode\u003EDataToolParameter\u003C\u002Fcode\u003E to be filtered on attributes other than \u003Ccode\u003Edbkey\u003C\u002Fcode\u003E (dynamic options).\u003C\u002Fli\u003E\n\u003Cli\u003EAllow \u003Ccode\u003EFromParamToolOutputActionOption\u003C\u002Fcode\u003E and \u003Ccode\u003EParamValueToolOutputActionOptionFilter\u003C\u002Fcode\u003E to access attributes of parameter values.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"tool-test-framework\"\u003E\u003Ca href=\"#tool-test-framework\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003ETool Test Framework\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003ESpecifying '\u003Ccode\u003Eftype=\"bam\"\u003C\u002Fcode\u003E' as a parameter on a test's \u003Ccode\u003E&#x3C;output\u003E\u003C\u002Fcode\u003E tag will now cause the test framework to use '\u003Ccode\u003Esamtools view\u003C\u002Fcode\u003E' to convert the file to \u003Cem\u003ESAM\u003C\u002Fem\u003E.  This should make it easier to debug why the test tool output differs from your baseline test output.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Cp\u003ETest framework enhancements:\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003EAllow \u003Ccode\u003Etoolbox tests\u003C\u002Fcode\u003E to upload a file found located in \u003Ccode\u003Esubdirectories\u003C\u002Fcode\u003E. \u003C\u002Fli\u003E\n\u003Cli\u003EFix a bug occurring on the determination of uploaded dataset name during the handling of the removal of \u003Ccode\u003E.gzip\u003C\u002Fcode\u003E or \u003Ccode\u003E.zip\u003C\u002Fcode\u003E extension from the uploaded filenames.\u003C\u002Fli\u003E\n\u003Cli\u003EWhen using \u003Ccode\u003Ere_match comparison method\u003C\u002Fcode\u003E in functional tests and line counts do not match, print out first 40 lines of the history file.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"admin-menu\"\u003E\u003Ca href=\"#admin-menu\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EAdmin Menu\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003EAdd a checkbox to the \u003Cstrong\u003ECreate Group\u003C\u002Fstrong\u003E page that if checked will create a new \u003Cem\u003E\u003Ccode\u003ERole\u003C\u002Fcode\u003E\u003C\u002Fem\u003E with the same name. This provides a similar feature the the existing checkbox on the \u003Cstrong\u003ECreate Role\u003C\u002Fstrong\u003E page. \u003C\u002Fli\u003E\n\u003Cli\u003EHowever, the behavior is now changed such that new associations are created when the checkbox is checked, whereas before, only the \u003Cem\u003E\u003Ccode\u003EGroup\u003C\u002Fcode\u003E\u003C\u002Fem\u003E or \u003Cem\u003E\u003Ccode\u003ERole\u003C\u002Fcode\u003E\u003C\u002Fem\u003E objects with the same name were created, but not associated with anything.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Ch3 id=\"bug-fixes\"\u003E\u003Ca href=\"#bug-fixes\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EBug Fixes\u003C\u002Fh3\u003E\n\u003Cul\u003E\n\u003Cli\u003EFollowing the \u003Cstrong\u003Elogin\u003C\u002Fstrong\u003E link from the \u003Cstrong\u003ELogout\u003C\u002Fstrong\u003E page no longer redirects you to the \u003Cstrong\u003ELogout\u003C\u002Fstrong\u003E page once you have logged in.\u003C\u002Fli\u003E\n\u003Cli\u003EThe peek setting method, \u003Ccode\u003Eset_peek()\u003C\u002Fcode\u003E, should now function consistently across datatypes descending from \u003Ccode\u003EText\u003C\u002Fcode\u003E, particularly with respect to line count estimation.\u003C\u002Fli\u003E\n\u003Cli\u003EApply patch from Ry4an Brase to correct \"NoneType preference on the jobs view\" issue.\u003C\u002Fli\u003E\n\u003Cli\u003EDon't alter the contents of a file while uploading to a data library if using one of the \u003Cstrong\u003EUpload files from file system paths\u003C\u002Fstrong\u003E or \u003Cstrong\u003EUpload a directory of files\u003C\u002Fstrong\u003E options in conjunction with the \u003Cstrong\u003ELink to files without copying into Galaxy\u003C\u002Fstrong\u003E option. This partially resolves the issue where a supposedly sorted \u003Cem\u003EBAM\u003C\u002Fem\u003E file was being resorted upon upload to a data library when using this option. A better implementation of determining whether a \u003Cem\u003EBAM\u003C\u002Fem\u003E file has been sorted (so that it does not get resorted) remains to be done.\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Cstrong\u003EExtract Genomic DNA\u003C\u002Fstrong\u003E tool: \u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003EDo not fix strand for \u003Cem\u003EGFF\u003C\u002Fem\u003E input.\u003C\u002Fli\u003E\n\u003Cli\u003EHandle \u003Cem\u003Enon-GFF\u003C\u002Fem\u003E files when interpret features is \u003Ccode\u003Etrue\u003C\u002Fcode\u003E.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003ESet default value for \u003Cem\u003ECuffdiff's\u003C\u002Fem\u003E minimum alignment count parameter to reflect v0.9.3.\u003C\u002Fli\u003E\n\u003Cli\u003EEnhance \u003Cem\u003EGFF\u003C\u002Fem\u003E reader to handle headers and comments and fix bug so that new feature starts when \u003Ccode\u003Etranscript_id\u003C\u002Fcode\u003E or \u003Ccode\u003Egene_id\u003C\u002Fcode\u003E changes.\u003C\u002Fli\u003E\n\u003Cli\u003ESuppress most logging for \u003Cem\u003ECufflinks\u003C\u002Fem\u003E and \u003Cem\u003ECuffdiff\u003C\u002Fem\u003E as the logging can generate output that is incompatible with \u003Ccode\u003EUTF-8 format\u003C\u002Fcode\u003E used by database.\u003C\u002Fli\u003E\n\u003Cli\u003ESet message type to error when history deletion fails due to sharing.\u003C\u002Fli\u003E\n\u003Cli\u003EShow masthead links to \u003Cstrong\u003EPublished Visualizations\u003C\u002Fstrong\u003E and \u003Cstrong\u003EPublished Pages\u003C\u002Fstrong\u003E only if \u003Cstrong\u003ETracks\u003C\u002Fstrong\u003E or \u003Cstrong\u003EPages\u003C\u002Fstrong\u003E are enabled.\u003C\u002Fli\u003E\n\u003Cli\u003EShared visualizations work again.\u003C\u002Fli\u003E\n\u003Cli\u003EFix bug in \u003Ccode\u003Erecently used tool submenu\u003C\u002Fcode\u003E.\u003C\u002Fli\u003E\n\u003Cli\u003EChange default value for \u003Cem\u003ETophat\u003C\u002Fem\u003E coverage search to \u003Ccode\u003Etrue\u003C\u002Fcode\u003E.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Chr\u003E\n\u003Ch2 id=\"about-galaxy\"\u003E\u003Ca href=\"#about-galaxy\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EAbout Galaxy\u003C\u002Fh2\u003E\n\u003Cp\u003EThe \u003Cstrong\u003EGalaxy team\u003C\u002Fstrong\u003E is a part of \u003Ca href=\"http:\u002F\u002Fwww.bx.psu.edu\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EBX\u003C\u002Fa\u003E at \u003Ca href=\"http:\u002F\u002Fwww.psu.edu\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EPenn State\u003C\u002Fa\u003E, and the \u003Ca href=\"http:\u002F\u002Fwww.biology.emory.edu\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EBiology\u003C\u002Fa\u003E and \u003Ca href=\"http:\u002F\u002Fwww.mathcs.emory.edu\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EMathematics and Computer Science\u003C\u002Fa\u003E departments at \u003Ca href=\"http:\u002F\u002Fwww.emory.edu\u002Fhome\u002Findex.html\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EEmory University\u003C\u002Fa\u003E. \u003C\u002Fp\u003E\n\u003Cp\u003E\u003Cstrong\u003EGalaxy\u003C\u002Fstrong\u003E is supported in part by \u003Ca href=\"http:\u002F\u002Fwww.nsf.gov\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003ENSF\u003C\u002Fa\u003E, \u003Ca href=\"http:\u002F\u002Fwww.genome.gov\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003ENHGRI\u003C\u002Fa\u003E, the \u003Ca href=\"http:\u002F\u002Fwww.huck.psu.edu\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EHuck Institutes of the Life Sciences\u003C\u002Fa\u003E, and \u003Ca href=\"http:\u002F\u002Fwww.ics.psu.edu\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EThe Institute for CyberScience at Penn State\u003C\u002Fa\u003E, and \u003Ca href=\"http:\u002F\u002Fwww.emory.edu\u002Fhome\u002Findex.html\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EEmory University\u003C\u002Fa\u003E.\u003C\u002Fp\u003E\n\u003Cp\u003E\u003Ca href=\"http:\u002F\u002Fbitbucket.org\u002Fgalaxy\u002Fgalaxy-central\u002Fwiki\u002FGalaxyTeam\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EGalaxyTeam\u003C\u002Fa\u003E\u003C\u002Fp\u003E\n\u003Cp\u003E\u003Ca href=\"http:\u002F\u002Fusegalaxy.org\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EUse Galaxy!\u003C\u002Fa\u003E\u003C\u002Fp\u003E\n\u003Cp\u003E\u003Ca href=\"http:\u002F\u002Fgalaxyproject.org\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EGalaxyProject.org\u003C\u002Fa\u003E\u003C\u002Fp\u003E\n\u003Cp\u003EJoin us at \u003Cstrong\u003ETwitter\u003C\u002Fstrong\u003E\u003C\u002Fp\u003E\n\u003Cp\u003E\u003Ccode\u003E#usegalaxy\u003C\u002Fcode\u003E\u003C\u002Fp\u003E\n\u003Cp\u003E\u003Ca href=\"http:\u002F\u002Ftwitter.com\u002F#!\u002Fsearch\u002Fgalaxyproject\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttp:\u002F\u002Ftwitter.com\u002F#!\u002Fsearch\u002Fgalaxyproject\u003C\u002Fa\u003E \u003C\u002Fp\u003E\n\u003Chr\u003E\n"}},"context":{}};(function(){var s;(s=document.currentScript||document.scripts[document.scripts.length-1]).parentNode.removeChild(s);}());</script><script src="/assets/js/app.f7f9943a.js" defer></script><script src="/assets/js/page--src--templates--article-vue.9c961af0.js" defer></script>
  </body>
</html>
