<!DOCTYPE html>
<html data-html-server-rendered="true" lang="en" data-vue-tag="%7B%22lang%22:%7B%22ssr%22:%22en%22%7D%7D">
  <head>
    <title> - Galaxy Community Hub</title><meta name="gridsome:hash" content="64a9e8afa68c1466f7ef441294c96aecf9a68066"><meta data-vue-tag="ssr" charset="utf-8"><meta data-vue-tag="ssr" name="generator" content="Gridsome v0.7.23"><meta data-vue-tag="ssr" data-key="viewport" name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover"><meta data-vue-tag="ssr" data-key="format-detection" name="format-detection" content="telephone=no"><meta data-vue-tag="ssr" data-key="description" name="description" content="All about Galaxy and its community."><link data-vue-tag="ssr" rel="icon" href="data:,"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="16x16" href="/assets/static/favicon.ce0531f.7d4a227415c9d9b704d56a079d49a3e1.png"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="32x32" href="/assets/static/favicon.ac8d93a.7d4a227415c9d9b704d56a079d49a3e1.png"><link rel="preload" href="/assets/css/0.styles.5b1099da.css" as="style"><link rel="preload" href="/assets/js/app.6a2146ae.js" as="script"><link rel="preload" href="/assets/js/page--src--templates--article-vue.59a31f82.js" as="script"><link rel="prefetch" href="/assets/js/page--node-modules--gridsome--app--pages--404-vue.9b913011.js"><link rel="prefetch" href="/assets/js/page--src--pages--blog-vue.b9d631d5.js"><link rel="prefetch" href="/assets/js/page--src--pages--careers-vue.7bdfa9aa.js"><link rel="prefetch" href="/assets/js/page--src--pages--data-explorer-vue.50f81452.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--archive-vue.bdcb994f.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests--papercuts-vue.9624a935.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--cofests-vue.3da6c251.js"><link rel="prefetch" href="/assets/js/page--src--pages--events--webinars-vue.4cf7aa8a.js"><link rel="prefetch" href="/assets/js/page--src--pages--events-vue.3f22c4a9.js"><link rel="prefetch" href="/assets/js/page--src--pages--index-vue.8b7e39d4.js"><link rel="prefetch" href="/assets/js/page--src--pages--news-vue.357930b0.js"><link rel="prefetch" href="/assets/js/page--src--pages--search-vue.1a4d1477.js"><link rel="prefetch" href="/assets/js/page--src--pages--tests-vue.093ab612.js"><link rel="prefetch" href="/assets/js/page--src--pages--use-vue.c890644b.js"><link rel="prefetch" href="/assets/js/page--src--templates--platform-vue.0db261b0.js"><link rel="prefetch" href="/assets/js/page--src--templates--vue-article-vue.454443f9.js"><link rel="prefetch" href="/assets/js/vendors~page--src--pages--careers-vue~page--src--pages--use-vue~page--src--templates--article-vue~pa~efade918.c3041708.js"><link rel="prefetch" href="/assets/js/vendors~page--src--pages--search-vue.5670e8f9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--index-md.7ef1364e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--md--imgtests--imgdir--index-md.2e934c33.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--partials--index-md.eeba3cb6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--imgdir--index-md.92886b7f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--imgtests--index-md.eab5717f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--index-md.e1baaf22.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--index-md.861c549e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--0-tests--vue-remark--syntaxtests--reldir--index-md.aa6f3c64.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--capacity-planning--index-md.bfc3219e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--index-md.bec55add.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--config--performance--cluster--legacy--index-md.e37560d2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--datatypes--index-md.534701f8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--disk-quotas--index-md.33c98ae0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--faq--index-md.8a142c3e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--data-model--index-md.dbe6d521.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--internals--galaxy-ops--index-md.ae261eb0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--ten-simple-steps-galaxy-as-a-service--index-md.7f58c8af.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--tool-config-syntax--index-md.cd8060f8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--admin--tools--writing-tests--index-md.67221f9c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--allocation--index-md.de0ec1f4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--faq--index-md.4669ecb3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--index-md.cf67f110.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--ssh--index-md.0e243102.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloud--jetstream--troubleshooting--index-md.35efbe50.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--capacity-planning--index-md.dc90802a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--getting-started--index-md.2ca2f401.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--aws--index-md.8415a706.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--building--index-md.d98aeff8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--capacity-planning--index-md.4510948e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--cluster-types--index-md.cac4089c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--customizing--index-md.e2b286b6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--faq--index-md.2b285c3a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--index-md.ea2ebc17.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--getting-started--pre-201703--index-md.11c5fe6a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--hadoop--index-md.bb206ee6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--ht-condor--index-md.9da34f44.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--index-md.233a66d8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--openstack--index-md.ab85a30a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--services--index-md.10c9f181.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--sharing--index-md.d6d1b6d6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--troubleshooting--index-md.9c79821f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--cloudman--userdata--index-md.53f02454.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--future--index-md.57f97361.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--index-md.9651e1c9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-09--index-md.ad9dcbc5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-07-27--index-md.7471bbe4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-09-19--index-md.e1e5e8d8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2012-11-14--index-md.9f83078a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-01-16--index-md.4539d357.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-03-20--index-md.70ed6c95.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-05-15--index-md.4688a0b3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2013-11-20--index-md.f2ba774b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-02-19--index-md.79bdf65d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-04-16--index-md.4a242376.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-06-18--index-md.440620d8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-08-20--index-md.376ae2ed.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2015-10-15--index-md.27dc34e9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-02-18--index-md.fdb95229.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-04-21--index-md.f92b0f6d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-06-16--index-md.fe11d204.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--2016-08-18--index-md.f52ed1f8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--index-md.9f7dc20a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--meetups--webinar-tech--index-md.d949fe23.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2012--index-md.c7266273.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--2014--index-md.e4286fb0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--community--galaxy-admins--surveys--index-md.d66e06d1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--data--index-md.20e48791.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--datafiles--index-md.2d10c564.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--architecture--index-md.4629c460.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--bitbucket--index-md.74b0114b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--css--index-md.657cefe3.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--getting-started--index-md.328bffc8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--index-md.09543e4e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--jsa--index-md.6375449f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--mercurial--index-md.86dc9b7b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--develop--security-tool-tips--index-md.f4faf0d9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--galaxy-project--index-md.3e02395c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--index-md.4d9d00ba.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--papers--index-md.c7f4278a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--posters--index-md.75155b75.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--presentations--index-md.28312b2f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--documents--videos--index-md.53f176ff.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--advanced-session--index-md.6bd7cd89.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--basics-session--index-md.a7aa2234.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--index-md.881dccf2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--logistics--index-md.1a0b965f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--publicity--index-md.5fc8d7c6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--admin-training-2016--registration--index-md.a85019e7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--abstracts--index-md.f45d8543.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--index-md.41cde30b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--key-dates--index-md.eb4461fb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--lightning-talks--index-md.416549ba.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--logistics--index-md.a40269b5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--organizing-committee--index-md.b5fc35e6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--photos--index-md.a9d9d78d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--automation-strategies--index-md.53a84cb2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--bioinformatics-training--index-md.5576edcf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--end-users--index-md.0b5fa62e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--breakouts--workflows-and-api--index-md.03c0504a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--program--index-md.b8fa162b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--promotion--index-md.f38b09af.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--register--index-md.f4320c18.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--sponsorships--index-md.3e9b6ca4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2012--training-day--index-md.30df8ca8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--index-md.53afc104.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--posters--index-md.5a1c5bbd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--abstracts--talks--index-md.24bf003b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--clinical-use-hpc-file-systems--index-md.54a9c0d6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-admins--index-md.4e804331.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-blast--index-md.5d5d7573.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--galaxy-professionals--index-md.ee96dc73.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--index-md.e819dfd1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--proteomics--index-md.adb5209b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--public-galaxy-servers--index-md.5b535d3b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--bof--tool-developers--index-md.934ab089.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--index-md.fc9e5d60.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--key-dates--index-md.9feb96c8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--lightning--index-md.986b956c.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--logistics--index-md.aafc45da.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--organizers--index-md.8129c4fe.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--photos--index-md.9f48888b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--program--index-md.afd0b377.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--promotion--index-md.84a5ce7a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--register--index-md.e8a8cb9e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--sponsorships--index-md.f3562bcc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--advance-tool-data--index-md.f69ec61d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--api--index-md.e7ae288d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--index-md.4419ab24.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--intro-tool-data--index-md.2505a2c4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2013--training-day--vms--index-md.31fa2f12.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--index-md.44077d78.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--posters--index-md.968309e7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--abstracts--talks--index-md.3bd80461.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--branch-release-merge--index-md.cc4e0d0a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-admins--index-md.c1aa37dd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-end-users--index-md.3ddb7f3b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--galaxy-training-network--index-md.43cdc565.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--heterogeneous--index-md.810f499a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--index-md.27205369.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--bofs--using-galaxy-ui-overview--index-md.770275d4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--hackathon--index-md.23dc0e2f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--index-md.b84a33d4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--key-dates--index-md.afa58daf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--lightning--index-md.6722fb11.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--logistics--index-md.74e0dab1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--organizers--index-md.80304844.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--photos--index-md.976fd862.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--program--index-md.d8e5ebc7.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--promotion--index-md.577f91cb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--register--index-md.09b57f47.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--sponsor-exhibit--index-md.d28c30a0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--admin-walkthrough--index-md.e70f4130.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--api--index-md.3e466840.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--data-managers--index-md.c4c90feb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--index-md.09eceaad.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2014--training-day--vms--index-md.a9f99c81.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--cwl--index-md.572d144e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-management-challenges--index-md.6519a99a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--data-splitting-and-parallelism--index-md.f9225e89.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--elixir--index-md.83e340c0.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--extended-workflow--index-md.84d907aa.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--fots--index-md.9eb9fbae.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-admins--index-md.013a0661.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-community-uk--index-md.73da832e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-scientists--index-md.8271737e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--galaxy-training-network--index-md.2c5cf472.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--bofs--index-md.50db5ff8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--data-hack-roundup--index-md.54209e2d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--header--index-md.99f7b8a4.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2015--index-md.b607d569.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2016--bofs--index-md.1a1d782b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--abstracts--index-md.8a86f254.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--childcare--index-md.46870d68.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--cofest--index-md.558efeda.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--conduct--index-md.4cdcc7b1.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--faq--index-md.54d3f72a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--index-md.03d62056.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--organizers--index-md.0ca6c313.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--promotion--index-md.a0d7a442.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--registration--index-md.be067239.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--sponsors--index-md.b823edb9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--training--index-md.8e06e99f.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gcc-2019--venue--index-md.bd503783.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--abstracts--index-md.7206d890.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--childcare--index-md.7f23990a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--code-of-conduct--index-md.9c7749ad.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--collaboration--index-md.d2df60a9.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--faq--index-md.3263be33.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--icebreaker--index-md.ab42ed08.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--index-md.2726d22d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--keynotes--index-md.dfee893e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--organizers--index-md.7cdb9d53.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--promotion--index-md.3881bf6d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--register--index-md.5ca35f56.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--sponsors--index-md.8fef601a.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--training--index-md.37b79dad.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--gccbosc-2018--venue--index-md.7c4dd44e.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2014--index-md.286c0184.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--germany-2016--index-md.6c02f7a2.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2014--index-md.e1f8864b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--sg-2016--index-md.eed2cd9b.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--index-md.5356abdf.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2014--trainingday--index-md.45d2d738.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--events--switzerland-2016--index-md.837d1830.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--example--vue-remark--index-md.972b7a38.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--faqs--index-md.ecb23aeb.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-06--index-md.dfe44fb6.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--galaxy-updates--2013-07--index-md.c147ef3d.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--icons--index-md.b3e66524.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--index-md.27f20470.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--logos--index-md.5cf6a844.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--news-graphics--index-md.68e69490.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--photos--index-md.96cfd5fc.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--images--screenshots--index-md.6188b2ad.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--index-md.c0be3976.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--basic-editing--workflow-editor-unannotated--index-md.c7e004d5.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--extract--index-md.1f732370.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--tool-panel--index-md.861a5048.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--advanced-workflow--variables--index-md.c521b297.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--faq--index-md.1591ea49.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--galaxy-pages--index-md.321b9ce8.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--learn--interval-operations--index-md.cab27468.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--main--faq--index-md.7ee220fd.js"><link rel="prefetch" href="/assets/js/vue-remark--build--content-vue--tools--requirements--index-md.bcb851a8.js"><link rel="stylesheet" href="/assets/css/0.styles.5b1099da.css"><noscript data-vue-tag="ssr"><style>.g-image--loading{display:none;}</style></noscript>
  </head>
  <body >
    <div data-server-rendered="true" id="app" class="layout" data-v-14d6d1e0><header id="masthead"><nav class="container navbar navbar-expand-lg navbar-dark justify-content-center"><a href="/" class="navbar-brand active"><img id="masthead-logo" src="/images/galaxy_logo_hub_white.svg" alt="Galaxy Community Hub" height="30"></a><div id="navbarToggleWrapper" class="collapse navbar-collapse mx-auto"><ul class="navbar-nav"><li class="nav-item"><a href="/use/" role="button" aria-expanded="false" class="nav-link dropdown">
          Use
        </a></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" aria-expanded="false" class="nav-link dropdown-toggle">
          Learn
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/learn/" class="dropdown-item">How to use Galaxy</a></li><li><a href="https://training.galaxyproject.org/" class="dropdown-item">Teach with Galaxy</a></li></ul></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" class="nav-link dropdown-toggle">
          Community
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/community/" class="dropdown-item">About</a></li><li><a href="/events/" class="dropdown-item">Events</a></li><li><a href="/news/" class="dropdown-item">News</a></li><li><a href="/blog/" class="dropdown-item">Blog</a></li><li><a href="/publication-library/" class="dropdown-item">Publications</a></li><li><a href="/galaxy-project/statistics/" class="dropdown-item">Stats</a></li><li><a href="/careers/" class="dropdown-item">Careers</a></li></ul></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" class="nav-link dropdown-toggle">
          Deploy &amp; Develop
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/admin/get-galaxy/" class="dropdown-item">Install Galaxy</a></li><li><a href="/admin/" class="dropdown-item">Server administration</a></li><li><a href="/tools/" class="dropdown-item">Tools</a></li><li><a href="/develop/api/" class="dropdown-item">Working with the API</a></li><li><a href="/develop/" class="dropdown-item">Contributing</a></li><li><a href="/docs/" class="dropdown-item">Code documentation</a></li></ul></li><li class="nav-item dropdown"><a href="#" data-toggle="dropdown" role="button" class="nav-link dropdown-toggle">
          Support
          <span class="caret"></span></a><ul class="dropdown-menu"><li><a href="/support/" class="dropdown-item active">FAQ</a></li><li><a href="https://help.galaxyproject.org/" class="dropdown-item">Galaxy Help Forum</a></li></ul></li><li class="nav-item"><a href="/jxtx/" role="button" aria-expanded="false" class="nav-link dropdown">
          @jxtx
        </a></li></ul><ul class="navbar-nav ml-auto"><li><form method="get" action="/search/" class="my-2 navbar-form"><div class="input-group"><input id="search-input" name="q" placeholder="Search Galaxy" class="form-control"><span class="input-group-btn"><button type="submit" class="btn btn-secondary"><span class="fa fa-search"></span></button></span></div></form></li></ul></div></nav></header><section id="maincontainer" class="container"><header data-v-08ecca1f data-v-14d6d1e0><a href="/" class="link active" data-v-08ecca1f>← Back to Home</a><!----><h1 class="title" data-v-08ecca1f></h1><!----><section class="metadata freetext" data-v-08ecca1f><!----><!----><!----></section><!----><!----></header><div class="content markdown" data-v-14d6d1e0><div class="toc-wrapper col-md-3">
<ul>
<li>
<p><a href="#chromosome-identifiers-in-reference-genomes-and-other--omes">Chromosome Identifiers in Reference Genomes (and other -omes)</a></p>
<ul>
<li><a href="#find-bam-dataset-identifiers">Find BAM dataset identifiers</a></li>
<li><a href="#directly-obtain-ucsc-sourced-genome-identifiers">Directly obtain UCSC sourced <em>genome</em> identifiers</a></li>
<li>
<p><a href="#adjusting-identifiers-or-input-source">Adjusting Identifiers or Input source</a></p>
<ul>
<li><a href="#ucsc-sourced-data-used-with-other-sourced-data">UCSC sourced data used with Other sourced data</a></li>
<li><a href="#any-mixed-sourced-data">Any mixed sourced data</a></li>
</ul>
</li>
<li><a href="#a-note-on-built-in-reference-genomes">A Note on Built-in Reference Genomes</a></li>
</ul>
</li>
</ul>
</div>
<div class="body-wrapper col-md-9">
<h1 id="chromosome-identifiers-in-reference-genomes-and-other--omes"><a href="#chromosome-identifiers-in-reference-genomes-and-other--omes" aria-hidden="true"><span class="icon icon-link"></span></a>Chromosome Identifiers in Reference Genomes (and other -omes)</h1>
<p><strong><a href="/support/">Back to Support Hub</a></strong>
<strong><a href="/support/#troubleshooting">Troubleshooting Help</a></strong></p>
<p>Methods described help to identify and correct errors or unexpected results linked to inputs having non-identical chromosome identifiers and/or different chromosome sequence content.</p>
<p><strong>If using a Custom Reference genome</strong>, the methods below also apply, but the first step is to make certain that the <a href="/learn/custom-genomes/">Custom Genome is formatted correctly</a>. Improper formating is the most common root cause of CG related errors.</p>
<h2 id="find-bam-dataset-identifiers"><a href="#find-bam-dataset-identifiers" aria-hidden="true"><span class="icon icon-link"></span></a>Find BAM dataset identifiers</h2>
<p>Quickly learn what the identifiers are in any <strong>BAM</strong> dataset that is the result from mapping.</p>
<p>Warning: This will <em>not</em> work for "sequence-only" <code>bam</code> datasets, as these usually have no header.</p>
<p><em>Method 1</em></p>
<ol>
<li>Run <strong>Samtools: IdxStats</strong> on the aligned data (<code>bam</code> dataset).</li>
<li>The "index header" chromosome names and lengths will be listed in the output (along with read counts).</li>
<li>Compare the chromosome identifiers to the chromosome (aka "chrom") field in all other inputs: VCF, GTF, GFF(3), BED, Interval, etc.</li>
<li>Note: The orignal mapping targat may have been a built-in genome index, custom genome (transcriptome, exome, other) -- the same <code>bam</code> data will still be summarized.</li>
</ol>
<h2 id="directly-obtain-ucsc-sourced-genome-identifiers"><a href="#directly-obtain-ucsc-sourced-genome-identifiers" aria-hidden="true"><span class="icon icon-link"></span></a>Directly obtain UCSC sourced <em>genome</em> identifiers</h2>
<p><em>Method 2</em></p>
<ol>
<li>Go to <a href="http://genome.ucsc.edu/" target="_blank" rel="noopener noreferrer">http://genome.ucsc.edu/</a>, navigate to "genomes", then the species of interest</li>
<li>On the home page for the genome build, immediately under the top navigation box, in the blue bar next to the full genome build name, is linked text like "(sequences)"</li>
<li>Click on this and it will take you to a detail page with a table listing out the contents</li>
</ol>
<p><em>Method 3</em></p>
<ol>
<li>Use the tool "Get Data -> UCSC Main"</li>
<li>In the Table Browser, choose the target genome and build</li>
<li>For "group" choose the last option "All Tables"</li>
<li>For "table" choose "chromInfo"</li>
<li>Leave all other options at default and send the output to Galaxy</li>
<li>This new dataset will load as a tabular dataset into your history</li>
<li>It will list out the contents of the genome build, including the chromosome identifiers (in the first column)</li>
</ol>
<h2 id="adjusting-identifiers-or-input-source"><a href="#adjusting-identifiers-or-input-source" aria-hidden="true"><span class="icon icon-link"></span></a>Adjusting Identifiers or Input source</h2>
<h3 id="ucsc-sourced-data-used-with-other-sourced-data"><a href="#ucsc-sourced-data-used-with-other-sourced-data" aria-hidden="true"><span class="icon icon-link"></span></a>UCSC sourced data used with Other sourced data</h3>
<p>A GTF formatted dataset (potentially a "reference annotation dataset"), with Ensembl/UCSC/Other based chromosome identifiers, is to be used with a tool that also makes use of a different sourced reference genome</p>
<p>Or, the reverse may be true, Ensembl/UCSC/Other sourced reference genome and a differnt sourced reference annotation</p>
<p><em>The underlying genome sequence content is otherwise identical.</em> If not, see the next section for alternative methods.</p>
<p><em>Method 4</em></p>
<p>To adjust the Ensembl/Other reference annotation to match a UCSC-sourced reference genome (or another source that uses UCSC-style chromosome names), add a "chr" to the chromosome name, so that "N" becomes "chrN". Using tools from the group "Text Manipulation". Examples below.</p>
<p>For <strong><a href="/learn/datatypes/#bed">bed</a></strong> data:</p>
<ol>
<li>Tool <strong>Add column</strong>: add "chr" to the original dataset as a new column.</li>
<li>Tool <strong>Merge Columns</strong>: merge "c7" with "c1"</li>
<li>Tool <strong>Cut</strong>: cut "c8,c2,c3,c4,c5,c6" (replace c1 &#x26; c7 - with merged c8 - the new chrom identifier)</li>
<li>Click on the pencil icon for the result dataset, then the tab for "Datatype". Assign "bed" and save. Allow the metadata to complete assignment (the "yellow" dataset state)</li>
<li>Now click on the tab for "Attributes" and assign the remaining columns. Strand = 6, name = 4, and score = 5. Save. For best results with certain downstream tools, allow the metadata to complete assignment</li>
</ol>
<p>For <strong><a href="/learn/datatypes/#wig-and-bigwig">wig/wiggle</a></strong> data (NOT compressed bigWig):</p>
<ol>
<li>Tool <strong>Replace parts of text</strong></li>
<li>File to process: Use Multi-select select wig datasets to fix (one or more)</li>
<li>Find pattern: chrom=</li>
<li>Replace with: chrom=chr</li>
<li>Remainder of options left at default</li>
</ol>
<h3 id="any-mixed-sourced-data"><a href="#any-mixed-sourced-data" aria-hidden="true"><span class="icon icon-link"></span></a>Any mixed sourced data</h3>
<p>The inputs are a match for sequence content but simply adding "chr" will not make all chromosomes identifiers synch up between the inputs. How to fix or replace the inputs so that a match is possible.</p>
<p><em>The underlying genome sequence content may or may not be identical.</em> Read method descriptions carefully to learn if that method is right for your usage case.</p>
<p><em>Method 5</em></p>
<p><strong>Sequence content is a match but adding "chr" is not enough to obtain an exact identifier match. You want to try to fix the identifiers anyway!!</strong></p>
<ol>
<li>If the data is in a tabular format (BED, Interval, GTF -- with any headers removed first), and a suitable identifier mapping file can be obtained or created, the tool <em>Replace column by values which are defined in a convert file</em> can be used. Note that this will NOT work with BAM, VCF, Wiggle or other structured formats, as these are not tabular formatted data.</li>
<li>Manipuations with tools can often be used to split up a dataset, perform text substitutions and additions, concatinate datasets, and most other common operations one could do with command-line shell tools.</li>
<li>The dataset could also be downloaded locally to your computer and manipulated there using command-line tools or the text editor of choice.</li>
</ol>
<p><em>Method 6</em></p>
<p><strong>Sequence content is a match but adding "chr" is not enough to obtain an exact identifier match. You DO NOT want to try to fix the identifiers or it is overly complicated or it is simply not possible to fix the data without an external reference mapping file (not always available).</strong></p>
<ol>
<li>Obtain a reference annotation dataset that is a match for the reference genome used</li>
<li>Sometimes the source is the same for both</li>
<li>Sometimes the source is the same, but the content of the reference annotation is not ideal for the tools used</li>
</ol>
<ul>
<li>Example: The tool Cuffdiff makes use of specific attributes in the reference annotation (p_id, tss_id, gene_name). If these attributes are not present in the GTF dataset, the resuls will not be fully annotated and some calculations will be skipped</li>
<li>Use the iGenomes version of the reference annotation, as described below</li>
<li>Using Cuffdiff and the Gene ID is not present? Check your GTF file - the attribute gene_name is probably missing</li>
</ul>
<ol start="4">
<li>Sometimes the source can be <strong>iGenomes</strong>, which does contain the extra specific attributes needed for RNA-seq and certain other operationsar</li>
</ol>
<ul>
<li>Example: The tool htseq-count is used and the attributes gene_id and transcript_id need to be distinct values (also true for the tool Cuffdiff for the best results)</li>
<li>Two sources: <a href="https://support.illumina.com/sequencing/sequencing_software/igenome.html" target="_blank" rel="noopener noreferrer">https://support.illumina.com/sequencing/sequencing_software/igenome.html</a> and <a href="http://cole-trapnell-lab.github.io/cufflinks/igenome_table/index.html" target="_blank" rel="noopener noreferrer">http://cole-trapnell-lab.github.io/cufflinks/igenome_table/index.html</a></li>
<li>Download the .tar file locally, uncompress it, then upload just the genes.gtf dataset to Galaxy</li>
<li>Note: the compression format .tar is not accepted by the Upload tool</li>
<li>If a .tar dataset is attempted to be uploaded, the load may fail or just the first file in the archive is uploaded (and it will not be the genes.gtf file)</li>
</ul>
<ol start="5">
<li><a href="https://www.gencodegenes.org" target="_blank" rel="noopener noreferrer">Genecode Genes</a> is also an annotation source for some genome builds.</li>
<li>In summary:</li>
</ol>
<ul>
<li>For <strong>Gencode</strong>, copy the link to the GTF and paste it into the <em>Upload</em> tool. Hg38 data is here <a href="https://www.gencodegenes.org/" target="_blank" rel="noopener noreferrer">https://www.gencodegenes.org/</a>. After it is loaded, remove the headers (lines that start with a "#") with the <em>Select</em> tool using the options "NOT Matching" with the regular expression <code>^#</code>. Once the formatting is fixed, change the datatype to be <code>gft</code> under Edit Attributes (pencil icon). The data will be given the datatype <code>gff</code> by default, which works fine with some tools and but not with others. Avoid the <code>gff3</code> version of this particular data (contains duplicated IDs and several RNA-seq tools do not work with annotation in that format anyway).</li>
<li>For <strong>iGenomes</strong>, the archive corresponding to the target genome/build needs to be locally downloaded, the tar archive unpacked, and then just the <code>genes.gtf</code> data uploaded to Galaxy (browse the local file, or use FTP). Find all available genome/builds here: <a href="https://support.illumina.com/sequencing/sequencing_software/igenome.html" target="_blank" rel="noopener noreferrer">https://support.illumina.com/sequencing/sequencing_software/igenome.html</a></li>
</ul>
<p><em>Method 7</em></p>
<p><strong>Sequence content is NOT a match or you want to try using a different reference genome instead of a different reference annotation source (reverse of <em>Method 6</em> above).</strong></p>
<ol>
<li>Map against the same reference genome that the reference annotation is based on</li>
<li>Where and if this reference genome is available will depend on the genome build</li>
<li>In most cases, the source will be the same for both</li>
<li>If loading your own genome, make sure it is formatted correctly as a Custom Genome</li>
<li>Promote the Custom Genome to a Custom Build and assign the genome/build metadata attribute to datasets</li>
<li><a href="/learn/custom-genomes/">Custom Reference Genome help</a></li>
<li>Be aware that if the genome is large, this option may result in a memory failure. Try <em>Method 2</em> or consider moving to a local or cloud Galaxy where you can control the resources</li>
</ol>
<h2 id="a-note-on-built-in-reference-genomes"><a href="#a-note-on-built-in-reference-genomes" aria-hidden="true"><span class="icon icon-link"></span></a>A Note on Built-in Reference Genomes</h2>
<p>The default variant for all genomes is "Full", defined as all primary chromosomes (or scaffolds/contigs) including mitochondrial plus associated unmapped, plasmid, and other segments.</p>
<p>When only one version of a genome is available for a tool, it represents the default "Full" variant.</p>
<p>Some genomes will have more than one variant available.</p>
<ul>
<li>The "Canonical Male" or sometimes simply "Canonical" variant contains the primary chromosomes for a genome. For example a human "Canonical" variant contains chr1-chr22, chrX, chrY, and chrM.</li>
<li>The "Canonical Female" variant contains the primary chromosomes excluding chrY.</li>
</ul>
</div>
</div><footer data-v-fcf53fc4 data-v-14d6d1e0><!----></footer><footer class="static-footer markdown"><p>The Galaxy Project is supported in part by NSF, NHGRI, The Huck Institutes of the Life Sciences, The Institute for CyberScience at Penn State, and Johns Hopkins University.</p>
</footer></section></div>
    <script>window.__INITIAL_STATE__={"data":{"article":{"id":"a8c96601e03e30ef7d83bce28d064d23","title":"","tease":"","category":null,"date":null,"days":null,"contact":"","contact_url":"","authors":"","location":"","location_url":"","source_blog":"","source_blog_url":"","skip_title_render":null,"links":[],"image":"","images":{},"external_url":"","content":"\u003Cdiv class=\"toc-wrapper col-md-3\"\u003E\n\u003Cul\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Ca href=\"#chromosome-identifiers-in-reference-genomes-and-other--omes\"\u003EChromosome Identifiers in Reference Genomes (and other -omes)\u003C\u002Fa\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"#find-bam-dataset-identifiers\"\u003EFind BAM dataset identifiers\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#directly-obtain-ucsc-sourced-genome-identifiers\"\u003EDirectly obtain UCSC sourced \u003Cem\u003Egenome\u003C\u002Fem\u003E identifiers\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\n\u003Cp\u003E\u003Ca href=\"#adjusting-identifiers-or-input-source\"\u003EAdjusting Identifiers or Input source\u003C\u002Fa\u003E\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003E\u003Ca href=\"#ucsc-sourced-data-used-with-other-sourced-data\"\u003EUCSC sourced data used with Other sourced data\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#any-mixed-sourced-data\"\u003EAny mixed sourced data\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"#a-note-on-built-in-reference-genomes\"\u003EA Note on Built-in Reference Genomes\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fdiv\u003E\n\u003Cdiv class=\"body-wrapper col-md-9\"\u003E\n\u003Ch1 id=\"chromosome-identifiers-in-reference-genomes-and-other--omes\"\u003E\u003Ca href=\"#chromosome-identifiers-in-reference-genomes-and-other--omes\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EChromosome Identifiers in Reference Genomes (and other -omes)\u003C\u002Fh1\u003E\n\u003Cp\u003E\u003Cstrong\u003E\u003Ca href=\"\u002Fsupport\u002F\"\u003EBack to Support Hub\u003C\u002Fa\u003E\u003C\u002Fstrong\u003E\n\u003Cstrong\u003E\u003Ca href=\"\u002Fsupport\u002F#troubleshooting\"\u003ETroubleshooting Help\u003C\u002Fa\u003E\u003C\u002Fstrong\u003E\u003C\u002Fp\u003E\n\u003Cp\u003EMethods described help to identify and correct errors or unexpected results linked to inputs having non-identical chromosome identifiers and\u002For different chromosome sequence content.\u003C\u002Fp\u003E\n\u003Cp\u003E\u003Cstrong\u003EIf using a Custom Reference genome\u003C\u002Fstrong\u003E, the methods below also apply, but the first step is to make certain that the \u003Ca href=\"\u002Flearn\u002Fcustom-genomes\u002F\"\u003ECustom Genome is formatted correctly\u003C\u002Fa\u003E. Improper formating is the most common root cause of CG related errors.\u003C\u002Fp\u003E\n\u003Ch2 id=\"find-bam-dataset-identifiers\"\u003E\u003Ca href=\"#find-bam-dataset-identifiers\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EFind BAM dataset identifiers\u003C\u002Fh2\u003E\n\u003Cp\u003EQuickly learn what the identifiers are in any \u003Cstrong\u003EBAM\u003C\u002Fstrong\u003E dataset that is the result from mapping.\u003C\u002Fp\u003E\n\u003Cp\u003EWarning: This will \u003Cem\u003Enot\u003C\u002Fem\u003E work for \"sequence-only\" \u003Ccode\u003Ebam\u003C\u002Fcode\u003E datasets, as these usually have no header.\u003C\u002Fp\u003E\n\u003Cp\u003E\u003Cem\u003EMethod 1\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Col\u003E\n\u003Cli\u003ERun \u003Cstrong\u003ESamtools: IdxStats\u003C\u002Fstrong\u003E on the aligned data (\u003Ccode\u003Ebam\u003C\u002Fcode\u003E dataset).\u003C\u002Fli\u003E\n\u003Cli\u003EThe \"index header\" chromosome names and lengths will be listed in the output (along with read counts).\u003C\u002Fli\u003E\n\u003Cli\u003ECompare the chromosome identifiers to the chromosome (aka \"chrom\") field in all other inputs: VCF, GTF, GFF(3), BED, Interval, etc.\u003C\u002Fli\u003E\n\u003Cli\u003ENote: The orignal mapping targat may have been a built-in genome index, custom genome (transcriptome, exome, other) -- the same \u003Ccode\u003Ebam\u003C\u002Fcode\u003E data will still be summarized.\u003C\u002Fli\u003E\n\u003C\u002Fol\u003E\n\u003Ch2 id=\"directly-obtain-ucsc-sourced-genome-identifiers\"\u003E\u003Ca href=\"#directly-obtain-ucsc-sourced-genome-identifiers\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EDirectly obtain UCSC sourced \u003Cem\u003Egenome\u003C\u002Fem\u003E identifiers\u003C\u002Fh2\u003E\n\u003Cp\u003E\u003Cem\u003EMethod 2\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Col\u003E\n\u003Cli\u003EGo to \u003Ca href=\"http:\u002F\u002Fgenome.ucsc.edu\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttp:\u002F\u002Fgenome.ucsc.edu\u002F\u003C\u002Fa\u003E, navigate to \"genomes\", then the species of interest\u003C\u002Fli\u003E\n\u003Cli\u003EOn the home page for the genome build, immediately under the top navigation box, in the blue bar next to the full genome build name, is linked text like \"(sequences)\"\u003C\u002Fli\u003E\n\u003Cli\u003EClick on this and it will take you to a detail page with a table listing out the contents\u003C\u002Fli\u003E\n\u003C\u002Fol\u003E\n\u003Cp\u003E\u003Cem\u003EMethod 3\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Col\u003E\n\u003Cli\u003EUse the tool \"Get Data -\u003E UCSC Main\"\u003C\u002Fli\u003E\n\u003Cli\u003EIn the Table Browser, choose the target genome and build\u003C\u002Fli\u003E\n\u003Cli\u003EFor \"group\" choose the last option \"All Tables\"\u003C\u002Fli\u003E\n\u003Cli\u003EFor \"table\" choose \"chromInfo\"\u003C\u002Fli\u003E\n\u003Cli\u003ELeave all other options at default and send the output to Galaxy\u003C\u002Fli\u003E\n\u003Cli\u003EThis new dataset will load as a tabular dataset into your history\u003C\u002Fli\u003E\n\u003Cli\u003EIt will list out the contents of the genome build, including the chromosome identifiers (in the first column)\u003C\u002Fli\u003E\n\u003C\u002Fol\u003E\n\u003Ch2 id=\"adjusting-identifiers-or-input-source\"\u003E\u003Ca href=\"#adjusting-identifiers-or-input-source\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EAdjusting Identifiers or Input source\u003C\u002Fh2\u003E\n\u003Ch3 id=\"ucsc-sourced-data-used-with-other-sourced-data\"\u003E\u003Ca href=\"#ucsc-sourced-data-used-with-other-sourced-data\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EUCSC sourced data used with Other sourced data\u003C\u002Fh3\u003E\n\u003Cp\u003EA GTF formatted dataset (potentially a \"reference annotation dataset\"), with Ensembl\u002FUCSC\u002FOther based chromosome identifiers, is to be used with a tool that also makes use of a different sourced reference genome\u003C\u002Fp\u003E\n\u003Cp\u003EOr, the reverse may be true, Ensembl\u002FUCSC\u002FOther sourced reference genome and a differnt sourced reference annotation\u003C\u002Fp\u003E\n\u003Cp\u003E\u003Cem\u003EThe underlying genome sequence content is otherwise identical.\u003C\u002Fem\u003E If not, see the next section for alternative methods.\u003C\u002Fp\u003E\n\u003Cp\u003E\u003Cem\u003EMethod 4\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Cp\u003ETo adjust the Ensembl\u002FOther reference annotation to match a UCSC-sourced reference genome (or another source that uses UCSC-style chromosome names), add a \"chr\" to the chromosome name, so that \"N\" becomes \"chrN\". Using tools from the group \"Text Manipulation\". Examples below.\u003C\u002Fp\u003E\n\u003Cp\u003EFor \u003Cstrong\u003E\u003Ca href=\"\u002Flearn\u002Fdatatypes\u002F#bed\"\u003Ebed\u003C\u002Fa\u003E\u003C\u002Fstrong\u003E data:\u003C\u002Fp\u003E\n\u003Col\u003E\n\u003Cli\u003ETool \u003Cstrong\u003EAdd column\u003C\u002Fstrong\u003E: add \"chr\" to the original dataset as a new column.\u003C\u002Fli\u003E\n\u003Cli\u003ETool \u003Cstrong\u003EMerge Columns\u003C\u002Fstrong\u003E: merge \"c7\" with \"c1\"\u003C\u002Fli\u003E\n\u003Cli\u003ETool \u003Cstrong\u003ECut\u003C\u002Fstrong\u003E: cut \"c8,c2,c3,c4,c5,c6\" (replace c1 &#x26; c7 - with merged c8 - the new chrom identifier)\u003C\u002Fli\u003E\n\u003Cli\u003EClick on the pencil icon for the result dataset, then the tab for \"Datatype\". Assign \"bed\" and save. Allow the metadata to complete assignment (the \"yellow\" dataset state)\u003C\u002Fli\u003E\n\u003Cli\u003ENow click on the tab for \"Attributes\" and assign the remaining columns. Strand = 6, name = 4, and score = 5. Save. For best results with certain downstream tools, allow the metadata to complete assignment\u003C\u002Fli\u003E\n\u003C\u002Fol\u003E\n\u003Cp\u003EFor \u003Cstrong\u003E\u003Ca href=\"\u002Flearn\u002Fdatatypes\u002F#wig-and-bigwig\"\u003Ewig\u002Fwiggle\u003C\u002Fa\u003E\u003C\u002Fstrong\u003E data (NOT compressed bigWig):\u003C\u002Fp\u003E\n\u003Col\u003E\n\u003Cli\u003ETool \u003Cstrong\u003EReplace parts of text\u003C\u002Fstrong\u003E\u003C\u002Fli\u003E\n\u003Cli\u003EFile to process: Use Multi-select select wig datasets to fix (one or more)\u003C\u002Fli\u003E\n\u003Cli\u003EFind pattern: chrom=\u003C\u002Fli\u003E\n\u003Cli\u003EReplace with: chrom=chr\u003C\u002Fli\u003E\n\u003Cli\u003ERemainder of options left at default\u003C\u002Fli\u003E\n\u003C\u002Fol\u003E\n\u003Ch3 id=\"any-mixed-sourced-data\"\u003E\u003Ca href=\"#any-mixed-sourced-data\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EAny mixed sourced data\u003C\u002Fh3\u003E\n\u003Cp\u003EThe inputs are a match for sequence content but simply adding \"chr\" will not make all chromosomes identifiers synch up between the inputs. How to fix or replace the inputs so that a match is possible.\u003C\u002Fp\u003E\n\u003Cp\u003E\u003Cem\u003EThe underlying genome sequence content may or may not be identical.\u003C\u002Fem\u003E Read method descriptions carefully to learn if that method is right for your usage case.\u003C\u002Fp\u003E\n\u003Cp\u003E\u003Cem\u003EMethod 5\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Cp\u003E\u003Cstrong\u003ESequence content is a match but adding \"chr\" is not enough to obtain an exact identifier match. You want to try to fix the identifiers anyway!!\u003C\u002Fstrong\u003E\u003C\u002Fp\u003E\n\u003Col\u003E\n\u003Cli\u003EIf the data is in a tabular format (BED, Interval, GTF -- with any headers removed first), and a suitable identifier mapping file can be obtained or created, the tool \u003Cem\u003EReplace column by values which are defined in a convert file\u003C\u002Fem\u003E can be used. Note that this will NOT work with BAM, VCF, Wiggle or other structured formats, as these are not tabular formatted data.\u003C\u002Fli\u003E\n\u003Cli\u003EManipuations with tools can often be used to split up a dataset, perform text substitutions and additions, concatinate datasets, and most other common operations one could do with command-line shell tools.\u003C\u002Fli\u003E\n\u003Cli\u003EThe dataset could also be downloaded locally to your computer and manipulated there using command-line tools or the text editor of choice.\u003C\u002Fli\u003E\n\u003C\u002Fol\u003E\n\u003Cp\u003E\u003Cem\u003EMethod 6\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Cp\u003E\u003Cstrong\u003ESequence content is a match but adding \"chr\" is not enough to obtain an exact identifier match. You DO NOT want to try to fix the identifiers or it is overly complicated or it is simply not possible to fix the data without an external reference mapping file (not always available).\u003C\u002Fstrong\u003E\u003C\u002Fp\u003E\n\u003Col\u003E\n\u003Cli\u003EObtain a reference annotation dataset that is a match for the reference genome used\u003C\u002Fli\u003E\n\u003Cli\u003ESometimes the source is the same for both\u003C\u002Fli\u003E\n\u003Cli\u003ESometimes the source is the same, but the content of the reference annotation is not ideal for the tools used\u003C\u002Fli\u003E\n\u003C\u002Fol\u003E\n\u003Cul\u003E\n\u003Cli\u003EExample: The tool Cuffdiff makes use of specific attributes in the reference annotation (p_id, tss_id, gene_name). If these attributes are not present in the GTF dataset, the resuls will not be fully annotated and some calculations will be skipped\u003C\u002Fli\u003E\n\u003Cli\u003EUse the iGenomes version of the reference annotation, as described below\u003C\u002Fli\u003E\n\u003Cli\u003EUsing Cuffdiff and the Gene ID is not present? Check your GTF file - the attribute gene_name is probably missing\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Col start=\"4\"\u003E\n\u003Cli\u003ESometimes the source can be \u003Cstrong\u003EiGenomes\u003C\u002Fstrong\u003E, which does contain the extra specific attributes needed for RNA-seq and certain other operationsar\u003C\u002Fli\u003E\n\u003C\u002Fol\u003E\n\u003Cul\u003E\n\u003Cli\u003EExample: The tool htseq-count is used and the attributes gene_id and transcript_id need to be distinct values (also true for the tool Cuffdiff for the best results)\u003C\u002Fli\u003E\n\u003Cli\u003ETwo sources: \u003Ca href=\"https:\u002F\u002Fsupport.illumina.com\u002Fsequencing\u002Fsequencing_software\u002Figenome.html\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttps:\u002F\u002Fsupport.illumina.com\u002Fsequencing\u002Fsequencing_software\u002Figenome.html\u003C\u002Fa\u003E and \u003Ca href=\"http:\u002F\u002Fcole-trapnell-lab.github.io\u002Fcufflinks\u002Figenome_table\u002Findex.html\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttp:\u002F\u002Fcole-trapnell-lab.github.io\u002Fcufflinks\u002Figenome_table\u002Findex.html\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003EDownload the .tar file locally, uncompress it, then upload just the genes.gtf dataset to Galaxy\u003C\u002Fli\u003E\n\u003Cli\u003ENote: the compression format .tar is not accepted by the Upload tool\u003C\u002Fli\u003E\n\u003Cli\u003EIf a .tar dataset is attempted to be uploaded, the load may fail or just the first file in the archive is uploaded (and it will not be the genes.gtf file)\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Col start=\"5\"\u003E\n\u003Cli\u003E\u003Ca href=\"https:\u002F\u002Fwww.gencodegenes.org\" target=\"_blank\" rel=\"noopener noreferrer\"\u003EGenecode Genes\u003C\u002Fa\u003E is also an annotation source for some genome builds.\u003C\u002Fli\u003E\n\u003Cli\u003EIn summary:\u003C\u002Fli\u003E\n\u003C\u002Fol\u003E\n\u003Cul\u003E\n\u003Cli\u003EFor \u003Cstrong\u003EGencode\u003C\u002Fstrong\u003E, copy the link to the GTF and paste it into the \u003Cem\u003EUpload\u003C\u002Fem\u003E tool. Hg38 data is here \u003Ca href=\"https:\u002F\u002Fwww.gencodegenes.org\u002F\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttps:\u002F\u002Fwww.gencodegenes.org\u002F\u003C\u002Fa\u003E. After it is loaded, remove the headers (lines that start with a \"#\") with the \u003Cem\u003ESelect\u003C\u002Fem\u003E tool using the options \"NOT Matching\" with the regular expression \u003Ccode\u003E^#\u003C\u002Fcode\u003E. Once the formatting is fixed, change the datatype to be \u003Ccode\u003Egft\u003C\u002Fcode\u003E under Edit Attributes (pencil icon). The data will be given the datatype \u003Ccode\u003Egff\u003C\u002Fcode\u003E by default, which works fine with some tools and but not with others. Avoid the \u003Ccode\u003Egff3\u003C\u002Fcode\u003E version of this particular data (contains duplicated IDs and several RNA-seq tools do not work with annotation in that format anyway).\u003C\u002Fli\u003E\n\u003Cli\u003EFor \u003Cstrong\u003EiGenomes\u003C\u002Fstrong\u003E, the archive corresponding to the target genome\u002Fbuild needs to be locally downloaded, the tar archive unpacked, and then just the \u003Ccode\u003Egenes.gtf\u003C\u002Fcode\u003E data uploaded to Galaxy (browse the local file, or use FTP). Find all available genome\u002Fbuilds here: \u003Ca href=\"https:\u002F\u002Fsupport.illumina.com\u002Fsequencing\u002Fsequencing_software\u002Figenome.html\" target=\"_blank\" rel=\"noopener noreferrer\"\u003Ehttps:\u002F\u002Fsupport.illumina.com\u002Fsequencing\u002Fsequencing_software\u002Figenome.html\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003Cp\u003E\u003Cem\u003EMethod 7\u003C\u002Fem\u003E\u003C\u002Fp\u003E\n\u003Cp\u003E\u003Cstrong\u003ESequence content is NOT a match or you want to try using a different reference genome instead of a different reference annotation source (reverse of \u003Cem\u003EMethod 6\u003C\u002Fem\u003E above).\u003C\u002Fstrong\u003E\u003C\u002Fp\u003E\n\u003Col\u003E\n\u003Cli\u003EMap against the same reference genome that the reference annotation is based on\u003C\u002Fli\u003E\n\u003Cli\u003EWhere and if this reference genome is available will depend on the genome build\u003C\u002Fli\u003E\n\u003Cli\u003EIn most cases, the source will be the same for both\u003C\u002Fli\u003E\n\u003Cli\u003EIf loading your own genome, make sure it is formatted correctly as a Custom Genome\u003C\u002Fli\u003E\n\u003Cli\u003EPromote the Custom Genome to a Custom Build and assign the genome\u002Fbuild metadata attribute to datasets\u003C\u002Fli\u003E\n\u003Cli\u003E\u003Ca href=\"\u002Flearn\u002Fcustom-genomes\u002F\"\u003ECustom Reference Genome help\u003C\u002Fa\u003E\u003C\u002Fli\u003E\n\u003Cli\u003EBe aware that if the genome is large, this option may result in a memory failure. Try \u003Cem\u003EMethod 2\u003C\u002Fem\u003E or consider moving to a local or cloud Galaxy where you can control the resources\u003C\u002Fli\u003E\n\u003C\u002Fol\u003E\n\u003Ch2 id=\"a-note-on-built-in-reference-genomes\"\u003E\u003Ca href=\"#a-note-on-built-in-reference-genomes\" aria-hidden=\"true\"\u003E\u003Cspan class=\"icon icon-link\"\u003E\u003C\u002Fspan\u003E\u003C\u002Fa\u003EA Note on Built-in Reference Genomes\u003C\u002Fh2\u003E\n\u003Cp\u003EThe default variant for all genomes is \"Full\", defined as all primary chromosomes (or scaffolds\u002Fcontigs) including mitochondrial plus associated unmapped, plasmid, and other segments.\u003C\u002Fp\u003E\n\u003Cp\u003EWhen only one version of a genome is available for a tool, it represents the default \"Full\" variant.\u003C\u002Fp\u003E\n\u003Cp\u003ESome genomes will have more than one variant available.\u003C\u002Fp\u003E\n\u003Cul\u003E\n\u003Cli\u003EThe \"Canonical Male\" or sometimes simply \"Canonical\" variant contains the primary chromosomes for a genome. For example a human \"Canonical\" variant contains chr1-chr22, chrX, chrY, and chrM.\u003C\u002Fli\u003E\n\u003Cli\u003EThe \"Canonical Female\" variant contains the primary chromosomes excluding chrY.\u003C\u002Fli\u003E\n\u003C\u002Ful\u003E\n\u003C\u002Fdiv\u003E\n"}},"context":{}};(function(){var s;(s=document.currentScript||document.scripts[document.scripts.length-1]).parentNode.removeChild(s);}());</script><script src="/assets/js/app.6a2146ae.js" defer></script><script src="/assets/js/page--src--templates--article-vue.59a31f82.js" defer></script>
  </body>
</html>
